<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (1.8.0_121) on Mon Feb 12 13:10:14 EST 2018 -->
<title>TargetedPcrMetrics (picard 2.17.9-SNAPSHOT API)</title>
<meta name="date" content="2018-02-12">
<link rel="stylesheet" type="text/css" href="../../../stylesheet.css" title="Style">
<script type="text/javascript" src="../../../script.js"></script>
</head>
<body>
<script type="text/javascript"><!--
    try {
        if (location.href.indexOf('is-external=true') == -1) {
            parent.document.title="TargetedPcrMetrics (picard 2.17.9-SNAPSHOT API)";
        }
    }
    catch(err) {
    }
//-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="topNav"><a name="navbar.top">
<!--   -->
</a>
<div class="skipNav"><a href="#skip.navbar.top" title="Skip navigation links">Skip navigation links</a></div>
<a name="navbar.top.firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../overview-summary.html">Overview</a></li>
<li><a href="package-summary.html">Package</a></li>
<li class="navBarCell1Rev">Class</li>
<li><a href="package-tree.html">Tree</a></li>
<li><a href="../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../index-all.html">Index</a></li>
<li><a href="../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li><a href="../../../picard/analysis/directed/RnaSeqMetricsCollector.StrandSpecificity.html" title="enum in picard.analysis.directed"><span class="typeNameLink">Prev&nbsp;Class</span></a></li>
<li><a href="../../../picard/analysis/directed/TargetedPcrMetricsCollector.html" title="class in picard.analysis.directed"><span class="typeNameLink">Next&nbsp;Class</span></a></li>
</ul>
<ul class="navList">
<li><a href="../../../index.html?picard/analysis/directed/TargetedPcrMetrics.html" target="_top">Frames</a></li>
<li><a href="TargetedPcrMetrics.html" target="_top">No&nbsp;Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_top">
<li><a href="../../../allclasses-noframe.html">All&nbsp;Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_top");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<div>
<ul class="subNavList">
<li>Summary:&nbsp;</li>
<li>Nested&nbsp;|&nbsp;</li>
<li><a href="#field.summary">Field</a>&nbsp;|&nbsp;</li>
<li><a href="#constructor.summary">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#methods.inherited.from.class.htsjdk.samtools.metrics.MetricBase">Method</a></li>
</ul>
<ul class="subNavList">
<li>Detail:&nbsp;</li>
<li><a href="#field.detail">Field</a>&nbsp;|&nbsp;</li>
<li><a href="#constructor.detail">Constr</a>&nbsp;|&nbsp;</li>
<li>Method</li>
</ul>
</div>
<a name="skip.navbar.top">
<!--   -->
</a></div>
<!-- ========= END OF TOP NAVBAR ========= -->
<!-- ======== START OF CLASS DATA ======== -->
<div class="header">
<div class="subTitle">picard.analysis.directed</div>
<h2 title="Class TargetedPcrMetrics" class="title">Class TargetedPcrMetrics</h2>
</div>
<div class="contentContainer">
<ul class="inheritance">
<li>java.lang.Object</li>
<li>
<ul class="inheritance">
<li>htsjdk.samtools.metrics.MetricBase</li>
<li>
<ul class="inheritance">
<li><a href="../../../picard/metrics/MultilevelMetrics.html" title="class in picard.metrics">picard.metrics.MultilevelMetrics</a></li>
<li>
<ul class="inheritance">
<li>picard.analysis.directed.TargetedPcrMetrics</li>
</ul>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<div class="description">
<ul class="blockList">
<li class="blockList">
<hr>
<br>
<pre>public class <span class="typeNameLabel">TargetedPcrMetrics</span>
extends <a href="../../../picard/metrics/MultilevelMetrics.html" title="class in picard.metrics">MultilevelMetrics</a></pre>
<div class="block">Metrics class for the analysis of reads obtained from targeted pcr experiments e.g. the TruSeq Custom Amplicon
 (TSCA) kit (Illumina).</div>
</li>
</ul>
</div>
<div class="summary">
<ul class="blockList">
<li class="blockList">
<!-- =========== FIELD SUMMARY =========== -->
<ul class="blockList">
<li class="blockList"><a name="field.summary">
<!--   -->
</a>
<h3>Field Summary</h3>
<table class="memberSummary" border="0" cellpadding="3" cellspacing="0" summary="Field Summary table, listing fields, and an explanation">
<caption><span>Fields</span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Field and Description</th>
</tr>
<tr class="altColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#AMPLICON_TERRITORY">AMPLICON_TERRITORY</a></span></code>
<div class="block">The number of unique bases covered by the intervals of all amplicons in the amplicon set</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#AT_DROPOUT">AT_DROPOUT</a></span></code>
<div class="block">A measure of how regions with low GC content (<= 50%), are undercovered relative to mean coverage.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.lang.String</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#CUSTOM_AMPLICON_SET">CUSTOM_AMPLICON_SET</a></span></code>
<div class="block">The name of the amplicon set used in this metrics collection run</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#FOLD_80_BASE_PENALTY">FOLD_80_BASE_PENALTY</a></span></code>
<div class="block">The fold over-coverage necessary to raise 80% of bases in "non-zero-cvg" targets to
 the mean coverage level in those targets.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#FOLD_ENRICHMENT">FOLD_ENRICHMENT</a></span></code>
<div class="block">The fold by which the amplicon region has been amplified above genomic background.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#GC_DROPOUT">GC_DROPOUT</a></span></code>
<div class="block">A measure of how regions of high GC content (>= 50% GC) are undercovered relative to the mean coverage
 value.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#GENOME_SIZE">GENOME_SIZE</a></span></code>
<div class="block">The number of bases in the reference genome used for alignment</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#HET_SNP_Q">HET_SNP_Q</a></span></code>
<div class="block">The Q Score of the theoretical HET SNP sensitivity.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#HET_SNP_SENSITIVITY">HET_SNP_SENSITIVITY</a></span></code>
<div class="block">The theoretical HET SNP sensitivity.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#MAX_TARGET_COVERAGE">MAX_TARGET_COVERAGE</a></span></code>
<div class="block">The maximum coverage of reads that mapped to target regions of an experiment.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#MEAN_AMPLICON_COVERAGE">MEAN_AMPLICON_COVERAGE</a></span></code>
<div class="block">The mean read coverage of all amplicon regions in the experiment.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#MEAN_TARGET_COVERAGE">MEAN_TARGET_COVERAGE</a></span></code>
<div class="block">The mean read coverage of all target regions in an experiment.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#MEDIAN_TARGET_COVERAGE">MEDIAN_TARGET_COVERAGE</a></span></code>
<div class="block">The median coverage of reads that mapped to target regions of an experiment.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#NEAR_AMPLICON_BASES">NEAR_AMPLICON_BASES</a></span></code>
<div class="block">The number of PF_BASES_ALIGNED that mapped to within a fixed interval of an amplified region, but not on a
 baited region.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#OFF_AMPLICON_BASES">OFF_AMPLICON_BASES</a></span></code>
<div class="block">The number of PF_BASES_ALIGNED that mapped neither on or near an amplicon.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#ON_AMPLICON_BASES">ON_AMPLICON_BASES</a></span></code>
<div class="block">The number of PF_BASES_ALIGNED that mapped to an amplified region of the genome.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#ON_AMPLICON_VS_SELECTED">ON_AMPLICON_VS_SELECTED</a></span></code>
<div class="block">The fraction of bases mapping to regions on or near amplicons, which mapped directly to but not near
 amplicons, ON_AMPLICON_BASES/(NEAR_AMPLICON_BASES + ON_AMPLICON_BASES)</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#ON_TARGET_BASES">ON_TARGET_BASES</a></span></code>
<div class="block">The number of PF_BASES_ALIGNED that mapped to a targeted region of the genome.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#ON_TARGET_FROM_PAIR_BASES">ON_TARGET_FROM_PAIR_BASES</a></span></code>
<div class="block">The number of bases from PF_SELECTED_UNIQUE_PAIRS that mapped to a targeted region of the genome.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#PCT_AMPLIFIED_BASES">PCT_AMPLIFIED_BASES</a></span></code>
<div class="block">The fraction of PF_BASES_ALIGNED that mapped to or near an amplicon, (ON_AMPLICON_BASES +
 NEAR_AMPLICON_BASES)/PF_BASES_ALIGNED.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#PCT_EXC_BASEQ">PCT_EXC_BASEQ</a></span></code>
<div class="block">The fraction of aligned bases that were filtered out because they were of low base quality.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#PCT_EXC_DUPE">PCT_EXC_DUPE</a></span></code>
<div class="block">The fraction of aligned bases that were filtered out because they were in reads marked as duplicates.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#PCT_EXC_MAPQ">PCT_EXC_MAPQ</a></span></code>
<div class="block">The fraction of aligned bases that were filtered out because they were in reads with low mapping quality.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#PCT_EXC_OFF_TARGET">PCT_EXC_OFF_TARGET</a></span></code>
<div class="block">The fraction of bases that were filtered out because they did not map to a base within a target region.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#PCT_EXC_OVERLAP">PCT_EXC_OVERLAP</a></span></code>
<div class="block">The fraction of aligned bases that were filtered out because they were the second observation from an
 insert with overlapping reads.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#PCT_OFF_AMPLICON">PCT_OFF_AMPLICON</a></span></code>
<div class="block">The fraction of PF_BASES_ALIGNED that mapped neither onto or near an amplicon,
 OFF_AMPLICON_BASES/PF_BASES_ALIGNED</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#PCT_PF_READS">PCT_PF_READS</a></span></code>
<div class="block">The fraction of reads passing filter, PF_READS/TOTAL_READS.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#PCT_PF_UQ_READS">PCT_PF_UQ_READS</a></span></code>
<div class="block">The fraction of TOTAL_READS that are unique, PF, and are not duplicates, PF_UNIQUE_READS/TOTAL_READS</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#PCT_PF_UQ_READS_ALIGNED">PCT_PF_UQ_READS_ALIGNED</a></span></code>
<div class="block">Fraction of PF_READS that are unique and align to the reference genome, PF_UQ_READS_ALIGNED/PF_READS</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#PCT_TARGET_BASES_10X">PCT_TARGET_BASES_10X</a></span></code>
<div class="block">The fraction of all target bases achieving 10X or greater coverage depth.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#PCT_TARGET_BASES_1X">PCT_TARGET_BASES_1X</a></span></code>
<div class="block">The fraction of all target bases achieving 1X or greater coverage.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#PCT_TARGET_BASES_20X">PCT_TARGET_BASES_20X</a></span></code>
<div class="block">The fraction of all target bases achieving 20X or greater coverage depth.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#PCT_TARGET_BASES_2X">PCT_TARGET_BASES_2X</a></span></code>
<div class="block">The fraction of all target bases achieving 2X or greater coverage depth.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#PCT_TARGET_BASES_30X">PCT_TARGET_BASES_30X</a></span></code>
<div class="block">The fraction of all target bases achieving 30X or greater coverage depth.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#PF_BASES">PF_BASES</a></span></code>
<div class="block">The total number of bases within the PF_READS of the SAM or BAM file to be examined</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#PF_BASES_ALIGNED">PF_BASES_ALIGNED</a></span></code>
<div class="block">The number of bases from PF_READS that align to the reference genome with mapping score > 0</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#PF_READS">PF_READS</a></span></code>
<div class="block">The total number of reads passing filter (PF), where the filter(s) can be platform/vendor quality controls</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#PF_SELECTED_PAIRS">PF_SELECTED_PAIRS</a></span></code>
<div class="block">Tracks the number of PF read pairs (used to calculate library size)</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#PF_SELECTED_UNIQUE_PAIRS">PF_SELECTED_UNIQUE_PAIRS</a></span></code>
<div class="block">Tracks the number of unique, PF, read pairs, observed (used to calculate library size)</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#PF_UNIQUE_READS">PF_UNIQUE_READS</a></span></code>
<div class="block">The number of PF_READS that were not marked as sample or optical duplicates.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#PF_UQ_BASES_ALIGNED">PF_UQ_BASES_ALIGNED</a></span></code>
<div class="block">The number of bases from PF_UNIQUE_READS that align to the reference genome and have a mapping score > 0</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#PF_UQ_READS_ALIGNED">PF_UQ_READS_ALIGNED</a></span></code>
<div class="block">The total number of PF_UNIQUE_READS that align to the reference genome with mapping scores > 0</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#TARGET_TERRITORY">TARGET_TERRITORY</a></span></code>
<div class="block">The number of unique bases covered by the intervals of all targets that should be covered</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#TOTAL_READS">TOTAL_READS</a></span></code>
<div class="block">The total number of reads in the SAM or BAM file examined</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#ZERO_CVG_TARGETS_PCT">ZERO_CVG_TARGETS_PCT</a></span></code>
<div class="block">The fraction of targets that did not reach coverage=1 over any base.</div>
</td>
</tr>
</table>
<ul class="blockList">
<li class="blockList"><a name="fields.inherited.from.class.picard.metrics.MultilevelMetrics">
<!--   -->
</a>
<h3>Fields inherited from class&nbsp;picard.metrics.<a href="../../../picard/metrics/MultilevelMetrics.html" title="class in picard.metrics">MultilevelMetrics</a></h3>
<code><a href="../../../picard/metrics/MultilevelMetrics.html#LIBRARY">LIBRARY</a>, <a href="../../../picard/metrics/MultilevelMetrics.html#READ_GROUP">READ_GROUP</a>, <a href="../../../picard/metrics/MultilevelMetrics.html#SAMPLE">SAMPLE</a></code></li>
</ul>
</li>
</ul>
<!-- ======== CONSTRUCTOR SUMMARY ======== -->
<ul class="blockList">
<li class="blockList"><a name="constructor.summary">
<!--   -->
</a>
<h3>Constructor Summary</h3>
<table class="memberSummary" border="0" cellpadding="3" cellspacing="0" summary="Constructor Summary table, listing constructors, and an explanation">
<caption><span>Constructors</span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Constructor and Description</th>
</tr>
<tr class="altColor">
<td class="colOne"><code><span class="memberNameLink"><a href="../../../picard/analysis/directed/TargetedPcrMetrics.html#TargetedPcrMetrics--">TargetedPcrMetrics</a></span>()</code>&nbsp;</td>
</tr>
</table>
</li>
</ul>
<!-- ========== METHOD SUMMARY =========== -->
<ul class="blockList">
<li class="blockList"><a name="method.summary">
<!--   -->
</a>
<h3>Method Summary</h3>
<ul class="blockList">
<li class="blockList"><a name="methods.inherited.from.class.htsjdk.samtools.metrics.MetricBase">
<!--   -->
</a>
<h3>Methods inherited from class&nbsp;htsjdk.samtools.metrics.MetricBase</h3>
<code>equals, hashCode, toString</code></li>
</ul>
<ul class="blockList">
<li class="blockList"><a name="methods.inherited.from.class.java.lang.Object">
<!--   -->
</a>
<h3>Methods inherited from class&nbsp;java.lang.Object</h3>
<code>clone, finalize, getClass, notify, notifyAll, wait, wait, wait</code></li>
</ul>
</li>
</ul>
</li>
</ul>
</div>
<div class="details">
<ul class="blockList">
<li class="blockList">
<!-- ============ FIELD DETAIL =========== -->
<ul class="blockList">
<li class="blockList"><a name="field.detail">
<!--   -->
</a>
<h3>Field Detail</h3>
<a name="CUSTOM_AMPLICON_SET">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>CUSTOM_AMPLICON_SET</h4>
<pre>public&nbsp;java.lang.String CUSTOM_AMPLICON_SET</pre>
<div class="block">The name of the amplicon set used in this metrics collection run</div>
</li>
</ul>
<a name="GENOME_SIZE">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>GENOME_SIZE</h4>
<pre>public&nbsp;long GENOME_SIZE</pre>
<div class="block">The number of bases in the reference genome used for alignment</div>
</li>
</ul>
<a name="AMPLICON_TERRITORY">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>AMPLICON_TERRITORY</h4>
<pre>public&nbsp;long AMPLICON_TERRITORY</pre>
<div class="block">The number of unique bases covered by the intervals of all amplicons in the amplicon set</div>
</li>
</ul>
<a name="TARGET_TERRITORY">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>TARGET_TERRITORY</h4>
<pre>public&nbsp;long TARGET_TERRITORY</pre>
<div class="block">The number of unique bases covered by the intervals of all targets that should be covered</div>
</li>
</ul>
<a name="TOTAL_READS">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>TOTAL_READS</h4>
<pre>public&nbsp;long TOTAL_READS</pre>
<div class="block">The total number of reads in the SAM or BAM file examined</div>
</li>
</ul>
<a name="PF_READS">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PF_READS</h4>
<pre>public&nbsp;long PF_READS</pre>
<div class="block">The total number of reads passing filter (PF), where the filter(s) can be platform/vendor quality controls</div>
</li>
</ul>
<a name="PF_BASES">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PF_BASES</h4>
<pre>public&nbsp;long PF_BASES</pre>
<div class="block">The total number of bases within the PF_READS of the SAM or BAM file to be examined</div>
</li>
</ul>
<a name="PF_UNIQUE_READS">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PF_UNIQUE_READS</h4>
<pre>public&nbsp;long PF_UNIQUE_READS</pre>
<div class="block">The number of PF_READS that were not marked as sample or optical duplicates.</div>
</li>
</ul>
<a name="PCT_PF_READS">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PCT_PF_READS</h4>
<pre>public&nbsp;double PCT_PF_READS</pre>
<div class="block">The fraction of reads passing filter, PF_READS/TOTAL_READS.</div>
</li>
</ul>
<a name="PCT_PF_UQ_READS">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PCT_PF_UQ_READS</h4>
<pre>public&nbsp;double PCT_PF_UQ_READS</pre>
<div class="block">The fraction of TOTAL_READS that are unique, PF, and are not duplicates, PF_UNIQUE_READS/TOTAL_READS</div>
</li>
</ul>
<a name="PF_UQ_READS_ALIGNED">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PF_UQ_READS_ALIGNED</h4>
<pre>public&nbsp;long PF_UQ_READS_ALIGNED</pre>
<div class="block">The total number of PF_UNIQUE_READS that align to the reference genome with mapping scores > 0</div>
</li>
</ul>
<a name="PF_SELECTED_PAIRS">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PF_SELECTED_PAIRS</h4>
<pre>public&nbsp;long PF_SELECTED_PAIRS</pre>
<div class="block">Tracks the number of PF read pairs (used to calculate library size)</div>
</li>
</ul>
<a name="PF_SELECTED_UNIQUE_PAIRS">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PF_SELECTED_UNIQUE_PAIRS</h4>
<pre>public&nbsp;long PF_SELECTED_UNIQUE_PAIRS</pre>
<div class="block">Tracks the number of unique, PF, read pairs, observed (used to calculate library size)</div>
</li>
</ul>
<a name="PCT_PF_UQ_READS_ALIGNED">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PCT_PF_UQ_READS_ALIGNED</h4>
<pre>public&nbsp;double PCT_PF_UQ_READS_ALIGNED</pre>
<div class="block">Fraction of PF_READS that are unique and align to the reference genome, PF_UQ_READS_ALIGNED/PF_READS</div>
</li>
</ul>
<a name="PF_BASES_ALIGNED">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PF_BASES_ALIGNED</h4>
<pre>public&nbsp;long PF_BASES_ALIGNED</pre>
<div class="block">The number of bases from PF_READS that align to the reference genome with mapping score > 0</div>
</li>
</ul>
<a name="PF_UQ_BASES_ALIGNED">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PF_UQ_BASES_ALIGNED</h4>
<pre>public&nbsp;long PF_UQ_BASES_ALIGNED</pre>
<div class="block">The number of bases from PF_UNIQUE_READS that align to the reference genome and have a mapping score > 0</div>
</li>
</ul>
<a name="ON_AMPLICON_BASES">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>ON_AMPLICON_BASES</h4>
<pre>public&nbsp;long ON_AMPLICON_BASES</pre>
<div class="block">The number of PF_BASES_ALIGNED that mapped to an amplified region of the genome.</div>
</li>
</ul>
<a name="NEAR_AMPLICON_BASES">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>NEAR_AMPLICON_BASES</h4>
<pre>public&nbsp;long NEAR_AMPLICON_BASES</pre>
<div class="block">The number of PF_BASES_ALIGNED that mapped to within a fixed interval of an amplified region, but not on a
 baited region.</div>
</li>
</ul>
<a name="OFF_AMPLICON_BASES">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>OFF_AMPLICON_BASES</h4>
<pre>public&nbsp;long OFF_AMPLICON_BASES</pre>
<div class="block">The number of PF_BASES_ALIGNED that mapped neither on or near an amplicon.</div>
</li>
</ul>
<a name="ON_TARGET_BASES">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>ON_TARGET_BASES</h4>
<pre>public&nbsp;long ON_TARGET_BASES</pre>
<div class="block">The number of PF_BASES_ALIGNED that mapped to a targeted region of the genome.</div>
</li>
</ul>
<a name="ON_TARGET_FROM_PAIR_BASES">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>ON_TARGET_FROM_PAIR_BASES</h4>
<pre>public&nbsp;long ON_TARGET_FROM_PAIR_BASES</pre>
<div class="block">The number of bases from PF_SELECTED_UNIQUE_PAIRS that mapped to a targeted region of the genome.</div>
</li>
</ul>
<a name="PCT_AMPLIFIED_BASES">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PCT_AMPLIFIED_BASES</h4>
<pre>public&nbsp;double PCT_AMPLIFIED_BASES</pre>
<div class="block">The fraction of PF_BASES_ALIGNED that mapped to or near an amplicon, (ON_AMPLICON_BASES +
 NEAR_AMPLICON_BASES)/PF_BASES_ALIGNED.</div>
</li>
</ul>
<a name="PCT_OFF_AMPLICON">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PCT_OFF_AMPLICON</h4>
<pre>public&nbsp;double PCT_OFF_AMPLICON</pre>
<div class="block">The fraction of PF_BASES_ALIGNED that mapped neither onto or near an amplicon,
 OFF_AMPLICON_BASES/PF_BASES_ALIGNED</div>
</li>
</ul>
<a name="ON_AMPLICON_VS_SELECTED">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>ON_AMPLICON_VS_SELECTED</h4>
<pre>public&nbsp;double ON_AMPLICON_VS_SELECTED</pre>
<div class="block">The fraction of bases mapping to regions on or near amplicons, which mapped directly to but not near
 amplicons, ON_AMPLICON_BASES/(NEAR_AMPLICON_BASES + ON_AMPLICON_BASES)</div>
</li>
</ul>
<a name="MEAN_AMPLICON_COVERAGE">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>MEAN_AMPLICON_COVERAGE</h4>
<pre>public&nbsp;double MEAN_AMPLICON_COVERAGE</pre>
<div class="block">The mean read coverage of all amplicon regions in the experiment.</div>
</li>
</ul>
<a name="MEAN_TARGET_COVERAGE">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>MEAN_TARGET_COVERAGE</h4>
<pre>public&nbsp;double MEAN_TARGET_COVERAGE</pre>
<div class="block">The mean read coverage of all target regions in an experiment.</div>
</li>
</ul>
<a name="MEDIAN_TARGET_COVERAGE">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>MEDIAN_TARGET_COVERAGE</h4>
<pre>public&nbsp;double MEDIAN_TARGET_COVERAGE</pre>
<div class="block">The median coverage of reads that mapped to target regions of an experiment.</div>
</li>
</ul>
<a name="MAX_TARGET_COVERAGE">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>MAX_TARGET_COVERAGE</h4>
<pre>public&nbsp;long MAX_TARGET_COVERAGE</pre>
<div class="block">The maximum coverage of reads that mapped to target regions of an experiment.</div>
</li>
</ul>
<a name="FOLD_ENRICHMENT">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>FOLD_ENRICHMENT</h4>
<pre>public&nbsp;double FOLD_ENRICHMENT</pre>
<div class="block">The fold by which the amplicon region has been amplified above genomic background.</div>
</li>
</ul>
<a name="ZERO_CVG_TARGETS_PCT">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>ZERO_CVG_TARGETS_PCT</h4>
<pre>public&nbsp;double ZERO_CVG_TARGETS_PCT</pre>
<div class="block">The fraction of targets that did not reach coverage=1 over any base.</div>
</li>
</ul>
<a name="PCT_EXC_DUPE">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PCT_EXC_DUPE</h4>
<pre>public&nbsp;double PCT_EXC_DUPE</pre>
<div class="block">The fraction of aligned bases that were filtered out because they were in reads marked as duplicates.</div>
</li>
</ul>
<a name="PCT_EXC_MAPQ">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PCT_EXC_MAPQ</h4>
<pre>public&nbsp;double PCT_EXC_MAPQ</pre>
<div class="block">The fraction of aligned bases that were filtered out because they were in reads with low mapping quality.</div>
</li>
</ul>
<a name="PCT_EXC_BASEQ">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PCT_EXC_BASEQ</h4>
<pre>public&nbsp;double PCT_EXC_BASEQ</pre>
<div class="block">The fraction of aligned bases that were filtered out because they were of low base quality.</div>
</li>
</ul>
<a name="PCT_EXC_OVERLAP">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PCT_EXC_OVERLAP</h4>
<pre>public&nbsp;double PCT_EXC_OVERLAP</pre>
<div class="block">The fraction of aligned bases that were filtered out because they were the second observation from an
 insert with overlapping reads.</div>
</li>
</ul>
<a name="PCT_EXC_OFF_TARGET">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PCT_EXC_OFF_TARGET</h4>
<pre>public&nbsp;double PCT_EXC_OFF_TARGET</pre>
<div class="block">The fraction of bases that were filtered out because they did not map to a base within a target region.</div>
</li>
</ul>
<a name="FOLD_80_BASE_PENALTY">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>FOLD_80_BASE_PENALTY</h4>
<pre>public&nbsp;double FOLD_80_BASE_PENALTY</pre>
<div class="block">The fold over-coverage necessary to raise 80% of bases in "non-zero-cvg" targets to
 the mean coverage level in those targets.</div>
</li>
</ul>
<a name="PCT_TARGET_BASES_1X">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PCT_TARGET_BASES_1X</h4>
<pre>public&nbsp;double PCT_TARGET_BASES_1X</pre>
<div class="block">The fraction of all target bases achieving 1X or greater coverage.</div>
</li>
</ul>
<a name="PCT_TARGET_BASES_2X">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PCT_TARGET_BASES_2X</h4>
<pre>public&nbsp;double PCT_TARGET_BASES_2X</pre>
<div class="block">The fraction of all target bases achieving 2X or greater coverage depth.</div>
</li>
</ul>
<a name="PCT_TARGET_BASES_10X">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PCT_TARGET_BASES_10X</h4>
<pre>public&nbsp;double PCT_TARGET_BASES_10X</pre>
<div class="block">The fraction of all target bases achieving 10X or greater coverage depth.</div>
</li>
</ul>
<a name="PCT_TARGET_BASES_20X">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PCT_TARGET_BASES_20X</h4>
<pre>public&nbsp;double PCT_TARGET_BASES_20X</pre>
<div class="block">The fraction of all target bases achieving 20X or greater coverage depth.</div>
</li>
</ul>
<a name="PCT_TARGET_BASES_30X">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PCT_TARGET_BASES_30X</h4>
<pre>public&nbsp;double PCT_TARGET_BASES_30X</pre>
<div class="block">The fraction of all target bases achieving 30X or greater coverage depth.</div>
</li>
</ul>
<a name="AT_DROPOUT">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>AT_DROPOUT</h4>
<pre>public&nbsp;double AT_DROPOUT</pre>
<div class="block">A measure of how regions with low GC content (<= 50%), are undercovered relative to mean coverage.
 After binning the GC content [0..50], we calculate a = fraction of target territory, and b = fraction of
 aligned reads aligned to these targets for each bin.  AT DROPOUT is then abs(sum(a-b when a-b < 0)).
 For example, if the AT_DROPOUT value is 5% this implies that 5% of total reads that
 should have mapped to GC<=50% regions, mapped elsewhere.</div>
</li>
</ul>
<a name="GC_DROPOUT">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>GC_DROPOUT</h4>
<pre>public&nbsp;double GC_DROPOUT</pre>
<div class="block">A measure of how regions of high GC content (>= 50% GC) are undercovered relative to the mean coverage
 value. For each GC bin [50..100], we calculate a = % of target territory, and b = % of aligned reads aligned
 to these targets.  GC DROPOUT is then abs(sum(a-b when a-b < 0)).  For example, if the value is 5%, this
 implies that 5% of total reads that should have mapped to GC>=50% regions, mapped elsewhere.</div>
</li>
</ul>
<a name="HET_SNP_SENSITIVITY">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>HET_SNP_SENSITIVITY</h4>
<pre>public&nbsp;double HET_SNP_SENSITIVITY</pre>
<div class="block">The theoretical HET SNP sensitivity.</div>
</li>
</ul>
<a name="HET_SNP_Q">
<!--   -->
</a>
<ul class="blockListLast">
<li class="blockList">
<h4>HET_SNP_Q</h4>
<pre>public&nbsp;double HET_SNP_Q</pre>
<div class="block">The Q Score of the theoretical HET SNP sensitivity.</div>
</li>
</ul>
</li>
</ul>
<!-- ========= CONSTRUCTOR DETAIL ======== -->
<ul class="blockList">
<li class="blockList"><a name="constructor.detail">
<!--   -->
</a>
<h3>Constructor Detail</h3>
<a name="TargetedPcrMetrics--">
<!--   -->
</a>
<ul class="blockListLast">
<li class="blockList">
<h4>TargetedPcrMetrics</h4>
<pre>public&nbsp;TargetedPcrMetrics()</pre>
</li>
</ul>
</li>
</ul>
</li>
</ul>
</div>
</div>
<!-- ========= END OF CLASS DATA ========= -->
<!-- ======= START OF BOTTOM NAVBAR ====== -->
<div class="bottomNav"><a name="navbar.bottom">
<!--   -->
</a>
<div class="skipNav"><a href="#skip.navbar.bottom" title="Skip navigation links">Skip navigation links</a></div>
<a name="navbar.bottom.firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../overview-summary.html">Overview</a></li>
<li><a href="package-summary.html">Package</a></li>
<li class="navBarCell1Rev">Class</li>
<li><a href="package-tree.html">Tree</a></li>
<li><a href="../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../index-all.html">Index</a></li>
<li><a href="../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li><a href="../../../picard/analysis/directed/RnaSeqMetricsCollector.StrandSpecificity.html" title="enum in picard.analysis.directed"><span class="typeNameLink">Prev&nbsp;Class</span></a></li>
<li><a href="../../../picard/analysis/directed/TargetedPcrMetricsCollector.html" title="class in picard.analysis.directed"><span class="typeNameLink">Next&nbsp;Class</span></a></li>
</ul>
<ul class="navList">
<li><a href="../../../index.html?picard/analysis/directed/TargetedPcrMetrics.html" target="_top">Frames</a></li>
<li><a href="TargetedPcrMetrics.html" target="_top">No&nbsp;Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_bottom">
<li><a href="../../../allclasses-noframe.html">All&nbsp;Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_bottom");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<div>
<ul class="subNavList">
<li>Summary:&nbsp;</li>
<li>Nested&nbsp;|&nbsp;</li>
<li><a href="#field.summary">Field</a>&nbsp;|&nbsp;</li>
<li><a href="#constructor.summary">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#methods.inherited.from.class.htsjdk.samtools.metrics.MetricBase">Method</a></li>
</ul>
<ul class="subNavList">
<li>Detail:&nbsp;</li>
<li><a href="#field.detail">Field</a>&nbsp;|&nbsp;</li>
<li><a href="#constructor.detail">Constr</a>&nbsp;|&nbsp;</li>
<li>Method</li>
</ul>
</div>
<a name="skip.navbar.bottom">
<!--   -->
</a></div>
<!-- ======== END OF BOTTOM NAVBAR ======= -->
</body>
</html>
