<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (1.8.0_77) on Tue Dec 12 15:39:49 EST 2017 -->
<title>Index (picard 2.14.0-5-g441d28a-SNAPSHOT API)</title>
<meta name="date" content="2017-12-12">
<link rel="stylesheet" type="text/css" href="stylesheet.css" title="Style">
<script type="text/javascript" src="script.js"></script>
</head>
<body>
<script type="text/javascript"><!--
    try {
        if (location.href.indexOf('is-external=true') == -1) {
            parent.document.title="Index (picard 2.14.0-5-g441d28a-SNAPSHOT API)";
        }
    }
    catch(err) {
    }
//-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="topNav"><a name="navbar.top">
<!--   -->
</a>
<div class="skipNav"><a href="#skip.navbar.top" title="Skip navigation links">Skip navigation links</a></div>
<a name="navbar.top.firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="overview-summary.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li><a href="overview-tree.html">Tree</a></li>
<li><a href="deprecated-list.html">Deprecated</a></li>
<li class="navBarCell1Rev">Index</li>
<li><a href="help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev</li>
<li>Next</li>
</ul>
<ul class="navList">
<li><a href="index.html?index-all.html" target="_top">Frames</a></li>
<li><a href="index-all.html" target="_top">No&nbsp;Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_top">
<li><a href="allclasses-noframe.html">All&nbsp;Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_top");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip.navbar.top">
<!--   -->
</a></div>
<!-- ========= END OF TOP NAVBAR ========= -->
<div class="contentContainer"><a href="#I:A">A</a>&nbsp;<a href="#I:B">B</a>&nbsp;<a href="#I:C">C</a>&nbsp;<a href="#I:D">D</a>&nbsp;<a href="#I:E">E</a>&nbsp;<a href="#I:F">F</a>&nbsp;<a href="#I:G">G</a>&nbsp;<a href="#I:H">H</a>&nbsp;<a href="#I:I">I</a>&nbsp;<a href="#I:J">J</a>&nbsp;<a href="#I:L">L</a>&nbsp;<a href="#I:M">M</a>&nbsp;<a href="#I:N">N</a>&nbsp;<a href="#I:O">O</a>&nbsp;<a href="#I:P">P</a>&nbsp;<a href="#I:Q">Q</a>&nbsp;<a href="#I:R">R</a>&nbsp;<a href="#I:S">S</a>&nbsp;<a href="#I:T">T</a>&nbsp;<a href="#I:U">U</a>&nbsp;<a href="#I:V">V</a>&nbsp;<a href="#I:W">W</a>&nbsp;<a href="#I:X">X</a>&nbsp;<a href="#I:Y">Y</a>&nbsp;<a href="#I:Z">Z</a>&nbsp;<a name="I:A">
<!--   -->
</a>
<h2 class="title">A</h2>
<dl>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/AlleleBalanceFilter.html#AB_FILTER">AB_FILTER</a></span> - Static variable in class picard.vcf.filter.<a href="picard/vcf/filter/AlleleBalanceFilter.html" title="class in picard.vcf.filter">AlleleBalanceFilter</a></dt>
<dd>
<div class="block">The filter string used for sites that fail the allele balance filter.</div>
</dd>
<dt><a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam"><span class="typeNameLink">AbstractAlignmentMerger</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block">Abstract class that coordinates the general task of taking in a set of alignment information,
 possibly in SAM format, possibly in other formats, and merging that with the set of all reads
 for which alignment was attempted, stored in an unmapped SAM file.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.html#AbstractAlignmentMerger-java.io.File-java.io.File-java.io.File-boolean-boolean-boolean-htsjdk.samtools.SAMProgramRecord-java.util.List-java.util.List-java.lang.Integer-java.lang.Integer-java.util.List-htsjdk.samtools.SAMFileHeader.SortOrder-picard.sam.PrimaryAlignmentSelectionStrategy-boolean-boolean-">AbstractAlignmentMerger(File, File, File, boolean, boolean, boolean, SAMProgramRecord, List&lt;String&gt;, List&lt;String&gt;, Integer, Integer, List&lt;SamPairUtil.PairOrientation&gt;, SAMFileHeader.SortOrder, PrimaryAlignmentSelectionStrategy, boolean, boolean)</a></span> - Constructor for class picard.sam.<a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</a></dt>
<dd>
<div class="block">constructor with a default setting for unmappingReadsStrategy.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.html#AbstractAlignmentMerger-java.io.File-java.io.File-java.io.File-boolean-boolean-boolean-htsjdk.samtools.SAMProgramRecord-java.util.List-java.util.List-java.lang.Integer-java.lang.Integer-java.util.List-htsjdk.samtools.SAMFileHeader.SortOrder-picard.sam.PrimaryAlignmentSelectionStrategy-boolean-boolean-picard.sam.AbstractAlignmentMerger.UnmappingReadStrategy-">AbstractAlignmentMerger(File, File, File, boolean, boolean, boolean, SAMProgramRecord, List&lt;String&gt;, List&lt;String&gt;, Integer, Integer, List&lt;SamPairUtil.PairOrientation&gt;, SAMFileHeader.SortOrder, PrimaryAlignmentSelectionStrategy, boolean, boolean, AbstractAlignmentMerger.UnmappingReadStrategy)</a></span> - Constructor for class picard.sam.<a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</a></dt>
<dd>
<div class="block">Constructor</div>
</dd>
<dt><a href="picard/sam/AbstractAlignmentMerger.UnmappingReadStrategy.html" title="enum in picard.sam"><span class="typeNameLink">AbstractAlignmentMerger.UnmappingReadStrategy</span></a> - Enum in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.html" title="class in picard.illumina.parser.readers"><span class="typeNameLink">AbstractIlluminaPositionFileReader</span></a> - Class in <a href="picard/illumina/parser/readers/package-summary.html">picard.illumina.parser.readers</a></dt>
<dd>
<div class="block">The position files of Illumina are nearly the same form:  Pos files consist of text based tabbed
 x-y coordinate float pairs, locs files are binary x-y float pairs, clocs are compressed binary
 x-y float pairs.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.html#AbstractIlluminaPositionFileReader-java.io.File-">AbstractIlluminaPositionFileReader(File)</a></span> - Constructor for class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.html" title="class in picard.illumina.parser.readers">AbstractIlluminaPositionFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.html#AbstractIlluminaPositionFileReader-java.io.File-int-int-">AbstractIlluminaPositionFileReader(File, int, int)</a></span> - Constructor for class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.html" title="class in picard.illumina.parser.readers">AbstractIlluminaPositionFileReader</a></dt>
<dd>
<div class="block">Use this ctor if lane and tile are not discernible from file name.</div>
</dd>
<dt><a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.PositionInfo.html" title="class in picard.illumina.parser.readers"><span class="typeNameLink">AbstractIlluminaPositionFileReader.PositionInfo</span></a> - Class in <a href="picard/illumina/parser/readers/package-summary.html">picard.illumina.parser.readers</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/AbstractInputParser.html" title="class in picard.util"><span class="typeNameLink">AbstractInputParser</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">Class for parsing text files where each line consists of fields separated by whitespace.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/AbstractInputParser.html#AbstractInputParser--">AbstractInputParser()</a></span> - Constructor for class picard.util.<a href="picard/util/AbstractInputParser.html" title="class in picard.util">AbstractInputParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html" title="class in picard.sam.markduplicates.util"><span class="typeNameLink">AbstractMarkDuplicatesCommandLineProgram</span></a> - Class in <a href="picard/sam/markduplicates/util/package-summary.html">picard.sam.markduplicates.util</a></dt>
<dd>
<div class="block">Abstract class that holds parameters and methods common to classes that perform duplicate
 detection and/or marking within SAM/BAM files.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html#AbstractMarkDuplicatesCommandLineProgram--">AbstractMarkDuplicatesCommandLineProgram()</a></span> - Constructor for class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html" title="class in picard.sam.markduplicates.util">AbstractMarkDuplicatesCommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.SamHeaderAndIterator.html" title="class in picard.sam.markduplicates.util"><span class="typeNameLink">AbstractMarkDuplicatesCommandLineProgram.SamHeaderAndIterator</span></a> - Class in <a href="picard/sam/markduplicates/util/package-summary.html">picard.sam.markduplicates.util</a></dt>
<dd>
<div class="block">Little class used to package up a header and an iterable/iterator.</div>
</dd>
<dt><a href="picard/sam/markduplicates/util/AbstractOpticalDuplicateFinderCommandLineProgram.html" title="class in picard.sam.markduplicates.util"><span class="typeNameLink">AbstractOpticalDuplicateFinderCommandLineProgram</span></a> - Class in <a href="picard/sam/markduplicates/util/package-summary.html">picard.sam.markduplicates.util</a></dt>
<dd>
<div class="block">Abstract class that holds parameters and methods common to classes that optical duplicate detection.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/AbstractOpticalDuplicateFinderCommandLineProgram.html#AbstractOpticalDuplicateFinderCommandLineProgram--">AbstractOpticalDuplicateFinderCommandLineProgram()</a></span> - Constructor for class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/AbstractOpticalDuplicateFinderCommandLineProgram.html" title="class in picard.sam.markduplicates.util">AbstractOpticalDuplicateFinderCommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/AbstractWgsMetricsCollector.html" title="class in picard.analysis"><span class="typeNameLink">AbstractWgsMetricsCollector</span></a>&lt;<a href="picard/analysis/AbstractWgsMetricsCollector.html" title="type parameter in AbstractWgsMetricsCollector">T</a> extends htsjdk.samtools.util.AbstractRecordAndOffset&gt; - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Class for collecting data on reference coverage, base qualities and excluded bases from one AbstractLocusInfo object for
 CollectWgsMetrics.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html#acceptRead-htsjdk.samtools.SAMRecord-htsjdk.samtools.reference.ReferenceSequence-">acceptRead(SAMRecord, ReferenceSequence)</a></span> - Method in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html" title="class in picard.analysis.artifacts">CollectSequencingArtifactMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectAlignmentSummaryMetrics.html#acceptRead-htsjdk.samtools.SAMRecord-htsjdk.samtools.reference.ReferenceSequence-">acceptRead(SAMRecord, ReferenceSequence)</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectAlignmentSummaryMetrics.html" title="class in picard.analysis">CollectAlignmentSummaryMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectBaseDistributionByCycle.html#acceptRead-htsjdk.samtools.SAMRecord-htsjdk.samtools.reference.ReferenceSequence-">acceptRead(SAMRecord, ReferenceSequence)</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectBaseDistributionByCycle.html" title="class in picard.analysis">CollectBaseDistributionByCycle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectGcBiasMetrics.html#acceptRead-htsjdk.samtools.SAMRecord-htsjdk.samtools.reference.ReferenceSequence-">acceptRead(SAMRecord, ReferenceSequence)</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectGcBiasMetrics.html" title="class in picard.analysis">CollectGcBiasMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectInsertSizeMetrics.html#acceptRead-htsjdk.samtools.SAMRecord-htsjdk.samtools.reference.ReferenceSequence-">acceptRead(SAMRecord, ReferenceSequence)</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectInsertSizeMetrics.html" title="class in picard.analysis">CollectInsertSizeMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectQualityYieldMetrics.html#acceptRead-htsjdk.samtools.SAMRecord-htsjdk.samtools.reference.ReferenceSequence-">acceptRead(SAMRecord, ReferenceSequence)</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectQualityYieldMetrics.html" title="class in picard.analysis">CollectQualityYieldMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRnaSeqMetrics.html#acceptRead-htsjdk.samtools.SAMRecord-htsjdk.samtools.reference.ReferenceSequence-">acceptRead(SAMRecord, ReferenceSequence)</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectRnaSeqMetrics.html" title="class in picard.analysis">CollectRnaSeqMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/MeanQualityByCycle.html#acceptRead-htsjdk.samtools.SAMRecord-htsjdk.samtools.reference.ReferenceSequence-">acceptRead(SAMRecord, ReferenceSequence)</a></span> - Method in class picard.analysis.<a href="picard/analysis/MeanQualityByCycle.html" title="class in picard.analysis">MeanQualityByCycle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/QualityScoreDistribution.html#acceptRead-htsjdk.samtools.SAMRecord-htsjdk.samtools.reference.ReferenceSequence-">acceptRead(SAMRecord, ReferenceSequence)</a></span> - Method in class picard.analysis.<a href="picard/analysis/QualityScoreDistribution.html" title="class in picard.analysis">QualityScoreDistribution</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/SinglePassSamProgram.html#acceptRead-htsjdk.samtools.SAMRecord-htsjdk.samtools.reference.ReferenceSequence-">acceptRead(SAMRecord, ReferenceSequence)</a></span> - Method in class picard.analysis.<a href="picard/analysis/SinglePassSamProgram.html" title="class in picard.analysis">SinglePassSamProgram</a></dt>
<dd>
<div class="block">Should be implemented by subclasses to accept SAMRecords one at a time.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AlignmentSummaryMetricsCollector.html#acceptRecord-htsjdk.samtools.SAMRecord-htsjdk.samtools.reference.ReferenceSequence-">acceptRecord(SAMRecord, ReferenceSequence)</a></span> - Method in class picard.analysis.<a href="picard/analysis/AlignmentSummaryMetricsCollector.html" title="class in picard.analysis">AlignmentSummaryMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetricsCollector.html#acceptRecord-htsjdk.samtools.SAMRecord-htsjdk.samtools.reference.ReferenceSequence-">acceptRecord(SAMRecord, ReferenceSequence)</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetricsCollector.html" title="class in picard.analysis">CollectQualityYieldMetrics.QualityYieldMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/InsertSizeMetricsCollector.html#acceptRecord-htsjdk.samtools.SAMRecord-htsjdk.samtools.reference.ReferenceSequence-">acceptRecord(SAMRecord, ReferenceSequence)</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/InsertSizeMetricsCollector.html" title="class in picard.analysis.directed">InsertSizeMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/InsertSizeMetricsCollector.PerUnitInsertSizeMetricsCollector.html#acceptRecord-picard.analysis.directed.InsertSizeCollectorArgs-">acceptRecord(InsertSizeCollectorArgs)</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/InsertSizeMetricsCollector.PerUnitInsertSizeMetricsCollector.html" title="class in picard.analysis.directed">InsertSizeMetricsCollector.PerUnitInsertSizeMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/RnaSeqMetricsCollector.PerUnitRnaSeqMetricsCollector.html#acceptRecord-htsjdk.samtools.SAMRecord-">acceptRecord(SAMRecord)</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/RnaSeqMetricsCollector.PerUnitRnaSeqMetricsCollector.html" title="class in picard.analysis.directed">RnaSeqMetricsCollector.PerUnitRnaSeqMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetMetricsCollector.PerUnitTargetMetricCollector.html#acceptRecord-htsjdk.samtools.SAMRecord-">acceptRecord(SAMRecord)</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/TargetMetricsCollector.PerUnitTargetMetricCollector.html" title="class in picard.analysis.directed">TargetMetricsCollector.PerUnitTargetMetricCollector</a></dt>
<dd>
<div class="block">Adds information about an individual SAMRecord to the statistics.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasMetricsCollector.PerUnitGcBiasMetricsCollector.html#acceptRecord-picard.analysis.GcBiasCollectorArgs-">acceptRecord(GcBiasCollectorArgs)</a></span> - Method in class picard.analysis.<a href="picard/analysis/GcBiasMetricsCollector.PerUnitGcBiasMetricsCollector.html" title="class in picard.analysis">GcBiasMetricsCollector.PerUnitGcBiasMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/metrics/MultiLevelCollector.html#acceptRecord-htsjdk.samtools.SAMRecord-htsjdk.samtools.reference.ReferenceSequence-">acceptRecord(SAMRecord, ReferenceSequence)</a></span> - Method in class picard.metrics.<a href="picard/metrics/MultiLevelCollector.html" title="class in picard.metrics">MultiLevelCollector</a></dt>
<dd>
<div class="block">Construct a argument of ARGTYPE using the given SAMRecord and ReferenceSequence then pass
 this value to all collectors that should include this record</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/metrics/PerUnitMetricCollector.html#acceptRecord-ARGTYPE-">acceptRecord(ARGTYPE)</a></span> - Method in interface picard.metrics.<a href="picard/metrics/PerUnitMetricCollector.html" title="interface in picard.metrics">PerUnitMetricCollector</a></dt>
<dd>
<div class="block">Add a SAMRecord (with ReferenceSequence and Read Group info) to the metric(s) being calculated)</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CallingMetricAccumulator.html#accumulate-htsjdk.variant.variantcontext.VariantContext-">accumulate(VariantContext)</a></span> - Method in class picard.vcf.<a href="picard/vcf/CallingMetricAccumulator.html" title="class in picard.vcf">CallingMetricAccumulator</a></dt>
<dd>
<div class="block">Incorporates the provided variant's data into the metric analysis.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GvcfMetricAccumulator.html#accumulate-htsjdk.variant.variantcontext.VariantContext-">accumulate(VariantContext)</a></span> - Method in class picard.vcf.<a href="picard/vcf/GvcfMetricAccumulator.html" title="class in picard.vcf">GvcfMetricAccumulator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VariantProcessor.Accumulator.html#accumulate-htsjdk.variant.variantcontext.VariantContext-">accumulate(VariantContext)</a></span> - Method in interface picard.vcf.processor.<a href="picard/vcf/processor/VariantProcessor.Accumulator.html" title="interface in picard.vcf.processor">VariantProcessor.Accumulator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/AccumulateVariantCallingMetrics.html" title="class in picard.vcf"><span class="typeNameLink">AccumulateVariantCallingMetrics</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">Combines multiple Variant Calling Metrics files into a single file.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/AccumulateVariantCallingMetrics.html#AccumulateVariantCallingMetrics--">AccumulateVariantCallingMetrics()</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/AccumulateVariantCallingMetrics.html" title="class in picard.vcf">AccumulateVariantCallingMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasDetailMetrics.html#ACCUMULATION_LEVEL">ACCUMULATION_LEVEL</a></span> - Variable in class picard.analysis.<a href="picard/analysis/GcBiasDetailMetrics.html" title="class in picard.analysis">GcBiasDetailMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasSummaryMetrics.html#ACCUMULATION_LEVEL">ACCUMULATION_LEVEL</a></span> - Variable in class picard.analysis.<a href="picard/analysis/GcBiasSummaryMetrics.html" title="class in picard.analysis">GcBiasSummaryMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VariantAccumulatorExecutor.html#accumulators--">accumulators()</a></span> - Method in interface picard.vcf.processor.<a href="picard/vcf/processor/VariantAccumulatorExecutor.html" title="interface in picard.vcf.processor">VariantAccumulatorExecutor</a></dt>
<dd>
<div class="block">Returns the <a href="picard/vcf/processor/VariantProcessor.Accumulator.html" title="interface in picard.vcf.processor"><code>VariantProcessor.Accumulator</code></a>s associated with this executor.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VariantAccumulatorExecutor.MultiThreadedChunkBased.html#accumulators--">accumulators()</a></span> - Method in class picard.vcf.processor.<a href="picard/vcf/processor/VariantAccumulatorExecutor.MultiThreadedChunkBased.html" title="class in picard.vcf.processor">VariantAccumulatorExecutor.MultiThreadedChunkBased</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/DownsampleSam.html#ACCURACY">ACCURACY</a></span> - Variable in class picard.sam.<a href="picard/sam/DownsampleSam.html" title="class in picard.sam">DownsampleSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IntervalListTools.html#ACTION">ACTION</a></span> - Variable in class picard.util.<a href="picard/util/IntervalListTools.html" title="class in picard.util">IntervalListTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectAlignmentSummaryMetrics.html#ADAPTER_SEQUENCE">ADAPTER_SEQUENCE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectAlignmentSummaryMetrics.html" title="class in picard.analysis">CollectAlignmentSummaryMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/MarkIlluminaAdapters.html#ADAPTER_TRUNCATION_LENGTH">ADAPTER_TRUNCATION_LENGTH</a></span> - Variable in class picard.illumina.<a href="picard/illumina/MarkIlluminaAdapters.html" title="class in picard.illumina">MarkIlluminaAdapters</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/AdapterMarker.html" title="class in picard.util"><span class="typeNameLink">AdapterMarker</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">Store one or more AdapterPairs to use to mark adapter sequence of SAMRecords.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/AdapterMarker.html#AdapterMarker-picard.util.AdapterPair...-">AdapterMarker(AdapterPair...)</a></span> - Constructor for class picard.util.<a href="picard/util/AdapterMarker.html" title="class in picard.util">AdapterMarker</a></dt>
<dd>
<div class="block">Truncates adapters to DEFAULT_ADAPTER_LENGTH</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/AdapterMarker.html#AdapterMarker-int-picard.util.AdapterPair...-">AdapterMarker(int, AdapterPair...)</a></span> - Constructor for class picard.util.<a href="picard/util/AdapterMarker.html" title="class in picard.util">AdapterMarker</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/AdapterPair.html" title="interface in picard.util"><span class="typeNameLink">AdapterPair</span></a> - Interface in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/MarkIlluminaAdapters.html#ADAPTERS">ADAPTERS</a></span> - Variable in class picard.illumina.<a href="picard/illumina/MarkIlluminaAdapters.html" title="class in picard.illumina">MarkIlluminaAdapters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToFastq.html#ADAPTERS_TO_CHECK">ADAPTERS_TO_CHECK</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToFastq.html" title="class in picard.illumina">IlluminaBasecallsToFastq</a></dt>
<dd>
<div class="block"><span class="deprecatedLabel">Deprecated.</span></div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#ADAPTERS_TO_CHECK">ADAPTERS_TO_CHECK</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AdapterMarker.html#adapterTrimIlluminaPairedReads-htsjdk.samtools.SAMRecord-htsjdk.samtools.SAMRecord-">adapterTrimIlluminaPairedReads(SAMRecord, SAMRecord)</a></span> - Method in class picard.util.<a href="picard/util/AdapterMarker.html" title="class in picard.util">AdapterMarker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AdapterMarker.html#adapterTrimIlluminaPairedReads-htsjdk.samtools.SAMRecord-htsjdk.samtools.SAMRecord-int-double-">adapterTrimIlluminaPairedReads(SAMRecord, SAMRecord, int, double)</a></span> - Method in class picard.util.<a href="picard/util/AdapterMarker.html" title="class in picard.util">AdapterMarker</a></dt>
<dd>
<div class="block">Overrides defaults for minMatchBases and maxErrorRate</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/ClippingUtility.html#adapterTrimIlluminaPairedReads-htsjdk.samtools.SAMRecord-htsjdk.samtools.SAMRecord-picard.util.AdapterPair-">adapterTrimIlluminaPairedReads(SAMRecord, SAMRecord, AdapterPair)</a></span> - Static method in class picard.util.<a href="picard/util/ClippingUtility.html" title="class in picard.util">ClippingUtility</a></dt>
<dd>
<div class="block"><span class="deprecatedLabel">Deprecated.</span>
<div class="block"><span class="deprecationComment">Use the varargs version.  This no longer returns a warning string..</span></div>
</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/ClippingUtility.html#adapterTrimIlluminaPairedReads-htsjdk.samtools.SAMRecord-htsjdk.samtools.SAMRecord-picard.util.AdapterPair-int-double-">adapterTrimIlluminaPairedReads(SAMRecord, SAMRecord, AdapterPair, int, double)</a></span> - Static method in class picard.util.<a href="picard/util/ClippingUtility.html" title="class in picard.util">ClippingUtility</a></dt>
<dd>
<div class="block"><span class="deprecatedLabel">Deprecated.</span>
<div class="block"><span class="deprecationComment">Use the varargs version.  This no longer returns a warning string..</span></div>
</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/ClippingUtility.html#adapterTrimIlluminaPairedReads-htsjdk.samtools.SAMRecord-htsjdk.samtools.SAMRecord-picard.util.AdapterPair...-">adapterTrimIlluminaPairedReads(SAMRecord, SAMRecord, AdapterPair...)</a></span> - Static method in class picard.util.<a href="picard/util/ClippingUtility.html" title="class in picard.util">ClippingUtility</a></dt>
<dd>
<div class="block">Invokes adapterTrimIlluminaPairedReads with default less stringent parameters for a pair of reads.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/ClippingUtility.html#adapterTrimIlluminaPairedReads-htsjdk.samtools.SAMRecord-htsjdk.samtools.SAMRecord-int-double-picard.util.AdapterPair...-">adapterTrimIlluminaPairedReads(SAMRecord, SAMRecord, int, double, AdapterPair...)</a></span> - Static method in class picard.util.<a href="picard/util/ClippingUtility.html" title="class in picard.util">ClippingUtility</a></dt>
<dd>
<div class="block">Invokes adapterTrimIlluminaRead with explicit parameters for a pair of reads.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/AdapterMarker.html#adapterTrimIlluminaSingleRead-htsjdk.samtools.SAMRecord-">adapterTrimIlluminaSingleRead(SAMRecord)</a></span> - Method in class picard.util.<a href="picard/util/AdapterMarker.html" title="class in picard.util">AdapterMarker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AdapterMarker.html#adapterTrimIlluminaSingleRead-htsjdk.samtools.SAMRecord-int-double-">adapterTrimIlluminaSingleRead(SAMRecord, int, double)</a></span> - Method in class picard.util.<a href="picard/util/AdapterMarker.html" title="class in picard.util">AdapterMarker</a></dt>
<dd>
<div class="block">Overrides defaults for minMatchBases and maxErrorRate</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/ClippingUtility.html#adapterTrimIlluminaSingleRead-htsjdk.samtools.SAMRecord-picard.util.AdapterPair-">adapterTrimIlluminaSingleRead(SAMRecord, AdapterPair)</a></span> - Static method in class picard.util.<a href="picard/util/ClippingUtility.html" title="class in picard.util">ClippingUtility</a></dt>
<dd>
<div class="block"><span class="deprecatedLabel">Deprecated.</span>
<div class="block"><span class="deprecationComment">Use the varargs version.  This no longer returns a warning string..</span></div>
</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/ClippingUtility.html#adapterTrimIlluminaSingleRead-htsjdk.samtools.SAMRecord-picard.util.AdapterPair-int-double-">adapterTrimIlluminaSingleRead(SAMRecord, AdapterPair, int, double)</a></span> - Static method in class picard.util.<a href="picard/util/ClippingUtility.html" title="class in picard.util">ClippingUtility</a></dt>
<dd>
<div class="block"><span class="deprecatedLabel">Deprecated.</span>
<div class="block"><span class="deprecationComment">Use the varargs version.  This no longer returns a warning string..</span></div>
</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/ClippingUtility.html#adapterTrimIlluminaSingleRead-htsjdk.samtools.SAMRecord-picard.util.AdapterPair...-">adapterTrimIlluminaSingleRead(SAMRecord, AdapterPair...)</a></span> - Static method in class picard.util.<a href="picard/util/ClippingUtility.html" title="class in picard.util">ClippingUtility</a></dt>
<dd>
<div class="block">Invokes adapterTrimIlluminRead with default parameters for a single read.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/ClippingUtility.html#adapterTrimIlluminaSingleRead-htsjdk.samtools.SAMRecord-int-double-picard.util.AdapterPair...-">adapterTrimIlluminaSingleRead(SAMRecord, int, double, AdapterPair...)</a></span> - Static method in class picard.util.<a href="picard/util/ClippingUtility.html" title="class in picard.util">ClippingUtility</a></dt>
<dd>
<div class="block">Invokes adapterTrimIlluminRead with explicit matching thresholds for a single read.</div>
</dd>
<dt><a href="picard/analysis/AdapterUtility.html" title="class in picard.analysis"><span class="typeNameLink">AdapterUtility</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">A utility class for matching reads to adapters.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AdapterUtility.html#AdapterUtility-java.util.List-">AdapterUtility(List&lt;String&gt;)</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/AdapterUtility.html" title="class in picard.analysis">AdapterUtility</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/Fingerprint.html#add-picard.fingerprint.HaplotypeProbabilities-">add(HaplotypeProbabilities)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/Fingerprint.html" title="class in picard.fingerprint">Fingerprint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/pedigree/PedFile.html#add-picard.pedigree.PedFile.PedTrio-">add(PedFile.PedTrio)</a></span> - Method in class picard.pedigree.<a href="picard/pedigree/PedFile.html" title="class in picard.pedigree">PedFile</a></dt>
<dd>
<div class="block">Adds a trio to the PedFile keyed by the individual id.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/MarkQueue.html#add-picard.sam.markduplicates.util.ReadEndsForMateCigar-htsjdk.samtools.util.SamRecordTrackingBuffer-picard.sam.DuplicationMetrics-">add(ReadEndsForMateCigar, SamRecordTrackingBuffer, DuplicationMetrics)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/MarkQueue.html" title="class in picard.sam.markduplicates.util">MarkQueue</a></dt>
<dd>
<div class="block">Add a record to the mark queue.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/PhysicalLocationForMateCigarSet.html#add-picard.sam.markduplicates.util.ReadEndsForMateCigar-">add(ReadEndsForMateCigar)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/PhysicalLocationForMateCigarSet.html" title="class in picard.sam.markduplicates.util">PhysicalLocationForMateCigarSet</a></dt>
<dd>
<div class="block">Adds the end to this set, if not already added based on physical location</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FixMateInformation.html#ADD_MATE_CIGAR">ADD_MATE_CIGAR</a></span> - Variable in class picard.sam.<a href="picard/sam/FixMateInformation.html" title="class in picard.sam">FixMateInformation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#ADD_MATE_CIGAR">ADD_MATE_CIGAR</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/ReadOutputCommandLineProgram.html#ADD_PG_TAG_TO_READS">ADD_PG_TAG_TO_READS</a></span> - Variable in class picard.sam.util.<a href="picard/sam/util/ReadOutputCommandLineProgram.html" title="class in picard.sam.util">ReadOutputCommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/metrics/MultiLevelCollector.html#addAllLevelsToFile-htsjdk.samtools.metrics.MetricsFile-">addAllLevelsToFile(MetricsFile&lt;METRIC_TYPE, Histogram_KEY&gt;)</a></span> - Method in class picard.metrics.<a href="picard/metrics/MultiLevelCollector.html" title="class in picard.metrics">MultiLevelCollector</a></dt>
<dd>
<div class="block">Add all metrics to the given file in the following MetricAccumulationLevel order
  ALL_READS, SAMPLE, LIBRARY, READ_GROUP.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetMetricsCollector.Coverage.html#addBase-int-">addBase(int)</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/TargetMetricsCollector.Coverage.html" title="class in picard.analysis.directed">TargetMetricsCollector.Coverage</a></dt>
<dd>
<div class="block">Adds a single point of depth at the desired offset into the coverage array.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetMetricsCollector.Coverage.html#addBase-int-int-">addBase(int, int)</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/TargetMetricsCollector.Coverage.html" title="class in picard.analysis.directed">TargetMetricsCollector.Coverage</a></dt>
<dd>
<div class="block">Adds some depth at the desired offset into the coverage array.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AbstractWgsMetricsCollector.html#addBaseQHistogram-htsjdk.samtools.metrics.MetricsFile-">addBaseQHistogram(MetricsFile&lt;CollectWgsMetrics.WgsMetrics, Integer&gt;)</a></span> - Method in class picard.analysis.<a href="picard/analysis/AbstractWgsMetricsCollector.html" title="class in picard.analysis">AbstractWgsMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/AddCommentsToBam.html" title="class in picard.sam"><span class="typeNameLink">AddCommentsToBam</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block">A tool to add comments to a BAM file header.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AddCommentsToBam.html#AddCommentsToBam--">AddCommentsToBam()</a></span> - Constructor for class picard.sam.<a href="picard/sam/AddCommentsToBam.html" title="class in picard.sam">AddCommentsToBam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/Gene.Transcript.html#addCoverageCounts-int-int-int:A-">addCoverageCounts(int, int, int[])</a></span> - Method in class picard.annotation.<a href="picard/annotation/Gene.Transcript.html" title="class in picard.annotation">Gene.Transcript</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/BaseIlluminaDataProvider.html#addData-picard.illumina.parser.ClusterData-picard.illumina.parser.PositionalData-">addData(ClusterData, PositionalData)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/BaseIlluminaDataProvider.html" title="class in picard.illumina.parser">BaseIlluminaDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/BaseIlluminaDataProvider.html#addData-picard.illumina.parser.ClusterData-picard.illumina.parser.PfData-">addData(ClusterData, PfData)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/BaseIlluminaDataProvider.html" title="class in picard.illumina.parser">BaseIlluminaDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/BaseIlluminaDataProvider.html#addData-picard.illumina.parser.ClusterData-picard.illumina.parser.BarcodeData-">addData(ClusterData, BarcodeData)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/BaseIlluminaDataProvider.html" title="class in picard.illumina.parser">BaseIlluminaDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/GraphUtils.Graph.html#addEdge-Node-Node-">addEdge(Node, Node)</a></span> - Method in class picard.util.<a href="picard/util/GraphUtils.Graph.html" title="class in picard.util">GraphUtils.Graph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/Gene.Transcript.html#addExon-int-int-">addExon(int, int)</a></span> - Method in class picard.annotation.<a href="picard/annotation/Gene.Transcript.html" title="class in picard.annotation">Gene.Transcript</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeMap.html#addHaplotype-picard.fingerprint.HaplotypeBlock-">addHaplotype(HaplotypeBlock)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeMap.html" title="class in picard.fingerprint">HaplotypeMap</a></dt>
<dd>
<div class="block">Adds a HaplotypeBlock to the map and updates all the relevant caches/indices.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AbstractWgsMetricsCollector.html#addInfo-htsjdk.samtools.util.AbstractLocusInfo-htsjdk.samtools.reference.ReferenceSequence-boolean-">addInfo(AbstractLocusInfo&lt;T&gt;, ReferenceSequence, boolean)</a></span> - Method in class picard.analysis.<a href="picard/analysis/AbstractWgsMetricsCollector.html" title="class in picard.analysis">AbstractWgsMetricsCollector</a></dt>
<dd>
<div class="block">Accumulates the data from AbstractLocusInfo in inner structures</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetricsCollector.html#addInfo-htsjdk.samtools.util.AbstractLocusInfo-htsjdk.samtools.reference.ReferenceSequence-boolean-">addInfo(AbstractLocusInfo&lt;SamLocusIterator.RecordAndOffset&gt;, ReferenceSequence, boolean)</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetricsCollector.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/FastWgsMetricsCollector.html#addInfo-htsjdk.samtools.util.AbstractLocusInfo-htsjdk.samtools.reference.ReferenceSequence-boolean-">addInfo(AbstractLocusInfo&lt;EdgingRecordAndOffset&gt;, ReferenceSequence, boolean)</a></span> - Method in class picard.analysis.<a href="picard/analysis/FastWgsMetricsCollector.html" title="class in picard.analysis">FastWgsMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/BarcodeFileFaker.html#addLeadingZeros--">addLeadingZeros()</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/BarcodeFileFaker.html" title="class in picard.illumina.parser.fakers">BarcodeFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/BciFileFaker.html#addLeadingZeros--">addLeadingZeros()</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/BciFileFaker.html" title="class in picard.illumina.parser.fakers">BciFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/BclFileFaker.html#addLeadingZeros--">addLeadingZeros()</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/BclFileFaker.html" title="class in picard.illumina.parser.fakers">BclFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/ClocsFileFaker.html#addLeadingZeros--">addLeadingZeros()</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/ClocsFileFaker.html" title="class in picard.illumina.parser.fakers">ClocsFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/FileFaker.html#addLeadingZeros--">addLeadingZeros()</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/FileFaker.html" title="class in picard.illumina.parser.fakers">FileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/FilterFileFaker.html#addLeadingZeros--">addLeadingZeros()</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/FilterFileFaker.html" title="class in picard.illumina.parser.fakers">FilterFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/LocsFileFaker.html#addLeadingZeros--">addLeadingZeros()</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/LocsFileFaker.html" title="class in picard.illumina.parser.fakers">LocsFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/MultiTileBclFileFaker.html#addLeadingZeros--">addLeadingZeros()</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/MultiTileBclFileFaker.html" title="class in picard.illumina.parser.fakers">MultiTileBclFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/MultiTileLocsFileFaker.html#addLeadingZeros--">addLeadingZeros()</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/MultiTileLocsFileFaker.html" title="class in picard.illumina.parser.fakers">MultiTileLocsFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/PosFileFaker.html#addLeadingZeros--">addLeadingZeros()</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/PosFileFaker.html" title="class in picard.illumina.parser.fakers">PosFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/ReadNameParser.html#addLocationInformation-java.lang.String-picard.sam.util.PhysicalLocation-">addLocationInformation(String, PhysicalLocation)</a></span> - Method in class picard.sam.util.<a href="picard/sam/util/ReadNameParser.html" title="class in picard.sam.util">ReadNameParser</a></dt>
<dd>
<div class="block">Method used to extract tile/x/y from the read name and add it to the PhysicalLocationShort so that it
 can be used later to determine optical duplication</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/MatchResults.html#addLocusResult-picard.fingerprint.LocusResult-">addLocusResult(LocusResult)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/MatchResults.html" title="class in picard.fingerprint">MatchResults</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/LibraryIdGenerator.html#addMetricsByLibrary-java.lang.String-picard.sam.DuplicationMetrics-">addMetricsByLibrary(String, DuplicationMetrics)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/LibraryIdGenerator.html" title="class in picard.sam.markduplicates.util">LibraryIdGenerator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetricsCollector.html#addMetricsToFile-htsjdk.samtools.metrics.MetricsFile-">addMetricsToFile(MetricsFile&lt;CollectQualityYieldMetrics.QualityYieldMetrics, Integer&gt;)</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetricsCollector.html" title="class in picard.analysis">CollectQualityYieldMetrics.QualityYieldMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/InsertSizeMetricsCollector.PerUnitInsertSizeMetricsCollector.html#addMetricsToFile-htsjdk.samtools.metrics.MetricsFile-">addMetricsToFile(MetricsFile&lt;InsertSizeMetrics, Integer&gt;)</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/InsertSizeMetricsCollector.PerUnitInsertSizeMetricsCollector.html" title="class in picard.analysis.directed">InsertSizeMetricsCollector.PerUnitInsertSizeMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/RnaSeqMetricsCollector.PerUnitRnaSeqMetricsCollector.html#addMetricsToFile-htsjdk.samtools.metrics.MetricsFile-">addMetricsToFile(MetricsFile&lt;RnaSeqMetrics, Integer&gt;)</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/RnaSeqMetricsCollector.PerUnitRnaSeqMetricsCollector.html" title="class in picard.analysis.directed">RnaSeqMetricsCollector.PerUnitRnaSeqMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetMetricsCollector.PerUnitTargetMetricCollector.html#addMetricsToFile-htsjdk.samtools.metrics.MetricsFile-">addMetricsToFile(MetricsFile&lt;METRIC_TYPE, Integer&gt;)</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/TargetMetricsCollector.PerUnitTargetMetricCollector.html" title="class in picard.analysis.directed">TargetMetricsCollector.PerUnitTargetMetricCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasMetricsCollector.PerUnitGcBiasMetricsCollector.html#addMetricsToFile-htsjdk.samtools.metrics.MetricsFile-">addMetricsToFile(MetricsFile&lt;GcBiasMetrics, Integer&gt;)</a></span> - Method in class picard.analysis.<a href="picard/analysis/GcBiasMetricsCollector.PerUnitGcBiasMetricsCollector.html" title="class in picard.analysis">GcBiasMetricsCollector.PerUnitGcBiasMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/metrics/PerUnitMetricCollector.html#addMetricsToFile-htsjdk.samtools.metrics.MetricsFile-">addMetricsToFile(MetricsFile&lt;BEAN, HKEY&gt;)</a></span> - Method in interface picard.metrics.<a href="picard/metrics/PerUnitMetricCollector.html" title="interface in picard.metrics">PerUnitMetricCollector</a></dt>
<dd>
<div class="block">Any metrics collected will be added to the metric file provided.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#addMissingTruthAndMissingCallStates-double-long-picard.vcf.GenotypeConcordanceCounts-">addMissingTruthAndMissingCallStates(double, long, GenotypeConcordanceCounts)</a></span> - Static method in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>
<div class="block">Method to add missing sites that are KNOWN to be HOM_REF in the case of the NIST truth data set.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/GraphUtils.Graph.html#addNode-Node-">addNode(Node)</a></span> - Method in class picard.util.<a href="picard/util/GraphUtils.Graph.html" title="class in picard.util">GraphUtils.Graph</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/AddOrReplaceReadGroups.html" title="class in picard.sam"><span class="typeNameLink">AddOrReplaceReadGroups</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block">Assigns all the reads in a file to a single new readgroup.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AddOrReplaceReadGroups.html#AddOrReplaceReadGroups--">AddOrReplaceReadGroups()</a></span> - Constructor for class picard.sam.<a href="picard/sam/AddOrReplaceReadGroups.html" title="class in picard.sam">AddOrReplaceReadGroups</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/BaseIlluminaDataProvider.html#addReadData-picard.illumina.parser.ClusterData-int-picard.illumina.parser.BaseData-">addReadData(ClusterData, int, BaseData)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/BaseIlluminaDataProvider.html" title="class in picard.illumina.parser">BaseIlluminaDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/BaseIlluminaDataProvider.html#addReadData-picard.illumina.parser.ClusterData-int-picard.illumina.parser.QualityData-">addReadData(ClusterData, int, QualityData)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/BaseIlluminaDataProvider.html" title="class in picard.illumina.parser">BaseIlluminaDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/BaseIlluminaDataProvider.html#addReadData-picard.illumina.parser.ClusterData-int-picard.illumina.parser.CbclData-">addReadData(ClusterData, int, CbclData)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/BaseIlluminaDataProvider.html" title="class in picard.illumina.parser">BaseIlluminaDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintResults.html#addResults-picard.fingerprint.MatchResults-">addResults(MatchResults)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/FingerprintResults.html" title="class in picard.fingerprint">FingerprintResults</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceScheme.html#addRow-picard.vcf.GenotypeConcordanceStates.CallState-picard.vcf.GenotypeConcordanceStates.ContingencyState:A...-">addRow(GenotypeConcordanceStates.CallState, GenotypeConcordanceStates.ContingencyState[]...)</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceScheme.html" title="class in picard.vcf">GenotypeConcordanceScheme</a></dt>
<dd>
<div class="block">Adds a row to the scheme</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeBlock.html#addSnp-picard.fingerprint.Snp-">addSnp(Snp)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeBlock.html" title="class in picard.fingerprint">HaplotypeBlock</a></dt>
<dd>
<div class="block">Adds a SNP to the haplotype.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilitiesFromGenotypeLikelihoods.html#addToLogLikelihoods-picard.fingerprint.Snp-java.util.List-double:A-">addToLogLikelihoods(Snp, List&lt;Allele&gt;, double[])</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilitiesFromGenotypeLikelihoods.html" title="class in picard.fingerprint">HaplotypeProbabilitiesFromGenotypeLikelihoods</a></dt>
<dd>
<div class="block">Adds a base observation with the observed quality to the evidence for this haplotype
 based on the fact that the SNP is part of the haplotype.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AbstractWgsMetricsCollector.html#addToMetricsFile-htsjdk.samtools.metrics.MetricsFile-boolean-picard.filter.CountingFilter-picard.filter.CountingFilter-picard.filter.CountingPairedFilter-">addToMetricsFile(MetricsFile&lt;CollectWgsMetrics.WgsMetrics, Integer&gt;, boolean, CountingFilter, CountingFilter, CountingPairedFilter)</a></span> - Method in class picard.analysis.<a href="picard/analysis/AbstractWgsMetricsCollector.html" title="class in picard.analysis">AbstractWgsMetricsCollector</a></dt>
<dd>
<div class="block">Adds collected metrics and depth histogram to file</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverageCollector.html#addToMetricsFile-htsjdk.samtools.metrics.MetricsFile-boolean-picard.filter.CountingFilter-picard.filter.CountingFilter-picard.filter.CountingPairedFilter-">addToMetricsFile(MetricsFile&lt;CollectWgsMetrics.WgsMetrics, Integer&gt;, boolean, CountingFilter, CountingFilter, CountingPairedFilter)</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverageCollector.html" title="class in picard.analysis">CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverageCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/WgsMetricsProcessor.html#addToMetricsFile-htsjdk.samtools.metrics.MetricsFile-boolean-picard.filter.CountingFilter-picard.filter.CountingFilter-picard.filter.CountingPairedFilter-">addToMetricsFile(MetricsFile&lt;CollectWgsMetrics.WgsMetrics, Integer&gt;, boolean, CountingFilter, CountingFilter, CountingPairedFilter)</a></span> - Method in interface picard.analysis.<a href="picard/analysis/WgsMetricsProcessor.html" title="interface in picard.analysis">WgsMetricsProcessor</a></dt>
<dd>
<div class="block">Adds result metric's data to input file</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/WgsMetricsProcessorImpl.html#addToMetricsFile-htsjdk.samtools.metrics.MetricsFile-boolean-picard.filter.CountingFilter-picard.filter.CountingFilter-picard.filter.CountingPairedFilter-">addToMetricsFile(MetricsFile&lt;CollectWgsMetrics.WgsMetrics, Integer&gt;, boolean, CountingFilter, CountingFilter, CountingPairedFilter)</a></span> - Method in class picard.analysis.<a href="picard/analysis/WgsMetricsProcessorImpl.html" title="class in picard.analysis">WgsMetricsProcessorImpl</a></dt>
<dd>
<div class="block">Adds result metric's data to input file</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilitiesFromContaminatorSequence.html#addToProbs-picard.fingerprint.Snp-byte-byte-">addToProbs(Snp, byte, byte)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilitiesFromContaminatorSequence.html" title="class in picard.fingerprint">HaplotypeProbabilitiesFromContaminatorSequence</a></dt>
<dd>
<div class="block">Adds a base observation with the observed quality to the evidence for this haplotype
 based on the fact that the SNP is part of the haplotype.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilitiesFromSequence.html#addToProbs-picard.fingerprint.Snp-byte-byte-">addToProbs(Snp, byte, byte)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilitiesFromSequence.html" title="class in picard.fingerprint">HaplotypeProbabilitiesFromSequence</a></dt>
<dd>
<div class="block">Adds a base observation with the observed quality to the evidence for this haplotype
 based on the fact that the SNP is part of the haplotype.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/Gene.html#addTranscript-java.lang.String-int-int-int-int-int-">addTranscript(String, int, int, int, int, int)</a></span> - Method in class picard.annotation.<a href="picard/annotation/Gene.html" title="class in picard.annotation">Gene</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/UmiMetrics.html#addUmiObservation-java.lang.String-java.lang.String-">addUmiObservation(String, String)</a></span> - Method in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/UmiMetrics.html" title="class in picard.sam.markduplicates">UmiMetrics</a></dt>
<dd>
<div class="block">Add an observation of a UMI to the metrics</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/AbstractInputParser.html#advance--">advance()</a></span> - Method in class picard.util.<a href="picard/util/AbstractInputParser.html" title="class in picard.util">AbstractInputParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/BasicInputParser.html#advanceFile--">advanceFile()</a></span> - Method in class picard.util.<a href="picard/util/BasicInputParser.html" title="class in picard.util">BasicInputParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/ThreadPoolExecutorWithExceptions.html#afterExecute-java.lang.Runnable-java.lang.Throwable-">afterExecute(Runnable, Throwable)</a></span> - Method in class picard.util.<a href="picard/util/ThreadPoolExecutorWithExceptions.html" title="class in picard.util">ThreadPoolExecutorWithExceptions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#ALIGNED_BAM">ALIGNED_BAM</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasSummaryMetrics.html#ALIGNED_READS">ALIGNED_READS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/GcBiasSummaryMetrics.html" title="class in picard.analysis">GcBiasSummaryMetrics</a></dt>
<dd>
<div class="block">The total number of aligned reads used to compute the gc bias metrics.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectBaseDistributionByCycle.html#ALIGNED_READS_ONLY">ALIGNED_READS_ONLY</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectBaseDistributionByCycle.html" title="class in picard.analysis">CollectBaseDistributionByCycle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/MeanQualityByCycle.html#ALIGNED_READS_ONLY">ALIGNED_READS_ONLY</a></span> - Variable in class picard.analysis.<a href="picard/analysis/MeanQualityByCycle.html" title="class in picard.analysis">MeanQualityByCycle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/QualityScoreDistribution.html#ALIGNED_READS_ONLY">ALIGNED_READS_ONLY</a></span> - Variable in class picard.analysis.<a href="picard/analysis/QualityScoreDistribution.html" title="class in picard.analysis">QualityScoreDistribution</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#ALIGNED_READS_ONLY">ALIGNED_READS_ONLY</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#ALIGNER_PROPER_PAIR_FLAGS">ALIGNER_PROPER_PAIR_FLAGS</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ViewSam.html#ALIGNMENT_STATUS">ALIGNMENT_STATUS</a></span> - Variable in class picard.sam.<a href="picard/sam/ViewSam.html" title="class in picard.sam">ViewSam</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/AlignmentSummaryMetrics.html" title="class in picard.analysis"><span class="typeNameLink">AlignmentSummaryMetrics</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">High level metrics about the alignment of reads within a SAM file, produced by
 the CollectAlignmentSummaryMetrics program and usually stored in a file with
 the extension ".alignment_summary_metrics".</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AlignmentSummaryMetrics.html#AlignmentSummaryMetrics--">AlignmentSummaryMetrics()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/AlignmentSummaryMetrics.html" title="class in picard.analysis">AlignmentSummaryMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/AlignmentSummaryMetrics.Category.html" title="enum in picard.analysis"><span class="typeNameLink">AlignmentSummaryMetrics.Category</span></a> - Enum in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/AlignmentSummaryMetricsCollector.html" title="class in picard.analysis"><span class="typeNameLink">AlignmentSummaryMetricsCollector</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AlignmentSummaryMetricsCollector.html#AlignmentSummaryMetricsCollector-java.util.Set-java.util.List-boolean-java.util.List-int-java.util.Set-boolean-">AlignmentSummaryMetricsCollector(Set&lt;MetricAccumulationLevel&gt;, List&lt;SAMReadGroupRecord&gt;, boolean, List&lt;String&gt;, int, Set&lt;SamPairUtil.PairOrientation&gt;, boolean)</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/AlignmentSummaryMetricsCollector.html" title="class in picard.analysis">AlignmentSummaryMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/FilterApplyingVariantIterator.html#ALL_GTS_FILTERED">ALL_GTS_FILTERED</a></span> - Static variable in class picard.vcf.filter.<a href="picard/vcf/filter/FilterApplyingVariantIterator.html" title="class in picard.vcf.filter">FilterApplyingVariantIterator</a></dt>
<dd>
<div class="block">Filter string that is used to filter a Variant when all variant genotypes are filtered out.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.Alleles.html#allAlleles">allAlleles</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.Alleles.html" title="class in picard.vcf">GenotypeConcordance.Alleles</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/filter/AlleleBalanceFilter.html" title="class in picard.vcf.filter"><span class="typeNameLink">AlleleBalanceFilter</span></a> - Class in <a href="picard/vcf/filter/package-summary.html">picard.vcf.filter</a></dt>
<dd>
<div class="block">Filters out a record if the allele balance for heterozygotes is out of a defined range across all samples.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/AlleleBalanceFilter.html#AlleleBalanceFilter-double-">AlleleBalanceFilter(double)</a></span> - Constructor for class picard.vcf.filter.<a href="picard/vcf/filter/AlleleBalanceFilter.html" title="class in picard.vcf.filter">AlleleBalanceFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.Alleles.html#Alleles-picard.vcf.OrderedSet-java.lang.String-java.lang.String-java.lang.String-java.lang.String-">Alleles(OrderedSet&lt;String&gt;, String, String, String, String)</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/GenotypeConcordance.Alleles.html" title="class in picard.vcf">GenotypeConcordance.Alleles</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/AlleleSubsettingUtils.html" title="class in picard.util"><span class="typeNameLink">AlleleSubsettingUtils</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">Utilities class containing methods for restricting <code>VariantContext</code> and <code>GenotypesContext</code> objects to a
 reduced set of alleles, as well as for choosing the best set of alleles to keep and for cleaning up annotations and
 genotypes after subsetting.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/AlleleSubsettingUtils.html#AlleleSubsettingUtils--">AlleleSubsettingUtils()</a></span> - Constructor for class picard.util.<a href="picard/util/AlleleSubsettingUtils.html" title="class in picard.util">AlleleSubsettingUtils</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FastqToSam.html#ALLOW_AND_IGNORE_EMPTY_LINES">ALLOW_AND_IGNORE_EMPTY_LINES</a></span> - Variable in class picard.sam.<a href="picard/sam/FastqToSam.html" title="class in picard.sam">FastqToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ReorderSam.html#ALLOW_CONTIG_LENGTH_DISCORDANCE">ALLOW_CONTIG_LENGTH_DISCORDANCE</a></span> - Variable in class picard.sam.<a href="picard/sam/ReorderSam.html" title="class in picard.sam">ReorderSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckFingerprints.html#ALLOW_DUPLICATE_READS">ALLOW_DUPLICATE_READS</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckFingerprints.html" title="class in picard.fingerprint">CrosscheckFingerprints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ReorderSam.html#ALLOW_INCOMPLETE_DICT_CONCORDANCE">ALLOW_INCOMPLETE_DICT_CONCORDANCE</a></span> - Variable in class picard.sam.<a href="picard/sam/ReorderSam.html" title="class in picard.sam">ReorderSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/LiftoverVcf.html#ALLOW_MISSING_FIELDS_IN_HEADER">ALLOW_MISSING_FIELDS_IN_HEADER</a></span> - Variable in class picard.vcf.<a href="picard/vcf/LiftoverVcf.html" title="class in picard.vcf">LiftoverVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/UmiAwareMarkDuplicatesWithMateCigar.html#ALLOW_MISSING_UMIS">ALLOW_MISSING_UMIS</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/UmiAwareMarkDuplicatesWithMateCigar.html" title="class in picard.sam.markduplicates">UmiAwareMarkDuplicatesWithMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/PositionBasedDownsampleSam.html#ALLOW_MULTIPLE_DOWNSAMPLING_DESPITE_WARNINGS">ALLOW_MULTIPLE_DOWNSAMPLING_DESPITE_WARNINGS</a></span> - Variable in class picard.sam.<a href="picard/sam/PositionBasedDownsampleSam.html" title="class in picard.sam">PositionBasedDownsampleSam</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/cmdline/programgroups/Alpha.html" title="class in picard.cmdline.programgroups"><span class="typeNameLink">Alpha</span></a> - Class in <a href="picard/cmdline/programgroups/package-summary.html">picard.cmdline.programgroups</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/programgroups/Alpha.html#Alpha--">Alpha()</a></span> - Constructor for class picard.cmdline.programgroups.<a href="picard/cmdline/programgroups/Alpha.html" title="class in picard.cmdline.programgroups">Alpha</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectGcBiasMetrics.html#ALSO_IGNORE_DUPLICATES">ALSO_IGNORE_DUPLICATES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectGcBiasMetrics.html" title="class in picard.analysis">CollectGcBiasMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/ErrorSummaryMetrics.html#ALT_BASE">ALT_BASE</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/ErrorSummaryMetrics.html" title="class in picard.analysis.artifacts">ErrorSummaryMetrics</a></dt>
<dd>
<div class="block">The alternative base (or it's complement).</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html#ALT_BASE">ALT_BASE</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.BaitBiasDetailMetrics</a></dt>
<dd>
<div class="block">The (upper-case) alternative base that is called as a result of DNA damage.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasSummaryMetrics.html#ALT_BASE">ALT_BASE</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasSummaryMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.BaitBiasSummaryMetrics</a></dt>
<dd>
<div class="block">The (upper-case) alternative base that is called as a result of DNA damage.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterDetailMetrics.html#ALT_BASE">ALT_BASE</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterDetailMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.PreAdapterDetailMetrics</a></dt>
<dd>
<div class="block">The (upper-case) alternative base that is called as a result of DNA damage.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterSummaryMetrics.html#ALT_BASE">ALT_BASE</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterSummaryMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.PreAdapterSummaryMetrics</a></dt>
<dd>
<div class="block">The (upper-case) alternative base that is called as a result of DNA damage.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/ErrorSummaryMetrics.html#ALT_COUNT">ALT_COUNT</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/ErrorSummaryMetrics.html" title="class in picard.analysis.artifacts">ErrorSummaryMetrics</a></dt>
<dd>
<div class="block">The number of alt bases observed.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html#ALT_NONOXO_BASES">ALT_NONOXO_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html" title="class in picard.analysis">CollectOxoGMetrics.CpcgMetrics</a></dt>
<dd>
<div class="block">The count of observed A basecalls at C reference positions and T basecalls
 at G reference bases that are correlated to instrument read number in a way
 that rules out oxidation as the cause</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html#ALT_OXO_BASES">ALT_OXO_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html" title="class in picard.analysis">CollectOxoGMetrics.CpcgMetrics</a></dt>
<dd>
<div class="block">The count of observed A basecalls at C reference positions and T basecalls
 at G reference bases that are correlated to instrument read number in a way
 that is consistent with oxidative damage.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/CollectTargetedPcrMetrics.html#AMPLICON_INTERVALS">AMPLICON_INTERVALS</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/CollectTargetedPcrMetrics.html" title="class in picard.analysis.directed">CollectTargetedPcrMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#AMPLICON_TERRITORY">AMPLICON_TERRITORY</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The number of unique bases covered by the intervals of all amplicons in the amplicon set</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fastq/BamToBfq.html#ANALYSIS_DIR">ANALYSIS_DIR</a></span> - Variable in class picard.fastq.<a href="picard/fastq/BamToBfq.html" title="class in picard.fastq">BamToBfq</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/annotation/AnnotationException.html" title="class in picard.annotation"><span class="typeNameLink">AnnotationException</span></a> - Exception in <a href="picard/annotation/package-summary.html">picard.annotation</a></dt>
<dd>
<div class="block">Exception thrown when loading gene annotations.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/AnnotationException.html#AnnotationException-java.lang.String-">AnnotationException(String)</a></span> - Constructor for exception picard.annotation.<a href="picard/annotation/AnnotationException.html" title="class in picard.annotation">AnnotationException</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/AnnotationException.html#AnnotationException-java.lang.String-java.lang.Throwable-">AnnotationException(String, Throwable)</a></span> - Constructor for exception picard.annotation.<a href="picard/annotation/AnnotationException.html" title="class in picard.annotation">AnnotationException</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToFastq.html#APPLY_EAMSS_FILTER">APPLY_EAMSS_FILTER</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToFastq.html" title="class in picard.illumina">IlluminaBasecallsToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#APPLY_EAMSS_FILTER">APPLY_EAMSS_FILTER</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CompareSAMs.html#areEqual--">areEqual()</a></span> - Method in class picard.sam.<a href="picard/sam/CompareSAMs.html" title="class in picard.sam">CompareSAMs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverageCollector.html#areHistogramsEmpty--">areHistogramsEmpty()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverageCollector.html" title="class in picard.analysis">CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverageCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasSummaryMetrics.html#ARTIFACT_NAME">ARTIFACT_NAME</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasSummaryMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.BaitBiasSummaryMetrics</a></dt>
<dd>
<div class="block">A "nickname" of this artifact, if it is a known error mode.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterSummaryMetrics.html#ARTIFACT_NAME">ARTIFACT_NAME</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterSummaryMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.PreAdapterSummaryMetrics</a></dt>
<dd>
<div class="block">A "nickname" of this artifact, if it is a known error mode.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.Alleles.html#asList--">asList()</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.Alleles.html" title="class in picard.vcf">GenotypeConcordance.Alleles</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/BclQualityEvaluationStrategy.html#assertMinimumQualities--">assertMinimumQualities()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/BclQualityEvaluationStrategy.html" title="class in picard.illumina.parser.readers">BclQualityEvaluationStrategy</a></dt>
<dd>
<div class="block">Reviews the qualities observed thus far and throws an exception if any are below the minimum quality threshold.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/BclReader.html#assertProperFileStructure-java.io.File-">assertProperFileStructure(File)</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/BclReader.html" title="class in picard.illumina.parser.readers">BclReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigarIterator.html#assertSorted-htsjdk.samtools.SAMFileHeader.SortOrder-">assertSorted(SAMFileHeader.SortOrder)</a></span> - Method in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigarIterator.html" title="class in picard.sam.markduplicates">MarkDuplicatesWithMateCigarIterator</a></dt>
<dd>
<div class="block">Establishes that records returned by this iterator are expected to
 be in the specified sort order.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/UmiAwareMarkDuplicatesWithMateCigar.html#ASSIGNED_UMI_TAG">ASSIGNED_UMI_TAG</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/UmiAwareMarkDuplicatesWithMateCigar.html" title="class in picard.sam.markduplicates">UmiAwareMarkDuplicatesWithMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/Gene.Transcript.html#assignLocusFunctionForRange-int-picard.annotation.LocusFunction:A-">assignLocusFunctionForRange(int, LocusFunction[])</a></span> - Method in class picard.annotation.<a href="picard/annotation/Gene.Transcript.html" title="class in picard.annotation">Gene.Transcript</a></dt>
<dd>
<div class="block">Write into locusFunctions the function of each position from start to start + locusFunctions.length
 relative to this transcript.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html#ASSUME_SORT_ORDER">ASSUME_SORT_ORDER</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html" title="class in picard.sam.markduplicates.util">AbstractMarkDuplicatesCommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/StandardOptionDefinitions.html#ASSUME_SORT_ORDER_SHORT_NAME">ASSUME_SORT_ORDER_SHORT_NAME</a></span> - Static variable in class picard.cmdline.<a href="picard/cmdline/StandardOptionDefinitions.html" title="class in picard.cmdline">StandardOptionDefinitions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectMultipleMetrics.html#ASSUME_SORTED">ASSUME_SORTED</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectMultipleMetrics.html" title="class in picard.analysis">CollectMultipleMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRrbsMetrics.html#ASSUME_SORTED">ASSUME_SORTED</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectRrbsMetrics.html" title="class in picard.analysis">CollectRrbsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/SinglePassSamProgram.html#ASSUME_SORTED">ASSUME_SORTED</a></span> - Variable in class picard.analysis.<a href="picard/analysis/SinglePassSamProgram.html" title="class in picard.analysis">SinglePassSamProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FixMateInformation.html#ASSUME_SORTED">ASSUME_SORTED</a></span> - Variable in class picard.sam.<a href="picard/sam/FixMateInformation.html" title="class in picard.sam">FixMateInformation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html#ASSUME_SORTED">ASSUME_SORTED</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html" title="class in picard.sam.markduplicates.util">AbstractMarkDuplicatesCommandLineProgram</a></dt>
<dd>
<div class="block"><span class="deprecatedLabel">Deprecated.</span></div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeSamFiles.html#ASSUME_SORTED">ASSUME_SORTED</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeSamFiles.html" title="class in picard.sam">MergeSamFiles</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/StandardOptionDefinitions.html#ASSUME_SORTED_SHORT_NAME">ASSUME_SORTED_SHORT_NAME</a></span> - Static variable in class picard.cmdline.<a href="picard/cmdline/StandardOptionDefinitions.html" title="class in picard.cmdline">StandardOptionDefinitions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeMap.html#asVcf-htsjdk.samtools.reference.ReferenceSequenceFile-">asVcf(ReferenceSequenceFile)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeMap.html" title="class in picard.fingerprint">HaplotypeMap</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/AsyncIterator.html" title="class in picard.util"><span class="typeNameLink">AsyncIterator</span></a>&lt;<a href="picard/util/AsyncIterator.html" title="type parameter in AsyncIterator">T</a>&gt; - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">Wrapper around a CloseableIterator that reads in a separate thread, for cases in which that might be
 efficient.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/AsyncIterator.html#AsyncIterator-htsjdk.samtools.util.CloseableIterator-int-java.lang.String-">AsyncIterator(CloseableIterator&lt;T&gt;, int, String)</a></span> - Constructor for class picard.util.<a href="picard/util/AsyncIterator.html" title="class in picard.util">AsyncIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#AT_DROPOUT">AT_DROPOUT</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">A measure of how undercovered <= 50% GC regions are relative to the mean.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#AT_DROPOUT">AT_DROPOUT</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">A measure of how regions with low GC content (<= 50%), are undercovered relative to mean coverage.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasSummaryMetrics.html#AT_DROPOUT">AT_DROPOUT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/GcBiasSummaryMetrics.html" title="class in picard.analysis">GcBiasSummaryMetrics</a></dt>
<dd>
<div class="block">Illumina-style AT dropout metric.</div>
</dd>
<dt><a href="picard/util/AtomicIterator.html" title="interface in picard.util"><span class="typeNameLink">AtomicIterator</span></a>&lt;<a href="picard/util/AtomicIterator.html" title="type parameter in AtomicIterator">T</a>&gt; - Interface in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">Describes</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/Iterators.html#atomicIteratorOf-java.lang.Iterable-">atomicIteratorOf(Iterable&lt;E&gt;)</a></span> - Static method in class picard.util.<a href="picard/util/Iterators.html" title="class in picard.util">Iterators</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/Iterators.html#atomicIteratorOf-java.util.Iterator-">atomicIteratorOf(Iterator&lt;E&gt;)</a></span> - Static method in class picard.util.<a href="picard/util/Iterators.html" title="class in picard.util">Iterators</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/LiftoverVcf.html#ATTEMPTED_LOCUS">ATTEMPTED_LOCUS</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/LiftoverVcf.html" title="class in picard.vcf">LiftoverVcf</a></dt>
<dd>
<div class="block">Attribute used to store the position of the failed variant on the target contig prior to finding out that alleles do not match.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertSam.html#ATTRIBUTE_TO_CLEAR">ATTRIBUTE_TO_CLEAR</a></span> - Variable in class picard.sam.<a href="picard/sam/RevertSam.html" title="class in picard.sam">RevertSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#ATTRIBUTES_TO_REMOVE">ATTRIBUTES_TO_REMOVE</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#ATTRIBUTES_TO_RETAIN">ATTRIBUTES_TO_RETAIN</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#ATTRIBUTES_TO_REVERSE">ATTRIBUTES_TO_REVERSE</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#ATTRIBUTES_TO_REVERSE_COMPLEMENT">ATTRIBUTES_TO_REVERSE_COMPLEMENT</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VariantAccumulatorExecutor.html#awaitCompletion--">awaitCompletion()</a></span> - Method in interface picard.vcf.processor.<a href="picard/vcf/processor/VariantAccumulatorExecutor.html" title="interface in picard.vcf.processor">VariantAccumulatorExecutor</a></dt>
<dd>
<div class="block">Blocks until the work is complete.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VariantAccumulatorExecutor.MultiThreadedChunkBased.html#awaitCompletion--">awaitCompletion()</a></span> - Method in class picard.vcf.processor.<a href="picard/vcf/processor/VariantAccumulatorExecutor.MultiThreadedChunkBased.html" title="class in picard.vcf.processor">VariantAccumulatorExecutor.MultiThreadedChunkBased</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="I:B">
<!--   -->
</a>
<h2 class="title">B</h2>
<dl>
<dt><span class="memberNameLink"><a href="picard/analysis/AlignmentSummaryMetrics.html#BAD_CYCLES">BAD_CYCLES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/AlignmentSummaryMetrics.html" title="class in picard.analysis">AlignmentSummaryMetrics</a></dt>
<dd>
<div class="block">The number of instrument cycles in which 80% or more of base calls were no-calls.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.html#BAIT_BIAS_DETAILS_EXT">BAIT_BIAS_DETAILS_EXT</a></span> - Static variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.html#BAIT_BIAS_SUMMARY_EXT">BAIT_BIAS_SUMMARY_EXT</a></span> - Static variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#BAIT_DESIGN_EFFICIENCY">BAIT_DESIGN_EFFICIENCY</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The ratio of TARGET_TERRITORY/BAIT_TERRITORY.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/CollectHsMetrics.html#BAIT_INTERVALS">BAIT_INTERVALS</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/CollectHsMetrics.html" title="class in picard.analysis.directed">CollectHsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/BaitDesigner.html#BAIT_OFFSET">BAIT_OFFSET</a></span> - Variable in class picard.util.<a href="picard/util/BaitDesigner.html" title="class in picard.util">BaitDesigner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#BAIT_SET">BAIT_SET</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The name of the bait set used in the hybrid selection.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/CollectHsMetrics.html#BAIT_SET_NAME">BAIT_SET_NAME</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/CollectHsMetrics.html" title="class in picard.analysis.directed">CollectHsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/BaitDesigner.html#BAIT_SIZE">BAIT_SIZE</a></span> - Variable in class picard.util.<a href="picard/util/BaitDesigner.html" title="class in picard.util">BaitDesigner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#BAIT_TERRITORY">BAIT_TERRITORY</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The number of bases which are localized to one or more baits.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html#BaitBiasDetailMetrics--">BaitBiasDetailMetrics()</a></span> - Constructor for class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.BaitBiasDetailMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasSummaryMetrics.html#BaitBiasSummaryMetrics--">BaitBiasSummaryMetrics()</a></span> - Constructor for class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasSummaryMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.BaitBiasSummaryMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/BaitDesigner.html" title="class in picard.util"><span class="typeNameLink">BaitDesigner</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">Designs baits for hybrid selection!</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/BaitDesigner.html#BaitDesigner--">BaitDesigner()</a></span> - Constructor for class picard.util.<a href="picard/util/BaitDesigner.html" title="class in picard.util">BaitDesigner</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/BaitDesigner.DesignStrategy.html" title="enum in picard.util"><span class="typeNameLink">BaitDesigner.DesignStrategy</span></a> - Enum in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">Set of possible design strategies for bait design.</div>
</dd>
<dt><a href="picard/sam/BamIndexStats.html" title="class in picard.sam"><span class="typeNameLink">BamIndexStats</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block">Command line program to print statistics from BAM index (.bai) file
 Statistics include count of aligned and unaligned reads for each reference sequence
 and a count of all records with no start coordinate.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/BamIndexStats.html#BamIndexStats--">BamIndexStats()</a></span> - Constructor for class picard.sam.<a href="picard/sam/BamIndexStats.html" title="class in picard.sam">BamIndexStats</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/fastq/BamToBfq.html" title="class in picard.fastq"><span class="typeNameLink">BamToBfq</span></a> - Class in <a href="picard/fastq/package-summary.html">picard.fastq</a></dt>
<dd>
<div class="block">CommandLineProgram to generate to bfq files for use by the Maq aligner</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fastq/BamToBfq.html#BamToBfq--">BamToBfq()</a></span> - Constructor for class picard.fastq.<a href="picard/fastq/BamToBfq.html" title="class in picard.fastq">BamToBfq</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/fastq/BamToBfqWriter.html" title="class in picard.fastq"><span class="typeNameLink">BamToBfqWriter</span></a> - Class in <a href="picard/fastq/package-summary.html">picard.fastq</a></dt>
<dd>
<div class="block">Class to take unmapped reads in BAM file format and create Maq binary fastq format file(s) --
 one or two of them, depending on whether it's a paired-end read.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fastq/BamToBfqWriter.html#BamToBfqWriter-java.io.File-java.lang.String-java.lang.Integer-java.lang.Integer-boolean-java.lang.String-boolean-boolean-java.lang.Integer-">BamToBfqWriter(File, String, Integer, Integer, boolean, String, boolean, boolean, Integer)</a></span> - Constructor for class picard.fastq.<a href="picard/fastq/BamToBfqWriter.html" title="class in picard.fastq">BamToBfqWriter</a></dt>
<dd>
<div class="block">Constructor</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fastq/BamToBfqWriter.html#BamToBfqWriter-java.io.File-java.lang.String-boolean-java.lang.String-boolean-">BamToBfqWriter(File, String, boolean, String, boolean)</a></span> - Constructor for class picard.fastq.<a href="picard/fastq/BamToBfqWriter.html" title="class in picard.fastq">BamToBfqWriter</a></dt>
<dd>
<div class="block">Constructor</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.html#BARCODE">BARCODE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.html" title="class in picard.illumina">ExtractIlluminaBarcodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html#BARCODE">BARCODE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html" title="class in picard.illumina">ExtractIlluminaBarcodes.BarcodeMetric</a></dt>
<dd>
<div class="block">The barcode (from the set of expected barcodes) for which the following metrics apply.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadType.html#Barcode">Barcode</a></span> - Static variable in enum picard.illumina.parser.<a href="picard/illumina/parser/ReadType.html" title="enum in picard.illumina.parser">ReadType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesWithBarcodes.html#barcode">barcode</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesWithBarcodes.html" title="class in picard.sam.markduplicates.util">ReadEndsForMarkDuplicatesWithBarcodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/CollectIndependentReplicateMetrics.html#BARCODE_BQ">BARCODE_BQ</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/CollectIndependentReplicateMetrics.html" title="class in picard.analysis.replicates">CollectIndependentReplicateMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IlluminaUtil.html#BARCODE_DELIMITER">BARCODE_DELIMITER</a></span> - Static variable in class picard.util.<a href="picard/util/IlluminaUtil.html" title="class in picard.util">IlluminaUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.html#BARCODE_FILE">BARCODE_FILE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.html" title="class in picard.illumina">ExtractIlluminaBarcodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html#BARCODE_NAME">BARCODE_NAME</a></span> - Variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html" title="class in picard.illumina">ExtractIlluminaBarcodes.BarcodeMetric</a></dt>
<dd>
<div class="block">The barcode name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.html#BARCODE_NAME_COLUMN">BARCODE_NAME_COLUMN</a></span> - Static variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.html" title="class in picard.illumina">ExtractIlluminaBarcodes</a></dt>
<dd>
<div class="block">Column header for the barcode name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#BARCODE_PARAMS">BARCODE_PARAMS</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.html#BARCODE_SEQUENCE_1_COLUMN">BARCODE_SEQUENCE_1_COLUMN</a></span> - Static variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.html" title="class in picard.illumina">ExtractIlluminaBarcodes</a></dt>
<dd>
<div class="block">Column header for the first barcode sequence.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.html#BARCODE_SEQUENCE_COLUMN">BARCODE_SEQUENCE_COLUMN</a></span> - Static variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.html" title="class in picard.illumina">ExtractIlluminaBarcodes</a></dt>
<dd>
<div class="block">Column header for the first barcode sequence (preferred).</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/CollectIndependentReplicateMetrics.html#BARCODE_TAG">BARCODE_TAG</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/CollectIndependentReplicateMetrics.html" title="class in picard.analysis.replicates">CollectIndependentReplicateMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/EstimateLibraryComplexity.html#BARCODE_TAG">BARCODE_TAG</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/EstimateLibraryComplexity.html" title="class in picard.sam.markduplicates">EstimateLibraryComplexity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicates.html#BARCODE_TAG">BARCODE_TAG</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicates.html" title="class in picard.sam.markduplicates">MarkDuplicates</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html#BARCODE_WITHOUT_DELIMITER">BARCODE_WITHOUT_DELIMITER</a></span> - Variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html" title="class in picard.illumina">ExtractIlluminaBarcodes.BarcodeMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html#barcodeBytes">barcodeBytes</a></span> - Variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html" title="class in picard.illumina">ExtractIlluminaBarcodes.BarcodeMetric</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/fakers/BarcodeFileFaker.html" title="class in picard.illumina.parser.fakers"><span class="typeNameLink">BarcodeFileFaker</span></a> - Class in <a href="picard/illumina/parser/fakers/package-summary.html">picard.illumina.parser.fakers</a></dt>
<dd>
<div class="block">Created by jcarey on 3/13/14.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/BarcodeFileFaker.html#BarcodeFileFaker--">BarcodeFileFaker()</a></span> - Constructor for class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/BarcodeFileFaker.html" title="class in picard.illumina.parser.fakers">BarcodeFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/readers/BarcodeFileReader.html" title="class in picard.illumina.parser.readers"><span class="typeNameLink">BarcodeFileReader</span></a> - Class in <a href="picard/illumina/parser/readers/package-summary.html">picard.illumina.parser.readers</a></dt>
<dd>
<div class="block">Reads a single barcode file line by line and returns the barcode if there was a match or NULL otherwise.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/BarcodeFileReader.html#BarcodeFileReader-java.io.File-">BarcodeFileReader(File)</a></span> - Constructor for class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/BarcodeFileReader.html" title="class in picard.illumina.parser.readers">BarcodeFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.PerTileBarcodeExtractor.BarcodeMatch.html#BarcodeMatch--">BarcodeMatch()</a></span> - Constructor for class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.PerTileBarcodeExtractor.BarcodeMatch.html" title="class in picard.illumina">ExtractIlluminaBarcodes.PerTileBarcodeExtractor.BarcodeMatch</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html#BarcodeMetric-java.lang.String-java.lang.String-java.lang.String-java.lang.String:A-">BarcodeMetric(String, String, String, String[])</a></span> - Constructor for class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html" title="class in picard.illumina">ExtractIlluminaBarcodes.BarcodeMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html#BarcodeMetric--">BarcodeMetric()</a></span> - Constructor for class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html" title="class in picard.illumina">ExtractIlluminaBarcodes.BarcodeMetric</a></dt>
<dd>
<div class="block">This ctor is necessary for when reading metrics from file</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CollectIlluminaBasecallingMetrics.html#BARCODES_DIR">BARCODES_DIR</a></span> - Variable in class picard.illumina.<a href="picard/illumina/CollectIlluminaBasecallingMetrics.html" title="class in picard.illumina">CollectIlluminaBasecallingMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToFastq.html#BARCODES_DIR">BARCODES_DIR</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToFastq.html" title="class in picard.illumina">IlluminaBasecallsToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#BARCODES_DIR">BARCODES_DIR</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IlluminaUtil.html#barcodeSeqsToString-java.util.List-">barcodeSeqsToString(List&lt;String&gt;)</a></span> - Static method in class picard.util.<a href="picard/util/IlluminaUtil.html" title="class in picard.util">IlluminaUtil</a></dt>
<dd>
<div class="block">Concatenates all the barcode sequences with BARCODE_DELIMITER</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/IlluminaUtil.html#barcodeSeqsToString-java.lang.String:A-">barcodeSeqsToString(String[])</a></span> - Static method in class picard.util.<a href="picard/util/IlluminaUtil.html" title="class in picard.util">IlluminaUtil</a></dt>
<dd>
<div class="block">Concatenates all the barcode sequences with BARCODE_DELIMITER</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/IlluminaUtil.html#barcodeSeqsToString-byte:A:A-">barcodeSeqsToString(byte[][])</a></span> - Static method in class picard.util.<a href="picard/util/IlluminaUtil.html" title="class in picard.util">IlluminaUtil</a></dt>
<dd>
<div class="block">Concatenates all the barcode sequences with BARCODE_DELIMITER</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ParameterizedFileUtil.html#base">base</a></span> - Variable in class picard.illumina.parser.<a href="picard/illumina/parser/ParameterizedFileUtil.html" title="class in picard.illumina.parser">ParameterizedFileUtil</a></dt>
<dd>
<div class="block">If you think of the file system as a tree, this is the deepest directory(node) on the tree that
 still contains all of the files for this given type (e.g.</div>
</dd>
<dt><a href="picard/illumina/parser/readers/BaseBclReader.html" title="class in picard.illumina.parser.readers"><span class="typeNameLink">BaseBclReader</span></a> - Class in <a href="picard/illumina/parser/readers/package-summary.html">picard.illumina.parser.readers</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/readers/BaseBclReader.CycleData.html" title="class in picard.illumina.parser.readers"><span class="typeNameLink">BaseBclReader.CycleData</span></a> - Class in <a href="picard/illumina/parser/readers/package-summary.html">picard.illumina.parser.readers</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/readers/BaseBclReader.TileData.html" title="class in picard.illumina.parser.readers"><span class="typeNameLink">BaseBclReader.TileData</span></a> - Class in <a href="picard/illumina/parser/readers/package-summary.html">picard.illumina.parser.readers</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CheckIlluminaDirectory.html#BASECALLS_DIR">BASECALLS_DIR</a></span> - Variable in class picard.illumina.<a href="picard/illumina/CheckIlluminaDirectory.html" title="class in picard.illumina">CheckIlluminaDirectory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CollectIlluminaBasecallingMetrics.html#BASECALLS_DIR">BASECALLS_DIR</a></span> - Variable in class picard.illumina.<a href="picard/illumina/CollectIlluminaBasecallingMetrics.html" title="class in picard.illumina">CollectIlluminaBasecallingMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.html#BASECALLS_DIR">BASECALLS_DIR</a></span> - Variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.html" title="class in picard.illumina">ExtractIlluminaBarcodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToFastq.html#BASECALLS_DIR">BASECALLS_DIR</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToFastq.html" title="class in picard.illumina">IlluminaBasecallsToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#BASECALLS_DIR">BASECALLS_DIR</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.html#BASECALLS_DIR">BASECALLS_DIR</a></span> - Variable in class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/BaseDistributionByCycleMetrics.html" title="class in picard.analysis"><span class="typeNameLink">BaseDistributionByCycleMetrics</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/BaseDistributionByCycleMetrics.html#BaseDistributionByCycleMetrics--">BaseDistributionByCycleMetrics()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/BaseDistributionByCycleMetrics.html" title="class in picard.analysis">BaseDistributionByCycleMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/BaseIlluminaDataProvider.html" title="class in picard.illumina.parser"><span class="typeNameLink">BaseIlluminaDataProvider</span></a> - Class in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>
<div class="block">Parse various formats and versions of Illumina Basecall files, and use them the to populate
 ClusterData objects.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/BaseIlluminaDataProvider.html#BaseIlluminaDataProvider-int-picard.illumina.parser.OutputMapping-">BaseIlluminaDataProvider(int, OutputMapping)</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/BaseIlluminaDataProvider.html" title="class in picard.illumina.parser">BaseIlluminaDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/BclData.html#bases">bases</a></span> - Variable in class picard.illumina.parser.<a href="picard/illumina/parser/BclData.html" title="class in picard.illumina.parser">BclData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fastq/BamToBfq.html#BASES_TO_WRITE">BASES_TO_WRITE</a></span> - Variable in class picard.fastq.<a href="picard/fastq/BamToBfq.html" title="class in picard.fastq">BamToBfq</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/BasicInputParser.html" title="class in picard.util"><span class="typeNameLink">BasicInputParser</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">TextFileParser which reads a single text file.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/BasicInputParser.html#BasicInputParser-boolean-java.io.InputStream...-">BasicInputParser(boolean, InputStream...)</a></span> - Constructor for class picard.util.<a href="picard/util/BasicInputParser.html" title="class in picard.util">BasicInputParser</a></dt>
<dd>
<div class="block">Constructor.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/BasicInputParser.html#BasicInputParser-boolean-int-java.io.InputStream...-">BasicInputParser(boolean, int, InputStream...)</a></span> - Constructor for class picard.util.<a href="picard/util/BasicInputParser.html" title="class in picard.util">BasicInputParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/BasicInputParser.html#BasicInputParser-boolean-java.io.File...-">BasicInputParser(boolean, File...)</a></span> - Constructor for class picard.util.<a href="picard/util/BasicInputParser.html" title="class in picard.util">BasicInputParser</a></dt>
<dd>
<div class="block">Constructor.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/BasicInputParser.html#BasicInputParser-boolean-int-java.io.File...-">BasicInputParser(boolean, int, File...)</a></span> - Constructor for class picard.util.<a href="picard/util/BasicInputParser.html" title="class in picard.util">BasicInputParser</a></dt>
<dd>
<div class="block">Constructor.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/MultiTileFileUtil.html#bci">bci</a></span> - Variable in class picard.illumina.parser.<a href="picard/illumina/parser/MultiTileFileUtil.html" title="class in picard.illumina.parser">MultiTileFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/fakers/BciFileFaker.html" title="class in picard.illumina.parser.fakers"><span class="typeNameLink">BciFileFaker</span></a> - Class in <a href="picard/illumina/parser/fakers/package-summary.html">picard.illumina.parser.fakers</a></dt>
<dd>
<div class="block">Created by jcarey on 3/14/14.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/BciFileFaker.html#BciFileFaker--">BciFileFaker()</a></span> - Constructor for class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/BciFileFaker.html" title="class in picard.illumina.parser.fakers">BciFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/BclData.html" title="class in picard.illumina.parser"><span class="typeNameLink">BclData</span></a> - Class in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>
<div class="block">A class that implements the IlluminaData interfaces provided by this parser
 One BclData object is returned to IlluminaDataProvider per cluster and each
 first level array in bases and qualities represents a single read in that
 cluster</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/BclData.html#BclData-int:A-">BclData(int[])</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/BclData.html" title="class in picard.illumina.parser">BclData</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/fakers/BclFileFaker.html" title="class in picard.illumina.parser.fakers"><span class="typeNameLink">BclFileFaker</span></a> - Class in <a href="picard/illumina/parser/fakers/package-summary.html">picard.illumina.parser.fakers</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/BclFileFaker.html#BclFileFaker--">BclFileFaker()</a></span> - Constructor for class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/BclFileFaker.html" title="class in picard.illumina.parser.fakers">BclFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/readers/BclIndexReader.html" title="class in picard.illumina.parser.readers"><span class="typeNameLink">BclIndexReader</span></a> - Class in <a href="picard/illumina/parser/readers/package-summary.html">picard.illumina.parser.readers</a></dt>
<dd>
<div class="block">Annoyingly, there are two different files with extension .bci in NextSeq output.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/BclIndexReader.html#BclIndexReader-java.io.File-">BclIndexReader(File)</a></span> - Constructor for class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/BclIndexReader.html" title="class in picard.illumina.parser.readers">BclIndexReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/readers/BclQualityEvaluationStrategy.html" title="class in picard.illumina.parser.readers"><span class="typeNameLink">BclQualityEvaluationStrategy</span></a> - Class in <a href="picard/illumina/parser/readers/package-summary.html">picard.illumina.parser.readers</a></dt>
<dd>
<div class="block">Describes a mechanism for revising and evaluating qualities read from a BCL file.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/BclQualityEvaluationStrategy.html#BclQualityEvaluationStrategy-int-">BclQualityEvaluationStrategy(int)</a></span> - Constructor for class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/BclQualityEvaluationStrategy.html" title="class in picard.illumina.parser.readers">BclQualityEvaluationStrategy</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/readers/BclReader.html" title="class in picard.illumina.parser.readers"><span class="typeNameLink">BclReader</span></a> - Class in <a href="picard/illumina/parser/readers/package-summary.html">picard.illumina.parser.readers</a></dt>
<dd>
<div class="block">BCL Files are base call and quality score binary files containing a (base,quality) pair for successive clusters.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/BclReader.html#BclReader-java.util.List-int:A-picard.illumina.parser.readers.BclQualityEvaluationStrategy-boolean-">BclReader(List&lt;File&gt;, int[], BclQualityEvaluationStrategy, boolean)</a></span> - Constructor for class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/BclReader.html" title="class in picard.illumina.parser.readers">BclReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/BclReader.html#BclReader-java.io.File-picard.illumina.parser.readers.BclQualityEvaluationStrategy-boolean-">BclReader(File, BclQualityEvaluationStrategy, boolean)</a></span> - Constructor for class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/BclReader.html" title="class in picard.illumina.parser.readers">BclReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/BedToIntervalList.html" title="class in picard.util"><span class="typeNameLink">BedToIntervalList</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/BedToIntervalList.html#BedToIntervalList--">BedToIntervalList()</a></span> - Constructor for class picard.util.<a href="picard/util/BedToIntervalList.html" title="class in picard.util">BedToIntervalList</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/BestEndMapqPrimaryAlignmentStrategy.html" title="class in picard.sam"><span class="typeNameLink">BestEndMapqPrimaryAlignmentStrategy</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block">For an aligner that aligns each end independently, select the alignment for each end with the best MAPQ, and
 make that the primary.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/BestEndMapqPrimaryAlignmentStrategy.html#BestEndMapqPrimaryAlignmentStrategy--">BestEndMapqPrimaryAlignmentStrategy()</a></span> - Constructor for class picard.sam.<a href="picard/sam/BestEndMapqPrimaryAlignmentStrategy.html" title="class in picard.sam">BestEndMapqPrimaryAlignmentStrategy</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/BestMapqPrimaryAlignmentSelectionStrategy.html" title="class in picard.sam"><span class="typeNameLink">BestMapqPrimaryAlignmentSelectionStrategy</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block">This strategy was designed for TopHat output, but could be of general utility.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/BestMapqPrimaryAlignmentSelectionStrategy.html#BestMapqPrimaryAlignmentSelectionStrategy--">BestMapqPrimaryAlignmentSelectionStrategy()</a></span> - Constructor for class picard.sam.<a href="picard/sam/BestMapqPrimaryAlignmentSelectionStrategy.html" title="class in picard.sam">BestMapqPrimaryAlignmentSelectionStrategy</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#biSiteHeterogeneityRate">biSiteHeterogeneityRate</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#biSiteHomogeneityRate">biSiteHomogeneityRate</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigar.html#BLOCK_SIZE">BLOCK_SIZE</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigar.html" title="class in picard.sam.markduplicates">MarkDuplicatesWithMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IntervalListTools.html#BREAK_BANDS_AT_MULTIPLES_OF">BREAK_BANDS_AT_MULTIPLES_OF</a></span> - Variable in class picard.util.<a href="picard/util/IntervalListTools.html" title="class in picard.util">IntervalListTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/FifoBuffer.html#BUFFER_SIZE">BUFFER_SIZE</a></span> - Variable in class picard.util.<a href="picard/util/FifoBuffer.html" title="class in picard.util">FifoBuffer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/BarcodeFileFaker.html#bufferSize--">bufferSize()</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/BarcodeFileFaker.html" title="class in picard.illumina.parser.fakers">BarcodeFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/BciFileFaker.html#bufferSize--">bufferSize()</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/BciFileFaker.html" title="class in picard.illumina.parser.fakers">BciFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/BclFileFaker.html#bufferSize--">bufferSize()</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/BclFileFaker.html" title="class in picard.illumina.parser.fakers">BclFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/ClocsFileFaker.html#bufferSize--">bufferSize()</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/ClocsFileFaker.html" title="class in picard.illumina.parser.fakers">ClocsFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/FileFaker.html#bufferSize--">bufferSize()</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/FileFaker.html" title="class in picard.illumina.parser.fakers">FileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/FilterFileFaker.html#bufferSize--">bufferSize()</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/FilterFileFaker.html" title="class in picard.illumina.parser.fakers">FilterFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/LocsFileFaker.html#bufferSize--">bufferSize()</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/LocsFileFaker.html" title="class in picard.illumina.parser.fakers">LocsFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/MultiTileBclFileFaker.html#bufferSize--">bufferSize()</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/MultiTileBclFileFaker.html" title="class in picard.illumina.parser.fakers">MultiTileBclFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/MultiTileLocsFileFaker.html#bufferSize--">bufferSize()</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/MultiTileLocsFileFaker.html" title="class in picard.illumina.parser.fakers">MultiTileLocsFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/PosFileFaker.html#bufferSize--">bufferSize()</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/PosFileFaker.html" title="class in picard.illumina.parser.fakers">PosFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VariantProcessor.AccumulatorGenerator.html#build--">build()</a></span> - Method in interface picard.vcf.processor.<a href="picard/vcf/processor/VariantProcessor.AccumulatorGenerator.html" title="interface in picard.vcf.processor">VariantProcessor.AccumulatorGenerator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VariantProcessor.Builder.html#build--">build()</a></span> - Method in class picard.vcf.processor.<a href="picard/vcf/processor/VariantProcessor.Builder.html" title="class in picard.vcf.processor">VariantProcessor.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/BuildBamIndex.html" title="class in picard.sam"><span class="typeNameLink">BuildBamIndex</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block">Command line program to generate a BAM index (.bai) file from a BAM (.bam) file</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/BuildBamIndex.html#BuildBamIndex--">BuildBamIndex()</a></span> - Constructor for class picard.sam.<a href="picard/sam/BuildBamIndex.html" title="class in picard.sam">BuildBamIndex</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VariantIteratorProducer.html#byHundredMegabaseChunks-java.util.List-">byHundredMegabaseChunks(List&lt;File&gt;)</a></span> - Static method in class picard.vcf.processor.<a href="picard/vcf/processor/VariantIteratorProducer.html" title="class in picard.vcf.processor">VariantIteratorProducer</a></dt>
<dd>
<div class="block">Produces a chunking with segments of size 100 megabases (or less if a contig boundary is reached).</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VariantIteratorProducer.html#byHundredMegabaseChunksWithOnTheFlyFilteringByInterval-java.util.List-htsjdk.samtools.util.IntervalList-">byHundredMegabaseChunksWithOnTheFlyFilteringByInterval(List&lt;File&gt;, IntervalList)</a></span> - Static method in class picard.vcf.processor.<a href="picard/vcf/processor/VariantIteratorProducer.html" title="class in picard.vcf.processor">VariantIteratorProducer</a></dt>
<dd>
<div class="block">Produces a chunking with segments of size 100 megabases (or less if a contig boundary is reached), that also performs on-the-fly
 filtering of <code>VariantContext</code></div>
</dd>
<dt><a href="picard/vcf/ByIntervalListVariantContextIterator.html" title="class in picard.vcf"><span class="typeNameLink">ByIntervalListVariantContextIterator</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">Takes a VCFFileReader and an IntervalList and provides a single iterator over all variants in all the intervals.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/ByIntervalListVariantContextIterator.html#ByIntervalListVariantContextIterator-htsjdk.variant.vcf.VCFFileReader-htsjdk.samtools.util.IntervalList-">ByIntervalListVariantContextIterator(VCFFileReader, IntervalList)</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/ByIntervalListVariantContextIterator.html" title="class in picard.vcf">ByIntervalListVariantContextIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IlluminaUtil.html#byteArrayToString-byte:A:A-java.lang.String-">byteArrayToString(byte[][], String)</a></span> - Static method in class picard.util.<a href="picard/util/IlluminaUtil.html" title="class in picard.util">IlluminaUtil</a></dt>
<dd>
<div class="block">Concatenates all the barcode sequences with BARCODE_DELIMITER</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VcfFileSegmentGenerator.html#byWholeContigSubdividingWithWidth-long-">byWholeContigSubdividingWithWidth(long)</a></span> - Static method in class picard.vcf.processor.<a href="picard/vcf/processor/VcfFileSegmentGenerator.html" title="class in picard.vcf.processor">VcfFileSegmentGenerator</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="I:C">
<!--   -->
</a>
<h2 class="title">C</h2>
<dl>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRrbsMetrics.html#C_QUALITY_THRESHOLD">C_QUALITY_THRESHOLD</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectRrbsMetrics.html" title="class in picard.analysis">CollectRrbsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html#C_REF_ALT_BASES">C_REF_ALT_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html" title="class in picard.analysis">CollectOxoGMetrics.CpcgMetrics</a></dt>
<dd>
<div class="block">The number of alt (A/T) basecalls observed at sites where the genome reference == C.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html#C_REF_OXO_ERROR_RATE">C_REF_OXO_ERROR_RATE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html" title="class in picard.analysis">CollectOxoGMetrics.CpcgMetrics</a></dt>
<dd>
<div class="block">The rate at which C>A and G>T substitutions are observed at C reference sites above the expected rate if there
 were no bias between sites with a C reference base vs.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html#C_REF_OXO_Q">C_REF_OXO_Q</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html" title="class in picard.analysis">CollectOxoGMetrics.CpcgMetrics</a></dt>
<dd>
<div class="block">C_REF_OXO_ERROR_RATE expressed as a phred-scaled quality score.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html#C_REF_REF_BASES">C_REF_REF_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html" title="class in picard.analysis">CollectOxoGMetrics.CpcgMetrics</a></dt>
<dd>
<div class="block">The number of ref basecalls observed at sites where the genome reference == C.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasUtils.html#calculateAllGcs-byte:A-int-int-">calculateAllGcs(byte[], int, int)</a></span> - Static method in class picard.analysis.<a href="picard/analysis/GcBiasUtils.html" title="class in picard.analysis">GcBiasUtils</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/ErrorSummaryMetrics.html#calculateDerivedFields--">calculateDerivedFields()</a></span> - Method in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/ErrorSummaryMetrics.html" title="class in picard.analysis.artifacts">ErrorSummaryMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#calculateDerivedFields--">calculateDerivedFields()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/MergeableMetricBase.html#calculateDerivedFields--">calculateDerivedFields()</a></span> - Method in class picard.analysis.<a href="picard/analysis/MergeableMetricBase.html" title="class in picard.analysis">MergeableMetricBase</a></dt>
<dd>
<div class="block">placeholder method that will calculate the derived fields from the other ones.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#calculateDerivedFields--">calculateDerivedFields()</a></span> - Method in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html#calculateDerivedFields--">calculateDerivedFields()</a></span> - Method in class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics.PFFailSummaryMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/DuplicationMetrics.html#calculateDerivedFields--">calculateDerivedFields()</a></span> - Method in class picard.sam.<a href="picard/sam/DuplicationMetrics.html" title="class in picard.sam">DuplicationMetrics</a></dt>
<dd>
<div class="block">Fills in the ESTIMATED_LIBRARY_SIZE based on the paired read data examined where
 possible and the PERCENT_DUPLICATION.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/UmiMetrics.html#calculateDerivedFields--">calculateDerivedFields()</a></span> - Method in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/UmiMetrics.html" title="class in picard.sam.markduplicates">UmiMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingDetailMetrics.html#calculateDerivedFields--">calculateDerivedFields()</a></span> - Method in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingDetailMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingDetailMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html#calculateDerivedFields--">calculateDerivedFields()</a></span> - Method in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingSummaryMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html#calculateDerivedFields--">calculateDerivedFields()</a></span> - Method in class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html" title="class in picard.vcf.MendelianViolations">MendelianViolationMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/DuplicationMetrics.html#calculateDerivedMetrics--">calculateDerivedMetrics()</a></span> - Method in class picard.sam.<a href="picard/sam/DuplicationMetrics.html" title="class in picard.sam">DuplicationMetrics</a></dt>
<dd>
<div class="block"><span class="deprecatedLabel">Deprecated.</span></div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html#calculateDerivedStatistics--">calculateDerivedStatistics()</a></span> - Method in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.BaitBiasDetailMetrics</a></dt>
<dd>
<div class="block">Calculate the error rate given the raw counts.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterDetailMetrics.html#calculateDerivedStatistics--">calculateDerivedStatistics()</a></span> - Method in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterDetailMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.PreAdapterDetailMetrics</a></dt>
<dd>
<div class="block">Calculate the error rate given the raw counts.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingDetailMetrics.html#calculateFromDerivedFields--">calculateFromDerivedFields()</a></span> - Method in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingDetailMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingDetailMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html#calculateFromDerivedFields-long-">calculateFromDerivedFields(long)</a></span> - Method in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingSummaryMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasUtils.html#calculateGc-byte:A-int-int-picard.analysis.GcBiasUtils.CalculateGcState-">calculateGc(byte[], int, int, GcBiasUtils.CalculateGcState)</a></span> - Static method in class picard.analysis.<a href="picard/analysis/GcBiasUtils.html" title="class in picard.analysis">GcBiasUtils</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceCounts.html#calculateGenotypeConcordance-picard.vcf.GenotypeConcordanceScheme-boolean-">calculateGenotypeConcordance(GenotypeConcordanceScheme, boolean)</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceCounts.html" title="class in picard.vcf">GenotypeConcordanceCounts</a></dt>
<dd>
<div class="block">Genotype Concordance is the number of times the truth and call states match exactly / all truth and call combinations made
 If the GA4GH scheme is being used, any MISSING sites in truth OR call will not be included in the discordance calculations.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintChecker.html#calculateMatchResults-picard.fingerprint.Fingerprint-picard.fingerprint.Fingerprint-double-double-">calculateMatchResults(Fingerprint, Fingerprint, double, double)</a></span> - Static method in class picard.fingerprint.<a href="picard/fingerprint/FingerprintChecker.html" title="class in picard.fingerprint">FingerprintChecker</a></dt>
<dd>
<div class="block">Compares two fingerprints and calculates a MatchResults object which contains detailed
 information about the match (or mismatch) between fingerprints including the LOD score
 for whether or not the two are likely from the same sample.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintChecker.html#calculateMatchResults-picard.fingerprint.Fingerprint-picard.fingerprint.Fingerprint-">calculateMatchResults(Fingerprint, Fingerprint)</a></span> - Static method in class picard.fingerprint.<a href="picard/fingerprint/FingerprintChecker.html" title="class in picard.fingerprint">FingerprintChecker</a></dt>
<dd>
<div class="block">Compares two fingerprints and calculates a MatchResults object which contains detailed
 information about the match (or mismatch) between fingerprints including the LOD score
 for whether or not the two are likely from the same sample.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceCounts.html#calculateNonRefGenotypeConcordance-picard.vcf.GenotypeConcordanceScheme-boolean-">calculateNonRefGenotypeConcordance(GenotypeConcordanceScheme, boolean)</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceCounts.html" title="class in picard.vcf">GenotypeConcordanceCounts</a></dt>
<dd>
<div class="block">Non Ref Genotype Concordance is the number of times the truth and call states match exactly for *vars only* / all truth and call *var* combinations made
 If the GA4GH scheme is being used, any MISSING sites in truth OR call will not be included in the discordance calculations.</div>
</dd>
<dt><a href="picard/sam/CalculateReadGroupChecksum.html" title="class in picard.sam"><span class="typeNameLink">CalculateReadGroupChecksum</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CalculateReadGroupChecksum.html#CalculateReadGroupChecksum--">CalculateReadGroupChecksum()</a></span> - Constructor for class picard.sam.<a href="picard/sam/CalculateReadGroupChecksum.html" title="class in picard.sam">CalculateReadGroupChecksum</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasUtils.html#calculateRefWindowsByGc-int-java.io.File-int-">calculateRefWindowsByGc(int, File, int)</a></span> - Static method in class picard.analysis.<a href="picard/analysis/GcBiasUtils.html" title="class in picard.analysis">GcBiasUtils</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/DuplicationMetrics.html#calculateRoiHistogram--">calculateRoiHistogram()</a></span> - Method in class picard.sam.<a href="picard/sam/DuplicationMetrics.html" title="class in picard.sam">DuplicationMetrics</a></dt>
<dd>
<div class="block">Calculates a histogram using the estimateRoi method to estimate the effective yield
 doing x sequencing for x=1..10.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/AbstractInputParser.html#calculateWordCount-byte:A-">calculateWordCount(byte[])</a></span> - Method in class picard.util.<a href="picard/util/AbstractInputParser.html" title="class in picard.util">AbstractInputParser</a></dt>
<dd>
<div class="block">Calculates the number of delimiter-separated "words" in a line and sets the value of <code>wordCount</code></div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#CALL_SAMPLE">CALL_SAMPLE</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceContingencyMetrics.html#CALL_SAMPLE">CALL_SAMPLE</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceContingencyMetrics.html" title="class in picard.vcf">GenotypeConcordanceContingencyMetrics</a></dt>
<dd>
<div class="block">The name of the 'call' sample</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceDetailMetrics.html#CALL_SAMPLE">CALL_SAMPLE</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceDetailMetrics.html" title="class in picard.vcf">GenotypeConcordanceDetailMetrics</a></dt>
<dd>
<div class="block">The name of the 'call' sample</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html#CALL_SAMPLE">CALL_SAMPLE</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html" title="class in picard.vcf">GenotypeConcordanceSummaryMetrics</a></dt>
<dd>
<div class="block">The name of the 'call' sample</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceDetailMetrics.html#CALL_STATE">CALL_STATE</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceDetailMetrics.html" title="class in picard.vcf">GenotypeConcordanceDetailMetrics</a></dt>
<dd>
<div class="block">The state of the 'call' sample (i.e.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#CALL_VCF">CALL_VCF</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.Alleles.html#callAllele1">callAllele1</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.Alleles.html" title="class in picard.vcf">GenotypeConcordance.Alleles</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.Alleles.html#callAllele2">callAllele2</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.Alleles.html" title="class in picard.vcf">GenotypeConcordance.Alleles</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.Alleles.html#callAlleles--">callAlleles()</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.Alleles.html" title="class in picard.vcf">GenotypeConcordance.Alleles</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/CallingMetricAccumulator.html" title="class in picard.vcf"><span class="typeNameLink">CallingMetricAccumulator</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">Collects variants and generates metrics about them.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CallingMetricAccumulator.html#CallingMetricAccumulator-picard.util.DbSnpBitSetUtil.DbSnpBitSets-">CallingMetricAccumulator(DbSnpBitSetUtil.DbSnpBitSets)</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/CallingMetricAccumulator.html" title="class in picard.vcf">CallingMetricAccumulator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/CallingMetricAccumulator.Result.html" title="class in picard.vcf"><span class="typeNameLink">CallingMetricAccumulator.Result</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceStates.TruthAndCallStates.html#callState">callState</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceStates.TruthAndCallStates.html" title="class in picard.vcf">GenotypeConcordanceStates.TruthAndCallStates</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/MergeableMetricBase.html#canMerge-picard.analysis.MergeableMetricBase-">canMerge(MergeableMetricBase)</a></span> - Method in class picard.analysis.<a href="picard/analysis/MergeableMetricBase.html" title="class in picard.analysis">MergeableMetricBase</a></dt>
<dd>
<div class="block">checks if this instance can be merged with another

 Other must have all the fields that this instance has, and
 the fields that are annotated as MergeByAssertEquals must contain the same value</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertOriginalBaseQualitiesAndAddMateCigar.CanSkipSamFile.html#canSkip--">canSkip()</a></span> - Method in enum picard.sam.<a href="picard/sam/RevertOriginalBaseQualitiesAndAddMateCigar.CanSkipSamFile.html" title="enum in picard.sam">RevertOriginalBaseQualitiesAndAddMateCigar.CanSkipSamFile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertOriginalBaseQualitiesAndAddMateCigar.html#canSkipSAMFile-java.io.File-int-boolean-java.io.File-">canSkipSAMFile(File, int, boolean, File)</a></span> - Static method in class picard.sam.<a href="picard/sam/RevertOriginalBaseQualitiesAndAddMateCigar.html" title="class in picard.sam">RevertOriginalBaseQualitiesAndAddMateCigar</a></dt>
<dd>
<div class="block">Checks if we can skip the SAM/BAM file when reverting origin base qualities and adding mate cigars.</div>
</dd>
<dt><a href="picard/fastq/Casava18ReadNameEncoder.html" title="class in picard.fastq"><span class="typeNameLink">Casava18ReadNameEncoder</span></a> - Class in <a href="picard/fastq/package-summary.html">picard.fastq</a></dt>
<dd>
<div class="block">A read name encoder conforming to the standard described by Illumina Casava 1.8.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fastq/Casava18ReadNameEncoder.html#Casava18ReadNameEncoder-java.lang.String-java.lang.String-java.lang.String-">Casava18ReadNameEncoder(String, String, String)</a></span> - Constructor for class picard.fastq.<a href="picard/fastq/Casava18ReadNameEncoder.html" title="class in picard.fastq">Casava18ReadNameEncoder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AlignmentSummaryMetrics.html#CATEGORY">CATEGORY</a></span> - Variable in class picard.analysis.<a href="picard/analysis/AlignmentSummaryMetrics.html" title="class in picard.analysis">AlignmentSummaryMetrics</a></dt>
<dd>
<div class="block">One of either UNPAIRED (for a fragment run), FIRST_OF_PAIR when metrics are for only the
 first read in a paired run, SECOND_OF_PAIR when the metrics are for only the second read
 in a paired run or PAIR when the metrics are aggregated for both first and second reads
 in a pair.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverage.html#CATEGORY">CATEGORY</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverage.html" title="class in picard.analysis">CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverage</a></dt>
<dd>
<div class="block">One of either WHOLE_GENOME or NON_ZERO_REGIONS</div>
</dd>
<dt><a href="picard/illumina/parser/CbclData.html" title="class in picard.illumina.parser"><span class="typeNameLink">CbclData</span></a> - Class in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>
<div class="block">This class provides that data structure for cbcls.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/CbclData.html#CbclData-int:A-int-">CbclData(int[], int)</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/CbclData.html" title="class in picard.illumina.parser">CbclData</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/readers/CbclReader.html" title="class in picard.illumina.parser.readers"><span class="typeNameLink">CbclReader</span></a> - Class in <a href="picard/illumina/parser/readers/package-summary.html">picard.illumina.parser.readers</a></dt>
<dd>
<div class="block">------------------------------------- CBCL Header -----------------------------------
 Bytes 0 - 1     Version number, current version is 1    unsigned 16 bits little endian integer
 Bytes 2 - 5     Header size                             unsigned 32 bits little endian integer
 Byte 6          Number of bits per basecall             unsigned
 Byte 7          Number of bits per q-score              unsigned</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/CbclReader.html#CbclReader-java.util.List-java.util.Map-int:A-int-java.util.List-int:A-boolean-">CbclReader(List&lt;File&gt;, Map&lt;Integer, File&gt;, int[], int, List&lt;AbstractIlluminaPositionFileReader.PositionInfo&gt;, int[], boolean)</a></span> - Constructor for class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/CbclReader.html" title="class in picard.illumina.parser.readers">CbclReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/LiftOverIntervalList.html#CHAIN">CHAIN</a></span> - Variable in class picard.util.<a href="picard/util/LiftOverIntervalList.html" title="class in picard.util">LiftOverIntervalList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/LiftoverVcf.html#CHAIN">CHAIN</a></span> - Variable in class picard.vcf.<a href="picard/vcf/LiftoverVcf.html" title="class in picard.vcf">LiftoverVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectBaseDistributionByCycle.html#CHART_OUTPUT">CHART_OUTPUT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectBaseDistributionByCycle.html" title="class in picard.analysis">CollectBaseDistributionByCycle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectGcBiasMetrics.html#CHART_OUTPUT">CHART_OUTPUT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectGcBiasMetrics.html" title="class in picard.analysis">CollectGcBiasMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRnaSeqMetrics.html#CHART_OUTPUT">CHART_OUTPUT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectRnaSeqMetrics.html" title="class in picard.analysis">CollectRnaSeqMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.html#CHART_OUTPUT">CHART_OUTPUT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.html" title="class in picard.analysis">CollectWgsMetricsWithNonZeroCoverage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/MeanQualityByCycle.html#CHART_OUTPUT">CHART_OUTPUT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/MeanQualityByCycle.html" title="class in picard.analysis">MeanQualityByCycle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/QualityScoreDistribution.html#CHART_OUTPUT">CHART_OUTPUT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/QualityScoreDistribution.html" title="class in picard.analysis">QualityScoreDistribution</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/FixVcfHeader.html#CHECK_FIRST_N_RECORDS">CHECK_FIRST_N_RECORDS</a></span> - Variable in class picard.vcf.<a href="picard/vcf/FixVcfHeader.html" title="class in picard.vcf">FixVcfHeader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/fingerprint/CheckFingerprint.html" title="class in picard.fingerprint"><span class="typeNameLink">CheckFingerprint</span></a> - Class in <a href="picard/fingerprint/package-summary.html">picard.fingerprint</a></dt>
<dd>
<div class="block">Attempts to check the sample identity of the sequence/genotype data in the provided file (SAM/BAM or VCF)
 against a set of known genotypes in the supplied genotype file (in either GELI or VCF format).</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CheckFingerprint.html#CheckFingerprint--">CheckFingerprint()</a></span> - Constructor for class picard.fingerprint.<a href="picard/fingerprint/CheckFingerprint.html" title="class in picard.fingerprint">CheckFingerprint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintChecker.html#checkFingerprints-java.util.List-java.util.List-java.lang.String-boolean-">checkFingerprints(List&lt;File&gt;, List&lt;File&gt;, String, boolean)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/FingerprintChecker.html" title="class in picard.fingerprint">FingerprintChecker</a></dt>
<dd>
<div class="block">Top level method to take a set of one or more SAM files and one or more Genotype files and compare
 each read group in each SAM file to each set of fingerprint genotypes.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintChecker.html#checkFingerprints-java.util.List-java.util.List-java.lang.String-java.lang.String-">checkFingerprints(List&lt;File&gt;, List&lt;File&gt;, String, String)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/FingerprintChecker.html" title="class in picard.fingerprint">FingerprintChecker</a></dt>
<dd>
<div class="block">Top level method to take a set of one or more observed genotype (VCF) files and one or more expected genotype (VCF) files and compare
 one or more sample in the observed genotype file with one or more in the expected file and generate results for each set.</div>
</dd>
<dt><a href="picard/illumina/CheckIlluminaDirectory.html" title="class in picard.illumina"><span class="typeNameLink">CheckIlluminaDirectory</span></a> - Class in <a href="picard/illumina/package-summary.html">picard.illumina</a></dt>
<dd>
<div class="block">Program to check a lane of an Illumina output directory.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CheckIlluminaDirectory.html#CheckIlluminaDirectory--">CheckIlluminaDirectory()</a></span> - Constructor for class picard.illumina.<a href="picard/illumina/CheckIlluminaDirectory.html" title="class in picard.illumina">CheckIlluminaDirectory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CounterManager.html#checkOutOfBounds-int-">checkOutOfBounds(int)</a></span> - Method in class picard.analysis.<a href="picard/analysis/CounterManager.html" title="class in picard.analysis">CounterManager</a></dt>
<dd>
<div class="block">Method checks that new locus position is not out of bounds of Counter arrays and there is enough
 space in them to hold information on at least one more read of length <a href="picard/analysis/CounterManager.html#readLength"><code>CounterManager.readLength</code></a>.</div>
</dd>
<dt><a href="picard/sam/CheckTerminatorBlock.html" title="class in picard.sam"><span class="typeNameLink">CheckTerminatorBlock</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block">Simple class to check the terminator block of a SAM file.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CheckTerminatorBlock.html#CheckTerminatorBlock--">CheckTerminatorBlock()</a></span> - Constructor for class picard.sam.<a href="picard/sam/CheckTerminatorBlock.html" title="class in picard.sam">CheckTerminatorBlock</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectJumpingLibraryMetrics.html#CHIMERA_KB_MIN">CHIMERA_KB_MIN</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectJumpingLibraryMetrics.html" title="class in picard.analysis">CollectJumpingLibraryMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/ChimeraUtil.html" title="class in picard.analysis"><span class="typeNameLink">ChimeraUtil</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/ChimeraUtil.html#ChimeraUtil--">ChimeraUtil()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/ChimeraUtil.html" title="class in picard.analysis">ChimeraUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/JumpingLibraryMetrics.html#CHIMERIC_PAIRS">CHIMERIC_PAIRS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/JumpingLibraryMetrics.html" title="class in picard.analysis">JumpingLibraryMetrics</a></dt>
<dd>
<div class="block">The number of pairs where either (a) the ends fall on different chromosomes or (b) the insert size
 is greater than the maximum of 100000 or 2 times the mode of the insert size for outward-facing pairs.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/FingerprintingDetailMetrics.html#CHROM">CHROM</a></span> - Variable in class picard.analysis.<a href="picard/analysis/FingerprintingDetailMetrics.html" title="class in picard.analysis">FingerprintingDetailMetrics</a></dt>
<dd>
<div class="block">The chromosome on which the SNP resides.</div>
</dd>
<dt><a href="picard/util/CircularByteBuffer.html" title="class in picard.util"><span class="typeNameLink">CircularByteBuffer</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">Implementation of a circular byte buffer that uses a large byte[] internally and supports basic
 read/write operations from/to other byte[]s passed as arguments.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/CircularByteBuffer.html#CircularByteBuffer-int-">CircularByteBuffer(int)</a></span> - Constructor for class picard.util.<a href="picard/util/CircularByteBuffer.html" title="class in picard.util">CircularByteBuffer</a></dt>
<dd>
<div class="block">Constructs a buffer capable of holding the given number of bytes.</div>
</dd>
<dt><a href="picard/cmdline/ClassFinder.html" title="class in picard.cmdline"><span class="typeNameLink">ClassFinder</span></a> - Class in <a href="picard/cmdline/package-summary.html">picard.cmdline</a></dt>
<dd>
<div class="block">Utility class that can scan for classes in the classpath and find all the ones
 annotated with a particular annotation.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/ClassFinder.html#ClassFinder--">ClassFinder()</a></span> - Constructor for class picard.cmdline.<a href="picard/cmdline/ClassFinder.html" title="class in picard.cmdline">ClassFinder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/ClassFinder.html#ClassFinder-java.lang.ClassLoader-">ClassFinder(ClassLoader)</a></span> - Constructor for class picard.cmdline.<a href="picard/cmdline/ClassFinder.html" title="class in picard.cmdline">ClassFinder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/ClassFinder.html#ClassFinder-java.io.File-">ClassFinder(File)</a></span> - Constructor for class picard.cmdline.<a href="picard/cmdline/ClassFinder.html" title="class in picard.cmdline">ClassFinder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailDetailedMetric.html#CLASSIFICATION">CLASSIFICATION</a></span> - Variable in class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailDetailedMetric.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics.PFFailDetailedMetric</a></dt>
<dd>
<div class="block">The classification of this read: {EMPTY, POLYCLONAL, MISALIGNED, UNKNOWN}
 (See PFFailSummaryMetric for explanation regarding the possible classification.)</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#classifyVariants-java.util.Optional-java.lang.String-java.util.Optional-java.lang.String-int-int-boolean-">classifyVariants(Optional&lt;VariantContext&gt;, String, Optional&lt;VariantContext&gt;, String, int, int, boolean)</a></span> - Static method in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#classifyVariants-java.util.Optional-java.lang.String-java.util.Optional-java.lang.String-java.util.Optional-java.util.Optional-int-int-boolean-">classifyVariants(Optional&lt;VariantContext&gt;, String, Optional&lt;VariantContext&gt;, String, Optional&lt;GenotypeConcordanceCounts&gt;, Optional&lt;GenotypeConcordanceCounts&gt;, int, int, boolean)</a></span> - Static method in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>
<div class="block">Attempts to determine the concordance state given the truth and all variant context and optionally increments the genotype concordance
 count for the given variant type (SNP or INDEL).</div>
</dd>
<dt><a href="picard/sam/CleanSam.html" title="class in picard.sam"><span class="typeNameLink">CleanSam</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CleanSam.html#CleanSam--">CleanSam()</a></span> - Constructor for class picard.sam.<a href="picard/sam/CleanSam.html" title="class in picard.sam">CleanSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CounterManager.html#clear--">clear()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CounterManager.html" title="class in picard.analysis">CounterManager</a></dt>
<dd>
<div class="block">Clears all inner Counter arrays</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/CbclReader.html#clear--">clear()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/CbclReader.html" title="class in picard.illumina.parser.readers">CbclReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicates.html#CLEAR_DT">CLEAR_DT</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicates.html" title="class in picard.sam.markduplicates">MarkDuplicates</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fastq/BamToBfq.html#CLIP_ADAPTERS">CLIP_ADAPTERS</a></span> - Variable in class picard.fastq.<a href="picard/fastq/BamToBfq.html" title="class in picard.fastq">BamToBfq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#CLIP_ADAPTERS">CLIP_ADAPTERS</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/CollectTargetedMetrics.html#CLIP_OVERLAPPING_READS">CLIP_OVERLAPPING_READS</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/CollectTargetedMetrics.html" title="class in picard.analysis.directed">CollectTargetedMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#CLIP_OVERLAPPING_READS">CLIP_OVERLAPPING_READS</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.html#clipForOverlappingReads-htsjdk.samtools.SAMRecord-htsjdk.samtools.SAMRecord-">clipForOverlappingReads(SAMRecord, SAMRecord)</a></span> - Static method in class picard.sam.<a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</a></dt>
<dd>
<div class="block">Checks to see whether the ends of the reads overlap and soft clips reads
 them if necessary.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamToFastq.html#CLIPPING_ACTION">CLIPPING_ACTION</a></span> - Variable in class picard.sam.<a href="picard/sam/SamToFastq.html" title="class in picard.sam">SamToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamToFastq.html#CLIPPING_ATTRIBUTE">CLIPPING_ATTRIBUTE</a></span> - Variable in class picard.sam.<a href="picard/sam/SamToFastq.html" title="class in picard.sam">SamToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamToFastq.html#CLIPPING_MIN_LENGTH">CLIPPING_MIN_LENGTH</a></span> - Variable in class picard.sam.<a href="picard/sam/SamToFastq.html" title="class in picard.sam">SamToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/ClippingUtility.html" title="class in picard.util"><span class="typeNameLink">ClippingUtility</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">Utilities to clip the adapter sequence from a SAMRecord read</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/ClippingUtility.html#ClippingUtility--">ClippingUtility()</a></span> - Constructor for class picard.util.<a href="picard/util/ClippingUtility.html" title="class in picard.util">ClippingUtility</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/fakers/ClocsFileFaker.html" title="class in picard.illumina.parser.fakers"><span class="typeNameLink">ClocsFileFaker</span></a> - Class in <a href="picard/illumina/parser/fakers/package-summary.html">picard.illumina.parser.fakers</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/ClocsFileFaker.html#ClocsFileFaker--">ClocsFileFaker()</a></span> - Constructor for class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/ClocsFileFaker.html" title="class in picard.illumina.parser.fakers">ClocsFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/readers/ClocsFileReader.html" title="class in picard.illumina.parser.readers"><span class="typeNameLink">ClocsFileReader</span></a> - Class in <a href="picard/illumina/parser/readers/package-summary.html">picard.illumina.parser.readers</a></dt>
<dd>
<div class="block">The clocs file format is one of 3 Illumina formats(pos, locs, and clocs) that stores position data exclusively.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/ClocsFileReader.html#ClocsFileReader-java.io.File-">ClocsFileReader(File)</a></span> - Constructor for class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/ClocsFileReader.html" title="class in picard.illumina.parser.readers">ClocsFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicates.html#clone--">clone()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicates.html" title="class in picard.sam.markduplicates.util">ReadEndsForMarkDuplicates</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesCodec.html#clone--">clone()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesCodec.html" title="class in picard.sam.markduplicates.util">ReadEndsForMarkDuplicatesCodec</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesWithBarcodes.html#clone--">clone()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesWithBarcodes.html" title="class in picard.sam.markduplicates.util">ReadEndsForMarkDuplicatesWithBarcodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesWithBarcodesCodec.html#clone--">clone()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesWithBarcodesCodec.html" title="class in picard.sam.markduplicates.util">ReadEndsForMarkDuplicatesWithBarcodesCodec</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/RepresentativeReadIndexerCodec.html#clone--">clone()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/RepresentativeReadIndexerCodec.html" title="class in picard.sam.markduplicates.util">RepresentativeReadIndexerCodec</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/BaseIlluminaDataProvider.html#close--">close()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/BaseIlluminaDataProvider.html" title="class in picard.illumina.parser">BaseIlluminaDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaDataProvider.html#close--">close()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/IlluminaDataProvider.html" title="class in picard.illumina.parser">IlluminaDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/MultiTileFilterParser.html#close--">close()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/MultiTileFilterParser.html" title="class in picard.illumina.parser">MultiTileFilterParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/MultiTileLocsParser.html#close--">close()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/MultiTileLocsParser.html" title="class in picard.illumina.parser">MultiTileLocsParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/NewIlluminaDataProvider.html#close--">close()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/NewIlluminaDataProvider.html" title="class in picard.illumina.parser">NewIlluminaDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PerTileParser.html#close--">close()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/PerTileParser.html" title="class in picard.illumina.parser">PerTileParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/BarcodeFileReader.html#close--">close()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/BarcodeFileReader.html" title="class in picard.illumina.parser.readers">BarcodeFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/BclReader.html#close--">close()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/BclReader.html" title="class in picard.illumina.parser.readers">BclReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/CbclReader.html#close--">close()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/CbclReader.html" title="class in picard.illumina.parser.readers">CbclReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/ClocsFileReader.html#close--">close()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/ClocsFileReader.html" title="class in picard.illumina.parser.readers">ClocsFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/LocsFileReader.html#close--">close()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/LocsFileReader.html" title="class in picard.illumina.parser.readers">LocsFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/PosFileReader.html#close--">close()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/PosFileReader.html" title="class in picard.illumina.parser.readers">PosFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.html#close--">close()</a></span> - Method in class picard.sam.<a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigarIterator.html#close--">close()</a></span> - Method in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigarIterator.html" title="class in picard.sam.markduplicates">MarkDuplicatesWithMateCigarIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AbstractInputParser.html#close--">close()</a></span> - Method in class picard.util.<a href="picard/util/AbstractInputParser.html" title="class in picard.util">AbstractInputParser</a></dt>
<dd>
<div class="block">Closes this stream and releases any system resources associated with it.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/AsyncIterator.html#close--">close()</a></span> - Method in class picard.util.<a href="picard/util/AsyncIterator.html" title="class in picard.util">AsyncIterator</a></dt>
<dd>
<div class="block">Stops the thread and then calls synchronouslyClose() to allow implementation to do any one time clean up.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/BasicInputParser.html#close--">close()</a></span> - Method in class picard.util.<a href="picard/util/BasicInputParser.html" title="class in picard.util">BasicInputParser</a></dt>
<dd>
<div class="block">Closes the underlying stream</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/CircularByteBuffer.html#close--">close()</a></span> - Method in class picard.util.<a href="picard/util/CircularByteBuffer.html" title="class in picard.util">CircularByteBuffer</a></dt>
<dd>
<div class="block">Signals that the buffer is closed and no further writes will occur.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/DelimitedTextFileWithHeaderIterator.html#close--">close()</a></span> - Method in class picard.util.<a href="picard/util/DelimitedTextFileWithHeaderIterator.html" title="class in picard.util">DelimitedTextFileWithHeaderIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/TabbedTextFileWithHeaderParser.html#close--">close()</a></span> - Method in class picard.util.<a href="picard/util/TabbedTextFileWithHeaderParser.html" title="class in picard.util">TabbedTextFileWithHeaderParser</a></dt>
<dd>
<div class="block">Release all resources associated with the parser.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/FilterApplyingVariantIterator.html#close--">close()</a></span> - Method in class picard.vcf.filter.<a href="picard/vcf/filter/FilterApplyingVariantIterator.html" title="class in picard.vcf.filter">FilterApplyingVariantIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/util/PredicateFilterDecoratingClosableIterator.html#close--">close()</a></span> - Method in class picard.vcf.processor.util.<a href="picard/vcf/processor/util/PredicateFilterDecoratingClosableIterator.html" title="class in picard.vcf.processor.util">PredicateFilterDecoratingClosableIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VariantIteratorProducer.html#close--">close()</a></span> - Method in class picard.vcf.processor.<a href="picard/vcf/processor/VariantIteratorProducer.html" title="class in picard.vcf.processor">VariantIteratorProducer</a></dt>
<dd>
<div class="block">Closes any latent file handles that may have been opened by calls to <a href="picard/vcf/processor/VariantIteratorProducer.html#iterators--"><code>VariantIteratorProducer.iterators()</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FixMateInformation.html#closeWriter--">closeWriter()</a></span> - Method in class picard.sam.<a href="picard/sam/FixMateInformation.html" title="class in picard.sam">FixMateInformation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/ClusteredCrosscheckMetric.html#CLUSTER">CLUSTER</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/ClusteredCrosscheckMetric.html" title="class in picard.fingerprint">ClusteredCrosscheckMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/GraphUtils.Graph.html#cluster--">cluster()</a></span> - Method in class picard.util.<a href="picard/util/GraphUtils.Graph.html" title="class in picard.util">GraphUtils.Graph</a></dt>
<dd>
<div class="block">returns the cluster map of connected components</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaLaneMetrics.html#CLUSTER_DENSITY">CLUSTER_DENSITY</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaLaneMetrics.html" title="class in picard.illumina">IlluminaLaneMetrics</a></dt>
<dd>
<div class="block">The number of clusters per unit area on the this lane expressed in units of [cluster / mm^2].</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/ClusteredCrosscheckMetric.html#CLUSTER_SIZE">CLUSTER_SIZE</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/ClusteredCrosscheckMetric.html" title="class in picard.fingerprint">ClusteredCrosscheckMetric</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/fingerprint/ClusterCrosscheckMetrics.html" title="class in picard.fingerprint"><span class="typeNameLink">ClusterCrosscheckMetrics</span></a> - Class in <a href="picard/fingerprint/package-summary.html">picard.fingerprint</a></dt>
<dd>
<div class="block">Program to check that all (read-)groups within the set of input files appear to come from the same
 individual.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/ClusterCrosscheckMetrics.html#ClusterCrosscheckMetrics--">ClusterCrosscheckMetrics()</a></span> - Constructor for class picard.fingerprint.<a href="picard/fingerprint/ClusterCrosscheckMetrics.html" title="class in picard.fingerprint">ClusterCrosscheckMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/ClusterData.html" title="class in picard.illumina.parser"><span class="typeNameLink">ClusterData</span></a> - Class in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>
<div class="block">Store the information from Illumina files for a single cluster with one or more reads.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ClusterData.html#ClusterData-picard.illumina.parser.ReadData...-">ClusterData(ReadData...)</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/ClusterData.html" title="class in picard.illumina.parser">ClusterData</a></dt>
<dd>
<div class="block">Used for testing, reads is set directly with no copying to the input array</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ClusterData.html#ClusterData-picard.illumina.parser.ReadType:A-">ClusterData(ReadType[])</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/ClusterData.html" title="class in picard.illumina.parser">ClusterData</a></dt>
<dd>
<div class="block">Creates a ClusterData with one read for each type provided</div>
</dd>
<dt><a href="picard/illumina/ClusterDataToSamConverter.html" title="class in picard.illumina"><span class="typeNameLink">ClusterDataToSamConverter</span></a> - Class in <a href="picard/illumina/package-summary.html">picard.illumina</a></dt>
<dd>
<div class="block">Takes ClusterData provided by an IlluminaDataProvider into one or two SAMRecords,
 as appropriate, and optionally marking adapter sequence.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ClusterDataToSamConverter.html#ClusterDataToSamConverter-java.lang.String-java.lang.String-picard.illumina.parser.ReadStructure-java.util.List-">ClusterDataToSamConverter(String, String, ReadStructure, List&lt;AdapterPair&gt;)</a></span> - Constructor for class picard.illumina.<a href="picard/illumina/ClusterDataToSamConverter.html" title="class in picard.illumina">ClusterDataToSamConverter</a></dt>
<dd>
<div class="block">Constructor</div>
</dd>
<dt><a href="picard/fingerprint/ClusteredCrosscheckMetric.html" title="class in picard.fingerprint"><span class="typeNameLink">ClusteredCrosscheckMetric</span></a> - Class in <a href="picard/fingerprint/package-summary.html">picard.fingerprint</a></dt>
<dd>
<div class="block">A Metric class to hold the result of clustered crosschecking fingerprints.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/ClusteredCrosscheckMetric.html#ClusteredCrosscheckMetric--">ClusteredCrosscheckMetric()</a></span> - Constructor for class picard.fingerprint.<a href="picard/fingerprint/ClusteredCrosscheckMetric.html" title="class in picard.fingerprint">ClusteredCrosscheckMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/ClusteredCrosscheckMetric.html#ClusteredCrosscheckMetric-picard.fingerprint.CrosscheckMetric-">ClusteredCrosscheckMetric(CrosscheckMetric)</a></span> - Constructor for class picard.fingerprint.<a href="picard/fingerprint/ClusteredCrosscheckMetric.html" title="class in picard.fingerprint">ClusteredCrosscheckMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicates.DuplicateType.html#code--">code()</a></span> - Method in enum picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicates.DuplicateType.html" title="enum in picard.sam.markduplicates">MarkDuplicates.DuplicateType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RnaSeqMetrics.html#CODING_BASES">CODING_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RnaSeqMetrics.html" title="class in picard.analysis">RnaSeqMetrics</a></dt>
<dd>
<div class="block">Number of bases in primary alignments that align to a non-UTR coding base for some gene, and not ribosomal sequence.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/Gene.Transcript.html#codingEnd">codingEnd</a></span> - Variable in class picard.annotation.<a href="picard/annotation/Gene.Transcript.html" title="class in picard.annotation">Gene.Transcript</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/Gene.Transcript.html#codingStart">codingStart</a></span> - Variable in class picard.annotation.<a href="picard/annotation/Gene.Transcript.html" title="class in picard.annotation">Gene.Transcript</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectAlignmentSummaryMetrics.CollectAlignmentRefArgCollection.html#CollectAlignmentRefArgCollection--">CollectAlignmentRefArgCollection()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/CollectAlignmentSummaryMetrics.CollectAlignmentRefArgCollection.html" title="class in picard.analysis">CollectAlignmentSummaryMetrics.CollectAlignmentRefArgCollection</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/CollectAlignmentSummaryMetrics.html" title="class in picard.analysis"><span class="typeNameLink">CollectAlignmentSummaryMetrics</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">A command line tool to read a BAM file and produce standard alignment metrics that would be applicable to any alignment.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectAlignmentSummaryMetrics.html#CollectAlignmentSummaryMetrics--">CollectAlignmentSummaryMetrics()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/CollectAlignmentSummaryMetrics.html" title="class in picard.analysis">CollectAlignmentSummaryMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/CollectAlignmentSummaryMetrics.CollectAlignmentRefArgCollection.html" title="class in picard.analysis"><span class="typeNameLink">CollectAlignmentSummaryMetrics.CollectAlignmentRefArgCollection</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/CollectBaseDistributionByCycle.html" title="class in picard.analysis"><span class="typeNameLink">CollectBaseDistributionByCycle</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectBaseDistributionByCycle.html#CollectBaseDistributionByCycle--">CollectBaseDistributionByCycle()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/CollectBaseDistributionByCycle.html" title="class in picard.analysis">CollectBaseDistributionByCycle</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/CollectGcBiasMetrics.html" title="class in picard.analysis"><span class="typeNameLink">CollectGcBiasMetrics</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Tool to collect information about GC bias in the reads in a given BAM file.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectGcBiasMetrics.html#CollectGcBiasMetrics--">CollectGcBiasMetrics()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/CollectGcBiasMetrics.html" title="class in picard.analysis">CollectGcBiasMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.html" title="class in picard.illumina.quality"><span class="typeNameLink">CollectHiSeqXPfFailMetrics</span></a> - Class in <a href="picard/illumina/quality/package-summary.html">picard.illumina.quality</a></dt>
<dd>
<div class="block">Collect metrics regarding the reason for reads (sequenced by HiSeqX) not passing the Illumina PF Filter.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.html#CollectHiSeqXPfFailMetrics--">CollectHiSeqXPfFailMetrics()</a></span> - Constructor for class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailDetailedMetric.html" title="class in picard.illumina.quality"><span class="typeNameLink">CollectHiSeqXPfFailMetrics.PFFailDetailedMetric</span></a> - Class in <a href="picard/illumina/quality/package-summary.html">picard.illumina.quality</a></dt>
<dd>
<div class="block">a metric class for describing FP failing reads from an Illumina HiSeqX lane *</div>
</dd>
<dt><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html" title="class in picard.illumina.quality"><span class="typeNameLink">CollectHiSeqXPfFailMetrics.PFFailSummaryMetric</span></a> - Class in <a href="picard/illumina/quality/package-summary.html">picard.illumina.quality</a></dt>
<dd>
<div class="block">Metrics produced by the GetHiSeqXPFFailMetrics program.</div>
</dd>
<dt><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.ReadClassifier.html" title="class in picard.illumina.quality"><span class="typeNameLink">CollectHiSeqXPfFailMetrics.ReadClassifier</span></a> - Class in <a href="picard/illumina/quality/package-summary.html">picard.illumina.quality</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.ReadClassifier.PfFailReason.html" title="enum in picard.illumina.quality"><span class="typeNameLink">CollectHiSeqXPfFailMetrics.ReadClassifier.PfFailReason</span></a> - Enum in <a href="picard/illumina/quality/package-summary.html">picard.illumina.quality</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/directed/CollectHsMetrics.html" title="class in picard.analysis.directed"><span class="typeNameLink">CollectHsMetrics</span></a> - Class in <a href="picard/analysis/directed/package-summary.html">picard.analysis.directed</a></dt>
<dd>
<div class="block">Collects a set of HS metrics from a sam or bam file.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/CollectHsMetrics.html#CollectHsMetrics--">CollectHsMetrics()</a></span> - Constructor for class picard.analysis.directed.<a href="picard/analysis/directed/CollectHsMetrics.html" title="class in picard.analysis.directed">CollectHsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/CollectIlluminaBasecallingMetrics.html" title="class in picard.illumina"><span class="typeNameLink">CollectIlluminaBasecallingMetrics</span></a> - Class in <a href="picard/illumina/package-summary.html">picard.illumina</a></dt>
<dd>
<div class="block">A Command line tool to collect Illumina Basecalling metrics for a sequencing run
 Requires a Lane and an input file of Barcodes to expect.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CollectIlluminaBasecallingMetrics.html#CollectIlluminaBasecallingMetrics--">CollectIlluminaBasecallingMetrics()</a></span> - Constructor for class picard.illumina.<a href="picard/illumina/CollectIlluminaBasecallingMetrics.html" title="class in picard.illumina">CollectIlluminaBasecallingMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/CollectIlluminaLaneMetrics.html" title="class in picard.illumina"><span class="typeNameLink">CollectIlluminaLaneMetrics</span></a> - Class in <a href="picard/illumina/package-summary.html">picard.illumina</a></dt>
<dd>
<div class="block">Command-line wrapper around <a href="picard/illumina/CollectIlluminaLaneMetrics.IlluminaLaneMetricsCollector.html" title="class in picard.illumina"><code>CollectIlluminaLaneMetrics.IlluminaLaneMetricsCollector</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CollectIlluminaLaneMetrics.html#CollectIlluminaLaneMetrics--">CollectIlluminaLaneMetrics()</a></span> - Constructor for class picard.illumina.<a href="picard/illumina/CollectIlluminaLaneMetrics.html" title="class in picard.illumina">CollectIlluminaLaneMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/CollectIlluminaLaneMetrics.IlluminaLaneMetricsCollector.html" title="class in picard.illumina"><span class="typeNameLink">CollectIlluminaLaneMetrics.IlluminaLaneMetricsCollector</span></a> - Class in <a href="picard/illumina/package-summary.html">picard.illumina</a></dt>
<dd>
<div class="block">Utility for collating Tile records from the Illumina TileMetrics file into lane-level and phasing-level metrics.</div>
</dd>
<dt><a href="picard/analysis/replicates/CollectIndependentReplicateMetrics.html" title="class in picard.analysis.replicates"><span class="typeNameLink">CollectIndependentReplicateMetrics</span></a> - Class in <a href="picard/analysis/replicates/package-summary.html">picard.analysis.replicates</a></dt>
<dd>
<div class="block">A CLP that, given a BAM and a VCF with genotypes of the same sample, estimates the rate of independent replication of reads within the bam.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/CollectIndependentReplicateMetrics.html#CollectIndependentReplicateMetrics--">CollectIndependentReplicateMetrics()</a></span> - Constructor for class picard.analysis.replicates.<a href="picard/analysis/replicates/CollectIndependentReplicateMetrics.html" title="class in picard.analysis.replicates">CollectIndependentReplicateMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/CollectInsertSizeMetrics.html" title="class in picard.analysis"><span class="typeNameLink">CollectInsertSizeMetrics</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Command line program to read non-duplicate insert sizes, create a Histogram
 and report distribution statistics.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectInsertSizeMetrics.html#CollectInsertSizeMetrics--">CollectInsertSizeMetrics()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/CollectInsertSizeMetrics.html" title="class in picard.analysis">CollectInsertSizeMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/CollectJumpingLibraryMetrics.html" title="class in picard.analysis"><span class="typeNameLink">CollectJumpingLibraryMetrics</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Command-line program to compute metrics about outward-facing pairs, inward-facing
 pairs, and chimeras in a jumping library.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectJumpingLibraryMetrics.html#CollectJumpingLibraryMetrics--">CollectJumpingLibraryMetrics()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/CollectJumpingLibraryMetrics.html" title="class in picard.analysis">CollectJumpingLibraryMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CollectIlluminaLaneMetrics.IlluminaLaneMetricsCollector.html#collectLaneMetrics-java.io.File-java.io.File-java.lang.String-htsjdk.samtools.metrics.MetricsFile-htsjdk.samtools.metrics.MetricsFile-picard.illumina.parser.ReadStructure-java.lang.String-htsjdk.samtools.ValidationStringency-boolean-">collectLaneMetrics(File, File, String, MetricsFile&lt;MetricBase, Comparable&lt;?&gt;&gt;, MetricsFile&lt;MetricBase, Comparable&lt;?&gt;&gt;, ReadStructure, String, ValidationStringency, boolean)</a></span> - Static method in class picard.illumina.<a href="picard/illumina/CollectIlluminaLaneMetrics.IlluminaLaneMetricsCollector.html" title="class in picard.illumina">CollectIlluminaLaneMetrics.IlluminaLaneMetricsCollector</a></dt>
<dd>
<div class="block">Parses the tile data from the basecall directory and writes to both the lane and phasing metrics files</div>
</dd>
<dt><a href="picard/analysis/CollectMultipleMetrics.html" title="class in picard.analysis"><span class="typeNameLink">CollectMultipleMetrics</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Class that is designed to instantiate and execute multiple metrics programs that extend
 SinglePassSamProgram while making only a single pass through the SAM file and supplying
 each program with the records as it goes.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectMultipleMetrics.html#CollectMultipleMetrics--">CollectMultipleMetrics()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/CollectMultipleMetrics.html" title="class in picard.analysis">CollectMultipleMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/CollectMultipleMetrics.Program.html" title="enum in picard.analysis"><span class="typeNameLink">CollectMultipleMetrics.Program</span></a> - Enum in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/CollectMultipleMetrics.ProgramInterface.html" title="interface in picard.analysis"><span class="typeNameLink">CollectMultipleMetrics.ProgramInterface</span></a> - Interface in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/CollectOxoGMetrics.html" title="class in picard.analysis"><span class="typeNameLink">CollectOxoGMetrics</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Class for trying to quantify the CpCG->CpCA error rate.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.html#CollectOxoGMetrics--">CollectOxoGMetrics()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.html" title="class in picard.analysis">CollectOxoGMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html" title="class in picard.analysis"><span class="typeNameLink">CollectOxoGMetrics.CpcgMetrics</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Metrics class for outputs.</div>
</dd>
<dt><a href="picard/analysis/CollectQualityYieldMetrics.html" title="class in picard.analysis"><span class="typeNameLink">CollectQualityYieldMetrics</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Command line program to calibrate quality yield metrics</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectQualityYieldMetrics.html#CollectQualityYieldMetrics--">CollectQualityYieldMetrics()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/CollectQualityYieldMetrics.html" title="class in picard.analysis">CollectQualityYieldMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetrics.html" title="class in picard.analysis"><span class="typeNameLink">CollectQualityYieldMetrics.QualityYieldMetrics</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">A set of metrics used to describe the general quality of a BAM file</div>
</dd>
<dt><a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetricsCollector.html" title="class in picard.analysis"><span class="typeNameLink">CollectQualityYieldMetrics.QualityYieldMetricsCollector</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/CollectRawWgsMetrics.html" title="class in picard.analysis"><span class="typeNameLink">CollectRawWgsMetrics</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Computes a number of metrics that are useful for evaluating coverage and performance of whole genome sequencing
 experiments, same implementation as CollectWgsMetrics, with different defaults: lacks baseQ and mappingQ filters
 and has much higher coverage cap.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRawWgsMetrics.html#CollectRawWgsMetrics--">CollectRawWgsMetrics()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/CollectRawWgsMetrics.html" title="class in picard.analysis">CollectRawWgsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/CollectRawWgsMetrics.RawWgsMetrics.html" title="class in picard.analysis"><span class="typeNameLink">CollectRawWgsMetrics.RawWgsMetrics</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/CollectRnaSeqMetrics.html" title="class in picard.analysis"><span class="typeNameLink">CollectRnaSeqMetrics</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRnaSeqMetrics.html#CollectRnaSeqMetrics--">CollectRnaSeqMetrics()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/CollectRnaSeqMetrics.html" title="class in picard.analysis">CollectRnaSeqMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/CollectRrbsMetrics.html" title="class in picard.analysis"><span class="typeNameLink">CollectRrbsMetrics</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Calculates and reports QC metrics for RRBS data based on the methylation status at individual C/G bases as well
 as CpG sites across all reads in the input BAM/SAM file.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRrbsMetrics.html#CollectRrbsMetrics--">CollectRrbsMetrics()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/CollectRrbsMetrics.html" title="class in picard.analysis">CollectRrbsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/CollectRrbsMetrics.CollectRrbsMetricsReferenceArgumentCollection.html" title="class in picard.analysis"><span class="typeNameLink">CollectRrbsMetrics.CollectRrbsMetricsReferenceArgumentCollection</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRrbsMetrics.CollectRrbsMetricsReferenceArgumentCollection.html#CollectRrbsMetricsReferenceArgumentCollection--">CollectRrbsMetricsReferenceArgumentCollection()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/CollectRrbsMetrics.CollectRrbsMetricsReferenceArgumentCollection.html" title="class in picard.analysis">CollectRrbsMetrics.CollectRrbsMetricsReferenceArgumentCollection</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html" title="class in picard.analysis.artifacts"><span class="typeNameLink">CollectSequencingArtifactMetrics</span></a> - Class in <a href="picard/analysis/artifacts/package-summary.html">picard.analysis.artifacts</a></dt>
<dd>
<div class="block">Quantify substitution errors caused by mismatched base pairings during various
 stages of sample / library prep.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html#CollectSequencingArtifactMetrics--">CollectSequencingArtifactMetrics()</a></span> - Constructor for class picard.analysis.artifacts.<a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html" title="class in picard.analysis.artifacts">CollectSequencingArtifactMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/directed/CollectTargetedMetrics.html" title="class in picard.analysis.directed"><span class="typeNameLink">CollectTargetedMetrics</span></a>&lt;<a href="picard/analysis/directed/CollectTargetedMetrics.html" title="type parameter in CollectTargetedMetrics">METRIC</a> extends <a href="picard/metrics/MultilevelMetrics.html" title="class in picard.metrics">MultilevelMetrics</a>,<a href="picard/analysis/directed/CollectTargetedMetrics.html" title="type parameter in CollectTargetedMetrics">COLLECTOR</a> extends <a href="picard/analysis/directed/TargetMetricsCollector.html" title="class in picard.analysis.directed">TargetMetricsCollector</a>&lt;<a href="picard/analysis/directed/CollectTargetedMetrics.html" title="type parameter in CollectTargetedMetrics">METRIC</a>&gt;&gt; - Class in <a href="picard/analysis/directed/package-summary.html">picard.analysis.directed</a></dt>
<dd>
<div class="block">Both CollectTargetedPCRMetrics and CollectHsSelection share virtually identical program structures except
 for the name of their targeting mechanisms (e.g.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/CollectTargetedMetrics.html#CollectTargetedMetrics--">CollectTargetedMetrics()</a></span> - Constructor for class picard.analysis.directed.<a href="picard/analysis/directed/CollectTargetedMetrics.html" title="class in picard.analysis.directed">CollectTargetedMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/directed/CollectTargetedPcrMetrics.html" title="class in picard.analysis.directed"><span class="typeNameLink">CollectTargetedPcrMetrics</span></a> - Class in <a href="picard/analysis/directed/package-summary.html">picard.analysis.directed</a></dt>
<dd>
<div class="block">Collect metric information for target pcr metrics runs.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/CollectTargetedPcrMetrics.html#CollectTargetedPcrMetrics--">CollectTargetedPcrMetrics()</a></span> - Constructor for class picard.analysis.directed.<a href="picard/analysis/directed/CollectTargetedPcrMetrics.html" title="class in picard.analysis.directed">CollectTargetedPcrMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/CollectVariantCallingMetrics.html" title="class in picard.vcf"><span class="typeNameLink">CollectVariantCallingMetrics</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">Collects summary and per-sample metrics about variant calls in a VCF file.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.html#CollectVariantCallingMetrics--">CollectVariantCallingMetrics()</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingDetailMetrics.html" title="class in picard.vcf"><span class="typeNameLink">CollectVariantCallingMetrics.VariantCallingDetailMetrics</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">A collection of metrics relating to snps and indels within a variant-calling file (VCF) for a given sample.</div>
</dd>
<dt><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html" title="class in picard.vcf"><span class="typeNameLink">CollectVariantCallingMetrics.VariantCallingSummaryMetrics</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">A collection of metrics relating to snps and indels within a variant-calling file (VCF).</div>
</dd>
<dt><a href="picard/analysis/CollectWgsMetrics.html" title="class in picard.analysis"><span class="typeNameLink">CollectWgsMetrics</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Computes a number of metrics that are useful for evaluating coverage and performance of whole genome sequencing experiments.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.html#CollectWgsMetrics--">CollectWgsMetrics()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/CollectWgsMetrics.CollectWgsMetricsIntervalArgumentCollection.html" title="class in picard.analysis"><span class="typeNameLink">CollectWgsMetrics.CollectWgsMetricsIntervalArgumentCollection</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis"><span class="typeNameLink">CollectWgsMetrics.WgsMetrics</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Metrics for evaluating the performance of whole genome sequencing experiments.</div>
</dd>
<dt><a href="picard/analysis/CollectWgsMetrics.WgsMetricsCollector.html" title="class in picard.analysis"><span class="typeNameLink">CollectWgsMetrics.WgsMetricsCollector</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.CollectWgsMetricsIntervalArgumentCollection.html#CollectWgsMetricsIntervalArgumentCollection--">CollectWgsMetricsIntervalArgumentCollection()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.CollectWgsMetricsIntervalArgumentCollection.html" title="class in picard.analysis">CollectWgsMetrics.CollectWgsMetricsIntervalArgumentCollection</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.html" title="class in picard.analysis"><span class="typeNameLink">CollectWgsMetricsWithNonZeroCoverage</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.html#CollectWgsMetricsWithNonZeroCoverage--">CollectWgsMetricsWithNonZeroCoverage()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.html" title="class in picard.analysis">CollectWgsMetricsWithNonZeroCoverage</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverage.html" title="class in picard.analysis"><span class="typeNameLink">CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverage</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Metrics for evaluating the performance of whole genome sequencing experiments.</div>
</dd>
<dt><a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverage.Category.html" title="enum in picard.analysis"><span class="typeNameLink">CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverage.Category</span></a> - Enum in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverageCollector.html" title="class in picard.analysis"><span class="typeNameLink">CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverageCollector</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineDefaults.html#COLOR_STATUS">COLOR_STATUS</a></span> - Static variable in class picard.cmdline.<a href="picard/cmdline/CommandLineDefaults.html" title="class in picard.cmdline">CommandLineDefaults</a></dt>
<dd>
<div class="block">Implementation note, this is duplicate code stolen from HTSJDK's Default.java

    /**
 Decides if we want to write colors to the terminal.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/DelimitedTextFileWithHeaderIterator.html#columnLabels--">columnLabels()</a></span> - Method in class picard.util.<a href="picard/util/DelimitedTextFileWithHeaderIterator.html" title="class in picard.util">DelimitedTextFileWithHeaderIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/TabbedTextFileWithHeaderParser.html#columnLabels--">columnLabels()</a></span> - Method in class picard.util.<a href="picard/util/TabbedTextFileWithHeaderParser.html" title="class in picard.util">TabbedTextFileWithHeaderParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/TabbedTextFileWithHeaderParser.html#columnLabelsList--">columnLabelsList()</a></span> - Method in class picard.util.<a href="picard/util/TabbedTextFileWithHeaderParser.html" title="class in picard.util">TabbedTextFileWithHeaderParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VariantProcessor.Builder.html#combiningResultsBy-picard.vcf.processor.VariantProcessor.ResultMerger-">combiningResultsBy(VariantProcessor.ResultMerger&lt;R&gt;)</a></span> - Method in class picard.vcf.processor.<a href="picard/vcf/processor/VariantProcessor.Builder.html" title="class in picard.vcf.processor">VariantProcessor.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/cmdline/CommandLineDefaults.html" title="class in picard.cmdline"><span class="typeNameLink">CommandLineDefaults</span></a> - Class in <a href="picard/cmdline/package-summary.html">picard.cmdline</a></dt>
<dd>
<div class="block">Embodies defaults for global values that affect how the Picard Command Line operates.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineDefaults.html#CommandLineDefaults--">CommandLineDefaults()</a></span> - Constructor for class picard.cmdline.<a href="picard/cmdline/CommandLineDefaults.html" title="class in picard.cmdline">CommandLineDefaults</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline"><span class="typeNameLink">CommandLineProgram</span></a> - Class in <a href="picard/cmdline/package-summary.html">picard.cmdline</a></dt>
<dd>
<div class="block">Abstract class to facilitate writing command-line programs.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#CommandLineProgram--">CommandLineProgram()</a></span> - Constructor for class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/cmdline/CommandLineSyntaxTranslater.html" title="class in picard.cmdline"><span class="typeNameLink">CommandLineSyntaxTranslater</span></a> - Class in <a href="picard/cmdline/package-summary.html">picard.cmdline</a></dt>
<dd>
<div class="block">Class for handling translation of Picard-style command line argument syntax to POSIX-style argument syntax;
 used for running tests written with Picard style syntax against the Barclay command line parser.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineSyntaxTranslater.html#CommandLineSyntaxTranslater--">CommandLineSyntaxTranslater()</a></span> - Constructor for class picard.cmdline.<a href="picard/cmdline/CommandLineSyntaxTranslater.html" title="class in picard.cmdline">CommandLineSyntaxTranslater</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AddCommentsToBam.html#COMMENT">COMMENT</a></span> - Variable in class picard.sam.<a href="picard/sam/AddCommentsToBam.html" title="class in picard.sam">AddCommentsToBam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FastqToSam.html#COMMENT">COMMENT</a></span> - Variable in class picard.sam.<a href="picard/sam/FastqToSam.html" title="class in picard.sam">FastqToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html#COMMENT">COMMENT</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html" title="class in picard.sam.markduplicates.util">AbstractMarkDuplicatesCommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeSamFiles.html#COMMENT">COMMENT</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeSamFiles.html" title="class in picard.sam">MergeSamFiles</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IntervalListTools.html#COMMENT">COMMENT</a></span> - Variable in class picard.util.<a href="picard/util/IntervalListTools.html" title="class in picard.util">IntervalListTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#compare-int-int-">compare(int, int)</a></span> - Static method in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/CompareMetrics.html" title="class in picard.analysis"><span class="typeNameLink">CompareMetrics</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Compare two metrics files.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CompareMetrics.html#CompareMetrics--">CompareMetrics()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/CompareMetrics.html" title="class in picard.analysis">CompareMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/CompareSAMs.html" title="class in picard.sam"><span class="typeNameLink">CompareSAMs</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block">Rudimentary SAM comparer.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CompareSAMs.html#CompareSAMs--">CompareSAMs()</a></span> - Constructor for class picard.sam.<a href="picard/sam/CompareSAMs.html" title="class in picard.sam">CompareSAMs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html#compareTo-picard.analysis.artifacts.SequencingArtifactMetrics.BaitBiasDetailMetrics-">compareTo(SequencingArtifactMetrics.BaitBiasDetailMetrics)</a></span> - Method in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.BaitBiasDetailMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterDetailMetrics.html#compareTo-picard.analysis.artifacts.SequencingArtifactMetrics.PreAdapterDetailMetrics-">compareTo(SequencingArtifactMetrics.PreAdapterDetailMetrics)</a></span> - Method in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterDetailMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.PreAdapterDetailMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/Gene.html#compareTo-picard.annotation.Gene-">compareTo(Gene)</a></span> - Method in class picard.annotation.<a href="picard/annotation/Gene.html" title="class in picard.annotation">Gene</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeBlock.html#compareTo-picard.fingerprint.HaplotypeBlock-">compareTo(HaplotypeBlock)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeBlock.html" title="class in picard.fingerprint">HaplotypeBlock</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/MatchResults.html#compareTo-picard.fingerprint.MatchResults-">compareTo(MatchResults)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/MatchResults.html" title="class in picard.fingerprint">MatchResults</a></dt>
<dd>
<div class="block">Provides a natural sort so that better matches (by LOD) sort earlier.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/Snp.html#compareTo-picard.fingerprint.Snp-">compareTo(Snp)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/Snp.html" title="class in picard.fingerprint">Snp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceStates.TruthAndCallStates.html#compareTo-picard.vcf.GenotypeConcordanceStates.TruthAndCallStates-">compareTo(GenotypeConcordanceStates.TruthAndCallStates)</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceStates.TruthAndCallStates.html" title="class in picard.vcf">GenotypeConcordanceStates.TruthAndCallStates</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.html#COMPRESS_OUTPUTS">COMPRESS_OUTPUTS</a></span> - Variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.html" title="class in picard.illumina">ExtractIlluminaBarcodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToFastq.html#COMPRESS_OUTPUTS">COMPRESS_OUTPUTS</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToFastq.html" title="class in picard.illumina">IlluminaBasecallsToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamToFastq.html#COMPRESS_OUTPUTS_PER_RG">COMPRESS_OUTPUTS_PER_RG</a></span> - Variable in class picard.sam.<a href="picard/sam/SamToFastq.html" title="class in picard.sam">SamToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#COMPRESSION_LEVEL">COMPRESSION_LEVEL</a></span> - Variable in class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterDetailMetrics.html#CON_ALT_BASES">CON_ALT_BASES</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterDetailMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.PreAdapterDetailMetrics</a></dt>
<dd>
<div class="block">The number of REF_BASE:ALT_BASE alignments having a read number and orientation that refutes the presence of this artifact.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterDetailMetrics.html#CON_REF_BASES">CON_REF_BASES</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterDetailMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.PreAdapterDetailMetrics</a></dt>
<dd>
<div class="block">The number of REF_BASE:REF_BASE alignments having a read number and orientation that refutes the presence of this artifact.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeBlock.html#contains-picard.fingerprint.Snp-">contains(Snp)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeBlock.html" title="class in picard.fingerprint">HaplotypeBlock</a></dt>
<dd>
<div class="block">Returns true if the SNP is contained within the haplotype block, false otherwise.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilitiesFromContaminatorSequence.html#contamination">contamination</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilitiesFromContaminatorSequence.html" title="class in picard.fingerprint">HaplotypeProbabilitiesFromContaminatorSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html#CONTEXT">CONTEXT</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.BaitBiasDetailMetrics</a></dt>
<dd>
<div class="block">The sequence context to which the analysis is constrained.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterDetailMetrics.html#CONTEXT">CONTEXT</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterDetailMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.PreAdapterDetailMetrics</a></dt>
<dd>
<div class="block">The sequence context to which the analysis is constrained.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html#CONTEXT">CONTEXT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html" title="class in picard.analysis">CollectOxoGMetrics.CpcgMetrics</a></dt>
<dd>
<div class="block">The sequence context being reported on.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html#CONTEXT_SIZE">CONTEXT_SIZE</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html" title="class in picard.analysis.artifacts">CollectSequencingArtifactMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.html#CONTEXT_SIZE">CONTEXT_SIZE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.html" title="class in picard.analysis">CollectOxoGMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.html#CONTEXTS">CONTEXTS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.html" title="class in picard.analysis">CollectOxoGMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html#CONTEXTS_TO_PRINT">CONTEXTS_TO_PRINT</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html" title="class in picard.analysis.artifacts">CollectSequencingArtifactMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VcfFileSegment.html#contig--">contig()</a></span> - Method in class picard.vcf.processor.<a href="picard/vcf/processor/VcfFileSegment.html" title="class in picard.vcf.processor">VcfFileSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#CONTINGENCY_METRICS_FILE_EXTENSION">CONTINGENCY_METRICS_FILE_EXTENSION</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#CONTINGENCY_STATE_HEADER_LINE">CONTINGENCY_STATE_HEADER_LINE</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#CONTINGENCY_STATE_TAG">CONTINGENCY_STATE_TAG</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceDetailMetrics.html#CONTINGENCY_VALUES">CONTINGENCY_VALUES</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceDetailMetrics.html" title="class in picard.vcf">GenotypeConcordanceDetailMetrics</a></dt>
<dd>
<div class="block">The list of contingency table values (TP, TN, FP, FN) that are deduced from the truth/call state comparison, given the reference.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ClusterDataToSamConverter.html#convertClusterToOutputRecord-picard.illumina.parser.ClusterData-">convertClusterToOutputRecord(ClusterData)</a></span> - Method in class picard.illumina.<a href="picard/illumina/ClusterDataToSamConverter.html" title="class in picard.illumina">ClusterDataToSamConverter</a></dt>
<dd>
<div class="block">Creates the SAMRecord for each read in the cluster</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RrbsCpgDetailMetrics.html#CONVERTED_SITES">CONVERTED_SITES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RrbsCpgDetailMetrics.html" title="class in picard.analysis">RrbsCpgDetailMetrics</a></dt>
<dd>
<div class="block">Number of times this CpG site was converted (TG for + strand, CA for - strand)</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetricCollector.html#convertMetric-picard.analysis.directed.TargetMetrics-">convertMetric(TargetMetrics)</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetricCollector.html" title="class in picard.analysis.directed">HsMetricCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetricsCollector.html#convertMetric-picard.analysis.directed.TargetMetrics-">convertMetric(TargetMetrics)</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetricsCollector.html" title="class in picard.analysis.directed">TargetedPcrMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetMetricsCollector.html#convertMetric-picard.analysis.directed.TargetMetrics-">convertMetric(TargetMetrics)</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/TargetMetricsCollector.html" title="class in picard.analysis.directed">TargetMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/artifacts/ConvertSequencingArtifactToOxoG.html" title="class in picard.analysis.artifacts"><span class="typeNameLink">ConvertSequencingArtifactToOxoG</span></a> - Class in <a href="picard/analysis/artifacts/package-summary.html">picard.analysis.artifacts</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/ConvertSequencingArtifactToOxoG.html#ConvertSequencingArtifactToOxoG--">ConvertSequencingArtifactToOxoG()</a></span> - Constructor for class picard.analysis.artifacts.<a href="picard/analysis/artifacts/ConvertSequencingArtifactToOxoG.html" title="class in picard.analysis.artifacts">ConvertSequencingArtifactToOxoG</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IlluminaUtil.html#convertSolexaQualityAscii_1_1_ToPhredBinary-byte:A-">convertSolexaQualityAscii_1_1_ToPhredBinary(byte[])</a></span> - Static method in class picard.util.<a href="picard/util/IlluminaUtil.html" title="class in picard.util">IlluminaUtil</a></dt>
<dd>
<div class="block">Converts from Solexa ASCII to Phred binary in place.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html#copy-picard.illumina.ExtractIlluminaBarcodes.BarcodeMetric-">copy(ExtractIlluminaBarcodes.BarcodeMetric)</a></span> - Static method in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html" title="class in picard.illumina">ExtractIlluminaBarcodes.BarcodeMetric</a></dt>
<dd>
<div class="block">Creates a copy of metric initialized with only non-accumulated and non-calculated values set</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/ReflectionUtil.html#copyFromBaseClass-T-S-">copyFromBaseClass(T, S)</a></span> - Static method in class picard.util.<a href="picard/util/ReflectionUtil.html" title="class in picard.util">ReflectionUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RnaSeqMetrics.html#CORRECT_STRAND_READS">CORRECT_STRAND_READS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RnaSeqMetrics.html" title="class in picard.analysis">RnaSeqMetrics</a></dt>
<dd>
<div class="block">Number of aligned reads that are mapped to the correct strand.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VcfFileSegment.html#correspondingInterval--">correspondingInterval()</a></span> - Method in class picard.vcf.processor.<a href="picard/vcf/processor/VcfFileSegment.html" title="class in picard.vcf.processor">VcfFileSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceDetailMetrics.html#COUNT">COUNT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceDetailMetrics.html" title="class in picard.vcf">GenotypeConcordanceDetailMetrics</a></dt>
<dd>
<div class="block">The number of events of type TRUTH_STATE and CALL_STATE for the EVENT_TYPE and SAMPLEs</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.html#COUNT_UNPAIRED">COUNT_UNPAIRED</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AbstractWgsMetricsCollector.html#counter">counter</a></span> - Variable in class picard.analysis.<a href="picard/analysis/AbstractWgsMetricsCollector.html" title="class in picard.analysis">AbstractWgsMetricsCollector</a></dt>
<dd>
<div class="block">The number of processed genomic bases</div>
</dd>
<dt><a href="picard/analysis/CounterManager.html" title="class in picard.analysis"><span class="typeNameLink">CounterManager</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Class for managing a list of Counters of integer,
 provides methods to access data from Counters with respect to an offset.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CounterManager.html#CounterManager-int-int-">CounterManager(int, int)</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/CounterManager.html" title="class in picard.analysis">CounterManager</a></dt>
<dd>
<div class="block">Constructor creates new CounterManager without any Counters,
 counters are added to CounterManager via newCounter() method</div>
</dd>
<dt><a href="picard/analysis/CounterManager.Counter.html" title="class in picard.analysis"><span class="typeNameLink">CounterManager.Counter</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Class represents an integer array with methods to increment and get the values from it with respect
 to offset of outer <a href="picard/analysis/CounterManager.html" title="class in picard.analysis"><code>CounterManager</code></a>.</div>
</dd>
<dt><a href="picard/filter/CountingDuplicateFilter.html" title="class in picard.filter"><span class="typeNameLink">CountingDuplicateFilter</span></a> - Class in <a href="picard/filter/package-summary.html">picard.filter</a></dt>
<dd>
<div class="block">Counting filter that discards reads that have been marked as duplicates.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/filter/CountingDuplicateFilter.html#CountingDuplicateFilter--">CountingDuplicateFilter()</a></span> - Constructor for class picard.filter.<a href="picard/filter/CountingDuplicateFilter.html" title="class in picard.filter">CountingDuplicateFilter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/filter/CountingFilter.html" title="class in picard.filter"><span class="typeNameLink">CountingFilter</span></a> - Class in <a href="picard/filter/package-summary.html">picard.filter</a></dt>
<dd>
<div class="block">A SamRecordFilter that counts the number of bases in the reads which it filters out.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/filter/CountingFilter.html#CountingFilter--">CountingFilter()</a></span> - Constructor for class picard.filter.<a href="picard/filter/CountingFilter.html" title="class in picard.filter">CountingFilter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/filter/CountingMapQFilter.html" title="class in picard.filter"><span class="typeNameLink">CountingMapQFilter</span></a> - Class in <a href="picard/filter/package-summary.html">picard.filter</a></dt>
<dd>
<div class="block">Counting filter that discards reads below a configurable mapping quality threshold.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/filter/CountingMapQFilter.html#CountingMapQFilter-int-">CountingMapQFilter(int)</a></span> - Constructor for class picard.filter.<a href="picard/filter/CountingMapQFilter.html" title="class in picard.filter">CountingMapQFilter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/filter/CountingPairedFilter.html" title="class in picard.filter"><span class="typeNameLink">CountingPairedFilter</span></a> - Class in <a href="picard/filter/package-summary.html">picard.filter</a></dt>
<dd>
<div class="block">Counting filter that discards reads that are unpaired in sequencing and paired reads whose mates are not mapped.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/filter/CountingPairedFilter.html#CountingPairedFilter--">CountingPairedFilter()</a></span> - Constructor for class picard.filter.<a href="picard/filter/CountingPairedFilter.html" title="class in picard.filter">CountingPairedFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetMetricsCollector.Coverage.html#Coverage-htsjdk.samtools.util.Interval-int-">Coverage(Interval, int)</a></span> - Constructor for class picard.analysis.directed.<a href="picard/analysis/directed/TargetMetricsCollector.Coverage.html" title="class in picard.analysis.directed">TargetMetricsCollector.Coverage</a></dt>
<dd>
<div class="block">Constructs a new coverage object for the provided mapping with the desired padding either side.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.html#COVERAGE_CAP">COVERAGE_CAP</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/CollectTargetedMetrics.html#COVERAGE_CAP">COVERAGE_CAP</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/CollectTargetedMetrics.html" title="class in picard.analysis.directed">CollectTargetedMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AbstractWgsMetricsCollector.html#coverageCap">coverageCap</a></span> - Variable in class picard.analysis.<a href="picard/analysis/AbstractWgsMetricsCollector.html" title="class in picard.analysis">AbstractWgsMetricsCollector</a></dt>
<dd>
<div class="block">Positions with coverage exceeding this value are treated as if they had coverage at this value</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#coverageCap">coverageCap</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">The maximum depth/coverage to consider.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html#CpcgMetrics--">CpcgMetrics()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html" title="class in picard.analysis">CollectOxoGMetrics.CpcgMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RrbsSummaryMetrics.html#CPG_BASES_CONVERTED">CPG_BASES_CONVERTED</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RrbsSummaryMetrics.html" title="class in picard.analysis">RrbsSummaryMetrics</a></dt>
<dd>
<div class="block">Number of CpG sites that were converted (TG for +, CA for -)</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RrbsSummaryMetrics.html#CPG_BASES_SEEN">CPG_BASES_SEEN</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RrbsSummaryMetrics.html" title="class in picard.analysis">RrbsSummaryMetrics</a></dt>
<dd>
<div class="block">Number of CpG sites encountered</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#CREATE_INDEX">CREATE_INDEX</a></span> - Variable in class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#CREATE_MD5_FILE">CREATE_MD5_FILE</a></span> - Variable in class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.html#createNewCigarsIfMapsOffEndOfReference-htsjdk.samtools.SAMRecord-">createNewCigarsIfMapsOffEndOfReference(SAMRecord)</a></span> - Static method in class picard.sam.<a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</a></dt>
<dd>
<div class="block">Soft-clip an alignment that hangs off the end of its reference sequence.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FastqToSam.html#createSamFileHeader--">createSamFileHeader()</a></span> - Method in class picard.sam.<a href="picard/sam/FastqToSam.html" title="class in picard.sam">FastqToSam</a></dt>
<dd>
<div class="block">Creates a simple header with the values provided on the command line.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FixMateInformation.html#createSamFileWriter-htsjdk.samtools.SAMFileHeader-">createSamFileWriter(SAMFileHeader)</a></span> - Method in class picard.sam.<a href="picard/sam/FixMateInformation.html" title="class in picard.sam">FixMateInformation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CreateSequenceDictionary.CreateSeqDictReferenceArgumentCollection.html#CreateSeqDictReferenceArgumentCollection--">CreateSeqDictReferenceArgumentCollection()</a></span> - Constructor for class picard.sam.<a href="picard/sam/CreateSequenceDictionary.CreateSeqDictReferenceArgumentCollection.html" title="class in picard.sam">CreateSequenceDictionary.CreateSeqDictReferenceArgumentCollection</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/CreateSequenceDictionary.html" title="class in picard.sam"><span class="typeNameLink">CreateSequenceDictionary</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block">Create a SAM/BAM file from a fasta containing reference sequence.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CreateSequenceDictionary.html#CreateSequenceDictionary--">CreateSequenceDictionary()</a></span> - Constructor for class picard.sam.<a href="picard/sam/CreateSequenceDictionary.html" title="class in picard.sam">CreateSequenceDictionary</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/CreateSequenceDictionary.CreateSeqDictReferenceArgumentCollection.html" title="class in picard.sam"><span class="typeNameLink">CreateSequenceDictionary.CreateSeqDictReferenceArgumentCollection</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/DbSnpBitSetUtil.html#createSnpAndIndelBitSets-java.io.File-htsjdk.samtools.SAMSequenceDictionary-">createSnpAndIndelBitSets(File, SAMSequenceDictionary)</a></span> - Static method in class picard.util.<a href="picard/util/DbSnpBitSetUtil.html" title="class in picard.util">DbSnpBitSetUtil</a></dt>
<dd>
<div class="block">Factory method to create both a SNP bitmask and an indel bitmask in a single pass of the VCF.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/DbSnpBitSetUtil.html#createSnpAndIndelBitSets-java.io.File-htsjdk.samtools.SAMSequenceDictionary-htsjdk.samtools.util.IntervalList-">createSnpAndIndelBitSets(File, SAMSequenceDictionary, IntervalList)</a></span> - Static method in class picard.util.<a href="picard/util/DbSnpBitSetUtil.html" title="class in picard.util">DbSnpBitSetUtil</a></dt>
<dd>
<div class="block">Factory method to create both a SNP bitmask and an indel bitmask in a single pass of the VCF.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/DbSnpBitSetUtil.html#createSnpAndIndelBitSets-java.io.File-htsjdk.samtools.SAMSequenceDictionary-htsjdk.samtools.util.IntervalList-java.util.Optional-">createSnpAndIndelBitSets(File, SAMSequenceDictionary, IntervalList, Optional&lt;Log&gt;)</a></span> - Static method in class picard.util.<a href="picard/util/DbSnpBitSetUtil.html" title="class in picard.util">DbSnpBitSetUtil</a></dt>
<dd>
<div class="block">Factory method to create both a SNP bitmask and an indel bitmask in a single pass of the VCF.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/help/PicardHelpDoclet.html#createWorkUnit-org.broadinstitute.barclay.help.DocumentedFeature-com.sun.javadoc.ClassDoc-java.lang.Class-">createWorkUnit(DocumentedFeature, ClassDoc, Class&lt;?&gt;)</a></span> - Method in class picard.util.help.<a href="picard/util/help/PicardHelpDoclet.html" title="class in picard.util.help">PicardHelpDoclet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckFingerprints.html#CROSSCHECK_BY">CROSSCHECK_BY</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckFingerprints.html" title="class in picard.fingerprint">CrosscheckFingerprints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckReadGroupFingerprints.html#CROSSCHECK_LIBRARIES">CROSSCHECK_LIBRARIES</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckReadGroupFingerprints.html" title="class in picard.fingerprint">CrosscheckReadGroupFingerprints</a></dt>
<dd>
<div class="block"><span class="deprecatedLabel">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckReadGroupFingerprints.html#CROSSCHECK_SAMPLES">CROSSCHECK_SAMPLES</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckReadGroupFingerprints.html" title="class in picard.fingerprint">CrosscheckReadGroupFingerprints</a></dt>
<dd>
<div class="block"><span class="deprecatedLabel">Deprecated.</span></div>
&nbsp;</dd>
<dt><a href="picard/fingerprint/CrosscheckFingerprints.html" title="class in picard.fingerprint"><span class="typeNameLink">CrosscheckFingerprints</span></a> - Class in <a href="picard/fingerprint/package-summary.html">picard.fingerprint</a></dt>
<dd>
<div class="block">Program to check that all fingerprints within the set of input files appear to come from the same
 individual.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckFingerprints.html#CrosscheckFingerprints--">CrosscheckFingerprints()</a></span> - Constructor for class picard.fingerprint.<a href="picard/fingerprint/CrosscheckFingerprints.html" title="class in picard.fingerprint">CrosscheckFingerprints</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/fingerprint/CrosscheckMetric.html" title="class in picard.fingerprint"><span class="typeNameLink">CrosscheckMetric</span></a> - Class in <a href="picard/fingerprint/package-summary.html">picard.fingerprint</a></dt>
<dd>
<div class="block">A class to hold the result of crosschecking fingerprints.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckMetric.html#CrosscheckMetric--">CrosscheckMetric()</a></span> - Constructor for class picard.fingerprint.<a href="picard/fingerprint/CrosscheckMetric.html" title="class in picard.fingerprint">CrosscheckMetric</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/fingerprint/CrosscheckMetric.DataType.html" title="enum in picard.fingerprint"><span class="typeNameLink">CrosscheckMetric.DataType</span></a> - Enum in <a href="picard/fingerprint/package-summary.html">picard.fingerprint</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/fingerprint/CrosscheckMetric.FingerprintResult.html" title="enum in picard.fingerprint"><span class="typeNameLink">CrosscheckMetric.FingerprintResult</span></a> - Enum in <a href="picard/fingerprint/package-summary.html">picard.fingerprint</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/fingerprint/CrosscheckReadGroupFingerprints.html" title="class in picard.fingerprint"><span class="typeNameLink">CrosscheckReadGroupFingerprints</span></a> - Class in <a href="picard/fingerprint/package-summary.html">picard.fingerprint</a></dt>
<dd>
<div class="block"><span class="deprecatedLabel">Deprecated.</span>
<div class="block"><span class="deprecationComment">6/6/2017 Use <a href="picard/fingerprint/CrosscheckFingerprints.html" title="class in picard.fingerprint"><code>CrosscheckFingerprints</code></a> instead.</span></div>
</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckReadGroupFingerprints.html#CrosscheckReadGroupFingerprints--">CrosscheckReadGroupFingerprints()</a></span> - Constructor for class picard.fingerprint.<a href="picard/fingerprint/CrosscheckReadGroupFingerprints.html" title="class in picard.fingerprint">CrosscheckReadGroupFingerprints</a></dt>
<dd>
<div class="block"><span class="deprecatedLabel">Deprecated.</span></div>
&nbsp;</dd>
<dt><a href="picard/util/CsvInputParser.html" title="class in picard.util"><span class="typeNameLink">CsvInputParser</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/CsvInputParser.html#CsvInputParser-boolean-java.io.InputStream...-">CsvInputParser(boolean, InputStream...)</a></span> - Constructor for class picard.util.<a href="picard/util/CsvInputParser.html" title="class in picard.util">CsvInputParser</a></dt>
<dd>
<div class="block">Constructor</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/CsvInputParser.html#CsvInputParser-boolean-java.io.File...-">CsvInputParser(boolean, File...)</a></span> - Constructor for class picard.util.<a href="picard/util/CsvInputParser.html" title="class in picard.util">CsvInputParser</a></dt>
<dd>
<div class="block">Constructor</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#CUSTOM_AMPLICON_SET">CUSTOM_AMPLICON_SET</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The name of the amplicon set used in this metrics collection run</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/CollectTargetedPcrMetrics.html#CUSTOM_AMPLICON_SET_NAME">CUSTOM_AMPLICON_SET_NAME</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/CollectTargetedPcrMetrics.html" title="class in picard.analysis.directed">CollectTargetedPcrMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html" title="class in picard.analysis.artifacts">CollectSequencingArtifactMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectInsertSizeMetrics.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectInsertSizeMetrics.html" title="class in picard.analysis">CollectInsertSizeMetrics</a></dt>
<dd>
<div class="block">Put any custom command-line validation in an override of this method.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectMultipleMetrics.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectMultipleMetrics.html" title="class in picard.analysis">CollectMultipleMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.html" title="class in picard.analysis">CollectOxoGMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRnaSeqMetrics.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectRnaSeqMetrics.html" title="class in picard.analysis">CollectRnaSeqMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRrbsMetrics.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectRrbsMetrics.html" title="class in picard.analysis">CollectRrbsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/CollectTargetedMetrics.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/CollectTargetedMetrics.html" title="class in picard.analysis.directed">CollectTargetedMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>
<div class="block">Put any custom command-line validation in an override of this method.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fastq/BamToBfq.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.fastq.<a href="picard/fastq/BamToBfq.html" title="class in picard.fastq">BamToBfq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CheckFingerprint.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/CheckFingerprint.html" title="class in picard.fingerprint">CheckFingerprint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckReadGroupFingerprints.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckReadGroupFingerprints.html" title="class in picard.fingerprint">CrosscheckReadGroupFingerprints</a></dt>
<dd>
<div class="block"><span class="deprecatedLabel">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CheckIlluminaDirectory.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.illumina.<a href="picard/illumina/CheckIlluminaDirectory.html" title="class in picard.illumina">CheckIlluminaDirectory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.html" title="class in picard.illumina">ExtractIlluminaBarcodes</a></dt>
<dd>
<div class="block">Validate that POSITION >= 1, and that all BARCODEs are the same length and unique</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToFastq.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToFastq.html" title="class in picard.illumina">IlluminaBasecallsToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>
<div class="block">Put any custom command-line validation in an override of this method.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/MarkIlluminaAdapters.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.illumina.<a href="picard/illumina/MarkIlluminaAdapters.html" title="class in picard.illumina">MarkIlluminaAdapters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AddOrReplaceReadGroups.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.sam.<a href="picard/sam/AddOrReplaceReadGroups.html" title="class in picard.sam">AddOrReplaceReadGroups</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CreateSequenceDictionary.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.sam.<a href="picard/sam/CreateSequenceDictionary.html" title="class in picard.sam">CreateSequenceDictionary</a></dt>
<dd>
<div class="block">Use reference filename to create URI to go into header if URI was not passed on cmd line.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/DownsampleSam.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.sam.<a href="picard/sam/DownsampleSam.html" title="class in picard.sam">DownsampleSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FastqToSam.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.sam.<a href="picard/sam/FastqToSam.html" title="class in picard.sam">FastqToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FilterSamReads.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.sam.<a href="picard/sam/FilterSamReads.html" title="class in picard.sam">FilterSamReads</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/EstimateLibraryComplexity.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/EstimateLibraryComplexity.html" title="class in picard.sam.markduplicates">EstimateLibraryComplexity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/AbstractOpticalDuplicateFinderCommandLineProgram.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/AbstractOpticalDuplicateFinderCommandLineProgram.html" title="class in picard.sam.markduplicates.util">AbstractOpticalDuplicateFinderCommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>
<div class="block">Put any custom command-line validation in an override of this method.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeSamFiles.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.sam.<a href="picard/sam/MergeSamFiles.html" title="class in picard.sam">MergeSamFiles</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/PositionBasedDownsampleSam.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.sam.<a href="picard/sam/PositionBasedDownsampleSam.html" title="class in picard.sam">PositionBasedDownsampleSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertSam.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.sam.<a href="picard/sam/RevertSam.html" title="class in picard.sam">RevertSam</a></dt>
<dd>
<div class="block">Enforce that output ordering is queryname when sanitization is turned on since it requires a queryname sort.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamToFastq.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.sam.<a href="picard/sam/SamToFastq.html" title="class in picard.sam">SamToFastq</a></dt>
<dd>
<div class="block">Put any custom command-line validation in an override of this method.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ValidateSamFile.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.sam.<a href="picard/sam/ValidateSamFile.html" title="class in picard.sam">ValidateSamFile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ViewSam.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.sam.<a href="picard/sam/ViewSam.html" title="class in picard.sam">ViewSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/BaitDesigner.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.util.<a href="picard/util/BaitDesigner.html" title="class in picard.util">BaitDesigner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IntervalListTools.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.util.<a href="picard/util/IntervalListTools.html" title="class in picard.util">IntervalListTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/FixVcfHeader.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.vcf.<a href="picard/vcf/FixVcfHeader.html" title="class in picard.vcf">FixVcfHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/FindMendelianViolations.html#customCommandLineValidation--">customCommandLineValidation()</a></span> - Method in class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/FindMendelianViolations.html" title="class in picard.vcf.MendelianViolations">FindMendelianViolations</a></dt>
<dd>
<div class="block">Validates that the sex chromosomes don't overlap and parses the pseudo-autosomal regions into usable
 objects to ensure their parsability</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/BaseDistributionByCycleMetrics.html#CYCLE">CYCLE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/BaseDistributionByCycleMetrics.html" title="class in picard.analysis">BaseDistributionByCycleMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/EmpiricalPhasingMetricsOutReader.IlluminaPhasingMetrics.html#cycle">cycle</a></span> - Variable in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/EmpiricalPhasingMetricsOutReader.IlluminaPhasingMetrics.html" title="class in picard.illumina.parser.readers">EmpiricalPhasingMetricsOutReader.IlluminaPhasingMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/TileMetricsUtil.html#CYCLE_CONTAINING_TILE_METRICS">CYCLE_CONTAINING_TILE_METRICS</a></span> - Static variable in class picard.illumina.parser.<a href="picard/illumina/parser/TileMetricsUtil.html" title="class in picard.illumina.parser">TileMetricsUtil</a></dt>
<dd>
<div class="block">The cycle directory that contain the tile metrics output.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaFileUtil.html#CYCLE_SUBDIRECTORY_PATTERN">CYCLE_SUBDIRECTORY_PATTERN</a></span> - Static variable in class picard.illumina.parser.<a href="picard/illumina/parser/IlluminaFileUtil.html" title="class in picard.illumina.parser">IlluminaFileUtil</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="I:D">
<!--   -->
</a>
<h2 class="title">D</h2>
<dl>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckMetric.html#DATA_TYPE">DATA_TYPE</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckMetric.html" title="class in picard.fingerprint">CrosscheckMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CheckIlluminaDirectory.html#DATA_TYPES">DATA_TYPES</a></span> - Variable in class picard.illumina.<a href="picard/illumina/CheckIlluminaDirectory.html" title="class in picard.illumina">CheckIlluminaDirectory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsConverter.html#DATA_TYPES_NO_BARCODE">DATA_TYPES_NO_BARCODE</a></span> - Static variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsConverter.html" title="class in picard.illumina">IlluminaBasecallsConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/MultiTileFileUtil.html#dataFile">dataFile</a></span> - Variable in class picard.illumina.parser.<a href="picard/illumina/parser/MultiTileFileUtil.html" title="class in picard.illumina.parser">MultiTileFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html#DB_SNP">DB_SNP</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html" title="class in picard.analysis.artifacts">CollectSequencingArtifactMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectMultipleMetrics.html#DB_SNP">DB_SNP</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectMultipleMetrics.html" title="class in picard.analysis">CollectMultipleMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.html#DB_SNP">DB_SNP</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.html" title="class in picard.analysis">CollectOxoGMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.html#DBSNP">DBSNP</a></span> - Variable in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html#DBSNP_INS_DEL_RATIO">DBSNP_INS_DEL_RATIO</a></span> - Variable in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingSummaryMetrics</a></dt>
<dd>
<div class="block">The Insertion/Deletion ratio of the indel calls made at dbSNP sites</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html#DBSNP_TITV">DBSNP_TITV</a></span> - Variable in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingSummaryMetrics</a></dt>
<dd>
<div class="block">The Transition/Transversion ratio of the passing bi-allelic SNP calls made at dbSNP sites</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/DbSnpBitSetUtil.DbSnpBitSets.html#DbSnpBitSets--">DbSnpBitSets()</a></span> - Constructor for class picard.util.<a href="picard/util/DbSnpBitSetUtil.DbSnpBitSets.html" title="class in picard.util">DbSnpBitSetUtil.DbSnpBitSets</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/DbSnpBitSetUtil.html" title="class in picard.util"><span class="typeNameLink">DbSnpBitSetUtil</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">Utility class to use with DbSnp files to determine is a locus is
 a dbSnp site.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/DbSnpBitSetUtil.html#DbSnpBitSetUtil-java.io.File-htsjdk.samtools.SAMSequenceDictionary-">DbSnpBitSetUtil(File, SAMSequenceDictionary)</a></span> - Constructor for class picard.util.<a href="picard/util/DbSnpBitSetUtil.html" title="class in picard.util">DbSnpBitSetUtil</a></dt>
<dd>
<div class="block">Constructor that creates a bit set with bits set to true for all variant types.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/DbSnpBitSetUtil.html#DbSnpBitSetUtil-java.io.File-htsjdk.samtools.SAMSequenceDictionary-java.util.Collection-">DbSnpBitSetUtil(File, SAMSequenceDictionary, Collection&lt;VariantType&gt;)</a></span> - Constructor for class picard.util.<a href="picard/util/DbSnpBitSetUtil.html" title="class in picard.util">DbSnpBitSetUtil</a></dt>
<dd>
<div class="block">Constructor that creates a bit set with bits set to true for the given variant types.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/DbSnpBitSetUtil.html#DbSnpBitSetUtil-java.io.File-htsjdk.samtools.SAMSequenceDictionary-java.util.Collection-htsjdk.samtools.util.IntervalList-">DbSnpBitSetUtil(File, SAMSequenceDictionary, Collection&lt;VariantType&gt;, IntervalList)</a></span> - Constructor for class picard.util.<a href="picard/util/DbSnpBitSetUtil.html" title="class in picard.util">DbSnpBitSetUtil</a></dt>
<dd>
<div class="block">Constructor that creates a bit set with bits set to true for the given variant types over the given regions.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/DbSnpBitSetUtil.html#DbSnpBitSetUtil-java.io.File-htsjdk.samtools.SAMSequenceDictionary-java.util.Collection-htsjdk.samtools.util.IntervalList-java.util.Optional-">DbSnpBitSetUtil(File, SAMSequenceDictionary, Collection&lt;VariantType&gt;, IntervalList, Optional&lt;Log&gt;)</a></span> - Constructor for class picard.util.<a href="picard/util/DbSnpBitSetUtil.html" title="class in picard.util">DbSnpBitSetUtil</a></dt>
<dd>
<div class="block">Constructor.</div>
</dd>
<dt><a href="picard/util/DbSnpBitSetUtil.DbSnpBitSets.html" title="class in picard.util"><span class="typeNameLink">DbSnpBitSetUtil.DbSnpBitSets</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">Little tuple class to contain one bitset for SNPs and another for Indels.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/FifoBuffer.html#DEBUG_FREQUENCY">DEBUG_FREQUENCY</a></span> - Variable in class picard.util.<a href="picard/util/FifoBuffer.html" title="class in picard.util">FifoBuffer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesCodec.html#decode--">decode()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesCodec.html" title="class in picard.sam.markduplicates.util">ReadEndsForMarkDuplicatesCodec</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesWithBarcodesCodec.html#decode--">decode()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesWithBarcodesCodec.html" title="class in picard.sam.markduplicates.util">ReadEndsForMarkDuplicatesWithBarcodesCodec</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/RepresentativeReadIndexerCodec.html#decode--">decode()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/RepresentativeReadIndexerCodec.html" title="class in picard.sam.markduplicates.util">RepresentativeReadIndexerCodec</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AdapterMarker.html#DEFAULT_ADAPTER_LENGTH">DEFAULT_ADAPTER_LENGTH</a></span> - Static variable in class picard.util.<a href="picard/util/AdapterMarker.html" title="class in picard.util">AdapterMarker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AdapterUtility.html#DEFAULT_ADAPTER_SEQUENCE">DEFAULT_ADAPTER_SEQUENCE</a></span> - Static variable in class picard.analysis.<a href="picard/analysis/AdapterUtility.html" title="class in picard.analysis">AdapterUtility</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/OpticalDuplicateFinder.html#DEFAULT_BIG_DUPLICATE_SET_SIZE">DEFAULT_BIG_DUPLICATE_SET_SIZE</a></span> - Static variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/OpticalDuplicateFinder.html" title="class in picard.sam.markduplicates.util">OpticalDuplicateFinder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/ChimeraUtil.html#DEFAULT_EXPECTED_ORIENTATIONS">DEFAULT_EXPECTED_ORIENTATIONS</a></span> - Static variable in class picard.analysis.<a href="picard/analysis/ChimeraUtil.html" title="class in picard.analysis">ChimeraUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintChecker.html#DEFAULT_GENOTYPING_ERROR_RATE">DEFAULT_GENOTYPING_ERROR_RATE</a></span> - Static variable in class picard.fingerprint.<a href="picard/fingerprint/FingerprintChecker.html" title="class in picard.fingerprint">FingerprintChecker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/ChimeraUtil.html#DEFAULT_INSERT_SIZE_LIMIT">DEFAULT_INSERT_SIZE_LIMIT</a></span> - Static variable in class picard.analysis.<a href="picard/analysis/ChimeraUtil.html" title="class in picard.analysis">ChimeraUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/OpticalDuplicateFinder.html#DEFAULT_MAX_DUPLICATE_SET_SIZE">DEFAULT_MAX_DUPLICATE_SET_SIZE</a></span> - Static variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/OpticalDuplicateFinder.html" title="class in picard.sam.markduplicates.util">OpticalDuplicateFinder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintChecker.html#DEFAULT_MAXIMAL_PL_DIFFERENCE">DEFAULT_MAXIMAL_PL_DIFFERENCE</a></span> - Static variable in class picard.fingerprint.<a href="picard/fingerprint/FingerprintChecker.html" title="class in picard.fingerprint">FingerprintChecker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintChecker.html#DEFAULT_MINIMUM_BASE_QUALITY">DEFAULT_MINIMUM_BASE_QUALITY</a></span> - Static variable in class picard.fingerprint.<a href="picard/fingerprint/FingerprintChecker.html" title="class in picard.fingerprint">FingerprintChecker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintChecker.html#DEFAULT_MINIMUM_MAPPING_QUALITY">DEFAULT_MINIMUM_MAPPING_QUALITY</a></span> - Static variable in class picard.fingerprint.<a href="picard/fingerprint/FingerprintChecker.html" title="class in picard.fingerprint">FingerprintChecker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AdapterMarker.html#DEFAULT_NUM_ADAPTERS_TO_KEEP">DEFAULT_NUM_ADAPTERS_TO_KEEP</a></span> - Static variable in class picard.util.<a href="picard/util/AdapterMarker.html" title="class in picard.util">AdapterMarker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/OpticalDuplicateFinder.html#DEFAULT_OPTICAL_DUPLICATE_DISTANCE">DEFAULT_OPTICAL_DUPLICATE_DISTANCE</a></span> - Static variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/OpticalDuplicateFinder.html" title="class in picard.sam.markduplicates.util">OpticalDuplicateFinder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AdapterMarker.html#DEFAULT_PRUNE_ADAPTER_LIST_AFTER_THIS_MANY_ADAPTERS_SEEN">DEFAULT_PRUNE_ADAPTER_LIST_AFTER_THIS_MANY_ADAPTERS_SEEN</a></span> - Static variable in class picard.util.<a href="picard/util/AdapterMarker.html" title="class in picard.util">AdapterMarker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AsyncIterator.html#DEFAULT_QUEUE_SIZE">DEFAULT_QUEUE_SIZE</a></span> - Static variable in class picard.util.<a href="picard/util/AsyncIterator.html" title="class in picard.util">AsyncIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/ReadNameParser.html#DEFAULT_READ_NAME_REGEX">DEFAULT_READ_NAME_REGEX</a></span> - Static variable in class picard.sam.util.<a href="picard/sam/util/ReadNameParser.html" title="class in picard.sam.util">ReadNameParser</a></dt>
<dd>
<div class="block">The read name regular expression (regex) is used to extract three pieces of information from the read name: tile, x location,
 and y location.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ParameterizedFileUtil.html#DefaultSkipEmptyFiles">DefaultSkipEmptyFiles</a></span> - Static variable in class picard.illumina.parser.<a href="picard/illumina/parser/ParameterizedFileUtil.html" title="class in picard.illumina.parser">ParameterizedFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/DelimitedTextFileWithHeaderIterator.html" title="class in picard.util"><span class="typeNameLink">DelimitedTextFileWithHeaderIterator</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">Iterate through a delimited text file in which columns are found by looking at a header line rather than by position.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/DelimitedTextFileWithHeaderIterator.html#DelimitedTextFileWithHeaderIterator-picard.util.BasicInputParser-">DelimitedTextFileWithHeaderIterator(BasicInputParser)</a></span> - Constructor for class picard.util.<a href="picard/util/DelimitedTextFileWithHeaderIterator.html" title="class in picard.util">DelimitedTextFileWithHeaderIterator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/DelimitedTextFileWithHeaderIterator.Row.html" title="class in picard.util"><span class="typeNameLink">DelimitedTextFileWithHeaderIterator.Row</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/filter/DepthFilter.html" title="class in picard.vcf.filter"><span class="typeNameLink">DepthFilter</span></a> - Class in <a href="picard/vcf/filter/package-summary.html">picard.vcf.filter</a></dt>
<dd>
<div class="block">Filters out a record if all variant samples have depth lower than the given value.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/DepthFilter.html#DepthFilter-int-">DepthFilter(int)</a></span> - Constructor for class picard.vcf.filter.<a href="picard/vcf/filter/DepthFilter.html" title="class in picard.vcf.filter">DepthFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FastqToSam.html#DESCRIPTION">DESCRIPTION</a></span> - Variable in class picard.sam.<a href="picard/sam/FastqToSam.html" title="class in picard.sam">FastqToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadStructure.html#descriptors">descriptors</a></span> - Variable in class picard.illumina.parser.<a href="picard/illumina/parser/ReadStructure.html" title="class in picard.illumina.parser">ReadStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/BaitDesigner.html#DESIGN_NAME">DESIGN_NAME</a></span> - Variable in class picard.util.<a href="picard/util/BaitDesigner.html" title="class in picard.util">BaitDesigner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/BaitDesigner.html#DESIGN_ON_TARGET_STRAND">DESIGN_ON_TARGET_STRAND</a></span> - Variable in class picard.util.<a href="picard/util/BaitDesigner.html" title="class in picard.util">BaitDesigner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/BaitDesigner.html#DESIGN_STRATEGY">DESIGN_STRATEGY</a></span> - Variable in class picard.util.<a href="picard/util/BaitDesigner.html" title="class in picard.util">BaitDesigner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRrbsMetrics.html#DETAIL_FILE_EXTENSION">DETAIL_FILE_EXTENSION</a></span> - Static variable in class picard.analysis.<a href="picard/analysis/CollectRrbsMetrics.html" title="class in picard.analysis">CollectRrbsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CheckFingerprint.html#DETAIL_OUTPUT">DETAIL_OUTPUT</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CheckFingerprint.html" title="class in picard.fingerprint">CheckFingerprint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#DETAILED_METRICS_FILE_EXTENSION">DETAILED_METRICS_FILE_EXTENSION</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.html#detailedMetricsExtension">detailedMetricsExtension</a></span> - Static variable in class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/metrics/GcBiasMetrics.html#DETAILS">DETAILS</a></span> - Variable in class picard.metrics.<a href="picard/metrics/GcBiasMetrics.html" title="class in picard.metrics">GcBiasMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaDataProviderFactory.html#determineFormats-java.util.Set-picard.illumina.parser.IlluminaFileUtil-">determineFormats(Set&lt;IlluminaDataType&gt;, IlluminaFileUtil)</a></span> - Static method in class picard.illumina.parser.<a href="picard/illumina/parser/IlluminaDataProviderFactory.html" title="class in picard.illumina.parser">IlluminaDataProviderFactory</a></dt>
<dd>
<div class="block">For all requestedDataTypes return a map of file format to set of provided data types that covers as many requestedDataTypes as possible and
 chooses the most preferred available formats possible</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FastqToSam.html#determineQualityFormat-htsjdk.samtools.fastq.FastqReader-htsjdk.samtools.fastq.FastqReader-htsjdk.samtools.util.FastqQualityFormat-">determineQualityFormat(FastqReader, FastqReader, FastqQualityFormat)</a></span> - Static method in class picard.sam.<a href="picard/sam/FastqToSam.html" title="class in picard.sam">FastqToSam</a></dt>
<dd>
<div class="block">Looks at fastq input(s) and attempts to determine the proper quality format

 Closes the reader(s) by side effect</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#determineState-htsjdk.variant.variantcontext.VariantContext-java.lang.String-htsjdk.variant.variantcontext.VariantContext-java.lang.String-int-int-java.lang.Boolean-">determineState(VariantContext, String, VariantContext, String, int, int, Boolean)</a></span> - Static method in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>
<div class="block">A method to determine the truth and call states for a pair of variant contexts representing truth and call.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectInsertSizeMetrics.html#DEVIATIONS">DEVIATIONS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectInsertSizeMetrics.html" title="class in picard.analysis">CollectInsertSizeMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/fingerprint/DiploidGenotype.html" title="enum in picard.fingerprint"><span class="typeNameLink">DiploidGenotype</span></a> - Enum in <a href="picard/fingerprint/package-summary.html">picard.fingerprint</a></dt>
<dd>
<div class="block">A genotype produced by one of the concrete implementations of AbstractAlleleCaller.</div>
</dd>
<dt><a href="picard/fingerprint/DiploidHaplotype.html" title="enum in picard.fingerprint"><span class="typeNameLink">DiploidHaplotype</span></a> - Enum in <a href="picard/fingerprint/package-summary.html">picard.fingerprint</a></dt>
<dd>
<div class="block">Simple enum to represent the three possible combinations of major/major, major/minor
 and minor/minor haplotypes for a diploid individual.</div>
</dd>
<dt><a href="picard/sam/markduplicates/util/DiskBasedReadEndsForMarkDuplicatesMap.html" title="class in picard.sam.markduplicates.util"><span class="typeNameLink">DiskBasedReadEndsForMarkDuplicatesMap</span></a> - Class in <a href="picard/sam/markduplicates/util/package-summary.html">picard.sam.markduplicates.util</a></dt>
<dd>
<div class="block">Disk-based implementation of ReadEndsForMarkDuplicatesMap.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/DiskBasedReadEndsForMarkDuplicatesMap.html#DiskBasedReadEndsForMarkDuplicatesMap-int-picard.sam.markduplicates.util.ReadEndsForMarkDuplicatesCodec-">DiskBasedReadEndsForMarkDuplicatesMap(int, ReadEndsForMarkDuplicatesCodec)</a></span> - Constructor for class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/DiskBasedReadEndsForMarkDuplicatesMap.html" title="class in picard.sam.markduplicates.util">DiskBasedReadEndsForMarkDuplicatesMap</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#divide-double-double-">divide(double, double)</a></span> - Static method in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>
<div class="block">Divide two Doubles but return 0.0 if the denominator is 0</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#divide-double:A-double:A-">divide(double[], double[])</a></span> - Static method in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>
<div class="block">Calculates the ratio of two arrays of the same length.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/FileChannelJDKBugWorkAround.html#doBugWorkAround--">doBugWorkAround()</a></span> - Static method in class picard.util.<a href="picard/util/FileChannelJDKBugWorkAround.html" title="class in picard.util">FileChannelJDKBugWorkAround</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/help/HelpConstants.html#DOC_CAT_FASTA">DOC_CAT_FASTA</a></span> - Static variable in class picard.util.help.<a href="picard/util/help/HelpConstants.html" title="class in picard.util.help">HelpConstants</a></dt>
<dd>
<div class="block">Definition of the group names / descriptions for documentation/help purposes.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/help/HelpConstants.html#DOC_CAT_FASTA_SUMMARY">DOC_CAT_FASTA_SUMMARY</a></span> - Static variable in class picard.util.help.<a href="picard/util/help/HelpConstants.html" title="class in picard.util.help">HelpConstants</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/help/HelpConstants.html#DOC_CAT_INTERVALS">DOC_CAT_INTERVALS</a></span> - Static variable in class picard.util.help.<a href="picard/util/help/HelpConstants.html" title="class in picard.util.help">HelpConstants</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/help/HelpConstants.html#DOC_CAT_INTERVALS_SUMMARY">DOC_CAT_INTERVALS_SUMMARY</a></span> - Static variable in class picard.util.help.<a href="picard/util/help/HelpConstants.html" title="class in picard.util.help">HelpConstants</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/help/HelpConstants.html#DOC_CAT_READS">DOC_CAT_READS</a></span> - Static variable in class picard.util.help.<a href="picard/util/help/HelpConstants.html" title="class in picard.util.help">HelpConstants</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/help/HelpConstants.html#DOC_CAT_READS_SUMMARY">DOC_CAT_READS_SUMMARY</a></span> - Static variable in class picard.util.help.<a href="picard/util/help/HelpConstants.html" title="class in picard.util.help">HelpConstants</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/help/HelpConstants.html#DOC_CAT_TEST">DOC_CAT_TEST</a></span> - Static variable in class picard.util.help.<a href="picard/util/help/HelpConstants.html" title="class in picard.util.help">HelpConstants</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/help/HelpConstants.html#DOC_CAT_TEST_SUMMARY">DOC_CAT_TEST_SUMMARY</a></span> - Static variable in class picard.util.help.<a href="picard/util/help/HelpConstants.html" title="class in picard.util.help">HelpConstants</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/help/HelpConstants.html#DOC_CAT_VARIANT">DOC_CAT_VARIANT</a></span> - Static variable in class picard.util.help.<a href="picard/util/help/HelpConstants.html" title="class in picard.util.help">HelpConstants</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/help/HelpConstants.html#DOC_CAT_VARIANT_SUMMARY">DOC_CAT_VARIANT_SUMMARY</a></span> - Static variable in class picard.util.help.<a href="picard/util/help/HelpConstants.html" title="class in picard.util.help">HelpConstants</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/help/HelpConstants.html#DOC_SUPERCAT_EXCLUDE">DOC_SUPERCAT_EXCLUDE</a></span> - Static variable in class picard.util.help.<a href="picard/util/help/HelpConstants.html" title="class in picard.util.help">HelpConstants</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/help/HelpConstants.html#DOC_SUPERCAT_TOOLS">DOC_SUPERCAT_TOOLS</a></span> - Static variable in class picard.util.help.<a href="picard/util/help/HelpConstants.html" title="class in picard.util.help">HelpConstants</a></dt>
<dd>
<div class="block">List of "supercategory" values used for doc purposes.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/help/HelpConstants.html#DOC_SUPERCAT_UTILITIES">DOC_SUPERCAT_UTILITIES</a></span> - Static variable in class picard.util.help.<a href="picard/util/help/HelpConstants.html" title="class in picard.util.help">HelpConstants</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FastqToSam.html#doPaired-htsjdk.samtools.fastq.FastqReader-htsjdk.samtools.fastq.FastqReader-htsjdk.samtools.SAMFileWriter-">doPaired(FastqReader, FastqReader, SAMFileWriter)</a></span> - Method in class picard.sam.<a href="picard/sam/FastqToSam.html" title="class in picard.sam">FastqToSam</a></dt>
<dd>
<div class="block">More complicated method that takes two fastq files and builds pairing information in the SAM.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsConverter.html#doTileProcessing--">doTileProcessing()</a></span> - Method in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsConverter.html" title="class in picard.illumina">IlluminaBasecallsConverter</a></dt>
<dd>
<div class="block">Do the work, i.e.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/NewIlluminaBasecallsConverter.html#doTileProcessing--">doTileProcessing()</a></span> - Method in class picard.illumina.<a href="picard/illumina/NewIlluminaBasecallsConverter.html" title="class in picard.illumina">NewIlluminaBasecallsConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FastqToSam.html#doUnpaired-htsjdk.samtools.fastq.FastqReader-htsjdk.samtools.SAMFileWriter-">doUnpaired(FastqReader, SAMFileWriter)</a></span> - Method in class picard.sam.<a href="picard/sam/FastqToSam.html" title="class in picard.sam">FastqToSam</a></dt>
<dd>
<div class="block">Creates a simple SAM file from a single fastq file.</div>
</dd>
<dt><a href="picard/sam/DownsampleSam.html" title="class in picard.sam"><span class="typeNameLink">DownsampleSam</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block">Summary</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/DownsampleSam.html#DownsampleSam--">DownsampleSam()</a></span> - Constructor for class picard.sam.<a href="picard/sam/DownsampleSam.html" title="class in picard.sam">DownsampleSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/ConvertSequencingArtifactToOxoG.html#doWork--">doWork()</a></span> - Method in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/ConvertSequencingArtifactToOxoG.html" title="class in picard.analysis.artifacts">ConvertSequencingArtifactToOxoG</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectJumpingLibraryMetrics.html#doWork--">doWork()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectJumpingLibraryMetrics.html" title="class in picard.analysis">CollectJumpingLibraryMetrics</a></dt>
<dd>
<div class="block">Calculates the detailed statistics about the jumping library and then generates the results.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectMultipleMetrics.html#doWork--">doWork()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectMultipleMetrics.html" title="class in picard.analysis">CollectMultipleMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.html#doWork--">doWork()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.html" title="class in picard.analysis">CollectOxoGMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRrbsMetrics.html#doWork--">doWork()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectRrbsMetrics.html" title="class in picard.analysis">CollectRrbsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.html#doWork--">doWork()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.html#doWork--">doWork()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.html" title="class in picard.analysis">CollectWgsMetricsWithNonZeroCoverage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CompareMetrics.html#doWork--">doWork()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CompareMetrics.html" title="class in picard.analysis">CompareMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/CollectTargetedMetrics.html#doWork--">doWork()</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/CollectTargetedMetrics.html" title="class in picard.analysis.directed">CollectTargetedMetrics</a></dt>
<dd>
<div class="block">Asserts that files are readable and writable and then fires off an
 HsMetricsCalculator instance to do the real work.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/CollectIndependentReplicateMetrics.html#doWork--">doWork()</a></span> - Method in class picard.analysis.replicates.<a href="picard/analysis/replicates/CollectIndependentReplicateMetrics.html" title="class in picard.analysis.replicates">CollectIndependentReplicateMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/SinglePassSamProgram.html#doWork--">doWork()</a></span> - Method in class picard.analysis.<a href="picard/analysis/SinglePassSamProgram.html" title="class in picard.analysis">SinglePassSamProgram</a></dt>
<dd>
<div class="block">Final implementation of doWork() that checks and loads the input and optionally reference
 sequence files and the runs the sublcass through the setup() acceptRead() and finish() steps.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#doWork--">doWork()</a></span> - Method in class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>
<div class="block">Do the work after command line has been parsed.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fastq/BamToBfq.html#doWork--">doWork()</a></span> - Method in class picard.fastq.<a href="picard/fastq/BamToBfq.html" title="class in picard.fastq">BamToBfq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CheckFingerprint.html#doWork--">doWork()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/CheckFingerprint.html" title="class in picard.fingerprint">CheckFingerprint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/ClusterCrosscheckMetrics.html#doWork--">doWork()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/ClusterCrosscheckMetrics.html" title="class in picard.fingerprint">ClusterCrosscheckMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckFingerprints.html#doWork--">doWork()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckFingerprints.html" title="class in picard.fingerprint">CrosscheckFingerprints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckReadGroupFingerprints.html#doWork--">doWork()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckReadGroupFingerprints.html" title="class in picard.fingerprint">CrosscheckReadGroupFingerprints</a></dt>
<dd>
<div class="block"><span class="deprecatedLabel">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CheckIlluminaDirectory.html#doWork--">doWork()</a></span> - Method in class picard.illumina.<a href="picard/illumina/CheckIlluminaDirectory.html" title="class in picard.illumina">CheckIlluminaDirectory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CollectIlluminaBasecallingMetrics.html#doWork--">doWork()</a></span> - Method in class picard.illumina.<a href="picard/illumina/CollectIlluminaBasecallingMetrics.html" title="class in picard.illumina">CollectIlluminaBasecallingMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CollectIlluminaLaneMetrics.html#doWork--">doWork()</a></span> - Method in class picard.illumina.<a href="picard/illumina/CollectIlluminaLaneMetrics.html" title="class in picard.illumina">CollectIlluminaLaneMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.html#doWork--">doWork()</a></span> - Method in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.html" title="class in picard.illumina">ExtractIlluminaBarcodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToFastq.html#doWork--">doWork()</a></span> - Method in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToFastq.html" title="class in picard.illumina">IlluminaBasecallsToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#doWork--">doWork()</a></span> - Method in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/MarkIlluminaAdapters.html#doWork--">doWork()</a></span> - Method in class picard.illumina.<a href="picard/illumina/MarkIlluminaAdapters.html" title="class in picard.illumina">MarkIlluminaAdapters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.html#doWork--">doWork()</a></span> - Method in class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/reference/ExtractSequences.html#doWork--">doWork()</a></span> - Method in class picard.reference.<a href="picard/reference/ExtractSequences.html" title="class in picard.reference">ExtractSequences</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/reference/NonNFastaSize.html#doWork--">doWork()</a></span> - Method in class picard.reference.<a href="picard/reference/NonNFastaSize.html" title="class in picard.reference">NonNFastaSize</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/reference/NormalizeFasta.html#doWork--">doWork()</a></span> - Method in class picard.reference.<a href="picard/reference/NormalizeFasta.html" title="class in picard.reference">NormalizeFasta</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AddCommentsToBam.html#doWork--">doWork()</a></span> - Method in class picard.sam.<a href="picard/sam/AddCommentsToBam.html" title="class in picard.sam">AddCommentsToBam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AddOrReplaceReadGroups.html#doWork--">doWork()</a></span> - Method in class picard.sam.<a href="picard/sam/AddOrReplaceReadGroups.html" title="class in picard.sam">AddOrReplaceReadGroups</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/BamIndexStats.html#doWork--">doWork()</a></span> - Method in class picard.sam.<a href="picard/sam/BamIndexStats.html" title="class in picard.sam">BamIndexStats</a></dt>
<dd>
<div class="block">Main method for the program.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/BuildBamIndex.html#doWork--">doWork()</a></span> - Method in class picard.sam.<a href="picard/sam/BuildBamIndex.html" title="class in picard.sam">BuildBamIndex</a></dt>
<dd>
<div class="block">Main method for the program.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CalculateReadGroupChecksum.html#doWork--">doWork()</a></span> - Method in class picard.sam.<a href="picard/sam/CalculateReadGroupChecksum.html" title="class in picard.sam">CalculateReadGroupChecksum</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CheckTerminatorBlock.html#doWork--">doWork()</a></span> - Method in class picard.sam.<a href="picard/sam/CheckTerminatorBlock.html" title="class in picard.sam">CheckTerminatorBlock</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CleanSam.html#doWork--">doWork()</a></span> - Method in class picard.sam.<a href="picard/sam/CleanSam.html" title="class in picard.sam">CleanSam</a></dt>
<dd>
<div class="block">Do the work after command line has been parsed.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CompareSAMs.html#doWork--">doWork()</a></span> - Method in class picard.sam.<a href="picard/sam/CompareSAMs.html" title="class in picard.sam">CompareSAMs</a></dt>
<dd>
<div class="block">Do the work after command line has been parsed.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CreateSequenceDictionary.html#doWork--">doWork()</a></span> - Method in class picard.sam.<a href="picard/sam/CreateSequenceDictionary.html" title="class in picard.sam">CreateSequenceDictionary</a></dt>
<dd>
<div class="block">Do the work after command line has been parsed.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/DownsampleSam.html#doWork--">doWork()</a></span> - Method in class picard.sam.<a href="picard/sam/DownsampleSam.html" title="class in picard.sam">DownsampleSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FastqToSam.html#doWork--">doWork()</a></span> - Method in class picard.sam.<a href="picard/sam/FastqToSam.html" title="class in picard.sam">FastqToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FilterSamReads.html#doWork--">doWork()</a></span> - Method in class picard.sam.<a href="picard/sam/FilterSamReads.html" title="class in picard.sam">FilterSamReads</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FixMateInformation.html#doWork--">doWork()</a></span> - Method in class picard.sam.<a href="picard/sam/FixMateInformation.html" title="class in picard.sam">FixMateInformation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/GatherBamFiles.html#doWork--">doWork()</a></span> - Method in class picard.sam.<a href="picard/sam/GatherBamFiles.html" title="class in picard.sam">GatherBamFiles</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/EstimateLibraryComplexity.html#doWork--">doWork()</a></span> - Method in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/EstimateLibraryComplexity.html" title="class in picard.sam.markduplicates">EstimateLibraryComplexity</a></dt>
<dd>
<div class="block">Method that does most of the work.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicates.html#doWork--">doWork()</a></span> - Method in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicates.html" title="class in picard.sam.markduplicates">MarkDuplicates</a></dt>
<dd>
<div class="block">Main work method.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigar.html#doWork--">doWork()</a></span> - Method in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigar.html" title="class in picard.sam.markduplicates">MarkDuplicatesWithMateCigar</a></dt>
<dd>
<div class="block">Main work method.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/SimpleMarkDuplicatesWithMateCigar.html#doWork--">doWork()</a></span> - Method in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/SimpleMarkDuplicatesWithMateCigar.html" title="class in picard.sam.markduplicates">SimpleMarkDuplicatesWithMateCigar</a></dt>
<dd>
<div class="block">Main work method.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/UmiAwareMarkDuplicatesWithMateCigar.html#doWork--">doWork()</a></span> - Method in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/UmiAwareMarkDuplicatesWithMateCigar.html" title="class in picard.sam.markduplicates">UmiAwareMarkDuplicatesWithMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#doWork--">doWork()</a></span> - Method in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeSamFiles.html#doWork--">doWork()</a></span> - Method in class picard.sam.<a href="picard/sam/MergeSamFiles.html" title="class in picard.sam">MergeSamFiles</a></dt>
<dd>
<div class="block">Combines multiple SAM/BAM files into one.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/PositionBasedDownsampleSam.html#doWork--">doWork()</a></span> - Method in class picard.sam.<a href="picard/sam/PositionBasedDownsampleSam.html" title="class in picard.sam">PositionBasedDownsampleSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ReorderSam.html#doWork--">doWork()</a></span> - Method in class picard.sam.<a href="picard/sam/ReorderSam.html" title="class in picard.sam">ReorderSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ReplaceSamHeader.html#doWork--">doWork()</a></span> - Method in class picard.sam.<a href="picard/sam/ReplaceSamHeader.html" title="class in picard.sam">ReplaceSamHeader</a></dt>
<dd>
<div class="block">Do the work after command line has been parsed.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertOriginalBaseQualitiesAndAddMateCigar.html#doWork--">doWork()</a></span> - Method in class picard.sam.<a href="picard/sam/RevertOriginalBaseQualitiesAndAddMateCigar.html" title="class in picard.sam">RevertOriginalBaseQualitiesAndAddMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertSam.html#doWork--">doWork()</a></span> - Method in class picard.sam.<a href="picard/sam/RevertSam.html" title="class in picard.sam">RevertSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamFormatConverter.html#doWork--">doWork()</a></span> - Method in class picard.sam.<a href="picard/sam/SamFormatConverter.html" title="class in picard.sam">SamFormatConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamToFastq.html#doWork--">doWork()</a></span> - Method in class picard.sam.<a href="picard/sam/SamToFastq.html" title="class in picard.sam">SamToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SetNmMdAndUqTags.html#doWork--">doWork()</a></span> - Method in class picard.sam.<a href="picard/sam/SetNmMdAndUqTags.html" title="class in picard.sam">SetNmMdAndUqTags</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SortSam.html#doWork--">doWork()</a></span> - Method in class picard.sam.<a href="picard/sam/SortSam.html" title="class in picard.sam">SortSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SplitSamByLibrary.html#doWork--">doWork()</a></span> - Method in class picard.sam.<a href="picard/sam/SplitSamByLibrary.html" title="class in picard.sam">SplitSamByLibrary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ValidateSamFile.html#doWork--">doWork()</a></span> - Method in class picard.sam.<a href="picard/sam/ValidateSamFile.html" title="class in picard.sam">ValidateSamFile</a></dt>
<dd>
<div class="block">Do the work after command line has been parsed.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ViewSam.html#doWork--">doWork()</a></span> - Method in class picard.sam.<a href="picard/sam/ViewSam.html" title="class in picard.sam">ViewSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/BaitDesigner.html#doWork--">doWork()</a></span> - Method in class picard.util.<a href="picard/util/BaitDesigner.html" title="class in picard.util">BaitDesigner</a></dt>
<dd>
<div class="block">Main method that coordinates the checking of inputs, designing of baits and then
 the writing out of all necessary files.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/BedToIntervalList.html#doWork--">doWork()</a></span> - Method in class picard.util.<a href="picard/util/BedToIntervalList.html" title="class in picard.util">BedToIntervalList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/FifoBuffer.html#doWork--">doWork()</a></span> - Method in class picard.util.<a href="picard/util/FifoBuffer.html" title="class in picard.util">FifoBuffer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IntervalListToBed.html#doWork--">doWork()</a></span> - Method in class picard.util.<a href="picard/util/IntervalListToBed.html" title="class in picard.util">IntervalListToBed</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IntervalListTools.html#doWork--">doWork()</a></span> - Method in class picard.util.<a href="picard/util/IntervalListTools.html" title="class in picard.util">IntervalListTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/LiftOverIntervalList.html#doWork--">doWork()</a></span> - Method in class picard.util.<a href="picard/util/LiftOverIntervalList.html" title="class in picard.util">LiftOverIntervalList</a></dt>
<dd>
<div class="block">Do the work after command line has been parsed.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/ScatterIntervalsByNs.html#doWork--">doWork()</a></span> - Method in class picard.util.<a href="picard/util/ScatterIntervalsByNs.html" title="class in picard.util">ScatterIntervalsByNs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/AccumulateVariantCallingMetrics.html#doWork--">doWork()</a></span> - Method in class picard.vcf.<a href="picard/vcf/AccumulateVariantCallingMetrics.html" title="class in picard.vcf">AccumulateVariantCallingMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.html#doWork--">doWork()</a></span> - Method in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/FilterVcf.html#doWork--">doWork()</a></span> - Method in class picard.vcf.filter.<a href="picard/vcf/filter/FilterVcf.html" title="class in picard.vcf.filter">FilterVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/FixVcfHeader.html#doWork--">doWork()</a></span> - Method in class picard.vcf.<a href="picard/vcf/FixVcfHeader.html" title="class in picard.vcf">FixVcfHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GatherVcfs.html#doWork--">doWork()</a></span> - Method in class picard.vcf.<a href="picard/vcf/GatherVcfs.html" title="class in picard.vcf">GatherVcfs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#doWork--">doWork()</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/LiftoverVcf.html#doWork--">doWork()</a></span> - Method in class picard.vcf.<a href="picard/vcf/LiftoverVcf.html" title="class in picard.vcf">LiftoverVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MakeSitesOnlyVcf.html#doWork--">doWork()</a></span> - Method in class picard.vcf.<a href="picard/vcf/MakeSitesOnlyVcf.html" title="class in picard.vcf">MakeSitesOnlyVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/FindMendelianViolations.html#doWork--">doWork()</a></span> - Method in class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/FindMendelianViolations.html" title="class in picard.vcf.MendelianViolations">FindMendelianViolations</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MergeVcfs.html#doWork--">doWork()</a></span> - Method in class picard.vcf.<a href="picard/vcf/MergeVcfs.html" title="class in picard.vcf">MergeVcfs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/RenameSampleInVcf.html#doWork--">doWork()</a></span> - Method in class picard.vcf.<a href="picard/vcf/RenameSampleInVcf.html" title="class in picard.vcf">RenameSampleInVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/SortVcf.html#doWork--">doWork()</a></span> - Method in class picard.vcf.<a href="picard/vcf/SortVcf.html" title="class in picard.vcf">SortVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/SplitVcfs.html#doWork--">doWork()</a></span> - Method in class picard.vcf.<a href="picard/vcf/SplitVcfs.html" title="class in picard.vcf">SplitVcfs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/UpdateVcfSequenceDictionary.html#doWork--">doWork()</a></span> - Method in class picard.vcf.<a href="picard/vcf/UpdateVcfSequenceDictionary.html" title="class in picard.vcf">UpdateVcfSequenceDictionary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/VcfFormatConverter.html#doWork--">doWork()</a></span> - Method in class picard.vcf.<a href="picard/vcf/VcfFormatConverter.html" title="class in picard.vcf">VcfFormatConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/VcfToIntervalList.html#doWork--">doWork()</a></span> - Method in class picard.vcf.<a href="picard/vcf/VcfToIntervalList.html" title="class in picard.vcf">VcfToIntervalList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/TheoreticalSensitivity.RouletteWheel.html#draw--">draw()</a></span> - Method in class picard.analysis.<a href="picard/analysis/TheoreticalSensitivity.RouletteWheel.html" title="class in picard.analysis">TheoreticalSensitivity.RouletteWheel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html#DUPLICATE_SCORING_STRATEGY">DUPLICATE_SCORING_STRATEGY</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html" title="class in picard.sam.markduplicates.util">AbstractMarkDuplicatesCommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicates.html#DUPLICATE_SET_INDEX_TAG">DUPLICATE_SET_INDEX_TAG</a></span> - Static variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicates.html" title="class in picard.sam.markduplicates">MarkDuplicates</a></dt>
<dd>
<div class="block">The attribute in the SAM/BAM file used to store which read was selected as representative out of a duplicate set</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicates.html#DUPLICATE_SET_SIZE_TAG">DUPLICATE_SET_SIZE_TAG</a></span> - Static variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicates.html" title="class in picard.sam.markduplicates">MarkDuplicates</a></dt>
<dd>
<div class="block">The attribute in the SAM/BAM file used to store the size of a duplicate set</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/UmiMetrics.html#DUPLICATE_SETS_IGNORING_UMI">DUPLICATE_SETS_IGNORING_UMI</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/UmiMetrics.html" title="class in picard.sam.markduplicates">UmiMetrics</a></dt>
<dd>
<div class="block">Number of duplicate sets found before taking UMIs into account</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/UmiMetrics.html#DUPLICATE_SETS_WITH_UMI">DUPLICATE_SETS_WITH_UMI</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/UmiMetrics.html" title="class in picard.sam.markduplicates">UmiMetrics</a></dt>
<dd>
<div class="block">Number of duplicate sets found after taking UMIs into account</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicates.html#DUPLICATE_TYPE_LIBRARY">DUPLICATE_TYPE_LIBRARY</a></span> - Static variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicates.html" title="class in picard.sam.markduplicates">MarkDuplicates</a></dt>
<dd>
<div class="block">The duplicate type tag value for duplicate type: library.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicates.html#DUPLICATE_TYPE_SEQUENCING">DUPLICATE_TYPE_SEQUENCING</a></span> - Static variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicates.html" title="class in picard.sam.markduplicates">MarkDuplicates</a></dt>
<dd>
<div class="block">The duplicate type tag value for duplicate type: sequencing (optical & pad-hopping, or "co-localized").</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicates.html#DUPLICATE_TYPE_TAG">DUPLICATE_TYPE_TAG</a></span> - Static variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicates.html" title="class in picard.sam.markduplicates">MarkDuplicates</a></dt>
<dd>
<div class="block">The optional attribute in SAM/BAM files used to store the duplicate type.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicates.html#duplicateSetSize">duplicateSetSize</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicates.html" title="class in picard.sam.markduplicates.util">ReadEndsForMarkDuplicates</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/DuplicationMetrics.html" title="class in picard.sam"><span class="typeNameLink">DuplicationMetrics</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block">Metrics that are calculated during the process of marking duplicates
 within a stream of SAMRecords.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/DuplicationMetrics.html#DuplicationMetrics--">DuplicationMetrics()</a></span> - Constructor for class picard.sam.<a href="picard/sam/DuplicationMetrics.html" title="class in picard.sam">DuplicationMetrics</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="I:E">
<!--   -->
</a>
<h2 class="title">E</h2>
<dl>
<dt><a href="picard/sam/EarliestFragmentPrimaryAlignmentSelectionStrategy.html" title="class in picard.sam"><span class="typeNameLink">EarliestFragmentPrimaryAlignmentSelectionStrategy</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block">When it is necessary to pick a primary alignment from a group of alignments for a read, pick the one that maps
 the earliest base in the read.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/EarliestFragmentPrimaryAlignmentSelectionStrategy.html#EarliestFragmentPrimaryAlignmentSelectionStrategy--">EarliestFragmentPrimaryAlignmentSelectionStrategy()</a></span> - Constructor for class picard.sam.<a href="picard/sam/EarliestFragmentPrimaryAlignmentSelectionStrategy.html" title="class in picard.sam">EarliestFragmentPrimaryAlignmentSelectionStrategy</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/readers/EmpiricalPhasingMetricsOutReader.html" title="class in picard.illumina.parser.readers"><span class="typeNameLink">EmpiricalPhasingMetricsOutReader</span></a> - Class in <a href="picard/illumina/parser/readers/package-summary.html">picard.illumina.parser.readers</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/EmpiricalPhasingMetricsOutReader.html#EmpiricalPhasingMetricsOutReader-java.io.File-">EmpiricalPhasingMetricsOutReader(File)</a></span> - Constructor for class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/EmpiricalPhasingMetricsOutReader.html" title="class in picard.illumina.parser.readers">EmpiricalPhasingMetricsOutReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/readers/EmpiricalPhasingMetricsOutReader.IlluminaPhasingMetrics.html" title="class in picard.illumina.parser.readers"><span class="typeNameLink">EmpiricalPhasingMetricsOutReader.IlluminaPhasingMetrics</span></a> - Class in <a href="picard/illumina/parser/readers/package-summary.html">picard.illumina.parser.readers</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceScheme.html#EMPTY">EMPTY</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceScheme.html" title="class in picard.vcf">GenotypeConcordanceScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceContingencyMetrics.html#EMPTY_COUNT">EMPTY_COUNT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceContingencyMetrics.html" title="class in picard.vcf">GenotypeConcordanceContingencyMetrics</a></dt>
<dd>
<div class="block">The empty (no contingency info) count across all variants</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesCodec.html#encode-picard.sam.markduplicates.util.ReadEndsForMarkDuplicates-">encode(ReadEndsForMarkDuplicates)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesCodec.html" title="class in picard.sam.markduplicates.util">ReadEndsForMarkDuplicatesCodec</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesWithBarcodesCodec.html#encode-picard.sam.markduplicates.util.ReadEndsForMarkDuplicates-">encode(ReadEndsForMarkDuplicates)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesWithBarcodesCodec.html" title="class in picard.sam.markduplicates.util">ReadEndsForMarkDuplicatesWithBarcodesCodec</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/RepresentativeReadIndexerCodec.html#encode-picard.sam.util.RepresentativeReadIndexer-">encode(RepresentativeReadIndexer)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/RepresentativeReadIndexerCodec.html" title="class in picard.sam.markduplicates.util">RepresentativeReadIndexerCodec</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/Gene.Transcript.html#end--">end()</a></span> - Method in class picard.annotation.<a href="picard/annotation/Gene.Transcript.html" title="class in picard.annotation">Gene.Transcript</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/Gene.Transcript.Exon.html#end">end</a></span> - Variable in class picard.annotation.<a href="picard/annotation/Gene.Transcript.Exon.html" title="class in picard.annotation">Gene.Transcript.Exon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/Range.html#end">end</a></span> - Variable in class picard.illumina.parser.<a href="picard/illumina/parser/Range.html" title="class in picard.illumina.parser">Range</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/FixVcfHeader.html#ENFORCE_SAME_SAMPLES">ENFORCE_SAME_SAMPLES</a></span> - Variable in class picard.vcf.<a href="picard/vcf/FixVcfHeader.html" title="class in picard.vcf">FixVcfHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/Gene.html#equals-java.lang.Object-">equals(Object)</a></span> - Method in class picard.annotation.<a href="picard/annotation/Gene.html" title="class in picard.annotation">Gene</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/Gene.Transcript.html#equals-java.lang.Object-">equals(Object)</a></span> - Method in class picard.annotation.<a href="picard/annotation/Gene.Transcript.html" title="class in picard.annotation">Gene.Transcript</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintIdDetails.html#equals-java.lang.Object-">equals(Object)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/FingerprintIdDetails.html" title="class in picard.fingerprint">FingerprintIdDetails</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeBlock.html#equals-java.lang.Object-">equals(Object)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeBlock.html" title="class in picard.fingerprint">HaplotypeBlock</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/Snp.html#equals-java.lang.Object-">equals(Object)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/Snp.html" title="class in picard.fingerprint">Snp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/FourChannelIntensityData.html#equals-java.lang.Object-">equals(Object)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/FourChannelIntensityData.html" title="class in picard.illumina.parser">FourChannelIntensityData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/Range.html#equals-java.lang.Object-">equals(Object)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/Range.html" title="class in picard.illumina.parser">Range</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadDescriptor.html#equals-java.lang.Object-">equals(Object)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ReadDescriptor.html" title="class in picard.illumina.parser">ReadDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.PositionInfo.html#equals-java.lang.Object-">equals(Object)</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.PositionInfo.html" title="class in picard.illumina.parser.readers">AbstractIlluminaPositionFileReader.PositionInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaLaneTileCode.html#equals-java.lang.Object-">equals(Object)</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaLaneTileCode.html" title="class in picard.illumina.parser.readers">TileMetricsOutReader.IlluminaLaneTileCode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaTileMetrics.html#equals-java.lang.Object-">equals(Object)</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaTileMetrics.html" title="class in picard.illumina.parser.readers">TileMetricsOutReader.IlluminaTileMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadStructure.html#equals-java.lang.Object-">equals(Object)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ReadStructure.html" title="class in picard.illumina.parser">ReadStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/PhysicalLocationForMateCigar.html#equals-java.lang.Object-">equals(Object)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/PhysicalLocationForMateCigar.html" title="class in picard.sam.markduplicates.util">PhysicalLocationForMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceStates.TruthAndCallStates.html#equals-java.lang.Object-">equals(Object)</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceStates.TruthAndCallStates.html" title="class in picard.vcf">GenotypeConcordanceStates.TruthAndCallStates</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasDetailMetrics.html#ERROR_BAR_WIDTH">ERROR_BAR_WIDTH</a></span> - Variable in class picard.analysis.<a href="picard/analysis/GcBiasDetailMetrics.html" title="class in picard.analysis">GcBiasDetailMetrics</a></dt>
<dd>
<div class="block">The radius of error bars in this bin based on the number of observations made.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html#ERROR_RATE">ERROR_RATE</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.BaitBiasDetailMetrics</a></dt>
<dd>
<div class="block">The bait bias error rate, calculated as max(1e-10, FWD_ERROR_RATE - REV_ERROR_RATE).</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterDetailMetrics.html#ERROR_RATE">ERROR_RATE</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterDetailMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.PreAdapterDetailMetrics</a></dt>
<dd>
<div class="block">The estimated error rate due to this artifact.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.html#ERROR_SUMMARY_EXT">ERROR_SUMMARY_EXT</a></span> - Static variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/artifacts/ErrorSummaryMetrics.html" title="class in picard.analysis.artifacts"><span class="typeNameLink">ErrorSummaryMetrics</span></a> - Class in <a href="picard/analysis/artifacts/package-summary.html">picard.analysis.artifacts</a></dt>
<dd>
<div class="block">Summary metrics produced by <a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html" title="class in picard.analysis.artifacts"><code>CollectSequencingArtifactMetrics</code></a> as a roll up of the
 context-specific error rates, to provide global error rates per type of base substitution.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/ErrorSummaryMetrics.html#ErrorSummaryMetrics--">ErrorSummaryMetrics()</a></span> - Constructor for class picard.analysis.artifacts.<a href="picard/analysis/artifacts/ErrorSummaryMetrics.html" title="class in picard.analysis.artifacts">ErrorSummaryMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ParameterizedFileUtil.html#escapePeriods-java.lang.String-">escapePeriods(String)</a></span> - Static method in class picard.illumina.parser.<a href="picard/illumina/parser/ParameterizedFileUtil.html" title="class in picard.illumina.parser">ParameterizedFileUtil</a></dt>
<dd>
<div class="block">Escape the period character.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/DuplicationMetrics.html#ESTIMATED_LIBRARY_SIZE">ESTIMATED_LIBRARY_SIZE</a></span> - Variable in class picard.sam.<a href="picard/sam/DuplicationMetrics.html" title="class in picard.sam">DuplicationMetrics</a></dt>
<dd>
<div class="block">The estimated number of unique molecules in the library based on PE duplication.</div>
</dd>
<dt><a href="picard/sam/markduplicates/EstimateLibraryComplexity.html" title="class in picard.sam.markduplicates"><span class="typeNameLink">EstimateLibraryComplexity</span></a> - Class in <a href="picard/sam/markduplicates/package-summary.html">picard.sam.markduplicates</a></dt>
<dd>
<div class="block">Attempts to estimate library complexity from sequence alone.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/EstimateLibraryComplexity.html#EstimateLibraryComplexity--">EstimateLibraryComplexity()</a></span> - Constructor for class picard.sam.markduplicates.<a href="picard/sam/markduplicates/EstimateLibraryComplexity.html" title="class in picard.sam.markduplicates">EstimateLibraryComplexity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/DuplicationMetrics.html#estimateLibrarySize-long-long-">estimateLibrarySize(long, long)</a></span> - Static method in class picard.sam.<a href="picard/sam/DuplicationMetrics.html" title="class in picard.sam">DuplicationMetrics</a></dt>
<dd>
<div class="block">Estimates the size of a library based on the number of paired end molecules observed
 and the number of unique pairs observed.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/DuplicationMetrics.html#estimateRoi-long-double-long-long-">estimateRoi(long, double, long, long)</a></span> - Static method in class picard.sam.<a href="picard/sam/DuplicationMetrics.html" title="class in picard.sam">DuplicationMetrics</a></dt>
<dd>
<div class="block">Estimates the ROI (return on investment) that one would see if a library was sequenced to
 x higher coverage than the observed coverage.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VcfFileSegmentGenerator.html#excludingNonOverlaps-picard.vcf.processor.VcfFileSegmentGenerator-htsjdk.samtools.util.OverlapDetector-">excludingNonOverlaps(VcfFileSegmentGenerator, OverlapDetector&lt;T&gt;)</a></span> - Static method in class picard.vcf.processor.<a href="picard/vcf/processor/VcfFileSegmentGenerator.html" title="class in picard.vcf.processor">VcfFileSegmentGenerator</a></dt>
<dd>
<div class="block">Returns a decorated <a href="picard/vcf/processor/VcfFileSegmentGenerator.html" title="class in picard.vcf.processor"><code>VcfFileSegmentGenerator</code></a> that filters out <a href="picard/vcf/processor/VcfFileSegment.html" title="class in picard.vcf.processor"><code>VcfFileSegment</code></a>s that have no overlap with the provided
 <code>OverlapDetector</code>.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/RExecutor.html#executeFromClasspath-java.lang.String-java.lang.String...-">executeFromClasspath(String, String...)</a></span> - Static method in class picard.util.<a href="picard/util/RExecutor.html" title="class in picard.util">RExecutor</a></dt>
<dd>
<div class="block">Executes the given R script that is stored in a file on the classpath.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/RExecutor.html#executeFromFile-java.io.File-java.lang.String...-">executeFromFile(File, String...)</a></span> - Static method in class picard.util.<a href="picard/util/RExecutor.html" title="class in picard.util">RExecutor</a></dt>
<dd>
<div class="block">Executes the given R script that is stored in a file by a call to Rscript.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/LiftoverVcf.html#EXIT_CODE_WHEN_CONTIG_NOT_IN_REFERENCE">EXIT_CODE_WHEN_CONTIG_NOT_IN_REFERENCE</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/LiftoverVcf.html" title="class in picard.vcf">LiftoverVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckFingerprints.html#EXIT_CODE_WHEN_MISMATCH">EXIT_CODE_WHEN_MISMATCH</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckFingerprints.html" title="class in picard.fingerprint">CrosscheckFingerprints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/Gene.Transcript.Exon.html#Exon-int-int-">Exon(int, int)</a></span> - Constructor for class picard.annotation.<a href="picard/annotation/Gene.Transcript.Exon.html" title="class in picard.annotation">Gene.Transcript.Exon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/Gene.Transcript.html#exons">exons</a></span> - Variable in class picard.annotation.<a href="picard/annotation/Gene.Transcript.html" title="class in picard.annotation">Gene.Transcript</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckFingerprints.html#EXPECT_ALL_GROUPS_TO_MATCH">EXPECT_ALL_GROUPS_TO_MATCH</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckFingerprints.html" title="class in picard.fingerprint">CrosscheckFingerprints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckReadGroupFingerprints.html#EXPECT_ALL_READ_GROUPS_TO_MATCH">EXPECT_ALL_READ_GROUPS_TO_MATCH</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckReadGroupFingerprints.html" title="class in picard.fingerprint">CrosscheckReadGroupFingerprints</a></dt>
<dd>
<div class="block"><span class="deprecatedLabel">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/FingerprintingDetailMetrics.html#EXPECTED_GENOTYPE">EXPECTED_GENOTYPE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/FingerprintingDetailMetrics.html" title="class in picard.analysis">FingerprintingDetailMetrics</a></dt>
<dd>
<div class="block">The expected genotype of the sample at the SNP locus.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/StandardOptionDefinitions.html#EXPECTED_INSERT_SIZE_SHORT_NAME">EXPECTED_INSERT_SIZE_SHORT_NAME</a></span> - Static variable in class picard.cmdline.<a href="picard/cmdline/StandardOptionDefinitions.html" title="class in picard.cmdline">StandardOptionDefinitions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#EXPECTED_ORIENTATIONS">EXPECTED_ORIENTATIONS</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectAlignmentSummaryMetrics.html#EXPECTED_PAIR_ORIENTATIONS">EXPECTED_PAIR_ORIENTATIONS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectAlignmentSummaryMetrics.html" title="class in picard.analysis">CollectAlignmentSummaryMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CheckFingerprint.html#EXPECTED_SAMPLE_ALIAS">EXPECTED_SAMPLE_ALIAS</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CheckFingerprint.html" title="class in picard.fingerprint">CheckFingerprint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/FilterFileReader.html#EXPECTED_VERSION">EXPECTED_VERSION</a></span> - Static variable in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/FilterFileReader.html" title="class in picard.illumina.parser.readers">FilterFileReader</a></dt>
<dd>
<div class="block">Expected Version</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ParameterizedFileUtil.html#extension">extension</a></span> - Variable in class picard.illumina.parser.<a href="picard/illumina/parser/ParameterizedFileUtil.html" title="class in picard.illumina.parser">ParameterizedFileUtil</a></dt>
<dd>
<div class="block">The file extension for this class, file extension does not have the standard meaning
 in this instance.</div>
</dd>
<dt><a href="picard/illumina/ExtractIlluminaBarcodes.html" title="class in picard.illumina"><span class="typeNameLink">ExtractIlluminaBarcodes</span></a> - Class in <a href="picard/illumina/package-summary.html">picard.illumina</a></dt>
<dd>
<div class="block">Determine the barcode for each read in an Illumina lane.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.html#ExtractIlluminaBarcodes--">ExtractIlluminaBarcodes()</a></span> - Constructor for class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.html" title="class in picard.illumina">ExtractIlluminaBarcodes</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html" title="class in picard.illumina"><span class="typeNameLink">ExtractIlluminaBarcodes.BarcodeMetric</span></a> - Class in <a href="picard/illumina/package-summary.html">picard.illumina</a></dt>
<dd>
<div class="block">Metrics produced by the ExtractIlluminaBarcodes program that is used to parse data in
 the basecalls directory and determine to which barcode each read should be assigned.</div>
</dd>
<dt><a href="picard/illumina/ExtractIlluminaBarcodes.PerTileBarcodeExtractor.html" title="class in picard.illumina"><span class="typeNameLink">ExtractIlluminaBarcodes.PerTileBarcodeExtractor</span></a> - Class in <a href="picard/illumina/package-summary.html">picard.illumina</a></dt>
<dd>
<div class="block">Extracts barcodes and accumulates metrics for an entire tile.</div>
</dd>
<dt><a href="picard/illumina/ExtractIlluminaBarcodes.PerTileBarcodeExtractor.BarcodeMatch.html" title="class in picard.illumina"><span class="typeNameLink">ExtractIlluminaBarcodes.PerTileBarcodeExtractor.BarcodeMatch</span></a> - Class in <a href="picard/illumina/package-summary.html">picard.illumina</a></dt>
<dd>
<div class="block">Utility class to hang onto data about the best match for a given barcode</div>
</dd>
<dt><a href="picard/reference/ExtractSequences.html" title="class in picard.reference"><span class="typeNameLink">ExtractSequences</span></a> - Class in <a href="picard/reference/package-summary.html">picard.reference</a></dt>
<dd>
<div class="block">Simple command line program that allows sub-sequences represented by an interval
 list to be extracted from a reference sequence file.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/reference/ExtractSequences.html#ExtractSequences--">ExtractSequences()</a></span> - Constructor for class picard.reference.<a href="picard/reference/ExtractSequences.html" title="class in picard.reference">ExtractSequences</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="I:F">
<!--   -->
</a>
<h2 class="title">F</h2>
<dl>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEnds.html#F">F</a></span> - Static variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEnds.html" title="class in picard.sam.markduplicates.util">ReadEnds</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CheckIlluminaDirectory.html#FAKE_FILES">FAKE_FILES</a></span> - Variable in class picard.illumina.<a href="picard/illumina/CheckIlluminaDirectory.html" title="class in picard.illumina">CheckIlluminaDirectory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/BciFileFaker.html#fakeBciFile-java.io.File-java.util.List-">fakeBciFile(File, List&lt;Integer&gt;)</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/BciFileFaker.html" title="class in picard.illumina.parser.fakers">BciFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/BarcodeFileFaker.html#fakeFile-java.nio.ByteBuffer-">fakeFile(ByteBuffer)</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/BarcodeFileFaker.html" title="class in picard.illumina.parser.fakers">BarcodeFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/BciFileFaker.html#fakeFile-java.nio.ByteBuffer-">fakeFile(ByteBuffer)</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/BciFileFaker.html" title="class in picard.illumina.parser.fakers">BciFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/BclFileFaker.html#fakeFile-java.nio.ByteBuffer-">fakeFile(ByteBuffer)</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/BclFileFaker.html" title="class in picard.illumina.parser.fakers">BclFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/ClocsFileFaker.html#fakeFile-java.nio.ByteBuffer-">fakeFile(ByteBuffer)</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/ClocsFileFaker.html" title="class in picard.illumina.parser.fakers">ClocsFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/FileFaker.html#fakeFile-java.nio.ByteBuffer-">fakeFile(ByteBuffer)</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/FileFaker.html" title="class in picard.illumina.parser.fakers">FileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/FileFaker.html#fakeFile-java.io.File-int-int-java.lang.String-">fakeFile(File, int, int, String)</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/FileFaker.html" title="class in picard.illumina.parser.fakers">FileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/FileFaker.html#fakeFile-java.io.File-java.util.List-int-java.lang.String-">fakeFile(File, List&lt;Integer&gt;, int, String)</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/FileFaker.html" title="class in picard.illumina.parser.fakers">FileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/FileFaker.html#fakeFile-java.io.File-java.lang.Integer-">fakeFile(File, Integer)</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/FileFaker.html" title="class in picard.illumina.parser.fakers">FileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/FilterFileFaker.html#fakeFile-java.nio.ByteBuffer-">fakeFile(ByteBuffer)</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/FilterFileFaker.html" title="class in picard.illumina.parser.fakers">FilterFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/LocsFileFaker.html#fakeFile-java.nio.ByteBuffer-">fakeFile(ByteBuffer)</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/LocsFileFaker.html" title="class in picard.illumina.parser.fakers">LocsFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/MultiTileBclFileFaker.html#fakeFile-java.nio.ByteBuffer-">fakeFile(ByteBuffer)</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/MultiTileBclFileFaker.html" title="class in picard.illumina.parser.fakers">MultiTileBclFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/MultiTileLocsFileFaker.html#fakeFile-java.nio.ByteBuffer-">fakeFile(ByteBuffer)</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/MultiTileLocsFileFaker.html" title="class in picard.illumina.parser.fakers">MultiTileLocsFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/PosFileFaker.html#fakeFile-java.nio.ByteBuffer-">fakeFile(ByteBuffer)</a></span> - Method in class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/PosFileFaker.html" title="class in picard.illumina.parser.fakers">PosFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/MultiTileBclFileUtil.html#fakeFiles-java.util.List-int:A-picard.illumina.parser.IlluminaFileUtil.SupportedIlluminaFormat-">fakeFiles(List&lt;Integer&gt;, int[], IlluminaFileUtil.SupportedIlluminaFormat)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/MultiTileBclFileUtil.html" title="class in picard.illumina.parser">MultiTileBclFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/MultiTileFileUtil.html#fakeFiles-java.util.List-int:A-picard.illumina.parser.IlluminaFileUtil.SupportedIlluminaFormat-">fakeFiles(List&lt;Integer&gt;, int[], IlluminaFileUtil.SupportedIlluminaFormat)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/MultiTileFileUtil.html" title="class in picard.illumina.parser">MultiTileFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ParameterizedFileUtil.html#fakeFiles-java.util.List-int:A-picard.illumina.parser.IlluminaFileUtil.SupportedIlluminaFormat-">fakeFiles(List&lt;Integer&gt;, int[], IlluminaFileUtil.SupportedIlluminaFormat)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ParameterizedFileUtil.html" title="class in picard.illumina.parser">ParameterizedFileUtil</a></dt>
<dd>
<div class="block">Given the expected tiles/expected cycles for this file type create a set of fake files such that the
 verification criteria are met.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PerTileFileUtil.html#fakeFiles-java.util.List-int:A-picard.illumina.parser.IlluminaFileUtil.SupportedIlluminaFormat-">fakeFiles(List&lt;Integer&gt;, int[], IlluminaFileUtil.SupportedIlluminaFormat)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/PerTileFileUtil.html" title="class in picard.illumina.parser">PerTileFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PerTilePerCycleFileUtil.html#fakeFiles-java.util.List-int:A-picard.illumina.parser.IlluminaFileUtil.SupportedIlluminaFormat-">fakeFiles(List&lt;Integer&gt;, int[], IlluminaFileUtil.SupportedIlluminaFormat)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/PerTilePerCycleFileUtil.html" title="class in picard.illumina.parser">PerTilePerCycleFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ParameterizedFileUtil.html#faker">faker</a></span> - Variable in class picard.illumina.parser.<a href="picard/illumina/parser/ParameterizedFileUtil.html" title="class in picard.illumina.parser">ParameterizedFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html#FAMILY_ID">FAMILY_ID</a></span> - Variable in class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html" title="class in picard.vcf.MendelianViolations">MendelianViolationMetrics</a></dt>
<dd>
<div class="block">The family ID assigned to the trio for which these metrics are calculated.</div>
</dd>
<dt><a href="picard/cmdline/programgroups/Fasta.html" title="class in picard.cmdline.programgroups"><span class="typeNameLink">Fasta</span></a> - Class in <a href="picard/cmdline/programgroups/package-summary.html">picard.cmdline.programgroups</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/programgroups/Fasta.html#Fasta--">Fasta()</a></span> - Constructor for class picard.cmdline.programgroups.<a href="picard/cmdline/programgroups/Fasta.html" title="class in picard.cmdline.programgroups">Fasta</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FastqToSam.html#FASTQ">FASTQ</a></span> - Variable in class picard.sam.<a href="picard/sam/FastqToSam.html" title="class in picard.sam">FastqToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamToFastq.html#FASTQ">FASTQ</a></span> - Variable in class picard.sam.<a href="picard/sam/SamToFastq.html" title="class in picard.sam">SamToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FastqToSam.html#FASTQ2">FASTQ2</a></span> - Variable in class picard.sam.<a href="picard/sam/FastqToSam.html" title="class in picard.sam">FastqToSam</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/FastqToSam.html" title="class in picard.sam"><span class="typeNameLink">FastqToSam</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block">Converts a fastq file to an unaligned BAM/SAM format.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FastqToSam.html#FastqToSam--">FastqToSam()</a></span> - Constructor for class picard.sam.<a href="picard/sam/FastqToSam.html" title="class in picard.sam">FastqToSam</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/FastWgsMetricsCollector.html" title="class in picard.analysis"><span class="typeNameLink">FastWgsMetricsCollector</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Class represents fast algorithm for collecting data from <code>AbstractLocusInfo</code>
 with a list of aligned <code>EdgingRecordAndOffset</code> objects.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/FastWgsMetricsCollector.html#FastWgsMetricsCollector-picard.analysis.CollectWgsMetrics-int-htsjdk.samtools.util.IntervalList-">FastWgsMetricsCollector(CollectWgsMetrics, int, IntervalList)</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/FastWgsMetricsCollector.html" title="class in picard.analysis">FastWgsMetricsCollector</a></dt>
<dd>
<div class="block">Creates a collector and initializes the inner data structures</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html#FATHER">FATHER</a></span> - Variable in class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html" title="class in picard.vcf.MendelianViolations">MendelianViolationMetrics</a></dt>
<dd>
<div class="block">The ID of the father within the trio.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/FindMendelianViolations.html#FEMALE_CHROMS">FEMALE_CHROMS</a></span> - Variable in class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/FindMendelianViolations.html" title="class in picard.vcf.MendelianViolations">FindMendelianViolations</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEnds.html#FF">FF</a></span> - Static variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEnds.html" title="class in picard.sam.markduplicates.util">ReadEnds</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/FifoBuffer.html" title="class in picard.util"><span class="typeNameLink">FifoBuffer</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">A program that is designed to act as a large memory buffer between processes that are
 connected with unix pipes where one or more processes either produce or consume their
 input or output in bursts.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/FifoBuffer.html#FifoBuffer-java.io.InputStream-java.io.PrintStream-">FifoBuffer(InputStream, PrintStream)</a></span> - Constructor for class picard.util.<a href="picard/util/FifoBuffer.html" title="class in picard.util">FifoBuffer</a></dt>
<dd>
<div class="block">Constructor that defaults to QUIET since Fifos don't do anything beyond buffering having their
 start/end information logged is often undesirable.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/FifoBuffer.html#FifoBuffer--">FifoBuffer()</a></span> - Constructor for class picard.util.<a href="picard/util/FifoBuffer.html" title="class in picard.util">FifoBuffer</a></dt>
<dd>
<div class="block">Constructor that defaults to QUIET since Fifos don't do anything beyond buffering having their
 start/end information logged is often undesirable.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html#FILE_EXTENSION">FILE_EXTENSION</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html" title="class in picard.analysis.artifacts">CollectSequencingArtifactMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectMultipleMetrics.html#FILE_EXTENSION">FILE_EXTENSION</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectMultipleMetrics.html" title="class in picard.analysis">CollectMultipleMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CollectIlluminaLaneMetrics.html#FILE_EXTENSION">FILE_EXTENSION</a></span> - Variable in class picard.illumina.<a href="picard/illumina/CollectIlluminaLaneMetrics.html" title="class in picard.illumina">CollectIlluminaLaneMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/FileChannelJDKBugWorkAround.html" title="class in picard.util"><span class="typeNameLink">FileChannelJDKBugWorkAround</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">Work-around for the following bug
 http://bugs.sun.com/bugdatabase/view_bug.do?bug_id=6503430

 Call the method in the main thread before launching threads that do memory-mapping.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/FileChannelJDKBugWorkAround.html#FileChannelJDKBugWorkAround--">FileChannelJDKBugWorkAround()</a></span> - Constructor for class picard.util.<a href="picard/util/FileChannelJDKBugWorkAround.html" title="class in picard.util">FileChannelJDKBugWorkAround</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/fakers/FileFaker.html" title="class in picard.illumina.parser.fakers"><span class="typeNameLink">FileFaker</span></a> - Class in <a href="picard/illumina/parser/fakers/package-summary.html">picard.illumina.parser.fakers</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/FileFaker.html#FileFaker--">FileFaker()</a></span> - Constructor for class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/FileFaker.html" title="class in picard.illumina.parser.fakers">FileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/MultiTileBclFileUtil.html#filesAvailable--">filesAvailable()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/MultiTileBclFileUtil.html" title="class in picard.illumina.parser">MultiTileBclFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/MultiTileFileUtil.html#filesAvailable--">filesAvailable()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/MultiTileFileUtil.html" title="class in picard.illumina.parser">MultiTileFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ParameterizedFileUtil.html#filesAvailable--">filesAvailable()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ParameterizedFileUtil.html" title="class in picard.illumina.parser">ParameterizedFileUtil</a></dt>
<dd>
<div class="block">Determine whether or not files are available</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PerTileFileUtil.html#filesAvailable--">filesAvailable()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/PerTileFileUtil.html" title="class in picard.illumina.parser">PerTileFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PerTilePerCycleFileUtil.html#filesAvailable--">filesAvailable()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/PerTilePerCycleFileUtil.html" title="class in picard.illumina.parser">PerTilePerCycleFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/NewIlluminaDataProvider.html#fileToTile-java.lang.String-">fileToTile(String)</a></span> - Static method in class picard.illumina.parser.<a href="picard/illumina/parser/NewIlluminaDataProvider.html" title="class in picard.illumina.parser">NewIlluminaDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ParameterizedFileUtil.html#fileToTile-java.lang.String-">fileToTile(String)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ParameterizedFileUtil.html" title="class in picard.illumina.parser">ParameterizedFileUtil</a></dt>
<dd>
<div class="block">Returns only lane and tile information as PerTileFt's do not have End information.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/BaitDesigner.html#FILL_POOLS">FILL_POOLS</a></span> - Variable in class picard.util.<a href="picard/util/BaitDesigner.html" title="class in picard.util">BaitDesigner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FilterSamReads.html#FILTER">FILTER</a></span> - Variable in class picard.sam.<a href="picard/sam/FilterSamReads.html" title="class in picard.sam">FilterSamReads</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/AlleleBalanceFilter.html#filter-htsjdk.variant.variantcontext.VariantContext-">filter(VariantContext)</a></span> - Method in class picard.vcf.filter.<a href="picard/vcf/filter/AlleleBalanceFilter.html" title="class in picard.vcf.filter">AlleleBalanceFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/DepthFilter.html#filter-htsjdk.variant.variantcontext.VariantContext-htsjdk.variant.variantcontext.Genotype-">filter(VariantContext, Genotype)</a></span> - Method in class picard.vcf.filter.<a href="picard/vcf/filter/DepthFilter.html" title="class in picard.vcf.filter">DepthFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/FisherStrandFilter.html#filter-htsjdk.variant.variantcontext.VariantContext-">filter(VariantContext)</a></span> - Method in class picard.vcf.filter.<a href="picard/vcf/filter/FisherStrandFilter.html" title="class in picard.vcf.filter">FisherStrandFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/GenotypeFilter.html#filter-htsjdk.variant.variantcontext.VariantContext-htsjdk.variant.variantcontext.Genotype-">filter(VariantContext, Genotype)</a></span> - Method in interface picard.vcf.filter.<a href="picard/vcf/filter/GenotypeFilter.html" title="interface in picard.vcf.filter">GenotypeFilter</a></dt>
<dd>
<div class="block">Test whether or not the genotype should be filtered out.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/GenotypeQualityFilter.html#filter-htsjdk.variant.variantcontext.VariantContext-htsjdk.variant.variantcontext.Genotype-">filter(VariantContext, Genotype)</a></span> - Method in class picard.vcf.filter.<a href="picard/vcf/filter/GenotypeQualityFilter.html" title="class in picard.vcf.filter">GenotypeQualityFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/QdFilter.html#filter-htsjdk.variant.variantcontext.VariantContext-">filter(VariantContext)</a></span> - Method in class picard.vcf.filter.<a href="picard/vcf/filter/QdFilter.html" title="class in picard.vcf.filter">QdFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/VariantFilter.html#filter-htsjdk.variant.variantcontext.VariantContext-">filter(VariantContext)</a></span> - Method in interface picard.vcf.filter.<a href="picard/vcf/filter/VariantFilter.html" title="interface in picard.vcf.filter">VariantFilter</a></dt>
<dd>
<div class="block">Check to see if the VariantContext should have a filter applied to it.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/LiftoverVcf.html#FILTER_CANNOT_LIFTOVER_INDEL">FILTER_CANNOT_LIFTOVER_INDEL</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/LiftoverVcf.html" title="class in picard.vcf">LiftoverVcf</a></dt>
<dd>
<div class="block">Filter name to use when a target cannot be lifted over.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/LiftoverVcf.html#FILTER_INDEL_STRADDLES_TWO_INTERVALS">FILTER_INDEL_STRADDLES_TWO_INTERVALS</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/LiftoverVcf.html" title="class in picard.vcf">LiftoverVcf</a></dt>
<dd>
<div class="block">Filter name to use when an indel cannot be lifted over since it straddles two intervals in a chain which means
 that it is unclear what are the right alleles to be used.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/LiftoverVcf.html#FILTER_MISMATCHING_REF_ALLELE">FILTER_MISMATCHING_REF_ALLELE</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/LiftoverVcf.html" title="class in picard.vcf">LiftoverVcf</a></dt>
<dd>
<div class="block">Filter name to use when a target is lifted over, but the reference allele doesn't match the new reference.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/QdFilter.html#FILTER_NAME">FILTER_NAME</a></span> - Static variable in class picard.vcf.filter.<a href="picard/vcf/filter/QdFilter.html" title="class in picard.vcf.filter">QdFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/LiftoverVcf.html#FILTER_NO_TARGET">FILTER_NO_TARGET</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/LiftoverVcf.html" title="class in picard.vcf">LiftoverVcf</a></dt>
<dd>
<div class="block">Filter name to use when a target cannot be lifted over.</div>
</dd>
<dt><a href="picard/vcf/filter/FilterApplyingVariantIterator.html" title="class in picard.vcf.filter"><span class="typeNameLink">FilterApplyingVariantIterator</span></a> - Class in <a href="picard/vcf/filter/package-summary.html">picard.vcf.filter</a></dt>
<dd>
<div class="block">Iterator that dynamically applies filter strings to VariantContext records supplied by an underlying
 iterator.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/FilterApplyingVariantIterator.html#FilterApplyingVariantIterator-java.util.Iterator-java.util.Collection-java.util.Collection-">FilterApplyingVariantIterator(Iterator&lt;VariantContext&gt;, Collection&lt;VariantFilter&gt;, Collection&lt;GenotypeFilter&gt;)</a></span> - Constructor for class picard.vcf.filter.<a href="picard/vcf/filter/FilterApplyingVariantIterator.html" title="class in picard.vcf.filter">FilterApplyingVariantIterator</a></dt>
<dd>
<div class="block">Constructs an iterator from an underlying iterator and the provided (possibly empty)
 collections of variant and genotype filters.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html#FILTERED_INDELS">FILTERED_INDELS</a></span> - Variable in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingSummaryMetrics</a></dt>
<dd>
<div class="block">The number of indels that are filtered</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html#FILTERED_SNPS">FILTERED_SNPS</a></span> - Variable in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingSummaryMetrics</a></dt>
<dd>
<div class="block">The number of SNPs that are filtered</div>
</dd>
<dt><a href="picard/illumina/parser/fakers/FilterFileFaker.html" title="class in picard.illumina.parser.fakers"><span class="typeNameLink">FilterFileFaker</span></a> - Class in <a href="picard/illumina/parser/fakers/package-summary.html">picard.illumina.parser.fakers</a></dt>
<dd>
<div class="block">Created by jcarey on 3/13/14.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/FilterFileFaker.html#FilterFileFaker--">FilterFileFaker()</a></span> - Constructor for class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/FilterFileFaker.html" title="class in picard.illumina.parser.fakers">FilterFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/readers/FilterFileReader.html" title="class in picard.illumina.parser.readers"><span class="typeNameLink">FilterFileReader</span></a> - Class in <a href="picard/illumina/parser/readers/package-summary.html">picard.illumina.parser.readers</a></dt>
<dd>
<div class="block">Illumina uses an algorithm described in "Theory of RTA" that determines whether or not a cluster passes filter("PF") or not.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/FilterFileReader.html#FilterFileReader-java.io.File-">FilterFileReader(File)</a></span> - Constructor for class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/FilterFileReader.html" title="class in picard.illumina.parser.readers">FilterFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/filter/CountingFilter.html#filterOut-htsjdk.samtools.SAMRecord-">filterOut(SAMRecord)</a></span> - Method in class picard.filter.<a href="picard/filter/CountingFilter.html" title="class in picard.filter">CountingFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/filter/CountingFilter.html#filterOut-htsjdk.samtools.SAMRecord-htsjdk.samtools.SAMRecord-">filterOut(SAMRecord, SAMRecord)</a></span> - Method in class picard.filter.<a href="picard/filter/CountingFilter.html" title="class in picard.filter">CountingFilter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/FilterSamReads.html" title="class in picard.sam"><span class="typeNameLink">FilterSamReads</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block">From a SAM or BAM file, produce a new SAM or BAM by filtering aligned reads or a list of read
 names provided in a file (one readname per line)
 <p/>
 $Id$</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FilterSamReads.html#FilterSamReads--">FilterSamReads()</a></span> - Constructor for class picard.sam.<a href="picard/sam/FilterSamReads.html" title="class in picard.sam">FilterSamReads</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/FilterSamReads.Filter.html" title="enum in picard.sam"><span class="typeNameLink">FilterSamReads.Filter</span></a> - Enum in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/Fingerprint.html#filterSuspectSites--">filterSuspectSites()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/Fingerprint.html" title="class in picard.fingerprint">Fingerprint</a></dt>
<dd>
<div class="block">Attempts to filter out haplotypes that may have suspect genotyping by removing haplotypes that reach
 a minimum confidence score yet have a significant fraction of observations from a third or fourth allele.</div>
</dd>
<dt><a href="picard/vcf/filter/FilterVcf.html" title="class in picard.vcf.filter"><span class="typeNameLink">FilterVcf</span></a> - Class in <a href="picard/vcf/filter/package-summary.html">picard.vcf.filter</a></dt>
<dd>
<div class="block">Applies a set of hard filters to Variants and to Genotypes within a VCF.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/FilterVcf.html#FilterVcf--">FilterVcf()</a></span> - Constructor for class picard.vcf.filter.<a href="picard/vcf/filter/FilterVcf.html" title="class in picard.vcf.filter">FilterVcf</a></dt>
<dd>
<div class="block">Constructor to default to having index creation on.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html#finalizeAndWriteMetrics-picard.sam.markduplicates.util.LibraryIdGenerator-">finalizeAndWriteMetrics(LibraryIdGenerator)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html" title="class in picard.sam.markduplicates.util">AbstractMarkDuplicatesCommandLineProgram</a></dt>
<dd>
<div class="block">Writes the metrics given by the libraryIdGenerator to the METRICS_FILE.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.html#finalizeMetrics-java.util.Map-picard.illumina.ExtractIlluminaBarcodes.BarcodeMetric-">finalizeMetrics(Map&lt;String, ExtractIlluminaBarcodes.BarcodeMetric&gt;, ExtractIlluminaBarcodes.BarcodeMetric)</a></span> - Static method in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.html" title="class in picard.illumina">ExtractIlluminaBarcodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/ClassFinder.html#find-java.lang.String-java.lang.Class-">find(String, Class&lt;?&gt;)</a></span> - Method in class picard.cmdline.<a href="picard/cmdline/ClassFinder.html" title="class in picard.cmdline">ClassFinder</a></dt>
<dd>
<div class="block">Scans the classpath for classes within the specified package and sub-packages that
 extend the parentType.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.PerTileBarcodeExtractor.html#findBestBarcodeAndUpdateMetrics-byte:A:A-byte:A:A-boolean-java.util.Map-picard.illumina.ExtractIlluminaBarcodes.BarcodeMetric-int-int-int-int-">findBestBarcodeAndUpdateMetrics(byte[][], byte[][], boolean, Map&lt;String, ExtractIlluminaBarcodes.BarcodeMetric&gt;, ExtractIlluminaBarcodes.BarcodeMetric, int, int, int, int)</a></span> - Static method in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.PerTileBarcodeExtractor.html" title="class in picard.illumina">ExtractIlluminaBarcodes.PerTileBarcodeExtractor</a></dt>
<dd>
<div class="block">Find the best barcode match for the given read sequence, and accumulate metrics</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/ClippingUtility.html#findIndexOfClipSequence-byte:A-byte:A-int-double-">findIndexOfClipSequence(byte[], byte[], int, double)</a></span> - Static method in class picard.util.<a href="picard/util/ClippingUtility.html" title="class in picard.util">ClippingUtility</a></dt>
<dd>
<div class="block">Finds the first index of the adapterSequence sequence in the read sequence requiring at least minMatch
 bases of pairwise alignment with a maximum number of errors dictated by maxErrorRate.</div>
</dd>
<dt><a href="picard/vcf/MendelianViolations/FindMendelianViolations.html" title="class in picard.vcf.MendelianViolations"><span class="typeNameLink">FindMendelianViolations</span></a> - Class in <a href="picard/vcf/MendelianViolations/package-summary.html">picard.vcf.MendelianViolations</a></dt>
<dd>
<div class="block">Finds mendelian violations of all types within a VCF.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/FindMendelianViolations.html#FindMendelianViolations--">FindMendelianViolations()</a></span> - Constructor for class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/FindMendelianViolations.html" title="class in picard.vcf.MendelianViolations">FindMendelianViolations</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/OpticalDuplicateFinder.html#findOpticalDuplicates-java.util.List-picard.sam.util.PhysicalLocation-">findOpticalDuplicates(List&lt;? extends PhysicalLocation&gt;, PhysicalLocation)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/OpticalDuplicateFinder.html" title="class in picard.sam.markduplicates.util">OpticalDuplicateFinder</a></dt>
<dd>
<div class="block">Finds which reads within the list of duplicates that are likely to be optical/co-localized duplicates of
 one another.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaDataProviderFactory.html#findPreferredFormat-picard.illumina.parser.IlluminaDataType-picard.illumina.parser.IlluminaFileUtil-">findPreferredFormat(IlluminaDataType, IlluminaFileUtil)</a></span> - Static method in class picard.illumina.parser.<a href="picard/illumina/parser/IlluminaDataProviderFactory.html" title="class in picard.illumina.parser">IlluminaDataProviderFactory</a></dt>
<dd>
<div class="block">Given a data type find the most preferred file format even if files are not available</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/TileIndex.html#findTile-int-">findTile(int)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/TileIndex.html" title="class in picard.illumina.parser">TileIndex</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaDataProviderFactory.html#findUnmatchedTypes-java.util.Set-java.util.Map-">findUnmatchedTypes(Set&lt;IlluminaDataType&gt;, Map&lt;IlluminaFileUtil.SupportedIlluminaFormat, Set&lt;IlluminaDataType&gt;&gt;)</a></span> - Static method in class picard.illumina.parser.<a href="picard/illumina/parser/IlluminaDataProviderFactory.html" title="class in picard.illumina.parser">IlluminaDataProviderFactory</a></dt>
<dd>
<div class="block">Given a set of formats to data types they provide, find any requested data types that do not have a format associated with them and return them</div>
</dd>
<dt><a href="picard/fingerprint/Fingerprint.html" title="class in picard.fingerprint"><span class="typeNameLink">Fingerprint</span></a> - Class in <a href="picard/fingerprint/package-summary.html">picard.fingerprint</a></dt>
<dd>
<div class="block">Small class to represent a genetic fingerprint as a set of HaplotypeProbabilities
 objects that give the relative probabilities of each of the possible haplotypes
 at a locus.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/Fingerprint.html#Fingerprint-java.lang.String-java.io.File-java.lang.String-">Fingerprint(String, File, String)</a></span> - Constructor for class picard.fingerprint.<a href="picard/fingerprint/Fingerprint.html" title="class in picard.fingerprint">Fingerprint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CheckFingerprint.html#FINGERPRINT_DETAIL_FILE_SUFFIX">FINGERPRINT_DETAIL_FILE_SUFFIX</a></span> - Static variable in class picard.fingerprint.<a href="picard/fingerprint/CheckFingerprint.html" title="class in picard.fingerprint">CheckFingerprint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CheckFingerprint.html#FINGERPRINT_SUMMARY_FILE_SUFFIX">FINGERPRINT_SUMMARY_FILE_SUFFIX</a></span> - Static variable in class picard.fingerprint.<a href="picard/fingerprint/CheckFingerprint.html" title="class in picard.fingerprint">CheckFingerprint</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/fingerprint/FingerprintChecker.html" title="class in picard.fingerprint"><span class="typeNameLink">FingerprintChecker</span></a> - Class in <a href="picard/fingerprint/package-summary.html">picard.fingerprint</a></dt>
<dd>
<div class="block">Major class that coordinates the activities involved in comparing genetic fingerprint
 data whether the source is from a genotyping platform or derived from sequence data.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintChecker.html#FingerprintChecker-java.io.File-">FingerprintChecker(File)</a></span> - Constructor for class picard.fingerprint.<a href="picard/fingerprint/FingerprintChecker.html" title="class in picard.fingerprint">FingerprintChecker</a></dt>
<dd>
<div class="block">Creates a fingerprint checker that will work with the set of haplotypes stored in
 the supplied file.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintChecker.html#FingerprintChecker-picard.fingerprint.HaplotypeMap-">FingerprintChecker(HaplotypeMap)</a></span> - Constructor for class picard.fingerprint.<a href="picard/fingerprint/FingerprintChecker.html" title="class in picard.fingerprint">FingerprintChecker</a></dt>
<dd>
<div class="block">Creates a fingerprint checker that will work with the set of haplotyped provided.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintChecker.html#fingerprintFiles-java.util.Collection-int-int-java.util.concurrent.TimeUnit-">fingerprintFiles(Collection&lt;File&gt;, int, int, TimeUnit)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/FingerprintChecker.html" title="class in picard.fingerprint">FingerprintChecker</a></dt>
<dd>
<div class="block">Fingerprints one or more SAM/BAM/VCF files at all available loci within the haplotype map, using multiple threads
 to speed up the processing.</div>
</dd>
<dt><a href="picard/fingerprint/FingerprintIdDetails.html" title="class in picard.fingerprint"><span class="typeNameLink">FingerprintIdDetails</span></a> - Class in <a href="picard/fingerprint/package-summary.html">picard.fingerprint</a></dt>
<dd>
<div class="block">class to hold the details of a element of fingerprinting PU tag</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintIdDetails.html#FingerprintIdDetails--">FingerprintIdDetails()</a></span> - Constructor for class picard.fingerprint.<a href="picard/fingerprint/FingerprintIdDetails.html" title="class in picard.fingerprint">FingerprintIdDetails</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintIdDetails.html#FingerprintIdDetails-java.lang.String-java.lang.String-">FingerprintIdDetails(String, String)</a></span> - Constructor for class picard.fingerprint.<a href="picard/fingerprint/FingerprintIdDetails.html" title="class in picard.fingerprint">FingerprintIdDetails</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintIdDetails.html#FingerprintIdDetails-htsjdk.samtools.SAMReadGroupRecord-java.lang.String-">FingerprintIdDetails(SAMReadGroupRecord, String)</a></span> - Constructor for class picard.fingerprint.<a href="picard/fingerprint/FingerprintIdDetails.html" title="class in picard.fingerprint">FingerprintIdDetails</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/cmdline/programgroups/Fingerprinting.html" title="class in picard.cmdline.programgroups"><span class="typeNameLink">Fingerprinting</span></a> - Class in <a href="picard/cmdline/programgroups/package-summary.html">picard.cmdline.programgroups</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/programgroups/Fingerprinting.html#Fingerprinting--">Fingerprinting()</a></span> - Constructor for class picard.cmdline.programgroups.<a href="picard/cmdline/programgroups/Fingerprinting.html" title="class in picard.cmdline.programgroups">Fingerprinting</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/FingerprintingDetailMetrics.html" title="class in picard.analysis"><span class="typeNameLink">FingerprintingDetailMetrics</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Detailed metrics about an individual SNP/Haplotype comparison within a fingerprint comparison.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/FingerprintingDetailMetrics.html#FingerprintingDetailMetrics--">FingerprintingDetailMetrics()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/FingerprintingDetailMetrics.html" title="class in picard.analysis">FingerprintingDetailMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/FingerprintingSummaryMetrics.html" title="class in picard.analysis"><span class="typeNameLink">FingerprintingSummaryMetrics</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Summary fingerprinting metrics and statistics about the comparison of the sequence data
 from a single read group (lane or index within a lane) vs.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/FingerprintingSummaryMetrics.html#FingerprintingSummaryMetrics--">FingerprintingSummaryMetrics()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/FingerprintingSummaryMetrics.html" title="class in picard.analysis">FingerprintingSummaryMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/fingerprint/FingerprintResults.html" title="class in picard.fingerprint"><span class="typeNameLink">FingerprintResults</span></a> - Class in <a href="picard/fingerprint/package-summary.html">picard.fingerprint</a></dt>
<dd>
<div class="block">Class that is used to represent the results of comparing a read group within a SAM file, or a sample
 within a VCF against one or more set of fingerprint genotypes.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintResults.html#FingerprintResults-java.io.File-java.lang.String-java.lang.String-">FingerprintResults(File, String, String)</a></span> - Constructor for class picard.fingerprint.<a href="picard/fingerprint/FingerprintResults.html" title="class in picard.fingerprint">FingerprintResults</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintChecker.html#fingerprintSamFile-java.io.File-htsjdk.samtools.util.IntervalList-">fingerprintSamFile(File, IntervalList)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/FingerprintChecker.html" title="class in picard.fingerprint">FingerprintChecker</a></dt>
<dd>
<div class="block">Generates a Fingerprint per read group in the supplied SAM file using the loci provided in
 the interval list.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintChecker.html#fingerprintVcf-java.io.File-">fingerprintVcf(File)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/FingerprintChecker.html" title="class in picard.fingerprint">FingerprintChecker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html#finish--">finish()</a></span> - Method in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html" title="class in picard.analysis.artifacts">CollectSequencingArtifactMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectAlignmentSummaryMetrics.html#finish--">finish()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectAlignmentSummaryMetrics.html" title="class in picard.analysis">CollectAlignmentSummaryMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectBaseDistributionByCycle.html#finish--">finish()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectBaseDistributionByCycle.html" title="class in picard.analysis">CollectBaseDistributionByCycle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectGcBiasMetrics.html#finish--">finish()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectGcBiasMetrics.html" title="class in picard.analysis">CollectGcBiasMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectInsertSizeMetrics.html#finish--">finish()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectInsertSizeMetrics.html" title="class in picard.analysis">CollectInsertSizeMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectQualityYieldMetrics.html#finish--">finish()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectQualityYieldMetrics.html" title="class in picard.analysis">CollectQualityYieldMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetricsCollector.html#finish--">finish()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetricsCollector.html" title="class in picard.analysis">CollectQualityYieldMetrics.QualityYieldMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRnaSeqMetrics.html#finish--">finish()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectRnaSeqMetrics.html" title="class in picard.analysis">CollectRnaSeqMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/InsertSizeMetricsCollector.PerUnitInsertSizeMetricsCollector.html#finish--">finish()</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/InsertSizeMetricsCollector.PerUnitInsertSizeMetricsCollector.html" title="class in picard.analysis.directed">InsertSizeMetricsCollector.PerUnitInsertSizeMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/RnaSeqMetricsCollector.PerUnitRnaSeqMetricsCollector.html#finish--">finish()</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/RnaSeqMetricsCollector.PerUnitRnaSeqMetricsCollector.html" title="class in picard.analysis.directed">RnaSeqMetricsCollector.PerUnitRnaSeqMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetMetricsCollector.PerUnitTargetMetricCollector.html#finish--">finish()</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/TargetMetricsCollector.PerUnitTargetMetricCollector.html" title="class in picard.analysis.directed">TargetMetricsCollector.PerUnitTargetMetricCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasMetricsCollector.PerUnitGcBiasMetricsCollector.html#finish--">finish()</a></span> - Method in class picard.analysis.<a href="picard/analysis/GcBiasMetricsCollector.PerUnitGcBiasMetricsCollector.html" title="class in picard.analysis">GcBiasMetricsCollector.PerUnitGcBiasMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/MeanQualityByCycle.html#finish--">finish()</a></span> - Method in class picard.analysis.<a href="picard/analysis/MeanQualityByCycle.html" title="class in picard.analysis">MeanQualityByCycle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/QualityScoreDistribution.html#finish--">finish()</a></span> - Method in class picard.analysis.<a href="picard/analysis/QualityScoreDistribution.html" title="class in picard.analysis">QualityScoreDistribution</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/SinglePassSamProgram.html#finish--">finish()</a></span> - Method in class picard.analysis.<a href="picard/analysis/SinglePassSamProgram.html" title="class in picard.analysis">SinglePassSamProgram</a></dt>
<dd>
<div class="block">Should be implemented by subclasses to do one-time finalization work.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/metrics/MultiLevelCollector.html#finish--">finish()</a></span> - Method in class picard.metrics.<a href="picard/metrics/MultiLevelCollector.html" title="class in picard.metrics">MultiLevelCollector</a></dt>
<dd>
<div class="block">Call finish on all PerUnitMetricCollectors</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/metrics/PerUnitMetricCollector.html#finish--">finish()</a></span> - Method in interface picard.metrics.<a href="picard/metrics/PerUnitMetricCollector.html" title="interface in picard.metrics">PerUnitMetricCollector</a></dt>
<dd>
<div class="block">When all records have been collected, compute any final values needed to finish constructing metrics/Histogram</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToFastq.html#FIRST_TILE">FIRST_TILE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToFastq.html" title="class in picard.illumina">IlluminaBasecallsToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#FIRST_TILE">FIRST_TILE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/filter/FisherStrandFilter.html" title="class in picard.vcf.filter"><span class="typeNameLink">FisherStrandFilter</span></a> - Class in <a href="picard/vcf/filter/package-summary.html">picard.vcf.filter</a></dt>
<dd>
<div class="block">Filters records based on the phred scaled p-value from the Fisher Strand test stored in
 the FS attribute.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/FisherStrandFilter.html#FisherStrandFilter-double-">FisherStrandFilter(double)</a></span> - Constructor for class picard.vcf.filter.<a href="picard/vcf/filter/FisherStrandFilter.html" title="class in picard.vcf.filter">FisherStrandFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#FIVE_PRIME_ADAPTER">FIVE_PRIME_ADAPTER</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/MarkIlluminaAdapters.html#FIVE_PRIME_ADAPTER">FIVE_PRIME_ADAPTER</a></span> - Variable in class picard.illumina.<a href="picard/illumina/MarkIlluminaAdapters.html" title="class in picard.illumina">MarkIlluminaAdapters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/LiftoverVcf.html#fixGenotypes-htsjdk.variant.variantcontext.GenotypesContext-java.util.Map-">fixGenotypes(GenotypesContext, Map&lt;Allele, Allele&gt;)</a></span> - Static method in class picard.vcf.<a href="picard/vcf/LiftoverVcf.html" title="class in picard.vcf">LiftoverVcf</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/FixMateInformation.html" title="class in picard.sam"><span class="typeNameLink">FixMateInformation</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block">This tool ensures that all mate-pair information is in sync between each read
 and its mate pair.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FixMateInformation.html#FixMateInformation--">FixMateInformation()</a></span> - Constructor for class picard.sam.<a href="picard/sam/FixMateInformation.html" title="class in picard.sam">FixMateInformation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.html#fixNmMdAndUq-htsjdk.samtools.SAMRecord-htsjdk.samtools.reference.ReferenceSequenceFileWalker-boolean-">fixNmMdAndUq(SAMRecord, ReferenceSequenceFileWalker, boolean)</a></span> - Static method in class picard.sam.<a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</a></dt>
<dd>
<div class="block">Calculates and sets the NM, MD, and and UQ tags from the record and the reference</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.html#fixUq-htsjdk.samtools.SAMRecord-htsjdk.samtools.reference.ReferenceSequenceFileWalker-boolean-">fixUq(SAMRecord, ReferenceSequenceFileWalker, boolean)</a></span> - Static method in class picard.sam.<a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</a></dt>
<dd>
<div class="block">Calculates and sets UQ tag from the record and the reference</div>
</dd>
<dt><a href="picard/vcf/FixVcfHeader.html" title="class in picard.vcf"><span class="typeNameLink">FixVcfHeader</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">Tool for replacing or fixing up a VCF header.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/FixVcfHeader.html#FixVcfHeader--">FixVcfHeader()</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/FixVcfHeader.html" title="class in picard.vcf">FixVcfHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/Snp.html#flip--">flip()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/Snp.html" title="class in picard.fingerprint">Snp</a></dt>
<dd>
<div class="block">Returns a new SNP object with the alleles swapped and MAF corrected.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/LiftoverVcf.html#flipIndel-htsjdk.variant.variantcontext.VariantContext-htsjdk.samtools.liftover.LiftOver-htsjdk.samtools.reference.ReferenceSequence-">flipIndel(VariantContext, LiftOver, ReferenceSequence)</a></span> - Static method in class picard.vcf.<a href="picard/vcf/LiftoverVcf.html" title="class in picard.vcf">LiftoverVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fastq/BamToBfq.html#FLOWCELL_BARCODE">FLOWCELL_BARCODE</a></span> - Variable in class picard.fastq.<a href="picard/fastq/BamToBfq.html" title="class in picard.fastq">BamToBfq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToFastq.html#FLOWCELL_BARCODE">FLOWCELL_BARCODE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToFastq.html" title="class in picard.illumina">IlluminaBasecallsToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceContingencyMetrics.html#FN_COUNT">FN_COUNT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceContingencyMetrics.html" title="class in picard.vcf">GenotypeConcordanceContingencyMetrics</a></dt>
<dd>
<div class="block">The FN (false negative) count across all variants</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceScheme.html#FN_ONLY">FN_ONLY</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceScheme.html" title="class in picard.vcf">GenotypeConcordanceScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#FOLD_80_BASE_PENALTY">FOLD_80_BASE_PENALTY</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The fold over-coverage necessary to raise 80% of bases in "non-zero-cvg" targets to the mean coverage level in those targets.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#FOLD_80_BASE_PENALTY">FOLD_80_BASE_PENALTY</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The fold over-coverage necessary to raise 80% of bases in "non-zero-cvg" targets to
 the mean coverage level in those targets.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#FOLD_ENRICHMENT">FOLD_ENRICHMENT</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The fold by which the baited region has been amplified above genomic background.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#FOLD_ENRICHMENT">FOLD_ENRICHMENT</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The fold by which the amplicon region has been amplified above genomic background.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html#foldInto-T-java.util.Collection-">foldInto(T, Collection&lt;T&gt;)</a></span> - Static method in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingSummaryMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToFastq.html#FORCE_GC">FORCE_GC</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToFastq.html" title="class in picard.illumina">IlluminaBasecallsToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#FORCE_GC">FORCE_GC</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaDataProviderFactory.html#formatToDataTypes">formatToDataTypes</a></span> - Variable in class picard.illumina.parser.<a href="picard/illumina/parser/IlluminaDataProviderFactory.html" title="class in picard.illumina.parser">IlluminaDataProviderFactory</a></dt>
<dd>
<div class="block">A Map of file formats to the dataTypes they will provide for this run.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VcfFileSegmentGenerator.html#forVcf-java.io.File-">forVcf(File)</a></span> - Method in class picard.vcf.processor.<a href="picard/vcf/processor/VcfFileSegmentGenerator.html" title="class in picard.vcf.processor">VcfFileSegmentGenerator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/FourChannelIntensityData.html" title="class in picard.illumina.parser"><span class="typeNameLink">FourChannelIntensityData</span></a> - Class in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>
<div class="block">Holds a 4 short values for each cycle of a read.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/FourChannelIntensityData.html#FourChannelIntensityData-int-">FourChannelIntensityData(int)</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/FourChannelIntensityData.html" title="class in picard.illumina.parser">FourChannelIntensityData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceContingencyMetrics.html#FP_COUNT">FP_COUNT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceContingencyMetrics.html" title="class in picard.vcf">GenotypeConcordanceContingencyMetrics</a></dt>
<dd>
<div class="block">The FP (false positive) count across all variants</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceScheme.html#FP_FN">FP_FN</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceScheme.html" title="class in picard.vcf">GenotypeConcordanceScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceScheme.html#FP_ONLY">FP_ONLY</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceScheme.html" title="class in picard.vcf">GenotypeConcordanceScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceScheme.html#FP_TN">FP_TN</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceScheme.html" title="class in picard.vcf">GenotypeConcordanceScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceScheme.html#FP_TN_FN">FP_TN_FN</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceScheme.html" title="class in picard.vcf">GenotypeConcordanceScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEnds.html#FR">FR</a></span> - Static variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEnds.html" title="class in picard.sam.markduplicates.util">ReadEnds</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/PositionBasedDownsampleSam.html#FRACTION">FRACTION</a></span> - Variable in class picard.sam.<a href="picard/sam/PositionBasedDownsampleSam.html" title="class in picard.sam">PositionBasedDownsampleSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/JumpingLibraryMetrics.html#FRAGMENTS">FRAGMENTS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/JumpingLibraryMetrics.html" title="class in picard.analysis">JumpingLibraryMetrics</a></dt>
<dd>
<div class="block">The number of fragments in the SAM file</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/DiploidGenotype.html#fromBases-byte:A-">fromBases(byte[])</a></span> - Static method in enum picard.fingerprint.<a href="picard/fingerprint/DiploidGenotype.html" title="enum in picard.fingerprint">DiploidGenotype</a></dt>
<dd>
<div class="block">Converts a pair of bases into a DiploidGenotype regardless of base order or case</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/DiploidGenotype.html#fromBases-byte-byte-">fromBases(byte, byte)</a></span> - Static method in enum picard.fingerprint.<a href="picard/fingerprint/DiploidGenotype.html" title="enum in picard.fingerprint">DiploidGenotype</a></dt>
<dd>
<div class="block">Converts a pair of bases into a DiploidGenotype regardless of base order or case</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/pedigree/Sex.html#fromCode-int-">fromCode(int)</a></span> - Static method in enum picard.pedigree.<a href="picard/pedigree/Sex.html" title="enum in picard.pedigree">Sex</a></dt>
<dd>
<div class="block">Decodes the Sex from a numeric code.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/pedigree/PedFile.html#fromFile-java.io.File-boolean-">fromFile(File, boolean)</a></span> - Static method in class picard.pedigree.<a href="picard/pedigree/PedFile.html" title="class in picard.pedigree">PedFile</a></dt>
<dd>
<div class="block">Attempts to read a pedigree file into memory.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/pedigree/PedFile.html#fromSexMap-java.util.Map-">fromSexMap(Map&lt;String, Sex&gt;)</a></span> - Static method in class picard.pedigree.<a href="picard/pedigree/PedFile.html" title="class in picard.pedigree">PedFile</a></dt>
<dd>
<div class="block">Function that accepts a map from sample-name to its sex and creates a PEDFile documenting the sexes.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html#FWD_CXT_ALT_BASES">FWD_CXT_ALT_BASES</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.BaitBiasDetailMetrics</a></dt>
<dd>
<div class="block">The number of REF_BASE:ALT_BASE alignments at sites with the given reference context.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html#FWD_CXT_REF_BASES">FWD_CXT_REF_BASES</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.BaitBiasDetailMetrics</a></dt>
<dd>
<div class="block">The number of REF_BASE:REF_BASE alignments at sites with the given reference context.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html#FWD_ERROR_RATE">FWD_ERROR_RATE</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.BaitBiasDetailMetrics</a></dt>
<dd>
<div class="block">The substitution rate of REF_BASE:ALT_BASE, calculated as max(1e-10, FWD_CXT_ALT_BASES / (FWD_CXT_ALT_BASES + FWD_CXT_REF_BASES)).</div>
</dd>
</dl>
<a name="I:G">
<!--   -->
</a>
<h2 class="title">G</h2>
<dl>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html#G_REF_ALT_BASES">G_REF_ALT_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html" title="class in picard.analysis">CollectOxoGMetrics.CpcgMetrics</a></dt>
<dd>
<div class="block">The number of alt (A/T) basecalls observed at sites where the genome reference == G.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html#G_REF_OXO_ERROR_RATE">G_REF_OXO_ERROR_RATE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html" title="class in picard.analysis">CollectOxoGMetrics.CpcgMetrics</a></dt>
<dd>
<div class="block">The rate at which C>A and G>T substitutions are observed at G reference sites above the expected rate if there
 were no bias between sites with a C reference base vs.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html#G_REF_OXO_Q">G_REF_OXO_Q</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html" title="class in picard.analysis">CollectOxoGMetrics.CpcgMetrics</a></dt>
<dd>
<div class="block">G_REF_OXO_ERROR_RATE expressed as a phred-scaled quality score.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html#G_REF_REF_BASES">G_REF_REF_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html" title="class in picard.analysis">CollectOxoGMetrics.CpcgMetrics</a></dt>
<dd>
<div class="block">The number of ref basecalls observed at sites where the genome reference == G.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#GA4GH_CLIENT_SECRETS">GA4GH_CLIENT_SECRETS</a></span> - Variable in class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/GA4GHScheme.html" title="class in picard.vcf"><span class="typeNameLink">GA4GHScheme</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">The scheme is defined in the constructor.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GA4GHScheme.html#GA4GHScheme--">GA4GHScheme()</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/GA4GHScheme.html" title="class in picard.vcf">GA4GHScheme</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/GA4GHSchemeWithMissingAsHomRef.html" title="class in picard.vcf"><span class="typeNameLink">GA4GHSchemeWithMissingAsHomRef</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">The default scheme is derived from the GA4GH Benchmarking Work Group's proposed evaluation scheme.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GA4GHSchemeWithMissingAsHomRef.html#GA4GHSchemeWithMissingAsHomRef--">GA4GHSchemeWithMissingAsHomRef()</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/GA4GHSchemeWithMissingAsHomRef.html" title="class in picard.vcf">GA4GHSchemeWithMissingAsHomRef</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/GatherBamFiles.html" title="class in picard.sam"><span class="typeNameLink">GatherBamFiles</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block">Program to perform a rapid "gather" operation on BAM files after a scatter operations where
 the same process has been performed on different regions of a BAM file creating many smaller
 BAM files that now need to be concatenated back together.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/GatherBamFiles.html#GatherBamFiles--">GatherBamFiles()</a></span> - Constructor for class picard.sam.<a href="picard/sam/GatherBamFiles.html" title="class in picard.sam">GatherBamFiles</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/GatherVcfs.html" title="class in picard.vcf"><span class="typeNameLink">GatherVcfs</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">Simple little class that combines multiple VCFs that have exactly the same set of samples
 and totally discrete sets of loci.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GatherVcfs.html#GatherVcfs--">GatherVcfs()</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/GatherVcfs.html" title="class in picard.vcf">GatherVcfs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasDetailMetrics.html#GC">GC</a></span> - Variable in class picard.analysis.<a href="picard/analysis/GcBiasDetailMetrics.html" title="class in picard.analysis">GcBiasDetailMetrics</a></dt>
<dd>
<div class="block">The G+C content of the reference sequence represented by this bin.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#GC_DROPOUT">GC_DROPOUT</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">A measure of how undercovered >= 50% GC regions are relative to the mean.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#GC_DROPOUT">GC_DROPOUT</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">A measure of how regions of high GC content (>= 50% GC) are undercovered relative to the mean coverage
 value.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasSummaryMetrics.html#GC_DROPOUT">GC_DROPOUT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/GcBiasSummaryMetrics.html" title="class in picard.analysis">GcBiasSummaryMetrics</a></dt>
<dd>
<div class="block">Illumina-style GC dropout metric.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasSummaryMetrics.html#GC_NC_0_19">GC_NC_0_19</a></span> - Variable in class picard.analysis.<a href="picard/analysis/GcBiasSummaryMetrics.html" title="class in picard.analysis">GcBiasSummaryMetrics</a></dt>
<dd>
<div class="block">Normalized coverage over quintile of GC content ranging from 0 - 19.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasSummaryMetrics.html#GC_NC_20_39">GC_NC_20_39</a></span> - Variable in class picard.analysis.<a href="picard/analysis/GcBiasSummaryMetrics.html" title="class in picard.analysis">GcBiasSummaryMetrics</a></dt>
<dd>
<div class="block">Normalized coverage over each quintile of GC content ranging from 20 - 39.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasSummaryMetrics.html#GC_NC_40_59">GC_NC_40_59</a></span> - Variable in class picard.analysis.<a href="picard/analysis/GcBiasSummaryMetrics.html" title="class in picard.analysis">GcBiasSummaryMetrics</a></dt>
<dd>
<div class="block">Normalized coverage over each quintile of GC content ranging from 40 - 59.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasSummaryMetrics.html#GC_NC_60_79">GC_NC_60_79</a></span> - Variable in class picard.analysis.<a href="picard/analysis/GcBiasSummaryMetrics.html" title="class in picard.analysis">GcBiasSummaryMetrics</a></dt>
<dd>
<div class="block">Normalized coverage over each quintile of GC content ranging from 60 - 79.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasSummaryMetrics.html#GC_NC_80_100">GC_NC_80_100</a></span> - Variable in class picard.analysis.<a href="picard/analysis/GcBiasSummaryMetrics.html" title="class in picard.analysis">GcBiasSummaryMetrics</a></dt>
<dd>
<div class="block">Normalized coverage over each quintile of GC content ranging from 80 - 100.</div>
</dd>
<dt><a href="picard/analysis/GcBiasDetailMetrics.html" title="class in picard.analysis"><span class="typeNameLink">GcBiasDetailMetrics</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Class that holds detailed metrics about reads that fall within windows of a certain
 GC bin on the reference genome.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasDetailMetrics.html#GcBiasDetailMetrics--">GcBiasDetailMetrics()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/GcBiasDetailMetrics.html" title="class in picard.analysis">GcBiasDetailMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/metrics/GcBiasMetrics.html" title="class in picard.metrics"><span class="typeNameLink">GcBiasMetrics</span></a> - Class in <a href="picard/metrics/package-summary.html">picard.metrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/metrics/GcBiasMetrics.html#GcBiasMetrics--">GcBiasMetrics()</a></span> - Constructor for class picard.metrics.<a href="picard/metrics/GcBiasMetrics.html" title="class in picard.metrics">GcBiasMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/GcBiasMetricsCollector.html" title="class in picard.analysis"><span class="typeNameLink">GcBiasMetricsCollector</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Calculates GC Bias Metrics on multiple levels
  Created by kbergin on 3/23/15.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasMetricsCollector.html#GcBiasMetricsCollector-java.util.Set-int:A-java.util.List-int-boolean-">GcBiasMetricsCollector(Set&lt;MetricAccumulationLevel&gt;, int[], List&lt;SAMReadGroupRecord&gt;, int, boolean)</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/GcBiasMetricsCollector.html" title="class in picard.analysis">GcBiasMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasMetricsCollector.html#GcBiasMetricsCollector-java.util.Set-int:A-java.util.List-int-boolean-boolean-">GcBiasMetricsCollector(Set&lt;MetricAccumulationLevel&gt;, int[], List&lt;SAMReadGroupRecord&gt;, int, boolean, boolean)</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/GcBiasMetricsCollector.html" title="class in picard.analysis">GcBiasMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/GcBiasMetricsCollector.PerUnitGcBiasMetricsCollector.html" title="class in picard.analysis"><span class="typeNameLink">GcBiasMetricsCollector.PerUnitGcBiasMetricsCollector</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/GcBiasSummaryMetrics.html" title="class in picard.analysis"><span class="typeNameLink">GcBiasSummaryMetrics</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">High level metrics that capture how biased the coverage in a certain lane is.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasSummaryMetrics.html#GcBiasSummaryMetrics--">GcBiasSummaryMetrics()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/GcBiasSummaryMetrics.html" title="class in picard.analysis">GcBiasSummaryMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/GcBiasUtils.html" title="class in picard.analysis"><span class="typeNameLink">GcBiasUtils</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Utilities to calculate GC Bias
 Created by kbergin on 9/23/15.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasUtils.html#GcBiasUtils--">GcBiasUtils()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/GcBiasUtils.html" title="class in picard.analysis">GcBiasUtils</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/annotation/Gene.html" title="class in picard.annotation"><span class="typeNameLink">Gene</span></a> - Class in <a href="picard/annotation/package-summary.html">picard.annotation</a></dt>
<dd>
<div class="block">Holds annotation of a gene for storage in an OverlapDetector.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/Gene.html#Gene-java.lang.String-int-int-boolean-java.lang.String-">Gene(String, int, int, boolean, String)</a></span> - Constructor for class picard.annotation.<a href="picard/annotation/Gene.html" title="class in picard.annotation">Gene</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/annotation/Gene.Transcript.html" title="class in picard.annotation"><span class="typeNameLink">Gene.Transcript</span></a> - Class in <a href="picard/annotation/package-summary.html">picard.annotation</a></dt>
<dd>
<div class="block">A single transcript of a gene.</div>
</dd>
<dt><a href="picard/annotation/Gene.Transcript.Exon.html" title="class in picard.annotation"><span class="typeNameLink">Gene.Transcript.Exon</span></a> - Class in <a href="picard/annotation/package-summary.html">picard.annotation</a></dt>
<dd>
<div class="block">1-based, inclusive representation of an exon.</div>
</dd>
<dt><a href="picard/annotation/GeneAnnotationReader.html" title="class in picard.annotation"><span class="typeNameLink">GeneAnnotationReader</span></a> - Class in <a href="picard/annotation/package-summary.html">picard.annotation</a></dt>
<dd>
<div class="block">Load gene annotations into an OverlapDetector of Gene objects.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/GeneAnnotationReader.html#GeneAnnotationReader--">GeneAnnotationReader()</a></span> - Constructor for class picard.annotation.<a href="picard/annotation/GeneAnnotationReader.html" title="class in picard.annotation">GeneAnnotationReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fastq/Casava18ReadNameEncoder.html#generateReadName-picard.illumina.parser.ClusterData-java.lang.Integer-">generateReadName(ClusterData, Integer)</a></span> - Method in class picard.fastq.<a href="picard/fastq/Casava18ReadNameEncoder.html" title="class in picard.fastq">Casava18ReadNameEncoder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fastq/IlluminaReadNameEncoder.html#generateReadName-picard.illumina.parser.ClusterData-java.lang.Integer-">generateReadName(ClusterData, Integer)</a></span> - Method in class picard.fastq.<a href="picard/fastq/IlluminaReadNameEncoder.html" title="class in picard.fastq">IlluminaReadNameEncoder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fastq/ReadNameEncoder.html#generateReadName-picard.illumina.parser.ClusterData-java.lang.Integer-">generateReadName(ClusterData, Integer)</a></span> - Method in interface picard.fastq.<a href="picard/fastq/ReadNameEncoder.html" title="interface in picard.fastq">ReadNameEncoder</a></dt>
<dd>
<div class="block">Generates a read name string for the provided cluster.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRawWgsMetrics.html#generateWgsMetrics-htsjdk.samtools.util.IntervalList-htsjdk.samtools.util.Histogram-htsjdk.samtools.util.Histogram-double-double-double-double-double-double-double-int-htsjdk.samtools.util.Histogram-int-">generateWgsMetrics(IntervalList, Histogram&lt;Integer&gt;, Histogram&lt;Integer&gt;, double, double, double, double, double, double, double, int, Histogram&lt;Integer&gt;, int)</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectRawWgsMetrics.html" title="class in picard.analysis">CollectRawWgsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.html#generateWgsMetrics-htsjdk.samtools.util.IntervalList-htsjdk.samtools.util.Histogram-htsjdk.samtools.util.Histogram-double-double-double-double-double-double-double-int-htsjdk.samtools.util.Histogram-int-">generateWgsMetrics(IntervalList, Histogram&lt;Integer&gt;, Histogram&lt;Integer&gt;, double, double, double, double, double, double, double, int, Histogram&lt;Integer&gt;, int)</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.html#generateWgsMetrics-htsjdk.samtools.util.IntervalList-htsjdk.samtools.util.Histogram-htsjdk.samtools.util.Histogram-double-double-double-double-double-double-double-int-htsjdk.samtools.util.Histogram-int-">generateWgsMetrics(IntervalList, Histogram&lt;Integer&gt;, Histogram&lt;Integer&gt;, double, double, double, double, double, double, double, int, Histogram&lt;Integer&gt;, int)</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.html" title="class in picard.analysis">CollectWgsMetricsWithNonZeroCoverage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VariantProcessor.Builder.html#generatingAccumulatorsBy-picard.vcf.processor.VariantProcessor.AccumulatorGenerator-">generatingAccumulatorsBy(VariantProcessor.AccumulatorGenerator&lt;A, R&gt;)</a></span> - Static method in class picard.vcf.processor.<a href="picard/vcf/processor/VariantProcessor.Builder.html" title="class in picard.vcf.processor">VariantProcessor.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CreateSequenceDictionary.html#GENOME_ASSEMBLY">GENOME_ASSEMBLY</a></span> - Variable in class picard.sam.<a href="picard/sam/CreateSequenceDictionary.html" title="class in picard.sam">CreateSequenceDictionary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#GENOME_SIZE">GENOME_SIZE</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The number of bases in the reference genome used for alignment.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#GENOME_SIZE">GENOME_SIZE</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The number of bases in the reference genome used for alignment</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#GENOME_TERRITORY">GENOME_TERRITORY</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">The number of non-N bases in the genome reference over which coverage will be evaluated.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html#GENOTYPE_CONCORDANCE">GENOTYPE_CONCORDANCE</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html" title="class in picard.vcf">GenotypeConcordanceSummaryMetrics</a></dt>
<dd>
<div class="block">The genotype concordance for all possible states.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CheckFingerprint.html#GENOTYPE_LOD_THRESHOLD">GENOTYPE_LOD_THRESHOLD</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CheckFingerprint.html" title="class in picard.fingerprint">CheckFingerprint</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf"><span class="typeNameLink">GenotypeConcordance</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">Calculates the concordance between genotype data for two samples in two different VCFs - one being considered the truth (or reference)
 the other being the call.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#GenotypeConcordance--">GenotypeConcordance()</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/GenotypeConcordance.Alleles.html" title="class in picard.vcf"><span class="typeNameLink">GenotypeConcordance.Alleles</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">A simple structure to return the results of getAlleles.</div>
</dd>
<dt><a href="picard/vcf/GenotypeConcordanceContingencyMetrics.html" title="class in picard.vcf"><span class="typeNameLink">GenotypeConcordanceContingencyMetrics</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">Class that holds metrics about the Genotype Concordance contingency tables.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceContingencyMetrics.html#GenotypeConcordanceContingencyMetrics--">GenotypeConcordanceContingencyMetrics()</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/GenotypeConcordanceContingencyMetrics.html" title="class in picard.vcf">GenotypeConcordanceContingencyMetrics</a></dt>
<dd>
<div class="block">Empty constructor - needed for unit tests</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceContingencyMetrics.html#GenotypeConcordanceContingencyMetrics-htsjdk.variant.variantcontext.VariantContext.Type-picard.vcf.GenotypeConcordanceCounts-java.lang.String-java.lang.String-boolean-">GenotypeConcordanceContingencyMetrics(VariantContext.Type, GenotypeConcordanceCounts, String, String, boolean)</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/GenotypeConcordanceContingencyMetrics.html" title="class in picard.vcf">GenotypeConcordanceContingencyMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/GenotypeConcordanceCounts.html" title="class in picard.vcf"><span class="typeNameLink">GenotypeConcordanceCounts</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">A class to store the counts for various truth and call state classifications relative to a reference.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceCounts.html#GenotypeConcordanceCounts--">GenotypeConcordanceCounts()</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/GenotypeConcordanceCounts.html" title="class in picard.vcf">GenotypeConcordanceCounts</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/GenotypeConcordanceDetailMetrics.html" title="class in picard.vcf"><span class="typeNameLink">GenotypeConcordanceDetailMetrics</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">Class that holds detail metrics about Genotype Concordance</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceDetailMetrics.html#GenotypeConcordanceDetailMetrics--">GenotypeConcordanceDetailMetrics()</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/GenotypeConcordanceDetailMetrics.html" title="class in picard.vcf">GenotypeConcordanceDetailMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/GenotypeConcordanceScheme.html" title="class in picard.vcf"><span class="typeNameLink">GenotypeConcordanceScheme</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">This defines for each valid TruthState and CallState tuple, the set of contingency table entries that to which the tuple should contribute.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceScheme.html#GenotypeConcordanceScheme--">GenotypeConcordanceScheme()</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/GenotypeConcordanceScheme.html" title="class in picard.vcf">GenotypeConcordanceScheme</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/GenotypeConcordanceSchemeFactory.html" title="class in picard.vcf"><span class="typeNameLink">GenotypeConcordanceSchemeFactory</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">Created by kbergin on 6/19/15.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceSchemeFactory.html#GenotypeConcordanceSchemeFactory--">GenotypeConcordanceSchemeFactory()</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/GenotypeConcordanceSchemeFactory.html" title="class in picard.vcf">GenotypeConcordanceSchemeFactory</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/GenotypeConcordanceStateCodes.html" title="enum in picard.vcf"><span class="typeNameLink">GenotypeConcordanceStateCodes</span></a> - Enum in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">Created by kbergin on 7/30/15.</div>
</dd>
<dt><a href="picard/vcf/GenotypeConcordanceStates.html" title="class in picard.vcf"><span class="typeNameLink">GenotypeConcordanceStates</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">A class to store the various classifications for:
 1.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceStates.html#GenotypeConcordanceStates--">GenotypeConcordanceStates()</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/GenotypeConcordanceStates.html" title="class in picard.vcf">GenotypeConcordanceStates</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/GenotypeConcordanceStates.CallState.html" title="enum in picard.vcf"><span class="typeNameLink">GenotypeConcordanceStates.CallState</span></a> - Enum in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">These states represent the relationship between the call genotype and the truth genotype relative to
 a reference sequence.</div>
</dd>
<dt><a href="picard/vcf/GenotypeConcordanceStates.ContingencyState.html" title="enum in picard.vcf"><span class="typeNameLink">GenotypeConcordanceStates.ContingencyState</span></a> - Enum in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">A specific state for a 2x2 contingency table.</div>
</dd>
<dt><a href="picard/vcf/GenotypeConcordanceStates.TruthAndCallStates.html" title="class in picard.vcf"><span class="typeNameLink">GenotypeConcordanceStates.TruthAndCallStates</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">A minute class to store the truth and call state respectively.</div>
</dd>
<dt><a href="picard/vcf/GenotypeConcordanceStates.TruthState.html" title="enum in picard.vcf"><span class="typeNameLink">GenotypeConcordanceStates.TruthState</span></a> - Enum in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">These states represent the relationship between a truth genotype and the reference sequence.</div>
</dd>
<dt><a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html" title="class in picard.vcf"><span class="typeNameLink">GenotypeConcordanceSummaryMetrics</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">Class that holds summary metrics about Genotype Concordance</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html#GenotypeConcordanceSummaryMetrics--">GenotypeConcordanceSummaryMetrics()</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html" title="class in picard.vcf">GenotypeConcordanceSummaryMetrics</a></dt>
<dd>
<div class="block">Empty constructor - needed for unit tests</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html#GenotypeConcordanceSummaryMetrics-htsjdk.variant.variantcontext.VariantContext.Type-picard.vcf.GenotypeConcordanceCounts-java.lang.String-java.lang.String-boolean-">GenotypeConcordanceSummaryMetrics(VariantContext.Type, GenotypeConcordanceCounts, String, String, boolean)</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html" title="class in picard.vcf">GenotypeConcordanceSummaryMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/filter/GenotypeFilter.html" title="interface in picard.vcf.filter"><span class="typeNameLink">GenotypeFilter</span></a> - Interface in <a href="picard/vcf/filter/package-summary.html">picard.vcf.filter</a></dt>
<dd>
<div class="block">An interface for classes that perform Genotype filtration.</div>
</dd>
<dt><a href="picard/vcf/filter/GenotypeQualityFilter.html" title="class in picard.vcf.filter"><span class="typeNameLink">GenotypeQualityFilter</span></a> - Class in <a href="picard/vcf/filter/package-summary.html">picard.vcf.filter</a></dt>
<dd>
<div class="block">Genotype filter that filters out genotypes below a given quality threshold.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/GenotypeQualityFilter.html#GenotypeQualityFilter-int-">GenotypeQualityFilter(int)</a></span> - Constructor for class picard.vcf.filter.<a href="picard/vcf/filter/GenotypeQualityFilter.html" title="class in picard.vcf.filter">GenotypeQualityFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CheckFingerprint.html#GENOTYPES">GENOTYPES</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CheckFingerprint.html" title="class in picard.fingerprint">CheckFingerprint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckFingerprints.html#GENOTYPING_ERROR_RATE">GENOTYPING_ERROR_RATE</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckFingerprints.html" title="class in picard.fingerprint">CrosscheckFingerprints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CounterManager.Counter.html#get-int-">get(int)</a></span> - Method in class picard.analysis.<a href="picard/analysis/CounterManager.Counter.html" title="class in picard.analysis">CounterManager.Counter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/BclIndexReader.html#get-int-">get(int)</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/BclIndexReader.html" title="class in picard.illumina.parser.readers">BclIndexReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadStructure.Substructure.html#get-int-">get(int)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ReadStructure.Substructure.html" title="class in picard.illumina.parser">ReadStructure.Substructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AdapterPair.html#get3PrimeAdapter--">get3PrimeAdapter()</a></span> - Method in interface picard.util.<a href="picard/util/AdapterPair.html" title="interface in picard.util">AdapterPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IlluminaUtil.IlluminaAdapterPair.html#get3PrimeAdapter--">get3PrimeAdapter()</a></span> - Method in enum picard.util.<a href="picard/util/IlluminaUtil.IlluminaAdapterPair.html" title="enum in picard.util">IlluminaUtil.IlluminaAdapterPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AdapterPair.html#get3PrimeAdapterBytes--">get3PrimeAdapterBytes()</a></span> - Method in interface picard.util.<a href="picard/util/AdapterPair.html" title="interface in picard.util">AdapterPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IlluminaUtil.IlluminaAdapterPair.html#get3PrimeAdapterBytes--">get3PrimeAdapterBytes()</a></span> - Method in enum picard.util.<a href="picard/util/IlluminaUtil.IlluminaAdapterPair.html" title="enum in picard.util">IlluminaUtil.IlluminaAdapterPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AdapterPair.html#get3PrimeAdapterBytesInReadOrder--">get3PrimeAdapterBytesInReadOrder()</a></span> - Method in interface picard.util.<a href="picard/util/AdapterPair.html" title="interface in picard.util">AdapterPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IlluminaUtil.IlluminaAdapterPair.html#get3PrimeAdapterBytesInReadOrder--">get3PrimeAdapterBytesInReadOrder()</a></span> - Method in enum picard.util.<a href="picard/util/IlluminaUtil.IlluminaAdapterPair.html" title="enum in picard.util">IlluminaUtil.IlluminaAdapterPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AdapterPair.html#get3PrimeAdapterInReadOrder--">get3PrimeAdapterInReadOrder()</a></span> - Method in interface picard.util.<a href="picard/util/AdapterPair.html" title="interface in picard.util">AdapterPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IlluminaUtil.IlluminaAdapterPair.html#get3PrimeAdapterInReadOrder--">get3PrimeAdapterInReadOrder()</a></span> - Method in enum picard.util.<a href="picard/util/IlluminaUtil.IlluminaAdapterPair.html" title="enum in picard.util">IlluminaUtil.IlluminaAdapterPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AdapterPair.html#get5PrimeAdapter--">get5PrimeAdapter()</a></span> - Method in interface picard.util.<a href="picard/util/AdapterPair.html" title="interface in picard.util">AdapterPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IlluminaUtil.IlluminaAdapterPair.html#get5PrimeAdapter--">get5PrimeAdapter()</a></span> - Method in enum picard.util.<a href="picard/util/IlluminaUtil.IlluminaAdapterPair.html" title="enum in picard.util">IlluminaUtil.IlluminaAdapterPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AdapterPair.html#get5PrimeAdapterBytes--">get5PrimeAdapterBytes()</a></span> - Method in interface picard.util.<a href="picard/util/AdapterPair.html" title="interface in picard.util">AdapterPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IlluminaUtil.IlluminaAdapterPair.html#get5PrimeAdapterBytes--">get5PrimeAdapterBytes()</a></span> - Method in enum picard.util.<a href="picard/util/IlluminaUtil.IlluminaAdapterPair.html" title="enum in picard.util">IlluminaUtil.IlluminaAdapterPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AdapterPair.html#get5PrimeAdapterBytesInReadOrder--">get5PrimeAdapterBytesInReadOrder()</a></span> - Method in interface picard.util.<a href="picard/util/AdapterPair.html" title="interface in picard.util">AdapterPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IlluminaUtil.IlluminaAdapterPair.html#get5PrimeAdapterBytesInReadOrder--">get5PrimeAdapterBytesInReadOrder()</a></span> - Method in enum picard.util.<a href="picard/util/IlluminaUtil.IlluminaAdapterPair.html" title="enum in picard.util">IlluminaUtil.IlluminaAdapterPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AdapterPair.html#get5PrimeAdapterInReadOrder--">get5PrimeAdapterInReadOrder()</a></span> - Method in interface picard.util.<a href="picard/util/AdapterPair.html" title="interface in picard.util">AdapterPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IlluminaUtil.IlluminaAdapterPair.html#get5PrimeAdapterInReadOrder--">get5PrimeAdapterInReadOrder()</a></span> - Method in enum picard.util.<a href="picard/util/IlluminaUtil.IlluminaAdapterPair.html" title="enum in picard.util">IlluminaUtil.IlluminaAdapterPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/FourChannelIntensityData.html#getA--">getA()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/FourChannelIntensityData.html" title="class in picard.illumina.parser">FourChannelIntensityData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaFileUtil.html#getActualTiles-java.util.List-">getActualTiles(List&lt;IlluminaFileUtil.SupportedIlluminaFormat&gt;)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/IlluminaFileUtil.html" title="class in picard.illumina.parser">IlluminaFileUtil</a></dt>
<dd>
<div class="block">Get the available tiles for the given formats, if the formats have tile lists that differ then
 throw an exception, if any of the format</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/DiploidGenotype.html#getAllele1--">getAllele1()</a></span> - Method in enum picard.fingerprint.<a href="picard/fingerprint/DiploidGenotype.html" title="enum in picard.fingerprint">DiploidGenotype</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/Snp.html#getAllele1--">getAllele1()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/Snp.html" title="class in picard.fingerprint">Snp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/DiploidGenotype.html#getAllele2--">getAllele2()</a></span> - Method in enum picard.fingerprint.<a href="picard/fingerprint/DiploidGenotype.html" title="enum in picard.fingerprint">DiploidGenotype</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/Snp.html#getAllele2--">getAllele2()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/Snp.html" title="class in picard.fingerprint">Snp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/Snp.html#getAlleleString--">getAlleleString()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/Snp.html" title="class in picard.fingerprint">Snp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/metrics/MultiLevelCollector.html#getAllReadsCollector--">getAllReadsCollector()</a></span> - Method in class picard.metrics.<a href="picard/metrics/MultiLevelCollector.html" title="class in picard.metrics">MultiLevelCollector</a></dt>
<dd>
<div class="block">Get the PerUnitMetricCollector that collects reads for all levels</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeMap.html#getAllSnps--">getAllSnps()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeMap.html" title="class in picard.fingerprint">HaplotypeMap</a></dt>
<dd>
<div class="block">Returns an unmodifiable collection of all SNPs in all Haplotype blocks.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/CbclReader.html#getAllTiles--">getAllTiles()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/CbclReader.html" title="class in picard.illumina.parser.readers">CbclReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.html#getAttributesToReverse--">getAttributesToReverse()</a></span> - Method in class picard.sam.<a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</a></dt>
<dd>
<div class="block">Gets the set of attributes to be reversed on reads marked as negative strand.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.html#getAttributesToReverseComplement--">getAttributesToReverseComplement()</a></span> - Method in class picard.sam.<a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</a></dt>
<dd>
<div class="block">Gets the set of attributes to be reverse complemented on reads marked as negative strand.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaDataProviderFactory.html#getAvailableTiles--">getAvailableTiles()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/IlluminaDataProviderFactory.html" title="class in picard.illumina.parser">IlluminaDataProviderFactory</a></dt>
<dd>
<div class="block">Return the list of tiles available for this flowcell and lane.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.PerTileBarcodeExtractor.BarcodeMatch.html#getBarcode--">getBarcode()</a></span> - Method in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.PerTileBarcodeExtractor.BarcodeMatch.html" title="class in picard.illumina">ExtractIlluminaBarcodes.PerTileBarcodeExtractor.BarcodeMatch</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/EstimateLibraryComplexity.html#getBarcodeValue-htsjdk.samtools.SAMRecord-">getBarcodeValue(SAMRecord)</a></span> - Method in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/EstimateLibraryComplexity.html" title="class in picard.sam.markduplicates">EstimateLibraryComplexity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/BclData.html#getBases--">getBases()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/BclData.html" title="class in picard.illumina.parser">BclData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadData.html#getBases--">getBases()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ReadData.html" title="class in picard.illumina.parser">ReadData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.html#getBasesExcludedBy-picard.filter.CountingFilter-">getBasesExcludedBy(CountingFilter)</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics</a></dt>
<dd>
<div class="block">If INTERVALS is specified, this will count bases beyond the interval list when the read overlaps the intervals and extends beyond the
 edge.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/BclIndexReader.html#getBciFile--">getBciFile()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/BclIndexReader.html" title="class in picard.illumina.parser.readers">BclIndexReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/MMapBackedIteratorFactory.html#getByteBufferIterator-int-int-java.io.File-">getByteBufferIterator(int, int, File)</a></span> - Static method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/MMapBackedIteratorFactory.html" title="class in picard.illumina.parser.readers">MMapBackedIteratorFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/MMapBackedIteratorFactory.html#getByteIterator-int-java.io.File-">getByteIterator(int, File)</a></span> - Static method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/MMapBackedIteratorFactory.html" title="class in picard.illumina.parser.readers">MMapBackedIteratorFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/CircularByteBuffer.html#getBytesAvailableToRead--">getBytesAvailableToRead()</a></span> - Method in class picard.util.<a href="picard/util/CircularByteBuffer.html" title="class in picard.util">CircularByteBuffer</a></dt>
<dd>
<div class="block">Returns the number of bytes that are in the buffer at the time of the method invocation.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/FourChannelIntensityData.html#getC--">getC()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/FourChannelIntensityData.html" title="class in picard.illumina.parser">FourChannelIntensityData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/CircularByteBuffer.html#getCapacity--">getCapacity()</a></span> - Method in class picard.util.<a href="picard/util/CircularByteBuffer.html" title="class in picard.util">CircularByteBuffer</a></dt>
<dd>
<div class="block">Returns the total capacity of the buffer (empty+filled).</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html#getChainedPgIds-htsjdk.samtools.SAMFileHeader-">getChainedPgIds(SAMFileHeader)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html" title="class in picard.sam.markduplicates.util">AbstractMarkDuplicatesCommandLineProgram</a></dt>
<dd>
<div class="block">We have to re-chain the program groups based on this algorithm.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/FourChannelIntensityData.html#getChannel-picard.illumina.parser.IntensityChannel-">getChannel(IntensityChannel)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/FourChannelIntensityData.html" title="class in picard.illumina.parser">FourChannelIntensityData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/Snp.html#getChrom--">getChrom()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/Snp.html" title="class in picard.fingerprint">Snp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/ClassFinder.html#getClasses--">getClasses()</a></span> - Method in class picard.cmdline.<a href="picard/cmdline/ClassFinder.html" title="class in picard.cmdline">ClassFinder</a></dt>
<dd>
<div class="block">Fetches the set of classes discovered so far.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/Tile.html#getClusterCount--">getClusterCount()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/Tile.html" title="class in picard.illumina.parser">Tile</a></dt>
<dd>
<div class="block">Returns the number of on this tile.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/Tile.html#getClusterDensity--">getClusterDensity()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/Tile.html" title="class in picard.illumina.parser">Tile</a></dt>
<dd>
<div class="block">Returns the cluster density of this tile, in units of [cluster/mm^2].</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceStates.CallState.html#getCode--">getCode()</a></span> - Method in enum picard.vcf.<a href="picard/vcf/GenotypeConcordanceStates.CallState.html" title="enum in picard.vcf">GenotypeConcordanceStates.CallState</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceStates.TruthState.html#getCode--">getCode()</a></span> - Method in enum picard.vcf.<a href="picard/vcf/GenotypeConcordanceStates.TruthState.html" title="enum in picard.vcf">GenotypeConcordanceStates.TruthState</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.html#getCollector-int-htsjdk.samtools.util.IntervalList-">getCollector(int, IntervalList)</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics</a></dt>
<dd>
<div class="block">Creates <a href="picard/analysis/AbstractWgsMetricsCollector.html" title="class in picard.analysis"><code>AbstractWgsMetricsCollector</code></a> implementation according to <code>this#USE_FAST_ALGORITHM</code> value.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.html#getCollector-int-htsjdk.samtools.util.IntervalList-">getCollector(int, IntervalList)</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.html" title="class in picard.analysis">CollectWgsMetricsWithNonZeroCoverage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/DelimitedTextFileWithHeaderIterator.html#getColumnNames--">getColumnNames()</a></span> - Method in class picard.util.<a href="picard/util/DelimitedTextFileWithHeaderIterator.html" title="class in picard.util">DelimitedTextFileWithHeaderIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/TabbedTextFileWithHeaderParser.html#getColumnNames--">getColumnNames()</a></span> - Method in class picard.util.<a href="picard/util/TabbedTextFileWithHeaderParser.html" title="class in picard.util">TabbedTextFileWithHeaderParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#getCommandLine--">getCommandLine()</a></span> - Method in class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#getCommandLineParser--">getCommandLineParser()</a></span> - Method in class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/BaseBclReader.TileData.html#getCompressedBlockSize--">getCompressedBlockSize()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/BaseBclReader.TileData.html" title="class in picard.illumina.parser.readers">BaseBclReader.TileData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceScheme.html#getConcordanceStateArray-picard.vcf.GenotypeConcordanceStates.TruthState-picard.vcf.GenotypeConcordanceStates.CallState-">getConcordanceStateArray(GenotypeConcordanceStates.TruthState, GenotypeConcordanceStates.CallState)</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceScheme.html" title="class in picard.vcf">GenotypeConcordanceScheme</a></dt>
<dd>
<div class="block">Get the concordance state array associate with the given truth state and call state tuple.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceScheme.html#getConcordanceStateArray-picard.vcf.GenotypeConcordanceStates.TruthAndCallStates-">getConcordanceStateArray(GenotypeConcordanceStates.TruthAndCallStates)</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceScheme.html" title="class in picard.vcf">GenotypeConcordanceScheme</a></dt>
<dd>
<div class="block">Get the concordance state array associate with the given truth state and call state tuple.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceCounts.html#getContingencyStateCounts-picard.vcf.GenotypeConcordanceScheme-">getContingencyStateCounts(GenotypeConcordanceScheme)</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceCounts.html" title="class in picard.vcf">GenotypeConcordanceCounts</a></dt>
<dd>
<div class="block">Returns the total number of times each contingency state is encountered, summed across all truth/call state pairs.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceScheme.html#getContingencyStateSet-picard.vcf.GenotypeConcordanceStates.ContingencyState:A-">getContingencyStateSet(GenotypeConcordanceStates.ContingencyState[])</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceScheme.html" title="class in picard.vcf">GenotypeConcordanceScheme</a></dt>
<dd>
<div class="block">Get the contingency state array as a set</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceScheme.html#getContingencyStateString-picard.vcf.GenotypeConcordanceStates.TruthState-picard.vcf.GenotypeConcordanceStates.CallState-">getContingencyStateString(GenotypeConcordanceStates.TruthState, GenotypeConcordanceStates.CallState)</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceScheme.html" title="class in picard.vcf">GenotypeConcordanceScheme</a></dt>
<dd>
<div class="block">Get the contingency state array as a parse-able string</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceCounts.html#getCount-picard.vcf.GenotypeConcordanceStates.TruthState-picard.vcf.GenotypeConcordanceStates.CallState-">getCount(GenotypeConcordanceStates.TruthState, GenotypeConcordanceStates.CallState)</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceCounts.html" title="class in picard.vcf">GenotypeConcordanceCounts</a></dt>
<dd>
<div class="block">Returns the count defined by the truth state set and call state set.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceCounts.html#getCount-picard.vcf.GenotypeConcordanceStates.TruthAndCallStates-">getCount(GenotypeConcordanceStates.TruthAndCallStates)</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceCounts.html" title="class in picard.vcf">GenotypeConcordanceCounts</a></dt>
<dd>
<div class="block">Returns the count defined by the truth state set and call state set.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceCounts.html#getCounterSize--">getCounterSize()</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceCounts.html" title="class in picard.vcf">GenotypeConcordanceCounts</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetMetricsCollector.PerUnitTargetMetricCollector.html#getCoverageByTarget--">getCoverageByTarget()</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/TargetMetricsCollector.PerUnitTargetMetricCollector.html" title="class in picard.analysis.directed">TargetMetricsCollector.PerUnitTargetMetricCollector</a></dt>
<dd>
<div class="block">Returns the accumulated coverage per target.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/BasicInputParser.html#getCurrentLine--">getCurrentLine()</a></span> - Method in class picard.util.<a href="picard/util/BasicInputParser.html" title="class in picard.util">BasicInputParser</a></dt>
<dd>
<div class="block">Provides access to the current (just parsed) line in pre-parsed format.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/DelimitedTextFileWithHeaderIterator.Row.html#getCurrentLine--">getCurrentLine()</a></span> - Method in class picard.util.<a href="picard/util/DelimitedTextFileWithHeaderIterator.Row.html" title="class in picard.util">DelimitedTextFileWithHeaderIterator.Row</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/TabbedTextFileWithHeaderParser.Row.html#getCurrentLine--">getCurrentLine()</a></span> - Method in class picard.util.<a href="picard/util/TabbedTextFileWithHeaderParser.Row.html" title="class in picard.util">TabbedTextFileWithHeaderParser.Row</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/BasicInputParser.html#getCurrentLineNumber--">getCurrentLineNumber()</a></span> - Method in class picard.util.<a href="picard/util/BasicInputParser.html" title="class in picard.util">BasicInputParser</a></dt>
<dd>
<div class="block">NOTE: Because AbstractInputParser pre-fetches the next line, this method actually returns the
 next line, not the most recent line returned by next().</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/DelimitedTextFileWithHeaderIterator.html#getCurrentLineNumber--">getCurrentLineNumber()</a></span> - Method in class picard.util.<a href="picard/util/DelimitedTextFileWithHeaderIterator.html" title="class in picard.util">DelimitedTextFileWithHeaderIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/TabbedTextFileWithHeaderParser.html#getCurrentLineNumber--">getCurrentLineNumber()</a></span> - Method in class picard.util.<a href="picard/util/TabbedTextFileWithHeaderParser.html" title="class in picard.util">TabbedTextFileWithHeaderParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/CbclReader.html#getCycleData--">getCycleData()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/CbclReader.html" title="class in picard.illumina.parser.readers">CbclReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PerTilePerCycleFileUtil.html#getCycleFromDir-java.io.File-">getCycleFromDir(File)</a></span> - Static method in class picard.illumina.parser.<a href="picard/illumina/parser/PerTilePerCycleFileUtil.html" title="class in picard.illumina.parser">PerTilePerCycleFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/OutputMapping.html#getCycleIndexRanges--">getCycleIndexRanges()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/OutputMapping.html" title="class in picard.illumina.parser">OutputMapping</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadStructure.Substructure.html#getCycleIndexRanges--">getCycleIndexRanges()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ReadStructure.Substructure.html" title="class in picard.illumina.parser">ReadStructure.Substructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadStructure.Substructure.html#getCycles--">getCycles()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ReadStructure.Substructure.html" title="class in picard.illumina.parser">ReadStructure.Substructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#getDefaultHeaders--">getDefaultHeaders()</a></span> - Method in class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/TileMetricsOutReader.html#getDensity--">getDensity()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/TileMetricsOutReader.html" title="class in picard.illumina.parser.readers">TileMetricsOutReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverageCollector.html#getDepthHistogram--">getDepthHistogram()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverageCollector.html" title="class in picard.analysis">CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverageCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetMetricsCollector.Coverage.html#getDepths--">getDepths()</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/TargetMetricsCollector.Coverage.html" title="class in picard.analysis.directed">TargetMetricsCollector.Coverage</a></dt>
<dd>
<div class="block">Gets the coverage depths as an array of ints.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/programgroups/Alpha.html#getDescription--">getDescription()</a></span> - Method in class picard.cmdline.programgroups.<a href="picard/cmdline/programgroups/Alpha.html" title="class in picard.cmdline.programgroups">Alpha</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/programgroups/Fasta.html#getDescription--">getDescription()</a></span> - Method in class picard.cmdline.programgroups.<a href="picard/cmdline/programgroups/Fasta.html" title="class in picard.cmdline.programgroups">Fasta</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/programgroups/Fingerprinting.html#getDescription--">getDescription()</a></span> - Method in class picard.cmdline.programgroups.<a href="picard/cmdline/programgroups/Fingerprinting.html" title="class in picard.cmdline.programgroups">Fingerprinting</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/programgroups/Illumina.html#getDescription--">getDescription()</a></span> - Method in class picard.cmdline.programgroups.<a href="picard/cmdline/programgroups/Illumina.html" title="class in picard.cmdline.programgroups">Illumina</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/programgroups/Intervals.html#getDescription--">getDescription()</a></span> - Method in class picard.cmdline.programgroups.<a href="picard/cmdline/programgroups/Intervals.html" title="class in picard.cmdline.programgroups">Intervals</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/programgroups/Metrics.html#getDescription--">getDescription()</a></span> - Method in class picard.cmdline.programgroups.<a href="picard/cmdline/programgroups/Metrics.html" title="class in picard.cmdline.programgroups">Metrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/programgroups/None.html#getDescription--">getDescription()</a></span> - Method in class picard.cmdline.programgroups.<a href="picard/cmdline/programgroups/None.html" title="class in picard.cmdline.programgroups">None</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/programgroups/SamOrBam.html#getDescription--">getDescription()</a></span> - Method in class picard.cmdline.programgroups.<a href="picard/cmdline/programgroups/SamOrBam.html" title="class in picard.cmdline.programgroups">SamOrBam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/programgroups/Testing.html#getDescription--">getDescription()</a></span> - Method in class picard.cmdline.programgroups.<a href="picard/cmdline/programgroups/Testing.html" title="class in picard.cmdline.programgroups">Testing</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/programgroups/VcfOrBcf.html#getDescription--">getDescription()</a></span> - Method in class picard.cmdline.programgroups.<a href="picard/cmdline/programgroups/VcfOrBcf.html" title="class in picard.cmdline.programgroups">VcfOrBcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadStructure.Substructure.html#getDescriptorLengths--">getDescriptorLengths()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ReadStructure.Substructure.html" title="class in picard.illumina.parser">ReadStructure.Substructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PerTilePerCycleFileUtil.html#getDetectedCycles--">getDetectedCycles()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/PerTilePerCycleFileUtil.html" title="class in picard.illumina.parser">PerTilePerCycleFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.html#getDictionaryForMergedBam--">getDictionaryForMergedBam()</a></span> - Method in class picard.sam.<a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamAlignmentMerger.html#getDictionaryForMergedBam--">getDictionaryForMergedBam()</a></span> - Method in class picard.sam.<a href="picard/sam/SamAlignmentMerger.html" title="class in picard.sam">SamAlignmentMerger</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeBlock.html#getDiploidHaplotype-picard.fingerprint.Snp-picard.fingerprint.DiploidGenotype-">getDiploidHaplotype(Snp, DiploidGenotype)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeBlock.html" title="class in picard.fingerprint">HaplotypeBlock</a></dt>
<dd>
<div class="block">Gets the diploid haplotype for this haplotype block given the provided SNP and SNP
 genotype.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/SimpleMarkDuplicatesWithMateCigar.html#getDuplicateSetIterator-picard.sam.markduplicates.util.AbstractMarkDuplicatesCommandLineProgram.SamHeaderAndIterator-htsjdk.samtools.SAMRecordDuplicateComparator-">getDuplicateSetIterator(AbstractMarkDuplicatesCommandLineProgram.SamHeaderAndIterator, SAMRecordDuplicateComparator)</a></span> - Method in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/SimpleMarkDuplicatesWithMateCigar.html" title="class in picard.sam.markduplicates">SimpleMarkDuplicatesWithMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/UmiAwareMarkDuplicatesWithMateCigar.html#getDuplicateSetIterator-picard.sam.markduplicates.util.AbstractMarkDuplicatesCommandLineProgram.SamHeaderAndIterator-htsjdk.samtools.SAMRecordDuplicateComparator-">getDuplicateSetIterator(AbstractMarkDuplicatesCommandLineProgram.SamHeaderAndIterator, SAMRecordDuplicateComparator)</a></span> - Method in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/UmiAwareMarkDuplicatesWithMateCigar.html" title="class in picard.sam.markduplicates">UmiAwareMarkDuplicatesWithMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.PerTileBarcodeExtractor.html#getException--">getException()</a></span> - Method in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.PerTileBarcodeExtractor.html" title="class in picard.illumina">ExtractIlluminaBarcodes.PerTileBarcodeExtractor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaFileUtil.html#getExpectedTiles--">getExpectedTiles()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/IlluminaFileUtil.html" title="class in picard.illumina.parser">IlluminaFileUtil</a></dt>
<dd>
<div class="block">Return the list of tiles we would expect for this lane based on the metrics found in InterOp/TileMetricsOut.bin</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaLaneMetrics.html#getExtension--">getExtension()</a></span> - Static method in class picard.illumina.<a href="picard/illumina/IlluminaLaneMetrics.html" title="class in picard.illumina">IlluminaLaneMetrics</a></dt>
<dd>
<div class="block">This property is not exposed in a field to avoid complications with MetricBase's dependency on reflection.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaPhasingMetrics.html#getExtension--">getExtension()</a></span> - Static method in class picard.illumina.<a href="picard/illumina/IlluminaPhasingMetrics.html" title="class in picard.illumina">IlluminaPhasingMetrics</a></dt>
<dd>
<div class="block">This property is not exposed in a field to avoid complications with MetricBase's dependency on reflection.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html#getExtension--">getExtension()</a></span> - Static method in class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html" title="class in picard.vcf.MendelianViolations">MendelianViolationMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/pedigree/PedFile.PedTrio.html#getFamilyId--">getFamilyId()</a></span> - Method in class picard.pedigree.<a href="picard/pedigree/PedFile.PedTrio.html" title="class in picard.pedigree">PedFile.PedTrio</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#getFaqLink--">getFaqLink()</a></span> - Static method in class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/DelimitedTextFileWithHeaderIterator.Row.html#getField-java.lang.String-">getField(String)</a></span> - Method in class picard.util.<a href="picard/util/DelimitedTextFileWithHeaderIterator.Row.html" title="class in picard.util">DelimitedTextFileWithHeaderIterator.Row</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/TabbedTextFileWithHeaderParser.Row.html#getField-java.lang.String-">getField(String)</a></span> - Method in class picard.util.<a href="picard/util/TabbedTextFileWithHeaderParser.Row.html" title="class in picard.util">TabbedTextFileWithHeaderParser.Row</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/DelimitedTextFileWithHeaderIterator.Row.html#getFields--">getFields()</a></span> - Method in class picard.util.<a href="picard/util/DelimitedTextFileWithHeaderIterator.Row.html" title="class in picard.util">DelimitedTextFileWithHeaderIterator.Row</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/TabbedTextFileWithHeaderParser.Row.html#getFields--">getFields()</a></span> - Method in class picard.util.<a href="picard/util/TabbedTextFileWithHeaderParser.Row.html" title="class in picard.util">TabbedTextFileWithHeaderParser.Row</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.html#getFile--">getFile()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.html" title="class in picard.illumina.parser.readers">AbstractIlluminaPositionFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/TileIndex.html#getFile--">getFile()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/TileIndex.html" title="class in picard.illumina.parser">TileIndex</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingDetailMetrics.html#getFileExtension--">getFileExtension()</a></span> - Static method in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingDetailMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingDetailMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html#getFileExtension--">getFileExtension()</a></span> - Static method in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingSummaryMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AbstractInputParser.html#getFileName--">getFileName()</a></span> - Method in class picard.util.<a href="picard/util/AbstractInputParser.html" title="class in picard.util">AbstractInputParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/BasicInputParser.html#getFileName--">getFileName()</a></span> - Method in class picard.util.<a href="picard/util/BasicInputParser.html" title="class in picard.util">BasicInputParser</a></dt>
<dd>
<div class="block">Gets the name of the file being parsed</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/MultiTileBclFileUtil.html#getFiles-java.util.List-int:A-">getFiles(List&lt;Integer&gt;, int[])</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/MultiTileBclFileUtil.html" title="class in picard.illumina.parser">MultiTileBclFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PerTileFileUtil.html#getFiles--">getFiles()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/PerTileFileUtil.html" title="class in picard.illumina.parser">PerTileFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PerTileFileUtil.html#getFiles-java.util.List-">getFiles(List&lt;Integer&gt;)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/PerTileFileUtil.html" title="class in picard.illumina.parser">PerTileFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PerTilePerCycleFileUtil.html#getFiles--">getFiles()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/PerTilePerCycleFileUtil.html" title="class in picard.illumina.parser">PerTilePerCycleFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PerTilePerCycleFileUtil.html#getFiles-java.util.List-">getFiles(List&lt;Integer&gt;)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/PerTilePerCycleFileUtil.html" title="class in picard.illumina.parser">PerTilePerCycleFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PerTilePerCycleFileUtil.html#getFiles-int:A-">getFiles(int[])</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/PerTilePerCycleFileUtil.html" title="class in picard.illumina.parser">PerTilePerCycleFileUtil</a></dt>
<dd>
<div class="block">Returns a cycleIlluminaFileMap with all available tiles but limited to the cycles passed in.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PerTilePerCycleFileUtil.html#getFiles-java.util.List-int:A-">getFiles(List&lt;Integer&gt;, int[])</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/PerTilePerCycleFileUtil.html" title="class in picard.illumina.parser">PerTilePerCycleFileUtil</a></dt>
<dd>
<div class="block">Returns a cycleIlluminaFileMap that contains only the tiles and cycles specified (and fewer if the original CycleIlluminaFileMap, created
 on util instantiation, doesn't contain any of these tiles/cycles).</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/CbclReader.html#getFilesForCycle-int-">getFilesForCycle(int)</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/CbclReader.html" title="class in picard.illumina.parser.readers">CbclReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/filter/CountingFilter.html#getFilteredBases--">getFilteredBases()</a></span> - Method in class picard.filter.<a href="picard/filter/CountingFilter.html" title="class in picard.filter">CountingFilter</a></dt>
<dd>
<div class="block">Gets the number of bases that have been filtered out thus far.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/filter/CountingFilter.html#getFilteredRecords--">getFilteredRecords()</a></span> - Method in class picard.filter.<a href="picard/filter/CountingFilter.html" title="class in picard.filter">CountingFilter</a></dt>
<dd>
<div class="block">Gets the number of records that have been filtered out thus far.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/MatchResults.html#getFingerprintFile--">getFingerprintFile()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/MatchResults.html" title="class in picard.fingerprint">MatchResults</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckFingerprints.html#getFingerprintIdDetailsStringFunction-picard.fingerprint.CrosscheckMetric.DataType-">getFingerprintIdDetailsStringFunction(CrosscheckMetric.DataType)</a></span> - Static method in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckFingerprints.html" title="class in picard.fingerprint">CrosscheckFingerprints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/Snp.html#getFingerprintPanels--">getFingerprintPanels()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/Snp.html" title="class in picard.fingerprint">Snp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeBlock.html#getFirstSnp--">getFirstSnp()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeBlock.html" title="class in picard.fingerprint">HaplotypeBlock</a></dt>
<dd>
<div class="block">Gets the arbitrarily first SNP in the haplotype.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/MMapBackedIteratorFactory.html#getFloatIterator-int-java.io.File-">getFloatIterator(int, File)</a></span> - Static method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/MMapBackedIteratorFactory.html" title="class in picard.illumina.parser.readers">MMapBackedIteratorFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamAlignmentMerger.html#getForceSort--">getForceSort()</a></span> - Method in class picard.sam.<a href="picard/sam/SamAlignmentMerger.html" title="class in picard.sam">SamAlignmentMerger</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilitiesFromSequence.html#getFractionUnexpectedAlleleObs--">getFractionUnexpectedAlleleObs()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilitiesFromSequence.html" title="class in picard.fingerprint">HaplotypeProbabilitiesFromSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/FourChannelIntensityData.html#getG--">getG()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/FourChannelIntensityData.html" title="class in picard.illumina.parser">FourChannelIntensityData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/Gene.Transcript.html#getGene--">getGene()</a></span> - Method in class picard.annotation.<a href="picard/annotation/Gene.Transcript.html" title="class in picard.annotation">Gene.Transcript</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/help/PicardHelpDoclet.html#getGroupMap-org.broadinstitute.barclay.help.DocWorkUnit-">getGroupMap(DocWorkUnit)</a></span> - Method in class picard.util.help.<a href="picard/util/help/PicardHelpDoclet.html" title="class in picard.util.help">PicardHelpDoclet</a></dt>
<dd>
<div class="block">Adds a super-category so that we can custom-order the categories in the doc index</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeMap.html#getHaplotype-picard.fingerprint.Snp-">getHaplotype(Snp)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeMap.html" title="class in picard.fingerprint">HaplotypeMap</a></dt>
<dd>
<div class="block">Queries a HaplotypeBlock by Snp object.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeMap.html#getHaplotype-java.lang.String-">getHaplotype(String)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeMap.html" title="class in picard.fingerprint">HaplotypeMap</a></dt>
<dd>
<div class="block">Queries a HaplotypeBlock by Snp name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeMap.html#getHaplotype-java.lang.String-int-">getHaplotype(String, int)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeMap.html" title="class in picard.fingerprint">HaplotypeMap</a></dt>
<dd>
<div class="block">Queries a HaplotypeBlock by Snp chromosome and position.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilities.html#getHaplotype--">getHaplotype()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilities.html" title="class in picard.fingerprint">HaplotypeProbabilities</a></dt>
<dd>
<div class="block">Returns the haplotype for which the probabilities apply.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeBlock.html#getHaplotypeFrequencies--">getHaplotypeFrequencies()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeBlock.html" title="class in picard.fingerprint">HaplotypeBlock</a></dt>
<dd>
<div class="block">Gets the set of haplotype frequencies.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeBlock.html#getHaplotypeFrequency-int-">getHaplotypeFrequency(int)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeBlock.html" title="class in picard.fingerprint">HaplotypeBlock</a></dt>
<dd>
<div class="block">Gets the frequency of the i'th diploid haplotype where haplotypes are ordered accorinding
 to DiploidHaplotype.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeMap.html#getHaplotypes--">getHaplotypes()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeMap.html" title="class in picard.fingerprint">HaplotypeMap</a></dt>
<dd>
<div class="block">Returns an unmodifiable collection of all the haplotype blocks in the map.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintChecker.html#getHeader--">getHeader()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/FingerprintChecker.html" title="class in picard.fingerprint">FingerprintChecker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeMap.html#getHeader--">getHeader()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeMap.html" title="class in picard.fingerprint">HaplotypeMap</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.html#getHeader--">getHeader()</a></span> - Method in class picard.sam.<a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/CbclReader.html#getHeaderSize--">getHeaderSize()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/CbclReader.html" title="class in picard.illumina.parser.readers">CbclReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FilterSamReads.Filter.html#getHelpDoc--">getHelpDoc()</a></span> - Method in enum picard.sam.<a href="picard/sam/FilterSamReads.Filter.html" title="enum in picard.sam">FilterSamReads.Filter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IntervalListTools.Action.html#getHelpDoc--">getHelpDoc()</a></span> - Method in enum picard.util.<a href="picard/util/IntervalListTools.Action.html" title="enum in picard.util">IntervalListTools.Action</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceStates.CallState.html#getHet-int-int-">getHet(int, int)</a></span> - Static method in enum picard.vcf.<a href="picard/vcf/GenotypeConcordanceStates.CallState.html" title="enum in picard.vcf">GenotypeConcordanceStates.CallState</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/Snp.html#getHeterogyzousGenotype--">getHeterogyzousGenotype()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/Snp.html" title="class in picard.fingerprint">Snp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AbstractWgsMetricsCollector.html#getHighQualityDepthHistogram--">getHighQualityDepthHistogram()</a></span> - Method in class picard.analysis.<a href="picard/analysis/AbstractWgsMetricsCollector.html" title="class in picard.analysis">AbstractWgsMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AbstractWgsMetricsCollector.html#getHistogram-long:A-java.lang.String-java.lang.String-">getHistogram(long[], String, String)</a></span> - Method in class picard.analysis.<a href="picard/analysis/AbstractWgsMetricsCollector.html" title="class in picard.analysis">AbstractWgsMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceStates.CallState.html#getHom-int-">getHom(int)</a></span> - Static method in enum picard.vcf.<a href="picard/vcf/GenotypeConcordanceStates.CallState.html" title="enum in picard.vcf">GenotypeConcordanceStates.CallState</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceStates.TruthState.html#getHom-int-">getHom(int)</a></span> - Static method in enum picard.vcf.<a href="picard/vcf/GenotypeConcordanceStates.TruthState.html" title="enum in picard.vcf">GenotypeConcordanceStates.TruthState</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/Snp.html#getHomozygousAllele1Genotype--">getHomozygousAllele1Genotype()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/Snp.html" title="class in picard.fingerprint">Snp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/Snp.html#getHomozygousAllele2Genotype--">getHomozygousAllele2Genotype()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/Snp.html" title="class in picard.fingerprint">Snp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#getIndelCounter--">getIndelCounter()</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/help/PicardHelpDoclet.html#getIndexTemplateName--">getIndexTemplateName()</a></span> - Method in class picard.util.help.<a href="picard/util/help/PicardHelpDoclet.html" title="class in picard.util.help">PicardHelpDoclet</a></dt>
<dd>
<div class="block">Return the name of the freemarker template to be used for the index generated by Barclay.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadStructure.Substructure.html#getIndices--">getIndices()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ReadStructure.Substructure.html" title="class in picard.illumina.parser">ReadStructure.Substructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/pedigree/PedFile.PedTrio.html#getIndividualId--">getIndividualId()</a></span> - Method in class picard.pedigree.<a href="picard/pedigree/PedFile.PedTrio.html" title="class in picard.pedigree">PedFile.PedTrio</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/Fingerprint.html#getInfo--">getInfo()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/Fingerprint.html" title="class in picard.fingerprint">Fingerprint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintResults.html#getInputFile--">getInputFile()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/FingerprintResults.html" title="class in picard.fingerprint">FingerprintResults</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesCodec.html#getInputStream--">getInputStream()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesCodec.html" title="class in picard.sam.markduplicates.util">ReadEndsForMarkDuplicatesCodec</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/RepresentativeReadIndexerCodec.html#getInputStream--">getInputStream()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/RepresentativeReadIndexerCodec.html" title="class in picard.sam.markduplicates.util">RepresentativeReadIndexerCodec</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/DelimitedTextFileWithHeaderIterator.Row.html#getIntegerField-java.lang.String-">getIntegerField(String)</a></span> - Method in class picard.util.<a href="picard/util/DelimitedTextFileWithHeaderIterator.Row.html" title="class in picard.util">DelimitedTextFileWithHeaderIterator.Row</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/TabbedTextFileWithHeaderParser.Row.html#getIntegerField-java.lang.String-">getIntegerField(String)</a></span> - Method in class picard.util.<a href="picard/util/TabbedTextFileWithHeaderParser.Row.html" title="class in picard.util">TabbedTextFileWithHeaderParser.Row</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/MMapBackedIteratorFactory.html#getIntegerIterator-int-java.io.File-">getIntegerIterator(int, File)</a></span> - Static method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/MMapBackedIteratorFactory.html" title="class in picard.illumina.parser.readers">MMapBackedIteratorFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.CollectWgsMetricsIntervalArgumentCollection.html#getIntervalFile--">getIntervalFile()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.CollectWgsMetricsIntervalArgumentCollection.html" title="class in picard.analysis">CollectWgsMetrics.CollectWgsMetricsIntervalArgumentCollection</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/argumentcollections/IntervalArgumentCollection.html#getIntervalFile--">getIntervalFile()</a></span> - Method in interface picard.cmdline.argumentcollections.<a href="picard/cmdline/argumentcollections/IntervalArgumentCollection.html" title="interface in picard.cmdline.argumentcollections">IntervalArgumentCollection</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeMap.html#getIntervalList--">getIntervalList()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeMap.html" title="class in picard.fingerprint">HaplotypeMap</a></dt>
<dd>
<div class="block">Returns an IntervalList with an entry for every SNP in every Haplotype in the map.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.html#getIntervalsToExamine--">getIntervalsToExamine()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics</a></dt>
<dd>
<div class="block">Gets the intervals over which we will calculate metrics.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ClusterData.html#getLane--">getLane()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ClusterData.html" title="class in picard.illumina.parser">ClusterData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaFileUtil.html#getLane--">getLane()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/IlluminaFileUtil.html" title="class in picard.illumina.parser">IlluminaFileUtil</a></dt>
<dd>
<div class="block">Return the lane we're inspecting</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.html#getLane--">getLane()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.html" title="class in picard.illumina.parser.readers">AbstractIlluminaPositionFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaLaneTileCode.html#getLaneNumber--">getLaneNumber()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaLaneTileCode.html" title="class in picard.illumina.parser.readers">TileMetricsOutReader.IlluminaLaneTileCode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaTileMetrics.html#getLaneNumber--">getLaneNumber()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaTileMetrics.html" title="class in picard.illumina.parser.readers">TileMetricsOutReader.IlluminaTileMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/Tile.html#getLaneNumber--">getLaneNumber()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/Tile.html" title="class in picard.illumina.parser">Tile</a></dt>
<dd>
<div class="block">Returns the number of this tile's parent lane.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaTileMetrics.html#getLaneTileCode--">getLaneTileCode()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaTileMetrics.html" title="class in picard.illumina.parser.readers">TileMetricsOutReader.IlluminaTileMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/ReadNameParser.html#getLastThreeFields-java.lang.String-char-int:A-">getLastThreeFields(String, char, int[])</a></span> - Static method in class picard.sam.util.<a href="picard/sam/util/ReadNameParser.html" title="class in picard.sam.util">ReadNameParser</a></dt>
<dd>
<div class="block">Given a string, splits the string by the delimiter, and returns the the last three fields parsed as integers.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/LibraryIdGenerator.html#getLibraryId-htsjdk.samtools.SAMRecord-">getLibraryId(SAMRecord)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/LibraryIdGenerator.html" title="class in picard.sam.markduplicates.util">LibraryIdGenerator</a></dt>
<dd>
<div class="block">Get the library ID for the given SAM record.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/PhysicalLocationForMateCigar.html#getLibraryId--">getLibraryId()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/PhysicalLocationForMateCigar.html" title="class in picard.sam.markduplicates.util">PhysicalLocationForMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEnds.html#getLibraryId--">getLibraryId()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEnds.html" title="class in picard.sam.markduplicates.util">ReadEnds</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/PhysicalLocation.html#getLibraryId--">getLibraryId()</a></span> - Method in interface picard.sam.util.<a href="picard/sam/util/PhysicalLocation.html" title="interface in picard.sam.util">PhysicalLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/PhysicalLocationInt.html#getLibraryId--">getLibraryId()</a></span> - Method in class picard.sam.util.<a href="picard/sam/util/PhysicalLocationInt.html" title="class in picard.sam.util">PhysicalLocationInt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigarIterator.html#getLibraryIdGenerator--">getLibraryIdGenerator()</a></span> - Method in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigarIterator.html" title="class in picard.sam.markduplicates">MarkDuplicatesWithMateCigarIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/LibraryIdGenerator.html#getLibraryIdsMap--">getLibraryIdsMap()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/LibraryIdGenerator.html" title="class in picard.sam.markduplicates.util">LibraryIdGenerator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/LibraryIdGenerator.html#getLibraryName-htsjdk.samtools.SAMFileHeader-htsjdk.samtools.SAMRecord-">getLibraryName(SAMFileHeader, SAMRecord)</a></span> - Static method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/LibraryIdGenerator.html" title="class in picard.sam.markduplicates.util">LibraryIdGenerator</a></dt>
<dd>
<div class="block">Gets the library name from the header for the record.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilities.html#getLikelihoods--">getLikelihoods()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilities.html" title="class in picard.fingerprint">HaplotypeProbabilities</a></dt>
<dd>
<div class="block">Returns the likelihoods, in order, of the AA, Aa and aa haplotypes given the evidence

 Mathematically this is P(evidence | haplotype) where haplotype={AA,Aa,aa}.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilitiesFromGenotype.html#getLikelihoods--">getLikelihoods()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilitiesFromGenotype.html" title="class in picard.fingerprint">HaplotypeProbabilitiesFromGenotype</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilityOfNormalGivenTumor.html#getLikelihoods--">getLikelihoods()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilityOfNormalGivenTumor.html" title="class in picard.fingerprint">HaplotypeProbabilityOfNormalGivenTumor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/MarkQueue.html#getLocations-picard.sam.markduplicates.util.ReadEndsForMateCigar-">getLocations(ReadEndsForMateCigar)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/MarkQueue.html" title="class in picard.sam.markduplicates.util">MarkQueue</a></dt>
<dd>
<div class="block">Returns the nonDuplicateReadEndsSet of read ends that should be considered for tracking optical duplicates.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMateCigar.html#getLocationSet--">getLocationSet()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMateCigar.html" title="class in picard.sam.markduplicates.util">ReadEndsForMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintChecker.html#getLociToGenotype-java.util.Collection-">getLociToGenotype(Collection&lt;Fingerprint&gt;)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/FingerprintChecker.html" title="class in picard.fingerprint">FingerprintChecker</a></dt>
<dd>
<div class="block">Takes a set of fingerprints and returns an IntervalList containing all the loci that
 can be productively examined in sequencing data to compare to one or more of the
 fingerprints.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.html#getLocusIterator-htsjdk.samtools.SamReader-">getLocusIterator(SamReader)</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics</a></dt>
<dd>
<div class="block">Creates <code>AbstractLocusIterator</code> implementation according to <code>this#USE_FAST_ALGORITHM</code> value.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/MatchResults.html#getLocusResults--">getLocusResults()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/MatchResults.html" title="class in picard.fingerprint">MatchResults</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/MatchResults.html#getLOD--">getLOD()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/MatchResults.html" title="class in picard.fingerprint">MatchResults</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilities.html#getLodMostProbableGenotype--">getLodMostProbableGenotype()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilities.html" title="class in picard.fingerprint">HaplotypeProbabilities</a></dt>
<dd>
<div class="block">Returns the LOD score between the most probable haplotype and the second most probable.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/MatchResults.html#getLodNT--">getLodNT()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/MatchResults.html" title="class in picard.fingerprint">MatchResults</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/MatchResults.html#getLodTN--">getLodTN()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/MatchResults.html" title="class in picard.fingerprint">MatchResults</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.LogMath.html#getLog_of_base--">getLog_of_base()</a></span> - Method in class picard.util.<a href="picard/util/MathUtil.LogMath.html" title="class in picard.util">MathUtil.LogMath</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilities.html#getLogLikelihoods--">getLogLikelihoods()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilities.html" title="class in picard.fingerprint">HaplotypeProbabilities</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilitiesFromContaminatorSequence.html#getLogLikelihoods--">getLogLikelihoods()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilitiesFromContaminatorSequence.html" title="class in picard.fingerprint">HaplotypeProbabilitiesFromContaminatorSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.LogMath.html#getLogValue-double-">getLogValue(double)</a></span> - Method in class picard.util.<a href="picard/util/MathUtil.LogMath.html" title="class in picard.util">MathUtil.LogMath</a></dt>
<dd>
<div class="block">Returns the log-representation of the provided decimal value.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.LogMath.html#getLogValue-double:A-">getLogValue(double[])</a></span> - Method in class picard.util.<a href="picard/util/MathUtil.LogMath.html" title="class in picard.util">MathUtil.LogMath</a></dt>
<dd>
<div class="block">Returns the log-representation of the provided decimal array.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/MMapBackedIteratorFactory.html#getLongIterator-int-java.io.File-">getLongIterator(int, File)</a></span> - Static method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/MMapBackedIteratorFactory.html" title="class in picard.illumina.parser.readers">MMapBackedIteratorFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeBlock.html#getMaf--">getMaf()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeBlock.html" title="class in picard.fingerprint">HaplotypeBlock</a></dt>
<dd>
<div class="block">Returns the minor allele frequency of this haplotype.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/Snp.html#getMaf--">getMaf()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/Snp.html" title="class in picard.fingerprint">Snp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CompareSAMs.html#getMappingsDiffer--">getMappingsDiffer()</a></span> - Method in class picard.sam.<a href="picard/sam/CompareSAMs.html" title="class in picard.sam">CompareSAMs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CompareSAMs.html#getMappingsMatch--">getMappingsMatch()</a></span> - Method in class picard.sam.<a href="picard/sam/CompareSAMs.html" title="class in picard.sam">CompareSAMs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/BaitDesigner.html#getMaskedBaseCount-byte:A-int-int-">getMaskedBaseCount(byte[], int, int)</a></span> - Static method in class picard.util.<a href="picard/util/BaitDesigner.html" title="class in picard.util">BaitDesigner</a></dt>
<dd>
<div class="block">Returns the total of soft or hard masked bases in the interval of bases.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ClusterData.html#getMatchedBarcode--">getMatchedBarcode()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ClusterData.html" title="class in picard.illumina.parser">ClusterData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintResults.html#getMatchResults--">getMatchResults()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/FingerprintResults.html" title="class in picard.fingerprint">FingerprintResults</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/pedigree/PedFile.PedTrio.html#getMaternalId--">getMaternalId()</a></span> - Method in class picard.pedigree.<a href="picard/pedigree/PedFile.PedTrio.html" title="class in picard.pedigree">PedFile.PedTrio</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AdapterMarker.html#getMaxPairErrorRate--">getMaxPairErrorRate()</a></span> - Method in class picard.util.<a href="picard/util/AdapterMarker.html" title="class in picard.util">AdapterMarker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AdapterMarker.html#getMaxSingleEndErrorRate--">getMaxSingleEndErrorRate()</a></span> - Method in class picard.util.<a href="picard/util/AdapterMarker.html" title="class in picard.util">AdapterMarker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/LanePhasingMetricsCollector.html#getMedianPhasingMap--">getMedianPhasingMap()</a></span> - Method in class picard.illumina.<a href="picard/illumina/LanePhasingMetricsCollector.html" title="class in picard.illumina">LanePhasingMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/LanePhasingMetricsCollector.html#getMedianPrePhasingMap--">getMedianPrePhasingMap()</a></span> - Method in class picard.illumina.<a href="picard/illumina/LanePhasingMetricsCollector.html" title="class in picard.illumina">LanePhasingMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertOriginalBaseQualitiesAndAddMateCigar.CanSkipSamFile.html#getMessage-int-">getMessage(int)</a></span> - Method in enum picard.sam.<a href="picard/sam/RevertOriginalBaseQualitiesAndAddMateCigar.CanSkipSamFile.html" title="enum in picard.sam">RevertOriginalBaseQualitiesAndAddMateCigar.CanSkipSamFile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaLaneTileCode.html#getMetricCode--">getMetricCode()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaLaneTileCode.html" title="class in picard.illumina.parser.readers">TileMetricsOutReader.IlluminaLaneTileCode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaTileMetrics.html#getMetricCode--">getMetricCode()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaTileMetrics.html" title="class in picard.illumina.parser.readers">TileMetricsOutReader.IlluminaTileMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AbstractWgsMetricsCollector.html#getMetrics-picard.filter.CountingFilter-picard.filter.CountingFilter-picard.filter.CountingPairedFilter-">getMetrics(CountingFilter, CountingFilter, CountingPairedFilter)</a></span> - Method in class picard.analysis.<a href="picard/analysis/AbstractWgsMetricsCollector.html" title="class in picard.analysis">AbstractWgsMetricsCollector</a></dt>
<dd>
<div class="block">Creates CollectWgsMetrics.WgsMetrics - the object holding the result of CollectWgsMetrics</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.PerTileBarcodeExtractor.html#getMetrics--">getMetrics()</a></span> - Method in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.PerTileBarcodeExtractor.html" title="class in picard.illumina">ExtractIlluminaBarcodes.PerTileBarcodeExtractor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/LibraryIdGenerator.html#getMetricsByLibrary-java.lang.String-">getMetricsByLibrary(String)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/LibraryIdGenerator.html" title="class in picard.sam.markduplicates.util">LibraryIdGenerator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/LibraryIdGenerator.html#getMetricsByLibraryMap--">getMetricsByLibraryMap()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/LibraryIdGenerator.html" title="class in picard.sam.markduplicates.util">LibraryIdGenerator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaMetricsCode.html#getMetricsCode--">getMetricsCode()</a></span> - Method in enum picard.illumina.parser.<a href="picard/illumina/parser/IlluminaMetricsCode.html" title="enum in picard.illumina.parser">IlluminaMetricsCode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#getMetricsFile--">getMetricsFile()</a></span> - Method in class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>
<div class="block">Gets a MetricsFile with default headers already written into it.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaTileMetrics.html#getMetricValue--">getMetricValue()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaTileMetrics.html" title="class in picard.illumina.parser.readers">TileMetricsOutReader.IlluminaTileMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AdapterMarker.html#getMinPairMatchBases--">getMinPairMatchBases()</a></span> - Method in class picard.util.<a href="picard/util/AdapterMarker.html" title="class in picard.util">AdapterMarker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AdapterMarker.html#getMinSingleEndMatchBases--">getMinSingleEndMatchBases()</a></span> - Method in class picard.util.<a href="picard/util/AdapterMarker.html" title="class in picard.util">AdapterMarker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CompareSAMs.html#getMissingLeft--">getMissingLeft()</a></span> - Method in class picard.sam.<a href="picard/sam/CompareSAMs.html" title="class in picard.sam">CompareSAMs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CompareSAMs.html#getMissingRight--">getMissingRight()</a></span> - Method in class picard.sam.<a href="picard/sam/CompareSAMs.html" title="class in picard.sam">CompareSAMs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilities.html#getMostLikelyGenotype-picard.fingerprint.Snp-">getMostLikelyGenotype(Snp)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilities.html" title="class in picard.fingerprint">HaplotypeProbabilities</a></dt>
<dd>
<div class="block">Gets the genotype for this Snp given the most likely haplotype.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilities.html#getMostLikelyHaplotype--">getMostLikelyHaplotype()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilities.html" title="class in picard.fingerprint">HaplotypeProbabilities</a></dt>
<dd>
<div class="block">Gets the most likely haplotype given the probabilities.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/programgroups/Alpha.html#getName--">getName()</a></span> - Method in class picard.cmdline.programgroups.<a href="picard/cmdline/programgroups/Alpha.html" title="class in picard.cmdline.programgroups">Alpha</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/programgroups/Fasta.html#getName--">getName()</a></span> - Method in class picard.cmdline.programgroups.<a href="picard/cmdline/programgroups/Fasta.html" title="class in picard.cmdline.programgroups">Fasta</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/programgroups/Fingerprinting.html#getName--">getName()</a></span> - Method in class picard.cmdline.programgroups.<a href="picard/cmdline/programgroups/Fingerprinting.html" title="class in picard.cmdline.programgroups">Fingerprinting</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/programgroups/Illumina.html#getName--">getName()</a></span> - Method in class picard.cmdline.programgroups.<a href="picard/cmdline/programgroups/Illumina.html" title="class in picard.cmdline.programgroups">Illumina</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/programgroups/Intervals.html#getName--">getName()</a></span> - Method in class picard.cmdline.programgroups.<a href="picard/cmdline/programgroups/Intervals.html" title="class in picard.cmdline.programgroups">Intervals</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/programgroups/Metrics.html#getName--">getName()</a></span> - Method in class picard.cmdline.programgroups.<a href="picard/cmdline/programgroups/Metrics.html" title="class in picard.cmdline.programgroups">Metrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/programgroups/None.html#getName--">getName()</a></span> - Method in class picard.cmdline.programgroups.<a href="picard/cmdline/programgroups/None.html" title="class in picard.cmdline.programgroups">None</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/programgroups/SamOrBam.html#getName--">getName()</a></span> - Method in class picard.cmdline.programgroups.<a href="picard/cmdline/programgroups/SamOrBam.html" title="class in picard.cmdline.programgroups">SamOrBam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/programgroups/Testing.html#getName--">getName()</a></span> - Method in class picard.cmdline.programgroups.<a href="picard/cmdline/programgroups/Testing.html" title="class in picard.cmdline.programgroups">Testing</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/programgroups/VcfOrBcf.html#getName--">getName()</a></span> - Method in class picard.cmdline.programgroups.<a href="picard/cmdline/programgroups/VcfOrBcf.html" title="class in picard.cmdline.programgroups">VcfOrBcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/Snp.html#getName--">getName()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/Snp.html" title="class in picard.fingerprint">Snp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AdapterPair.html#getName--">getName()</a></span> - Method in interface picard.util.<a href="picard/util/AdapterPair.html" title="interface in picard.util">AdapterPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IlluminaUtil.IlluminaAdapterPair.html#getName--">getName()</a></span> - Method in enum picard.util.<a href="picard/util/IlluminaUtil.IlluminaAdapterPair.html" title="enum in picard.util">IlluminaUtil.IlluminaAdapterPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetMetricsCollector.html#getNearProbeDistance--">getNearProbeDistance()</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/TargetMetricsCollector.html" title="class in picard.analysis.directed">TargetMetricsCollector</a></dt>
<dd>
<div class="block">Gets the distance that is allowed between a read and the nearest probe for it to be considered "near probe" and "selected.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/QuerySortedReadPairIteratorUtil.html#getNextReadPair-htsjdk.samtools.util.PeekableIterator-">getNextReadPair(PeekableIterator&lt;SAMRecord&gt;)</a></span> - Static method in class picard.util.<a href="picard/util/QuerySortedReadPairIteratorUtil.html" title="class in picard.util">QuerySortedReadPairIteratorUtil</a></dt>
<dd>
<div class="block">Get the next read pair (where both have the same read name).</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadData.html#getNoise--">getNoise()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ReadData.html" title="class in picard.illumina.parser">ReadData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.PerTileBarcodeExtractor.html#getNoMatchMetric--">getNoMatchMetric()</a></span> - Method in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.PerTileBarcodeExtractor.html" title="class in picard.illumina">ExtractIlluminaBarcodes.PerTileBarcodeExtractor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.LogMath.html#getNonLogValue-double-">getNonLogValue(double)</a></span> - Method in class picard.util.<a href="picard/util/MathUtil.LogMath.html" title="class in picard.util">MathUtil.LogMath</a></dt>
<dd>
<div class="block">Returns the decimal representation of the provided log values.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/AdapterMarker.html#getNumAdaptersToKeep--">getNumAdaptersToKeep()</a></span> - Method in class picard.util.<a href="picard/util/AdapterMarker.html" title="class in picard.util">AdapterMarker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/RnaSeqMetricsCollector.PerUnitRnaSeqMetricsCollector.html#getNumAlignedBases-htsjdk.samtools.SAMRecord-">getNumAlignedBases(SAMRecord)</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/RnaSeqMetricsCollector.PerUnitRnaSeqMetricsCollector.html" title="class in picard.analysis.directed">RnaSeqMetricsCollector.PerUnitRnaSeqMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetMetricsCollector.html#getNumBasesPassingMinimumBaseQuality-htsjdk.samtools.SAMRecord-htsjdk.samtools.AlignmentBlock-int-">getNumBasesPassingMinimumBaseQuality(SAMRecord, AlignmentBlock, int)</a></span> - Static method in class picard.analysis.directed.<a href="picard/analysis/directed/TargetMetricsCollector.html" title="class in picard.analysis.directed">TargetMetricsCollector</a></dt>
<dd>
<div class="block">Get the the number of bases in the given alignment block and record that have base quality greater or equal to the minimum</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/BclReader.html#getNumberOfClusters-java.io.File-">getNumberOfClusters(File)</a></span> - Static method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/BclReader.html" title="class in picard.illumina.parser.readers">BclReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/LibraryIdGenerator.html#getNumberOfOpticalDuplicateClusters--">getNumberOfOpticalDuplicateClusters()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/LibraryIdGenerator.html" title="class in picard.sam.markduplicates.util">LibraryIdGenerator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/TileIndex.TileIndexRecord.html#getNumClustersInTile--">getNumClustersInTile()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/TileIndex.TileIndexRecord.html" title="class in picard.illumina.parser">TileIndex.TileIndexRecord</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadStructure.html#getNumDescriptors--">getNumDescriptors()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ReadStructure.html" title="class in picard.illumina.parser">ReadStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigarIterator.html#getNumDuplicates--">getNumDuplicates()</a></span> - Method in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigarIterator.html" title="class in picard.sam.markduplicates">MarkDuplicatesWithMateCigarIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/MarkQueue.html#getNumDuplicates--">getNumDuplicates()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/MarkQueue.html" title="class in picard.sam.markduplicates.util">MarkQueue</a></dt>
<dd>
<div class="block">Returns the number of duplicates detected</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ClusterData.html#getNumReads--">getNumReads()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ClusterData.html" title="class in picard.illumina.parser">ClusterData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigarIterator.html#getNumRecordsWithNoMateCigar--">getNumRecordsWithNoMateCigar()</a></span> - Method in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigarIterator.html" title="class in picard.sam.markduplicates">MarkDuplicatesWithMateCigarIterator</a></dt>
<dd>
<div class="block">Useful for statistics after the iterator has been exhausted and closed.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/BclIndexReader.html#getNumTiles--">getNumTiles()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/BclIndexReader.html" title="class in picard.illumina.parser.readers">BclIndexReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/TileIndex.html#getNumTiles--">getNumTiles()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/TileIndex.html" title="class in picard.illumina.parser">TileIndex</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilities.html#getObsAllele1--">getObsAllele1()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilities.html" title="class in picard.fingerprint">HaplotypeProbabilities</a></dt>
<dd>
<div class="block">Returns the number of observations of alleles supporting the first/major haplotype allele.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilitiesFromSequence.html#getObsAllele1--">getObsAllele1()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilitiesFromSequence.html" title="class in picard.fingerprint">HaplotypeProbabilitiesFromSequence</a></dt>
<dd>
<div class="block">Returns the number of bases/reads that support the first allele.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilityOfNormalGivenTumor.html#getObsAllele1--">getObsAllele1()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilityOfNormalGivenTumor.html" title="class in picard.fingerprint">HaplotypeProbabilityOfNormalGivenTumor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilities.html#getObsAllele2--">getObsAllele2()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilities.html" title="class in picard.fingerprint">HaplotypeProbabilities</a></dt>
<dd>
<div class="block">Returns the number of observations of alleles supporting the second/minor haplotype allele.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilitiesFromSequence.html#getObsAllele2--">getObsAllele2()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilitiesFromSequence.html" title="class in picard.fingerprint">HaplotypeProbabilitiesFromSequence</a></dt>
<dd>
<div class="block">Returns the number of bases/reads that support the second allele.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilityOfNormalGivenTumor.html#getObsAllele2--">getObsAllele2()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilityOfNormalGivenTumor.html" title="class in picard.fingerprint">HaplotypeProbabilityOfNormalGivenTumor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigarIterator.html#getOpticalDupesByLibraryId--">getOpticalDupesByLibraryId()</a></span> - Method in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigarIterator.html" title="class in picard.sam.markduplicates">MarkDuplicatesWithMateCigarIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/LibraryIdGenerator.html#getOpticalDuplicatesByLibraryIdMap--">getOpticalDuplicatesByLibraryIdMap()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/LibraryIdGenerator.html" title="class in picard.sam.markduplicates.util">LibraryIdGenerator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEnds.html#getOrientationByte-boolean-boolean-">getOrientationByte(boolean, boolean)</a></span> - Static method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEnds.html" title="class in picard.sam.markduplicates.util">ReadEnds</a></dt>
<dd>
<div class="block">Returns a single byte that encodes the orientation of the two reads in a pair.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/OutputMapping.html#getOutputCycles--">getOutputCycles()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/OutputMapping.html" title="class in picard.illumina.parser">OutputMapping</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/OutputMapping.html#getOutputDescriptors--">getOutputDescriptors()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/OutputMapping.html" title="class in picard.illumina.parser">OutputMapping</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CalculateReadGroupChecksum.html#getOutputFileName-java.io.File-">getOutputFileName(File)</a></span> - Static method in class picard.sam.<a href="picard/sam/CalculateReadGroupChecksum.html" title="class in picard.sam">CalculateReadGroupChecksum</a></dt>
<dd>
<div class="block">Creates a file name (not including the path) for an RG MD5 file based on the name of the input file.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/OutputMapping.html#getOutputIndexForCycle-int-">getOutputIndexForCycle(int)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/OutputMapping.html" title="class in picard.illumina.parser">OutputMapping</a></dt>
<dd>
<div class="block">Return an index that where:
     index.arrayIndex - represents either the read number the cycle will be output too, or (in some cases)
     an array index into a two dimensional array of byte[][] where the top level array corresponds to read number

     index.elementIndex - represents the element a cycle will appear in inside it's give read, or the element
     in an array as described above</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/OutputMapping.html#getOutputReadLengths--">getOutputReadLengths()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/OutputMapping.html" title="class in picard.illumina.parser">OutputMapping</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaDataProviderFactory.html#getOutputReadStructure--">getOutputReadStructure()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/IlluminaDataProviderFactory.html" title="class in picard.illumina.parser">IlluminaDataProviderFactory</a></dt>
<dd>
<div class="block">Sometimes (in the case of skipped reads) the logical read structure of the output cluster data is different from the input
 readStructure</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/OutputMapping.html#getOutputReadStructure--">getOutputReadStructure()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/OutputMapping.html" title="class in picard.illumina.parser">OutputMapping</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesCodec.html#getOutputStream--">getOutputStream()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesCodec.html" title="class in picard.sam.markduplicates.util">ReadEndsForMarkDuplicatesCodec</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/RepresentativeReadIndexerCodec.html#getOutputStream--">getOutputStream()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/RepresentativeReadIndexerCodec.html" title="class in picard.sam.markduplicates.util">RepresentativeReadIndexerCodec</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/PicardCommandLine.html#getPackageList--">getPackageList()</a></span> - Static method in class picard.cmdline.<a href="picard/cmdline/PicardCommandLine.html" title="class in picard.cmdline">PicardCommandLine</a></dt>
<dd>
<div class="block">The packages we wish to include in our command line</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/pedigree/PedFile.PedTrio.html#getPaternalId--">getPaternalId()</a></span> - Method in class picard.pedigree.<a href="picard/pedigree/PedFile.PedTrio.html" title="class in picard.pedigree">PedFile.PedTrio</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PerTilePerCycleFileUtil.html#getPerTilePerCycleFiles--">getPerTilePerCycleFiles()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/PerTilePerCycleFileUtil.html" title="class in picard.illumina.parser">PerTilePerCycleFileUtil</a></dt>
<dd>
<div class="block">For the given tiles, populate a CycleIlluminaFileMap that contains all these tiles and will iterate through
 all the files for these tiles in expectedBase
 Side Effect: Assigns numCycles</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaMetricsCode.html#getPhasingCode-int-picard.illumina.parser.IlluminaMetricsCode-">getPhasingCode(int, IlluminaMetricsCode)</a></span> - Static method in enum picard.illumina.parser.<a href="picard/illumina/parser/IlluminaMetricsCode.html" title="enum in picard.illumina.parser">IlluminaMetricsCode</a></dt>
<dd>
<div class="block">Phasing codes are between 200 and 299 (inclusive).</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/Tile.html#getPhasingMap--">getPhasingMap()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/Tile.html" title="class in picard.illumina.parser">Tile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaPhasingMetrics.html#getPhasingMetricsForTiles-long-java.util.Collection-boolean-">getPhasingMetricsForTiles(long, Collection&lt;Tile&gt;, boolean)</a></span> - Static method in class picard.illumina.<a href="picard/illumina/IlluminaPhasingMetrics.html" title="class in picard.illumina">IlluminaPhasingMetrics</a></dt>
<dd>
<div class="block">Calculate the median phasing & prephasing values for a lane's tiles and create the appropriate IlluminaPhasingMetrics for them</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/TilePhasingValue.html#getPhasingValue--">getPhasingValue()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/TilePhasingValue.html" title="class in picard.illumina.parser">TilePhasingValue</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/pedigree/PedFile.PedTrio.html#getPhenotype--">getPhenotype()</a></span> - Method in class picard.pedigree.<a href="picard/pedigree/PedFile.PedTrio.html" title="class in picard.pedigree">PedFile.PedTrio</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintIdDetails.html#getPlatformUnit--">getPlatformUnit()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/FingerprintIdDetails.html" title="class in picard.fingerprint">FingerprintIdDetails</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/BclQualityEvaluationStrategy.html#getPoorQualityFrequencies--">getPoorQualityFrequencies()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/BclQualityEvaluationStrategy.html" title="class in picard.illumina.parser.readers">BclQualityEvaluationStrategy</a></dt>
<dd>
<div class="block">Returns a view of number of qualities that failed, where the key is the quality score and the value is the number of observations.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/MatchResults.html#getPopulationLikelihood--">getPopulationLikelihood()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/MatchResults.html" title="class in picard.fingerprint">MatchResults</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/Snp.html#getPos--">getPos()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/Snp.html" title="class in picard.fingerprint">Snp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/CbclData.html#getPositionInfo--">getPositionInfo()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/CbclData.html" title="class in picard.illumina.parser">CbclData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilities.html#getPosteriorProbabilities--">getPosteriorProbabilities()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilities.html" title="class in picard.fingerprint">HaplotypeProbabilities</a></dt>
<dd>
<div class="block">Returns the posterior probabilities using the population frequency as a prior.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/Tile.html#getPrePhasingMap--">getPrePhasingMap()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/Tile.html" title="class in picard.illumina.parser">Tile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/TilePhasingValue.html#getPrePhasingValue--">getPrePhasingValue()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/TilePhasingValue.html" title="class in picard.illumina.parser">TilePhasingValue</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/Fingerprint.html#getPrintableId--">getPrintableId()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/Fingerprint.html" title="class in picard.fingerprint">Fingerprint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilities.html#getPriorProbablities--">getPriorProbablities()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilities.html" title="class in picard.fingerprint">HaplotypeProbabilities</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/CollectHsMetrics.html#getProbeIntervals--">getProbeIntervals()</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/CollectHsMetrics.html" title="class in picard.analysis.directed">CollectHsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/CollectTargetedMetrics.html#getProbeIntervals--">getProbeIntervals()</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/CollectTargetedMetrics.html" title="class in picard.analysis.directed">CollectTargetedMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/CollectTargetedPcrMetrics.html#getProbeIntervals--">getProbeIntervals()</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/CollectTargetedPcrMetrics.html" title="class in picard.analysis.directed">CollectTargetedPcrMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/CollectHsMetrics.html#getProbeSetName--">getProbeSetName()</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/CollectHsMetrics.html" title="class in picard.analysis.directed">CollectHsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/CollectTargetedMetrics.html#getProbeSetName--">getProbeSetName()</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/CollectTargetedMetrics.html" title="class in picard.analysis.directed">CollectTargetedMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/CollectTargetedPcrMetrics.html#getProbeSetName--">getProbeSetName()</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/CollectTargetedPcrMetrics.html" title="class in picard.analysis.directed">CollectTargetedPcrMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/PicardCommandLine.html#getProgramProperty-java.lang.Class-">getProgramProperty(Class)</a></span> - Static method in class picard.cmdline.<a href="picard/cmdline/PicardCommandLine.html" title="class in picard.cmdline">PicardCommandLine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.html#getProgramRecord--">getProgramRecord()</a></span> - Method in class picard.sam.<a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/BclData.html#getQualities--">getQualities()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/BclData.html" title="class in picard.illumina.parser">BclData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadData.html#getQualities--">getQualities()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ReadData.html" title="class in picard.illumina.parser">ReadData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.html#getQuerynameSortedAlignedRecords--">getQuerynameSortedAlignedRecords()</a></span> - Method in class picard.sam.<a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamAlignmentMerger.html#getQuerynameSortedAlignedRecords--">getQuerynameSortedAlignedRecords()</a></span> - Method in class picard.sam.<a href="picard/sam/SamAlignmentMerger.html" title="class in picard.sam">SamAlignmentMerger</a></dt>
<dd>
<div class="block">Reads the aligned SAM records into a SortingCollection and returns an iterator over that collection</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadData.html#getRawIntensities--">getRawIntensities()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ReadData.html" title="class in picard.illumina.parser">ReadData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ClusterData.html#getRead-int-">getRead(int)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ClusterData.html" title="class in picard.illumina.parser">ClusterData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/EstimateLibraryComplexity.html#getReadBarcodeValue-htsjdk.samtools.SAMRecord-java.lang.String-">getReadBarcodeValue(SAMRecord, String)</a></span> - Static method in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/EstimateLibraryComplexity.html" title="class in picard.sam.markduplicates">EstimateLibraryComplexity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/PhysicalLocationForMateCigarSet.html#getReadEnds--">getReadEnds()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/PhysicalLocationForMateCigarSet.html" title="class in picard.sam.markduplicates.util">PhysicalLocationForMateCigarSet</a></dt>
<dd>
<div class="block">Gets the set of read ends</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMateCigar.html#getReadEndSetForOpticalDuplicates--">getReadEndSetForOpticalDuplicates()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMateCigar.html" title="class in picard.sam.markduplicates.util">ReadEndsForMateCigar</a></dt>
<dd>
<div class="block">Gets the read ends for optical duplicate tracking</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintResults.html#getReadGroup--">getReadGroup()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/FingerprintResults.html" title="class in picard.fingerprint">FingerprintResults</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/PhysicalLocationForMateCigar.html#getReadGroup--">getReadGroup()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/PhysicalLocationForMateCigar.html" title="class in picard.sam.markduplicates.util">PhysicalLocationForMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEnds.html#getReadGroup--">getReadGroup()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEnds.html" title="class in picard.sam.markduplicates.util">ReadEnds</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/PhysicalLocation.html#getReadGroup--">getReadGroup()</a></span> - Method in interface picard.sam.util.<a href="picard/sam/util/PhysicalLocation.html" title="interface in picard.sam.util">PhysicalLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/PhysicalLocationInt.html#getReadGroup--">getReadGroup()</a></span> - Method in class picard.sam.util.<a href="picard/sam/util/PhysicalLocationInt.html" title="class in picard.sam.util">PhysicalLocationInt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/LibraryIdGenerator.html#getReadGroupLibraryName-htsjdk.samtools.SAMReadGroupRecord-">getReadGroupLibraryName(SAMReadGroupRecord)</a></span> - Static method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/LibraryIdGenerator.html" title="class in picard.sam.markduplicates.util">LibraryIdGenerator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadData.html#getReadType--">getReadType()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ReadData.html" title="class in picard.illumina.parser">ReadData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMateCigar.html#getRecord--">getRecord()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMateCigar.html" title="class in picard.sam.markduplicates.util">ReadEndsForMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMateCigar.html#getRecordReadName--">getRecordReadName()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMateCigar.html" title="class in picard.sam.markduplicates.util">ReadEndsForMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectAlignmentSummaryMetrics.CollectAlignmentRefArgCollection.html#getReferenceFile--">getReferenceFile()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectAlignmentSummaryMetrics.CollectAlignmentRefArgCollection.html" title="class in picard.analysis">CollectAlignmentSummaryMetrics.CollectAlignmentRefArgCollection</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRrbsMetrics.CollectRrbsMetricsReferenceArgumentCollection.html#getReferenceFile--">getReferenceFile()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectRrbsMetrics.CollectRrbsMetricsReferenceArgumentCollection.html" title="class in picard.analysis">CollectRrbsMetrics.CollectRrbsMetricsReferenceArgumentCollection</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/argumentcollections/OptionalReferenceArgumentCollection.html#getReferenceFile--">getReferenceFile()</a></span> - Method in class picard.cmdline.argumentcollections.<a href="picard/cmdline/argumentcollections/OptionalReferenceArgumentCollection.html" title="class in picard.cmdline.argumentcollections">OptionalReferenceArgumentCollection</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/argumentcollections/ReferenceArgumentCollection.html#getReferenceFile--">getReferenceFile()</a></span> - Method in interface picard.cmdline.argumentcollections.<a href="picard/cmdline/argumentcollections/ReferenceArgumentCollection.html" title="interface in picard.cmdline.argumentcollections">ReferenceArgumentCollection</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/argumentcollections/RequiredReferenceArgumentCollection.html#getReferenceFile--">getReferenceFile()</a></span> - Method in class picard.cmdline.argumentcollections.<a href="picard/cmdline/argumentcollections/RequiredReferenceArgumentCollection.html" title="class in picard.cmdline.argumentcollections">RequiredReferenceArgumentCollection</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CreateSequenceDictionary.CreateSeqDictReferenceArgumentCollection.html#getReferenceFile--">getReferenceFile()</a></span> - Method in class picard.sam.<a href="picard/sam/CreateSequenceDictionary.CreateSeqDictReferenceArgumentCollection.html" title="class in picard.sam">CreateSequenceDictionary.CreateSeqDictReferenceArgumentCollection</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ReorderSam.ReorderSamReferenceArgumentCollection.html#getReferenceFile--">getReferenceFile()</a></span> - Method in class picard.sam.<a href="picard/sam/ReorderSam.ReorderSamReferenceArgumentCollection.html" title="class in picard.sam">ReorderSam.ReorderSamReferenceArgumentCollection</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/ScatterIntervalsByNs.ScatterIntervalsByNReferenceArgumentCollection.html#getReferenceFile--">getReferenceFile()</a></span> - Method in class picard.util.<a href="picard/util/ScatterIntervalsByNs.ScatterIntervalsByNReferenceArgumentCollection.html" title="class in picard.util">ScatterIntervalsByNs.ScatterIntervalsByNReferenceArgumentCollection</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/metrics/SAMRecordAndReference.html#getReferenceSequence--">getReferenceSequence()</a></span> - Method in class picard.metrics.<a href="picard/metrics/SAMRecordAndReference.html" title="class in picard.metrics">SAMRecordAndReference</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilities.html#getRepresentativeSnp--">getRepresentativeSnp()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilities.html" title="class in picard.fingerprint">HaplotypeProbabilities</a></dt>
<dd>
<div class="block">Returns a representative SNP for this haplotype.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilitiesFromGenotype.html#getRepresentativeSnp--">getRepresentativeSnp()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilitiesFromGenotype.html" title="class in picard.fingerprint">HaplotypeProbabilitiesFromGenotype</a></dt>
<dd>
<div class="block">Returns the SNP who's genotype was used to construct the likelihoods.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilityOfNormalGivenTumor.html#getRepresentativeSnp--">getRepresentativeSnp()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilityOfNormalGivenTumor.html" title="class in picard.fingerprint">HaplotypeProbabilityOfNormalGivenTumor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.html#getSamFileHeader--">getSamFileHeader()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics</a></dt>
<dd>
<div class="block">This method should only be called after <code>this.getSamReader()</code> is called.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/Fingerprint.html#getSample--">getSample()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/Fingerprint.html" title="class in picard.fingerprint">Fingerprint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintIdDetails.html#getSample--">getSample()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/FingerprintIdDetails.html" title="class in picard.fingerprint">FingerprintIdDetails</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/MatchResults.html#getSample--">getSample()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/MatchResults.html" title="class in picard.fingerprint">MatchResults</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintResults.html#getSampleAlias--">getSampleAlias()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/FingerprintResults.html" title="class in picard.fingerprint">FingerprintResults</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/MatchResults.html#getSampleLikelihood--">getSampleLikelihood()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/MatchResults.html" title="class in picard.fingerprint">MatchResults</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.html#getSamReader--">getSamReader()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics</a></dt>
<dd>
<div class="block">Gets the SamReader from which records will be examined.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.html#getSamReader--">getSamReader()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.html" title="class in picard.analysis">CollectWgsMetricsWithNonZeroCoverage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/metrics/SAMRecordAndReference.html#getSamRecord--">getSamRecord()</a></span> - Method in class picard.metrics.<a href="picard/metrics/SAMRecordAndReference.html" title="class in picard.metrics">SAMRecordAndReference</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMateCigar.html#getSamRecordIndex--">getSamRecordIndex()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMateCigar.html" title="class in picard.sam.markduplicates.util">ReadEndsForMateCigar</a></dt>
<dd>
<div class="block">A number of convenience functions</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/IntervalListTools.html#getScatteredFileName-java.io.File-long-java.lang.String-">getScatteredFileName(File, long, String)</a></span> - Static method in class picard.util.<a href="picard/util/IntervalListTools.html" title="class in picard.util">IntervalListTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceSchemeFactory.html#getScheme-boolean-">getScheme(boolean)</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceSchemeFactory.html" title="class in picard.vcf">GenotypeConcordanceSchemeFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceCounts.html#getSensitivity-picard.vcf.GenotypeConcordanceScheme-picard.vcf.GenotypeConcordanceStates.TruthState:A-">getSensitivity(GenotypeConcordanceScheme, GenotypeConcordanceStates.TruthState[])</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceCounts.html" title="class in picard.vcf">GenotypeConcordanceCounts</a></dt>
<dd>
<div class="block">Returns the sensitivity defined by the scheme across the subset of truth states.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FastqToSam.html#getSequentialFileList-java.io.File-">getSequentialFileList(File)</a></span> - Static method in class picard.sam.<a href="picard/sam/FastqToSam.html" title="class in picard.sam">FastqToSam</a></dt>
<dd>
<div class="block">Get a list of FASTQs that are sequentially numbered based on the first (base) fastq.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/pedigree/PedFile.PedTrio.html#getSex--">getSex()</a></span> - Method in class picard.pedigree.<a href="picard/pedigree/PedFile.PedTrio.html" title="class in picard.pedigree">PedFile.PedTrio</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CallingMetricAccumulator.html#getSingletonSample-htsjdk.variant.variantcontext.VariantContext-">getSingletonSample(VariantContext)</a></span> - Static method in class picard.vcf.<a href="picard/vcf/CallingMetricAccumulator.html" title="class in picard.vcf">CallingMetricAccumulator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicates.html#getSizeOf--">getSizeOf()</a></span> - Static method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicates.html" title="class in picard.sam.markduplicates.util">ReadEndsForMarkDuplicates</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesWithBarcodes.html#getSizeOf--">getSizeOf()</a></span> - Static method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesWithBarcodes.html" title="class in picard.sam.markduplicates.util">ReadEndsForMarkDuplicatesWithBarcodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeBlock.html#getSnp-java.lang.String-">getSnp(String)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeBlock.html" title="class in picard.fingerprint">HaplotypeBlock</a></dt>
<dd>
<div class="block">Gets a SNP by name if it belongs to this haplotype.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeMap.html#getSnp-java.lang.String-int-">getSnp(String, int)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeMap.html" title="class in picard.fingerprint">HaplotypeMap</a></dt>
<dd>
<div class="block">Queries a Snp by chromosome and position.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#getSnpCounter--">getSnpCounter()</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeBlock.html#getSnpGenotype-picard.fingerprint.Snp-picard.fingerprint.DiploidHaplotype-">getSnpGenotype(Snp, DiploidHaplotype)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeBlock.html" title="class in picard.fingerprint">HaplotypeBlock</a></dt>
<dd>
<div class="block">Gets the expected genotype of the provided SNP given the provided haplotype of this
 haplotype block.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeBlock.html#getSnps--">getSnps()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeBlock.html" title="class in picard.fingerprint">HaplotypeBlock</a></dt>
<dd>
<div class="block">Returns an unmodifiable, unordered, collection of all SNPs in this haplotype block.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/IlluminaUtil.html#getSolexaQualityCharFromFourQualities-java.lang.String:A-int-htsjdk.samtools.util.FormatUtil-">getSolexaQualityCharFromFourQualities(String[], int, FormatUtil)</a></span> - Static method in class picard.util.<a href="picard/util/IlluminaUtil.html" title="class in picard.util">IlluminaUtil</a></dt>
<dd>
<div class="block">Get a Solexa ASCII quality value from an array of strings that are integer qualities in this order:
 [cycle-1-A, cycle-1-C, cycle-1-G, cycle-1-T, cycle-2-A, ...].</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/Fingerprint.html#getSource--">getSource()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/Fingerprint.html" title="class in picard.fingerprint">Fingerprint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceCounts.html#getSpecificity-picard.vcf.GenotypeConcordanceScheme-picard.vcf.GenotypeConcordanceStates.TruthState:A-">getSpecificity(GenotypeConcordanceScheme, GenotypeConcordanceStates.TruthState[])</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceCounts.html" title="class in picard.vcf">GenotypeConcordanceCounts</a></dt>
<dd>
<div class="block">Returns the specificity defined by the scheme across the subset of truth states.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#getStandardUsagePreamble--">getStandardUsagePreamble()</a></span> - Method in class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#getStandardUsagePreamble-java.lang.Class-">getStandardUsagePreamble(Class&lt;?&gt;)</a></span> - Static method in class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>
<div class="block">A typical command line program will call this to get the beginning of the usage message,
 and then typically append a description of the program, like this:

 public String USAGE = getStandardUsagePreamble(getClass()) + "Frobnicate the freebozle."</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceCounts.html#getSum-java.util.Set-java.util.Set-">getSum(Set&lt;GenotypeConcordanceStates.TruthState&gt;, Set&lt;GenotypeConcordanceStates.CallState&gt;)</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceCounts.html" title="class in picard.vcf">GenotypeConcordanceCounts</a></dt>
<dd>
<div class="block">Returns the sum of all pairs of tuples defined by the truth state set and call state set.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceCounts.html#getSum--">getSum()</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceCounts.html" title="class in picard.vcf">GenotypeConcordanceCounts</a></dt>
<dd>
<div class="block">Returns the sum of all pairs of tuples defined by the truth state set and call state set.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/help/HelpConstants.html#getSuperCategoryMap--">getSuperCategoryMap()</a></span> - Static method in class picard.util.help.<a href="picard/util/help/HelpConstants.html" title="class in picard.util.help">HelpConstants</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/help/HelpConstants.html#getSuperCategoryProperty-java.lang.String-">getSuperCategoryProperty(String)</a></span> - Static method in class picard.util.help.<a href="picard/util/help/HelpConstants.html" title="class in picard.util.help">HelpConstants</a></dt>
<dd>
<div class="block">Given a group name, return a supercategory string for use by the online doc system to determine which
 supercateogry the group is in.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/FourChannelIntensityData.html#getT--">getT()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/FourChannelIntensityData.html" title="class in picard.illumina.parser">FourChannelIntensityData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/help/PicardHelpDocWorkUnitHandler.html#getTagFilterPrefix--">getTagFilterPrefix()</a></span> - Method in class picard.util.help.<a href="picard/util/help/PicardHelpDocWorkUnitHandler.html" title="class in picard.util.help">PicardHelpDocWorkUnitHandler</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/help/PicardHelpDocWorkUnitHandler.html#getTemplateName-org.broadinstitute.barclay.help.DocWorkUnit-">getTemplateName(DocWorkUnit)</a></span> - Method in class picard.util.help.<a href="picard/util/help/PicardHelpDocWorkUnitHandler.html" title="class in picard.util.help">PicardHelpDocWorkUnitHandler</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AdapterMarker.html#getThresholdForSelectingAdaptersToKeep--">getThresholdForSelectingAdaptersToKeep()</a></span> - Method in class picard.util.<a href="picard/util/AdapterMarker.html" title="class in picard.util">AdapterMarker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/CbclData.html#getTile--">getTile()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/CbclData.html" title="class in picard.illumina.parser">CbclData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ClusterData.html#getTile--">getTile()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ClusterData.html" title="class in picard.illumina.parser">ClusterData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.html#getTile--">getTile()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.html" title="class in picard.illumina.parser.readers">AbstractIlluminaPositionFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/PhysicalLocation.html#getTile--">getTile()</a></span> - Method in interface picard.sam.util.<a href="picard/sam/util/PhysicalLocation.html" title="interface in picard.sam.util">PhysicalLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/PhysicalLocationInt.html#getTile--">getTile()</a></span> - Method in class picard.sam.util.<a href="picard/sam/util/PhysicalLocationInt.html" title="class in picard.sam.util">PhysicalLocationInt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/NewIlluminaBasecallsConverter.html#getTiledFiles-java.io.File-java.util.regex.Pattern-">getTiledFiles(File, Pattern)</a></span> - Static method in class picard.illumina.<a href="picard/illumina/NewIlluminaBasecallsConverter.html" title="class in picard.illumina">NewIlluminaBasecallsConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ParameterizedFileUtil.html#getTiledFiles-java.io.File-java.util.regex.Pattern-">getTiledFiles(File, Pattern)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ParameterizedFileUtil.html" title="class in picard.illumina.parser">ParameterizedFileUtil</a></dt>
<dd>
<div class="block">Return all files that match pattern of the given file type in the given base directory</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/IlluminaUtil.html#getTileFromReadName-java.lang.String-">getTileFromReadName(String)</a></span> - Static method in class picard.util.<a href="picard/util/IlluminaUtil.html" title="class in picard.util">IlluminaUtil</a></dt>
<dd>
<div class="block">Parse the tile # from the read name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/BaseBclReader.CycleData.html#getTileInfo--">getTileInfo()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/BaseBclReader.CycleData.html" title="class in picard.illumina.parser.readers">BaseBclReader.CycleData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/BaseBclReader.TileData.html#getTileNum--">getTileNum()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/BaseBclReader.TileData.html" title="class in picard.illumina.parser.readers">BaseBclReader.TileData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaLaneTileCode.html#getTileNumber--">getTileNumber()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaLaneTileCode.html" title="class in picard.illumina.parser.readers">TileMetricsOutReader.IlluminaLaneTileCode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaTileMetrics.html#getTileNumber--">getTileNumber()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaTileMetrics.html" title="class in picard.illumina.parser.readers">TileMetricsOutReader.IlluminaTileMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/Tile.html#getTileNumber--">getTileNumber()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/Tile.html" title="class in picard.illumina.parser">Tile</a></dt>
<dd>
<div class="block">Returns the number/name of this tile.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/MultiTileParser.html#getTileOfNextCluster--">getTileOfNextCluster()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/MultiTileParser.html" title="class in picard.illumina.parser">MultiTileParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PerTileParser.html#getTileOfNextCluster--">getTileOfNextCluster()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/PerTileParser.html" title="class in picard.illumina.parser">PerTileParser</a></dt>
<dd>
<div class="block">Return the tile of the NEXT ILLUMINA_DATA object to be returned by the method next.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/MultiTileBclFileUtil.html#getTiles--">getTiles()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/MultiTileBclFileUtil.html" title="class in picard.illumina.parser">MultiTileBclFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/MultiTileFileUtil.html#getTiles--">getTiles()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/MultiTileFileUtil.html" title="class in picard.illumina.parser">MultiTileFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ParameterizedFileUtil.html#getTiles--">getTiles()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ParameterizedFileUtil.html" title="class in picard.illumina.parser">ParameterizedFileUtil</a></dt>
<dd>
<div class="block">Return a list of all tiles available for this file format and run</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PerTilePerCycleFileUtil.html#getTiles--">getTiles()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/PerTilePerCycleFileUtil.html" title="class in picard.illumina.parser">PerTilePerCycleFileUtil</a></dt>
<dd>
<div class="block">Discover all files of this type in expectedBase that match pattern and construct a list of tiles
 available based on these files.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/TileIndex.html#getTiles--">getTiles()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/TileIndex.html" title="class in picard.illumina.parser">TileIndex</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/TilePhasingValue.html#getTileTemplateRead--">getTileTemplateRead()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/TilePhasingValue.html" title="class in picard.illumina.parser">TilePhasingValue</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/MarkQueue.html#getToMarkQueueMinimumDistance--">getToMarkQueueMinimumDistance()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/MarkQueue.html" title="class in picard.sam.markduplicates.util">MarkQueue</a></dt>
<dd>
<div class="block">Returns the minimum genomic distance such that we can be assured that all duplicates have been considered.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetMetricsCollector.Coverage.html#getTotal--">getTotal()</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/TargetMetricsCollector.Coverage.html" title="class in picard.analysis.directed">TargetMetricsCollector.Coverage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadStructure.Substructure.html#getTotalCycles--">getTotalCycles()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ReadStructure.Substructure.html" title="class in picard.illumina.parser">ReadStructure.Substructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/InsertSizeMetricsCollector.PerUnitInsertSizeMetricsCollector.html#getTotalInserts--">getTotalInserts()</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/InsertSizeMetricsCollector.PerUnitInsertSizeMetricsCollector.html" title="class in picard.analysis.directed">InsertSizeMetricsCollector.PerUnitInsertSizeMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilities.html#getTotalObs--">getTotalObs()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilities.html" title="class in picard.fingerprint">HaplotypeProbabilities</a></dt>
<dd>
<div class="block">Returns the total number of observations of any allele.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilitiesFromSequence.html#getTotalObs--">getTotalObs()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilitiesFromSequence.html" title="class in picard.fingerprint">HaplotypeProbabilitiesFromSequence</a></dt>
<dd>
<div class="block">Gets the total number of observations presented at this locus.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilityOfNormalGivenTumor.html#getTotalObs--">getTotalObs()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilityOfNormalGivenTumor.html" title="class in picard.fingerprint">HaplotypeProbabilityOfNormalGivenTumor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/OutputMapping.html#getTotalOutputCycles--">getTotalOutputCycles()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/OutputMapping.html" title="class in picard.illumina.parser">OutputMapping</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/Gene.Transcript.html#getTranscriptCoordinate-int-">getTranscriptCoordinate(int)</a></span> - Method in class picard.annotation.<a href="picard/annotation/Gene.Transcript.html" title="class in picard.annotation">Gene.Transcript</a></dt>
<dd>
<div class="block">Given a coordinate on the genome (same chromosome) give the corresponding coordinate in the transcript.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AbstractWgsMetricsCollector.html#getUnfilteredBaseQHistogram--">getUnfilteredBaseQHistogram()</a></span> - Method in class picard.analysis.<a href="picard/analysis/AbstractWgsMetricsCollector.html" title="class in picard.analysis">AbstractWgsMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AbstractWgsMetricsCollector.html#getUnfilteredDepthHistogram--">getUnfilteredDepthHistogram()</a></span> - Method in class picard.analysis.<a href="picard/analysis/AbstractWgsMetricsCollector.html" title="class in picard.analysis">AbstractWgsMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CompareSAMs.html#getUnmappedBoth--">getUnmappedBoth()</a></span> - Method in class picard.sam.<a href="picard/sam/CompareSAMs.html" title="class in picard.sam">CompareSAMs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CompareSAMs.html#getUnmappedLeft--">getUnmappedLeft()</a></span> - Method in class picard.sam.<a href="picard/sam/CompareSAMs.html" title="class in picard.sam">CompareSAMs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CompareSAMs.html#getUnmappedRight--">getUnmappedRight()</a></span> - Method in class picard.sam.<a href="picard/sam/CompareSAMs.html" title="class in picard.sam">CompareSAMs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaFileUtil.html#getUtil-picard.illumina.parser.IlluminaFileUtil.SupportedIlluminaFormat-">getUtil(IlluminaFileUtil.SupportedIlluminaFormat)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/IlluminaFileUtil.html" title="class in picard.illumina.parser">IlluminaFileUtil</a></dt>
<dd>
<div class="block">Given a file type, get the Parameterized File Util object associated with it</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintChecker.html#getValidationStringency--">getValidationStringency()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/FingerprintChecker.html" title="class in picard.fingerprint">FingerprintChecker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceStates.TruthState.html#getVar-int-int-">getVar(int, int)</a></span> - Static method in enum picard.vcf.<a href="picard/vcf/GenotypeConcordanceStates.TruthState.html" title="enum in picard.vcf">GenotypeConcordanceStates.TruthState</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/VariantType.html#getVariantTypeFromOrdinal-int-">getVariantTypeFromOrdinal(int)</a></span> - Static method in enum picard.util.<a href="picard/util/VariantType.html" title="enum in picard.util">VariantType</a></dt>
<dd>
<div class="block">Gets the enum for a given ordinal</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#getVersion--">getVersion()</a></span> - Method in class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AbstractInputParser.html#getWordCount--">getWordCount()</a></span> - Method in class picard.util.<a href="picard/util/AbstractInputParser.html" title="class in picard.util">AbstractInputParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ClusterData.html#getX--">getX()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ClusterData.html" title="class in picard.illumina.parser">ClusterData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/PhysicalLocation.html#getX--">getX()</a></span> - Method in interface picard.sam.util.<a href="picard/sam/util/PhysicalLocation.html" title="interface in picard.sam.util">PhysicalLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/PhysicalLocationInt.html#getX--">getX()</a></span> - Method in class picard.sam.util.<a href="picard/sam/util/PhysicalLocationInt.html" title="class in picard.sam.util">PhysicalLocationInt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/CbclData.html#getXCoordinate--">getXCoordinate()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/CbclData.html" title="class in picard.illumina.parser">CbclData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ClusterData.html#getY--">getY()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ClusterData.html" title="class in picard.illumina.parser">ClusterData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/PhysicalLocation.html#getY--">getY()</a></span> - Method in interface picard.sam.util.<a href="picard/sam/util/PhysicalLocation.html" title="interface in picard.sam.util">PhysicalLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/PhysicalLocationInt.html#getY--">getY()</a></span> - Method in class picard.sam.util.<a href="picard/sam/util/PhysicalLocationInt.html" title="class in picard.sam.util">PhysicalLocationInt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/CbclData.html#getYCoordinate--">getYCoordinate()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/CbclData.html" title="class in picard.illumina.parser">CbclData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/TileIndex.TileIndexRecord.html#getZeroBasedTileNumber--">getZeroBasedTileNumber()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/TileIndex.TileIndexRecord.html" title="class in picard.illumina.parser">TileIndex.TileIndexRecord</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/GraphUtils.Graph.html#Graph--">Graph()</a></span> - Constructor for class picard.util.<a href="picard/util/GraphUtils.Graph.html" title="class in picard.util">GraphUtils.Graph</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/GraphUtils.html" title="class in picard.util"><span class="typeNameLink">GraphUtils</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">Created by farjoun on 11/2/16.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/GraphUtils.html#GraphUtils--">GraphUtils()</a></span> - Constructor for class picard.util.<a href="picard/util/GraphUtils.html" title="class in picard.util">GraphUtils</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/GraphUtils.Graph.html" title="class in picard.util"><span class="typeNameLink">GraphUtils.Graph</span></a>&lt;<a href="picard/util/GraphUtils.Graph.html" title="type parameter in GraphUtils.Graph">Node</a> extends java.lang.Comparable&lt;<a href="picard/util/GraphUtils.Graph.html" title="type parameter in GraphUtils.Graph">Node</a>&gt;&gt; - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.html#GVCF_INPUT">GVCF_INPUT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/GvcfMetricAccumulator.html" title="class in picard.vcf"><span class="typeNameLink">GvcfMetricAccumulator</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">An accumulator for collecting metrics about a single-sample GVCF.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GvcfMetricAccumulator.html#GvcfMetricAccumulator-picard.util.DbSnpBitSetUtil.DbSnpBitSets-">GvcfMetricAccumulator(DbSnpBitSetUtil.DbSnpBitSets)</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/GvcfMetricAccumulator.html" title="class in picard.vcf">GvcfMetricAccumulator</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="I:H">
<!--   -->
</a>
<h2 class="title">H</h2>
<dl>
<dt><span class="memberNameLink"><a href="picard/cmdline/ClassFinder.html#handleItem-java.lang.String-">handleItem(String)</a></span> - Method in class picard.cmdline.<a href="picard/cmdline/ClassFinder.html" title="class in picard.cmdline">ClassFinder</a></dt>
<dd>
<div class="block">Checks an item to see if it is a class and is annotated with the specified
 annotation.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CheckFingerprint.html#HAPLOTYPE_MAP">HAPLOTYPE_MAP</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CheckFingerprint.html" title="class in picard.fingerprint">CheckFingerprint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckFingerprints.html#HAPLOTYPE_MAP">HAPLOTYPE_MAP</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckFingerprints.html" title="class in picard.fingerprint">CrosscheckFingerprints</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/fingerprint/HaplotypeBlock.html" title="class in picard.fingerprint"><span class="typeNameLink">HaplotypeBlock</span></a> - Class in <a href="picard/fingerprint/package-summary.html">picard.fingerprint</a></dt>
<dd>
<div class="block">Represents information about a group of SNPs that form a haplotype in perfect LD
 with one another.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeBlock.html#HaplotypeBlock-double-">HaplotypeBlock(double)</a></span> - Constructor for class picard.fingerprint.<a href="picard/fingerprint/HaplotypeBlock.html" title="class in picard.fingerprint">HaplotypeBlock</a></dt>
<dd>
<div class="block">Constructs a haplotype block with the provided minor allele frequency.</div>
</dd>
<dt><a href="picard/fingerprint/HaplotypeMap.html" title="class in picard.fingerprint"><span class="typeNameLink">HaplotypeMap</span></a> - Class in <a href="picard/fingerprint/package-summary.html">picard.fingerprint</a></dt>
<dd>
<div class="block">A collection of metadata about Haplotype Blocks including multiple in memory "indices" of the data
 to make it easy to query the correct HaplotypeBlock or Snp by snp names, positions etc.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeMap.html#HaplotypeMap-htsjdk.samtools.SAMFileHeader-">HaplotypeMap(SAMFileHeader)</a></span> - Constructor for class picard.fingerprint.<a href="picard/fingerprint/HaplotypeMap.html" title="class in picard.fingerprint">HaplotypeMap</a></dt>
<dd>
<div class="block">Constructs an empty HaplotypeMap using the provided SAMFileHeader's sequence dictionary.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeMap.html#HaplotypeMap-java.io.File-">HaplotypeMap(File)</a></span> - Constructor for class picard.fingerprint.<a href="picard/fingerprint/HaplotypeMap.html" title="class in picard.fingerprint">HaplotypeMap</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/fingerprint/HaplotypeProbabilities.html" title="class in picard.fingerprint"><span class="typeNameLink">HaplotypeProbabilities</span></a> - Class in <a href="picard/fingerprint/package-summary.html">picard.fingerprint</a></dt>
<dd>
<div class="block">Abstract class for storing and calculating various likelihoods and probabilities
 for haplotype alleles given evidence.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilities.html#HaplotypeProbabilities-picard.fingerprint.HaplotypeBlock-">HaplotypeProbabilities(HaplotypeBlock)</a></span> - Constructor for class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilities.html" title="class in picard.fingerprint">HaplotypeProbabilities</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/fingerprint/HaplotypeProbabilities.Genotype.html" title="enum in picard.fingerprint"><span class="typeNameLink">HaplotypeProbabilities.Genotype</span></a> - Enum in <a href="picard/fingerprint/package-summary.html">picard.fingerprint</a></dt>
<dd>
<div class="block">Log10(P(evidence| haplotype)) for the 3 different possible haplotypes
 {aa, ab, bb}</div>
</dd>
<dt><a href="picard/fingerprint/HaplotypeProbabilitiesFromContaminatorSequence.html" title="class in picard.fingerprint"><span class="typeNameLink">HaplotypeProbabilitiesFromContaminatorSequence</span></a> - Class in <a href="picard/fingerprint/package-summary.html">picard.fingerprint</a></dt>
<dd>
<div class="block">Represents the probability of the underlying haplotype of the contaminating sample given the data.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilitiesFromContaminatorSequence.html#HaplotypeProbabilitiesFromContaminatorSequence-picard.fingerprint.HaplotypeBlock-double-">HaplotypeProbabilitiesFromContaminatorSequence(HaplotypeBlock, double)</a></span> - Constructor for class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilitiesFromContaminatorSequence.html" title="class in picard.fingerprint">HaplotypeProbabilitiesFromContaminatorSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/fingerprint/HaplotypeProbabilitiesFromGenotype.html" title="class in picard.fingerprint"><span class="typeNameLink">HaplotypeProbabilitiesFromGenotype</span></a> - Class in <a href="picard/fingerprint/package-summary.html">picard.fingerprint</a></dt>
<dd>
<div class="block">Represents a set of HaplotypeProbabilities that were derived from a single SNP
 genotype at a point in time.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilitiesFromGenotype.html#HaplotypeProbabilitiesFromGenotype-picard.fingerprint.Snp-picard.fingerprint.HaplotypeBlock-double-double-double-">HaplotypeProbabilitiesFromGenotype(Snp, HaplotypeBlock, double, double, double)</a></span> - Constructor for class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilitiesFromGenotype.html" title="class in picard.fingerprint">HaplotypeProbabilitiesFromGenotype</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/fingerprint/HaplotypeProbabilitiesFromGenotypeLikelihoods.html" title="class in picard.fingerprint"><span class="typeNameLink">HaplotypeProbabilitiesFromGenotypeLikelihoods</span></a> - Class in <a href="picard/fingerprint/package-summary.html">picard.fingerprint</a></dt>
<dd>
<div class="block">Represents the likelihood of the HaplotypeBlock given the GenotypeLikelihoods (GL field from a VCF, which is actually a log10-likelihood)
 for each of the SNPs in that block.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilitiesFromGenotypeLikelihoods.html#HaplotypeProbabilitiesFromGenotypeLikelihoods-picard.fingerprint.HaplotypeBlock-">HaplotypeProbabilitiesFromGenotypeLikelihoods(HaplotypeBlock)</a></span> - Constructor for class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilitiesFromGenotypeLikelihoods.html" title="class in picard.fingerprint">HaplotypeProbabilitiesFromGenotypeLikelihoods</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/fingerprint/HaplotypeProbabilitiesFromSequence.html" title="class in picard.fingerprint"><span class="typeNameLink">HaplotypeProbabilitiesFromSequence</span></a> - Class in <a href="picard/fingerprint/package-summary.html">picard.fingerprint</a></dt>
<dd>
<div class="block">Represents the probability of the underlying haplotype given the data.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilitiesFromSequence.html#HaplotypeProbabilitiesFromSequence-picard.fingerprint.HaplotypeBlock-">HaplotypeProbabilitiesFromSequence(HaplotypeBlock)</a></span> - Constructor for class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilitiesFromSequence.html" title="class in picard.fingerprint">HaplotypeProbabilitiesFromSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/fingerprint/HaplotypeProbabilityOfNormalGivenTumor.html" title="class in picard.fingerprint"><span class="typeNameLink">HaplotypeProbabilityOfNormalGivenTumor</span></a> - Class in <a href="picard/fingerprint/package-summary.html">picard.fingerprint</a></dt>
<dd>
<div class="block">A wrapper class for any HaplotypeProbabilities instance that will assume that the given evidence is that of a tumor sample and
 provide an hp for the normal sample that tumor came from.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilityOfNormalGivenTumor.html#HaplotypeProbabilityOfNormalGivenTumor-picard.fingerprint.HaplotypeProbabilities-double-">HaplotypeProbabilityOfNormalGivenTumor(HaplotypeProbabilities, double)</a></span> - Constructor for class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilityOfNormalGivenTumor.html" title="class in picard.fingerprint">HaplotypeProbabilityOfNormalGivenTumor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/FingerprintingSummaryMetrics.html#HAPLOTYPES_CONFIDENTLY_CHECKED">HAPLOTYPES_CONFIDENTLY_CHECKED</a></span> - Variable in class picard.analysis.<a href="picard/analysis/FingerprintingSummaryMetrics.html" title="class in picard.analysis">FingerprintingSummaryMetrics</a></dt>
<dd>
<div class="block">The subset of genotyped haplotypes for which there was sufficient sequence data to
 confidently genotype the haplotype.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/FingerprintingSummaryMetrics.html#HAPLOTYPES_CONFIDENTLY_MATCHING">HAPLOTYPES_CONFIDENTLY_MATCHING</a></span> - Variable in class picard.analysis.<a href="picard/analysis/FingerprintingSummaryMetrics.html" title="class in picard.analysis">FingerprintingSummaryMetrics</a></dt>
<dd>
<div class="block">The subset of confidently checked haplotypes that match the expected genotypes.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/FingerprintingSummaryMetrics.html#HAPLOTYPES_WITH_GENOTYPES">HAPLOTYPES_WITH_GENOTYPES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/FingerprintingSummaryMetrics.html" title="class in picard.analysis">FingerprintingSummaryMetrics</a></dt>
<dd>
<div class="block">The number of haplotypes that had expected genotypes to compare to.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/pedigree/PedFile.PedTrio.html#hasBothParents--">hasBothParents()</a></span> - Method in class picard.pedigree.<a href="picard/pedigree/PedFile.PedTrio.html" title="class in picard.pedigree">PedFile.PedTrio</a></dt>
<dd>
<div class="block">True if this record has paternal and maternal ids, otherwise false.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaFileUtil.html#hasCbcls-java.io.File-int-">hasCbcls(File, int)</a></span> - Static method in class picard.illumina.parser.<a href="picard/illumina/parser/IlluminaFileUtil.html" title="class in picard.illumina.parser">IlluminaFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/DelimitedTextFileWithHeaderIterator.html#hasColumn-java.lang.String-">hasColumn(String)</a></span> - Method in class picard.util.<a href="picard/util/DelimitedTextFileWithHeaderIterator.html" title="class in picard.util">DelimitedTextFileWithHeaderIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/TabbedTextFileWithHeaderParser.html#hasColumn-java.lang.String-">hasColumn(String)</a></span> - Method in class picard.util.<a href="picard/util/TabbedTextFileWithHeaderParser.html" title="class in picard.util">TabbedTextFileWithHeaderParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetMetricsCollector.Coverage.html#hasCoverage--">hasCoverage()</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/TargetMetricsCollector.Coverage.html" title="class in picard.analysis.directed">TargetMetricsCollector.Coverage</a></dt>
<dd>
<div class="block">Returns true if any base in the range has coverage of > 0</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilities.html#hasEvidence--">hasEvidence()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilities.html" title="class in picard.fingerprint">HaplotypeProbabilities</a></dt>
<dd>
<div class="block">Returns true if evidence has been added, false if the probabilities are just the priors.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilitiesFromSequence.html#hasEvidence--">hasEvidence()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilitiesFromSequence.html" title="class in picard.fingerprint">HaplotypeProbabilitiesFromSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilityOfNormalGivenTumor.html#hasEvidence--">hasEvidence()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilityOfNormalGivenTumor.html" title="class in picard.fingerprint">HaplotypeProbabilityOfNormalGivenTumor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/Gene.Transcript.html#hashCode--">hashCode()</a></span> - Method in class picard.annotation.<a href="picard/annotation/Gene.Transcript.html" title="class in picard.annotation">Gene.Transcript</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintIdDetails.html#hashCode--">hashCode()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/FingerprintIdDetails.html" title="class in picard.fingerprint">FingerprintIdDetails</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeBlock.html#hashCode--">hashCode()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeBlock.html" title="class in picard.fingerprint">HaplotypeBlock</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/Snp.html#hashCode--">hashCode()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/Snp.html" title="class in picard.fingerprint">Snp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/FourChannelIntensityData.html#hashCode--">hashCode()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/FourChannelIntensityData.html" title="class in picard.illumina.parser">FourChannelIntensityData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/Range.html#hashCode--">hashCode()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/Range.html" title="class in picard.illumina.parser">Range</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadDescriptor.html#hashCode--">hashCode()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ReadDescriptor.html" title="class in picard.illumina.parser">ReadDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaLaneTileCode.html#hashCode--">hashCode()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaLaneTileCode.html" title="class in picard.illumina.parser.readers">TileMetricsOutReader.IlluminaLaneTileCode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaTileMetrics.html#hashCode--">hashCode()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaTileMetrics.html" title="class in picard.illumina.parser.readers">TileMetricsOutReader.IlluminaTileMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadStructure.html#hashCode--">hashCode()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ReadStructure.html" title="class in picard.illumina.parser">ReadStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/PhysicalLocationForMateCigar.html#hashCode--">hashCode()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/PhysicalLocationForMateCigar.html" title="class in picard.sam.markduplicates.util">PhysicalLocationForMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceStates.TruthAndCallStates.html#hashCode--">hashCode()</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceStates.TruthAndCallStates.html" title="class in picard.vcf">GenotypeConcordanceStates.TruthAndCallStates</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/PhysicalLocation.html#hasLocation--">hasLocation()</a></span> - Method in interface picard.sam.util.<a href="picard/sam/util/PhysicalLocation.html" title="interface in picard.sam.util">PhysicalLocation</a></dt>
<dd>
<div class="block">Default implementation of a method to check whether real location data has been set.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaDataProvider.html#hasNext--">hasNext()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/IlluminaDataProvider.html" title="class in picard.illumina.parser">IlluminaDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/MultiTileParser.html#hasNext--">hasNext()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/MultiTileParser.html" title="class in picard.illumina.parser">MultiTileParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/NewIlluminaDataProvider.html#hasNext--">hasNext()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/NewIlluminaDataProvider.html" title="class in picard.illumina.parser">NewIlluminaDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PerTileParser.html#hasNext--">hasNext()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/PerTileParser.html" title="class in picard.illumina.parser">PerTileParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.html#hasNext--">hasNext()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.html" title="class in picard.illumina.parser.readers">AbstractIlluminaPositionFileReader</a></dt>
<dd>
<div class="block">Return true if the file has more elements to return, false otherwise</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/BarcodeFileReader.html#hasNext--">hasNext()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/BarcodeFileReader.html" title="class in picard.illumina.parser.readers">BarcodeFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/BclReader.html#hasNext--">hasNext()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/BclReader.html" title="class in picard.illumina.parser.readers">BclReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/CbclReader.html#hasNext--">hasNext()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/CbclReader.html" title="class in picard.illumina.parser.readers">CbclReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/ClocsFileReader.html#hasNext--">hasNext()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/ClocsFileReader.html" title="class in picard.illumina.parser.readers">ClocsFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/EmpiricalPhasingMetricsOutReader.html#hasNext--">hasNext()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/EmpiricalPhasingMetricsOutReader.html" title="class in picard.illumina.parser.readers">EmpiricalPhasingMetricsOutReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/FilterFileReader.html#hasNext--">hasNext()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/FilterFileReader.html" title="class in picard.illumina.parser.readers">FilterFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/LocsFileReader.html#hasNext--">hasNext()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/LocsFileReader.html" title="class in picard.illumina.parser.readers">LocsFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/PosFileReader.html#hasNext--">hasNext()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/PosFileReader.html" title="class in picard.illumina.parser.readers">PosFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/TileMetricsOutReader.html#hasNext--">hasNext()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/TileMetricsOutReader.html" title="class in picard.illumina.parser.readers">TileMetricsOutReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigarIterator.html#hasNext--">hasNext()</a></span> - Method in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigarIterator.html" title="class in picard.sam.markduplicates">MarkDuplicatesWithMateCigarIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AsyncIterator.html#hasNext--">hasNext()</a></span> - Method in class picard.util.<a href="picard/util/AsyncIterator.html" title="class in picard.util">AsyncIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/DelimitedTextFileWithHeaderIterator.html#hasNext--">hasNext()</a></span> - Method in class picard.util.<a href="picard/util/DelimitedTextFileWithHeaderIterator.html" title="class in picard.util">DelimitedTextFileWithHeaderIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/ByIntervalListVariantContextIterator.html#hasNext--">hasNext()</a></span> - Method in class picard.vcf.<a href="picard/vcf/ByIntervalListVariantContextIterator.html" title="class in picard.vcf">ByIntervalListVariantContextIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/FilterApplyingVariantIterator.html#hasNext--">hasNext()</a></span> - Method in class picard.vcf.filter.<a href="picard/vcf/filter/FilterApplyingVariantIterator.html" title="class in picard.vcf.filter">FilterApplyingVariantIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/PairedVariantSubContextIterator.html#hasNext--">hasNext()</a></span> - Method in class picard.vcf.<a href="picard/vcf/PairedVariantSubContextIterator.html" title="class in picard.vcf">PairedVariantSubContextIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/util/PredicateFilterDecoratingClosableIterator.html#hasNext--">hasNext()</a></span> - Method in class picard.vcf.processor.util.<a href="picard/vcf/processor/util/PredicateFilterDecoratingClosableIterator.html" title="class in picard.vcf.processor.util">PredicateFilterDecoratingClosableIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#hasWebDocumentation-java.lang.Class-">hasWebDocumentation(Class&lt;?&gt;)</a></span> - Static method in class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>
<div class="block">Determine if a class has web documentation based on its package name</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.SamHeaderAndIterator.html#header">header</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.SamHeaderAndIterator.html" title="class in picard.sam.markduplicates.util">AbstractMarkDuplicatesCommandLineProgram.SamHeaderAndIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ReplaceSamHeader.html#HEADER">HEADER</a></span> - Variable in class picard.sam.<a href="picard/sam/ReplaceSamHeader.html" title="class in picard.sam">ReplaceSamHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/FixVcfHeader.html#HEADER">HEADER</a></span> - Variable in class picard.vcf.<a href="picard/vcf/FixVcfHeader.html" title="class in picard.vcf">FixVcfHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ViewSam.html#HEADER_ONLY">HEADER_ONLY</a></span> - Variable in class picard.sam.<a href="picard/sam/ViewSam.html" title="class in picard.sam">ViewSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/AlleleBalanceFilter.html#headerLines--">headerLines()</a></span> - Method in class picard.vcf.filter.<a href="picard/vcf/filter/AlleleBalanceFilter.html" title="class in picard.vcf.filter">AlleleBalanceFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/FisherStrandFilter.html#headerLines--">headerLines()</a></span> - Method in class picard.vcf.filter.<a href="picard/vcf/filter/FisherStrandFilter.html" title="class in picard.vcf.filter">FisherStrandFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/QdFilter.html#headerLines--">headerLines()</a></span> - Method in class picard.vcf.filter.<a href="picard/vcf/filter/QdFilter.html" title="class in picard.vcf.filter">QdFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/VariantFilter.html#headerLines--">headerLines()</a></span> - Method in interface picard.vcf.filter.<a href="picard/vcf/filter/VariantFilter.html" title="interface in picard.vcf.filter">VariantFilter</a></dt>
<dd>
<div class="block">Return VCF header lines that define filters that may be applied by the VariantFilter.</div>
</dd>
<dt><a href="picard/util/help/HelpConstants.html" title="class in picard.util.help"><span class="typeNameLink">HelpConstants</span></a> - Class in <a href="picard/util/help/package-summary.html">picard.util.help</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/FingerprintingSummaryMetrics.html#HET_AS_HOM">HET_AS_HOM</a></span> - Variable in class picard.analysis.<a href="picard/analysis/FingerprintingSummaryMetrics.html" title="class in picard.analysis">FingerprintingSummaryMetrics</a></dt>
<dd>
<div class="block">num of hets, observed as homs with LOD > threshold</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeMap.html#HET_GENOTYPE_FOR_PHASING">HET_GENOTYPE_FOR_PHASING</a></span> - Static variable in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeMap.html" title="class in picard.fingerprint">HaplotypeMap</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingDetailMetrics.html#HET_HOMVAR_RATIO">HET_HOMVAR_RATIO</a></span> - Variable in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingDetailMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingDetailMetrics</a></dt>
<dd>
<div class="block">(count of hets)/(count of homozygous non-ref) for this sample</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html#HET_PPV">HET_PPV</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html" title="class in picard.vcf">GenotypeConcordanceSummaryMetrics</a></dt>
<dd>
<div class="block">The ppv (positive predictive value) for all heterozygous variants (PPV is the TP / (TP + FP))</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html#HET_SENSITIVITY">HET_SENSITIVITY</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html" title="class in picard.vcf">GenotypeConcordanceSummaryMetrics</a></dt>
<dd>
<div class="block">The sensitivity for all heterozygous variants (Sensitivity is TP / (TP + FN))</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#HET_SNP_Q">HET_SNP_Q</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">The Phred Scaled Q Score of the theoretical HET SNP sensitivity.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#HET_SNP_Q">HET_SNP_Q</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The Phred Scaled Q Score of the theoretical HET SNP sensitivity.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#HET_SNP_Q">HET_SNP_Q</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The Q Score of the theoretical HET SNP sensitivity.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#HET_SNP_SENSITIVITY">HET_SNP_SENSITIVITY</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">The theoretical HET SNP sensitivity.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#HET_SNP_SENSITIVITY">HET_SNP_SENSITIVITY</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The theoretical HET SNP sensitivity.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#HET_SNP_SENSITIVITY">HET_SNP_SENSITIVITY</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The theoretical HET SNP sensitivity.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html#HET_SPECIFICITY">HET_SPECIFICITY</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html" title="class in picard.vcf">GenotypeConcordanceSummaryMetrics</a></dt>
<dd>
<div class="block">The specificity for all heterozygous variants cannot be calculated</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/TheoreticalSensitivity.html#hetAltDepthDistribution-int-">hetAltDepthDistribution(int)</a></span> - Static method in class picard.analysis.<a href="picard/analysis/TheoreticalSensitivity.html" title="class in picard.analysis">TheoreticalSensitivity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/TheoreticalSensitivity.html#hetSNPSensitivity-double:A-double:A-int-double-">hetSNPSensitivity(double[], double[], int, double)</a></span> - Static method in class picard.analysis.<a href="picard/analysis/TheoreticalSensitivity.html" title="class in picard.analysis">TheoreticalSensitivity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/TheoreticalSensitivity.html#hetSNPSensitivity-double:A-double:A-int-double-boolean-">hetSNPSensitivity(double[], double[], int, double, boolean)</a></span> - Static method in class picard.analysis.<a href="picard/analysis/TheoreticalSensitivity.html" title="class in picard.analysis">TheoreticalSensitivity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#highQualityDepthHistogram">highQualityDepthHistogram</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">Count of sites with a given depth of coverage.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AbstractWgsMetricsCollector.html#highQualityDepthHistogramArray">highQualityDepthHistogramArray</a></span> - Variable in class picard.analysis.<a href="picard/analysis/AbstractWgsMetricsCollector.html" title="class in picard.analysis">AbstractWgsMetricsCollector</a></dt>
<dd>
<div class="block">Count of sites with a given depth of coverage.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectInsertSizeMetrics.html#Histogram_FILE">Histogram_FILE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectInsertSizeMetrics.html" title="class in picard.analysis">CollectInsertSizeMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectInsertSizeMetrics.html#Histogram_R_SCRIPT">Histogram_R_SCRIPT</a></span> - Static variable in class picard.analysis.<a href="picard/analysis/CollectInsertSizeMetrics.html" title="class in picard.analysis">CollectInsertSizeMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectInsertSizeMetrics.html#HISTOGRAM_WIDTH">HISTOGRAM_WIDTH</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectInsertSizeMetrics.html" title="class in picard.analysis">CollectInsertSizeMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/FingerprintingSummaryMetrics.html#HOM_AS_HET">HOM_AS_HET</a></span> - Variable in class picard.analysis.<a href="picard/analysis/FingerprintingSummaryMetrics.html" title="class in picard.analysis">FingerprintingSummaryMetrics</a></dt>
<dd>
<div class="block">num of homs, observed as hets with LOD > threshold</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/FingerprintingSummaryMetrics.html#HOM_AS_OTHER_HOM">HOM_AS_OTHER_HOM</a></span> - Variable in class picard.analysis.<a href="picard/analysis/FingerprintingSummaryMetrics.html" title="class in picard.analysis">FingerprintingSummaryMetrics</a></dt>
<dd>
<div class="block">num of homs, observed as other homs with LOD > threshold</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html#HOMVAR_PPV">HOMVAR_PPV</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html" title="class in picard.vcf">GenotypeConcordanceSummaryMetrics</a></dt>
<dd>
<div class="block">The ppv (positive predictive value) for all homozygous variants (PPV is the TP / (TP + FP))</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html#HOMVAR_SENSITIVITY">HOMVAR_SENSITIVITY</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html" title="class in picard.vcf">GenotypeConcordanceSummaryMetrics</a></dt>
<dd>
<div class="block">The sensitivity for all homozygous variants (Sensitivity is TP / (TP + FN))</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html#HOMVAR_SPECIFICITY">HOMVAR_SPECIFICITY</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html" title="class in picard.vcf">GenotypeConcordanceSummaryMetrics</a></dt>
<dd>
<div class="block">The specificity for all homozygous variants cannot be calculated.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#HS_LIBRARY_SIZE">HS_LIBRARY_SIZE</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The estimated number of unique molecules in the selected part of the library.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#HS_PENALTY_100X">HS_PENALTY_100X</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The "hybrid selection penalty" incurred to get 80% of target bases to 100X.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#HS_PENALTY_10X">HS_PENALTY_10X</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The "hybrid selection penalty" incurred to get 80% of target bases to 10X.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#HS_PENALTY_20X">HS_PENALTY_20X</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The "hybrid selection penalty" incurred to get 80% of target bases to 20X.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#HS_PENALTY_30X">HS_PENALTY_30X</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The "hybrid selection penalty" incurred to get 80% of target bases to 30X.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#HS_PENALTY_40X">HS_PENALTY_40X</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The "hybrid selection penalty" incurred to get 80% of target bases to 40X.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#HS_PENALTY_50X">HS_PENALTY_50X</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The "hybrid selection penalty" incurred to get 80% of target bases to 50X.</div>
</dd>
<dt><a href="picard/analysis/directed/HsMetricCollector.html" title="class in picard.analysis.directed"><span class="typeNameLink">HsMetricCollector</span></a> - Class in <a href="picard/analysis/directed/package-summary.html">picard.analysis.directed</a></dt>
<dd>
<div class="block">Calculates HS metrics for a given SAM or BAM file.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetricCollector.html#HsMetricCollector-java.util.Set-java.util.List-htsjdk.samtools.reference.ReferenceSequenceFile-java.io.File-java.io.File-htsjdk.samtools.util.IntervalList-htsjdk.samtools.util.IntervalList-java.lang.String-int-int-int-boolean-int-int-">HsMetricCollector(Set&lt;MetricAccumulationLevel&gt;, List&lt;SAMReadGroupRecord&gt;, ReferenceSequenceFile, File, File, IntervalList, IntervalList, String, int, int, int, boolean, int, int)</a></span> - Constructor for class picard.analysis.directed.<a href="picard/analysis/directed/HsMetricCollector.html" title="class in picard.analysis.directed">HsMetricCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetricCollector.html#HsMetricCollector-java.util.Set-java.util.List-htsjdk.samtools.reference.ReferenceSequenceFile-java.io.File-java.io.File-htsjdk.samtools.util.IntervalList-htsjdk.samtools.util.IntervalList-java.lang.String-int-int-int-boolean-boolean-int-int-">HsMetricCollector(Set&lt;MetricAccumulationLevel&gt;, List&lt;SAMReadGroupRecord&gt;, ReferenceSequenceFile, File, File, IntervalList, IntervalList, String, int, int, int, boolean, boolean, int, int)</a></span> - Constructor for class picard.analysis.directed.<a href="picard/analysis/directed/HsMetricCollector.html" title="class in picard.analysis.directed">HsMetricCollector</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed"><span class="typeNameLink">HsMetrics</span></a> - Class in <a href="picard/analysis/directed/package-summary.html">picard.analysis.directed</a></dt>
<dd>
<div class="block">Metrics generated by CollectHsMetrics for the analysis of target-capture sequencing experiments.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#HsMetrics--">HsMetrics()</a></span> - Constructor for class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="I:I">
<!--   -->
</a>
<h2 class="title">I</h2>
<dl>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintChecker.html#identifyContaminant-java.io.File-double-int-">identifyContaminant(File, double, int)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/FingerprintChecker.html" title="class in picard.fingerprint">FingerprintChecker</a></dt>
<dd>
<div class="block">Generates a per-sample Fingerprint for the contaminant in the supplied SAM file.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ValidateSamFile.html#IGNORE">IGNORE</a></span> - Variable in class picard.sam.<a href="picard/sam/ValidateSamFile.html" title="class in picard.sam">ValidateSamFile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#IGNORE_FILTER_STATUS">IGNORE_FILTER_STATUS</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FixMateInformation.html#IGNORE_MISSING_MATES">IGNORE_MISSING_MATES</a></span> - Variable in class picard.sam.<a href="picard/sam/FixMateInformation.html" title="class in picard.sam">FixMateInformation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CheckFingerprint.html#IGNORE_READ_GROUPS">IGNORE_READ_GROUPS</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CheckFingerprint.html" title="class in picard.fingerprint">CheckFingerprint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRnaSeqMetrics.html#IGNORE_SEQUENCE">IGNORE_SEQUENCE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectRnaSeqMetrics.html" title="class in picard.analysis">CollectRnaSeqMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToFastq.html#IGNORE_UNEXPECTED_BARCODES">IGNORE_UNEXPECTED_BARCODES</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToFastq.html" title="class in picard.illumina">IlluminaBasecallsToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#IGNORE_UNEXPECTED_BARCODES">IGNORE_UNEXPECTED_BARCODES</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ValidateSamFile.html#IGNORE_WARNINGS">IGNORE_WARNINGS</a></span> - Variable in class picard.sam.<a href="picard/sam/ValidateSamFile.html" title="class in picard.sam">ValidateSamFile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.html#ignoreAlignment-htsjdk.samtools.SAMRecord-">ignoreAlignment(SAMRecord)</a></span> - Method in class picard.sam.<a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamAlignmentMerger.html#ignoreAlignment-htsjdk.samtools.SAMRecord-">ignoreAlignment(SAMRecord)</a></span> - Method in class picard.sam.<a href="picard/sam/SamAlignmentMerger.html" title="class in picard.sam">SamAlignmentMerger</a></dt>
<dd>
<div class="block">For now, we only ignore those alignments that have more than <a href="picard/sam/SamAlignmentMerger.html#maxGaps"><code>SamAlignmentMerger.maxGaps</code></a> insertions
 or deletions.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RnaSeqMetrics.html#IGNORED_READS">IGNORED_READS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RnaSeqMetrics.html" title="class in picard.analysis">RnaSeqMetrics</a></dt>
<dd>
<div class="block">Number of primary alignments that are mapped to a sequence specified on command-line as IGNORED_SEQUENCE.</div>
</dd>
<dt><a href="picard/cmdline/programgroups/Illumina.html" title="class in picard.cmdline.programgroups"><span class="typeNameLink">Illumina</span></a> - Class in <a href="picard/cmdline/programgroups/package-summary.html">picard.cmdline.programgroups</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/programgroups/Illumina.html#Illumina--">Illumina()</a></span> - Constructor for class picard.cmdline.programgroups.<a href="picard/cmdline/programgroups/Illumina.html" title="class in picard.cmdline.programgroups">Illumina</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/BclQualityEvaluationStrategy.html#ILLUMINA_ALLEGED_MINIMUM_QUALITY">ILLUMINA_ALLEGED_MINIMUM_QUALITY</a></span> - Static variable in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/BclQualityEvaluationStrategy.html" title="class in picard.illumina.parser.readers">BclQualityEvaluationStrategy</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/IlluminaBasecallingMetrics.html" title="class in picard.illumina"><span class="typeNameLink">IlluminaBasecallingMetrics</span></a> - Class in <a href="picard/illumina/package-summary.html">picard.illumina</a></dt>
<dd>
<div class="block">Metric for Illumina Basecalling that stores means and standard deviations on a per-barcode per-lane basis.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallingMetrics.html#IlluminaBasecallingMetrics--">IlluminaBasecallingMetrics()</a></span> - Constructor for class picard.illumina.<a href="picard/illumina/IlluminaBasecallingMetrics.html" title="class in picard.illumina">IlluminaBasecallingMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/IlluminaBasecallsConverter.html" title="class in picard.illumina"><span class="typeNameLink">IlluminaBasecallsConverter</span></a>&lt;<a href="picard/illumina/IlluminaBasecallsConverter.html" title="type parameter in IlluminaBasecallsConverter">CLUSTER_OUTPUT_RECORD</a>&gt; - Class in <a href="picard/illumina/package-summary.html">picard.illumina</a></dt>
<dd>
<div class="block">Manages the conversion of Illumina basecalls into some output format.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsConverter.html#IlluminaBasecallsConverter-java.io.File-int-picard.illumina.parser.ReadStructure-java.util.Map-boolean-int-java.util.List-int-boolean-java.lang.Integer-java.lang.Integer-java.util.Comparator-htsjdk.samtools.util.SortingCollection.Codec-java.lang.Class-picard.illumina.parser.readers.BclQualityEvaluationStrategy-boolean-boolean-boolean-">IlluminaBasecallsConverter(File, int, ReadStructure, Map&lt;String, ? extends BasecallsConverter.ConvertedClusterDataWriter&lt;CLUSTER_OUTPUT_RECORD&gt;&gt;, boolean, int, List&lt;File&gt;, int, boolean, Integer, Integer, Comparator&lt;CLUSTER_OUTPUT_RECORD&gt;, SortingCollection.Codec&lt;CLUSTER_OUTPUT_RECORD&gt;, Class&lt;CLUSTER_OUTPUT_RECORD&gt;, BclQualityEvaluationStrategy, boolean, boolean, boolean)</a></span> - Constructor for class picard.illumina.<a href="picard/illumina/IlluminaBasecallsConverter.html" title="class in picard.illumina">IlluminaBasecallsConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsConverter.html#IlluminaBasecallsConverter-java.io.File-java.io.File-int-picard.illumina.parser.ReadStructure-java.util.Map-boolean-int-java.util.List-int-boolean-java.lang.Integer-java.lang.Integer-java.util.Comparator-htsjdk.samtools.util.SortingCollection.Codec-java.lang.Class-picard.illumina.parser.readers.BclQualityEvaluationStrategy-boolean-boolean-boolean-">IlluminaBasecallsConverter(File, File, int, ReadStructure, Map&lt;String, ? extends BasecallsConverter.ConvertedClusterDataWriter&lt;CLUSTER_OUTPUT_RECORD&gt;&gt;, boolean, int, List&lt;File&gt;, int, boolean, Integer, Integer, Comparator&lt;CLUSTER_OUTPUT_RECORD&gt;, SortingCollection.Codec&lt;CLUSTER_OUTPUT_RECORD&gt;, Class&lt;CLUSTER_OUTPUT_RECORD&gt;, BclQualityEvaluationStrategy, boolean, boolean, boolean)</a></span> - Constructor for class picard.illumina.<a href="picard/illumina/IlluminaBasecallsConverter.html" title="class in picard.illumina">IlluminaBasecallsConverter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/IlluminaBasecallsToFastq.html" title="class in picard.illumina"><span class="typeNameLink">IlluminaBasecallsToFastq</span></a> - Class in <a href="picard/illumina/package-summary.html">picard.illumina</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToFastq.html#IlluminaBasecallsToFastq--">IlluminaBasecallsToFastq()</a></span> - Constructor for class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToFastq.html" title="class in picard.illumina">IlluminaBasecallsToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/IlluminaBasecallsToFastq.ReadNameFormat.html" title="enum in picard.illumina"><span class="typeNameLink">IlluminaBasecallsToFastq.ReadNameFormat</span></a> - Enum in <a href="picard/illumina/package-summary.html">picard.illumina</a></dt>
<dd>
<div class="block">Simple switch to control the read name format to emit.</div>
</dd>
<dt><a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina"><span class="typeNameLink">IlluminaBasecallsToSam</span></a> - Class in <a href="picard/illumina/package-summary.html">picard.illumina</a></dt>
<dd>
<div class="block">IlluminaBasecallsToSam transforms a lane of Illumina data file formats (bcl, locs, clocs, qseqs, etc.) into
 SAM or BAM file format.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#IlluminaBasecallsToSam--">IlluminaBasecallsToSam()</a></span> - Constructor for class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/IlluminaDataProvider.html" title="class in picard.illumina.parser"><span class="typeNameLink">IlluminaDataProvider</span></a> - Class in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>
<div class="block">Parse various formats and versions of Illumina Basecall files, and use them the to populate
 ClusterData objects.</div>
</dd>
<dt><a href="picard/illumina/parser/IlluminaDataProviderFactory.html" title="class in picard.illumina.parser"><span class="typeNameLink">IlluminaDataProviderFactory</span></a> - Class in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>
<div class="block">IlluminaDataProviderFactory accepts options for parsing Illumina data files for a lane and creates an
 IlluminaDataProvider, an iterator over the ClusterData for that lane, which utilizes these options.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaDataProviderFactory.html#IlluminaDataProviderFactory-java.io.File-int-picard.illumina.parser.ReadStructure-picard.illumina.parser.readers.BclQualityEvaluationStrategy-picard.illumina.parser.IlluminaDataType...-">IlluminaDataProviderFactory(File, int, ReadStructure, BclQualityEvaluationStrategy, IlluminaDataType...)</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/IlluminaDataProviderFactory.html" title="class in picard.illumina.parser">IlluminaDataProviderFactory</a></dt>
<dd>
<div class="block">Create factory with the specified options, one that favors using QSeqs over all other files</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaDataProviderFactory.html#IlluminaDataProviderFactory-java.io.File-java.io.File-int-picard.illumina.parser.ReadStructure-picard.illumina.parser.readers.BclQualityEvaluationStrategy-picard.illumina.parser.IlluminaDataType...-">IlluminaDataProviderFactory(File, File, int, ReadStructure, BclQualityEvaluationStrategy, IlluminaDataType...)</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/IlluminaDataProviderFactory.html" title="class in picard.illumina.parser">IlluminaDataProviderFactory</a></dt>
<dd>
<div class="block">Create factory with the specified options, one that favors using QSeqs over all other files</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaDataProviderFactory.html#IlluminaDataProviderFactory-java.io.File-java.io.File-int-picard.illumina.parser.ReadStructure-picard.illumina.parser.readers.BclQualityEvaluationStrategy-">IlluminaDataProviderFactory(File, File, int, ReadStructure, BclQualityEvaluationStrategy)</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/IlluminaDataProviderFactory.html" title="class in picard.illumina.parser">IlluminaDataProviderFactory</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/IlluminaDataType.html" title="enum in picard.illumina.parser"><span class="typeNameLink">IlluminaDataType</span></a> - Enum in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>
<div class="block">List of data types of interest when parsing Illumina data.</div>
</dd>
<dt><a href="picard/illumina/parser/IlluminaFileUtil.html" title="class in picard.illumina.parser"><span class="typeNameLink">IlluminaFileUtil</span></a> - Class in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>
<div class="block">General utils for dealing with IlluminaFiles as well as utils for specific, support formats.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaFileUtil.html#IlluminaFileUtil-java.io.File-int-">IlluminaFileUtil(File, int)</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/IlluminaFileUtil.html" title="class in picard.illumina.parser">IlluminaFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaFileUtil.html#IlluminaFileUtil-java.io.File-java.io.File-int-">IlluminaFileUtil(File, File, int)</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/IlluminaFileUtil.html" title="class in picard.illumina.parser">IlluminaFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/IlluminaFileUtil.SupportedIlluminaFormat.html" title="enum in picard.illumina.parser"><span class="typeNameLink">IlluminaFileUtil.SupportedIlluminaFormat</span></a> - Enum in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/IlluminaLaneMetrics.html" title="class in picard.illumina"><span class="typeNameLink">IlluminaLaneMetrics</span></a> - Class in <a href="picard/illumina/package-summary.html">picard.illumina</a></dt>
<dd>
<div class="block">Embodies characteristics that describe a lane.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaLaneMetrics.html#IlluminaLaneMetrics--">IlluminaLaneMetrics()</a></span> - Constructor for class picard.illumina.<a href="picard/illumina/IlluminaLaneMetrics.html" title="class in picard.illumina">IlluminaLaneMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CollectIlluminaLaneMetrics.IlluminaLaneMetricsCollector.html#IlluminaLaneMetricsCollector--">IlluminaLaneMetricsCollector()</a></span> - Constructor for class picard.illumina.<a href="picard/illumina/CollectIlluminaLaneMetrics.IlluminaLaneMetricsCollector.html" title="class in picard.illumina">CollectIlluminaLaneMetrics.IlluminaLaneMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaLaneTileCode.html#IlluminaLaneTileCode-int-int-int-">IlluminaLaneTileCode(int, int, int)</a></span> - Constructor for class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaLaneTileCode.html" title="class in picard.illumina.parser.readers">TileMetricsOutReader.IlluminaLaneTileCode</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/IlluminaMetricsCode.html" title="enum in picard.illumina.parser"><span class="typeNameLink">IlluminaMetricsCode</span></a> - Enum in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>
<div class="block">Illumina's TileMetricsOut.bin file codes various metrics, both concrete (all density id's are code 100) or as a base code
 (e.g.</div>
</dd>
<dt><a href="picard/illumina/IlluminaPhasingMetrics.html" title="class in picard.illumina"><span class="typeNameLink">IlluminaPhasingMetrics</span></a> - Class in <a href="picard/illumina/package-summary.html">picard.illumina</a></dt>
<dd>
<div class="block">Metrics for Illumina Basecalling that stores median phasing and prephasing percentages on a per-template-read, per-lane basis.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaPhasingMetrics.html#IlluminaPhasingMetrics--">IlluminaPhasingMetrics()</a></span> - Constructor for class picard.illumina.<a href="picard/illumina/IlluminaPhasingMetrics.html" title="class in picard.illumina">IlluminaPhasingMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/fastq/IlluminaReadNameEncoder.html" title="class in picard.fastq"><span class="typeNameLink">IlluminaReadNameEncoder</span></a> - Class in <a href="picard/fastq/package-summary.html">picard.fastq</a></dt>
<dd>
<div class="block">A read name encoder following the encoding initially produced by picard fastq writers.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fastq/IlluminaReadNameEncoder.html#IlluminaReadNameEncoder-java.lang.String-">IlluminaReadNameEncoder(String)</a></span> - Constructor for class picard.fastq.<a href="picard/fastq/IlluminaReadNameEncoder.html" title="class in picard.fastq">IlluminaReadNameEncoder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaTileMetrics.html#IlluminaTileMetrics-java.nio.ByteBuffer-picard.illumina.parser.readers.TileMetricsOutReader.TileMetricsVersion-">IlluminaTileMetrics(ByteBuffer, TileMetricsOutReader.TileMetricsVersion)</a></span> - Constructor for class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaTileMetrics.html" title="class in picard.illumina.parser.readers">TileMetricsOutReader.IlluminaTileMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaTileMetrics.html#IlluminaTileMetrics-int-int-int-float-">IlluminaTileMetrics(int, int, int, float)</a></span> - Constructor for class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaTileMetrics.html" title="class in picard.illumina.parser.readers">TileMetricsOutReader.IlluminaTileMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/IlluminaUtil.html" title="class in picard.util"><span class="typeNameLink">IlluminaUtil</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">Misc utilities for working with Illumina specific files and data</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/IlluminaUtil.html#IlluminaUtil--">IlluminaUtil()</a></span> - Constructor for class picard.util.<a href="picard/util/IlluminaUtil.html" title="class in picard.util">IlluminaUtil</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/IlluminaUtil.IlluminaAdapterPair.html" title="enum in picard.util"><span class="typeNameLink">IlluminaUtil.IlluminaAdapterPair</span></a> - Enum in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">Describes adapters used on each pair of strands</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesCodec.html#in">in</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesCodec.html" title="class in picard.sam.markduplicates.util">ReadEndsForMarkDuplicatesCodec</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/RepresentativeReadIndexerCodec.html#in">in</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/RepresentativeReadIndexerCodec.html" title="class in picard.sam.markduplicates.util">RepresentativeReadIndexerCodec</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.html#INCLUDE_BQ_HISTOGRAM">INCLUDE_BQ_HISTOGRAM</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html#INCLUDE_DUPLICATES">INCLUDE_DUPLICATES</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html" title="class in picard.analysis.artifacts">CollectSequencingArtifactMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectInsertSizeMetrics.html#INCLUDE_DUPLICATES">INCLUDE_DUPLICATES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectInsertSizeMetrics.html" title="class in picard.analysis">CollectInsertSizeMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IntervalListTools.html#INCLUDE_FILTERED">INCLUDE_FILTERED</a></span> - Variable in class picard.util.<a href="picard/util/IntervalListTools.html" title="class in picard.util">IntervalListTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/VcfToIntervalList.html#INCLUDE_FILTERED">INCLUDE_FILTERED</a></span> - Variable in class picard.vcf.<a href="picard/vcf/VcfToIntervalList.html" title="class in picard.vcf">VcfToIntervalList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/VcfToIntervalList.html#INCLUDE_FILTERED_SHORT_NAME">INCLUDE_FILTERED_SHORT_NAME</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/VcfToIntervalList.html" title="class in picard.vcf">VcfToIntervalList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/QualityScoreDistribution.html#INCLUDE_NO_CALLS">INCLUDE_NO_CALLS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/QualityScoreDistribution.html" title="class in picard.analysis">QualityScoreDistribution</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html#INCLUDE_NON_PF_READS">INCLUDE_NON_PF_READS</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html" title="class in picard.analysis.artifacts">CollectSequencingArtifactMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.html#INCLUDE_NON_PF_READS">INCLUDE_NON_PF_READS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.html" title="class in picard.analysis">CollectOxoGMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fastq/BamToBfq.html#INCLUDE_NON_PF_READS">INCLUDE_NON_PF_READS</a></span> - Variable in class picard.fastq.<a href="picard/fastq/BamToBfq.html" title="class in picard.fastq">BamToBfq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToFastq.html#INCLUDE_NON_PF_READS">INCLUDE_NON_PF_READS</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToFastq.html" title="class in picard.illumina">IlluminaBasecallsToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#INCLUDE_NON_PF_READS">INCLUDE_NON_PF_READS</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamToFastq.html#INCLUDE_NON_PF_READS">INCLUDE_NON_PF_READS</a></span> - Variable in class picard.sam.<a href="picard/sam/SamToFastq.html" title="class in picard.sam">SamToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamToFastq.html#INCLUDE_NON_PRIMARY_ALIGNMENTS">INCLUDE_NON_PRIMARY_ALIGNMENTS</a></span> - Variable in class picard.sam.<a href="picard/sam/SamToFastq.html" title="class in picard.sam">SamToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectQualityYieldMetrics.html#INCLUDE_SECONDARY_ALIGNMENTS">INCLUDE_SECONDARY_ALIGNMENTS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectQualityYieldMetrics.html" title="class in picard.analysis">CollectQualityYieldMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#INCLUDE_SECONDARY_ALIGNMENTS">INCLUDE_SECONDARY_ALIGNMENTS</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectQualityYieldMetrics.html#INCLUDE_SUPPLEMENTAL_ALIGNMENTS">INCLUDE_SUPPLEMENTAL_ALIGNMENTS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectQualityYieldMetrics.html" title="class in picard.analysis">CollectQualityYieldMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html#INCLUDE_UNPAIRED">INCLUDE_UNPAIRED</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html" title="class in picard.analysis.artifacts">CollectSequencingArtifactMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectMultipleMetrics.html#INCLUDE_UNPAIRED">INCLUDE_UNPAIRED</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectMultipleMetrics.html" title="class in picard.analysis">CollectMultipleMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetricsCollector.html#includeSupplementalAlignments">includeSupplementalAlignments</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetricsCollector.html" title="class in picard.analysis">CollectQualityYieldMetrics.QualityYieldMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RnaSeqMetrics.html#INCORRECT_STRAND_READS">INCORRECT_STRAND_READS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RnaSeqMetrics.html" title="class in picard.analysis">RnaSeqMetrics</a></dt>
<dd>
<div class="block">Number of aligned reads that are mapped to the incorrect strand.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CounterManager.Counter.html#increment-int-">increment(int)</a></span> - Method in class picard.analysis.<a href="picard/analysis/CounterManager.Counter.html" title="class in picard.analysis">CounterManager.Counter</a></dt>
<dd>
<div class="block">Increments value corresponding to reference sequence index</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceCounts.html#increment-picard.vcf.GenotypeConcordanceStates.TruthAndCallStates-">increment(GenotypeConcordanceStates.TruthAndCallStates)</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceCounts.html" title="class in picard.vcf">GenotypeConcordanceCounts</a></dt>
<dd>
<div class="block">Increments a count for the truth/call state tuple.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceCounts.html#increment-picard.vcf.GenotypeConcordanceStates.TruthAndCallStates-double-">increment(GenotypeConcordanceStates.TruthAndCallStates, double)</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceCounts.html" title="class in picard.vcf">GenotypeConcordanceCounts</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetMetricsCollector.Coverage.html#incrementReadCount--">incrementReadCount()</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/TargetMetricsCollector.Coverage.html" title="class in picard.analysis.directed">TargetMetricsCollector.Coverage</a></dt>
<dd>
<div class="block">Increments the # of reads mapping to this target.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/SplitVcfs.html#INDEL_OUTPUT">INDEL_OUTPUT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/SplitVcfs.html" title="class in picard.vcf">SplitVcfs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#indelCounter">indelCounter</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/DbSnpBitSetUtil.DbSnpBitSets.html#indels">indels</a></span> - Variable in class picard.util.<a href="picard/util/DbSnpBitSetUtil.DbSnpBitSets.html" title="class in picard.util">DbSnpBitSetUtil.DbSnpBitSets</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates"><span class="typeNameLink">IndependentReplicateMetric</span></a> - Class in <a href="picard/analysis/replicates/package-summary.html">picard.analysis.replicates</a></dt>
<dd>
<div class="block">A class to store information relevant for biological rate estimation</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#IndependentReplicateMetric--">IndependentReplicateMetric()</a></span> - Constructor for class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#independentReplicationRateFromBiDups">independentReplicationRateFromBiDups</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#independentReplicationRateFromTriDups">independentReplicationRateFromTriDups</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#independentReplicationRateFromUmi">independentReplicationRateFromUmi</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ValidateSamFile.html#INDEX_VALIDATION_STRINGENCY">INDEX_VALIDATION_STRINGENCY</a></span> - Variable in class picard.sam.<a href="picard/sam/ValidateSamFile.html" title="class in picard.sam">ValidateSamFile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#indexOfMax-double:A-">indexOfMax(double[])</a></span> - Static method in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>
<div class="block">Returns the index of the largest element in the array.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#indexOfMax-long:A-">indexOfMax(long[])</a></span> - Static method in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>
<div class="block">Returns the index of the largest element in the array.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#indexOfMin-int:A-">indexOfMin(int[])</a></span> - Static method in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>
<div class="block">Returns the index of the smallest element in the array.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#indexOfMin-double:A-">indexOfMin(double[])</a></span> - Static method in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>
<div class="block">Returns the index of the smallest element in the array.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasSummaryMetrics.html#inferArtifactName--">inferArtifactName()</a></span> - Method in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasSummaryMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.BaitBiasSummaryMetrics</a></dt>
<dd>
<div class="block">Label the artifacts corresponding to known error modes.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterSummaryMetrics.html#inferArtifactName--">inferArtifactName()</a></span> - Method in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterSummaryMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.PreAdapterSummaryMetrics</a></dt>
<dd>
<div class="block">Label the artifacts corresponding to known error modes.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/UmiMetrics.html#INFERRED_UMI_ENTROPY">INFERRED_UMI_ENTROPY</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/UmiMetrics.html" title="class in picard.sam.markduplicates">UmiMetrics</a></dt>
<dd>
<div class="block">Entropy (in base 4) of the inferred UMI sequences, indicating the
 effective number of bases in the inferred UMIs.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/UmiMetrics.html#INFERRED_UNIQUE_UMIS">INFERRED_UNIQUE_UMIS</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/UmiMetrics.html" title="class in picard.sam.markduplicates">UmiMetrics</a></dt>
<dd>
<div class="block">Number of different inferred UMI sequences derived</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/MultiTileBclParser.html#initialize--">initialize()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/MultiTileBclParser.html" title="class in picard.illumina.parser">MultiTileBclParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GA4GHScheme.html#initiateScheme--">initiateScheme()</a></span> - Method in class picard.vcf.<a href="picard/vcf/GA4GHScheme.html" title="class in picard.vcf">GA4GHScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GA4GHSchemeWithMissingAsHomRef.html#initiateScheme--">initiateScheme()</a></span> - Method in class picard.vcf.<a href="picard/vcf/GA4GHSchemeWithMissingAsHomRef.html" title="class in picard.vcf">GA4GHSchemeWithMissingAsHomRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceScheme.html#initiateScheme--">initiateScheme()</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceScheme.html" title="class in picard.vcf">GenotypeConcordanceScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectJumpingLibraryMetrics.html#INPUT">INPUT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectJumpingLibraryMetrics.html" title="class in picard.analysis">CollectJumpingLibraryMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectMultipleMetrics.html#INPUT">INPUT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectMultipleMetrics.html" title="class in picard.analysis">CollectMultipleMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.html#INPUT">INPUT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.html" title="class in picard.analysis">CollectOxoGMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRrbsMetrics.html#INPUT">INPUT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectRrbsMetrics.html" title="class in picard.analysis">CollectRrbsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.html#INPUT">INPUT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/CollectTargetedMetrics.html#INPUT">INPUT</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/CollectTargetedMetrics.html" title="class in picard.analysis.directed">CollectTargetedMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/CollectIndependentReplicateMetrics.html#INPUT">INPUT</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/CollectIndependentReplicateMetrics.html" title="class in picard.analysis.replicates">CollectIndependentReplicateMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/SinglePassSamProgram.html#INPUT">INPUT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/SinglePassSamProgram.html" title="class in picard.analysis">SinglePassSamProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fastq/BamToBfq.html#INPUT">INPUT</a></span> - Variable in class picard.fastq.<a href="picard/fastq/BamToBfq.html" title="class in picard.fastq">BamToBfq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CheckFingerprint.html#INPUT">INPUT</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CheckFingerprint.html" title="class in picard.fingerprint">CheckFingerprint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/ClusterCrosscheckMetrics.html#INPUT">INPUT</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/ClusterCrosscheckMetrics.html" title="class in picard.fingerprint">ClusterCrosscheckMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckFingerprints.html#INPUT">INPUT</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckFingerprints.html" title="class in picard.fingerprint">CrosscheckFingerprints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CollectIlluminaBasecallingMetrics.html#INPUT">INPUT</a></span> - Variable in class picard.illumina.<a href="picard/illumina/CollectIlluminaBasecallingMetrics.html" title="class in picard.illumina">CollectIlluminaBasecallingMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/MarkIlluminaAdapters.html#INPUT">INPUT</a></span> - Variable in class picard.illumina.<a href="picard/illumina/MarkIlluminaAdapters.html" title="class in picard.illumina">MarkIlluminaAdapters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/reference/NonNFastaSize.html#INPUT">INPUT</a></span> - Variable in class picard.reference.<a href="picard/reference/NonNFastaSize.html" title="class in picard.reference">NonNFastaSize</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/reference/NormalizeFasta.html#INPUT">INPUT</a></span> - Variable in class picard.reference.<a href="picard/reference/NormalizeFasta.html" title="class in picard.reference">NormalizeFasta</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AddCommentsToBam.html#INPUT">INPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/AddCommentsToBam.html" title="class in picard.sam">AddCommentsToBam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AddOrReplaceReadGroups.html#INPUT">INPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/AddOrReplaceReadGroups.html" title="class in picard.sam">AddOrReplaceReadGroups</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/BamIndexStats.html#INPUT">INPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/BamIndexStats.html" title="class in picard.sam">BamIndexStats</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/BuildBamIndex.html#INPUT">INPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/BuildBamIndex.html" title="class in picard.sam">BuildBamIndex</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CalculateReadGroupChecksum.html#INPUT">INPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/CalculateReadGroupChecksum.html" title="class in picard.sam">CalculateReadGroupChecksum</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CheckTerminatorBlock.html#INPUT">INPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/CheckTerminatorBlock.html" title="class in picard.sam">CheckTerminatorBlock</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CleanSam.html#INPUT">INPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/CleanSam.html" title="class in picard.sam">CleanSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/DownsampleSam.html#INPUT">INPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/DownsampleSam.html" title="class in picard.sam">DownsampleSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FilterSamReads.html#INPUT">INPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/FilterSamReads.html" title="class in picard.sam">FilterSamReads</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FixMateInformation.html#INPUT">INPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/FixMateInformation.html" title="class in picard.sam">FixMateInformation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/GatherBamFiles.html#INPUT">INPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/GatherBamFiles.html" title="class in picard.sam">GatherBamFiles</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/EstimateLibraryComplexity.html#INPUT">INPUT</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/EstimateLibraryComplexity.html" title="class in picard.sam.markduplicates">EstimateLibraryComplexity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html#INPUT">INPUT</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html" title="class in picard.sam.markduplicates.util">AbstractMarkDuplicatesCommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeSamFiles.html#INPUT">INPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeSamFiles.html" title="class in picard.sam">MergeSamFiles</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/PositionBasedDownsampleSam.html#INPUT">INPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/PositionBasedDownsampleSam.html" title="class in picard.sam">PositionBasedDownsampleSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ReorderSam.html#INPUT">INPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/ReorderSam.html" title="class in picard.sam">ReorderSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ReplaceSamHeader.html#INPUT">INPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/ReplaceSamHeader.html" title="class in picard.sam">ReplaceSamHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertOriginalBaseQualitiesAndAddMateCigar.html#INPUT">INPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/RevertOriginalBaseQualitiesAndAddMateCigar.html" title="class in picard.sam">RevertOriginalBaseQualitiesAndAddMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertSam.html#INPUT">INPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/RevertSam.html" title="class in picard.sam">RevertSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamFormatConverter.html#INPUT">INPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/SamFormatConverter.html" title="class in picard.sam">SamFormatConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamToFastq.html#INPUT">INPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/SamToFastq.html" title="class in picard.sam">SamToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SetNmMdAndUqTags.html#INPUT">INPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/SetNmMdAndUqTags.html" title="class in picard.sam">SetNmMdAndUqTags</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SortSam.html#INPUT">INPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/SortSam.html" title="class in picard.sam">SortSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SplitSamByLibrary.html#INPUT">INPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/SplitSamByLibrary.html" title="class in picard.sam">SplitSamByLibrary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ValidateSamFile.html#INPUT">INPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/ValidateSamFile.html" title="class in picard.sam">ValidateSamFile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ViewSam.html#INPUT">INPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/ViewSam.html" title="class in picard.sam">ViewSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/BedToIntervalList.html#INPUT">INPUT</a></span> - Variable in class picard.util.<a href="picard/util/BedToIntervalList.html" title="class in picard.util">BedToIntervalList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IntervalListToBed.html#INPUT">INPUT</a></span> - Variable in class picard.util.<a href="picard/util/IntervalListToBed.html" title="class in picard.util">IntervalListToBed</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IntervalListTools.html#INPUT">INPUT</a></span> - Variable in class picard.util.<a href="picard/util/IntervalListTools.html" title="class in picard.util">IntervalListTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/LiftOverIntervalList.html#INPUT">INPUT</a></span> - Variable in class picard.util.<a href="picard/util/LiftOverIntervalList.html" title="class in picard.util">LiftOverIntervalList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/AccumulateVariantCallingMetrics.html#INPUT">INPUT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/AccumulateVariantCallingMetrics.html" title="class in picard.vcf">AccumulateVariantCallingMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.html#INPUT">INPUT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/FilterVcf.html#INPUT">INPUT</a></span> - Variable in class picard.vcf.filter.<a href="picard/vcf/filter/FilterVcf.html" title="class in picard.vcf.filter">FilterVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/FixVcfHeader.html#INPUT">INPUT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/FixVcfHeader.html" title="class in picard.vcf">FixVcfHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GatherVcfs.html#INPUT">INPUT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GatherVcfs.html" title="class in picard.vcf">GatherVcfs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/LiftoverVcf.html#INPUT">INPUT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/LiftoverVcf.html" title="class in picard.vcf">LiftoverVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MakeSitesOnlyVcf.html#INPUT">INPUT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/MakeSitesOnlyVcf.html" title="class in picard.vcf">MakeSitesOnlyVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/FindMendelianViolations.html#INPUT">INPUT</a></span> - Variable in class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/FindMendelianViolations.html" title="class in picard.vcf.MendelianViolations">FindMendelianViolations</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MergeVcfs.html#INPUT">INPUT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/MergeVcfs.html" title="class in picard.vcf">MergeVcfs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/RenameSampleInVcf.html#INPUT">INPUT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/RenameSampleInVcf.html" title="class in picard.vcf">RenameSampleInVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/SortVcf.html#INPUT">INPUT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/SortVcf.html" title="class in picard.vcf">SortVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/SplitVcfs.html#INPUT">INPUT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/SplitVcfs.html" title="class in picard.vcf">SplitVcfs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/UpdateVcfSequenceDictionary.html#INPUT">INPUT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/UpdateVcfSequenceDictionary.html" title="class in picard.vcf">UpdateVcfSequenceDictionary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/VcfFormatConverter.html#INPUT">INPUT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/VcfFormatConverter.html" title="class in picard.vcf">VcfFormatConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/VcfToIntervalList.html#INPUT">INPUT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/VcfToIntervalList.html" title="class in picard.vcf">VcfToIntervalList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/ConvertSequencingArtifactToOxoG.html#INPUT_BASE">INPUT_BASE</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/ConvertSequencingArtifactToOxoG.html" title="class in picard.analysis.artifacts">ConvertSequencingArtifactToOxoG</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/StandardOptionDefinitions.html#INPUT_SHORT_NAME">INPUT_SHORT_NAME</a></span> - Static variable in class picard.cmdline.<a href="picard/cmdline/StandardOptionDefinitions.html" title="class in picard.cmdline">StandardOptionDefinitions</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/InsertSizeMetrics.html" title="class in picard.analysis"><span class="typeNameLink">InsertSizeMetrics</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Metrics about the insert size distribution of a paired-end library, created by the
 CollectInsertSizeMetrics program and usually written to a file with the extension
 ".insert_size_metrics".</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/InsertSizeMetrics.html#InsertSizeMetrics--">InsertSizeMetrics()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/InsertSizeMetrics.html" title="class in picard.analysis">InsertSizeMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/directed/InsertSizeMetricsCollector.html" title="class in picard.analysis.directed"><span class="typeNameLink">InsertSizeMetricsCollector</span></a> - Class in <a href="picard/analysis/directed/package-summary.html">picard.analysis.directed</a></dt>
<dd>
<div class="block">Collects InsertSizeMetrics on the specified accumulationLevels using</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/InsertSizeMetricsCollector.html#InsertSizeMetricsCollector-java.util.Set-java.util.List-double-java.lang.Integer-double-boolean-">InsertSizeMetricsCollector(Set&lt;MetricAccumulationLevel&gt;, List&lt;SAMReadGroupRecord&gt;, double, Integer, double, boolean)</a></span> - Constructor for class picard.analysis.directed.<a href="picard/analysis/directed/InsertSizeMetricsCollector.html" title="class in picard.analysis.directed">InsertSizeMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/directed/InsertSizeMetricsCollector.PerUnitInsertSizeMetricsCollector.html" title="class in picard.analysis.directed"><span class="typeNameLink">InsertSizeMetricsCollector.PerUnitInsertSizeMetricsCollector</span></a> - Class in <a href="picard/analysis/directed/package-summary.html">picard.analysis.directed</a></dt>
<dd>
<div class="block">A Collector for individual InsertSizeMetrics for a given SAMPLE or SAMPLE/LIBRARY or SAMPLE/LIBRARY/READ_GROUP (depending on aggregation levels)</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#instanceMain-java.lang.String:A-">instanceMain(String[])</a></span> - Method in class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/PicardCommandLine.html#instanceMain-java.lang.String:A-java.util.List-java.lang.String-">instanceMain(String[], List&lt;String&gt;, String)</a></span> - Method in class picard.cmdline.<a href="picard/cmdline/PicardCommandLine.html" title="class in picard.cmdline">PicardCommandLine</a></dt>
<dd>
<div class="block">The main method.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/PicardCommandLine.html#instanceMain-java.lang.String:A-">instanceMain(String[])</a></span> - Method in class picard.cmdline.<a href="picard/cmdline/PicardCommandLine.html" title="class in picard.cmdline">PicardCommandLine</a></dt>
<dd>
<div class="block">For testing</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#instanceMainWithExit-java.lang.String:A-">instanceMainWithExit(String[])</a></span> - Method in class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/IntensityChannel.html" title="enum in picard.illumina.parser"><span class="typeNameLink">IntensityChannel</span></a> - Enum in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>
<div class="block">The channels in a FourChannelIntensityData object, and the channels produced by a ClusterIntensityFileReader,
 for cases in which it is desirable to handle these abstractly rather than having the specific names
 in the source code.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RnaSeqMetrics.html#INTERGENIC_BASES">INTERGENIC_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RnaSeqMetrics.html" title="class in picard.analysis">RnaSeqMetrics</a></dt>
<dd>
<div class="block">Number of bases in primary alignments that do not align to any gene.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamToFastq.html#INTERLEAVE">INTERLEAVE</a></span> - Variable in class picard.sam.<a href="picard/sam/SamToFastq.html" title="class in picard.sam">SamToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/TileMetricsUtil.html#INTEROP_SUBDIRECTORY_NAME">INTEROP_SUBDIRECTORY_NAME</a></span> - Static variable in class picard.illumina.parser.<a href="picard/illumina/parser/TileMetricsUtil.html" title="class in picard.illumina.parser">TileMetricsUtil</a></dt>
<dd>
<div class="block">The path to the directory containing the tile metrics file relative to the basecalling directory.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#INTERSECT_INTERVALS">INTERSECT_INTERVALS</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/reference/ExtractSequences.html#INTERVAL_LIST">INTERVAL_LIST</a></span> - Variable in class picard.reference.<a href="picard/reference/ExtractSequences.html" title="class in picard.reference">ExtractSequences</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FilterSamReads.html#INTERVAL_LIST">INTERVAL_LIST</a></span> - Variable in class picard.sam.<a href="picard/sam/FilterSamReads.html" title="class in picard.sam">FilterSamReads</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ViewSam.html#INTERVAL_LIST">INTERVAL_LIST</a></span> - Variable in class picard.sam.<a href="picard/sam/ViewSam.html" title="class in picard.sam">ViewSam</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/cmdline/argumentcollections/IntervalArgumentCollection.html" title="interface in picard.cmdline.argumentcollections"><span class="typeNameLink">IntervalArgumentCollection</span></a> - Interface in <a href="picard/cmdline/argumentcollections/package-summary.html">picard.cmdline.argumentcollections</a></dt>
<dd>
<div class="block">Base interface for an interval argument collection.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.html#intervalArugmentCollection">intervalArugmentCollection</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/IntervalListScatterer.html" title="class in picard.util"><span class="typeNameLink">IntervalListScatterer</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IntervalListScatterer.html#IntervalListScatterer-picard.util.IntervalListScatterer.Mode-">IntervalListScatterer(IntervalListScatterer.Mode)</a></span> - Constructor for class picard.util.<a href="picard/util/IntervalListScatterer.html" title="class in picard.util">IntervalListScatterer</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/IntervalListScatterer.Mode.html" title="enum in picard.util"><span class="typeNameLink">IntervalListScatterer.Mode</span></a> - Enum in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/IntervalListToBed.html" title="class in picard.util"><span class="typeNameLink">IntervalListToBed</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">Trivially simple command line program to convert an IntervalList file to a BED file.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/IntervalListToBed.html#IntervalListToBed--">IntervalListToBed()</a></span> - Constructor for class picard.util.<a href="picard/util/IntervalListToBed.html" title="class in picard.util">IntervalListToBed</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/IntervalListTools.html" title="class in picard.util"><span class="typeNameLink">IntervalListTools</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">Little class to aid working with interval lists.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/IntervalListTools.html#IntervalListTools--">IntervalListTools()</a></span> - Constructor for class picard.util.<a href="picard/util/IntervalListTools.html" title="class in picard.util">IntervalListTools</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/IntervalListTools.Action.html" title="enum in picard.util"><span class="typeNameLink">IntervalListTools.Action</span></a> - Enum in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AbstractWgsMetricsCollector.html#intervals">intervals</a></span> - Variable in class picard.analysis.<a href="picard/analysis/AbstractWgsMetricsCollector.html" title="class in picard.analysis">AbstractWgsMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html#INTERVALS">INTERVALS</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html" title="class in picard.analysis.artifacts">CollectSequencingArtifactMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectMultipleMetrics.html#INTERVALS">INTERVALS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectMultipleMetrics.html" title="class in picard.analysis">CollectMultipleMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.html#INTERVALS">INTERVALS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.html" title="class in picard.analysis">CollectOxoGMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.CollectWgsMetricsIntervalArgumentCollection.html#INTERVALS">INTERVALS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.CollectWgsMetricsIntervalArgumentCollection.html" title="class in picard.analysis">CollectWgsMetrics.CollectWgsMetricsIntervalArgumentCollection</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.html#INTERVALS">INTERVALS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#intervals">intervals</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">The intervals over which this metric was computed.</div>
</dd>
<dt><a href="picard/cmdline/programgroups/Intervals.html" title="class in picard.cmdline.programgroups"><span class="typeNameLink">Intervals</span></a> - Class in <a href="picard/cmdline/programgroups/package-summary.html">picard.cmdline.programgroups</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/programgroups/Intervals.html#Intervals--">Intervals()</a></span> - Constructor for class picard.cmdline.programgroups.<a href="picard/cmdline/programgroups/Intervals.html" title="class in picard.cmdline.programgroups">Intervals</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/reference/NonNFastaSize.html#INTERVALS">INTERVALS</a></span> - Variable in class picard.reference.<a href="picard/reference/NonNFastaSize.html" title="class in picard.reference">NonNFastaSize</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeSamFiles.html#INTERVALS">INTERVALS</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeSamFiles.html" title="class in picard.sam">MergeSamFiles</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#INTERVALS">INTERVALS</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RnaSeqMetrics.html#INTRONIC_BASES">INTRONIC_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RnaSeqMetrics.html" title="class in picard.analysis">RnaSeqMetrics</a></dt>
<dd>
<div class="block">Number of bases in primary alignments that align to an intronic base for some gene, and not a coding or UTR base.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/IntervalListTools.html#INVERT">INVERT</a></span> - Variable in class picard.util.<a href="picard/util/IntervalListTools.html" title="class in picard.util">IntervalListTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/FifoBuffer.html#IO_SIZE">IO_SIZE</a></span> - Variable in class picard.util.<a href="picard/util/FifoBuffer.html" title="class in picard.util">FifoBuffer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#IS_BISULFITE_SEQUENCE">IS_BISULFITE_SEQUENCE</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SetNmMdAndUqTags.html#IS_BISULFITE_SEQUENCE">IS_BISULFITE_SEQUENCE</a></span> - Variable in class picard.sam.<a href="picard/sam/SetNmMdAndUqTags.html" title="class in picard.sam">SetNmMdAndUqTags</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectAlignmentSummaryMetrics.html#IS_BISULFITE_SEQUENCED">IS_BISULFITE_SEQUENCED</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectAlignmentSummaryMetrics.html" title="class in picard.analysis">CollectAlignmentSummaryMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectGcBiasMetrics.html#IS_BISULFITE_SEQUENCED">IS_BISULFITE_SEQUENCED</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectGcBiasMetrics.html" title="class in picard.analysis">CollectGcBiasMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ValidateSamFile.html#IS_BISULFITE_SEQUENCED">IS_BISULFITE_SEQUENCED</a></span> - Variable in class picard.sam.<a href="picard/sam/ValidateSamFile.html" title="class in picard.sam">ValidateSamFile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CollectIlluminaLaneMetrics.html#IS_NOVASEQ">IS_NOVASEQ</a></span> - Variable in class picard.illumina.<a href="picard/illumina/CollectIlluminaLaneMetrics.html" title="class in picard.illumina">CollectIlluminaLaneMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AdapterUtility.html#isAdapterSequence-byte:A-">isAdapterSequence(byte[])</a></span> - Method in class picard.analysis.<a href="picard/analysis/AdapterUtility.html" title="class in picard.analysis">AdapterUtility</a></dt>
<dd>
<div class="block">Checks the first ADAPTER_MATCH_LENGTH bases of the read against known adapter sequences and returns
 true if the read matches an adapter sequence with MAX_ADAPTER_ERRORS mismsatches or fewer.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/AbstractInputParser.html#isBlank-byte:A-">isBlank(byte[])</a></span> - Method in class picard.util.<a href="picard/util/AbstractInputParser.html" title="class in picard.util">AbstractInputParser</a></dt>
<dd>
<div class="block">Determines whether a given line is a comment</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/BclReader.html#isBlockGzipped-java.io.File-">isBlockGzipped(File)</a></span> - Static method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/BclReader.html" title="class in picard.illumina.parser.readers">BclReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/ChimeraUtil.html#isChimeric-htsjdk.samtools.SAMRecord-int-java.util.Set-">isChimeric(SAMRecord, int, Set&lt;SamPairUtil.PairOrientation&gt;)</a></span> - Static method in class picard.analysis.<a href="picard/analysis/ChimeraUtil.html" title="class in picard.analysis">ChimeraUtil</a></dt>
<dd>
<div class="block">Checks whether the given read is part of a chimeric pair.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/ChimeraUtil.html#isChimeric-htsjdk.samtools.SAMRecord-htsjdk.samtools.SAMRecord-int-java.util.Set-">isChimeric(SAMRecord, SAMRecord, int, Set&lt;SamPairUtil.PairOrientation&gt;)</a></span> - Static method in class picard.analysis.<a href="picard/analysis/ChimeraUtil.html" title="class in picard.analysis">ChimeraUtil</a></dt>
<dd>
<div class="block">Checks whether the given read is part of a chimeric pair.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.html#isClipOverlappingReads--">isClipOverlappingReads()</a></span> - Method in class picard.sam.<a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/CircularByteBuffer.html#isClosed--">isClosed()</a></span> - Method in class picard.util.<a href="picard/util/CircularByteBuffer.html" title="class in picard.util">CircularByteBuffer</a></dt>
<dd>
<div class="block">Returns true if the buffer is closed, false otherwise.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaTileMetrics.html#isClusterRecord--">isClusterRecord()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaTileMetrics.html" title="class in picard.illumina.parser.readers">TileMetricsOutReader.IlluminaTileMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AbstractInputParser.html#isComment-byte:A-">isComment(byte[])</a></span> - Method in class picard.util.<a href="picard/util/AbstractInputParser.html" title="class in picard.util">AbstractInputParser</a></dt>
<dd>
<div class="block">Determines whether a given line is a comment</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.html#isContaminant-picard.sam.HitsForInsert-">isContaminant(HitsForInsert)</a></span> - Method in class picard.sam.<a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamAlignmentMerger.html#isContaminant-picard.sam.HitsForInsert-">isContaminant(HitsForInsert)</a></span> - Method in class picard.sam.<a href="picard/sam/SamAlignmentMerger.html" title="class in picard.sam">SamAlignmentMerger</a></dt>
<dd>
<div class="block">Criteria for contaminant reads:
 1.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/DbSnpBitSetUtil.html#isDbSnpSite-java.lang.String-int-">isDbSnpSite(String, int)</a></span> - Method in class picard.util.<a href="picard/util/DbSnpBitSetUtil.html" title="class in picard.util">DbSnpBitSetUtil</a></dt>
<dd>
<div class="block">Returns true if there is a dbSnp entry at pos in sequenceName, otherwise false</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/AbstractInputParser.html#isDelimiter-byte-">isDelimiter(byte)</a></span> - Method in class picard.util.<a href="picard/util/AbstractInputParser.html" title="class in picard.util">AbstractInputParser</a></dt>
<dd>
<div class="block">Determines whether a given character is a delimiter</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/CsvInputParser.html#isDelimiter-byte-">isDelimiter(byte)</a></span> - Method in class picard.util.<a href="picard/util/CsvInputParser.html" title="class in picard.util">CsvInputParser</a></dt>
<dd>
<div class="block">Determines whether a given character is a delimiter</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/TabbedInputParser.html#isDelimiter-byte-">isDelimiter(byte)</a></span> - Method in class picard.util.<a href="picard/util/TabbedInputParser.html" title="class in picard.util">TabbedInputParser</a></dt>
<dd>
<div class="block">Determines whether a given character is a delimiter</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadStructure.Substructure.html#isEmpty--">isEmpty()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ReadStructure.Substructure.html" title="class in picard.illumina.parser">ReadStructure.Substructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/MarkQueue.html#isEmpty--">isEmpty()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/MarkQueue.html" title="class in picard.sam.markduplicates.util">MarkQueue</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckMetric.FingerprintResult.html#isExpected--">isExpected()</a></span> - Method in enum picard.fingerprint.<a href="picard/fingerprint/CrosscheckMetric.FingerprintResult.html" title="enum in picard.fingerprint">CrosscheckMetric.FingerprintResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/BclReader.html#isGzipped-java.io.File-">isGzipped(File)</a></span> - Static method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/BclReader.html" title="class in picard.illumina.parser.readers">BclReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/DiploidGenotype.html#isHeterozygous--">isHeterozygous()</a></span> - Method in enum picard.fingerprint.<a href="picard/fingerprint/DiploidGenotype.html" title="enum in picard.fingerprint">DiploidGenotype</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/DiploidGenotype.html#isHomomozygous--">isHomomozygous()</a></span> - Method in enum picard.fingerprint.<a href="picard/fingerprint/DiploidGenotype.html" title="enum in picard.fingerprint">DiploidGenotype</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.html#isKeepAlignerProperPairFlags--">isKeepAlignerProperPairFlags()</a></span> - Method in class picard.sam.<a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckMetric.FingerprintResult.html#isMatch--">isMatch()</a></span> - Method in enum picard.fingerprint.<a href="picard/fingerprint/CrosscheckMetric.FingerprintResult.html" title="enum in picard.fingerprint">CrosscheckMetric.FingerprintResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.PerTileBarcodeExtractor.BarcodeMatch.html#isMatched--">isMatched()</a></span> - Method in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.PerTileBarcodeExtractor.BarcodeMatch.html" title="class in picard.illumina">ExtractIlluminaBarcodes.PerTileBarcodeExtractor.BarcodeMatch</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceCounts.html#isMissing-picard.vcf.GenotypeConcordanceStates.TruthState-picard.vcf.GenotypeConcordanceStates.CallState-">isMissing(GenotypeConcordanceStates.TruthState, GenotypeConcordanceStates.CallState)</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceCounts.html" title="class in picard.vcf">GenotypeConcordanceCounts</a></dt>
<dd>
<div class="block">Returns true if EITHER the truth or call state is MISSING.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEnds.html#isOpticalDuplicate">isOpticalDuplicate</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEnds.html" title="class in picard.sam.markduplicates.util">ReadEnds</a></dt>
<dd>
<div class="block">A *transient* flag marking this read end as being an optical duplicate.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEnds.html#isPaired--">isPaired()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEnds.html" title="class in picard.sam.markduplicates.util">ReadEnds</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMateCigar.html#isPaired--">isPaired()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMateCigar.html" title="class in picard.sam.markduplicates.util">ReadEndsForMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/CbclData.html#isPf--">isPf()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/CbclData.html" title="class in picard.illumina.parser">CbclData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ClusterData.html#isPf--">isPf()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ClusterData.html" title="class in picard.illumina.parser">ClusterData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaMetricsCode.html#isPhasing-picard.illumina.parser.IlluminaMetricsCode-">isPhasing(IlluminaMetricsCode)</a></span> - Static method in enum picard.illumina.parser.<a href="picard/illumina/parser/IlluminaMetricsCode.html" title="enum in picard.illumina.parser">IlluminaMetricsCode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.UnmappingReadStrategy.html#isPopulatePaTag--">isPopulatePaTag()</a></span> - Method in enum picard.sam.<a href="picard/sam/AbstractAlignmentMerger.UnmappingReadStrategy.html" title="enum in picard.sam">AbstractAlignmentMerger.UnmappingReadStrategy</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.html#isReservedTag-java.lang.String-">isReservedTag(String)</a></span> - Method in class picard.sam.<a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.UnmappingReadStrategy.html#isResetMappingInformation--">isResetMappingInformation()</a></span> - Method in enum picard.sam.<a href="picard/sam/AbstractAlignmentMerger.UnmappingReadStrategy.html" title="enum in picard.sam">AbstractAlignmentMerger.UnmappingReadStrategy</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AbstractInputParser.html#isSkipBlankLines--">isSkipBlankLines()</a></span> - Method in class picard.util.<a href="picard/util/AbstractInputParser.html" title="class in picard.util">AbstractInputParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/Gene.Transcript.html#isSoloTranscript--">isSoloTranscript()</a></span> - Method in class picard.annotation.<a href="picard/annotation/Gene.Transcript.html" title="class in picard.annotation">Gene.Transcript</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AbstractInputParser.html#isTreatGroupedDelimitersAsOne--">isTreatGroupedDelimitersAsOne()</a></span> - Method in class picard.util.<a href="picard/util/AbstractInputParser.html" title="class in picard.util">AbstractInputParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/DiploidGenotype.html#isValidBase-byte-">isValidBase(byte)</a></span> - Static method in enum picard.fingerprint.<a href="picard/fingerprint/DiploidGenotype.html" title="enum in picard.fingerprint">DiploidGenotype</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceCounts.html#isVar-picard.vcf.GenotypeConcordanceStates.TruthState-picard.vcf.GenotypeConcordanceStates.CallState-">isVar(GenotypeConcordanceStates.TruthState, GenotypeConcordanceStates.CallState)</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceCounts.html" title="class in picard.vcf">GenotypeConcordanceCounts</a></dt>
<dd>
<div class="block">Returns true if EITHER the truth or call state is a VAR.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/VcfUtils.html#isVariantFile-java.io.File-">isVariantFile(File)</a></span> - Static method in class picard.vcf.<a href="picard/vcf/VcfUtils.html" title="class in picard.vcf">VcfUtils</a></dt>
<dd>
<div class="block">Checks if the suffix is one of those that are allowed for the various
 formats that contain variants (currently vcf and bcf)</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/Gene.html#iterator--">iterator()</a></span> - Method in class picard.annotation.<a href="picard/annotation/Gene.html" title="class in picard.annotation">Gene</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/BaseIlluminaDataProvider.html#iterator--">iterator()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/BaseIlluminaDataProvider.html" title="class in picard.illumina.parser">BaseIlluminaDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadStructure.Substructure.html#iterator--">iterator()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ReadStructure.Substructure.html" title="class in picard.illumina.parser">ReadStructure.Substructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/TileIndex.html#iterator--">iterator()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/TileIndex.html" title="class in picard.illumina.parser">TileIndex</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.SamHeaderAndIterator.html#iterator">iterator</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.SamHeaderAndIterator.html" title="class in picard.sam.markduplicates.util">AbstractMarkDuplicatesCommandLineProgram.SamHeaderAndIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AbstractInputParser.html#iterator--">iterator()</a></span> - Method in class picard.util.<a href="picard/util/AbstractInputParser.html" title="class in picard.util">AbstractInputParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/TabbedTextFileWithHeaderParser.html#iterator--">iterator()</a></span> - Method in class picard.util.<a href="picard/util/TabbedTextFileWithHeaderParser.html" title="class in picard.util">TabbedTextFileWithHeaderParser</a></dt>
<dd>
<div class="block">Creates the iterator object.</div>
</dd>
<dt><a href="picard/util/Iterators.html" title="class in picard.util"><span class="typeNameLink">Iterators</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/Iterators.html#Iterators--">Iterators()</a></span> - Constructor for class picard.util.<a href="picard/util/Iterators.html" title="class in picard.util">Iterators</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VariantIteratorProducer.html#iterators--">iterators()</a></span> - Method in class picard.vcf.processor.<a href="picard/vcf/processor/VariantIteratorProducer.html" title="class in picard.vcf.processor">VariantIteratorProducer</a></dt>
<dd>
<div class="block">Renders the embodied regions of the VCF files in the form of <code>CloseableIterator</code>s over
 <code>VariantContext</code>s.</div>
</dd>
</dl>
<a name="I:J">
<!--   -->
</a>
<h2 class="title">J</h2>
<dl>
<dt><span class="memberNameLink"><a href="picard/sam/FilterSamReads.html#JAVASCRIPT_FILE">JAVASCRIPT_FILE</a></span> - Variable in class picard.sam.<a href="picard/sam/FilterSamReads.html" title="class in picard.sam">FilterSamReads</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/FilterVcf.html#JAVASCRIPT_FILE">JAVASCRIPT_FILE</a></span> - Variable in class picard.vcf.filter.<a href="picard/vcf/filter/FilterVcf.html" title="class in picard.vcf.filter">FilterVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/JumpingLibraryMetrics.html#JUMP_DUPLICATE_PAIRS">JUMP_DUPLICATE_PAIRS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/JumpingLibraryMetrics.html" title="class in picard.analysis">JumpingLibraryMetrics</a></dt>
<dd>
<div class="block">The number of outward-facing pairs that are duplicates</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/JumpingLibraryMetrics.html#JUMP_DUPLICATE_PCT">JUMP_DUPLICATE_PCT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/JumpingLibraryMetrics.html" title="class in picard.analysis">JumpingLibraryMetrics</a></dt>
<dd>
<div class="block">The fraction of outward-facing pairs that are marked as duplicates</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/JumpingLibraryMetrics.html#JUMP_LIBRARY_SIZE">JUMP_LIBRARY_SIZE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/JumpingLibraryMetrics.html" title="class in picard.analysis">JumpingLibraryMetrics</a></dt>
<dd>
<div class="block">The estimated library size for outward-facing pairs</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/JumpingLibraryMetrics.html#JUMP_MEAN_INSERT_SIZE">JUMP_MEAN_INSERT_SIZE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/JumpingLibraryMetrics.html" title="class in picard.analysis">JumpingLibraryMetrics</a></dt>
<dd>
<div class="block">The mean insert size for outward-facing pairs</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/JumpingLibraryMetrics.html#JUMP_PAIRS">JUMP_PAIRS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/JumpingLibraryMetrics.html" title="class in picard.analysis">JumpingLibraryMetrics</a></dt>
<dd>
<div class="block">The number of outward-facing pairs in the SAM file</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#JUMP_SIZE">JUMP_SIZE</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/JumpingLibraryMetrics.html#JUMP_STDEV_INSERT_SIZE">JUMP_STDEV_INSERT_SIZE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/JumpingLibraryMetrics.html" title="class in picard.analysis">JumpingLibraryMetrics</a></dt>
<dd>
<div class="block">The standard deviation on the insert size for outward-facing pairs</div>
</dd>
<dt><a href="picard/analysis/JumpingLibraryMetrics.html" title="class in picard.analysis"><span class="typeNameLink">JumpingLibraryMetrics</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">High level metrics about the presence of  outward- and inward-facing pairs
 within a SAM file generated with a jumping library, produced by
 the CollectJumpingLibraryMetrics program and usually stored in a file with
 the extension ".jump_metrics".</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/JumpingLibraryMetrics.html#JumpingLibraryMetrics--">JumpingLibraryMetrics()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/JumpingLibraryMetrics.html" title="class in picard.analysis">JumpingLibraryMetrics</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="I:L">
<!--   -->
</a>
<h2 class="title">L</h2>
<dl>
<dt><span class="memberNameLink"><a href="picard/fastq/BamToBfq.html#LANE">LANE</a></span> - Variable in class picard.fastq.<a href="picard/fastq/BamToBfq.html" title="class in picard.fastq">BamToBfq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CollectIlluminaBasecallingMetrics.html#LANE">LANE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/CollectIlluminaBasecallingMetrics.html" title="class in picard.illumina">CollectIlluminaBasecallingMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.html#LANE">LANE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.html" title="class in picard.illumina">ExtractIlluminaBarcodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallingMetrics.html#LANE">LANE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallingMetrics.html" title="class in picard.illumina">IlluminaBasecallingMetrics</a></dt>
<dd>
<div class="block">The lane for which the metrics were calculated.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToFastq.html#LANE">LANE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToFastq.html" title="class in picard.illumina">IlluminaBasecallsToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#LANE">LANE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaLaneMetrics.html#LANE">LANE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaLaneMetrics.html" title="class in picard.illumina">IlluminaLaneMetrics</a></dt>
<dd>
<div class="block">This lane's number.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaPhasingMetrics.html#LANE">LANE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaPhasingMetrics.html" title="class in picard.illumina">IlluminaPhasingMetrics</a></dt>
<dd>
<div class="block">Illumina flowcell lane number</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/BaseIlluminaDataProvider.html#lane">lane</a></span> - Variable in class picard.illumina.parser.<a href="picard/illumina/parser/BaseIlluminaDataProvider.html" title="class in picard.illumina.parser">BaseIlluminaDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ParameterizedFileUtil.html#lane">lane</a></span> - Variable in class picard.illumina.parser.<a href="picard/illumina/parser/ParameterizedFileUtil.html" title="class in picard.illumina.parser">ParameterizedFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.PositionInfo.html#lane">lane</a></span> - Variable in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.PositionInfo.html" title="class in picard.illumina.parser.readers">AbstractIlluminaPositionFileReader.PositionInfo</a></dt>
<dd>
<div class="block">The lane, which is determined from the file name</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.html#LANE">LANE</a></span> - Variable in class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/StandardOptionDefinitions.html#LANE_SHORT_NAME">LANE_SHORT_NAME</a></span> - Static variable in class picard.cmdline.<a href="picard/cmdline/StandardOptionDefinitions.html" title="class in picard.cmdline">StandardOptionDefinitions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ClusterData.html#laneIsSet--">laneIsSet()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ClusterData.html" title="class in picard.illumina.parser">ClusterData</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/LanePhasingMetricsCollector.html" title="class in picard.illumina"><span class="typeNameLink">LanePhasingMetricsCollector</span></a> - Class in <a href="picard/illumina/package-summary.html">picard.illumina</a></dt>
<dd>
<div class="block">Helper class used to transform tile data for a lane into a collection of IlluminaPhasingMetrics</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/LanePhasingMetricsCollector.html#LanePhasingMetricsCollector-java.util.Collection-boolean-">LanePhasingMetricsCollector(Collection&lt;Tile&gt;, boolean)</a></span> - Constructor for class picard.illumina.<a href="picard/illumina/LanePhasingMetricsCollector.html" title="class in picard.illumina">LanePhasingMetricsCollector</a></dt>
<dd>
<div class="block">Constructor takes a lane's collection of Tiles and calculates the median phasing/prephasing for the
 first and second (if available) reads</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CheckIlluminaDirectory.html#LANES">LANES</a></span> - Variable in class picard.illumina.<a href="picard/illumina/CheckIlluminaDirectory.html" title="class in picard.illumina">CheckIlluminaDirectory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/EmpiricalPhasingMetricsOutReader.IlluminaPhasingMetrics.html#laneTileCode">laneTileCode</a></span> - Variable in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/EmpiricalPhasingMetricsOutReader.IlluminaPhasingMetrics.html" title="class in picard.illumina.parser.readers">EmpiricalPhasingMetricsOutReader.IlluminaPhasingMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckMetric.html#LEFT_FILE">LEFT_FILE</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckMetric.html" title="class in picard.fingerprint">CrosscheckMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckMetric.html#LEFT_GROUP_VALUE">LEFT_GROUP_VALUE</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckMetric.html" title="class in picard.fingerprint">CrosscheckMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckMetric.html#LEFT_LANE">LEFT_LANE</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckMetric.html" title="class in picard.fingerprint">CrosscheckMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckMetric.html#LEFT_LIBRARY">LEFT_LIBRARY</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckMetric.html" title="class in picard.fingerprint">CrosscheckMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckMetric.html#LEFT_MOLECULAR_BARCODE_SEQUENCE">LEFT_MOLECULAR_BARCODE_SEQUENCE</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckMetric.html" title="class in picard.fingerprint">CrosscheckMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/BaitDesigner.html#LEFT_PRIMER">LEFT_PRIMER</a></span> - Variable in class picard.util.<a href="picard/util/BaitDesigner.html" title="class in picard.util">BaitDesigner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckMetric.html#LEFT_RUN_BARCODE">LEFT_RUN_BARCODE</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckMetric.html" title="class in picard.fingerprint">CrosscheckMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckMetric.html#LEFT_SAMPLE">LEFT_SAMPLE</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckMetric.html" title="class in picard.fingerprint">CrosscheckMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/LiftoverVcf.html#leftAlignVariant-htsjdk.variant.variantcontext.VariantContext-htsjdk.samtools.reference.ReferenceSequence-">leftAlignVariant(VariantContext, ReferenceSequence)</a></span> - Static method in class picard.vcf.<a href="picard/vcf/LiftoverVcf.html" title="class in picard.vcf">LiftoverVcf</a></dt>
<dd>
<div class="block">Normalizes and left aligns a <code>VariantContext</code>.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/PairedVariantSubContextIterator.VcfTuple.html#leftVariantContext">leftVariantContext</a></span> - Variable in class picard.vcf.<a href="picard/vcf/PairedVariantSubContextIterator.VcfTuple.html" title="class in picard.vcf">PairedVariantSubContextIterator.VcfTuple</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/Gene.Transcript.html#length--">length()</a></span> - Method in class picard.annotation.<a href="picard/annotation/Gene.Transcript.html" title="class in picard.annotation">Gene.Transcript</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/Range.html#length">length</a></span> - Variable in class picard.illumina.parser.<a href="picard/illumina/parser/Range.html" title="class in picard.illumina.parser">Range</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadDescriptor.html#length">length</a></span> - Variable in class picard.illumina.parser.<a href="picard/illumina/parser/ReadDescriptor.html" title="class in picard.illumina.parser">ReadDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadStructure.Substructure.html#length--">length()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ReadStructure.Substructure.html" title="class in picard.illumina.parser">ReadStructure.Substructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html#LIBRARY">LIBRARY</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.BaitBiasDetailMetrics</a></dt>
<dd>
<div class="block">The name of the library being assayed.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasSummaryMetrics.html#LIBRARY">LIBRARY</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasSummaryMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.BaitBiasSummaryMetrics</a></dt>
<dd>
<div class="block">The name of the library being assayed.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterDetailMetrics.html#LIBRARY">LIBRARY</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterDetailMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.PreAdapterDetailMetrics</a></dt>
<dd>
<div class="block">The name of the library being assayed.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterSummaryMetrics.html#LIBRARY">LIBRARY</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterSummaryMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.PreAdapterSummaryMetrics</a></dt>
<dd>
<div class="block">The name of the library being assayed.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html#LIBRARY">LIBRARY</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html" title="class in picard.analysis">CollectOxoGMetrics.CpcgMetrics</a></dt>
<dd>
<div class="block">The name of the library being assayed.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/metrics/MultilevelMetrics.html#LIBRARY">LIBRARY</a></span> - Variable in class picard.metrics.<a href="picard/metrics/MultilevelMetrics.html" title="class in picard.metrics">MultilevelMetrics</a></dt>
<dd>
<div class="block">The library to which these metrics apply.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/DuplicationMetrics.html#LIBRARY">LIBRARY</a></span> - Variable in class picard.sam.<a href="picard/sam/DuplicationMetrics.html" title="class in picard.sam">DuplicationMetrics</a></dt>
<dd>
<div class="block">The library on which the duplicate marking was performed.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html#LIBRARY_NAME">LIBRARY_NAME</a></span> - Variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html" title="class in picard.illumina">ExtractIlluminaBarcodes.BarcodeMetric</a></dt>
<dd>
<div class="block">The name of the library</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#LIBRARY_NAME">LIBRARY_NAME</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FastqToSam.html#LIBRARY_NAME">LIBRARY_NAME</a></span> - Variable in class picard.sam.<a href="picard/sam/FastqToSam.html" title="class in picard.sam">FastqToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertSam.html#LIBRARY_NAME">LIBRARY_NAME</a></span> - Variable in class picard.sam.<a href="picard/sam/RevertSam.html" title="class in picard.sam">RevertSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.html#LIBRARY_NAME_COLUMN">LIBRARY_NAME_COLUMN</a></span> - Static variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.html" title="class in picard.illumina">ExtractIlluminaBarcodes</a></dt>
<dd>
<div class="block">Column header for the library name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/StandardOptionDefinitions.html#LIBRARY_NAME_SHORT_NAME">LIBRARY_NAME_SHORT_NAME</a></span> - Static variable in class picard.cmdline.<a href="picard/cmdline/StandardOptionDefinitions.html" title="class in picard.cmdline">StandardOptionDefinitions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#LIBRARY_PARAMS">LIBRARY_PARAMS</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEnds.html#libraryId">libraryId</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEnds.html" title="class in picard.sam.markduplicates.util">ReadEnds</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicates.html#libraryIdGenerator">libraryIdGenerator</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicates.html" title="class in picard.sam.markduplicates">MarkDuplicates</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/markduplicates/util/LibraryIdGenerator.html" title="class in picard.sam.markduplicates.util"><span class="typeNameLink">LibraryIdGenerator</span></a> - Class in <a href="picard/sam/markduplicates/util/package-summary.html">picard.sam.markduplicates.util</a></dt>
<dd>
<div class="block">A class to generate library Ids and keep duplication metrics by library IDs.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/LibraryIdGenerator.html#LibraryIdGenerator-htsjdk.samtools.SAMFileHeader-">LibraryIdGenerator(SAMFileHeader)</a></span> - Constructor for class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/LibraryIdGenerator.html" title="class in picard.sam.markduplicates.util">LibraryIdGenerator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/LiftoverVcf.html#LIFTOVER_MIN_MATCH">LIFTOVER_MIN_MATCH</a></span> - Variable in class picard.vcf.<a href="picard/vcf/LiftoverVcf.html" title="class in picard.vcf">LiftoverVcf</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/LiftOverIntervalList.html" title="class in picard.util"><span class="typeNameLink">LiftOverIntervalList</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">This tool adjusts the coordinates in an interval list on one reference to its homologous interval list on another
 reference, based on a chain file that describes the correspondence between the two references.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/LiftOverIntervalList.html#LiftOverIntervalList--">LiftOverIntervalList()</a></span> - Constructor for class picard.util.<a href="picard/util/LiftOverIntervalList.html" title="class in picard.util">LiftOverIntervalList</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/LiftoverVcf.html" title="class in picard.vcf"><span class="typeNameLink">LiftoverVcf</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">Tool for lifting over a VCF to another genome build and producing a properly header'd,
 sorted and indexed VCF in one go.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/LiftoverVcf.html#LiftoverVcf--">LiftoverVcf()</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/LiftoverVcf.html" title="class in picard.vcf">LiftoverVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/LiftoverVcf.html#liftSimpleVariant-htsjdk.variant.variantcontext.VariantContext-htsjdk.samtools.util.Interval-">liftSimpleVariant(VariantContext, Interval)</a></span> - Static method in class picard.vcf.<a href="picard/vcf/LiftoverVcf.html" title="class in picard.vcf">LiftoverVcf</a></dt>
<dd>
<div class="block">liftsOver snps on either positive or negative strand and biallelic indels on positive strand only</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VariantProcessor.Builder.html#limitingProcessedRegionsTo-htsjdk.samtools.util.IntervalList-">limitingProcessedRegionsTo(IntervalList)</a></span> - Method in class picard.vcf.processor.<a href="picard/vcf/processor/VariantProcessor.Builder.html" title="class in picard.vcf.processor">VariantProcessor.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/reference/ExtractSequences.html#LINE_LENGTH">LINE_LENGTH</a></span> - Variable in class picard.reference.<a href="picard/reference/ExtractSequences.html" title="class in picard.reference">ExtractSequences</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/reference/NormalizeFasta.html#LINE_LENGTH">LINE_LENGTH</a></span> - Variable in class picard.reference.<a href="picard/reference/NormalizeFasta.html" title="class in picard.reference">NormalizeFasta</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CheckIlluminaDirectory.html#LINK_LOCS">LINK_LOCS</a></span> - Variable in class picard.illumina.<a href="picard/illumina/CheckIlluminaDirectory.html" title="class in picard.illumina">CheckIlluminaDirectory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/FingerprintingSummaryMetrics.html#LL_EXPECTED_SAMPLE">LL_EXPECTED_SAMPLE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/FingerprintingSummaryMetrics.html" title="class in picard.analysis">FingerprintingSummaryMetrics</a></dt>
<dd>
<div class="block">The Log Likelihood of the sequence data given the expected sample's genotypes.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/FingerprintingSummaryMetrics.html#LL_RANDOM_SAMPLE">LL_RANDOM_SAMPLE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/FingerprintingSummaryMetrics.html" title="class in picard.analysis">FingerprintingSummaryMetrics</a></dt>
<dd>
<div class="block">The Log Likelihood of the sequence data given a random sample from the human population.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintChecker.html#loadFingerprints-java.io.File-java.lang.String-">loadFingerprints(File, String)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/FingerprintChecker.html" title="class in picard.fingerprint">FingerprintChecker</a></dt>
<dd>
<div class="block">Loads genotypes from the supplied file into one or more Fingerprint objects and returns them in a
 Map of Sample->Fingerprint.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintChecker.html#loadFingerprintsFromIndexedVcf-java.io.File-java.lang.String-">loadFingerprintsFromIndexedVcf(File, String)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/FingerprintChecker.html" title="class in picard.fingerprint">FingerprintChecker</a></dt>
<dd>
<div class="block">Loads genotypes from the supplied file into one or more Fingerprint objects and returns them in a
 Map of Sample->Fingerprint.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintChecker.html#loadFingerprintsFromNonIndexedVcf-java.io.File-java.lang.String-">loadFingerprintsFromNonIndexedVcf(File, String)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/FingerprintChecker.html" title="class in picard.fingerprint">FingerprintChecker</a></dt>
<dd>
<div class="block">Loads genotypes from the supplied file into one or more Fingerprint objects and returns them in a
 Map of Sample->Fingerprint.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/PropertyUtils.html#loadPropertiesFile-java.lang.String-java.lang.Class-">loadPropertiesFile(String, Class&lt;?&gt;)</a></span> - Static method in class picard.util.<a href="picard/util/PropertyUtils.html" title="class in picard.util">PropertyUtils</a></dt>
<dd>
<div class="block">Attempt to load a Properties object from an on-disk properties file.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/GeneAnnotationReader.html#loadRefFlat-java.io.File-htsjdk.samtools.SAMSequenceDictionary-">loadRefFlat(File, SAMSequenceDictionary)</a></span> - Static method in class picard.annotation.<a href="picard/annotation/GeneAnnotationReader.html" title="class in picard.annotation">GeneAnnotationReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/fakers/LocsFileFaker.html" title="class in picard.illumina.parser.fakers"><span class="typeNameLink">LocsFileFaker</span></a> - Class in <a href="picard/illumina/parser/fakers/package-summary.html">picard.illumina.parser.fakers</a></dt>
<dd>
<div class="block">Created by jcarey on 3/13/14.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/LocsFileFaker.html#LocsFileFaker--">LocsFileFaker()</a></span> - Constructor for class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/LocsFileFaker.html" title="class in picard.illumina.parser.fakers">LocsFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/readers/LocsFileReader.html" title="class in picard.illumina.parser.readers"><span class="typeNameLink">LocsFileReader</span></a> - Class in <a href="picard/illumina/parser/readers/package-summary.html">picard.illumina.parser.readers</a></dt>
<dd>
<div class="block">The locs file format is one 3 Illumina formats(pos, locs, and clocs) that stores position data exclusively.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/LocsFileReader.html#LocsFileReader-java.io.File-">LocsFileReader(File)</a></span> - Constructor for class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/LocsFileReader.html" title="class in picard.illumina.parser.readers">LocsFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/LocsFileReader.html#LocsFileReader-java.io.File-int-int-">LocsFileReader(File, int, int)</a></span> - Constructor for class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/LocsFileReader.html" title="class in picard.illumina.parser.readers">LocsFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.html#LOCUS_ACCUMULATION_CAP">LOCUS_ACCUMULATION_CAP</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/annotation/LocusFunction.html" title="enum in picard.annotation"><span class="typeNameLink">LocusFunction</span></a> - Enum in <a href="picard/annotation/package-summary.html">picard.annotation</a></dt>
<dd>
<div class="block">Describes the behavior of a locus relative to a gene.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/FingerprintingDetailMetrics.html#LOD">LOD</a></span> - Variable in class picard.analysis.<a href="picard/analysis/FingerprintingDetailMetrics.html" title="class in picard.analysis">FingerprintingDetailMetrics</a></dt>
<dd>
<div class="block">The LOD score for OBSERVED_GENOTYPE vs.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/FingerprintingSummaryMetrics.html#LOD_EXPECTED_SAMPLE">LOD_EXPECTED_SAMPLE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/FingerprintingSummaryMetrics.html" title="class in picard.analysis">FingerprintingSummaryMetrics</a></dt>
<dd>
<div class="block">The LOD for Expected Sample vs.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckMetric.html#LOD_SCORE">LOD_SCORE</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckMetric.html" title="class in picard.fingerprint">CrosscheckMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckMetric.html#LOD_SCORE_NORMAL_TUMOR">LOD_SCORE_NORMAL_TUMOR</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckMetric.html" title="class in picard.fingerprint">CrosscheckMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckMetric.html#LOD_SCORE_TUMOR_NORMAL">LOD_SCORE_TUMOR_NORMAL</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckMetric.html" title="class in picard.fingerprint">CrosscheckMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/ClusterCrosscheckMetrics.html#LOD_THRESHOLD">LOD_THRESHOLD</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/ClusterCrosscheckMetrics.html" title="class in picard.fingerprint">ClusterCrosscheckMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckFingerprints.html#LOD_THRESHOLD">LOD_THRESHOLD</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckFingerprints.html" title="class in picard.fingerprint">CrosscheckFingerprints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/AbstractOpticalDuplicateFinderCommandLineProgram.html#LOG">LOG</a></span> - Static variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/AbstractOpticalDuplicateFinderCommandLineProgram.html" title="class in picard.sam.markduplicates.util">AbstractOpticalDuplicateFinderCommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/VcfFormatConverter.html#LOG">LOG</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/VcfFormatConverter.html" title="class in picard.vcf">VcfFormatConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/VcfToIntervalList.html#LOG">LOG</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/VcfToIntervalList.html" title="class in picard.vcf">VcfToIntervalList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#log10_1p-double-">log10_1p(double)</a></span> - Static method in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>
<div class="block">this function mimics the behavior of log_1p but resulting in log _base 10_ of (1+x) instead of natural log of 1+x</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#LOG_10_MATH">LOG_10_MATH</a></span> - Static variable in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#LOG_2_MATH">LOG_2_MATH</a></span> - Static variable in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#LOG_4_BASE_E">LOG_4_BASE_E</a></span> - Static variable in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>
<div class="block">Constant to convert between the natural base e and 4.0.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#logLikelihoodsToProbs-double:A-">logLikelihoodsToProbs(double[])</a></span> - Static method in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>
<div class="block"><span class="deprecatedLabel">Deprecated.</span></div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigarIterator.html#logMemoryStats-htsjdk.samtools.util.Log-">logMemoryStats(Log)</a></span> - Method in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigarIterator.html" title="class in picard.sam.markduplicates">MarkDuplicatesWithMateCigarIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaFileUtil.html#longLaneStr-int-">longLaneStr(int)</a></span> - Static method in class picard.illumina.parser.<a href="picard/illumina/parser/IlluminaFileUtil.html" title="class in picard.illumina.parser">IlluminaFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckFingerprints.html#LOSS_OF_HET_RATE">LOSS_OF_HET_RATE</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckFingerprints.html" title="class in picard.fingerprint">CrosscheckFingerprints</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="I:M">
<!--   -->
</a>
<h2 class="title">M</h2>
<dl>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToFastq.html#MACHINE_NAME">MACHINE_NAME</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToFastq.html" title="class in picard.illumina">IlluminaBasecallsToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#MAD_COVERAGE">MAD_COVERAGE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">The median absolute deviation of coverage of the genome after all filters are applied.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/ConvertSequencingArtifactToOxoG.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/ConvertSequencingArtifactToOxoG.html" title="class in picard.analysis.artifacts">ConvertSequencingArtifactToOxoG</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectAlignmentSummaryMetrics.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.analysis.<a href="picard/analysis/CollectAlignmentSummaryMetrics.html" title="class in picard.analysis">CollectAlignmentSummaryMetrics</a></dt>
<dd>
<div class="block">Required main method implementation.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectBaseDistributionByCycle.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.analysis.<a href="picard/analysis/CollectBaseDistributionByCycle.html" title="class in picard.analysis">CollectBaseDistributionByCycle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectGcBiasMetrics.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.analysis.<a href="picard/analysis/CollectGcBiasMetrics.html" title="class in picard.analysis">CollectGcBiasMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectInsertSizeMetrics.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.analysis.<a href="picard/analysis/CollectInsertSizeMetrics.html" title="class in picard.analysis">CollectInsertSizeMetrics</a></dt>
<dd>
<div class="block">Required main method implementation.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectJumpingLibraryMetrics.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.analysis.<a href="picard/analysis/CollectJumpingLibraryMetrics.html" title="class in picard.analysis">CollectJumpingLibraryMetrics</a></dt>
<dd>
<div class="block">Stock main method.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectMultipleMetrics.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.analysis.<a href="picard/analysis/CollectMultipleMetrics.html" title="class in picard.analysis">CollectMultipleMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.html" title="class in picard.analysis">CollectOxoGMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRnaSeqMetrics.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.analysis.<a href="picard/analysis/CollectRnaSeqMetrics.html" title="class in picard.analysis">CollectRnaSeqMetrics</a></dt>
<dd>
<div class="block">Required main method implementation.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRrbsMetrics.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.analysis.<a href="picard/analysis/CollectRrbsMetrics.html" title="class in picard.analysis">CollectRrbsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.analysis.<a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.html" title="class in picard.analysis">CollectWgsMetricsWithNonZeroCoverage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CompareMetrics.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.analysis.<a href="picard/analysis/CompareMetrics.html" title="class in picard.analysis">CompareMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/CollectHsMetrics.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.analysis.directed.<a href="picard/analysis/directed/CollectHsMetrics.html" title="class in picard.analysis.directed">CollectHsMetrics</a></dt>
<dd>
<div class="block">Stock main method.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/CollectTargetedPcrMetrics.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.analysis.directed.<a href="picard/analysis/directed/CollectTargetedPcrMetrics.html" title="class in picard.analysis.directed">CollectTargetedPcrMetrics</a></dt>
<dd>
<div class="block">Stock main method.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/MeanQualityByCycle.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.analysis.<a href="picard/analysis/MeanQualityByCycle.html" title="class in picard.analysis">MeanQualityByCycle</a></dt>
<dd>
<div class="block">Required main method.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/QualityScoreDistribution.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.analysis.<a href="picard/analysis/QualityScoreDistribution.html" title="class in picard.analysis">QualityScoreDistribution</a></dt>
<dd>
<div class="block">Required main method.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/PicardCommandLine.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.cmdline.<a href="picard/cmdline/PicardCommandLine.html" title="class in picard.cmdline">PicardCommandLine</a></dt>
<dd>
<div class="block">Override this if you want to include different java packages to search for classes that extend CommandLineProgram.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fastq/BamToBfq.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.fastq.<a href="picard/fastq/BamToBfq.html" title="class in picard.fastq">BamToBfq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CheckFingerprint.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.fingerprint.<a href="picard/fingerprint/CheckFingerprint.html" title="class in picard.fingerprint">CheckFingerprint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CheckIlluminaDirectory.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.illumina.<a href="picard/illumina/CheckIlluminaDirectory.html" title="class in picard.illumina">CheckIlluminaDirectory</a></dt>
<dd>
<div class="block">Required main method implementation.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CollectIlluminaBasecallingMetrics.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.illumina.<a href="picard/illumina/CollectIlluminaBasecallingMetrics.html" title="class in picard.illumina">CollectIlluminaBasecallingMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CollectIlluminaLaneMetrics.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.illumina.<a href="picard/illumina/CollectIlluminaLaneMetrics.html" title="class in picard.illumina">CollectIlluminaLaneMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.html" title="class in picard.illumina">ExtractIlluminaBarcodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToFastq.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToFastq.html" title="class in picard.illumina">IlluminaBasecallsToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/MarkIlluminaAdapters.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.illumina.<a href="picard/illumina/MarkIlluminaAdapters.html" title="class in picard.illumina">MarkIlluminaAdapters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics</a></dt>
<dd>
<div class="block">Stock main method.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/reference/ExtractSequences.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.reference.<a href="picard/reference/ExtractSequences.html" title="class in picard.reference">ExtractSequences</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/reference/NonNFastaSize.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.reference.<a href="picard/reference/NonNFastaSize.html" title="class in picard.reference">NonNFastaSize</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/reference/NormalizeFasta.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.reference.<a href="picard/reference/NormalizeFasta.html" title="class in picard.reference">NormalizeFasta</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AddCommentsToBam.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.sam.<a href="picard/sam/AddCommentsToBam.html" title="class in picard.sam">AddCommentsToBam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/BamIndexStats.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.sam.<a href="picard/sam/BamIndexStats.html" title="class in picard.sam">BamIndexStats</a></dt>
<dd>
<div class="block">Stock main method for a command line program.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/BuildBamIndex.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.sam.<a href="picard/sam/BuildBamIndex.html" title="class in picard.sam">BuildBamIndex</a></dt>
<dd>
<div class="block">Stock main method for a command line program.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CalculateReadGroupChecksum.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.sam.<a href="picard/sam/CalculateReadGroupChecksum.html" title="class in picard.sam">CalculateReadGroupChecksum</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CheckTerminatorBlock.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.sam.<a href="picard/sam/CheckTerminatorBlock.html" title="class in picard.sam">CheckTerminatorBlock</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CleanSam.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.sam.<a href="picard/sam/CleanSam.html" title="class in picard.sam">CleanSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CompareSAMs.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.sam.<a href="picard/sam/CompareSAMs.html" title="class in picard.sam">CompareSAMs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CreateSequenceDictionary.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.sam.<a href="picard/sam/CreateSequenceDictionary.html" title="class in picard.sam">CreateSequenceDictionary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/DuplicationMetrics.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.sam.<a href="picard/sam/DuplicationMetrics.html" title="class in picard.sam">DuplicationMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FastqToSam.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.sam.<a href="picard/sam/FastqToSam.html" title="class in picard.sam">FastqToSam</a></dt>
<dd>
<div class="block">Stock main method.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FixMateInformation.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.sam.<a href="picard/sam/FixMateInformation.html" title="class in picard.sam">FixMateInformation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/GatherBamFiles.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.sam.<a href="picard/sam/GatherBamFiles.html" title="class in picard.sam">GatherBamFiles</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/EstimateLibraryComplexity.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/EstimateLibraryComplexity.html" title="class in picard.sam.markduplicates">EstimateLibraryComplexity</a></dt>
<dd>
<div class="block">Stock main method.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicates.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicates.html" title="class in picard.sam.markduplicates">MarkDuplicates</a></dt>
<dd>
<div class="block">Stock main method.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigar.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigar.html" title="class in picard.sam.markduplicates">MarkDuplicatesWithMateCigar</a></dt>
<dd>
<div class="block">Stock main method.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/SimpleMarkDuplicatesWithMateCigar.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/SimpleMarkDuplicatesWithMateCigar.html" title="class in picard.sam.markduplicates">SimpleMarkDuplicatesWithMateCigar</a></dt>
<dd>
<div class="block">Stock main method.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>
<div class="block">Required main method implementation.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeSamFiles.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.sam.<a href="picard/sam/MergeSamFiles.html" title="class in picard.sam">MergeSamFiles</a></dt>
<dd>
<div class="block">Required main method implementation.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ReorderSam.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.sam.<a href="picard/sam/ReorderSam.html" title="class in picard.sam">ReorderSam</a></dt>
<dd>
<div class="block">Required main method implementation.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ReplaceSamHeader.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.sam.<a href="picard/sam/ReplaceSamHeader.html" title="class in picard.sam">ReplaceSamHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertOriginalBaseQualitiesAndAddMateCigar.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.sam.<a href="picard/sam/RevertOriginalBaseQualitiesAndAddMateCigar.html" title="class in picard.sam">RevertOriginalBaseQualitiesAndAddMateCigar</a></dt>
<dd>
<div class="block">Default main method impl.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertSam.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.sam.<a href="picard/sam/RevertSam.html" title="class in picard.sam">RevertSam</a></dt>
<dd>
<div class="block">Default main method impl.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamFormatConverter.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.sam.<a href="picard/sam/SamFormatConverter.html" title="class in picard.sam">SamFormatConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamToFastq.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.sam.<a href="picard/sam/SamToFastq.html" title="class in picard.sam">SamToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SetNmMdAndUqTags.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.sam.<a href="picard/sam/SetNmMdAndUqTags.html" title="class in picard.sam">SetNmMdAndUqTags</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SortSam.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.sam.<a href="picard/sam/SortSam.html" title="class in picard.sam">SortSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SplitSamByLibrary.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.sam.<a href="picard/sam/SplitSamByLibrary.html" title="class in picard.sam">SplitSamByLibrary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ValidateSamFile.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.sam.<a href="picard/sam/ValidateSamFile.html" title="class in picard.sam">ValidateSamFile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ViewSam.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.sam.<a href="picard/sam/ViewSam.html" title="class in picard.sam">ViewSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/Test.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.<a href="picard/Test.html" title="class in picard">Test</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/BaitDesigner.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.util.<a href="picard/util/BaitDesigner.html" title="class in picard.util">BaitDesigner</a></dt>
<dd>
<div class="block">Stock main method.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/BedToIntervalList.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.util.<a href="picard/util/BedToIntervalList.html" title="class in picard.util">BedToIntervalList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/FifoBuffer.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.util.<a href="picard/util/FifoBuffer.html" title="class in picard.util">FifoBuffer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IntervalListTools.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.util.<a href="picard/util/IntervalListTools.html" title="class in picard.util">IntervalListTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/LiftOverIntervalList.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.util.<a href="picard/util/LiftOverIntervalList.html" title="class in picard.util">LiftOverIntervalList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/ScatterIntervalsByNs.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.util.<a href="picard/util/ScatterIntervalsByNs.html" title="class in picard.util">ScatterIntervalsByNs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/FilterVcf.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.vcf.filter.<a href="picard/vcf/filter/FilterVcf.html" title="class in picard.vcf.filter">FilterVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/FixVcfHeader.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.vcf.<a href="picard/vcf/FixVcfHeader.html" title="class in picard.vcf">FixVcfHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GatherVcfs.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.vcf.<a href="picard/vcf/GatherVcfs.html" title="class in picard.vcf">GatherVcfs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/LiftoverVcf.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.vcf.<a href="picard/vcf/LiftoverVcf.html" title="class in picard.vcf">LiftoverVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MakeSitesOnlyVcf.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.vcf.<a href="picard/vcf/MakeSitesOnlyVcf.html" title="class in picard.vcf">MakeSitesOnlyVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MergeVcfs.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.vcf.<a href="picard/vcf/MergeVcfs.html" title="class in picard.vcf">MergeVcfs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/RenameSampleInVcf.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.vcf.<a href="picard/vcf/RenameSampleInVcf.html" title="class in picard.vcf">RenameSampleInVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/SortVcf.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.vcf.<a href="picard/vcf/SortVcf.html" title="class in picard.vcf">SortVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/SplitVcfs.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.vcf.<a href="picard/vcf/SplitVcfs.html" title="class in picard.vcf">SplitVcfs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/UpdateVcfSequenceDictionary.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.vcf.<a href="picard/vcf/UpdateVcfSequenceDictionary.html" title="class in picard.vcf">UpdateVcfSequenceDictionary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/VcfFormatConverter.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.vcf.<a href="picard/vcf/VcfFormatConverter.html" title="class in picard.vcf">VcfFormatConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/VcfToIntervalList.html#main-java.lang.String:A-">main(String[])</a></span> - Static method in class picard.vcf.<a href="picard/vcf/VcfToIntervalList.html" title="class in picard.vcf">VcfToIntervalList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetMetricsCollector.html#makeAllReadCollector--">makeAllReadCollector()</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/TargetMetricsCollector.html" title="class in picard.analysis.directed">TargetMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/metrics/MultiLevelCollector.html#makeAllReadCollector--">makeAllReadCollector()</a></span> - Method in class picard.metrics.<a href="picard/metrics/MultiLevelCollector.html" title="class in picard.metrics">MultiLevelCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/InsertSizeMetricsCollector.html#makeArg-htsjdk.samtools.SAMRecord-htsjdk.samtools.reference.ReferenceSequence-">makeArg(SAMRecord, ReferenceSequence)</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/InsertSizeMetricsCollector.html" title="class in picard.analysis.directed">InsertSizeMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasMetricsCollector.html#makeArg-htsjdk.samtools.SAMRecord-htsjdk.samtools.reference.ReferenceSequence-">makeArg(SAMRecord, ReferenceSequence)</a></span> - Method in class picard.analysis.<a href="picard/analysis/GcBiasMetricsCollector.html" title="class in picard.analysis">GcBiasMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/metrics/MultiLevelCollector.html#makeArg-htsjdk.samtools.SAMRecord-htsjdk.samtools.reference.ReferenceSequence-">makeArg(SAMRecord, ReferenceSequence)</a></span> - Method in class picard.metrics.<a href="picard/metrics/MultiLevelCollector.html" title="class in picard.metrics">MultiLevelCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/metrics/SAMRecordAndReferenceMultiLevelCollector.html#makeArg-htsjdk.samtools.SAMRecord-htsjdk.samtools.reference.ReferenceSequence-">makeArg(SAMRecord, ReferenceSequence)</a></span> - Method in class picard.metrics.<a href="picard/metrics/SAMRecordAndReferenceMultiLevelCollector.html" title="class in picard.metrics">SAMRecordAndReferenceMultiLevelCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/metrics/SAMRecordMultiLevelCollector.html#makeArg-htsjdk.samtools.SAMRecord-htsjdk.samtools.reference.ReferenceSequence-">makeArg(SAMRecord, ReferenceSequence)</a></span> - Method in class picard.metrics.<a href="picard/metrics/SAMRecordMultiLevelCollector.html" title="class in picard.metrics">SAMRecordMultiLevelCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AlignmentSummaryMetricsCollector.html#makeChildCollector-java.lang.String-java.lang.String-java.lang.String-">makeChildCollector(String, String, String)</a></span> - Method in class picard.analysis.<a href="picard/analysis/AlignmentSummaryMetricsCollector.html" title="class in picard.analysis">AlignmentSummaryMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/InsertSizeMetricsCollector.html#makeChildCollector-java.lang.String-java.lang.String-java.lang.String-">makeChildCollector(String, String, String)</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/InsertSizeMetricsCollector.html" title="class in picard.analysis.directed">InsertSizeMetricsCollector</a></dt>
<dd>
<div class="block">Make an InsertSizeCollector with the given arguments</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/RnaSeqMetricsCollector.html#makeChildCollector-java.lang.String-java.lang.String-java.lang.String-">makeChildCollector(String, String, String)</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/RnaSeqMetricsCollector.html" title="class in picard.analysis.directed">RnaSeqMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetMetricsCollector.html#makeChildCollector-java.lang.String-java.lang.String-java.lang.String-">makeChildCollector(String, String, String)</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/TargetMetricsCollector.html" title="class in picard.analysis.directed">TargetMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasMetricsCollector.html#makeChildCollector-java.lang.String-java.lang.String-java.lang.String-">makeChildCollector(String, String, String)</a></span> - Method in class picard.analysis.<a href="picard/analysis/GcBiasMetricsCollector.html" title="class in picard.analysis">GcBiasMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RrbsMetricsCollector.html#makeChildCollector-java.lang.String-java.lang.String-java.lang.String-">makeChildCollector(String, String, String)</a></span> - Method in class picard.analysis.<a href="picard/analysis/RrbsMetricsCollector.html" title="class in picard.analysis">RrbsMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/metrics/MultiLevelCollector.html#makeChildCollector-java.lang.String-java.lang.String-java.lang.String-">makeChildCollector(String, String, String)</a></span> - Method in class picard.metrics.<a href="picard/metrics/MultiLevelCollector.html" title="class in picard.metrics">MultiLevelCollector</a></dt>
<dd>
<div class="block">Construct a PerUnitMetricCollector with the given arguments.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/CollectHsMetrics.html#makeCollector-java.util.Set-java.util.List-htsjdk.samtools.reference.ReferenceSequenceFile-java.io.File-java.io.File-htsjdk.samtools.util.IntervalList-htsjdk.samtools.util.IntervalList-java.lang.String-int-">makeCollector(Set&lt;MetricAccumulationLevel&gt;, List&lt;SAMReadGroupRecord&gt;, ReferenceSequenceFile, File, File, IntervalList, IntervalList, String, int)</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/CollectHsMetrics.html" title="class in picard.analysis.directed">CollectHsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/CollectTargetedMetrics.html#makeCollector-java.util.Set-java.util.List-htsjdk.samtools.reference.ReferenceSequenceFile-java.io.File-java.io.File-htsjdk.samtools.util.IntervalList-htsjdk.samtools.util.IntervalList-java.lang.String-int-">makeCollector(Set&lt;MetricAccumulationLevel&gt;, List&lt;SAMReadGroupRecord&gt;, ReferenceSequenceFile, File, File, IntervalList, IntervalList, String, int)</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/CollectTargetedMetrics.html" title="class in picard.analysis.directed">CollectTargetedMetrics</a></dt>
<dd>
<div class="block">A factory method for the TargetMetricsCollector to use this time.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/CollectTargetedPcrMetrics.html#makeCollector-java.util.Set-java.util.List-htsjdk.samtools.reference.ReferenceSequenceFile-java.io.File-java.io.File-htsjdk.samtools.util.IntervalList-htsjdk.samtools.util.IntervalList-java.lang.String-int-">makeCollector(Set&lt;MetricAccumulationLevel&gt;, List&lt;SAMReadGroupRecord&gt;, ReferenceSequenceFile, File, File, IntervalList, IntervalList, String, int)</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/CollectTargetedPcrMetrics.html" title="class in picard.analysis.directed">CollectTargetedPcrMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/MultiTileBclParser.html#makeCycleFileParser-java.util.List-picard.illumina.parser.PerTileCycleParser.CycleFilesParser-">makeCycleFileParser(List&lt;File&gt;, PerTileCycleParser.CycleFilesParser&lt;BclData&gt;)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/MultiTileBclParser.html" title="class in picard.illumina.parser">MultiTileBclParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/MultiTileBclParser.html#makeCycleFileParser-java.util.List-">makeCycleFileParser(List&lt;File&gt;)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/MultiTileBclParser.html" title="class in picard.illumina.parser">MultiTileBclParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaDataProviderFactory.html#makeDataProvider-java.util.List-java.util.List-java.io.File:A-int-java.io.File-">makeDataProvider(List&lt;File&gt;, List&lt;AbstractIlluminaPositionFileReader.PositionInfo&gt;, File[], int, File)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/IlluminaDataProviderFactory.html" title="class in picard.illumina.parser">IlluminaDataProviderFactory</a></dt>
<dd>
<div class="block">Call this method to create a ClusterData iterator over all clusters for a given tile.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaDataProviderFactory.html#makeDataProvider--">makeDataProvider()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/IlluminaDataProviderFactory.html" title="class in picard.illumina.parser">IlluminaDataProviderFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaDataProviderFactory.html#makeDataProvider-java.util.List-">makeDataProvider(List&lt;Integer&gt;)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/IlluminaDataProviderFactory.html" title="class in picard.illumina.parser">IlluminaDataProviderFactory</a></dt>
<dd>
<div class="block">Call this method to create a ClusterData iterator over the specified tiles.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.html#makeExceptionMsg--">makeExceptionMsg()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.html" title="class in picard.illumina.parser.readers">AbstractIlluminaPositionFileReader</a></dt>
<dd>
<div class="block">Create a string that will be included in any NoSuchElementException thrown by the next() method</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/ClocsFileReader.html#makeExceptionMsg--">makeExceptionMsg()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/ClocsFileReader.html" title="class in picard.illumina.parser.readers">ClocsFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/LocsFileReader.html#makeExceptionMsg--">makeExceptionMsg()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/LocsFileReader.html" title="class in picard.illumina.parser.readers">LocsFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/PosFileReader.html#makeExceptionMsg--">makeExceptionMsg()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/PosFileReader.html" title="class in picard.illumina.parser.readers">PosFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/RnaSeqMetricsCollector.html#makeIgnoredSequenceIndicesSet-htsjdk.samtools.SAMFileHeader-java.util.Set-">makeIgnoredSequenceIndicesSet(SAMFileHeader, Set&lt;String&gt;)</a></span> - Static method in class picard.analysis.directed.<a href="picard/analysis/directed/RnaSeqMetricsCollector.html" title="class in picard.analysis.directed">RnaSeqMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectMultipleMetrics.ProgramInterface.html#makeInstance-java.lang.String-java.lang.String-java.io.File-java.io.File-java.util.Set-java.io.File-java.io.File-boolean-">makeInstance(String, String, File, File, Set&lt;MetricAccumulationLevel&gt;, File, File, boolean)</a></span> - Method in interface picard.analysis.<a href="picard/analysis/CollectMultipleMetrics.ProgramInterface.html" title="interface in picard.analysis">CollectMultipleMetrics.ProgramInterface</a></dt>
<dd>
<div class="block">By default, this method calls the <a href="picard/analysis/CollectMultipleMetrics.ProgramInterface.html#makeInstance-java.lang.String-java.lang.String-java.io.File-java.io.File-java.util.Set-java.io.File-java.io.File-"><code>CollectMultipleMetrics.ProgramInterface.makeInstance(String, String, File, File, Set, File, File)</code></a> method without 'includeUnpaired' parameter.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectMultipleMetrics.ProgramInterface.html#makeInstance-java.lang.String-java.lang.String-java.io.File-java.io.File-java.util.Set-java.io.File-java.io.File-">makeInstance(String, String, File, File, Set&lt;MetricAccumulationLevel&gt;, File, File)</a></span> - Method in interface picard.analysis.<a href="picard/analysis/CollectMultipleMetrics.ProgramInterface.html" title="interface in picard.analysis">CollectMultipleMetrics.ProgramInterface</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.html#makeIntervalArgumentCollection--">makeIntervalArgumentCollection()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/SinglePassSamProgram.html#makeItSo-java.io.File-java.io.File-boolean-long-java.util.Collection-">makeItSo(File, File, boolean, long, Collection&lt;SinglePassSamProgram&gt;)</a></span> - Static method in class picard.analysis.<a href="picard/analysis/SinglePassSamProgram.html" title="class in picard.analysis">SinglePassSamProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FastqToSam.html#makeItSo-htsjdk.samtools.fastq.FastqReader-htsjdk.samtools.fastq.FastqReader-htsjdk.samtools.SAMFileWriter-">makeItSo(FastqReader, FastqReader, SAMFileWriter)</a></span> - Method in class picard.sam.<a href="picard/sam/FastqToSam.html" title="class in picard.sam">FastqToSam</a></dt>
<dd>
<div class="block">Handles the FastqToSam execution on the FastqReader(s).</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ParameterizedFileUtil.html#makeLaneTileRegex-java.lang.String-int-">makeLaneTileRegex(String, int)</a></span> - Static method in class picard.illumina.parser.<a href="picard/illumina/parser/ParameterizedFileUtil.html" title="class in picard.illumina.parser">ParameterizedFileUtil</a></dt>
<dd>
<div class="block">Return a regex string for finding Lane and Tile given a file extension pattern</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/metrics/MultiLevelCollector.html#makeLibraryCollector-htsjdk.samtools.SAMReadGroupRecord-">makeLibraryCollector(SAMReadGroupRecord)</a></span> - Method in class picard.metrics.<a href="picard/metrics/MultiLevelCollector.html" title="class in picard.metrics">MultiLevelCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/RnaSeqMetricsCollector.html#makeOverlapDetector-java.io.File-htsjdk.samtools.SAMFileHeader-java.io.File-htsjdk.samtools.util.Log-">makeOverlapDetector(File, SAMFileHeader, File, Log)</a></span> - Static method in class picard.analysis.directed.<a href="picard/analysis/directed/RnaSeqMetricsCollector.html" title="class in picard.analysis.directed">RnaSeqMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IlluminaUtil.html#makePhredBinaryFromSolexaQualityAscii_1_3-java.lang.String-">makePhredBinaryFromSolexaQualityAscii_1_3(String)</a></span> - Static method in class picard.util.<a href="picard/util/IlluminaUtil.html" title="class in picard.util">IlluminaUtil</a></dt>
<dd>
<div class="block">Convert from Solexa-scaled ASCII qualities to Phred-scaled binary.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/IlluminaUtil.html#makePhredBinaryFromSolexaQualityAscii_1_3-java.lang.String-int-int-">makePhredBinaryFromSolexaQualityAscii_1_3(String, int, int)</a></span> - Static method in class picard.util.<a href="picard/util/IlluminaUtil.html" title="class in picard.util">IlluminaUtil</a></dt>
<dd>
<div class="block">Convert from Solexa-scaled ASCII qualities to Phred-scaled binary.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/metrics/MultiLevelCollector.html#makeReadGroupCollector-htsjdk.samtools.SAMReadGroupRecord-">makeReadGroupCollector(SAMReadGroupRecord)</a></span> - Method in class picard.metrics.<a href="picard/metrics/MultiLevelCollector.html" title="class in picard.metrics">MultiLevelCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectAlignmentSummaryMetrics.html#makeReferenceArgumentCollection--">makeReferenceArgumentCollection()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectAlignmentSummaryMetrics.html" title="class in picard.analysis">CollectAlignmentSummaryMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRrbsMetrics.html#makeReferenceArgumentCollection--">makeReferenceArgumentCollection()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectRrbsMetrics.html" title="class in picard.analysis">CollectRrbsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#makeReferenceArgumentCollection--">makeReferenceArgumentCollection()</a></span> - Method in class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CreateSequenceDictionary.html#makeReferenceArgumentCollection--">makeReferenceArgumentCollection()</a></span> - Method in class picard.sam.<a href="picard/sam/CreateSequenceDictionary.html" title="class in picard.sam">CreateSequenceDictionary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/DownsampleSam.html#makeReferenceArgumentCollection--">makeReferenceArgumentCollection()</a></span> - Method in class picard.sam.<a href="picard/sam/DownsampleSam.html" title="class in picard.sam">DownsampleSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ReorderSam.html#makeReferenceArgumentCollection--">makeReferenceArgumentCollection()</a></span> - Method in class picard.sam.<a href="picard/sam/ReorderSam.html" title="class in picard.sam">ReorderSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/ScatterIntervalsByNs.html#makeReferenceArgumentCollection--">makeReferenceArgumentCollection()</a></span> - Method in class picard.util.<a href="picard/util/ScatterIntervalsByNs.html" title="class in picard.util">ScatterIntervalsByNs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/LiftoverVcf.html#makeReferenceArgumentCollection--">makeReferenceArgumentCollection()</a></span> - Method in class picard.vcf.<a href="picard/vcf/LiftoverVcf.html" title="class in picard.vcf">LiftoverVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/metrics/MultiLevelCollector.html#makeSampleCollector-htsjdk.samtools.SAMReadGroupRecord-">makeSampleCollector(SAMReadGroupRecord)</a></span> - Method in class picard.metrics.<a href="picard/metrics/MultiLevelCollector.html" title="class in picard.metrics">MultiLevelCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/BclReader.html#makeSeekable-java.util.List-picard.illumina.parser.readers.BclQualityEvaluationStrategy-int:A-">makeSeekable(List&lt;File&gt;, BclQualityEvaluationStrategy, int[])</a></span> - Static method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/BclReader.html" title="class in picard.illumina.parser.readers">BclReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CreateSequenceDictionary.html#makeSequenceDictionary-java.io.File-">makeSequenceDictionary(File)</a></span> - Method in class picard.sam.<a href="picard/sam/CreateSequenceDictionary.html" title="class in picard.sam">CreateSequenceDictionary</a></dt>
<dd>
<div class="block"><span class="deprecatedLabel">Deprecated.</span></div>
</dd>
<dt><a href="picard/vcf/MakeSitesOnlyVcf.html" title="class in picard.vcf"><span class="typeNameLink">MakeSitesOnlyVcf</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">Writes out a VCF that contains all the site-level information for all records in the input VCF and no per-sample information.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MakeSitesOnlyVcf.html#MakeSitesOnlyVcf--">MakeSitesOnlyVcf()</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/MakeSitesOnlyVcf.html" title="class in picard.vcf">MakeSitesOnlyVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PerTileParser.html#makeTileIterator-java.io.File-">makeTileIterator(File)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/PerTileParser.html" title="class in picard.illumina.parser">PerTileParser</a></dt>
<dd>
<div class="block">Factory method for the iterator of each tile</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PosParser.html#makeTileIterator-java.io.File-">makeTileIterator(File)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/PosParser.html" title="class in picard.illumina.parser">PosParser</a></dt>
<dd>
<div class="block">Make an CloseableIterator<PositionalData> based on the given file and fileType specified at construction.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/FindMendelianViolations.html#MALE_CHROMS">MALE_CHROMS</a></span> - Variable in class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/FindMendelianViolations.html" title="class in picard.vcf.MendelianViolations">FindMendelianViolations</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/markduplicates/MarkDuplicates.html" title="class in picard.sam.markduplicates"><span class="typeNameLink">MarkDuplicates</span></a> - Class in <a href="picard/sam/markduplicates/package-summary.html">picard.sam.markduplicates</a></dt>
<dd>
<div class="block">A better duplication marking algorithm that handles all cases including clipped
 and gapped alignments.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicates.html#MarkDuplicates--">MarkDuplicates()</a></span> - Constructor for class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicates.html" title="class in picard.sam.markduplicates">MarkDuplicates</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/markduplicates/MarkDuplicates.DuplicateTaggingPolicy.html" title="enum in picard.sam.markduplicates"><span class="typeNameLink">MarkDuplicates.DuplicateTaggingPolicy</span></a> - Enum in <a href="picard/sam/markduplicates/package-summary.html">picard.sam.markduplicates</a></dt>
<dd>
<div class="block">Enum used to control how duplicates are flagged in the DT optional tag on each read.</div>
</dd>
<dt><a href="picard/sam/markduplicates/MarkDuplicates.DuplicateType.html" title="enum in picard.sam.markduplicates"><span class="typeNameLink">MarkDuplicates.DuplicateType</span></a> - Enum in <a href="picard/sam/markduplicates/package-summary.html">picard.sam.markduplicates</a></dt>
<dd>
<div class="block">Enum for the possible values that a duplicate read can be tagged with in the DT attribute.</div>
</dd>
<dt><a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigar.html" title="class in picard.sam.markduplicates"><span class="typeNameLink">MarkDuplicatesWithMateCigar</span></a> - Class in <a href="picard/sam/markduplicates/package-summary.html">picard.sam.markduplicates</a></dt>
<dd>
<div class="block">An even better duplication marking algorithm that handles all cases including clipped
 and gapped alignments.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigar.html#MarkDuplicatesWithMateCigar--">MarkDuplicatesWithMateCigar()</a></span> - Constructor for class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigar.html" title="class in picard.sam.markduplicates">MarkDuplicatesWithMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigarIterator.html" title="class in picard.sam.markduplicates"><span class="typeNameLink">MarkDuplicatesWithMateCigarIterator</span></a> - Class in <a href="picard/sam/markduplicates/package-summary.html">picard.sam.markduplicates</a></dt>
<dd>
<div class="block">This will iterate through a coordinate sorted SAM file (iterator) and either mark or
 remove duplicates as appropriate.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigarIterator.html#MarkDuplicatesWithMateCigarIterator-htsjdk.samtools.SAMFileHeader-htsjdk.samtools.util.CloseableIterator-picard.sam.markduplicates.util.OpticalDuplicateFinder-htsjdk.samtools.DuplicateScoringStrategy.ScoringStrategy-int-boolean-boolean-int-int-java.util.List-">MarkDuplicatesWithMateCigarIterator(SAMFileHeader, CloseableIterator&lt;SAMRecord&gt;, OpticalDuplicateFinder, DuplicateScoringStrategy.ScoringStrategy, int, boolean, boolean, int, int, List&lt;File&gt;)</a></span> - Constructor for class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigarIterator.html" title="class in picard.sam.markduplicates">MarkDuplicatesWithMateCigarIterator</a></dt>
<dd>
<div class="block">Initializes the mark duplicates iterator.</div>
</dd>
<dt><a href="picard/illumina/MarkIlluminaAdapters.html" title="class in picard.illumina"><span class="typeNameLink">MarkIlluminaAdapters</span></a> - Class in <a href="picard/illumina/package-summary.html">picard.illumina</a></dt>
<dd>
<div class="block">Command line program to mark the location of adapter sequences.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/MarkIlluminaAdapters.html#MarkIlluminaAdapters--">MarkIlluminaAdapters()</a></span> - Constructor for class picard.illumina.<a href="picard/illumina/MarkIlluminaAdapters.html" title="class in picard.illumina">MarkIlluminaAdapters</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/markduplicates/util/MarkQueue.html" title="class in picard.sam.markduplicates.util"><span class="typeNameLink">MarkQueue</span></a> - Class in <a href="picard/sam/markduplicates/util/package-summary.html">picard.sam.markduplicates.util</a></dt>
<dd>
<div class="block">This is the mark queue.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/MarkQueue.html#MarkQueue-htsjdk.samtools.DuplicateScoringStrategy.ScoringStrategy-">MarkQueue(DuplicateScoringStrategy.ScoringStrategy)</a></span> - Constructor for class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/MarkQueue.html" title="class in picard.sam.markduplicates.util">MarkQueue</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#MATCHING_DICTIONARY_TAGS">MATCHING_DICTIONARY_TAGS</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ParameterizedFileUtil.html#matchPattern">matchPattern</a></span> - Variable in class picard.illumina.parser.<a href="picard/illumina/parser/ParameterizedFileUtil.html" title="class in picard.illumina.parser">ParameterizedFileUtil</a></dt>
<dd>
<div class="block">A pattern that will match files of this type for this lane</div>
</dd>
<dt><a href="picard/fingerprint/MatchResults.html" title="class in picard.fingerprint"><span class="typeNameLink">MatchResults</span></a> - Class in <a href="picard/fingerprint/package-summary.html">picard.fingerprint</a></dt>
<dd>
<div class="block">Represents the results of a fingerprint comparison between one dataset and a specific
 fingerprint file.</div>
</dd>
<dt><a href="picard/util/MathUtil.html" title="class in picard.util"><span class="typeNameLink">MathUtil</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">General math utilities</div>
</dd>
<dt><a href="picard/util/MathUtil.LogMath.html" title="class in picard.util"><span class="typeNameLink">MathUtil.LogMath</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">A collection of common math operations that work with log values.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/CollectIndependentReplicateMetrics.html#MATRIX_OUTPUT">MATRIX_OUTPUT</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/CollectIndependentReplicateMetrics.html" title="class in picard.analysis.replicates">CollectIndependentReplicateMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckFingerprints.html#MATRIX_OUTPUT">MATRIX_OUTPUT</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckFingerprints.html" title="class in picard.fingerprint">CrosscheckFingerprints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#max-double:A-">max(double[])</a></span> - Static method in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>
<div class="block">Returns the largest value stored in the array.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#max-long:A-">max(long[])</a></span> - Static method in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>
<div class="block">Returns the largest value stored in the array.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/EstimateLibraryComplexity.html#MAX_DIFF_RATE">MAX_DIFF_RATE</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/EstimateLibraryComplexity.html" title="class in picard.sam.markduplicates">EstimateLibraryComplexity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertSam.html#MAX_DISCARD_FRACTION">MAX_DISCARD_FRACTION</a></span> - Variable in class picard.sam.<a href="picard/sam/RevertSam.html" title="class in picard.sam">RevertSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/UmiAwareMarkDuplicatesWithMateCigar.html#MAX_EDIT_DISTANCE_TO_JOIN">MAX_EDIT_DISTANCE_TO_JOIN</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/UmiAwareMarkDuplicatesWithMateCigar.html" title="class in picard.sam.markduplicates">UmiAwareMarkDuplicatesWithMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/ClippingUtility.html#MAX_ERROR_RATE">MAX_ERROR_RATE</a></span> - Static variable in class picard.util.<a href="picard/util/ClippingUtility.html" title="class in picard.util">ClippingUtility</a></dt>
<dd>
<div class="block">The default value used for the maximum error rate when matching read bases to clippable sequence.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/MarkIlluminaAdapters.html#MAX_ERROR_RATE_PE">MAX_ERROR_RATE_PE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/MarkIlluminaAdapters.html" title="class in picard.illumina">MarkIlluminaAdapters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/MarkIlluminaAdapters.html#MAX_ERROR_RATE_SE">MAX_ERROR_RATE_SE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/MarkIlluminaAdapters.html" title="class in picard.illumina">MarkIlluminaAdapters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicates.html#MAX_FILE_HANDLES_FOR_READ_ENDS_MAP">MAX_FILE_HANDLES_FOR_READ_ENDS_MAP</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicates.html" title="class in picard.sam.markduplicates">MarkDuplicates</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/FilterVcf.html#MAX_FS">MAX_FS</a></span> - Variable in class picard.vcf.filter.<a href="picard/vcf/filter/FilterVcf.html" title="class in picard.vcf.filter">FilterVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/EstimateLibraryComplexity.html#MAX_GROUP_RATIO">MAX_GROUP_RATIO</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/EstimateLibraryComplexity.html" title="class in picard.sam.markduplicates">EstimateLibraryComplexity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectAlignmentSummaryMetrics.html#MAX_INSERT_SIZE">MAX_INSERT_SIZE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectAlignmentSummaryMetrics.html" title="class in picard.analysis">CollectAlignmentSummaryMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/InsertSizeMetrics.html#MAX_INSERT_SIZE">MAX_INSERT_SIZE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/InsertSizeMetrics.html" title="class in picard.analysis">InsertSizeMetrics</a></dt>
<dd>
<div class="block">The maximum measure insert size by alignment.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#MAX_INSERTIONS_OR_DELETIONS">MAX_INSERTIONS_OR_DELETIONS</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRrbsMetrics.html#MAX_MISMATCH_RATE">MAX_MISMATCH_RATE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectRrbsMetrics.html" title="class in picard.analysis">CollectRrbsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.html#MAX_MISMATCHES">MAX_MISMATCHES</a></span> - Variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.html" title="class in picard.illumina">ExtractIlluminaBarcodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.html#MAX_NO_CALLS">MAX_NO_CALLS</a></span> - Variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.html" title="class in picard.illumina">ExtractIlluminaBarcodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ValidateSamFile.html#MAX_OPEN_TEMP_FILES">MAX_OPEN_TEMP_FILES</a></span> - Variable in class picard.sam.<a href="picard/sam/ValidateSamFile.html" title="class in picard.sam">ValidateSamFile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/AbstractOpticalDuplicateFinderCommandLineProgram.html#MAX_OPTICAL_DUPLICATE_SET_SIZE">MAX_OPTICAL_DUPLICATE_SET_SIZE</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/AbstractOpticalDuplicateFinderCommandLineProgram.html" title="class in picard.sam.markduplicates.util">AbstractOpticalDuplicateFinderCommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ValidateSamFile.html#MAX_OUTPUT">MAX_OUTPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/ValidateSamFile.html" title="class in picard.sam">ValidateSamFile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/ClippingUtility.html#MAX_PE_ERROR_RATE">MAX_PE_ERROR_RATE</a></span> - Static variable in class picard.util.<a href="picard/util/ClippingUtility.html" title="class in picard.util">ClippingUtility</a></dt>
<dd>
<div class="block">The default value used for the maximum error rate when matching paired end read bases to clippable sequence.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.html#MAX_POS">MAX_POS</a></span> - Static variable in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.html" title="class in picard.illumina.parser.readers">AbstractIlluminaPositionFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#MAX_PROB_BELOW_ONE">MAX_PROB_BELOW_ONE</a></span> - Static variable in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>
<div class="block">The double value closest to 1 while still being less than 1.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FastqToSam.html#MAX_Q">MAX_Q</a></span> - Variable in class picard.sam.<a href="picard/sam/FastqToSam.html" title="class in picard.sam">FastqToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/EstimateLibraryComplexity.html#MAX_READ_LENGTH">MAX_READ_LENGTH</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/EstimateLibraryComplexity.html" title="class in picard.sam.markduplicates">EstimateLibraryComplexity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToFastq.html#MAX_READS_IN_RAM_PER_TILE">MAX_READS_IN_RAM_PER_TILE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToFastq.html" title="class in picard.illumina">IlluminaBasecallsToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#MAX_READS_IN_RAM_PER_TILE">MAX_READS_IN_RAM_PER_TILE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#MAX_RECORDS_IN_RAM">MAX_RECORDS_IN_RAM</a></span> - Variable in class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.html#MAX_RECORDS_IN_RAM">MAX_RECORDS_IN_RAM</a></span> - Static variable in class picard.sam.<a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertOriginalBaseQualitiesAndAddMateCigar.html#MAX_RECORDS_TO_EXAMINE">MAX_RECORDS_TO_EXAMINE</a></span> - Variable in class picard.sam.<a href="picard/sam/RevertOriginalBaseQualitiesAndAddMateCigar.html" title="class in picard.sam">RevertOriginalBaseQualitiesAndAddMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicates.html#MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP">MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicates.html" title="class in picard.sam.markduplicates">MarkDuplicates</a></dt>
<dd>
<div class="block">If more than this many sequences in SAM file, don't spill to disk because there will not
 be enough file handles.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#MAX_TARGET_COVERAGE">MAX_TARGET_COVERAGE</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The maximum coverage of reads that mapped to target regions of an experiment.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#MAX_TARGET_COVERAGE">MAX_TARGET_COVERAGE</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The maximum coverage of reads that mapped to target regions of an experiment.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/ScatterIntervalsByNs.html#MAX_TO_MERGE">MAX_TO_MERGE</a></span> - Variable in class picard.util.<a href="picard/util/ScatterIntervalsByNs.html" title="class in picard.util">ScatterIntervalsByNs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html#MAXIMUM_INSERT_SIZE">MAXIMUM_INSERT_SIZE</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html" title="class in picard.analysis.artifacts">CollectSequencingArtifactMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.html#MAXIMUM_INSERT_SIZE">MAXIMUM_INSERT_SIZE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.html" title="class in picard.analysis">CollectOxoGMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PerTileParser.html#maybeAdvance--">maybeAdvance()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/PerTileParser.html" title="class in picard.illumina.parser">PerTileParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.LogMath.html#mean-double...-">mean(double...)</a></span> - Method in class picard.util.<a href="picard/util/MathUtil.LogMath.html" title="class in picard.util">MathUtil.LogMath</a></dt>
<dd>
<div class="block">Computes the mean of the provided log values.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#mean-double:A-int-int-">mean(double[], int, int)</a></span> - Static method in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>
<div class="block">Calculated the mean of an array of doubles.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#MEAN_AMPLICON_COVERAGE">MEAN_AMPLICON_COVERAGE</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The mean read coverage of all amplicon regions in the experiment.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#MEAN_BAIT_COVERAGE">MEAN_BAIT_COVERAGE</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The mean coverage of all baits in the experiment.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasDetailMetrics.html#MEAN_BASE_QUALITY">MEAN_BASE_QUALITY</a></span> - Variable in class picard.analysis.<a href="picard/analysis/GcBiasDetailMetrics.html" title="class in picard.analysis">GcBiasDetailMetrics</a></dt>
<dd>
<div class="block">The mean quality (determined via the error rate) of all bases of all reads that are assigned to windows of this GC.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallingMetrics.html#MEAN_CLUSTERS_PER_TILE">MEAN_CLUSTERS_PER_TILE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallingMetrics.html" title="class in picard.illumina">IlluminaBasecallingMetrics</a></dt>
<dd>
<div class="block">The mean number of clusters per tile.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#MEAN_COVERAGE">MEAN_COVERAGE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">The mean coverage in bases of the genome territory, after all filters are applied.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RrbsSummaryMetrics.html#MEAN_CPG_COVERAGE">MEAN_CPG_COVERAGE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RrbsSummaryMetrics.html" title="class in picard.analysis">RrbsSummaryMetrics</a></dt>
<dd>
<div class="block">Mean coverage of CpG sites</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/InsertSizeMetrics.html#MEAN_INSERT_SIZE">MEAN_INSERT_SIZE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/InsertSizeMetrics.html" title="class in picard.analysis">InsertSizeMetrics</a></dt>
<dd>
<div class="block">The mean insert size of the "core" of the distribution.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallingMetrics.html#MEAN_PCT_PF_CLUSTERS_PER_TILE">MEAN_PCT_PF_CLUSTERS_PER_TILE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallingMetrics.html" title="class in picard.illumina">IlluminaBasecallingMetrics</a></dt>
<dd>
<div class="block">The mean percentage of pf clusters per tile.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallingMetrics.html#MEAN_PF_CLUSTERS_PER_TILE">MEAN_PF_CLUSTERS_PER_TILE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallingMetrics.html" title="class in picard.illumina">IlluminaBasecallingMetrics</a></dt>
<dd>
<div class="block">The mean number of pf clusters per tile.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AlignmentSummaryMetrics.html#MEAN_READ_LENGTH">MEAN_READ_LENGTH</a></span> - Variable in class picard.analysis.<a href="picard/analysis/AlignmentSummaryMetrics.html" title="class in picard.analysis">AlignmentSummaryMetrics</a></dt>
<dd>
<div class="block">The mean read length of the set of reads examined.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#MEAN_TARGET_COVERAGE">MEAN_TARGET_COVERAGE</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The mean coverage of a target region.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#MEAN_TARGET_COVERAGE">MEAN_TARGET_COVERAGE</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The mean read coverage of all target regions in an experiment.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/UmiMetrics.html#MEAN_UMI_LENGTH">MEAN_UMI_LENGTH</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/UmiMetrics.html" title="class in picard.sam.markduplicates">UmiMetrics</a></dt>
<dd>
<div class="block">Number of bases in each UMI</div>
</dd>
<dt><a href="picard/analysis/MeanQualityByCycle.html" title="class in picard.analysis"><span class="typeNameLink">MeanQualityByCycle</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Program to generate a data table and chart of mean quality by cycle from a
 BAM file.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/MeanQualityByCycle.html#MeanQualityByCycle--">MeanQualityByCycle()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/MeanQualityByCycle.html" title="class in picard.analysis">MeanQualityByCycle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#median-double...-">median(double...)</a></span> - Static method in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>
<div class="block">Calculate the median of an array of doubles.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RnaSeqMetrics.html#MEDIAN_3PRIME_BIAS">MEDIAN_3PRIME_BIAS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RnaSeqMetrics.html" title="class in picard.analysis">RnaSeqMetrics</a></dt>
<dd>
<div class="block">The median 3 prime bias of the 1000 most highly expressed transcripts, where 3 prime bias is calculated per
 transcript as: mean coverage of the 3 prime-most 100 bases divided by the mean coverage of the whole transcript.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RnaSeqMetrics.html#MEDIAN_5PRIME_BIAS">MEDIAN_5PRIME_BIAS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RnaSeqMetrics.html" title="class in picard.analysis">RnaSeqMetrics</a></dt>
<dd>
<div class="block">The median 5 prime bias of the 1000 most highly expressed transcripts.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RnaSeqMetrics.html#MEDIAN_5PRIME_TO_3PRIME_BIAS">MEDIAN_5PRIME_TO_3PRIME_BIAS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RnaSeqMetrics.html" title="class in picard.analysis">RnaSeqMetrics</a></dt>
<dd>
<div class="block">The ratio of coverage at the 5 prime end to the 3 prime end based on the 1000 most highly expressed transcripts.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/InsertSizeMetrics.html#MEDIAN_ABSOLUTE_DEVIATION">MEDIAN_ABSOLUTE_DEVIATION</a></span> - Variable in class picard.analysis.<a href="picard/analysis/InsertSizeMetrics.html" title="class in picard.analysis">InsertSizeMetrics</a></dt>
<dd>
<div class="block">The median absolute deviation of the distribution.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#MEDIAN_COVERAGE">MEDIAN_COVERAGE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">The median coverage in bases of the genome territory, after all filters are applied.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RrbsSummaryMetrics.html#MEDIAN_CPG_COVERAGE">MEDIAN_CPG_COVERAGE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RrbsSummaryMetrics.html" title="class in picard.analysis">RrbsSummaryMetrics</a></dt>
<dd>
<div class="block">Median coverage of CpG sites</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RnaSeqMetrics.html#MEDIAN_CV_COVERAGE">MEDIAN_CV_COVERAGE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RnaSeqMetrics.html" title="class in picard.analysis">RnaSeqMetrics</a></dt>
<dd>
<div class="block">The median coefficient of variation (CV) or stdev/mean for coverage values of the 1000 most highly expressed transcripts.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/InsertSizeMetrics.html#MEDIAN_INSERT_SIZE">MEDIAN_INSERT_SIZE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/InsertSizeMetrics.html" title="class in picard.analysis">InsertSizeMetrics</a></dt>
<dd>
<div class="block">The MEDIAN insert size of all paired end reads where both ends mapped to the same chromosome.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#MEDIAN_TARGET_COVERAGE">MEDIAN_TARGET_COVERAGE</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The median coverage of a target region.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#MEDIAN_TARGET_COVERAGE">MEDIAN_TARGET_COVERAGE</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The median coverage of reads that mapped to target regions of an experiment.</div>
</dd>
<dt><a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html" title="class in picard.vcf.MendelianViolations"><span class="typeNameLink">MendelianViolationMetrics</span></a> - Class in <a href="picard/vcf/MendelianViolations/package-summary.html">picard.vcf.MendelianViolations</a></dt>
<dd>
<div class="block">Describes the type and number of mendelian violations found within a Trio.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html#MendelianViolationMetrics--">MendelianViolationMetrics()</a></span> - Constructor for class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html" title="class in picard.vcf.MendelianViolations">MendelianViolationMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/MendelianViolations/MendelianViolationsByFamily.html" title="class in picard.vcf.MendelianViolations"><span class="typeNameLink">MendelianViolationsByFamily</span></a> - Class in <a href="picard/vcf/MendelianViolations/package-summary.html">picard.vcf.MendelianViolations</a></dt>
<dd>
<div class="block">Created by farjoun on 6/25/16.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/MendelianViolationsByFamily.html#MendelianViolationsByFamily--">MendelianViolationsByFamily()</a></span> - Constructor for class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/MendelianViolationsByFamily.html" title="class in picard.vcf.MendelianViolations">MendelianViolationsByFamily</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#merge-picard.analysis.MergeableMetricBase-">merge(MergeableMetricBase)</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">Merges the various PCT_EXC_* metrics.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/MergeableMetricBase.html#merge-java.util.Collection-">merge(Collection&lt;? extends MergeableMetricBase&gt;)</a></span> - Method in class picard.analysis.<a href="picard/analysis/MergeableMetricBase.html" title="class in picard.analysis">MergeableMetricBase</a></dt>
<dd>
<div class="block">for a collection of MergeableMetricBase, merge them all into "this" one.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/MergeableMetricBase.html#merge-picard.analysis.MergeableMetricBase-">merge(MergeableMetricBase)</a></span> - Method in class picard.analysis.<a href="picard/analysis/MergeableMetricBase.html" title="class in picard.analysis">MergeableMetricBase</a></dt>
<dd>
<div class="block">Merge another metric into this one</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/Fingerprint.html#merge-picard.fingerprint.Fingerprint-">merge(Fingerprint)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/Fingerprint.html" title="class in picard.fingerprint">Fingerprint</a></dt>
<dd>
<div class="block">Merges the likelihoods from the supplied Fingerprint into the likelihoods for this fingerprint.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintIdDetails.html#merge-picard.fingerprint.FingerprintIdDetails-">merge(FingerprintIdDetails)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/FingerprintIdDetails.html" title="class in picard.fingerprint">FingerprintIdDetails</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilities.html#merge-picard.fingerprint.HaplotypeProbabilities-">merge(HaplotypeProbabilities)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilities.html" title="class in picard.fingerprint">HaplotypeProbabilities</a></dt>
<dd>
<div class="block">Merges in the likelihood information from the supplied haplotype probabilities object.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilitiesFromContaminatorSequence.html#merge-picard.fingerprint.HaplotypeProbabilities-">merge(HaplotypeProbabilities)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilitiesFromContaminatorSequence.html" title="class in picard.fingerprint">HaplotypeProbabilitiesFromContaminatorSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilitiesFromGenotype.html#merge-picard.fingerprint.HaplotypeProbabilities-">merge(HaplotypeProbabilities)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilitiesFromGenotype.html" title="class in picard.fingerprint">HaplotypeProbabilitiesFromGenotype</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilitiesFromSequence.html#merge-picard.fingerprint.HaplotypeProbabilities-">merge(HaplotypeProbabilities)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilitiesFromSequence.html" title="class in picard.fingerprint">HaplotypeProbabilitiesFromSequence</a></dt>
<dd>
<div class="block">Merges information from another haplotype probabilities object for the same haplotype into
 this object.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilityOfNormalGivenTumor.html#merge-picard.fingerprint.HaplotypeProbabilities-">merge(HaplotypeProbabilities)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilityOfNormalGivenTumor.html" title="class in picard.fingerprint">HaplotypeProbabilityOfNormalGivenTumor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html#merge-picard.illumina.ExtractIlluminaBarcodes.BarcodeMetric-">merge(ExtractIlluminaBarcodes.BarcodeMetric)</a></span> - Method in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html" title="class in picard.illumina">ExtractIlluminaBarcodes.BarcodeMetric</a></dt>
<dd>
<div class="block">Adds the non-calculated</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html#merge-picard.illumina.quality.CollectHiSeqXPfFailMetrics.PFFailSummaryMetric-">merge(CollectHiSeqXPfFailMetrics.PFFailSummaryMetric)</a></span> - Method in class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics.PFFailSummaryMetric</a></dt>
<dd>
<div class="block">Adds the non-calculated fields from the other metric to this one.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CallingMetricAccumulator.Result.html#merge-java.util.Collection-">merge(Collection&lt;CallingMetricAccumulator.Result&gt;)</a></span> - Static method in class picard.vcf.<a href="picard/vcf/CallingMetricAccumulator.Result.html" title="class in picard.vcf">CallingMetricAccumulator.Result</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VariantProcessor.ResultMerger.html#merge-java.util.Collection-">merge(Collection&lt;RESULT&gt;)</a></span> - Method in interface picard.vcf.processor.<a href="picard/vcf/processor/VariantProcessor.ResultMerger.html" title="interface in picard.vcf.processor">VariantProcessor.ResultMerger</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/BaitDesigner.html#MERGE_NEARBY_TARGETS">MERGE_NEARBY_TARGETS</a></span> - Variable in class picard.util.<a href="picard/util/BaitDesigner.html" title="class in picard.util">BaitDesigner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeSamFiles.html#MERGE_SEQUENCE_DICTIONARIES">MERGE_SEQUENCE_DICTIONARIES</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeSamFiles.html" title="class in picard.sam">MergeSamFiles</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/MergeableMetricBase.html" title="class in picard.analysis"><span class="typeNameLink">MergeableMetricBase</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">An extension of MetricBase that knows how to merge-by-adding fields that are appropriately annotated.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/MergeableMetricBase.html#MergeableMetricBase--">MergeableMetricBase()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/MergeableMetricBase.html" title="class in picard.analysis">MergeableMetricBase</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/MergeableMetricBase.MergeByAdding.html" title="annotation in picard.analysis"><span class="typeNameLink">MergeableMetricBase.MergeByAdding</span></a> - Annotation Type in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Metrics whose values can be merged by adding.</div>
</dd>
<dt><a href="picard/analysis/MergeableMetricBase.MergeByAssertEquals.html" title="annotation in picard.analysis"><span class="typeNameLink">MergeableMetricBase.MergeByAssertEquals</span></a> - Annotation Type in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Metrics whose values should be equal when merging.</div>
</dd>
<dt><a href="picard/analysis/MergeableMetricBase.MergingIsManual.html" title="annotation in picard.analysis"><span class="typeNameLink">MergeableMetricBase.MergingIsManual</span></a> - Annotation Type in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Metrics that are merged manually in the <a href="picard/analysis/MergeableMetricBase.html#merge-picard.analysis.MergeableMetricBase-"><code>MergeableMetricBase.merge(MergeableMetricBase)</code></a> ()}.</div>
</dd>
<dt><a href="picard/analysis/MergeableMetricBase.NoMergingIsDerived.html" title="annotation in picard.analysis"><span class="typeNameLink">MergeableMetricBase.NoMergingIsDerived</span></a> - Annotation Type in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Metrics that are not merged, but are subsequently derived from other metrics, for example by
 <a href="picard/analysis/MergeableMetricBase.html#calculateDerivedFields--"><code>MergeableMetricBase.calculateDerivedFields()</code></a>.</div>
</dd>
<dt><a href="picard/analysis/MergeableMetricBase.NoMergingKeepsValue.html" title="annotation in picard.analysis"><span class="typeNameLink">MergeableMetricBase.NoMergingKeepsValue</span></a> - Annotation Type in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Metrics that are not merged.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.html#mergeAlignment-java.io.File-">mergeAlignment(File)</a></span> - Method in class picard.sam.<a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</a></dt>
<dd>
<div class="block">Merges the alignment data with the non-aligned records from the source BAM file.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamAlignmentMerger.html#mergeAlignment-java.io.File-">mergeAlignment(File)</a></span> - Method in class picard.sam.<a href="picard/sam/SamAlignmentMerger.html" title="class in picard.sam">SamAlignmentMerger</a></dt>
<dd>
<div class="block">Merges the alignment from the map file with the non-aligned records from the source BAM file.</div>
</dd>
<dt><a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam"><span class="typeNameLink">MergeBamAlignment</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block">A command-line tool to merge BAM/SAM alignment info from a third-party aligner with the data in an
 unmapped BAM file, producing a third BAM file that has alignment data and all the additional data
 from the unmapped BAM</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#MergeBamAlignment--">MergeBamAlignment()</a></span> - Constructor for class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckFingerprints.html#mergeFingerprintsBy-java.util.Map-java.util.function.Function-">mergeFingerprintsBy(Map&lt;FingerprintIdDetails, Fingerprint&gt;, Function&lt;FingerprintIdDetails, String&gt;)</a></span> - Static method in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckFingerprints.html" title="class in picard.fingerprint">CrosscheckFingerprints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/MergeableMetricBase.html#mergeIfCan-picard.analysis.MergeableMetricBase-">mergeIfCan(MergeableMetricBase)</a></span> - Method in class picard.analysis.<a href="picard/analysis/MergeableMetricBase.html" title="class in picard.analysis">MergeableMetricBase</a></dt>
<dd>
<div class="block">Merges another MergableMetricBase if possible</div>
</dd>
<dt><a href="picard/sam/MergeSamFiles.html" title="class in picard.sam"><span class="typeNameLink">MergeSamFiles</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block">Reads a SAM or BAM file and combines the output to one file</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeSamFiles.html#MergeSamFiles--">MergeSamFiles()</a></span> - Constructor for class picard.sam.<a href="picard/sam/MergeSamFiles.html" title="class in picard.sam">MergeSamFiles</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/MergeVcfs.html" title="class in picard.vcf"><span class="typeNameLink">MergeVcfs</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">Combines multiple VCF files into a single file.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MergeVcfs.html#MergeVcfs--">MergeVcfs()</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/MergeVcfs.html" title="class in picard.vcf">MergeVcfs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectAlignmentSummaryMetrics.html#METRIC_ACCUMULATION_LEVEL">METRIC_ACCUMULATION_LEVEL</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectAlignmentSummaryMetrics.html" title="class in picard.analysis">CollectAlignmentSummaryMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectGcBiasMetrics.html#METRIC_ACCUMULATION_LEVEL">METRIC_ACCUMULATION_LEVEL</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectGcBiasMetrics.html" title="class in picard.analysis">CollectGcBiasMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectInsertSizeMetrics.html#METRIC_ACCUMULATION_LEVEL">METRIC_ACCUMULATION_LEVEL</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectInsertSizeMetrics.html" title="class in picard.analysis">CollectInsertSizeMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectMultipleMetrics.html#METRIC_ACCUMULATION_LEVEL">METRIC_ACCUMULATION_LEVEL</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectMultipleMetrics.html" title="class in picard.analysis">CollectMultipleMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRnaSeqMetrics.html#METRIC_ACCUMULATION_LEVEL">METRIC_ACCUMULATION_LEVEL</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectRnaSeqMetrics.html" title="class in picard.analysis">CollectRnaSeqMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRrbsMetrics.html#METRIC_ACCUMULATION_LEVEL">METRIC_ACCUMULATION_LEVEL</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectRrbsMetrics.html" title="class in picard.analysis">CollectRrbsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/CollectTargetedMetrics.html#METRIC_ACCUMULATION_LEVEL">METRIC_ACCUMULATION_LEVEL</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/CollectTargetedMetrics.html" title="class in picard.analysis.directed">CollectTargetedMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/MetricAccumulationLevel.html" title="enum in picard.analysis"><span class="typeNameLink">MetricAccumulationLevel</span></a> - Enum in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">For use with Picard metrics programs that may output metrics for multiple levels
 of aggregation with an analysis.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/RnaSeqMetricsCollector.PerUnitRnaSeqMetricsCollector.html#metrics">metrics</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/RnaSeqMetricsCollector.PerUnitRnaSeqMetricsCollector.html" title="class in picard.analysis.directed">RnaSeqMetricsCollector.PerUnitRnaSeqMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/cmdline/programgroups/Metrics.html" title="class in picard.cmdline.programgroups"><span class="typeNameLink">Metrics</span></a> - Class in <a href="picard/cmdline/programgroups/package-summary.html">picard.cmdline.programgroups</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/programgroups/Metrics.html#Metrics--">Metrics()</a></span> - Constructor for class picard.cmdline.programgroups.<a href="picard/cmdline/programgroups/Metrics.html" title="class in picard.cmdline.programgroups">Metrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/MarkIlluminaAdapters.html#METRICS">METRICS</a></span> - Variable in class picard.illumina.<a href="picard/illumina/MarkIlluminaAdapters.html" title="class in picard.illumina">MarkIlluminaAdapters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.html#METRICS_FILE">METRICS_FILE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.html" title="class in picard.illumina">ExtractIlluminaBarcodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/DownsampleSam.html#METRICS_FILE">METRICS_FILE</a></span> - Variable in class picard.sam.<a href="picard/sam/DownsampleSam.html" title="class in picard.sam">DownsampleSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html#METRICS_FILE">METRICS_FILE</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html" title="class in picard.sam.markduplicates.util">AbstractMarkDuplicatesCommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRrbsMetrics.html#METRICS_FILE_PREFIX">METRICS_FILE_PREFIX</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectRrbsMetrics.html" title="class in picard.analysis">CollectRrbsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/StandardOptionDefinitions.html#METRICS_FILE_SHORT_NAME">METRICS_FILE_SHORT_NAME</a></span> - Static variable in class picard.cmdline.<a href="picard/cmdline/StandardOptionDefinitions.html" title="class in picard.cmdline">StandardOptionDefinitions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CompareMetrics.html#metricsFiles">metricsFiles</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CompareMetrics.html" title="class in picard.analysis">CompareMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#min-double:A-">min(double[])</a></span> - Static method in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>
<div class="block">Returns the smallest value stored in the array.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#min-int:A-">min(int[])</a></span> - Static method in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>
<div class="block">Returns the smallest value stored in the array.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#min-short:A-">min(short[])</a></span> - Static method in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>
<div class="block">Returns the smallest value stored in the array.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#min-byte:A-">min(byte[])</a></span> - Static method in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>
<div class="block">Returns the smallest value stored in the array.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/FilterVcf.html#MIN_AB">MIN_AB</a></span> - Variable in class picard.vcf.filter.<a href="picard/vcf/filter/FilterVcf.html" title="class in picard.vcf.filter">FilterVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/FilterVcf.html#MIN_DP">MIN_DP</a></span> - Variable in class picard.vcf.filter.<a href="picard/vcf/filter/FilterVcf.html" title="class in picard.vcf.filter">FilterVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#MIN_DP">MIN_DP</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/FindMendelianViolations.html#MIN_DP">MIN_DP</a></span> - Variable in class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/FindMendelianViolations.html" title="class in picard.vcf.MendelianViolations">FindMendelianViolations</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/FilterVcf.html#MIN_GQ">MIN_GQ</a></span> - Variable in class picard.vcf.filter.<a href="picard/vcf/filter/FilterVcf.html" title="class in picard.vcf.filter">FilterVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#MIN_GQ">MIN_GQ</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/FindMendelianViolations.html#MIN_GQ">MIN_GQ</a></span> - Variable in class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/FindMendelianViolations.html" title="class in picard.vcf.MendelianViolations">FindMendelianViolations</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/EstimateLibraryComplexity.html#MIN_GROUP_COUNT">MIN_GROUP_COUNT</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/EstimateLibraryComplexity.html" title="class in picard.sam.markduplicates">EstimateLibraryComplexity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/FindMendelianViolations.html#MIN_HET_FRACTION">MIN_HET_FRACTION</a></span> - Variable in class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/FindMendelianViolations.html" title="class in picard.vcf.MendelianViolations">FindMendelianViolations</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/EstimateLibraryComplexity.html#MIN_IDENTICAL_BASES">MIN_IDENTICAL_BASES</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/EstimateLibraryComplexity.html" title="class in picard.sam.markduplicates">EstimateLibraryComplexity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/InsertSizeMetrics.html#MIN_INSERT_SIZE">MIN_INSERT_SIZE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/InsertSizeMetrics.html" title="class in picard.analysis">InsertSizeMetrics</a></dt>
<dd>
<div class="block">The minimum measured insert size.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/LiftOverIntervalList.html#MIN_LIFTOVER_PCT">MIN_LIFTOVER_PCT</a></span> - Variable in class picard.util.<a href="picard/util/LiftOverIntervalList.html" title="class in picard.util">LiftOverIntervalList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/ClippingUtility.html#MIN_MATCH_BASES">MIN_MATCH_BASES</a></span> - Static variable in class picard.util.<a href="picard/util/ClippingUtility.html" title="class in picard.util">ClippingUtility</a></dt>
<dd>
<div class="block">The default value used for the minimum number of contiguous bases to match against.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/MarkIlluminaAdapters.html#MIN_MATCH_BASES_PE">MIN_MATCH_BASES_PE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/MarkIlluminaAdapters.html" title="class in picard.illumina">MarkIlluminaAdapters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/MarkIlluminaAdapters.html#MIN_MATCH_BASES_SE">MIN_MATCH_BASES_SE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/MarkIlluminaAdapters.html" title="class in picard.illumina">MarkIlluminaAdapters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/ClippingUtility.html#MIN_MATCH_PE_BASES">MIN_MATCH_PE_BASES</a></span> - Static variable in class picard.util.<a href="picard/util/ClippingUtility.html" title="class in picard.util">ClippingUtility</a></dt>
<dd>
<div class="block">The default value used for the minimum number of contiguous bases to match against in a paired end read</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/EstimateLibraryComplexity.html#MIN_MEAN_QUALITY">MIN_MEAN_QUALITY</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/EstimateLibraryComplexity.html" title="class in picard.sam.markduplicates">EstimateLibraryComplexity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.html#MIN_MISMATCH_DELTA">MIN_MISMATCH_DELTA</a></span> - Variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.html" title="class in picard.illumina">ExtractIlluminaBarcodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.html#MIN_POS">MIN_POS</a></span> - Static variable in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.html" title="class in picard.illumina.parser.readers">AbstractIlluminaPositionFileReader</a></dt>
<dd>
<div class="block">At least one NextSeq run produced a small negative value for y coordinate (-5), so allow small
 negative values and see what happens.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FastqToSam.html#MIN_Q">MIN_Q</a></span> - Variable in class picard.sam.<a href="picard/sam/FastqToSam.html" title="class in picard.sam">FastqToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/FilterVcf.html#MIN_QD">MIN_QD</a></span> - Variable in class picard.vcf.filter.<a href="picard/vcf/filter/FilterVcf.html" title="class in picard.vcf.filter">FilterVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#MIN_UNCLIPPED_BASES">MIN_UNCLIPPED_BASES</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/BaitDesigner.html#MINIMUM_BAITS_PER_TARGET">MINIMUM_BAITS_PER_TARGET</a></span> - Variable in class picard.util.<a href="picard/util/BaitDesigner.html" title="class in picard.util">BaitDesigner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/CollectIndependentReplicateMetrics.html#MINIMUM_BARCODE_BQ">MINIMUM_BARCODE_BQ</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/CollectIndependentReplicateMetrics.html" title="class in picard.analysis.replicates">CollectIndependentReplicateMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.html#MINIMUM_BASE_QUALITY">MINIMUM_BASE_QUALITY</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/CollectTargetedMetrics.html#MINIMUM_BASE_QUALITY">MINIMUM_BASE_QUALITY</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/CollectTargetedMetrics.html" title="class in picard.analysis.directed">CollectTargetedMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.html#MINIMUM_BASE_QUALITY">MINIMUM_BASE_QUALITY</a></span> - Variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.html" title="class in picard.illumina">ExtractIlluminaBarcodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/CollectIndependentReplicateMetrics.html#MINIMUM_BQ">MINIMUM_BQ</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/CollectIndependentReplicateMetrics.html" title="class in picard.analysis.replicates">CollectIndependentReplicateMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigar.html#MINIMUM_DISTANCE">MINIMUM_DISTANCE</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigar.html" title="class in picard.sam.markduplicates">MarkDuplicatesWithMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectGcBiasMetrics.html#MINIMUM_GENOME_FRACTION">MINIMUM_GENOME_FRACTION</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectGcBiasMetrics.html" title="class in picard.analysis">CollectGcBiasMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/CollectIndependentReplicateMetrics.html#MINIMUM_GQ">MINIMUM_GQ</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/CollectIndependentReplicateMetrics.html" title="class in picard.analysis.replicates">CollectIndependentReplicateMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html#MINIMUM_INSERT_SIZE">MINIMUM_INSERT_SIZE</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html" title="class in picard.analysis.artifacts">CollectSequencingArtifactMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.html#MINIMUM_INSERT_SIZE">MINIMUM_INSERT_SIZE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.html" title="class in picard.analysis">CollectOxoGMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRnaSeqMetrics.html#MINIMUM_LENGTH">MINIMUM_LENGTH</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectRnaSeqMetrics.html" title="class in picard.analysis">CollectRnaSeqMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/StandardOptionDefinitions.html#MINIMUM_LOD_SHORT_NAME">MINIMUM_LOD_SHORT_NAME</a></span> - Static variable in class picard.cmdline.<a href="picard/cmdline/StandardOptionDefinitions.html" title="class in picard.cmdline">StandardOptionDefinitions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html#MINIMUM_MAPPING_QUALITY">MINIMUM_MAPPING_QUALITY</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html" title="class in picard.analysis.artifacts">CollectSequencingArtifactMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectJumpingLibraryMetrics.html#MINIMUM_MAPPING_QUALITY">MINIMUM_MAPPING_QUALITY</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectJumpingLibraryMetrics.html" title="class in picard.analysis">CollectJumpingLibraryMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.html#MINIMUM_MAPPING_QUALITY">MINIMUM_MAPPING_QUALITY</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.html" title="class in picard.analysis">CollectOxoGMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.html#MINIMUM_MAPPING_QUALITY">MINIMUM_MAPPING_QUALITY</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/CollectTargetedMetrics.html#MINIMUM_MAPPING_QUALITY">MINIMUM_MAPPING_QUALITY</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/CollectTargetedMetrics.html" title="class in picard.analysis.directed">CollectTargetedMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/StandardOptionDefinitions.html#MINIMUM_MAPPING_QUALITY_SHORT_NAME">MINIMUM_MAPPING_QUALITY_SHORT_NAME</a></span> - Static variable in class picard.cmdline.<a href="picard/cmdline/StandardOptionDefinitions.html" title="class in picard.cmdline">StandardOptionDefinitions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/CollectIndependentReplicateMetrics.html#MINIMUM_MQ">MINIMUM_MQ</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/CollectIndependentReplicateMetrics.html" title="class in picard.analysis.replicates">CollectIndependentReplicateMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectInsertSizeMetrics.html#MINIMUM_PCT">MINIMUM_PCT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectInsertSizeMetrics.html" title="class in picard.analysis">CollectInsertSizeMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.html#MINIMUM_QUALITY">MINIMUM_QUALITY</a></span> - Variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.html" title="class in picard.illumina">ExtractIlluminaBarcodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToFastq.html#MINIMUM_QUALITY">MINIMUM_QUALITY</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToFastq.html" title="class in picard.illumina">IlluminaBasecallsToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#MINIMUM_QUALITY">MINIMUM_QUALITY</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html#MINIMUM_QUALITY_SCORE">MINIMUM_QUALITY_SCORE</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html" title="class in picard.analysis.artifacts">CollectSequencingArtifactMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.html#MINIMUM_QUALITY_SCORE">MINIMUM_QUALITY_SCORE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.html" title="class in picard.analysis">CollectOxoGMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRrbsMetrics.html#MINIMUM_READ_LENGTH">MINIMUM_READ_LENGTH</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectRrbsMetrics.html" title="class in picard.analysis">CollectRrbsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#MISSING_SITES_HOM_REF">MISSING_SITES_HOM_REF</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/readers/MMapBackedIteratorFactory.html" title="class in picard.illumina.parser.readers"><span class="typeNameLink">MMapBackedIteratorFactory</span></a> - Class in <a href="picard/illumina/parser/readers/package-summary.html">picard.illumina.parser.readers</a></dt>
<dd>
<div class="block">MMapBackedIteratorFactory a file reader that takes a header size and a binary file, maps the file to
 a read-only byte buffer and provides methods to retrieve the header as it's own bytebuffer and create
 iterators of different data types over the values of file (starting after the end of the header).</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/MMapBackedIteratorFactory.html#MMapBackedIteratorFactory--">MMapBackedIteratorFactory()</a></span> - Constructor for class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/MMapBackedIteratorFactory.html" title="class in picard.illumina.parser.readers">MMapBackedIteratorFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ValidateSamFile.html#MODE">MODE</a></span> - Variable in class picard.sam.<a href="picard/sam/ValidateSamFile.html" title="class in picard.sam">ValidateSamFile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallingMetrics.html#MOLECULAR_BARCODE_NAME">MOLECULAR_BARCODE_NAME</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallingMetrics.html" title="class in picard.illumina">IlluminaBasecallingMetrics</a></dt>
<dd>
<div class="block">The barcode name for which the metrics were calculated.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallingMetrics.html#MOLECULAR_BARCODE_SEQUENCE_1">MOLECULAR_BARCODE_SEQUENCE_1</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallingMetrics.html" title="class in picard.illumina">IlluminaBasecallingMetrics</a></dt>
<dd>
<div class="block">The barcode sequence for which the metrics were calculated.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#MOLECULAR_INDEX_BASE_QUALITY_TAG">MOLECULAR_INDEX_BASE_QUALITY_TAG</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#MOLECULAR_INDEX_TAG">MOLECULAR_INDEX_TAG</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadStructure.html#molecularBarcode">molecularBarcode</a></span> - Variable in class picard.illumina.parser.<a href="picard/illumina/parser/ReadStructure.html" title="class in picard.illumina.parser">ReadStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadType.html#MolecularIndex">MolecularIndex</a></span> - Static variable in enum picard.illumina.parser.<a href="picard/illumina/parser/ReadType.html" title="enum in picard.illumina.parser">ReadType</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/MostDistantPrimaryAlignmentSelectionStrategy.html" title="class in picard.sam"><span class="typeNameLink">MostDistantPrimaryAlignmentSelectionStrategy</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block">For a paired-end aligner that aligns each end independently, select the pair of alignments that result
 in the largest insert size.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MostDistantPrimaryAlignmentSelectionStrategy.html#MostDistantPrimaryAlignmentSelectionStrategy--">MostDistantPrimaryAlignmentSelectionStrategy()</a></span> - Constructor for class picard.sam.<a href="picard/sam/MostDistantPrimaryAlignmentSelectionStrategy.html" title="class in picard.sam">MostDistantPrimaryAlignmentSelectionStrategy</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html#MOTHER">MOTHER</a></span> - Variable in class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html" title="class in picard.vcf.MendelianViolations">MendelianViolationMetrics</a></dt>
<dd>
<div class="block">The ID of the mother within the trio.</div>
</dd>
<dt><a href="picard/metrics/MultiLevelCollector.html" title="class in picard.metrics"><span class="typeNameLink">MultiLevelCollector</span></a>&lt;<a href="picard/metrics/MultiLevelCollector.html" title="type parameter in MultiLevelCollector">METRIC_TYPE</a> extends htsjdk.samtools.metrics.MetricBase,<a href="picard/metrics/MultiLevelCollector.html" title="type parameter in MultiLevelCollector">Histogram_KEY</a> extends java.lang.Comparable,<a href="picard/metrics/MultiLevelCollector.html" title="type parameter in MultiLevelCollector">ARGTYPE</a>&gt; - Class in <a href="picard/metrics/package-summary.html">picard.metrics</a></dt>
<dd>
<div class="block">MultiLevelCollector handles accumulating Metrics at different MetricAccumulationLevels(ALL_READS, SAMPLE, LIBRARY, READ_GROUP).</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/metrics/MultiLevelCollector.html#MultiLevelCollector--">MultiLevelCollector()</a></span> - Constructor for class picard.metrics.<a href="picard/metrics/MultiLevelCollector.html" title="class in picard.metrics">MultiLevelCollector</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/metrics/MultilevelMetrics.html" title="class in picard.metrics"><span class="typeNameLink">MultilevelMetrics</span></a> - Class in <a href="picard/metrics/package-summary.html">picard.metrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/metrics/MultilevelMetrics.html#MultilevelMetrics--">MultilevelMetrics()</a></span> - Constructor for class picard.metrics.<a href="picard/metrics/MultilevelMetrics.html" title="class in picard.metrics">MultilevelMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToFastq.html#MULTIPLEX_PARAMS">MULTIPLEX_PARAMS</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToFastq.html" title="class in picard.illumina">IlluminaBasecallsToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#multiply-double:A-double:A-">multiply(double[], double[])</a></span> - Static method in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>
<div class="block">Calculates the product of two arrays of the same length.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VariantAccumulatorExecutor.MultiThreadedChunkBased.html#MultiThreadedChunkBased-int-picard.vcf.processor.VariantIteratorProducer-picard.vcf.processor.VariantProcessor.AccumulatorGenerator-">MultiThreadedChunkBased(int, VariantIteratorProducer, VariantProcessor.AccumulatorGenerator&lt;A, R&gt;)</a></span> - Constructor for class picard.vcf.processor.<a href="picard/vcf/processor/VariantAccumulatorExecutor.MultiThreadedChunkBased.html" title="class in picard.vcf.processor">VariantAccumulatorExecutor.MultiThreadedChunkBased</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VariantProcessor.Builder.html#multithreadingBy-int-">multithreadingBy(int)</a></span> - Method in class picard.vcf.processor.<a href="picard/vcf/processor/VariantProcessor.Builder.html" title="class in picard.vcf.processor">VariantProcessor.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/fakers/MultiTileBclFileFaker.html" title="class in picard.illumina.parser.fakers"><span class="typeNameLink">MultiTileBclFileFaker</span></a> - Class in <a href="picard/illumina/parser/fakers/package-summary.html">picard.illumina.parser.fakers</a></dt>
<dd>
<div class="block">Created by jcarey on 3/13/14.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/MultiTileBclFileFaker.html#MultiTileBclFileFaker--">MultiTileBclFileFaker()</a></span> - Constructor for class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/MultiTileBclFileFaker.html" title="class in picard.illumina.parser.fakers">MultiTileBclFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/MultiTileBclFileUtil.html" title="class in picard.illumina.parser"><span class="typeNameLink">MultiTileBclFileUtil</span></a> - Class in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>
<div class="block">NextSeq-style bcl's have all tiles for a cycle in a single file.</div>
</dd>
<dt><a href="picard/illumina/parser/MultiTileBclParser.html" title="class in picard.illumina.parser"><span class="typeNameLink">MultiTileBclParser</span></a> - Class in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>
<div class="block">Parse .bcl.bgzf files that contain multiple tiles in a single file.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/MultiTileBclParser.html#MultiTileBclParser-java.io.File-int-picard.illumina.parser.CycleIlluminaFileMap-picard.illumina.parser.OutputMapping-boolean-picard.illumina.parser.readers.BclQualityEvaluationStrategy-picard.illumina.parser.TileIndex-">MultiTileBclParser(File, int, CycleIlluminaFileMap, OutputMapping, boolean, BclQualityEvaluationStrategy, TileIndex)</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/MultiTileBclParser.html" title="class in picard.illumina.parser">MultiTileBclParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/MultiTileFileUtil.html" title="class in picard.illumina.parser"><span class="typeNameLink">MultiTileFileUtil</span></a>&lt;<a href="picard/illumina/parser/MultiTileFileUtil.html" title="type parameter in MultiTileFileUtil">OUTPUT_RECORD</a> extends picard.illumina.parser.IlluminaData&gt; - Class in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>
<div class="block">For file types for which there is one file per lane, with fixed record size, and all the tiles in it,
 so the s_<lane>.bci file can be used to figure out where each tile starts and ends.</div>
</dd>
<dt><a href="picard/illumina/parser/MultiTileFilterParser.html" title="class in picard.illumina.parser"><span class="typeNameLink">MultiTileFilterParser</span></a> - Class in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>
<div class="block">Read filter file that contains multiple tiles in a single file.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/MultiTileFilterParser.html#MultiTileFilterParser-picard.illumina.parser.TileIndex-java.util.List-java.io.File-">MultiTileFilterParser(TileIndex, List&lt;Integer&gt;, File)</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/MultiTileFilterParser.html" title="class in picard.illumina.parser">MultiTileFilterParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/fakers/MultiTileLocsFileFaker.html" title="class in picard.illumina.parser.fakers"><span class="typeNameLink">MultiTileLocsFileFaker</span></a> - Class in <a href="picard/illumina/parser/fakers/package-summary.html">picard.illumina.parser.fakers</a></dt>
<dd>
<div class="block">Created by jcarey on 3/13/14.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/MultiTileLocsFileFaker.html#MultiTileLocsFileFaker--">MultiTileLocsFileFaker()</a></span> - Constructor for class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/MultiTileLocsFileFaker.html" title="class in picard.illumina.parser.fakers">MultiTileLocsFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/MultiTileLocsParser.html" title="class in picard.illumina.parser"><span class="typeNameLink">MultiTileLocsParser</span></a> - Class in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>
<div class="block">Read locs file that contains multiple tiles in a single file.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/MultiTileLocsParser.html#MultiTileLocsParser-picard.illumina.parser.TileIndex-java.util.List-java.io.File-int-">MultiTileLocsParser(TileIndex, List&lt;Integer&gt;, File, int)</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/MultiTileLocsParser.html" title="class in picard.illumina.parser">MultiTileLocsParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/MultiTileParser.html" title="class in picard.illumina.parser"><span class="typeNameLink">MultiTileParser</span></a>&lt;<a href="picard/illumina/parser/MultiTileParser.html" title="type parameter in MultiTileParser">OUTPUT_RECORD</a> extends picard.illumina.parser.IlluminaData&gt; - Class in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>
<div class="block">Abstract class for files with fixed-length records for multiple tiles, e.g.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/MultiTileParser.html#MultiTileParser-picard.illumina.parser.TileIndex-java.util.List-java.util.Set-">MultiTileParser(TileIndex, List&lt;Integer&gt;, Set&lt;IlluminaDataType&gt;)</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/MultiTileParser.html" title="class in picard.illumina.parser">MultiTileParser</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="I:N">
<!--   -->
</a>
<h2 class="title">N</h2>
<dl>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.html#N_CYCLES">N_CYCLES</a></span> - Variable in class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceScheme.html#NA">NA</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceScheme.html" title="class in picard.vcf">GenotypeConcordanceScheme</a></dt>
<dd>
<div class="block">These are convenience variables for defining a scheme.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#nAlternateAllelesBiDups">nAlternateAllelesBiDups</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#nAlternateAllelesTriDups">nAlternateAllelesTriDups</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#nAlternateReads">nAlternateReads</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/Gene.Transcript.html#name">name</a></span> - Variable in class picard.annotation.<a href="picard/annotation/Gene.Transcript.html" title="class in picard.annotation">Gene.Transcript</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/FifoBuffer.html#NAME">NAME</a></span> - Variable in class picard.util.<a href="picard/util/FifoBuffer.html" title="class in picard.util">FifoBuffer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#NATURAL_LOG_MATH">NATURAL_LOG_MATH</a></span> - Static variable in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#nBadBarcodes">nBadBarcodes</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#nDifferentAllelesBiDups">nDifferentAllelesBiDups</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#nDifferentAllelesTriDups">nDifferentAllelesTriDups</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#nDuplicateSets">nDuplicateSets</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#NEAR_AMPLICON_BASES">NEAR_AMPLICON_BASES</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The number of PF_BASES_ALIGNED that mapped to within a fixed interval of an amplified region, but not on a
 baited region.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#NEAR_BAIT_BASES">NEAR_BAIT_BASES</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The number of PF_BASES_ALIGNED that are mapped to within a fixed interval containing a baited region, but not within the baited section per se.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/CollectTargetedMetrics.html#NEAR_DISTANCE">NEAR_DISTANCE</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/CollectTargetedMetrics.html" title="class in picard.analysis.directed">CollectTargetedMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetMetricsCollector.html#NEAR_PROBE_DISTANCE_DEFAULT">NEAR_PROBE_DISTANCE_DEFAULT</a></span> - Static variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetMetricsCollector.html" title="class in picard.analysis.directed">TargetMetricsCollector</a></dt>
<dd>
<div class="block">Default distance for a read to be considered "selected".</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectMultipleMetrics.ProgramInterface.html#needsReferenceSequence--">needsReferenceSequence()</a></span> - Method in interface picard.analysis.<a href="picard/analysis/CollectMultipleMetrics.ProgramInterface.html" title="interface in picard.analysis">CollectMultipleMetrics.ProgramInterface</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/RenameSampleInVcf.html#NEW_SAMPLE_NAME">NEW_SAMPLE_NAME</a></span> - Variable in class picard.vcf.<a href="picard/vcf/RenameSampleInVcf.html" title="class in picard.vcf">RenameSampleInVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CounterManager.html#newCounter--">newCounter()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CounterManager.html" title="class in picard.analysis">CounterManager</a></dt>
<dd>
<div class="block">Creates a new Counter object and adds it to the list of managed Counters.</div>
</dd>
<dt><a href="picard/illumina/NewIlluminaBasecallsConverter.html" title="class in picard.illumina"><span class="typeNameLink">NewIlluminaBasecallsConverter</span></a>&lt;<a href="picard/illumina/NewIlluminaBasecallsConverter.html" title="type parameter in NewIlluminaBasecallsConverter">CLUSTER_OUTPUT_RECORD</a>&gt; - Class in <a href="picard/illumina/package-summary.html">picard.illumina</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/NewIlluminaBasecallsConverter.html#NewIlluminaBasecallsConverter-java.io.File-java.io.File-int-picard.illumina.parser.ReadStructure-java.util.Map-boolean-int-java.util.List-int-java.lang.Integer-java.lang.Integer-java.util.Comparator-htsjdk.samtools.util.SortingCollection.Codec-java.lang.Class-picard.illumina.parser.readers.BclQualityEvaluationStrategy-boolean-">NewIlluminaBasecallsConverter(File, File, int, ReadStructure, Map&lt;String, ? extends BasecallsConverter.ConvertedClusterDataWriter&lt;CLUSTER_OUTPUT_RECORD&gt;&gt;, boolean, int, List&lt;File&gt;, int, Integer, Integer, Comparator&lt;CLUSTER_OUTPUT_RECORD&gt;, SortingCollection.Codec&lt;CLUSTER_OUTPUT_RECORD&gt;, Class&lt;CLUSTER_OUTPUT_RECORD&gt;, BclQualityEvaluationStrategy, boolean)</a></span> - Constructor for class picard.illumina.<a href="picard/illumina/NewIlluminaBasecallsConverter.html" title="class in picard.illumina">NewIlluminaBasecallsConverter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/NewIlluminaDataProvider.html" title="class in picard.illumina.parser"><span class="typeNameLink">NewIlluminaDataProvider</span></a> - Class in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>
<div class="block">Parse cbcls Illumina Basecall files, and use them the to populate
 ClusterData objects.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#nExactlyDouble">nExactlyDouble</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#nExactlyTriple">nExactlyTriple</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaDataProvider.html#next--">next()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/IlluminaDataProvider.html" title="class in picard.illumina.parser">IlluminaDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/MultiTileParser.html#next--">next()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/MultiTileParser.html" title="class in picard.illumina.parser">MultiTileParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/NewIlluminaDataProvider.html#next--">next()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/NewIlluminaDataProvider.html" title="class in picard.illumina.parser">NewIlluminaDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PerTileParser.html#next--">next()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/PerTileParser.html" title="class in picard.illumina.parser">PerTileParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.html#next--">next()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.html" title="class in picard.illumina.parser.readers">AbstractIlluminaPositionFileReader</a></dt>
<dd>
<div class="block">Return the next set of coordinates in a given file.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/BarcodeFileReader.html#next--">next()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/BarcodeFileReader.html" title="class in picard.illumina.parser.readers">BarcodeFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/BclReader.html#next--">next()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/BclReader.html" title="class in picard.illumina.parser.readers">BclReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/CbclReader.html#next--">next()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/CbclReader.html" title="class in picard.illumina.parser.readers">CbclReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/EmpiricalPhasingMetricsOutReader.html#next--">next()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/EmpiricalPhasingMetricsOutReader.html" title="class in picard.illumina.parser.readers">EmpiricalPhasingMetricsOutReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/FilterFileReader.html#next--">next()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/FilterFileReader.html" title="class in picard.illumina.parser.readers">FilterFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/TileMetricsOutReader.html#next--">next()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/TileMetricsOutReader.html" title="class in picard.illumina.parser.readers">TileMetricsOutReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigarIterator.html#next--">next()</a></span> - Method in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigarIterator.html" title="class in picard.sam.markduplicates">MarkDuplicatesWithMateCigarIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AsyncIterator.html#next--">next()</a></span> - Method in class picard.util.<a href="picard/util/AsyncIterator.html" title="class in picard.util">AsyncIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AtomicIterator.html#next--">next()</a></span> - Method in interface picard.util.<a href="picard/util/AtomicIterator.html" title="interface in picard.util">AtomicIterator</a></dt>
<dd>
<div class="block">Produces the next element from the iterator, if there is one; otherwise, produces <code>Optional.EMPTY</code></div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/DelimitedTextFileWithHeaderIterator.html#next--">next()</a></span> - Method in class picard.util.<a href="picard/util/DelimitedTextFileWithHeaderIterator.html" title="class in picard.util">DelimitedTextFileWithHeaderIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/ByIntervalListVariantContextIterator.html#next--">next()</a></span> - Method in class picard.vcf.<a href="picard/vcf/ByIntervalListVariantContextIterator.html" title="class in picard.vcf">ByIntervalListVariantContextIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/FilterApplyingVariantIterator.html#next--">next()</a></span> - Method in class picard.vcf.filter.<a href="picard/vcf/filter/FilterApplyingVariantIterator.html" title="class in picard.vcf.filter">FilterApplyingVariantIterator</a></dt>
<dd>
<div class="block">Provides the next record from the underlying iterator after applying filter strings generated
 by the set of filters in use by the iterator.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/PairedVariantSubContextIterator.html#next--">next()</a></span> - Method in class picard.vcf.<a href="picard/vcf/PairedVariantSubContextIterator.html" title="class in picard.vcf">PairedVariantSubContextIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/util/PredicateFilterDecoratingClosableIterator.html#next--">next()</a></span> - Method in class picard.vcf.processor.util.<a href="picard/vcf/processor/util/PredicateFilterDecoratingClosableIterator.html" title="class in picard.vcf.processor.util">PredicateFilterDecoratingClosableIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRrbsMetrics.html#NEXT_BASE_QUALITY_THRESHOLD">NEXT_BASE_QUALITY_THRESHOLD</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectRrbsMetrics.html" title="class in picard.analysis">CollectRrbsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#nGoodBarcodes">nGoodBarcodes</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#nMatchingUMIsInDiffBiDups">nMatchingUMIsInDiffBiDups</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#nMatchingUMIsInSameBiDups">nMatchingUMIsInSameBiDups</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#nMismatchingAllelesBiDups">nMismatchingAllelesBiDups</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#nMismatchingAllelesTriDups">nMismatchingAllelesTriDups</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#nMismatchingUMIsInContraOrientedBiDups">nMismatchingUMIsInContraOrientedBiDups</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#nMismatchingUMIsInCoOrientedBiDups">nMismatchingUMIsInCoOrientedBiDups</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#nMismatchingUMIsInDiffBiDups">nMismatchingUMIsInDiffBiDups</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#nMismatchingUMIsInSameBiDups">nMismatchingUMIsInSameBiDups</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SplitSamByLibrary.html#NO_LIBRARIES_SPECIFIED_IN_HEADER">NO_LIBRARIES_SPECIFIED_IN_HEADER</a></span> - Static variable in class picard.sam.<a href="picard/sam/SplitSamByLibrary.html" title="class in picard.sam">SplitSamByLibrary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/ClippingUtility.html#NO_MATCH">NO_MATCH</a></span> - Static variable in class picard.util.<a href="picard/util/ClippingUtility.html" title="class in picard.util">ClippingUtility</a></dt>
<dd>
<div class="block">The value returned by methods returning int when no match is found.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/pedigree/PedFile.html#NO_PHENO">NO_PHENO</a></span> - Static variable in class picard.pedigree.<a href="picard/pedigree/PedFile.html" title="class in picard.pedigree">PedFile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/PhysicalLocation.html#NO_VALUE">NO_VALUE</a></span> - Static variable in interface picard.sam.util.<a href="picard/sam/util/PhysicalLocation.html" title="interface in picard.sam.util">PhysicalLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RrbsSummaryMetrics.html#NON_CPG_BASES">NON_CPG_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RrbsSummaryMetrics.html" title="class in picard.analysis">RrbsSummaryMetrics</a></dt>
<dd>
<div class="block">Number of times a non-CpG cytosine was encountered</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RrbsSummaryMetrics.html#NON_CPG_CONVERTED_BASES">NON_CPG_CONVERTED_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RrbsSummaryMetrics.html" title="class in picard.analysis">RrbsSummaryMetrics</a></dt>
<dd>
<div class="block">Number of times a non-CpG cytosine was converted (C->T for +, G->A for -)</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/AlleleSubsettingUtils.html#NON_REF_ALLELE">NON_REF_ALLELE</a></span> - Static variable in class picard.util.<a href="picard/util/AlleleSubsettingUtils.html" title="class in picard.util">AlleleSubsettingUtils</a></dt>
<dd>
<div class="block"><span class="deprecatedLabel">Deprecated.</span>
<div class="block"><span class="deprecationComment">from inception, as one should use a version of this from htsjdk.
 Will be deleted when htsjdk includes a replacement and that version is in picard.</span></div>
</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html#NON_REF_GENOTYPE_CONCORDANCE">NON_REF_GENOTYPE_CONCORDANCE</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html" title="class in picard.vcf">GenotypeConcordanceSummaryMetrics</a></dt>
<dd>
<div class="block">The non-ref genotype concordance, ie for all var states only.</div>
</dd>
<dt><a href="picard/cmdline/programgroups/None.html" title="class in picard.cmdline.programgroups"><span class="typeNameLink">None</span></a> - Class in <a href="picard/cmdline/programgroups/package-summary.html">picard.cmdline.programgroups</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/programgroups/None.html#None--">None()</a></span> - Constructor for class picard.cmdline.programgroups.<a href="picard/cmdline/programgroups/None.html" title="class in picard.cmdline.programgroups">None</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/JumpingLibraryMetrics.html#NONJUMP_DUPLICATE_PAIRS">NONJUMP_DUPLICATE_PAIRS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/JumpingLibraryMetrics.html" title="class in picard.analysis">JumpingLibraryMetrics</a></dt>
<dd>
<div class="block">The number of inward-facing pais that are duplicates</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/JumpingLibraryMetrics.html#NONJUMP_DUPLICATE_PCT">NONJUMP_DUPLICATE_PCT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/JumpingLibraryMetrics.html" title="class in picard.analysis">JumpingLibraryMetrics</a></dt>
<dd>
<div class="block">The fraction of inward-facing pairs that are marked as duplicates</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/JumpingLibraryMetrics.html#NONJUMP_LIBRARY_SIZE">NONJUMP_LIBRARY_SIZE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/JumpingLibraryMetrics.html" title="class in picard.analysis">JumpingLibraryMetrics</a></dt>
<dd>
<div class="block">The estimated library size for inward-facing pairs</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/JumpingLibraryMetrics.html#NONJUMP_MEAN_INSERT_SIZE">NONJUMP_MEAN_INSERT_SIZE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/JumpingLibraryMetrics.html" title="class in picard.analysis">JumpingLibraryMetrics</a></dt>
<dd>
<div class="block">The mean insert size for inward-facing pairs</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/JumpingLibraryMetrics.html#NONJUMP_PAIRS">NONJUMP_PAIRS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/JumpingLibraryMetrics.html" title="class in picard.analysis">JumpingLibraryMetrics</a></dt>
<dd>
<div class="block">The number of inward-facing pairs in the SAM file</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/JumpingLibraryMetrics.html#NONJUMP_STDEV_INSERT_SIZE">NONJUMP_STDEV_INSERT_SIZE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/JumpingLibraryMetrics.html" title="class in picard.analysis">JumpingLibraryMetrics</a></dt>
<dd>
<div class="block">The standard deviation on the insert size for inward-facing pairs</div>
</dd>
<dt><a href="picard/reference/NonNFastaSize.html" title="class in picard.reference"><span class="typeNameLink">NonNFastaSize</span></a> - Class in <a href="picard/reference/package-summary.html">picard.reference</a></dt>
<dd>
<div class="block">A tool to count the number of non-N bases in a fasta file</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/reference/NonNFastaSize.html#NonNFastaSize--">NonNFastaSize()</a></span> - Constructor for class picard.reference.<a href="picard/reference/NonNFastaSize.html" title="class in picard.reference">NonNFastaSize</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadStructure.html#nonSkips">nonSkips</a></span> - Variable in class picard.illumina.parser.<a href="picard/illumina/parser/ReadStructure.html" title="class in picard.illumina.parser">ReadStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#normalizeAlleles-htsjdk.variant.variantcontext.VariantContext-java.lang.String-htsjdk.variant.variantcontext.VariantContext-java.lang.String-java.lang.Boolean-">normalizeAlleles(VariantContext, String, VariantContext, String, Boolean)</a></span> - Static method in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>
<div class="block">Gets the alleles for the truth and call genotypes.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasDetailMetrics.html#NORMALIZED_COVERAGE">NORMALIZED_COVERAGE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/GcBiasDetailMetrics.html" title="class in picard.analysis">GcBiasDetailMetrics</a></dt>
<dd>
<div class="block">The ratio of "coverage" in this GC bin vs.</div>
</dd>
<dt><a href="picard/reference/NormalizeFasta.html" title="class in picard.reference"><span class="typeNameLink">NormalizeFasta</span></a> - Class in <a href="picard/reference/package-summary.html">picard.reference</a></dt>
<dd>
<div class="block">Little program to "normalize" a fasta file to ensure that all line of sequence are the
 same length, and are a reasonable length!</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/reference/NormalizeFasta.html#NormalizeFasta--">NormalizeFasta()</a></span> - Constructor for class picard.reference.<a href="picard/reference/NormalizeFasta.html" title="class in picard.reference">NormalizeFasta</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/TheoreticalSensitivity.html#normalizeHistogram-htsjdk.samtools.util.Histogram-">normalizeHistogram(Histogram&lt;Integer&gt;)</a></span> - Static method in class picard.analysis.<a href="picard/analysis/TheoreticalSensitivity.html" title="class in picard.analysis">TheoreticalSensitivity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html#NOVEL_INDELS">NOVEL_INDELS</a></span> - Variable in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingSummaryMetrics</a></dt>
<dd>
<div class="block">The number of passing indels called that were not found in dbSNP</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html#NOVEL_INS_DEL_RATIO">NOVEL_INS_DEL_RATIO</a></span> - Variable in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingSummaryMetrics</a></dt>
<dd>
<div class="block">The Insertion/Deletion ratio of the indel calls made at non-dbSNP sites</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html#NOVEL_SNPS">NOVEL_SNPS</a></span> - Variable in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingSummaryMetrics</a></dt>
<dd>
<div class="block">The number of passing bi-allelic SNPS called that were not found in dbSNP</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html#NOVEL_TITV">NOVEL_TITV</a></span> - Variable in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingSummaryMetrics</a></dt>
<dd>
<div class="block">The Transition/Transversion ratio of the passing bi-allelic SNP calls made at non-dbSNP sites</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#nReadsInBigSets">nReadsInBigSets</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#nReferenceAllelesBiDups">nReferenceAllelesBiDups</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#nReferenceAllelesTriDups">nReferenceAllelesTriDups</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#nReferenceReads">nReferenceReads</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#nSites">nSites</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#nThreeAllelesSites">nThreeAllelesSites</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#nTotalReads">nTotalReads</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/MarkIlluminaAdapters.html#NUM_ADAPTERS_TO_KEEP">NUM_ADAPTERS_TO_KEEP</a></span> - Variable in class picard.illumina.<a href="picard/illumina/MarkIlluminaAdapters.html" title="class in picard.illumina">MarkIlluminaAdapters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IntensityChannel.html#NUM_CHANNELS">NUM_CHANNELS</a></span> - Static variable in enum picard.illumina.parser.<a href="picard/illumina/parser/IntensityChannel.html" title="enum in picard.illumina.parser">IntensityChannel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html#NUM_DIPLOID_DENOVO">NUM_DIPLOID_DENOVO</a></span> - Variable in class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html" title="class in picard.vcf.MendelianViolations">MendelianViolationMetrics</a></dt>
<dd>
<div class="block">The number of diploid sites at which a potential de-novo mutation was observed (i.e.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html#NUM_HAPLOID_DENOVO">NUM_HAPLOID_DENOVO</a></span> - Variable in class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html" title="class in picard.vcf.MendelianViolations">MendelianViolationMetrics</a></dt>
<dd>
<div class="block">The number of sites at which the offspring is haploid, the parent is homozygous reference and the offspring is non-reference.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html#NUM_HAPLOID_OTHER">NUM_HAPLOID_OTHER</a></span> - Variable in class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html" title="class in picard.vcf.MendelianViolations">MendelianViolationMetrics</a></dt>
<dd>
<div class="block">The number of sites at which the offspring is haploid and exhibits a reference allele that is not present in the parent.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html#NUM_HOM_HET_HOM">NUM_HOM_HET_HOM</a></span> - Variable in class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html" title="class in picard.vcf.MendelianViolations">MendelianViolationMetrics</a></dt>
<dd>
<div class="block">The number of sites at which one parent is homozygous, the other is heterozygous and the offspring is the alternative homozygote.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html#NUM_HOMREF_HOMVAR_HOM">NUM_HOMREF_HOMVAR_HOM</a></span> - Variable in class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html" title="class in picard.vcf.MendelianViolations">MendelianViolationMetrics</a></dt>
<dd>
<div class="block">The number of sites at which the one parent is homozygous reference, the other homozygous variant and the offspring is homozygous.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html#NUM_HOMVAR_HOMVAR_HET">NUM_HOMVAR_HOMVAR_HET</a></span> - Variable in class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html" title="class in picard.vcf.MendelianViolations">MendelianViolationMetrics</a></dt>
<dd>
<div class="block">The number of sites at which both parents are homozygous for a non-reference allele and the offspring is heterozygous.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html#NUM_IN_DB_SNP">NUM_IN_DB_SNP</a></span> - Variable in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingSummaryMetrics</a></dt>
<dd>
<div class="block">The number of passing bi-allelic SNPs found in dbSNP</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html#NUM_IN_DB_SNP_COMPLEX_INDELS">NUM_IN_DB_SNP_COMPLEX_INDELS</a></span> - Variable in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingSummaryMetrics</a></dt>
<dd>
<div class="block">The number of passing complex indels found in dbSNP</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html#NUM_IN_DB_SNP_INDELS">NUM_IN_DB_SNP_INDELS</a></span> - Variable in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingSummaryMetrics</a></dt>
<dd>
<div class="block">The number of passing indels found in dbSNP</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html#NUM_IN_DB_SNP_MULTIALLELIC">NUM_IN_DB_SNP_MULTIALLELIC</a></span> - Variable in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingSummaryMetrics</a></dt>
<dd>
<div class="block">The number of passing multi-allelic SNPs found in dbSNP</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailDetailedMetric.html#NUM_N">NUM_N</a></span> - Variable in class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailDetailedMetric.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics.PFFailDetailedMetric</a></dt>
<dd>
<div class="block">The number of Ns found in this read</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html#NUM_OTHER">NUM_OTHER</a></span> - Variable in class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html" title="class in picard.vcf.MendelianViolations">MendelianViolationMetrics</a></dt>
<dd>
<div class="block">The number of otherwise unclassified events.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.html#NUM_PROCESSORS">NUM_PROCESSORS</a></span> - Variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.html" title="class in picard.illumina">ExtractIlluminaBarcodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToFastq.html#NUM_PROCESSORS">NUM_PROCESSORS</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToFastq.html" title="class in picard.illumina">IlluminaBasecallsToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#NUM_PROCESSORS">NUM_PROCESSORS</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.html#NUM_PROCESSORS">NUM_PROCESSORS</a></span> - Variable in class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailDetailedMetric.html#NUM_Q_GT_TWO">NUM_Q_GT_TWO</a></span> - Variable in class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailDetailedMetric.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics.PFFailDetailedMetric</a></dt>
<dd>
<div class="block">The number of Quality scores greater than 2 found in this read</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RnaSeqMetrics.html#NUM_R1_TRANSCRIPT_STRAND_READS">NUM_R1_TRANSCRIPT_STRAND_READS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RnaSeqMetrics.html" title="class in picard.analysis">RnaSeqMetrics</a></dt>
<dd>
<div class="block">The number of reads that support the model where R1 is on the strand of transcription and R2 is on the
 opposite strand.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RnaSeqMetrics.html#NUM_R2_TRANSCRIPT_STRAND_READS">NUM_R2_TRANSCRIPT_STRAND_READS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RnaSeqMetrics.html" title="class in picard.analysis">RnaSeqMetrics</a></dt>
<dd>
<div class="block">The fraction of reads that support the model where R2 is on the strand of transcription and R1 is on the opposite
 strand.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CreateSequenceDictionary.html#NUM_SEQUENCES">NUM_SEQUENCES</a></span> - Variable in class picard.sam.<a href="picard/sam/CreateSequenceDictionary.html" title="class in picard.sam">CreateSequenceDictionary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html#NUM_SINGLETONS">NUM_SINGLETONS</a></span> - Variable in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingSummaryMetrics</a></dt>
<dd>
<div class="block">For summary metrics, the number of variants that appear in only one sample.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckFingerprints.html#NUM_THREADS">NUM_THREADS</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckFingerprints.html" title="class in picard.fingerprint">CrosscheckFingerprints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RnaSeqMetrics.html#NUM_UNEXPLAINED_READS">NUM_UNEXPLAINED_READS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RnaSeqMetrics.html" title="class in picard.analysis">RnaSeqMetrics</a></dt>
<dd>
<div class="block">The fraction of reads for which the transcription strand model could not be inferred.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html#NUM_VARIANT_SITES">NUM_VARIANT_SITES</a></span> - Variable in class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html" title="class in picard.vcf.MendelianViolations">MendelianViolationMetrics</a></dt>
<dd>
<div class="block">The number of biallelic, SNP sites at which all relevant samples exceeded the minimum genotype quality and depth and at least one of the samples was variant.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/FilterFileReader.html#numClusters">numClusters</a></span> - Variable in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/FilterFileReader.html" title="class in picard.illumina.parser.readers">FilterFileReader</a></dt>
<dd>
<div class="block">The number of cluster's pf values stored in this file</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/OutputMapping.html#numOutputReads--">numOutputReads()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/OutputMapping.html" title="class in picard.illumina.parser">OutputMapping</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="I:O">
<!--   -->
</a>
<h2 class="title">O</h2>
<dl>
<dt><span class="memberNameLink"><a href="picard/analysis/FingerprintingDetailMetrics.html#OBS_A">OBS_A</a></span> - Variable in class picard.analysis.<a href="picard/analysis/FingerprintingDetailMetrics.html" title="class in picard.analysis">FingerprintingDetailMetrics</a></dt>
<dd>
<div class="block">The number of observations of the first, or A,  allele of the SNP in the sequencing data.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/FingerprintingDetailMetrics.html#OBS_B">OBS_B</a></span> - Variable in class picard.analysis.<a href="picard/analysis/FingerprintingDetailMetrics.html" title="class in picard.analysis">FingerprintingDetailMetrics</a></dt>
<dd>
<div class="block">The number of observations of the second, or B, allele of the SNP in the sequencing data.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilitiesFromSequence.html#obsAllele1">obsAllele1</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilitiesFromSequence.html" title="class in picard.fingerprint">HaplotypeProbabilitiesFromSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilitiesFromSequence.html#obsAllele2">obsAllele2</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilitiesFromSequence.html" title="class in picard.fingerprint">HaplotypeProbabilitiesFromSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilitiesFromSequence.html#obsAlleleOther">obsAlleleOther</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilitiesFromSequence.html" title="class in picard.fingerprint">HaplotypeProbabilitiesFromSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/UmiMetrics.html#OBSERVED_BASE_ERRORS">OBSERVED_BASE_ERRORS</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/UmiMetrics.html" title="class in picard.sam.markduplicates">UmiMetrics</a></dt>
<dd>
<div class="block">Number of errors inferred by comparing the observed and inferred UMIs</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/FingerprintingDetailMetrics.html#OBSERVED_GENOTYPE">OBSERVED_GENOTYPE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/FingerprintingDetailMetrics.html" title="class in picard.analysis">FingerprintingDetailMetrics</a></dt>
<dd>
<div class="block">The most likely genotype given the observed evidence at the SNP locus in the sequencing data.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CheckFingerprint.html#OBSERVED_SAMPLE_ALIAS">OBSERVED_SAMPLE_ALIAS</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CheckFingerprint.html" title="class in picard.fingerprint">CheckFingerprint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/UmiMetrics.html#OBSERVED_UMI_ENTROPY">OBSERVED_UMI_ENTROPY</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/UmiMetrics.html" title="class in picard.sam.markduplicates">UmiMetrics</a></dt>
<dd>
<div class="block">Entropy (in base 4) of the observed UMI sequences, indicating the
 effective number of bases in the UMIs.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/UmiMetrics.html#OBSERVED_UNIQUE_UMIS">OBSERVED_UNIQUE_UMIS</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/UmiMetrics.html" title="class in picard.sam.markduplicates">UmiMetrics</a></dt>
<dd>
<div class="block">Number of different UMI sequences observed</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#OFF_AMPLICON_BASES">OFF_AMPLICON_BASES</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The number of PF_BASES_ALIGNED that mapped neither on or near an amplicon.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#OFF_BAIT_BASES">OFF_BAIT_BASES</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The number of PF_BASES_ALIGNED that are mapped away from any baited region.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html#OFFSPRING">OFFSPRING</a></span> - Variable in class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html" title="class in picard.vcf.MendelianViolations">MendelianViolationMetrics</a></dt>
<dd>
<div class="block">The ID of the offspring within the trio.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html#OFFSPRING_SEX">OFFSPRING_SEX</a></span> - Variable in class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html" title="class in picard.vcf.MendelianViolations">MendelianViolationMetrics</a></dt>
<dd>
<div class="block">The sex of the offspring.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/RenameSampleInVcf.html#OLD_SAMPLE_NAME">OLD_SAMPLE_NAME</a></span> - Variable in class picard.vcf.<a href="picard/vcf/RenameSampleInVcf.html" title="class in picard.vcf">RenameSampleInVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#ON_AMPLICON_BASES">ON_AMPLICON_BASES</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The number of PF_BASES_ALIGNED that mapped to an amplified region of the genome.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#ON_AMPLICON_VS_SELECTED">ON_AMPLICON_VS_SELECTED</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The fraction of bases mapping to regions on or near amplicons, which mapped directly to but not near
 amplicons, ON_AMPLICON_BASES/(NEAR_AMPLICON_BASES + ON_AMPLICON_BASES)</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#ON_BAIT_BASES">ON_BAIT_BASES</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The number of PF_BASES_ALIGNED that are mapped to the baited regions of the genome.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#ON_BAIT_VS_SELECTED">ON_BAIT_VS_SELECTED</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The fraction of bases on or near baits that are covered by baits, ON_BAIT_BASES/(ON_BAIT_BASES + NEAR_BAIT_BASES).</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#ON_TARGET_BASES">ON_TARGET_BASES</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The number of PF_BASES_ALIGNED that are mapped to a targeted region of the genome.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#ON_TARGET_BASES">ON_TARGET_BASES</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The number of PF_BASES_ALIGNED that mapped to a targeted region of the genome.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#ON_TARGET_FROM_PAIR_BASES">ON_TARGET_FROM_PAIR_BASES</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The number of bases from PF_SELECTED_UNIQUE_PAIRS that mapped to a targeted region of the genome.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html#ONE_MISMATCH_MATCHES">ONE_MISMATCH_MATCHES</a></span> - Variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html" title="class in picard.illumina">ExtractIlluminaBarcodes.BarcodeMetric</a></dt>
<dd>
<div class="block">The number of all reads matching this barcode that matched with 1 error or no-call.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html#openInputs-boolean-">openInputs(boolean)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html" title="class in picard.sam.markduplicates.util">AbstractMarkDuplicatesCommandLineProgram</a></dt>
<dd>
<div class="block">Since this may read its inputs more than once this method does all the opening
 and checking of the inputs.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/AbstractOpticalDuplicateFinderCommandLineProgram.html#OPTICAL_DUPLICATE_PIXEL_DISTANCE">OPTICAL_DUPLICATE_PIXEL_DISTANCE</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/AbstractOpticalDuplicateFinderCommandLineProgram.html" title="class in picard.sam.markduplicates.util">AbstractOpticalDuplicateFinderCommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/AbstractOpticalDuplicateFinderCommandLineProgram.html#opticalDuplicateFinder">opticalDuplicateFinder</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/AbstractOpticalDuplicateFinderCommandLineProgram.html" title="class in picard.sam.markduplicates.util">AbstractOpticalDuplicateFinderCommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/markduplicates/util/OpticalDuplicateFinder.html" title="class in picard.sam.markduplicates.util"><span class="typeNameLink">OpticalDuplicateFinder</span></a> - Class in <a href="picard/sam/markduplicates/util/package-summary.html">picard.sam.markduplicates.util</a></dt>
<dd>
<div class="block">Contains methods for finding optical/co-localized/sequencing duplicates.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/OpticalDuplicateFinder.html#OpticalDuplicateFinder--">OpticalDuplicateFinder()</a></span> - Constructor for class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/OpticalDuplicateFinder.html" title="class in picard.sam.markduplicates.util">OpticalDuplicateFinder</a></dt>
<dd>
<div class="block">Uses the default duplicate distance <a href="picard/sam/markduplicates/util/OpticalDuplicateFinder.html#DEFAULT_OPTICAL_DUPLICATE_DISTANCE"><code>OpticalDuplicateFinder.DEFAULT_OPTICAL_DUPLICATE_DISTANCE</code></a>
 (<a href="picard/sam/markduplicates/util/OpticalDuplicateFinder.html#DEFAULT_OPTICAL_DUPLICATE_DISTANCE">100</a>) and the default read name regex
 <a href="picard/sam/util/ReadNameParser.html#DEFAULT_READ_NAME_REGEX"><code>ReadNameParser.DEFAULT_READ_NAME_REGEX</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/OpticalDuplicateFinder.html#OpticalDuplicateFinder-java.lang.String-int-htsjdk.samtools.util.Log-">OpticalDuplicateFinder(String, int, Log)</a></span> - Constructor for class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/OpticalDuplicateFinder.html" title="class in picard.sam.markduplicates.util">OpticalDuplicateFinder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/OpticalDuplicateFinder.html#OpticalDuplicateFinder-java.lang.String-int-long-htsjdk.samtools.util.Log-">OpticalDuplicateFinder(String, int, long, Log)</a></span> - Constructor for class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/OpticalDuplicateFinder.html" title="class in picard.sam.markduplicates.util">OpticalDuplicateFinder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/OpticalDuplicateFinder.html#opticalDuplicatePixelDistance">opticalDuplicatePixelDistance</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/OpticalDuplicateFinder.html" title="class in picard.sam.markduplicates.util">OpticalDuplicateFinder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/cmdline/argumentcollections/OptionalReferenceArgumentCollection.html" title="class in picard.cmdline.argumentcollections"><span class="typeNameLink">OptionalReferenceArgumentCollection</span></a> - Class in <a href="picard/cmdline/argumentcollections/package-summary.html">picard.cmdline.argumentcollections</a></dt>
<dd>
<div class="block">Picard default argument collection for an optional reference.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/argumentcollections/OptionalReferenceArgumentCollection.html#OptionalReferenceArgumentCollection--">OptionalReferenceArgumentCollection()</a></span> - Constructor for class picard.cmdline.argumentcollections.<a href="picard/cmdline/argumentcollections/OptionalReferenceArgumentCollection.html" title="class in picard.cmdline.argumentcollections">OptionalReferenceArgumentCollection</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEnds.html#orientation">orientation</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEnds.html" title="class in picard.sam.markduplicates.util">ReadEnds</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEnds.html#orientationForOpticalDuplicates">orientationForOpticalDuplicates</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEnds.html" title="class in picard.sam.markduplicates.util">ReadEnds</a></dt>
<dd>
<div class="block">For optical duplicate detection the orientation matters regard to 1st or 2nd end of a mate</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/LiftoverVcf.html#ORIGINAL_CONTIG">ORIGINAL_CONTIG</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/LiftoverVcf.html" title="class in picard.vcf">LiftoverVcf</a></dt>
<dd>
<div class="block">Attribute used to store the name of the source contig/chromosome prior to liftover.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/LiftoverVcf.html#ORIGINAL_START">ORIGINAL_START</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/LiftoverVcf.html" title="class in picard.vcf">LiftoverVcf</a></dt>
<dd>
<div class="block">Attribute used to store the position of the variant on the source contig prior to liftover.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FixMateInformation.html#out">out</a></span> - Variable in class picard.sam.<a href="picard/sam/FixMateInformation.html" title="class in picard.sam">FixMateInformation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesCodec.html#out">out</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesCodec.html" title="class in picard.sam.markduplicates.util">ReadEndsForMarkDuplicatesCodec</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/RepresentativeReadIndexerCodec.html#out">out</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/RepresentativeReadIndexerCodec.html" title="class in picard.sam.markduplicates.util">RepresentativeReadIndexerCodec</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectJumpingLibraryMetrics.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectJumpingLibraryMetrics.html" title="class in picard.analysis">CollectJumpingLibraryMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectMultipleMetrics.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectMultipleMetrics.html" title="class in picard.analysis">CollectMultipleMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.html" title="class in picard.analysis">CollectOxoGMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/CollectTargetedMetrics.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/CollectTargetedMetrics.html" title="class in picard.analysis.directed">CollectTargetedMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/CollectIndependentReplicateMetrics.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/CollectIndependentReplicateMetrics.html" title="class in picard.analysis.replicates">CollectIndependentReplicateMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/SinglePassSamProgram.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/SinglePassSamProgram.html" title="class in picard.analysis">SinglePassSamProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CheckFingerprint.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CheckFingerprint.html" title="class in picard.fingerprint">CheckFingerprint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/ClusterCrosscheckMetrics.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/ClusterCrosscheckMetrics.html" title="class in picard.fingerprint">ClusterCrosscheckMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckFingerprints.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckFingerprints.html" title="class in picard.fingerprint">CrosscheckFingerprints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CollectIlluminaBasecallingMetrics.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.illumina.<a href="picard/illumina/CollectIlluminaBasecallingMetrics.html" title="class in picard.illumina">CollectIlluminaBasecallingMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/MarkIlluminaAdapters.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.illumina.<a href="picard/illumina/MarkIlluminaAdapters.html" title="class in picard.illumina">MarkIlluminaAdapters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/reference/ExtractSequences.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.reference.<a href="picard/reference/ExtractSequences.html" title="class in picard.reference">ExtractSequences</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/reference/NonNFastaSize.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.reference.<a href="picard/reference/NonNFastaSize.html" title="class in picard.reference">NonNFastaSize</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/reference/NormalizeFasta.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.reference.<a href="picard/reference/NormalizeFasta.html" title="class in picard.reference">NormalizeFasta</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AddCommentsToBam.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/AddCommentsToBam.html" title="class in picard.sam">AddCommentsToBam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AddOrReplaceReadGroups.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/AddOrReplaceReadGroups.html" title="class in picard.sam">AddOrReplaceReadGroups</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/BuildBamIndex.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/BuildBamIndex.html" title="class in picard.sam">BuildBamIndex</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CalculateReadGroupChecksum.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/CalculateReadGroupChecksum.html" title="class in picard.sam">CalculateReadGroupChecksum</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CleanSam.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/CleanSam.html" title="class in picard.sam">CleanSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CreateSequenceDictionary.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/CreateSequenceDictionary.html" title="class in picard.sam">CreateSequenceDictionary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/DownsampleSam.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/DownsampleSam.html" title="class in picard.sam">DownsampleSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FastqToSam.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/FastqToSam.html" title="class in picard.sam">FastqToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FilterSamReads.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/FilterSamReads.html" title="class in picard.sam">FilterSamReads</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FixMateInformation.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/FixMateInformation.html" title="class in picard.sam">FixMateInformation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/GatherBamFiles.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/GatherBamFiles.html" title="class in picard.sam">GatherBamFiles</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/EstimateLibraryComplexity.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/EstimateLibraryComplexity.html" title="class in picard.sam.markduplicates">EstimateLibraryComplexity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html" title="class in picard.sam.markduplicates.util">AbstractMarkDuplicatesCommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeSamFiles.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeSamFiles.html" title="class in picard.sam">MergeSamFiles</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/PositionBasedDownsampleSam.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/PositionBasedDownsampleSam.html" title="class in picard.sam">PositionBasedDownsampleSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ReorderSam.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/ReorderSam.html" title="class in picard.sam">ReorderSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ReplaceSamHeader.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/ReplaceSamHeader.html" title="class in picard.sam">ReplaceSamHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertOriginalBaseQualitiesAndAddMateCigar.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/RevertOriginalBaseQualitiesAndAddMateCigar.html" title="class in picard.sam">RevertOriginalBaseQualitiesAndAddMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertSam.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/RevertSam.html" title="class in picard.sam">RevertSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamFormatConverter.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/SamFormatConverter.html" title="class in picard.sam">SamFormatConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SetNmMdAndUqTags.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/SetNmMdAndUqTags.html" title="class in picard.sam">SetNmMdAndUqTags</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SortSam.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/SortSam.html" title="class in picard.sam">SortSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SplitSamByLibrary.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/SplitSamByLibrary.html" title="class in picard.sam">SplitSamByLibrary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ValidateSamFile.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.sam.<a href="picard/sam/ValidateSamFile.html" title="class in picard.sam">ValidateSamFile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/BedToIntervalList.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.util.<a href="picard/util/BedToIntervalList.html" title="class in picard.util">BedToIntervalList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IntervalListToBed.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.util.<a href="picard/util/IntervalListToBed.html" title="class in picard.util">IntervalListToBed</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IntervalListTools.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.util.<a href="picard/util/IntervalListTools.html" title="class in picard.util">IntervalListTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/LiftOverIntervalList.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.util.<a href="picard/util/LiftOverIntervalList.html" title="class in picard.util">LiftOverIntervalList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/ScatterIntervalsByNs.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.util.<a href="picard/util/ScatterIntervalsByNs.html" title="class in picard.util">ScatterIntervalsByNs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/AccumulateVariantCallingMetrics.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/AccumulateVariantCallingMetrics.html" title="class in picard.vcf">AccumulateVariantCallingMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/FilterVcf.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.vcf.filter.<a href="picard/vcf/filter/FilterVcf.html" title="class in picard.vcf.filter">FilterVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/FixVcfHeader.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/FixVcfHeader.html" title="class in picard.vcf">FixVcfHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GatherVcfs.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GatherVcfs.html" title="class in picard.vcf">GatherVcfs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/LiftoverVcf.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/LiftoverVcf.html" title="class in picard.vcf">LiftoverVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MakeSitesOnlyVcf.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/MakeSitesOnlyVcf.html" title="class in picard.vcf">MakeSitesOnlyVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/FindMendelianViolations.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/FindMendelianViolations.html" title="class in picard.vcf.MendelianViolations">FindMendelianViolations</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MergeVcfs.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/MergeVcfs.html" title="class in picard.vcf">MergeVcfs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/RenameSampleInVcf.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/RenameSampleInVcf.html" title="class in picard.vcf">RenameSampleInVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/SortVcf.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/SortVcf.html" title="class in picard.vcf">SortVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/UpdateVcfSequenceDictionary.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/UpdateVcfSequenceDictionary.html" title="class in picard.vcf">UpdateVcfSequenceDictionary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/VcfFormatConverter.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/VcfFormatConverter.html" title="class in picard.vcf">VcfFormatConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/VcfToIntervalList.html#OUTPUT">OUTPUT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/VcfToIntervalList.html" title="class in picard.vcf">VcfToIntervalList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/BaitDesigner.html#OUTPUT_AGILENT_FILES">OUTPUT_AGILENT_FILES</a></span> - Variable in class picard.util.<a href="picard/util/BaitDesigner.html" title="class in picard.util">BaitDesigner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#OUTPUT_ALL_ROWS">OUTPUT_ALL_ROWS</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/ConvertSequencingArtifactToOxoG.html#OUTPUT_BASE">OUTPUT_BASE</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/ConvertSequencingArtifactToOxoG.html" title="class in picard.analysis.artifacts">ConvertSequencingArtifactToOxoG</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertSam.html#OUTPUT_BY_READGROUP">OUTPUT_BY_READGROUP</a></span> - Variable in class picard.sam.<a href="picard/sam/RevertSam.html" title="class in picard.sam">RevertSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertSam.html#OUTPUT_BY_READGROUP_FILE_FORMAT">OUTPUT_BY_READGROUP_FILE_FORMAT</a></span> - Variable in class picard.sam.<a href="picard/sam/RevertSam.html" title="class in picard.sam">RevertSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.html#OUTPUT_DIR">OUTPUT_DIR</a></span> - Variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.html" title="class in picard.illumina">ExtractIlluminaBarcodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamToFastq.html#OUTPUT_DIR">OUTPUT_DIR</a></span> - Variable in class picard.sam.<a href="picard/sam/SamToFastq.html" title="class in picard.sam">SamToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CollectIlluminaLaneMetrics.html#OUTPUT_DIRECTORY">OUTPUT_DIRECTORY</a></span> - Variable in class picard.illumina.<a href="picard/illumina/CollectIlluminaLaneMetrics.html" title="class in picard.illumina">CollectIlluminaLaneMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/BaitDesigner.html#OUTPUT_DIRECTORY">OUTPUT_DIRECTORY</a></span> - Variable in class picard.util.<a href="picard/util/BaitDesigner.html" title="class in picard.util">BaitDesigner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckFingerprints.html#OUTPUT_ERRORS_ONLY">OUTPUT_ERRORS_ONLY</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckFingerprints.html" title="class in picard.fingerprint">CrosscheckFingerprints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fastq/BamToBfq.html#OUTPUT_FILE_PREFIX">OUTPUT_FILE_PREFIX</a></span> - Variable in class picard.fastq.<a href="picard/fastq/BamToBfq.html" title="class in picard.fastq">BamToBfq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertSam.html#OUTPUT_MAP">OUTPUT_MAP</a></span> - Variable in class picard.sam.<a href="picard/sam/RevertSam.html" title="class in picard.sam">RevertSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamToFastq.html#OUTPUT_PER_RG">OUTPUT_PER_RG</a></span> - Variable in class picard.sam.<a href="picard/sam/SamToFastq.html" title="class in picard.sam">SamToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CollectIlluminaLaneMetrics.html#OUTPUT_PREFIX">OUTPUT_PREFIX</a></span> - Variable in class picard.illumina.<a href="picard/illumina/CollectIlluminaLaneMetrics.html" title="class in picard.illumina">CollectIlluminaLaneMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToFastq.html#OUTPUT_PREFIX">OUTPUT_PREFIX</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToFastq.html" title="class in picard.illumina">IlluminaBasecallsToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/StandardOptionDefinitions.html#OUTPUT_SHORT_NAME">OUTPUT_SHORT_NAME</a></span> - Static variable in class picard.cmdline.<a href="picard/cmdline/StandardOptionDefinitions.html" title="class in picard.cmdline">StandardOptionDefinitions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/ScatterIntervalsByNs.html#OUTPUT_TYPE">OUTPUT_TYPE</a></span> - Variable in class picard.util.<a href="picard/util/ScatterIntervalsByNs.html" title="class in picard.util">ScatterIntervalsByNs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#OUTPUT_VCF">OUTPUT_VCF</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#OUTPUT_VCF_CALL_SAMPLE_NAME">OUTPUT_VCF_CALL_SAMPLE_NAME</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#OUTPUT_VCF_FILE_EXTENSION">OUTPUT_VCF_FILE_EXTENSION</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#OUTPUT_VCF_TRUTH_SAMPLE_NAME">OUTPUT_VCF_TRUTH_SAMPLE_NAME</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#outputDetailMetricsFile-htsjdk.variant.variantcontext.VariantContext.Type-htsjdk.samtools.metrics.MetricsFile-picard.vcf.GenotypeConcordanceCounts-java.lang.String-java.lang.String-boolean-boolean-">outputDetailMetricsFile(VariantContext.Type, MetricsFile&lt;GenotypeConcordanceDetailMetrics, ?&gt;, GenotypeConcordanceCounts, String, String, boolean, boolean)</a></span> - Static method in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>
<div class="block">Outputs the detailed statistics tables for SNP and Indel match categories.</div>
</dd>
<dt><a href="picard/illumina/parser/OutputMapping.html" title="class in picard.illumina.parser"><span class="typeNameLink">OutputMapping</span></a> - Class in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>
<div class="block">In multiple locations we need to know what cycles are output, as of now we output all non-skip cycles, but rather than sprinkle
 this knowledge throughout the parser code, instead OutputMapping provides all the data a client might want about the
 cycles to be output including what ReadType they are.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/OutputMapping.html#OutputMapping-picard.illumina.parser.ReadStructure-">OutputMapping(ReadStructure)</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/OutputMapping.html" title="class in picard.illumina.parser">OutputMapping</a></dt>
<dd>
<div class="block">Create an OutputMapping from a readStructure, currently the outputSubstructure just references the readStructure.nonSkips
 Substructure</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html#OXIDATION_ERROR_RATE">OXIDATION_ERROR_RATE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html" title="class in picard.analysis">CollectOxoGMetrics.CpcgMetrics</a></dt>
<dd>
<div class="block">The oxo error rate, calculated as max(ALT_OXO_BASES - ALT_NONOXO_BASES, 1) / TOTAL_BASES</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html#OXIDATION_Q">OXIDATION_Q</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html" title="class in picard.analysis">CollectOxoGMetrics.CpcgMetrics</a></dt>
<dd>
<div class="block">-10 * log10(OXIDATION_ERROR_RATE)</div>
</dd>
</dl>
<a name="I:P">
<!--   -->
</a>
<h2 class="title">P</h2>
<dl>
<dt><span class="memberNameLink"><a href="picard/util/BaitDesigner.html#PADDING">PADDING</a></span> - Variable in class picard.util.<a href="picard/util/BaitDesigner.html" title="class in picard.util">BaitDesigner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IntervalListTools.html#PADDING">PADDING</a></span> - Variable in class picard.util.<a href="picard/util/IntervalListTools.html" title="class in picard.util">IntervalListTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/InsertSizeMetrics.html#PAIR_ORIENTATION">PAIR_ORIENTATION</a></span> - Variable in class picard.analysis.<a href="picard/analysis/InsertSizeMetrics.html" title="class in picard.analysis">InsertSizeMetrics</a></dt>
<dd>
<div class="block">The pair orientation of the reads in this data category.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fastq/BamToBfq.html#PAIRED_RUN">PAIRED_RUN</a></span> - Variable in class picard.fastq.<a href="picard/fastq/BamToBfq.html" title="class in picard.fastq">BamToBfq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/MarkIlluminaAdapters.html#PAIRED_RUN">PAIRED_RUN</a></span> - Variable in class picard.illumina.<a href="picard/illumina/MarkIlluminaAdapters.html" title="class in picard.illumina">MarkIlluminaAdapters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#PAIRED_RUN">PAIRED_RUN</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>
<div class="block"><span class="deprecatedLabel">Deprecated.</span></div>
</dd>
<dt><a href="picard/vcf/PairedVariantSubContextIterator.html" title="class in picard.vcf"><span class="typeNameLink">PairedVariantSubContextIterator</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">An iterator that takes a pair of iterators over VariantContexts and iterates over them in tandem.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/PairedVariantSubContextIterator.html#PairedVariantSubContextIterator-java.util.Iterator-java.lang.String-java.util.Iterator-java.lang.String-htsjdk.samtools.SAMSequenceDictionary-">PairedVariantSubContextIterator(Iterator&lt;VariantContext&gt;, String, Iterator&lt;VariantContext&gt;, String, SAMSequenceDictionary)</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/PairedVariantSubContextIterator.html" title="class in picard.vcf">PairedVariantSubContextIterator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/PairedVariantSubContextIterator.VcfTuple.html" title="class in picard.vcf"><span class="typeNameLink">PairedVariantSubContextIterator.VcfTuple</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">Little class to hold a pair of VariantContexts that are in sync with one another.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadStructure.html#PARAMETER_DOC">PARAMETER_DOC</a></span> - Static variable in class picard.illumina.parser.<a href="picard/illumina/parser/ReadStructure.html" title="class in picard.illumina.parser">ReadStructure</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/ParameterizedFileUtil.html" title="class in picard.illumina.parser"><span class="typeNameLink">ParameterizedFileUtil</span></a> - Class in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ParameterizedFileUtil.html#ParameterizedFileUtil-boolean-java.lang.String-java.io.File-picard.illumina.parser.fakers.FileFaker-int-boolean-">ParameterizedFileUtil(boolean, String, File, FileFaker, int, boolean)</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/ParameterizedFileUtil.html" title="class in picard.illumina.parser">ParameterizedFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ParameterizedFileUtil.html#ParameterizedFileUtil-boolean-java.lang.String-java.io.File-picard.illumina.parser.fakers.FileFaker-int-">ParameterizedFileUtil(boolean, String, File, FileFaker, int)</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/ParameterizedFileUtil.html" title="class in picard.illumina.parser">ParameterizedFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ParameterizedFileUtil.html#ParameterizedFileUtil-java.lang.String-java.lang.String-java.io.File-picard.illumina.parser.fakers.FileFaker-int-">ParameterizedFileUtil(String, String, File, FileFaker, int)</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/ParameterizedFileUtil.html" title="class in picard.illumina.parser">ParameterizedFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#parseArgs-java.lang.String:A-">parseArgs(String[])</a></span> - Method in class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/TileMetricsUtil.html#parseTileMetrics-java.io.File-java.util.Map-picard.illumina.parser.ReadStructure-htsjdk.samtools.ValidationStringency-">parseTileMetrics(File, Map&lt;Integer, File&gt;, ReadStructure, ValidationStringency)</a></span> - Static method in class picard.illumina.parser.<a href="picard/illumina/parser/TileMetricsUtil.html" title="class in picard.illumina.parser">TileMetricsUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/TileMetricsUtil.html#parseTileMetrics-java.io.File-picard.illumina.parser.ReadStructure-htsjdk.samtools.ValidationStringency-">parseTileMetrics(File, ReadStructure, ValidationStringency)</a></span> - Static method in class picard.illumina.parser.<a href="picard/illumina/parser/TileMetricsUtil.html" title="class in picard.illumina.parser">TileMetricsUtil</a></dt>
<dd>
<div class="block">Returns an unmodifiable collection of tile data read from the provided file.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/FilterApplyingVariantIterator.html#PASS_FILTER">PASS_FILTER</a></span> - Static variable in class picard.vcf.filter.<a href="picard/vcf/filter/FilterApplyingVariantIterator.html" title="class in picard.vcf.filter">FilterApplyingVariantIterator</a></dt>
<dd>
<div class="block">The "PASS"ing filter String.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_100X">PCT_100X</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">The fraction of bases that attained at least 100X sequence coverage in post-filtering bases.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_10X">PCT_10X</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">The fraction of bases that attained at least 10X sequence coverage in post-filtering bases.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_15X">PCT_15X</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">The fraction of bases that attained at least 15X sequence coverage in post-filtering bases.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_1X">PCT_1X</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">The fraction of bases that attained at least 1X sequence coverage in post-filtering bases.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_20X">PCT_20X</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">The fraction of bases that attained at least 20X sequence coverage in post-filtering bases.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_25X">PCT_25X</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">The fraction of bases that attained at least 25X sequence coverage in post-filtering bases.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_30X">PCT_30X</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">The fraction of bases that attained at least 30X sequence coverage in post-filtering bases.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_40X">PCT_40X</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">The fraction of bases that attained at least 40X sequence coverage in post-filtering bases.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_50X">PCT_50X</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">The fraction of bases that attained at least 50X sequence coverage in post-filtering bases.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_5X">PCT_5X</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">The fraction of bases that attained at least 5X sequence coverage in post-filtering bases.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_60X">PCT_60X</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">The fraction of bases that attained at least 60X sequence coverage in post-filtering bases.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_70X">PCT_70X</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">The fraction of bases that attained at least 70X sequence coverage in post-filtering bases.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_80X">PCT_80X</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">The fraction of bases that attained at least 80X sequence coverage in post-filtering bases.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_90X">PCT_90X</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">The fraction of bases that attained at least 90X sequence coverage in post-filtering bases.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/BaseDistributionByCycleMetrics.html#PCT_A">PCT_A</a></span> - Variable in class picard.analysis.<a href="picard/analysis/BaseDistributionByCycleMetrics.html" title="class in picard.analysis">BaseDistributionByCycleMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AlignmentSummaryMetrics.html#PCT_ADAPTER">PCT_ADAPTER</a></span> - Variable in class picard.analysis.<a href="picard/analysis/AlignmentSummaryMetrics.html" title="class in picard.analysis">AlignmentSummaryMetrics</a></dt>
<dd>
<div class="block">The fraction of PF reads that are unaligned and match to a known adapter sequence right from the
 start of the read.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#PCT_AMPLIFIED_BASES">PCT_AMPLIFIED_BASES</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The fraction of PF_BASES_ALIGNED that mapped to or near an amplicon, (ON_AMPLICON_BASES +
 NEAR_AMPLICON_BASES)/PF_BASES_ALIGNED.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/BaseDistributionByCycleMetrics.html#PCT_C">PCT_C</a></span> - Variable in class picard.analysis.<a href="picard/analysis/BaseDistributionByCycleMetrics.html" title="class in picard.analysis">BaseDistributionByCycleMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AlignmentSummaryMetrics.html#PCT_CHIMERAS">PCT_CHIMERAS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/AlignmentSummaryMetrics.html" title="class in picard.analysis">AlignmentSummaryMetrics</a></dt>
<dd>
<div class="block">The fraction of reads that map outside of a maximum insert size (usually 100kb) or that have
 the two ends mapping to different chromosomes.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/JumpingLibraryMetrics.html#PCT_CHIMERAS">PCT_CHIMERAS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/JumpingLibraryMetrics.html" title="class in picard.analysis">JumpingLibraryMetrics</a></dt>
<dd>
<div class="block">The number of chimeric pairs expressed as a fraction of the total of all outward facing pairs,
 inward-facing pairs, and chimeric pairs.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RnaSeqMetrics.html#PCT_CODING_BASES">PCT_CODING_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RnaSeqMetrics.html" title="class in picard.analysis">RnaSeqMetrics</a></dt>
<dd>
<div class="block">Fraction of PF_ALIGNED_BASES that mapped to protein coding regions of genes, CODING_BASES/PF_ALIGNED_BASES</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RrbsCpgDetailMetrics.html#PCT_CONVERTED">PCT_CONVERTED</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RrbsCpgDetailMetrics.html" title="class in picard.analysis">RrbsCpgDetailMetrics</a></dt>
<dd>
<div class="block">CpG CONVERTED_BASES / CpG TOTAL_BASES (fraction)</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RnaSeqMetrics.html#PCT_CORRECT_STRAND_READS">PCT_CORRECT_STRAND_READS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RnaSeqMetrics.html" title="class in picard.analysis">RnaSeqMetrics</a></dt>
<dd>
<div class="block">Fraction of reads corresponding to mRNA transcripts which map to the correct strand of a reference genome
    = CORRECT_STRAND_READS/(CORRECT_STRAND_READS + INCORRECT_STRAND_READS).</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RrbsSummaryMetrics.html#PCT_CPG_BASES_CONVERTED">PCT_CPG_BASES_CONVERTED</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RrbsSummaryMetrics.html" title="class in picard.analysis">RrbsSummaryMetrics</a></dt>
<dd>
<div class="block">CPG_BASES_CONVERTED / CPG_BASES_SEEN (fraction)</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html#PCT_DBSNP">PCT_DBSNP</a></span> - Variable in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingSummaryMetrics</a></dt>
<dd>
<div class="block">The fraction of passing bi-allelic SNPs in dbSNP</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html#PCT_DBSNP_INDELS">PCT_DBSNP_INDELS</a></span> - Variable in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingSummaryMetrics</a></dt>
<dd>
<div class="block">The fraction of passing indels in dbSNP</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_EXC_BASEQ">PCT_EXC_BASEQ</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">The fraction of aligned bases that were filtered out because they were of low base quality (default is < 20).</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#PCT_EXC_BASEQ">PCT_EXC_BASEQ</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The fraction of aligned bases that were filtered out because they were of low base quality.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#PCT_EXC_BASEQ">PCT_EXC_BASEQ</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The fraction of aligned bases that were filtered out because they were of low base quality.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_EXC_CAPPED">PCT_EXC_CAPPED</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">The fraction of aligned bases that were filtered out because they would have raised coverage above the capped value (default cap = 250x).</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_EXC_DUPE">PCT_EXC_DUPE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">The fraction of aligned bases that were filtered out because they were in reads marked as duplicates.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#PCT_EXC_DUPE">PCT_EXC_DUPE</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The fraction of aligned bases that were filtered out because they were in reads marked as duplicates.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#PCT_EXC_DUPE">PCT_EXC_DUPE</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The fraction of aligned bases that were filtered out because they were in reads marked as duplicates.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_EXC_MAPQ">PCT_EXC_MAPQ</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">The fraction of aligned bases that were filtered out because they were in reads with low mapping quality (default is < 20).</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#PCT_EXC_MAPQ">PCT_EXC_MAPQ</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The fraction of aligned bases that were filtered out because they were in reads with low mapping quality.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#PCT_EXC_MAPQ">PCT_EXC_MAPQ</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The fraction of aligned bases that were filtered out because they were in reads with low mapping quality.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#PCT_EXC_OFF_TARGET">PCT_EXC_OFF_TARGET</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The fraction of aligned bases that were filtered out because they did not align over a target base.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#PCT_EXC_OFF_TARGET">PCT_EXC_OFF_TARGET</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The fraction of bases that were filtered out because they did not map to a base within a target region.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_EXC_OVERLAP">PCT_EXC_OVERLAP</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">The fraction of aligned bases that were filtered out because they were the second observation from an insert with overlapping reads.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#PCT_EXC_OVERLAP">PCT_EXC_OVERLAP</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The fraction of aligned bases that were filtered out because they were the second observation from an insert with overlapping reads.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#PCT_EXC_OVERLAP">PCT_EXC_OVERLAP</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The fraction of aligned bases that were filtered out because they were the second observation from an
 insert with overlapping reads.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_EXC_TOTAL">PCT_EXC_TOTAL</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">The total fraction of aligned bases excluded due to all filters.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_EXC_UNPAIRED">PCT_EXC_UNPAIRED</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">The fraction of aligned bases that were filtered out because they were in reads without a mapped mate pair.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/BaseDistributionByCycleMetrics.html#PCT_G">PCT_G</a></span> - Variable in class picard.analysis.<a href="picard/analysis/BaseDistributionByCycleMetrics.html" title="class in picard.analysis">BaseDistributionByCycleMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingDetailMetrics.html#PCT_GQ0_VARIANTS">PCT_GQ0_VARIANTS</a></span> - Variable in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingDetailMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingDetailMetrics</a></dt>
<dd>
<div class="block">The percentage of variants in a particular sample that have a GQ score of 0.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RnaSeqMetrics.html#PCT_INTERGENIC_BASES">PCT_INTERGENIC_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RnaSeqMetrics.html" title="class in picard.analysis">RnaSeqMetrics</a></dt>
<dd>
<div class="block">Fraction of PF_ALIGNED_BASES that mapped to intergenic regions of genomic DNA, INTERGENIC_BASES/PF_ALIGNED_BASES</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RnaSeqMetrics.html#PCT_INTRONIC_BASES">PCT_INTRONIC_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RnaSeqMetrics.html" title="class in picard.analysis">RnaSeqMetrics</a></dt>
<dd>
<div class="block">Fraction of PF_ALIGNED_BASES that correspond to gene introns, INTRONIC_BASES/PF_ALIGNED_BASES</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/JumpingLibraryMetrics.html#PCT_JUMPS">PCT_JUMPS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/JumpingLibraryMetrics.html" title="class in picard.analysis">JumpingLibraryMetrics</a></dt>
<dd>
<div class="block">The number of outward-facing pairs expressed as a fraction of the total of all outward facing pairs,
 inward-facing pairs, and chimeric pairs.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html#PCT_MATCHES">PCT_MATCHES</a></span> - Variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html" title="class in picard.illumina">ExtractIlluminaBarcodes.BarcodeMetric</a></dt>
<dd>
<div class="block">The fraction of all reads in the lane that matched to this barcode.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RnaSeqMetrics.html#PCT_MRNA_BASES">PCT_MRNA_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RnaSeqMetrics.html" title="class in picard.analysis">RnaSeqMetrics</a></dt>
<dd>
<div class="block">Sum of bases mapped to regions corresponding to UTRs and coding regions of mRNA transcripts, PCT_UTR_BASES + PCT_CODING_BASES</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/BaseDistributionByCycleMetrics.html#PCT_N">PCT_N</a></span> - Variable in class picard.analysis.<a href="picard/analysis/BaseDistributionByCycleMetrics.html" title="class in picard.analysis">BaseDistributionByCycleMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RrbsSummaryMetrics.html#PCT_NON_CPG_BASES_CONVERTED">PCT_NON_CPG_BASES_CONVERTED</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RrbsSummaryMetrics.html" title="class in picard.analysis">RrbsSummaryMetrics</a></dt>
<dd>
<div class="block">NON_CPG_CONVERTED_BASES / NON_CPG_BASES (fraction)</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/JumpingLibraryMetrics.html#PCT_NONJUMPS">PCT_NONJUMPS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/JumpingLibraryMetrics.html" title="class in picard.analysis">JumpingLibraryMetrics</a></dt>
<dd>
<div class="block">The number of inward-facing pairs expressed as a fraction of the total of all outward facing pairs,
 inward-facing pairs, and chimeric pairs.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#PCT_OFF_AMPLICON">PCT_OFF_AMPLICON</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The fraction of PF_BASES_ALIGNED that mapped neither onto or near an amplicon,
 OFF_AMPLICON_BASES/PF_BASES_ALIGNED</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#PCT_OFF_BAIT">PCT_OFF_BAIT</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The fraction of PF_BASES_ALIGNED that are mapped away from any baited region, OFF_BAIT_BASES/PF_BASES_ALIGNED.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html#PCT_PF_FAIL_EMPTY">PCT_PF_FAIL_EMPTY</a></span> - Variable in class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics.PFFailSummaryMetric</a></dt>
<dd>
<div class="block">The fraction of non-PF reads in this tile that are deemed empty (as fraction of all non-PF reads).</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html#PCT_PF_FAIL_MISALIGNED">PCT_PF_FAIL_MISALIGNED</a></span> - Variable in class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics.PFFailSummaryMetric</a></dt>
<dd>
<div class="block">The fraction of non-PF reads in this tile that are deemed "misaligned" (as fraction of all non-PF reads).</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html#PCT_PF_FAIL_POLYCLONAL">PCT_PF_FAIL_POLYCLONAL</a></span> - Variable in class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics.PFFailSummaryMetric</a></dt>
<dd>
<div class="block">The fraction of non-PF reads in this tile that are deemed multiclonal (as fraction of all non-PF reads).</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html#PCT_PF_FAIL_READS">PCT_PF_FAIL_READS</a></span> - Variable in class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics.PFFailSummaryMetric</a></dt>
<dd>
<div class="block">The fraction of PF_READS</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html#PCT_PF_FAIL_UNKNOWN">PCT_PF_FAIL_UNKNOWN</a></span> - Variable in class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics.PFFailSummaryMetric</a></dt>
<dd>
<div class="block">The fraction of non-PF reads in this tile that have not been classified (as fraction of all non-PF reads).</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AlignmentSummaryMetrics.html#PCT_PF_READS">PCT_PF_READS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/AlignmentSummaryMetrics.html" title="class in picard.analysis">AlignmentSummaryMetrics</a></dt>
<dd>
<div class="block">The fraction of reads that are PF (PF_READS / TOTAL_READS)</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#PCT_PF_READS">PCT_PF_READS</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The fraction of reads passing the vendor's filter, PF_READS/TOTAL_READS.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#PCT_PF_READS">PCT_PF_READS</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The fraction of reads passing filter, PF_READS/TOTAL_READS.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AlignmentSummaryMetrics.html#PCT_PF_READS_ALIGNED">PCT_PF_READS_ALIGNED</a></span> - Variable in class picard.analysis.<a href="picard/analysis/AlignmentSummaryMetrics.html" title="class in picard.analysis">AlignmentSummaryMetrics</a></dt>
<dd>
<div class="block">The percentage of PF reads that aligned to the reference sequence.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AlignmentSummaryMetrics.html#PCT_PF_READS_IMPROPER_PAIRS">PCT_PF_READS_IMPROPER_PAIRS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/AlignmentSummaryMetrics.html" title="class in picard.analysis">AlignmentSummaryMetrics</a></dt>
<dd>
<div class="block">The fraction of (primary) reads that are *not* "properly" aligned in pairs (as per SAM flag 0x2).</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#PCT_PF_UQ_READS">PCT_PF_UQ_READS</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The fraction of PF_UNIQUE_READS from the TOTAL_READS, PF_UNIQUE_READS/TOTAL_READS.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#PCT_PF_UQ_READS">PCT_PF_UQ_READS</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The fraction of TOTAL_READS that are unique, PF, and are not duplicates, PF_UNIQUE_READS/TOTAL_READS</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#PCT_PF_UQ_READS_ALIGNED">PCT_PF_UQ_READS_ALIGNED</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The fraction of PF_UQ_READS_ALIGNED from the total number of PF reads.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#PCT_PF_UQ_READS_ALIGNED">PCT_PF_UQ_READS_ALIGNED</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">Fraction of PF_READS that are unique and align to the reference genome, PF_UQ_READS_ALIGNED/PF_READS</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RnaSeqMetrics.html#PCT_R1_TRANSCRIPT_STRAND_READS">PCT_R1_TRANSCRIPT_STRAND_READS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RnaSeqMetrics.html" title="class in picard.analysis">RnaSeqMetrics</a></dt>
<dd>
<div class="block">The fraction of reads that support the model where R1 is on the strand of transcription and R2 is on the
 opposite strand.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RnaSeqMetrics.html#PCT_R2_TRANSCRIPT_STRAND_READS">PCT_R2_TRANSCRIPT_STRAND_READS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RnaSeqMetrics.html" title="class in picard.analysis">RnaSeqMetrics</a></dt>
<dd>
<div class="block">The fraction of reads that support the model where R2 is on the strand of transcription and R1 is on the opposite
 strand.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AlignmentSummaryMetrics.html#PCT_READS_ALIGNED_IN_PAIRS">PCT_READS_ALIGNED_IN_PAIRS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/AlignmentSummaryMetrics.html" title="class in picard.analysis">AlignmentSummaryMetrics</a></dt>
<dd>
<div class="block">The fraction of reads whose mate pair was also aligned to the reference.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RnaSeqMetrics.html#PCT_RIBOSOMAL_BASES">PCT_RIBOSOMAL_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RnaSeqMetrics.html" title="class in picard.analysis">RnaSeqMetrics</a></dt>
<dd>
<div class="block">Fraction of PF_ALIGNED_BASES that mapped to regions encoding ribosomal RNA, RIBOSOMAL_BASES/PF_ALIGNED_BASES</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#PCT_SELECTED_BASES">PCT_SELECTED_BASES</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The fraction of PF_BASES_ALIGNED located on or near a baited region (ON_BAIT_BASES + NEAR_BAIT_BASES)/PF_BASES_ALIGNED.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/BaseDistributionByCycleMetrics.html#PCT_T">PCT_T</a></span> - Variable in class picard.analysis.<a href="picard/analysis/BaseDistributionByCycleMetrics.html" title="class in picard.analysis">BaseDistributionByCycleMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#PCT_TARGET_BASES_100X">PCT_TARGET_BASES_100X</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The fraction of all target bases achieving 100X or greater coverage.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#PCT_TARGET_BASES_10X">PCT_TARGET_BASES_10X</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The fraction of all target bases achieving 10X or greater coverage.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#PCT_TARGET_BASES_10X">PCT_TARGET_BASES_10X</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The fraction of all target bases achieving 10X or greater coverage depth.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#PCT_TARGET_BASES_1X">PCT_TARGET_BASES_1X</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The fraction of all target bases achieving 1X or greater coverage.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#PCT_TARGET_BASES_1X">PCT_TARGET_BASES_1X</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The fraction of all target bases achieving 1X or greater coverage.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#PCT_TARGET_BASES_20X">PCT_TARGET_BASES_20X</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The fraction of all target bases achieving 20X or greater coverage.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#PCT_TARGET_BASES_20X">PCT_TARGET_BASES_20X</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The fraction of all target bases achieving 20X or greater coverage depth.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#PCT_TARGET_BASES_2X">PCT_TARGET_BASES_2X</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The fraction of all target bases achieving 2X or greater coverage.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#PCT_TARGET_BASES_2X">PCT_TARGET_BASES_2X</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The fraction of all target bases achieving 2X or greater coverage depth.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#PCT_TARGET_BASES_30X">PCT_TARGET_BASES_30X</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The fraction of all target bases achieving 30X or greater coverage.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#PCT_TARGET_BASES_30X">PCT_TARGET_BASES_30X</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The fraction of all target bases achieving 30X or greater coverage depth.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#PCT_TARGET_BASES_40X">PCT_TARGET_BASES_40X</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The fraction of all target bases achieving 40X or greater coverage.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#PCT_TARGET_BASES_50X">PCT_TARGET_BASES_50X</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The fraction of all target bases achieving 50X or greater coverage.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RnaSeqMetrics.html#PCT_USABLE_BASES">PCT_USABLE_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RnaSeqMetrics.html" title="class in picard.analysis">RnaSeqMetrics</a></dt>
<dd>
<div class="block">The fraction of bases mapping to mRNA divided by the total number of PF bases, (CODING_BASES + UTR_BASES)/PF_BASES.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#PCT_USABLE_BASES_ON_BAIT">PCT_USABLE_BASES_ON_BAIT</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The number of aligned, de-duped, on-bait bases out of the PF bases available.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#PCT_USABLE_BASES_ON_TARGET">PCT_USABLE_BASES_ON_TARGET</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The number of aligned, de-duped, on-target bases out of all of the PF bases available.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RnaSeqMetrics.html#PCT_UTR_BASES">PCT_UTR_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RnaSeqMetrics.html" title="class in picard.analysis">RnaSeqMetrics</a></dt>
<dd>
<div class="block">Fraction of PF_ALIGNED_BASES that mapped to untranslated regions (UTR) of genes, UTR_BASES/PF_ALIGNED_BASES</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRrbsMetrics.html#PDF_FILE_EXTENSION">PDF_FILE_EXTENSION</a></span> - Static variable in class picard.analysis.<a href="picard/analysis/CollectRrbsMetrics.html" title="class in picard.analysis">CollectRrbsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/pedigree/PedFile.html" title="class in picard.pedigree"><span class="typeNameLink">PedFile</span></a> - Class in <a href="picard/pedigree/package-summary.html">picard.pedigree</a></dt>
<dd>
<div class="block">Represents a .ped file of family information as documented here:
    http://pngu.mgh.harvard.edu/~purcell/plink/data.shtml

 Stores the information in memory as a map of individualId -> Pedigree information for that individual</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/pedigree/PedFile.html#PedFile-boolean-">PedFile(boolean)</a></span> - Constructor for class picard.pedigree.<a href="picard/pedigree/PedFile.html" title="class in picard.pedigree">PedFile</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/pedigree/PedFile.PedTrio.html" title="class in picard.pedigree"><span class="typeNameLink">PedFile.PedTrio</span></a> - Class in <a href="picard/pedigree/package-summary.html">picard.pedigree</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/pedigree/PedFile.PedTrio.html#PedTrio-java.lang.String-java.lang.String-java.lang.String-java.lang.String-picard.pedigree.Sex-java.lang.Number-">PedTrio(String, String, String, String, Sex, Number)</a></span> - Constructor for class picard.pedigree.<a href="picard/pedigree/PedFile.PedTrio.html" title="class in picard.pedigree">PedFile.PedTrio</a></dt>
<dd>
<div class="block">Constructs a TRIO that cannot be modified after the fact.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/MarkQueue.html#peek--">peek()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/MarkQueue.html" title="class in picard.sam.markduplicates.util">MarkQueue</a></dt>
<dd>
<div class="block">Returns the first element in this queue</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/CollectTargetedMetrics.html#PER_BASE_COVERAGE">PER_BASE_COVERAGE</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/CollectTargetedMetrics.html" title="class in picard.analysis.directed">CollectTargetedMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/CollectTargetedMetrics.html#PER_TARGET_COVERAGE">PER_TARGET_COVERAGE</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/CollectTargetedMetrics.html" title="class in picard.analysis.directed">CollectTargetedMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ParameterizedFileUtil.html#PER_TILE_PATTERN_STRING">PER_TILE_PATTERN_STRING</a></span> - Static variable in class picard.illumina.parser.<a href="picard/illumina/parser/ParameterizedFileUtil.html" title="class in picard.illumina.parser">ParameterizedFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/DuplicationMetrics.html#PERCENT_DUPLICATION">PERCENT_DUPLICATION</a></span> - Variable in class picard.sam.<a href="picard/sam/DuplicationMetrics.html" title="class in picard.sam">DuplicationMetrics</a></dt>
<dd>
<div class="block">The fraction of mapped sequence that is marked as duplicate.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#percentageOrNull-java.lang.Long-java.lang.Long-">percentageOrNull(Long, Long)</a></span> - Static method in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>
<div class="block">Obtains percentage of two Longs</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html#PERFECT_MATCHES">PERFECT_MATCHES</a></span> - Variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html" title="class in picard.illumina">ExtractIlluminaBarcodes.BarcodeMetric</a></dt>
<dd>
<div class="block">The number of all reads matching this barcode that matched with 0 errors or no-calls.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.PerTileBarcodeExtractor.html#PerTileBarcodeExtractor-int-java.io.File-java.util.Map-picard.illumina.ExtractIlluminaBarcodes.BarcodeMetric-picard.illumina.parser.IlluminaDataProviderFactory-int-int-int-int-java.util.List-java.util.List-java.io.File:A-">PerTileBarcodeExtractor(int, File, Map&lt;String, ExtractIlluminaBarcodes.BarcodeMetric&gt;, ExtractIlluminaBarcodes.BarcodeMetric, IlluminaDataProviderFactory, int, int, int, int, List&lt;File&gt;, List&lt;AbstractIlluminaPositionFileReader.PositionInfo&gt;, File[])</a></span> - Constructor for class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.PerTileBarcodeExtractor.html" title="class in picard.illumina">ExtractIlluminaBarcodes.PerTileBarcodeExtractor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.PerTileBarcodeExtractor.html#PerTileBarcodeExtractor-int-java.io.File-java.util.Map-picard.illumina.ExtractIlluminaBarcodes.BarcodeMetric-picard.illumina.parser.IlluminaDataProviderFactory-int-int-int-int-">PerTileBarcodeExtractor(int, File, Map&lt;String, ExtractIlluminaBarcodes.BarcodeMetric&gt;, ExtractIlluminaBarcodes.BarcodeMetric, IlluminaDataProviderFactory, int, int, int, int)</a></span> - Constructor for class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.PerTileBarcodeExtractor.html" title="class in picard.illumina">ExtractIlluminaBarcodes.PerTileBarcodeExtractor</a></dt>
<dd>
<div class="block">Constructor</div>
</dd>
<dt><a href="picard/illumina/parser/PerTileFileUtil.html" title="class in picard.illumina.parser"><span class="typeNameLink">PerTileFileUtil</span></a> - Class in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PerTileFileUtil.html#PerTileFileUtil-java.lang.String-java.io.File-picard.illumina.parser.fakers.FileFaker-int-">PerTileFileUtil(String, File, FileFaker, int)</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/PerTileFileUtil.html" title="class in picard.illumina.parser">PerTileFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PerTileFileUtil.html#PerTileFileUtil-java.lang.String-java.io.File-picard.illumina.parser.fakers.FileFaker-int-boolean-">PerTileFileUtil(String, File, FileFaker, int, boolean)</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/PerTileFileUtil.html" title="class in picard.illumina.parser">PerTileFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/PerTileParser.html" title="class in picard.illumina.parser"><span class="typeNameLink">PerTileParser</span></a>&lt;<a href="picard/illumina/parser/PerTileParser.html" title="type parameter in PerTileParser">ILLUMINA_DATA</a> extends picard.illumina.parser.IlluminaData&gt; - Class in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>
<div class="block">Abstract base class for Parsers that open a single tile file at a time and iterate through them.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PerTileParser.html#PerTileParser-picard.illumina.parser.IlluminaFileMap-">PerTileParser(IlluminaFileMap)</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/PerTileParser.html" title="class in picard.illumina.parser">PerTileParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PerTileParser.html#PerTileParser-picard.illumina.parser.IlluminaFileMap-int-">PerTileParser(IlluminaFileMap, int)</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/PerTileParser.html" title="class in picard.illumina.parser">PerTileParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/PerTilePerCycleFileUtil.html" title="class in picard.illumina.parser"><span class="typeNameLink">PerTilePerCycleFileUtil</span></a> - Class in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PerTilePerCycleFileUtil.html#PerTilePerCycleFileUtil-java.lang.String-java.io.File-picard.illumina.parser.fakers.FileFaker-int-">PerTilePerCycleFileUtil(String, File, FileFaker, int)</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/PerTilePerCycleFileUtil.html" title="class in picard.illumina.parser">PerTilePerCycleFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasMetricsCollector.PerUnitGcBiasMetricsCollector.html#PerUnitGcBiasMetricsCollector-java.lang.String-java.lang.String-java.lang.String-">PerUnitGcBiasMetricsCollector(String, String, String)</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/GcBiasMetricsCollector.PerUnitGcBiasMetricsCollector.html" title="class in picard.analysis">GcBiasMetricsCollector.PerUnitGcBiasMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/InsertSizeMetricsCollector.PerUnitInsertSizeMetricsCollector.html#PerUnitInsertSizeMetricsCollector-java.lang.String-java.lang.String-java.lang.String-">PerUnitInsertSizeMetricsCollector(String, String, String)</a></span> - Constructor for class picard.analysis.directed.<a href="picard/analysis/directed/InsertSizeMetricsCollector.PerUnitInsertSizeMetricsCollector.html" title="class in picard.analysis.directed">InsertSizeMetricsCollector.PerUnitInsertSizeMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/metrics/PerUnitMetricCollector.html" title="interface in picard.metrics"><span class="typeNameLink">PerUnitMetricCollector</span></a>&lt;<a href="picard/metrics/PerUnitMetricCollector.html" title="type parameter in PerUnitMetricCollector">BEAN</a> extends htsjdk.samtools.metrics.MetricBase,<a href="picard/metrics/PerUnitMetricCollector.html" title="type parameter in PerUnitMetricCollector">HKEY</a> extends java.lang.Comparable,<a href="picard/metrics/PerUnitMetricCollector.html" title="type parameter in PerUnitMetricCollector">ARGTYPE</a>&gt; - Interface in <a href="picard/metrics/package-summary.html">picard.metrics</a></dt>
<dd>
<div class="block">PerRecordCollector - An interface for classes that collect data in order to generate one or more metrics.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/RnaSeqMetricsCollector.PerUnitRnaSeqMetricsCollector.html#PerUnitRnaSeqMetricsCollector-picard.analysis.RnaSeqMetrics-java.lang.String-java.lang.String-java.lang.String-java.lang.Long-">PerUnitRnaSeqMetricsCollector(RnaSeqMetrics, String, String, String, Long)</a></span> - Constructor for class picard.analysis.directed.<a href="picard/analysis/directed/RnaSeqMetricsCollector.PerUnitRnaSeqMetricsCollector.html" title="class in picard.analysis.directed">RnaSeqMetricsCollector.PerUnitRnaSeqMetricsCollector</a></dt>
<dd>
<div class="block">Derived classes that need to capture some additional metrics can use this ctor to supply a metrics instance
 that is a subclass of RnaSeqMetrics.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/RnaSeqMetricsCollector.PerUnitRnaSeqMetricsCollector.html#PerUnitRnaSeqMetricsCollector-java.lang.String-java.lang.String-java.lang.String-java.lang.Long-">PerUnitRnaSeqMetricsCollector(String, String, String, Long)</a></span> - Constructor for class picard.analysis.directed.<a href="picard/analysis/directed/RnaSeqMetricsCollector.PerUnitRnaSeqMetricsCollector.html" title="class in picard.analysis.directed">RnaSeqMetricsCollector.PerUnitRnaSeqMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetMetricsCollector.PerUnitTargetMetricCollector.html#PerUnitTargetMetricCollector-java.lang.String-java.util.Set-java.lang.String-java.lang.String-java.lang.String-long-long-long-java.util.Map-int-int-boolean-">PerUnitTargetMetricCollector(String, Set&lt;Interval&gt;, String, String, String, long, long, long, Map&lt;Interval, Double&gt;, int, int, boolean)</a></span> - Constructor for class picard.analysis.directed.<a href="picard/analysis/directed/TargetMetricsCollector.PerUnitTargetMetricCollector.html" title="class in picard.analysis.directed">TargetMetricsCollector.PerUnitTargetMetricCollector</a></dt>
<dd>
<div class="block">Constructor that parses the squashed reference to genome reference file and stores the
 information in a map for later use.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AlignmentSummaryMetrics.html#PF_ALIGNED_BASES">PF_ALIGNED_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/AlignmentSummaryMetrics.html" title="class in picard.analysis">AlignmentSummaryMetrics</a></dt>
<dd>
<div class="block">The total number of aligned bases, in all mapped PF reads, that are aligned to the reference sequence.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RnaSeqMetrics.html#PF_ALIGNED_BASES">PF_ALIGNED_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RnaSeqMetrics.html" title="class in picard.analysis">RnaSeqMetrics</a></dt>
<dd>
<div class="block">The total number of aligned PF bases.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetrics.html#PF_BASES">PF_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetrics.html" title="class in picard.analysis">CollectQualityYieldMetrics.QualityYieldMetrics</a></dt>
<dd>
<div class="block">The total number of bases in all PF reads</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#PF_BASES">PF_BASES</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The total number of bases within the PF_READS of the SAM or BAM file to be examined</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RnaSeqMetrics.html#PF_BASES">PF_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RnaSeqMetrics.html" title="class in picard.analysis">RnaSeqMetrics</a></dt>
<dd>
<div class="block">The total number of PF bases including non-aligned reads.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallingMetrics.html#PF_BASES">PF_BASES</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallingMetrics.html" title="class in picard.illumina">IlluminaBasecallingMetrics</a></dt>
<dd>
<div class="block">The total number of passing-filter bases assigned to the index.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#PF_BASES_ALIGNED">PF_BASES_ALIGNED</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The number of PF unique bases that are aligned to the reference genome with mapping scores > 0.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#PF_BASES_ALIGNED">PF_BASES_ALIGNED</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The number of bases from PF_READS that align to the reference genome with mapping score > 0</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallingMetrics.html#PF_CLUSTERS">PF_CLUSTERS</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallingMetrics.html" title="class in picard.illumina">IlluminaBasecallingMetrics</a></dt>
<dd>
<div class="block">The total number of PF clusters assigned to the index.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html#PF_FAIL_EMPTY">PF_FAIL_EMPTY</a></span> - Variable in class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics.PFFailSummaryMetric</a></dt>
<dd>
<div class="block">The number of non-PF reads in this tile that are deemed empty.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html#PF_FAIL_MISALIGNED">PF_FAIL_MISALIGNED</a></span> - Variable in class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics.PFFailSummaryMetric</a></dt>
<dd>
<div class="block">The number of non-PF reads in this tile that are deemed "misaligned".</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html#PF_FAIL_POLYCLONAL">PF_FAIL_POLYCLONAL</a></span> - Variable in class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics.PFFailSummaryMetric</a></dt>
<dd>
<div class="block">The number of non-PF reads in this tile that are deemed multiclonal.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html#PF_FAIL_READS">PF_FAIL_READS</a></span> - Variable in class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics.PFFailSummaryMetric</a></dt>
<dd>
<div class="block">The number of non-PF reads in this tile.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html#PF_FAIL_UNKNOWN">PF_FAIL_UNKNOWN</a></span> - Variable in class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics.PFFailSummaryMetric</a></dt>
<dd>
<div class="block">The number of non-PF reads in this tile that have not been classified.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AlignmentSummaryMetrics.html#PF_HQ_ALIGNED_BASES">PF_HQ_ALIGNED_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/AlignmentSummaryMetrics.html" title="class in picard.analysis">AlignmentSummaryMetrics</a></dt>
<dd>
<div class="block">The number of bases aligned to the reference sequence in reads that were mapped at high
 quality.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AlignmentSummaryMetrics.html#PF_HQ_ALIGNED_Q20_BASES">PF_HQ_ALIGNED_Q20_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/AlignmentSummaryMetrics.html" title="class in picard.analysis">AlignmentSummaryMetrics</a></dt>
<dd>
<div class="block">The subset of PF_HQ_ALIGNED_BASES where the base call quality was Q20 or higher.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AlignmentSummaryMetrics.html#PF_HQ_ALIGNED_READS">PF_HQ_ALIGNED_READS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/AlignmentSummaryMetrics.html" title="class in picard.analysis">AlignmentSummaryMetrics</a></dt>
<dd>
<div class="block">The number of PF reads that were aligned to the reference sequence with a mapping quality of
 Q20 or higher signifying that the aligner estimates a 1/100 (or smaller) chance that the
 alignment is wrong.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AlignmentSummaryMetrics.html#PF_HQ_ERROR_RATE">PF_HQ_ERROR_RATE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/AlignmentSummaryMetrics.html" title="class in picard.analysis">AlignmentSummaryMetrics</a></dt>
<dd>
<div class="block">The fraction of bases that mismatch the reference in PF HQ aligned reads.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AlignmentSummaryMetrics.html#PF_HQ_MEDIAN_MISMATCHES">PF_HQ_MEDIAN_MISMATCHES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/AlignmentSummaryMetrics.html" title="class in picard.analysis">AlignmentSummaryMetrics</a></dt>
<dd>
<div class="block">The median number of mismatches versus the reference sequence in reads that were aligned
 to the reference at high quality (i.e.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AlignmentSummaryMetrics.html#PF_INDEL_RATE">PF_INDEL_RATE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/AlignmentSummaryMetrics.html" title="class in picard.analysis">AlignmentSummaryMetrics</a></dt>
<dd>
<div class="block">The number of insertion and deletion events per 100 aligned bases.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AlignmentSummaryMetrics.html#PF_MISMATCH_RATE">PF_MISMATCH_RATE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/AlignmentSummaryMetrics.html" title="class in picard.analysis">AlignmentSummaryMetrics</a></dt>
<dd>
<div class="block">The rate of bases mismatching the reference for all bases aligned to the reference sequence.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AlignmentSummaryMetrics.html#PF_NOISE_READS">PF_NOISE_READS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/AlignmentSummaryMetrics.html" title="class in picard.analysis">AlignmentSummaryMetrics</a></dt>
<dd>
<div class="block">The number of PF reads that are marked as noise reads.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html#PF_NORMALIZED_MATCHES">PF_NORMALIZED_MATCHES</a></span> - Variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html" title="class in picard.illumina">ExtractIlluminaBarcodes.BarcodeMetric</a></dt>
<dd>
<div class="block">The "normalized" matches to each barcode.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html#PF_ONE_MISMATCH_MATCHES">PF_ONE_MISMATCH_MATCHES</a></span> - Variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html" title="class in picard.illumina">ExtractIlluminaBarcodes.BarcodeMetric</a></dt>
<dd>
<div class="block">The number of PF reads matching this barcode that matched with 1 error or no-call.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html#PF_PCT_MATCHES">PF_PCT_MATCHES</a></span> - Variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html" title="class in picard.illumina">ExtractIlluminaBarcodes.BarcodeMetric</a></dt>
<dd>
<div class="block">The fraction of PF reads in the lane that matched to this barcode.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html#PF_PERFECT_MATCHES">PF_PERFECT_MATCHES</a></span> - Variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html" title="class in picard.illumina">ExtractIlluminaBarcodes.BarcodeMetric</a></dt>
<dd>
<div class="block">The number of PF reads matching this barcode that matched with 0 errors or no-calls.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetrics.html#PF_Q20_BASES">PF_Q20_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetrics.html" title="class in picard.analysis">CollectQualityYieldMetrics.QualityYieldMetrics</a></dt>
<dd>
<div class="block">The number of bases in PF reads that achieve quality score 20 or higher</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetrics.html#PF_Q20_EQUIVALENT_YIELD">PF_Q20_EQUIVALENT_YIELD</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetrics.html" title="class in picard.analysis">CollectQualityYieldMetrics.QualityYieldMetrics</a></dt>
<dd>
<div class="block">The sum of quality scores of all bases in PF reads divided by 20</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetrics.html#PF_Q30_BASES">PF_Q30_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetrics.html" title="class in picard.analysis">CollectQualityYieldMetrics.QualityYieldMetrics</a></dt>
<dd>
<div class="block">The number of bases in PF reads that achieve quality score 30 or higher</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html#PF_RATIO_THIS_BARCODE_TO_BEST_BARCODE_PCT">PF_RATIO_THIS_BARCODE_TO_BEST_BARCODE_PCT</a></span> - Variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html" title="class in picard.illumina">ExtractIlluminaBarcodes.BarcodeMetric</a></dt>
<dd>
<div class="block">The rate of PF reads matching this barcode to PF reads matching the most prevelant barcode.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AlignmentSummaryMetrics.html#PF_READS">PF_READS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/AlignmentSummaryMetrics.html" title="class in picard.analysis">AlignmentSummaryMetrics</a></dt>
<dd>
<div class="block">The number of PF reads where PF is defined as passing Illumina's filter.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetrics.html#PF_READS">PF_READS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetrics.html" title="class in picard.analysis">CollectQualityYieldMetrics.QualityYieldMetrics</a></dt>
<dd>
<div class="block">The number of reads that are PF - pass filter</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#PF_READS">PF_READS</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The total number of reads that pass the vendor's filter.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#PF_READS">PF_READS</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The total number of reads passing filter (PF), where the filter(s) can be platform/vendor quality controls</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html#PF_READS">PF_READS</a></span> - Variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html" title="class in picard.illumina">ExtractIlluminaBarcodes.BarcodeMetric</a></dt>
<dd>
<div class="block">The number of PF reads matching this barcode (always less than or equal to READS).</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallingMetrics.html#PF_READS">PF_READS</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallingMetrics.html" title="class in picard.illumina">IlluminaBasecallingMetrics</a></dt>
<dd>
<div class="block">The total number of passing-filter reads assigned to the index.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AlignmentSummaryMetrics.html#PF_READS_ALIGNED">PF_READS_ALIGNED</a></span> - Variable in class picard.analysis.<a href="picard/analysis/AlignmentSummaryMetrics.html" title="class in picard.analysis">AlignmentSummaryMetrics</a></dt>
<dd>
<div class="block">The number of PF reads that were aligned to the reference sequence.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AlignmentSummaryMetrics.html#PF_READS_IMPROPER_PAIRS">PF_READS_IMPROPER_PAIRS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/AlignmentSummaryMetrics.html" title="class in picard.analysis">AlignmentSummaryMetrics</a></dt>
<dd>
<div class="block">The number of (primary) aligned reads that are **not** "properly" aligned in pairs (as per SAM flag 0x2).</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectBaseDistributionByCycle.html#PF_READS_ONLY">PF_READS_ONLY</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectBaseDistributionByCycle.html" title="class in picard.analysis">CollectBaseDistributionByCycle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/MeanQualityByCycle.html#PF_READS_ONLY">PF_READS_ONLY</a></span> - Variable in class picard.analysis.<a href="picard/analysis/MeanQualityByCycle.html" title="class in picard.analysis">MeanQualityByCycle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/QualityScoreDistribution.html#PF_READS_ONLY">PF_READS_ONLY</a></span> - Variable in class picard.analysis.<a href="picard/analysis/QualityScoreDistribution.html" title="class in picard.analysis">QualityScoreDistribution</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/StandardOptionDefinitions.html#PF_READS_ONLY_SHORT_NAME">PF_READS_ONLY_SHORT_NAME</a></span> - Static variable in class picard.cmdline.<a href="picard/cmdline/StandardOptionDefinitions.html" title="class in picard.cmdline">StandardOptionDefinitions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#PF_SELECTED_PAIRS">PF_SELECTED_PAIRS</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">Tracks the number of PF read pairs (used to calculate library size)</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#PF_SELECTED_UNIQUE_PAIRS">PF_SELECTED_UNIQUE_PAIRS</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">Tracks the number of unique, PF, read pairs, observed (used to calculate library size)</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ViewSam.html#PF_STATUS">PF_STATUS</a></span> - Variable in class picard.sam.<a href="picard/sam/ViewSam.html" title="class in picard.sam">ViewSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#PF_UNIQUE_READS">PF_UNIQUE_READS</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The number of PF reads that are not marked as duplicates.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#PF_UNIQUE_READS">PF_UNIQUE_READS</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The number of PF_READS that were not marked as sample or optical duplicates.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#PF_UQ_BASES_ALIGNED">PF_UQ_BASES_ALIGNED</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The number of bases in the PF_UQ_READS_ALIGNED reads.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#PF_UQ_BASES_ALIGNED">PF_UQ_BASES_ALIGNED</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The number of bases from PF_UNIQUE_READS that align to the reference genome and have a mapping score > 0</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#PF_UQ_READS_ALIGNED">PF_UQ_READS_ALIGNED</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The number of PF_UNIQUE_READS that aligned to the reference genome with a mapping score > 0.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#PF_UQ_READS_ALIGNED">PF_UQ_READS_ALIGNED</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The total number of PF_UNIQUE_READS that align to the reference genome with mapping scores > 0</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailDetailedMetric.html#PFFailDetailedMetric-java.lang.Integer-int-int-int-int-picard.illumina.quality.CollectHiSeqXPfFailMetrics.ReadClassifier.PfFailReason-">PFFailDetailedMetric(Integer, int, int, int, int, CollectHiSeqXPfFailMetrics.ReadClassifier.PfFailReason)</a></span> - Constructor for class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailDetailedMetric.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics.PFFailDetailedMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailDetailedMetric.html#PFFailDetailedMetric--">PFFailDetailedMetric()</a></span> - Constructor for class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailDetailedMetric.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics.PFFailDetailedMetric</a></dt>
<dd>
<div class="block">This ctor is necessary for when reading metrics from file</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html#PFFailSummaryMetric-java.lang.String-">PFFailSummaryMetric(String)</a></span> - Constructor for class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics.PFFailSummaryMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html#PFFailSummaryMetric--">PFFailSummaryMetric()</a></span> - Constructor for class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics.PFFailSummaryMetric</a></dt>
<dd>
<div class="block">This ctor is necessary for when reading metrics from file</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/PositionBasedDownsampleSam.html#PG_PROGRAM_NAME">PG_PROGRAM_NAME</a></span> - Static variable in class picard.sam.<a href="picard/sam/PositionBasedDownsampleSam.html" title="class in picard.sam">PositionBasedDownsampleSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html#pgIdsSeen">pgIdsSeen</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html" title="class in picard.sam.markduplicates.util">AbstractMarkDuplicatesCommandLineProgram</a></dt>
<dd>
<div class="block">The program groups that have been seen during the course of examining the input records.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeMap.html#PHASESET_PREFIX">PHASESET_PREFIX</a></span> - Static variable in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeMap.html" title="class in picard.fingerprint">HaplotypeMap</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaPhasingMetrics.html#PHASING_APPLIED">PHASING_APPLIED</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaPhasingMetrics.html" title="class in picard.illumina">IlluminaPhasingMetrics</a></dt>
<dd>
<div class="block">Median phasing value across all tiles in a lane, applied to the first and second template reads</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/EmpiricalPhasingMetricsOutReader.IlluminaPhasingMetrics.html#phasingWeight">phasingWeight</a></span> - Variable in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/EmpiricalPhasingMetricsOutReader.IlluminaPhasingMetrics.html" title="class in picard.illumina.parser.readers">EmpiricalPhasingMetricsOutReader.IlluminaPhasingMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/util/PhysicalLocation.html" title="interface in picard.sam.util"><span class="typeNameLink">PhysicalLocation</span></a> - Interface in <a href="picard/sam/util/package-summary.html">picard.sam.util</a></dt>
<dd>
<div class="block">Small interface that provides access to the physical location information about a cluster.</div>
</dd>
<dt><a href="picard/sam/markduplicates/util/PhysicalLocationForMateCigar.html" title="class in picard.sam.markduplicates.util"><span class="typeNameLink">PhysicalLocationForMateCigar</span></a> - Class in <a href="picard/sam/markduplicates/util/package-summary.html">picard.sam.markduplicates.util</a></dt>
<dd>
<div class="block">Stores the minimal information needed for optical duplicate detection.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/PhysicalLocationForMateCigar.html#PhysicalLocationForMateCigar-picard.sam.util.PhysicalLocation-">PhysicalLocationForMateCigar(PhysicalLocation)</a></span> - Constructor for class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/PhysicalLocationForMateCigar.html" title="class in picard.sam.markduplicates.util">PhysicalLocationForMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/markduplicates/util/PhysicalLocationForMateCigarSet.html" title="class in picard.sam.markduplicates.util"><span class="typeNameLink">PhysicalLocationForMateCigarSet</span></a> - Class in <a href="picard/sam/markduplicates/util/package-summary.html">picard.sam.markduplicates.util</a></dt>
<dd>
<div class="block">This stores records that are comparable for detecting optical duplicates.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/PhysicalLocationForMateCigarSet.html#PhysicalLocationForMateCigarSet--">PhysicalLocationForMateCigarSet()</a></span> - Constructor for class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/PhysicalLocationForMateCigarSet.html" title="class in picard.sam.markduplicates.util">PhysicalLocationForMateCigarSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/util/PhysicalLocationInt.html" title="class in picard.sam.util"><span class="typeNameLink">PhysicalLocationInt</span></a> - Class in <a href="picard/sam/util/package-summary.html">picard.sam.util</a></dt>
<dd>
<div class="block">Small class that provides access to the physical location information about a cluster.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/PhysicalLocationInt.html#PhysicalLocationInt--">PhysicalLocationInt()</a></span> - Constructor for class picard.sam.util.<a href="picard/sam/util/PhysicalLocationInt.html" title="class in picard.sam.util">PhysicalLocationInt</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/util/PhysicalLocationShort.html" title="class in picard.sam.util"><span class="typeNameLink">PhysicalLocationShort</span></a> - Class in <a href="picard/sam/util/package-summary.html">picard.sam.util</a></dt>
<dd>
<div class="block">Small class that provides access to the physical location information about a cluster.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/PhysicalLocationShort.html#PhysicalLocationShort--">PhysicalLocationShort()</a></span> - Constructor for class picard.sam.util.<a href="picard/sam/util/PhysicalLocationShort.html" title="class in picard.sam.util">PhysicalLocationShort</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/package-summary.html">picard</a> - package picard</dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/package-summary.html">picard.analysis</a> - package picard.analysis</dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/artifacts/package-summary.html">picard.analysis.artifacts</a> - package picard.analysis.artifacts</dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/directed/package-summary.html">picard.analysis.directed</a> - package picard.analysis.directed</dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/replicates/package-summary.html">picard.analysis.replicates</a> - package picard.analysis.replicates</dt>
<dd>&nbsp;</dd>
<dt><a href="picard/annotation/package-summary.html">picard.annotation</a> - package picard.annotation</dt>
<dd>&nbsp;</dd>
<dt><a href="picard/cmdline/package-summary.html">picard.cmdline</a> - package picard.cmdline</dt>
<dd>&nbsp;</dd>
<dt><a href="picard/cmdline/argumentcollections/package-summary.html">picard.cmdline.argumentcollections</a> - package picard.cmdline.argumentcollections</dt>
<dd>&nbsp;</dd>
<dt><a href="picard/cmdline/programgroups/package-summary.html">picard.cmdline.programgroups</a> - package picard.cmdline.programgroups</dt>
<dd>&nbsp;</dd>
<dt><a href="picard/fastq/package-summary.html">picard.fastq</a> - package picard.fastq</dt>
<dd>&nbsp;</dd>
<dt><a href="picard/filter/package-summary.html">picard.filter</a> - package picard.filter</dt>
<dd>&nbsp;</dd>
<dt><a href="picard/fingerprint/package-summary.html">picard.fingerprint</a> - package picard.fingerprint</dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/package-summary.html">picard.illumina</a> - package picard.illumina</dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a> - package picard.illumina.parser</dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/fakers/package-summary.html">picard.illumina.parser.fakers</a> - package picard.illumina.parser.fakers</dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/readers/package-summary.html">picard.illumina.parser.readers</a> - package picard.illumina.parser.readers</dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/quality/package-summary.html">picard.illumina.quality</a> - package picard.illumina.quality</dt>
<dd>&nbsp;</dd>
<dt><a href="picard/metrics/package-summary.html">picard.metrics</a> - package picard.metrics</dt>
<dd>&nbsp;</dd>
<dt><a href="picard/pedigree/package-summary.html">picard.pedigree</a> - package picard.pedigree</dt>
<dd>&nbsp;</dd>
<dt><a href="picard/reference/package-summary.html">picard.reference</a> - package picard.reference</dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/package-summary.html">picard.sam</a> - package picard.sam</dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/markduplicates/package-summary.html">picard.sam.markduplicates</a> - package picard.sam.markduplicates</dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/markduplicates/util/package-summary.html">picard.sam.markduplicates.util</a> - package picard.sam.markduplicates.util</dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/util/package-summary.html">picard.sam.util</a> - package picard.sam.util</dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/package-summary.html">picard.util</a> - package picard.util</dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/help/package-summary.html">picard.util.help</a> - package picard.util.help</dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/package-summary.html">picard.vcf</a> - package picard.vcf</dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/filter/package-summary.html">picard.vcf.filter</a> - package picard.vcf.filter</dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/MendelianViolations/package-summary.html">picard.vcf.MendelianViolations</a> - package picard.vcf.MendelianViolations</dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/processor/package-summary.html">picard.vcf.processor</a> - package picard.vcf.processor</dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/processor/util/package-summary.html">picard.vcf.processor.util</a> - package picard.vcf.processor.util</dt>
<dd>&nbsp;</dd>
<dt><a href="picard/cmdline/PicardCommandLine.html" title="class in picard.cmdline"><span class="typeNameLink">PicardCommandLine</span></a> - Class in <a href="picard/cmdline/package-summary.html">picard.cmdline</a></dt>
<dd>
<div class="block">This is the main class of Picard and is the way of executing individual command line programs.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/PicardCommandLine.html#PicardCommandLine--">PicardCommandLine()</a></span> - Constructor for class picard.cmdline.<a href="picard/cmdline/PicardCommandLine.html" title="class in picard.cmdline">PicardCommandLine</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/PicardException.html" title="class in picard"><span class="typeNameLink">PicardException</span></a> - Exception in <a href="picard/package-summary.html">picard</a></dt>
<dd>
<div class="block">Basic Picard runtime exception that, for now, does nothing much</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/PicardException.html#PicardException-java.lang.String-">PicardException(String)</a></span> - Constructor for exception picard.<a href="picard/PicardException.html" title="class in picard">PicardException</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/PicardException.html#PicardException-java.lang.String-java.lang.Throwable-">PicardException(String, Throwable)</a></span> - Constructor for exception picard.<a href="picard/PicardException.html" title="class in picard">PicardException</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/help/PicardHelpDoclet.html" title="class in picard.util.help"><span class="typeNameLink">PicardHelpDoclet</span></a> - Class in <a href="picard/util/help/package-summary.html">picard.util.help</a></dt>
<dd>
<div class="block">Custom Barclay-based Javadoc Doclet used for generating Picard help/documentation.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/help/PicardHelpDoclet.html#PicardHelpDoclet--">PicardHelpDoclet()</a></span> - Constructor for class picard.util.help.<a href="picard/util/help/PicardHelpDoclet.html" title="class in picard.util.help">PicardHelpDoclet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/help/PicardHelpDocWorkUnitHandler.html" title="class in picard.util.help"><span class="typeNameLink">PicardHelpDocWorkUnitHandler</span></a> - Class in <a href="picard/util/help/package-summary.html">picard.util.help</a></dt>
<dd>
<div class="block">The Picard Documentation work unit handler class that is the companion to PicardHelpDoclet.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/help/PicardHelpDocWorkUnitHandler.html#PicardHelpDocWorkUnitHandler-org.broadinstitute.barclay.help.HelpDoclet-">PicardHelpDocWorkUnitHandler(HelpDoclet)</a></span> - Constructor for class picard.util.help.<a href="picard/util/help/PicardHelpDocWorkUnitHandler.html" title="class in picard.util.help">PicardHelpDocWorkUnitHandler</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/BestEndMapqPrimaryAlignmentStrategy.html#pickPrimaryAlignment-picard.sam.HitsForInsert-">pickPrimaryAlignment(HitsForInsert)</a></span> - Method in class picard.sam.<a href="picard/sam/BestEndMapqPrimaryAlignmentStrategy.html" title="class in picard.sam">BestEndMapqPrimaryAlignmentStrategy</a></dt>
<dd>
<div class="block">Primary alignment was filtered out.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/BestMapqPrimaryAlignmentSelectionStrategy.html#pickPrimaryAlignment-picard.sam.HitsForInsert-">pickPrimaryAlignment(HitsForInsert)</a></span> - Method in class picard.sam.<a href="picard/sam/BestMapqPrimaryAlignmentSelectionStrategy.html" title="class in picard.sam">BestMapqPrimaryAlignmentSelectionStrategy</a></dt>
<dd>
<div class="block">Primary alignment was filtered out.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/EarliestFragmentPrimaryAlignmentSelectionStrategy.html#pickPrimaryAlignment-picard.sam.HitsForInsert-">pickPrimaryAlignment(HitsForInsert)</a></span> - Method in class picard.sam.<a href="picard/sam/EarliestFragmentPrimaryAlignmentSelectionStrategy.html" title="class in picard.sam">EarliestFragmentPrimaryAlignmentSelectionStrategy</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MostDistantPrimaryAlignmentSelectionStrategy.html#pickPrimaryAlignment-picard.sam.HitsForInsert-">pickPrimaryAlignment(HitsForInsert)</a></span> - Method in class picard.sam.<a href="picard/sam/MostDistantPrimaryAlignmentSelectionStrategy.html" title="class in picard.sam">MostDistantPrimaryAlignmentSelectionStrategy</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/PrimaryAlignmentSelectionStrategy.html#pickPrimaryAlignment-picard.sam.HitsForInsert-">pickPrimaryAlignment(HitsForInsert)</a></span> - Method in interface picard.sam.<a href="picard/sam/PrimaryAlignmentSelectionStrategy.html" title="interface in picard.sam">PrimaryAlignmentSelectionStrategy</a></dt>
<dd>
<div class="block">When this method returns, one alignment has been marked as primary according to the implementation's strategy.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/RnaSeqMetricsCollector.PerUnitRnaSeqMetricsCollector.html#pickTranscripts-java.util.Map-">pickTranscripts(Map&lt;Gene.Transcript, int[]&gt;)</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/RnaSeqMetricsCollector.PerUnitRnaSeqMetricsCollector.html" title="class in picard.analysis.directed">RnaSeqMetricsCollector.PerUnitRnaSeqMetricsCollector</a></dt>
<dd>
<div class="block">Picks the set of transcripts on which the coverage metrics are to be calculated.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#PLATFORM">PLATFORM</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FastqToSam.html#PLATFORM">PLATFORM</a></span> - Variable in class picard.sam.<a href="picard/sam/FastqToSam.html" title="class in picard.sam">FastqToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FastqToSam.html#PLATFORM_MODEL">PLATFORM_MODEL</a></span> - Variable in class picard.sam.<a href="picard/sam/FastqToSam.html" title="class in picard.sam">FastqToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FastqToSam.html#PLATFORM_UNIT">PLATFORM_UNIT</a></span> - Variable in class picard.sam.<a href="picard/sam/FastqToSam.html" title="class in picard.sam">FastqToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#pNormalizeLogProbability-double:A-">pNormalizeLogProbability(double[])</a></span> - Static method in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>
<div class="block">Takes a complete set of mutually exclusive logPosteriors and converts them to probabilities
 that sum to 1 with as much fidelity as possible.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#pNormalizeVector-double:A-">pNormalizeVector(double[])</a></span> - Static method in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>
<div class="block">Takes a vector of numbers and converts it to a vector of probabilities
 that sum to 1 with as much fidelity as possible.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/MarkQueue.html#poll-htsjdk.samtools.util.SamRecordTrackingBuffer-htsjdk.samtools.SAMFileHeader-picard.sam.markduplicates.util.OpticalDuplicateFinder-picard.sam.markduplicates.util.LibraryIdGenerator-">poll(SamRecordTrackingBuffer, SAMFileHeader, OpticalDuplicateFinder, LibraryIdGenerator)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/MarkQueue.html" title="class in picard.sam.markduplicates.util">MarkQueue</a></dt>
<dd>
<div class="block">The poll method will return the read end that is *not* the duplicate of all comparable read ends that
 have been seen.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/BaitDesigner.html#POOL_SIZE">POOL_SIZE</a></span> - Variable in class picard.util.<a href="picard/util/BaitDesigner.html" title="class in picard.util">BaitDesigner</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/fakers/PosFileFaker.html" title="class in picard.illumina.parser.fakers"><span class="typeNameLink">PosFileFaker</span></a> - Class in <a href="picard/illumina/parser/fakers/package-summary.html">picard.illumina.parser.fakers</a></dt>
<dd>
<div class="block">Created by jcarey on 3/13/14.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/fakers/PosFileFaker.html#PosFileFaker--">PosFileFaker()</a></span> - Constructor for class picard.illumina.parser.fakers.<a href="picard/illumina/parser/fakers/PosFileFaker.html" title="class in picard.illumina.parser.fakers">PosFileFaker</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/readers/PosFileReader.html" title="class in picard.illumina.parser.readers"><span class="typeNameLink">PosFileReader</span></a> - Class in <a href="picard/illumina/parser/readers/package-summary.html">picard.illumina.parser.readers</a></dt>
<dd>
<div class="block">The pos file format is one 3 Illumina formats(pos, locs, and clocs) that stores position data exclusively.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/PosFileReader.html#PosFileReader-java.io.File-">PosFileReader(File)</a></span> - Constructor for class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/PosFileReader.html" title="class in picard.illumina.parser.readers">PosFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/FingerprintingDetailMetrics.html#POSITION">POSITION</a></span> - Variable in class picard.analysis.<a href="picard/analysis/FingerprintingDetailMetrics.html" title="class in picard.analysis">FingerprintingDetailMetrics</a></dt>
<dd>
<div class="block">The position of the SNP on the chromosome.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RrbsCpgDetailMetrics.html#POSITION">POSITION</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RrbsCpgDetailMetrics.html" title="class in picard.analysis">RrbsCpgDetailMetrics</a></dt>
<dd>
<div class="block">Position within the sequence of the CpG site</div>
</dd>
<dt><a href="picard/sam/PositionBasedDownsampleSam.html" title="class in picard.sam"><span class="typeNameLink">PositionBasedDownsampleSam</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block">Summary</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/PositionBasedDownsampleSam.html#PositionBasedDownsampleSam--">PositionBasedDownsampleSam()</a></span> - Constructor for class picard.sam.<a href="picard/sam/PositionBasedDownsampleSam.html" title="class in picard.sam">PositionBasedDownsampleSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.PositionInfo.html#PositionInfo-float-float-int-int-">PositionInfo(float, float, int, int)</a></span> - Constructor for class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.PositionInfo.html" title="class in picard.illumina.parser.readers">AbstractIlluminaPositionFileReader.PositionInfo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/PosParser.html" title="class in picard.illumina.parser"><span class="typeNameLink">PosParser</span></a> - Class in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>
<div class="block">PosParser parses multiple files formatted as one of the three file formats that contain position information
 only (pos, locs, and clocs).</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PosParser.html#PosParser-picard.illumina.parser.IlluminaFileMap-picard.illumina.parser.IlluminaFileUtil.SupportedIlluminaFormat-">PosParser(IlluminaFileMap, IlluminaFileUtil.SupportedIlluminaFormat)</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/PosParser.html" title="class in picard.illumina.parser">PosParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PosParser.html#PosParser-picard.illumina.parser.IlluminaFileMap-int-picard.illumina.parser.IlluminaFileUtil.SupportedIlluminaFormat-">PosParser(IlluminaFileMap, int, IlluminaFileUtil.SupportedIlluminaFormat)</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/PosParser.html" title="class in picard.illumina.parser">PosParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceCounts.html#Ppv-picard.vcf.GenotypeConcordanceScheme-picard.vcf.GenotypeConcordanceStates.CallState:A-">Ppv(GenotypeConcordanceScheme, GenotypeConcordanceStates.CallState[])</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceCounts.html" title="class in picard.vcf">GenotypeConcordanceCounts</a></dt>
<dd>
<div class="block">Returns the PPV defined by the scheme across the subset of call states.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.html#PRE_ADAPTER_DETAILS_EXT">PRE_ADAPTER_DETAILS_EXT</a></span> - Static variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.html#PRE_ADAPTER_SUMMARY_EXT">PRE_ADAPTER_SUMMARY_EXT</a></span> - Static variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterDetailMetrics.html#PreAdapterDetailMetrics--">PreAdapterDetailMetrics()</a></span> - Constructor for class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterDetailMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.PreAdapterDetailMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterSummaryMetrics.html#PreAdapterSummaryMetrics--">PreAdapterSummaryMetrics()</a></span> - Constructor for class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterSummaryMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.PreAdapterSummaryMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/processor/util/PredicateFilterDecoratingClosableIterator.html" title="class in picard.vcf.processor.util"><span class="typeNameLink">PredicateFilterDecoratingClosableIterator</span></a>&lt;<a href="picard/vcf/processor/util/PredicateFilterDecoratingClosableIterator.html" title="type parameter in PredicateFilterDecoratingClosableIterator">T</a>&gt; - Class in <a href="picard/vcf/processor/util/package-summary.html">picard.vcf.processor.util</a></dt>
<dd>
<div class="block">Performs on-the-fly filtering of the provided <code>VariantContext</code> <code>Iterator</code> such that only variants that satisfy
 all predicates are emitted.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/util/PredicateFilterDecoratingClosableIterator.html#PredicateFilterDecoratingClosableIterator-htsjdk.samtools.util.CloseableIterator-java.util.Collection-">PredicateFilterDecoratingClosableIterator(CloseableIterator&lt;T&gt;, Collection&lt;Predicate&lt;T&gt;&gt;)</a></span> - Constructor for class picard.vcf.processor.util.<a href="picard/vcf/processor/util/PredicateFilterDecoratingClosableIterator.html" title="class in picard.vcf.processor.util">PredicateFilterDecoratingClosableIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FastqToSam.html#PREDICTED_INSERT_SIZE">PREDICTED_INSERT_SIZE</a></span> - Variable in class picard.sam.<a href="picard/sam/FastqToSam.html" title="class in picard.sam">FastqToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaPhasingMetrics.html#PREPHASING_APPLIED">PREPHASING_APPLIED</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaPhasingMetrics.html" title="class in picard.illumina">IlluminaPhasingMetrics</a></dt>
<dd>
<div class="block">Median pre-phasing value across all tiles in a lane, applied to the first and second template reads</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/EmpiricalPhasingMetricsOutReader.IlluminaPhasingMetrics.html#prephasingWeight">prephasingWeight</a></span> - Variable in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/EmpiricalPhasingMetricsOutReader.IlluminaPhasingMetrics.html" title="class in picard.illumina.parser.readers">EmpiricalPhasingMetricsOutReader.IlluminaPhasingMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#PRIMARY_ALIGNMENT_STRATEGY">PRIMARY_ALIGNMENT_STRATEGY</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/PrimaryAlignmentSelectionStrategy.html" title="interface in picard.sam"><span class="typeNameLink">PrimaryAlignmentSelectionStrategy</span></a> - Interface in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block">Given a set of alignments for a read or read pair, mark one alignment as primary, according to whatever
 strategy is appropriate.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/PicardCommandLine.html#printUnknown-java.util.Set-java.lang.String-">printUnknown(Set&lt;Class&lt;?&gt;&gt;, String)</a></span> - Static method in class picard.cmdline.<a href="picard/cmdline/PicardCommandLine.html" title="class in picard.cmdline">PicardCommandLine</a></dt>
<dd>
<div class="block">When a command does not match any known command, searches for similar commands, using the same method as GIT</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterDetailMetrics.html#PRO_ALT_BASES">PRO_ALT_BASES</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterDetailMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.PreAdapterDetailMetrics</a></dt>
<dd>
<div class="block">The number of REF_BASE:ALT_BASE alignments having a read number and orientation that supports the presence of this artifact.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterDetailMetrics.html#PRO_REF_BASES">PRO_REF_BASES</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterDetailMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.PreAdapterDetailMetrics</a></dt>
<dd>
<div class="block">The number of REF_BASE:REF_BASE alignments having a read number and orientation that supports the presence of this artifact.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.html#PROB_EXPLICIT_READS">PROB_EXPLICIT_READS</a></span> - Variable in class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/DownsampleSam.html#PROBABILITY">PROBABILITY</a></span> - Variable in class picard.sam.<a href="picard/sam/DownsampleSam.html" title="class in picard.sam">DownsampleSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VariantProcessor.html#process--">process()</a></span> - Method in class picard.vcf.processor.<a href="picard/vcf/processor/VariantProcessor.html" title="class in picard.vcf.processor">VariantProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/PicardCommandLine.html#processAllCommandLinePrograms-java.util.List-java.util.function.BiConsumer-">processAllCommandLinePrograms(List&lt;String&gt;, BiConsumer&lt;Class&lt;CommandLineProgram&gt;, CommandLineProgramProperties&gt;)</a></span> - Static method in class picard.cmdline.<a href="picard/cmdline/PicardCommandLine.html" title="class in picard.cmdline">PicardCommandLine</a></dt>
<dd>
<div class="block">Process each <code>CommandLineProgram</code>-derived class given a list of packages.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/WgsMetricsProcessor.html#processFile--">processFile()</a></span> - Method in interface picard.analysis.<a href="picard/analysis/WgsMetricsProcessor.html" title="interface in picard.analysis">WgsMetricsProcessor</a></dt>
<dd>
<div class="block">Method processes the input data and accumulates result data</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/WgsMetricsProcessorImpl.html#processFile--">processFile()</a></span> - Method in class picard.analysis.<a href="picard/analysis/WgsMetricsProcessorImpl.html" title="class in picard.analysis">WgsMetricsProcessorImpl</a></dt>
<dd>
<div class="block">Method gets the data from iterator for each locus and processes it with the help of collector.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.LogMath.html#product-double...-">product(double...)</a></span> - Method in class picard.util.<a href="picard/util/MathUtil.LogMath.html" title="class in picard.util">MathUtil.LogMath</a></dt>
<dd>
<div class="block">Computes the sum of the provided log values.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectMultipleMetrics.html#PROGRAM">PROGRAM</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectMultipleMetrics.html" title="class in picard.analysis">CollectMultipleMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FastqToSam.html#PROGRAM_GROUP">PROGRAM_GROUP</a></span> - Variable in class picard.sam.<a href="picard/sam/FastqToSam.html" title="class in picard.sam">FastqToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html#PROGRAM_GROUP_COMMAND_LINE">PROGRAM_GROUP_COMMAND_LINE</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html" title="class in picard.sam.markduplicates.util">AbstractMarkDuplicatesCommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#PROGRAM_GROUP_COMMAND_LINE">PROGRAM_GROUP_COMMAND_LINE</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html#PROGRAM_GROUP_NAME">PROGRAM_GROUP_NAME</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html" title="class in picard.sam.markduplicates.util">AbstractMarkDuplicatesCommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#PROGRAM_GROUP_NAME">PROGRAM_GROUP_NAME</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html#PROGRAM_GROUP_VERSION">PROGRAM_GROUP_VERSION</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html" title="class in picard.sam.markduplicates.util">AbstractMarkDuplicatesCommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#PROGRAM_GROUP_VERSION">PROGRAM_GROUP_VERSION</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html#PROGRAM_RECORD_ID">PROGRAM_RECORD_ID</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html" title="class in picard.sam.markduplicates.util">AbstractMarkDuplicatesCommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#PROGRAM_RECORD_ID">PROGRAM_RECORD_ID</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/StandardOptionDefinitions.html#PROGRAM_RECORD_ID_SHORT_NAME">PROGRAM_RECORD_ID_SHORT_NAME</a></span> - Static variable in class picard.cmdline.<a href="picard/cmdline/StandardOptionDefinitions.html" title="class in picard.cmdline">StandardOptionDefinitions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#promote-int:A-">promote(int[])</a></span> - Static method in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>
<div class="block">"Promotes" an int[] into a double array with the same values (or as close as precision allows).</div>
</dd>
<dt><a href="picard/util/PropertyUtils.html" title="class in picard.util"><span class="typeNameLink">PropertyUtils</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">Utility for loading properties files from resources.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/PropertyUtils.html#PropertyUtils--">PropertyUtils()</a></span> - Constructor for class picard.util.<a href="picard/util/PropertyUtils.html" title="class in picard.util">PropertyUtils</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/TheoreticalSensitivity.html#proportionsAboveThresholds-java.util.List-java.util.List-">proportionsAboveThresholds(List&lt;ArrayList&lt;Integer&gt;&gt;, List&lt;Double&gt;)</a></span> - Static method in class picard.analysis.<a href="picard/analysis/TheoreticalSensitivity.html" title="class in picard.analysis">TheoreticalSensitivity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/MarkIlluminaAdapters.html#PRUNE_ADAPTER_LIST_AFTER_THIS_MANY_ADAPTERS_SEEN">PRUNE_ADAPTER_LIST_AFTER_THIS_MANY_ADAPTERS_SEEN</a></span> - Variable in class picard.illumina.<a href="picard/illumina/MarkIlluminaAdapters.html" title="class in picard.illumina">MarkIlluminaAdapters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#pSameAlleleWhenMismatchingUmi">pSameAlleleWhenMismatchingUmi</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#pSameUmiInIndependentBiDup">pSameUmiInIndependentBiDup</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/FindMendelianViolations.html#PSEUDO_AUTOSOMAL_REGIONS">PSEUDO_AUTOSOMAL_REGIONS</a></span> - Variable in class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/FindMendelianViolations.html" title="class in picard.vcf.MendelianViolations">FindMendelianViolations</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/DiskBasedReadEndsForMarkDuplicatesMap.html#put-int-java.lang.String-picard.sam.markduplicates.util.ReadEndsForMarkDuplicates-">put(int, String, ReadEndsForMarkDuplicates)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/DiskBasedReadEndsForMarkDuplicatesMap.html" title="class in picard.sam.markduplicates.util">DiskBasedReadEndsForMarkDuplicatesMap</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesMap.html#put-int-java.lang.String-picard.sam.markduplicates.util.ReadEndsForMarkDuplicates-">put(int, String, ReadEndsForMarkDuplicates)</a></span> - Method in interface picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesMap.html" title="interface in picard.sam.markduplicates.util">ReadEndsForMarkDuplicatesMap</a></dt>
<dd>
<div class="block">Store the element in the map with the given key.</div>
</dd>
</dl>
<a name="I:Q">
<!--   -->
</a>
<h2 class="title">Q</h2>
<dl>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetrics.html#Q20_BASES">Q20_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetrics.html" title="class in picard.analysis">CollectQualityYieldMetrics.QualityYieldMetrics</a></dt>
<dd>
<div class="block">The number of bases in all reads that achieve quality score 20 or higher</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetrics.html#Q20_EQUIVALENT_YIELD">Q20_EQUIVALENT_YIELD</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetrics.html" title="class in picard.analysis">CollectQualityYieldMetrics.QualityYieldMetrics</a></dt>
<dd>
<div class="block">The sum of quality scores of all bases divided by 20</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetrics.html#Q30_BASES">Q30_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetrics.html" title="class in picard.analysis">CollectQualityYieldMetrics.QualityYieldMetrics</a></dt>
<dd>
<div class="block">The number of bases in all reads that achieve quality score 30 or higher</div>
</dd>
<dt><a href="picard/vcf/filter/QdFilter.html" title="class in picard.vcf.filter"><span class="typeNameLink">QdFilter</span></a> - Class in <a href="picard/vcf/filter/package-summary.html">picard.vcf.filter</a></dt>
<dd>
<div class="block">Filters out sites that have a QD annotation applied to them and where the QD value is lower than a
 lower limit.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/QdFilter.html#QdFilter-double-">QdFilter(double)</a></span> - Constructor for class picard.vcf.filter.<a href="picard/vcf/filter/QdFilter.html" title="class in picard.vcf.filter">QdFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html#QSCORE">QSCORE</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.BaitBiasDetailMetrics</a></dt>
<dd>
<div class="block">The Phred-scaled quality score of the artifact, calculated as -10 * log10(ERROR_RATE).</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterDetailMetrics.html#QSCORE">QSCORE</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterDetailMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.PreAdapterDetailMetrics</a></dt>
<dd>
<div class="block">The Phred-scaled quality score of the artifact, calculated as -10 * log10(ERROR_RATE).</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/BclData.html#qualities">qualities</a></span> - Variable in class picard.illumina.parser.<a href="picard/illumina/parser/BclData.html" title="class in picard.illumina.parser">BclData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamToFastq.html#QUALITY">QUALITY</a></span> - Variable in class picard.sam.<a href="picard/sam/SamToFastq.html" title="class in picard.sam">SamToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FastqToSam.html#QUALITY_FORMAT">QUALITY_FORMAT</a></span> - Variable in class picard.sam.<a href="picard/sam/FastqToSam.html" title="class in picard.sam">FastqToSam</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/QualityScoreDistribution.html" title="class in picard.analysis"><span class="typeNameLink">QualityScoreDistribution</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Charts quality score distribution within a BAM file.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/QualityScoreDistribution.html#QualityScoreDistribution--">QualityScoreDistribution()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/QualityScoreDistribution.html" title="class in picard.analysis">QualityScoreDistribution</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetrics.html#QualityYieldMetrics--">QualityYieldMetrics()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetrics.html" title="class in picard.analysis">CollectQualityYieldMetrics.QualityYieldMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetricsCollector.html#QualityYieldMetricsCollector-boolean-boolean-boolean-">QualityYieldMetricsCollector(boolean, boolean, boolean)</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetricsCollector.html" title="class in picard.analysis">CollectQualityYieldMetrics.QualityYieldMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/QuerySortedReadPairIteratorUtil.html" title="class in picard.util"><span class="typeNameLink">QuerySortedReadPairIteratorUtil</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">A collection of helper utilities for iterating through reads that are in query-name sorted
 read order as pairs</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/QuerySortedReadPairIteratorUtil.html#QuerySortedReadPairIteratorUtil--">QuerySortedReadPairIteratorUtil()</a></span> - Constructor for class picard.util.<a href="picard/util/QuerySortedReadPairIteratorUtil.html" title="class in picard.util">QuerySortedReadPairIteratorUtil</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/QuerySortedReadPairIteratorUtil.ReadPair.html" title="class in picard.util"><span class="typeNameLink">QuerySortedReadPairIteratorUtil.ReadPair</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/BclReader.html#queue">queue</a></span> - Variable in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/BclReader.html" title="class in picard.illumina.parser.readers">BclReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#QUIET">QUIET</a></span> - Variable in class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="I:R">
<!--   -->
</a>
<h2 class="title">R</h2>
<dl>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEnds.html#R">R</a></span> - Static variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEnds.html" title="class in picard.sam.markduplicates.util">ReadEnds</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/DownsampleSam.html#RANDOM_SEED">RANDOM_SEED</a></span> - Variable in class picard.sam.<a href="picard/sam/DownsampleSam.html" title="class in picard.sam">DownsampleSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintChecker.html#randomSublist-java.util.List-int-">randomSublist(List&lt;T&gt;, int)</a></span> - Static method in class picard.fingerprint.<a href="picard/fingerprint/FingerprintChecker.html" title="class in picard.fingerprint">FingerprintChecker</a></dt>
<dd>
<div class="block">A small utility function to choose n random elements (un-shuffled) from a list</div>
</dd>
<dt><a href="picard/illumina/parser/Range.html" title="class in picard.illumina.parser"><span class="typeNameLink">Range</span></a> - Class in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>
<div class="block">While structurally identical to CompositeIndex, this class is maintained as it makes code more readable when the two are used together (see QSeqParser)</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/Range.html#Range-int-int-">Range(int, int)</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/Range.html" title="class in picard.illumina.parser">Range</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/ReadNameParser.html#rapidParseInt-java.lang.String-">rapidParseInt(String)</a></span> - Static method in class picard.sam.util.<a href="picard/sam/util/ReadNameParser.html" title="class in picard.sam.util">ReadNameParser</a></dt>
<dd>
<div class="block">Very specialized method to rapidly parse a sequence of digits from a String up until the first
 non-digit character.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html#RATIO_THIS_BARCODE_TO_BEST_BARCODE_PCT">RATIO_THIS_BARCODE_TO_BEST_BARCODE_PCT</a></span> - Variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html" title="class in picard.illumina">ExtractIlluminaBarcodes.BarcodeMetric</a></dt>
<dd>
<div class="block">The rate of all reads matching this barcode to all reads matching the most prevelant barcode.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRawWgsMetrics.RawWgsMetrics.html#RawWgsMetrics--">RawWgsMetrics()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/CollectRawWgsMetrics.RawWgsMetrics.html" title="class in picard.analysis">CollectRawWgsMetrics.RawWgsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRawWgsMetrics.RawWgsMetrics.html#RawWgsMetrics-htsjdk.samtools.util.IntervalList-htsjdk.samtools.util.Histogram-htsjdk.samtools.util.Histogram-double-double-double-double-double-double-double-int-htsjdk.samtools.util.Histogram-int-">RawWgsMetrics(IntervalList, Histogram&lt;Integer&gt;, Histogram&lt;Integer&gt;, double, double, double, double, double, double, double, int, Histogram&lt;Integer&gt;, int)</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/CollectRawWgsMetrics.RawWgsMetrics.html" title="class in picard.analysis">CollectRawWgsMetrics.RawWgsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamToFastq.html#RE_REVERSE">RE_REVERSE</a></span> - Variable in class picard.sam.<a href="picard/sam/SamToFastq.html" title="class in picard.sam">SamToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/CircularByteBuffer.html#read-byte:A-int-int-">read(byte[], int, int)</a></span> - Method in class picard.util.<a href="picard/util/CircularByteBuffer.html" title="class in picard.util">CircularByteBuffer</a></dt>
<dd>
<div class="block">Read bytes from the buffer into the supplied array.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/QuerySortedReadPairIteratorUtil.ReadPair.html#read1">read1</a></span> - Variable in class picard.util.<a href="picard/util/QuerySortedReadPairIteratorUtil.ReadPair.html" title="class in picard.util">QuerySortedReadPairIteratorUtil.ReadPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#READ1_ALIGNED_BAM">READ1_ALIGNED_BAM</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamToFastq.html#READ1_MAX_BASES_TO_WRITE">READ1_MAX_BASES_TO_WRITE</a></span> - Variable in class picard.sam.<a href="picard/sam/SamToFastq.html" title="class in picard.sam">SamToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#READ1_TRIM">READ1_TRIM</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamToFastq.html#READ1_TRIM">READ1_TRIM</a></span> - Variable in class picard.sam.<a href="picard/sam/SamToFastq.html" title="class in picard.sam">SamToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEnds.html#read1Coordinate">read1Coordinate</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEnds.html" title="class in picard.sam.markduplicates.util">ReadEnds</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicates.html#read1IndexInFile">read1IndexInFile</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicates.html" title="class in picard.sam.markduplicates.util">ReadEndsForMarkDuplicates</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEnds.html#read1ReferenceIndex">read1ReferenceIndex</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEnds.html" title="class in picard.sam.markduplicates.util">ReadEnds</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/QuerySortedReadPairIteratorUtil.ReadPair.html#read2">read2</a></span> - Variable in class picard.util.<a href="picard/util/QuerySortedReadPairIteratorUtil.ReadPair.html" title="class in picard.util">QuerySortedReadPairIteratorUtil.ReadPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#READ2_ALIGNED_BAM">READ2_ALIGNED_BAM</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamToFastq.html#READ2_MAX_BASES_TO_WRITE">READ2_MAX_BASES_TO_WRITE</a></span> - Variable in class picard.sam.<a href="picard/sam/SamToFastq.html" title="class in picard.sam">SamToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#READ2_TRIM">READ2_TRIM</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamToFastq.html#READ2_TRIM">READ2_TRIM</a></span> - Variable in class picard.sam.<a href="picard/sam/SamToFastq.html" title="class in picard.sam">SamToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEnds.html#read2Coordinate">read2Coordinate</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEnds.html" title="class in picard.sam.markduplicates.util">ReadEnds</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicates.html#read2IndexInFile">read2IndexInFile</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicates.html" title="class in picard.sam.markduplicates.util">ReadEndsForMarkDuplicates</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEnds.html#read2ReferenceIndex">read2ReferenceIndex</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEnds.html" title="class in picard.sam.markduplicates.util">ReadEnds</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fastq/BamToBfq.html#READ_CHUNK_SIZE">READ_CHUNK_SIZE</a></span> - Variable in class picard.fastq.<a href="picard/fastq/BamToBfq.html" title="class in picard.fastq">BamToBfq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/BaseDistributionByCycleMetrics.html#READ_END">READ_END</a></span> - Variable in class picard.analysis.<a href="picard/analysis/BaseDistributionByCycleMetrics.html" title="class in picard.analysis">BaseDistributionByCycleMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/FingerprintingDetailMetrics.html#READ_GROUP">READ_GROUP</a></span> - Variable in class picard.analysis.<a href="picard/analysis/FingerprintingDetailMetrics.html" title="class in picard.analysis">FingerprintingDetailMetrics</a></dt>
<dd>
<div class="block">The sequencing read group from which sequence data was fingerprinted.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/FingerprintingSummaryMetrics.html#READ_GROUP">READ_GROUP</a></span> - Variable in class picard.analysis.<a href="picard/analysis/FingerprintingSummaryMetrics.html" title="class in picard.analysis">FingerprintingSummaryMetrics</a></dt>
<dd>
<div class="block">The read group from which sequence data was drawn for comparison.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/metrics/MultilevelMetrics.html#READ_GROUP">READ_GROUP</a></span> - Variable in class picard.metrics.<a href="picard/metrics/MultilevelMetrics.html" title="class in picard.metrics">MultilevelMetrics</a></dt>
<dd>
<div class="block">The read group to which these metrics apply.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#READ_GROUP_ID">READ_GROUP_ID</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/StandardOptionDefinitions.html#READ_GROUP_ID_SHORT_NAME">READ_GROUP_ID_SHORT_NAME</a></span> - Static variable in class picard.cmdline.<a href="picard/cmdline/StandardOptionDefinitions.html" title="class in picard.cmdline">StandardOptionDefinitions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FastqToSam.html#READ_GROUP_NAME">READ_GROUP_NAME</a></span> - Variable in class picard.sam.<a href="picard/sam/FastqToSam.html" title="class in picard.sam">FastqToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetrics.html#READ_LENGTH">READ_LENGTH</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetrics.html" title="class in picard.analysis">CollectQualityYieldMetrics.QualityYieldMetrics</a></dt>
<dd>
<div class="block">The average read length of all the reads (will be fixed for a lane)</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.html#READ_LENGTH">READ_LENGTH</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FilterSamReads.html#READ_LIST_FILE">READ_LIST_FILE</a></span> - Variable in class picard.sam.<a href="picard/sam/FilterSamReads.html" title="class in picard.sam">FilterSamReads</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToFastq.html#READ_NAME_FORMAT">READ_NAME_FORMAT</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToFastq.html" title="class in picard.illumina">IlluminaBasecallsToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fastq/BamToBfq.html#READ_NAME_PREFIX">READ_NAME_PREFIX</a></span> - Variable in class picard.fastq.<a href="picard/fastq/BamToBfq.html" title="class in picard.fastq">BamToBfq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/AbstractOpticalDuplicateFinderCommandLineProgram.html#READ_NAME_REGEX">READ_NAME_REGEX</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/AbstractOpticalDuplicateFinderCommandLineProgram.html" title="class in picard.sam.markduplicates.util">AbstractOpticalDuplicateFinderCommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/PositionBasedDownsampleSam.html#READ_NAME_REGEX">READ_NAME_REGEX</a></span> - Variable in class picard.sam.<a href="picard/sam/PositionBasedDownsampleSam.html" title="class in picard.sam">PositionBasedDownsampleSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/EstimateLibraryComplexity.html#READ_ONE_BARCODE_TAG">READ_ONE_BARCODE_TAG</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/EstimateLibraryComplexity.html" title="class in picard.sam.markduplicates">EstimateLibraryComplexity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicates.html#READ_ONE_BARCODE_TAG">READ_ONE_BARCODE_TAG</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicates.html" title="class in picard.sam.markduplicates">MarkDuplicates</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/DuplicationMetrics.html#READ_PAIR_DUPLICATES">READ_PAIR_DUPLICATES</a></span> - Variable in class picard.sam.<a href="picard/sam/DuplicationMetrics.html" title="class in picard.sam">DuplicationMetrics</a></dt>
<dd>
<div class="block">The number of read pairs that were marked as duplicates.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/DuplicationMetrics.html#READ_PAIR_OPTICAL_DUPLICATES">READ_PAIR_OPTICAL_DUPLICATES</a></span> - Variable in class picard.sam.<a href="picard/sam/DuplicationMetrics.html" title="class in picard.sam">DuplicationMetrics</a></dt>
<dd>
<div class="block">The number of read pairs duplicates that were caused by optical duplication.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/InsertSizeMetrics.html#READ_PAIRS">READ_PAIRS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/InsertSizeMetrics.html" title="class in picard.analysis">InsertSizeMetrics</a></dt>
<dd>
<div class="block">The total number of read pairs that were examined in the entire distribution.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/DuplicationMetrics.html#READ_PAIRS_EXAMINED">READ_PAIRS_EXAMINED</a></span> - Variable in class picard.sam.<a href="picard/sam/DuplicationMetrics.html" title="class in picard.sam">DuplicationMetrics</a></dt>
<dd>
<div class="block">The number of mapped read pairs examined.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasDetailMetrics.html#READ_STARTS">READ_STARTS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/GcBiasDetailMetrics.html" title="class in picard.analysis">GcBiasDetailMetrics</a></dt>
<dd>
<div class="block">The number of reads whose start position is at the start of a window of this GC.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CheckIlluminaDirectory.html#READ_STRUCTURE">READ_STRUCTURE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/CheckIlluminaDirectory.html" title="class in picard.illumina">CheckIlluminaDirectory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CollectIlluminaBasecallingMetrics.html#READ_STRUCTURE">READ_STRUCTURE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/CollectIlluminaBasecallingMetrics.html" title="class in picard.illumina">CollectIlluminaBasecallingMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CollectIlluminaLaneMetrics.html#READ_STRUCTURE">READ_STRUCTURE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/CollectIlluminaLaneMetrics.html" title="class in picard.illumina">CollectIlluminaLaneMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.html#READ_STRUCTURE">READ_STRUCTURE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.html" title="class in picard.illumina">ExtractIlluminaBarcodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToFastq.html#READ_STRUCTURE">READ_STRUCTURE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToFastq.html" title="class in picard.illumina">IlluminaBasecallsToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#READ_STRUCTURE">READ_STRUCTURE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/EstimateLibraryComplexity.html#READ_TWO_BARCODE_TAG">READ_TWO_BARCODE_TAG</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/EstimateLibraryComplexity.html" title="class in picard.sam.markduplicates">EstimateLibraryComplexity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicates.html#READ_TWO_BARCODE_TAG">READ_TWO_BARCODE_TAG</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicates.html" title="class in picard.sam.markduplicates">MarkDuplicates</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.ReadClassifier.html#ReadClassifier-picard.illumina.parser.ReadData-">ReadClassifier(ReadData)</a></span> - Constructor for class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.ReadClassifier.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics.ReadClassifier</a></dt>
<dd>
<div class="block">Heart of CLP.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetMetricsCollector.Coverage.html#readCount">readCount</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetMetricsCollector.Coverage.html" title="class in picard.analysis.directed">TargetMetricsCollector.Coverage</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/ReadData.html" title="class in picard.illumina.parser"><span class="typeNameLink">ReadData</span></a> - Class in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>
<div class="block">Data for a single end of a paired-end read, a barcode read, or for the entire read if not paired end.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadData.html#ReadData--">ReadData()</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/ReadData.html" title="class in picard.illumina.parser">ReadData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadData.html#ReadData-picard.illumina.parser.ReadType-">ReadData(ReadType)</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/ReadData.html" title="class in picard.illumina.parser">ReadData</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/ReadDescriptor.html" title="class in picard.illumina.parser"><span class="typeNameLink">ReadDescriptor</span></a> - Class in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>
<div class="block">Represents one set of cycles in an ReadStructure (e.g.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadDescriptor.html#ReadDescriptor-int-picard.illumina.parser.ReadType-">ReadDescriptor(int, ReadType)</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/ReadDescriptor.html" title="class in picard.illumina.parser">ReadDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/markduplicates/util/ReadEnds.html" title="class in picard.sam.markduplicates.util"><span class="typeNameLink">ReadEnds</span></a> - Class in <a href="picard/sam/markduplicates/util/package-summary.html">picard.sam.markduplicates.util</a></dt>
<dd>
<div class="block">Little struct-like class to hold read pair (and fragment) end data for duplicate marking.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEnds.html#ReadEnds--">ReadEnds()</a></span> - Constructor for class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEnds.html" title="class in picard.sam.markduplicates.util">ReadEnds</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicates.html" title="class in picard.sam.markduplicates.util"><span class="typeNameLink">ReadEndsForMarkDuplicates</span></a> - Class in <a href="picard/sam/markduplicates/util/package-summary.html">picard.sam.markduplicates.util</a></dt>
<dd>
<div class="block">Little struct-like class to hold read pair (and fragment) end data for MarkDuplicatesWithMateCigar</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicates.html#ReadEndsForMarkDuplicates--">ReadEndsForMarkDuplicates()</a></span> - Constructor for class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicates.html" title="class in picard.sam.markduplicates.util">ReadEndsForMarkDuplicates</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicates.html#ReadEndsForMarkDuplicates-picard.sam.markduplicates.util.ReadEndsForMarkDuplicates-">ReadEndsForMarkDuplicates(ReadEndsForMarkDuplicates)</a></span> - Constructor for class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicates.html" title="class in picard.sam.markduplicates.util">ReadEndsForMarkDuplicates</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesCodec.html" title="class in picard.sam.markduplicates.util"><span class="typeNameLink">ReadEndsForMarkDuplicatesCodec</span></a> - Class in <a href="picard/sam/markduplicates/util/package-summary.html">picard.sam.markduplicates.util</a></dt>
<dd>
<div class="block">Codec for ReadEnds that just outputs the primitive fields and reads them back.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesCodec.html#ReadEndsForMarkDuplicatesCodec--">ReadEndsForMarkDuplicatesCodec()</a></span> - Constructor for class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesCodec.html" title="class in picard.sam.markduplicates.util">ReadEndsForMarkDuplicatesCodec</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesMap.html" title="interface in picard.sam.markduplicates.util"><span class="typeNameLink">ReadEndsForMarkDuplicatesMap</span></a> - Interface in <a href="picard/sam/markduplicates/util/package-summary.html">picard.sam.markduplicates.util</a></dt>
<dd>
<div class="block">Interface for storing and retrieving ReadEnds objects.</div>
</dd>
<dt><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesWithBarcodes.html" title="class in picard.sam.markduplicates.util"><span class="typeNameLink">ReadEndsForMarkDuplicatesWithBarcodes</span></a> - Class in <a href="picard/sam/markduplicates/util/package-summary.html">picard.sam.markduplicates.util</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesWithBarcodes.html#ReadEndsForMarkDuplicatesWithBarcodes--">ReadEndsForMarkDuplicatesWithBarcodes()</a></span> - Constructor for class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesWithBarcodes.html" title="class in picard.sam.markduplicates.util">ReadEndsForMarkDuplicatesWithBarcodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesWithBarcodes.html#ReadEndsForMarkDuplicatesWithBarcodes-picard.sam.markduplicates.util.ReadEndsForMarkDuplicates-">ReadEndsForMarkDuplicatesWithBarcodes(ReadEndsForMarkDuplicates)</a></span> - Constructor for class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesWithBarcodes.html" title="class in picard.sam.markduplicates.util">ReadEndsForMarkDuplicatesWithBarcodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesWithBarcodes.html#ReadEndsForMarkDuplicatesWithBarcodes-picard.sam.markduplicates.util.ReadEndsForMarkDuplicatesWithBarcodes-">ReadEndsForMarkDuplicatesWithBarcodes(ReadEndsForMarkDuplicatesWithBarcodes)</a></span> - Constructor for class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesWithBarcodes.html" title="class in picard.sam.markduplicates.util">ReadEndsForMarkDuplicatesWithBarcodes</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesWithBarcodesCodec.html" title="class in picard.sam.markduplicates.util"><span class="typeNameLink">ReadEndsForMarkDuplicatesWithBarcodesCodec</span></a> - Class in <a href="picard/sam/markduplicates/util/package-summary.html">picard.sam.markduplicates.util</a></dt>
<dd>
<div class="block">Created by nhomer on 9/13/15.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesWithBarcodesCodec.html#ReadEndsForMarkDuplicatesWithBarcodesCodec--">ReadEndsForMarkDuplicatesWithBarcodesCodec()</a></span> - Constructor for class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesWithBarcodesCodec.html" title="class in picard.sam.markduplicates.util">ReadEndsForMarkDuplicatesWithBarcodesCodec</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/markduplicates/util/ReadEndsForMateCigar.html" title="class in picard.sam.markduplicates.util"><span class="typeNameLink">ReadEndsForMateCigar</span></a> - Class in <a href="picard/sam/markduplicates/util/package-summary.html">picard.sam.markduplicates.util</a></dt>
<dd>
<div class="block">A class to store individual records for MarkDuplicatesWithMateCigar.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMateCigar.html#ReadEndsForMateCigar-htsjdk.samtools.SAMFileHeader-htsjdk.samtools.util.SamRecordWithOrdinal-picard.sam.markduplicates.util.OpticalDuplicateFinder-short-">ReadEndsForMateCigar(SAMFileHeader, SamRecordWithOrdinal, OpticalDuplicateFinder, short)</a></span> - Constructor for class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMateCigar.html" title="class in picard.sam.markduplicates.util">ReadEndsForMateCigar</a></dt>
<dd>
<div class="block">Builds a read ends object that represents a single read.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMateCigar.html#ReadEndsForMateCigar-picard.sam.markduplicates.util.ReadEndsForMateCigar-htsjdk.samtools.util.SamRecordWithOrdinal-">ReadEndsForMateCigar(ReadEndsForMateCigar, SamRecordWithOrdinal)</a></span> - Constructor for class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMateCigar.html" title="class in picard.sam.markduplicates.util">ReadEndsForMateCigar</a></dt>
<dd>
<div class="block">Creates a shallow copy from the "other"</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEnds.html#readGroup">readGroup</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEnds.html" title="class in picard.sam.markduplicates.util">ReadEnds</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/RepresentativeReadIndexer.html#readIndexInFile">readIndexInFile</a></span> - Variable in class picard.sam.util.<a href="picard/sam/util/RepresentativeReadIndexer.html" title="class in picard.sam.util">RepresentativeReadIndexer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CollectIlluminaLaneMetrics.IlluminaLaneMetricsCollector.html#readLaneTiles-java.io.File-picard.illumina.parser.ReadStructure-htsjdk.samtools.ValidationStringency-boolean-">readLaneTiles(File, ReadStructure, ValidationStringency, boolean)</a></span> - Static method in class picard.illumina.<a href="picard/illumina/CollectIlluminaLaneMetrics.IlluminaLaneMetricsCollector.html" title="class in picard.illumina">CollectIlluminaLaneMetrics.IlluminaLaneMetricsCollector</a></dt>
<dd>
<div class="block">Returns a partitioned collection of lane number to Tile objects from the provided basecall directory.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadStructure.html#readLengths">readLengths</a></span> - Variable in class picard.illumina.parser.<a href="picard/illumina/parser/ReadStructure.html" title="class in picard.illumina.parser">ReadStructure</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/fastq/ReadNameEncoder.html" title="interface in picard.fastq"><span class="typeNameLink">ReadNameEncoder</span></a> - Interface in <a href="picard/fastq/package-summary.html">picard.fastq</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/util/ReadNameParser.html" title="class in picard.sam.util"><span class="typeNameLink">ReadNameParser</span></a> - Class in <a href="picard/sam/util/package-summary.html">picard.sam.util</a></dt>
<dd>
<div class="block">Provides access to the physical location information about a cluster.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/ReadNameParser.html#ReadNameParser--">ReadNameParser()</a></span> - Constructor for class picard.sam.util.<a href="picard/sam/util/ReadNameParser.html" title="class in picard.sam.util">ReadNameParser</a></dt>
<dd>
<div class="block">Creates are read name parser using the default read name regex and optical duplicate distance.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/ReadNameParser.html#ReadNameParser-java.lang.String-">ReadNameParser(String)</a></span> - Constructor for class picard.sam.util.<a href="picard/sam/util/ReadNameParser.html" title="class in picard.sam.util">ReadNameParser</a></dt>
<dd>
<div class="block">Creates are read name parser using the given read name regex.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/ReadNameParser.html#ReadNameParser-java.lang.String-htsjdk.samtools.util.Log-">ReadNameParser(String, Log)</a></span> - Constructor for class picard.sam.util.<a href="picard/sam/util/ReadNameParser.html" title="class in picard.sam.util">ReadNameParser</a></dt>
<dd>
<div class="block">Creates are read name parser using the given read name regex.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/AbstractInputParser.html#readNextLine--">readNextLine()</a></span> - Method in class picard.util.<a href="picard/util/AbstractInputParser.html" title="class in picard.util">AbstractInputParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/BasicInputParser.html#readNextLine--">readNextLine()</a></span> - Method in class picard.util.<a href="picard/util/BasicInputParser.html" title="class in picard.util">BasicInputParser</a></dt>
<dd>
<div class="block">Workhorse method that reads the next line from the underlying reader</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesWithBarcodes.html#readOneBarcode">readOneBarcode</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesWithBarcodes.html" title="class in picard.sam.markduplicates.util">ReadEndsForMarkDuplicatesWithBarcodes</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/util/ReadOutputCommandLineProgram.html" title="class in picard.sam.util"><span class="typeNameLink">ReadOutputCommandLineProgram</span></a> - Class in <a href="picard/sam/util/package-summary.html">picard.sam.util</a></dt>
<dd>
<div class="block">Abstract class that holds parameters and methods common to classes that want to add PG tags to reads in SAM/BAM files</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/ReadOutputCommandLineProgram.html#ReadOutputCommandLineProgram--">ReadOutputCommandLineProgram()</a></span> - Constructor for class picard.sam.util.<a href="picard/sam/util/ReadOutputCommandLineProgram.html" title="class in picard.sam.util">ReadOutputCommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/QuerySortedReadPairIteratorUtil.ReadPair.html#ReadPair--">ReadPair()</a></span> - Constructor for class picard.util.<a href="picard/util/QuerySortedReadPairIteratorUtil.ReadPair.html" title="class in picard.util">QuerySortedReadPairIteratorUtil.ReadPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html#READS">READS</a></span> - Variable in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.BarcodeMetric.html" title="class in picard.illumina">ExtractIlluminaBarcodes.BarcodeMetric</a></dt>
<dd>
<div class="block">The total number of reads matching the barcode.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html#READS">READS</a></span> - Variable in class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics.PFFailSummaryMetric</a></dt>
<dd>
<div class="block">The total number of reads examined</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RrbsSummaryMetrics.html#READS_ALIGNED">READS_ALIGNED</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RrbsSummaryMetrics.html" title="class in picard.analysis">RrbsSummaryMetrics</a></dt>
<dd>
<div class="block">Number of mapped reads processed</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AlignmentSummaryMetrics.html#READS_ALIGNED_IN_PAIRS">READS_ALIGNED_IN_PAIRS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/AlignmentSummaryMetrics.html" title="class in picard.analysis">AlignmentSummaryMetrics</a></dt>
<dd>
<div class="block">The number of aligned reads whose mate pair was also aligned to the reference.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RrbsSummaryMetrics.html#READS_IGNORED_MISMATCHES">READS_IGNORED_MISMATCHES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RrbsSummaryMetrics.html" title="class in picard.analysis">RrbsSummaryMetrics</a></dt>
<dd>
<div class="block">Number of reads discarded for exceeding the mismatch threshold</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RrbsSummaryMetrics.html#READS_IGNORED_SHORT">READS_IGNORED_SHORT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RrbsSummaryMetrics.html" title="class in picard.analysis">RrbsSummaryMetrics</a></dt>
<dd>
<div class="block">Number of reads discarded due to being too short</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fastq/BamToBfq.html#READS_TO_ALIGN">READS_TO_ALIGN</a></span> - Variable in class picard.fastq.<a href="picard/fastq/BamToBfq.html" title="class in picard.fastq">BamToBfq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasDetailMetrics.html#READS_USED">READS_USED</a></span> - Variable in class picard.analysis.<a href="picard/analysis/GcBiasDetailMetrics.html" title="class in picard.analysis">GcBiasDetailMetrics</a></dt>
<dd>
<div class="block">This option is used to mark including or excluding duplicates.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasSummaryMetrics.html#READS_USED">READS_USED</a></span> - Variable in class picard.analysis.<a href="picard/analysis/GcBiasSummaryMetrics.html" title="class in picard.analysis">GcBiasSummaryMetrics</a></dt>
<dd>
<div class="block">This option is used to mark including or excluding duplicates.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RrbsSummaryMetrics.html#READS_WITH_NO_CPG">READS_WITH_NO_CPG</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RrbsSummaryMetrics.html" title="class in picard.analysis">RrbsSummaryMetrics</a></dt>
<dd>
<div class="block">Number of reads discarded for having no CpG sites</div>
</dd>
<dt><a href="picard/illumina/parser/ReadStructure.html" title="class in picard.illumina.parser"><span class="typeNameLink">ReadStructure</span></a> - Class in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>
<div class="block">Describes the intended logical output structure of clusters of an Illumina run.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadStructure.html#ReadStructure-java.util.List-">ReadStructure(List&lt;ReadDescriptor&gt;)</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/ReadStructure.html" title="class in picard.illumina.parser">ReadStructure</a></dt>
<dd>
<div class="block">Copies collection into descriptors (making descriptors unmodifiable) and then calculates relevant statistics about descriptors.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadStructure.html#ReadStructure-java.lang.String-">ReadStructure(String)</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/ReadStructure.html" title="class in picard.illumina.parser">ReadStructure</a></dt>
<dd>
<div class="block">Converts readStructureString into a List<ReadDescriptor> and calls the primary constructor using this List as it's argument.</div>
</dd>
<dt><a href="picard/illumina/parser/ReadStructure.Substructure.html" title="class in picard.illumina.parser"><span class="typeNameLink">ReadStructure.Substructure</span></a> - Class in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>
<div class="block">Represents a subset of ReadDescriptors in the containing ReadStructure, they ARE NOT necessarily contiguous
  in the containing ReadStructure but they ARE in the order they appear in the containing ReadStructure</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesWithBarcodes.html#readTwoBarcode">readTwoBarcode</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesWithBarcodes.html" title="class in picard.sam.markduplicates.util">ReadEndsForMarkDuplicatesWithBarcodes</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/ReadType.html" title="enum in picard.illumina.parser"><span class="typeNameLink">ReadType</span></a> - Enum in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>
<div class="block">A read type describes a stretch of cycles in an ReadStructure
 (e.g.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/filter/CountingDuplicateFilter.html#reallyFilterOut-htsjdk.samtools.SAMRecord-">reallyFilterOut(SAMRecord)</a></span> - Method in class picard.filter.<a href="picard/filter/CountingDuplicateFilter.html" title="class in picard.filter">CountingDuplicateFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/filter/CountingFilter.html#reallyFilterOut-htsjdk.samtools.SAMRecord-">reallyFilterOut(SAMRecord)</a></span> - Method in class picard.filter.<a href="picard/filter/CountingFilter.html" title="class in picard.filter">CountingFilter</a></dt>
<dd>
<div class="block">Return true if we are to filter this record out, false otherwise.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/filter/CountingMapQFilter.html#reallyFilterOut-htsjdk.samtools.SAMRecord-">reallyFilterOut(SAMRecord)</a></span> - Method in class picard.filter.<a href="picard/filter/CountingMapQFilter.html" title="class in picard.filter">CountingMapQFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/filter/CountingPairedFilter.html#reallyFilterOut-htsjdk.samtools.SAMRecord-">reallyFilterOut(SAMRecord)</a></span> - Method in class picard.filter.<a href="picard/filter/CountingPairedFilter.html" title="class in picard.filter">CountingPairedFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ViewSam.html#RECORDS_ONLY">RECORDS_ONLY</a></span> - Variable in class picard.sam.<a href="picard/sam/ViewSam.html" title="class in picard.sam">ViewSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/ErrorSummaryMetrics.html#REF_BASE">REF_BASE</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/ErrorSummaryMetrics.html" title="class in picard.analysis.artifacts">ErrorSummaryMetrics</a></dt>
<dd>
<div class="block">The reference base (or it's complement).</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html#REF_BASE">REF_BASE</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.BaitBiasDetailMetrics</a></dt>
<dd>
<div class="block">The (upper-case) original base on the reference strand.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasSummaryMetrics.html#REF_BASE">REF_BASE</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasSummaryMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.BaitBiasSummaryMetrics</a></dt>
<dd>
<div class="block">The (upper-case) original base on the reference strand.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterDetailMetrics.html#REF_BASE">REF_BASE</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterDetailMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.PreAdapterDetailMetrics</a></dt>
<dd>
<div class="block">The (upper-case) original base on the reference strand.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterSummaryMetrics.html#REF_BASE">REF_BASE</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterSummaryMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.PreAdapterSummaryMetrics</a></dt>
<dd>
<div class="block">The (upper-case) original base on the reference strand.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/ErrorSummaryMetrics.html#REF_COUNT">REF_COUNT</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/ErrorSummaryMetrics.html" title="class in picard.analysis.artifacts">ErrorSummaryMetrics</a></dt>
<dd>
<div class="block">The number of reference bases observed.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRnaSeqMetrics.html#REF_FLAT">REF_FLAT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectRnaSeqMetrics.html" title="class in picard.analysis">CollectRnaSeqMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html#REF_NONOXO_BASES">REF_NONOXO_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html" title="class in picard.analysis">CollectOxoGMetrics.CpcgMetrics</a></dt>
<dd>
<div class="block">The number of reference alleles observed as C in read 1 and G in read 2.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html#REF_OXO_BASES">REF_OXO_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html" title="class in picard.analysis">CollectOxoGMetrics.CpcgMetrics</a></dt>
<dd>
<div class="block">The number of reference alleles observed as G in read 1 and C in read 2.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html#REF_TOTAL_BASES">REF_TOTAL_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html" title="class in picard.analysis">CollectOxoGMetrics.CpcgMetrics</a></dt>
<dd>
<div class="block">The total number of reference alleles observed</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRrbsMetrics.CollectRrbsMetricsReferenceArgumentCollection.html#REFERENCE">REFERENCE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectRrbsMetrics.CollectRrbsMetricsReferenceArgumentCollection.html" title="class in picard.analysis">CollectRrbsMetrics.CollectRrbsMetricsReferenceArgumentCollection</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CreateSequenceDictionary.CreateSeqDictReferenceArgumentCollection.html#REFERENCE">REFERENCE</a></span> - Variable in class picard.sam.<a href="picard/sam/CreateSequenceDictionary.CreateSeqDictReferenceArgumentCollection.html" title="class in picard.sam">CreateSequenceDictionary.CreateSeqDictReferenceArgumentCollection</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ReorderSam.ReorderSamReferenceArgumentCollection.html#REFERENCE">REFERENCE</a></span> - Variable in class picard.sam.<a href="picard/sam/ReorderSam.ReorderSamReferenceArgumentCollection.html" title="class in picard.sam">ReorderSam.ReorderSamReferenceArgumentCollection</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/ScatterIntervalsByNs.ScatterIntervalsByNReferenceArgumentCollection.html#REFERENCE">REFERENCE</a></span> - Variable in class picard.util.<a href="picard/util/ScatterIntervalsByNs.ScatterIntervalsByNReferenceArgumentCollection.html" title="class in picard.util">ScatterIntervalsByNs.ScatterIntervalsByNReferenceArgumentCollection</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectAlignmentSummaryMetrics.CollectAlignmentRefArgCollection.html#REFERENCE_SEQUENCE">REFERENCE_SEQUENCE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectAlignmentSummaryMetrics.CollectAlignmentRefArgCollection.html" title="class in picard.analysis">CollectAlignmentSummaryMetrics.CollectAlignmentRefArgCollection</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/argumentcollections/OptionalReferenceArgumentCollection.html#REFERENCE_SEQUENCE">REFERENCE_SEQUENCE</a></span> - Variable in class picard.cmdline.argumentcollections.<a href="picard/cmdline/argumentcollections/OptionalReferenceArgumentCollection.html" title="class in picard.cmdline.argumentcollections">OptionalReferenceArgumentCollection</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/argumentcollections/RequiredReferenceArgumentCollection.html#REFERENCE_SEQUENCE">REFERENCE_SEQUENCE</a></span> - Variable in class picard.cmdline.argumentcollections.<a href="picard/cmdline/argumentcollections/RequiredReferenceArgumentCollection.html" title="class in picard.cmdline.argumentcollections">RequiredReferenceArgumentCollection</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#REFERENCE_SEQUENCE">REFERENCE_SEQUENCE</a></span> - Variable in class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/StandardOptionDefinitions.html#REFERENCE_SHORT_NAME">REFERENCE_SHORT_NAME</a></span> - Static variable in class picard.cmdline.<a href="picard/cmdline/StandardOptionDefinitions.html" title="class in picard.cmdline">StandardOptionDefinitions</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/cmdline/argumentcollections/ReferenceArgumentCollection.html" title="interface in picard.cmdline.argumentcollections"><span class="typeNameLink">ReferenceArgumentCollection</span></a> - Interface in <a href="picard/cmdline/argumentcollections/package-summary.html">picard.cmdline.argumentcollections</a></dt>
<dd>
<div class="block">Base interface for a reference argument collection.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.html#referenceFasta">referenceFasta</a></span> - Variable in class picard.sam.<a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#referenceSequence">referenceSequence</a></span> - Variable in class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/annotation/RefFlatReader.html" title="class in picard.annotation"><span class="typeNameLink">RefFlatReader</span></a> - Class in <a href="picard/annotation/package-summary.html">picard.annotation</a></dt>
<dd>
<div class="block">Loads gene annotations from a refFlat file into an OverlapDetector<Gene>.</div>
</dd>
<dt><a href="picard/annotation/RefFlatReader.RefFlatColumns.html" title="enum in picard.annotation"><span class="typeNameLink">RefFlatReader.RefFlatColumns</span></a> - Enum in <a href="picard/annotation/package-summary.html">picard.annotation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/ReflectionUtil.html" title="class in picard.util"><span class="typeNameLink">ReflectionUtil</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">Class which contains utility functions that use reflection.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/ReflectionUtil.html#ReflectionUtil--">ReflectionUtil()</a></span> - Constructor for class picard.util.<a href="picard/util/ReflectionUtil.html" title="class in picard.util">ReflectionUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetMetricsCollector.html#reflectiveCopy-picard.analysis.directed.TargetMetrics-MT-java.lang.String:A-java.lang.String:A-">reflectiveCopy(TargetMetrics, MT, String[], String[])</a></span> - Static method in class picard.analysis.directed.<a href="picard/analysis/directed/TargetMetricsCollector.html" title="class in picard.analysis.directed">TargetMetricsCollector</a></dt>
<dd>
<div class="block">Since the targeted metrics (HsMetrics, TargetedPcrMetrics,...) share many of the same values as TargetMetrics, this copy will copy all public attributes in targetMetrics
 to the outputMetrics' attributes of the same name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/LiftoverVcf.html#REJECT">REJECT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/LiftoverVcf.html" title="class in picard.vcf">LiftoverVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaDataProvider.html#remove--">remove()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/IlluminaDataProvider.html" title="class in picard.illumina.parser">IlluminaDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/MultiTileParser.html#remove--">remove()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/MultiTileParser.html" title="class in picard.illumina.parser">MultiTileParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PerTileParser.html#remove--">remove()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/PerTileParser.html" title="class in picard.illumina.parser">PerTileParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.html#remove--">remove()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.html" title="class in picard.illumina.parser.readers">AbstractIlluminaPositionFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/BarcodeFileReader.html#remove--">remove()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/BarcodeFileReader.html" title="class in picard.illumina.parser.readers">BarcodeFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/BclReader.html#remove--">remove()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/BclReader.html" title="class in picard.illumina.parser.readers">BclReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/FilterFileReader.html#remove--">remove()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/FilterFileReader.html" title="class in picard.illumina.parser.readers">FilterFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/TileMetricsOutReader.html#remove--">remove()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/TileMetricsOutReader.html" title="class in picard.illumina.parser.readers">TileMetricsOutReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigarIterator.html#remove--">remove()</a></span> - Method in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicatesWithMateCigarIterator.html" title="class in picard.sam.markduplicates">MarkDuplicatesWithMateCigarIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/DiskBasedReadEndsForMarkDuplicatesMap.html#remove-int-java.lang.String-">remove(int, String)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/DiskBasedReadEndsForMarkDuplicatesMap.html" title="class in picard.sam.markduplicates.util">DiskBasedReadEndsForMarkDuplicatesMap</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/PhysicalLocationForMateCigarSet.html#remove-picard.sam.markduplicates.util.ReadEndsForMateCigar-">remove(ReadEndsForMateCigar)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/PhysicalLocationForMateCigarSet.html" title="class in picard.sam.markduplicates.util">PhysicalLocationForMateCigarSet</a></dt>
<dd>
<div class="block">Removes the end from this set, if present</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesMap.html#remove-int-java.lang.String-">remove(int, String)</a></span> - Method in interface picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesMap.html" title="interface in picard.sam.markduplicates.util">ReadEndsForMarkDuplicatesMap</a></dt>
<dd>
<div class="block">Remove element with given key from the map.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/AsyncIterator.html#remove--">remove()</a></span> - Method in class picard.util.<a href="picard/util/AsyncIterator.html" title="class in picard.util">AsyncIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/DelimitedTextFileWithHeaderIterator.html#remove--">remove()</a></span> - Method in class picard.util.<a href="picard/util/DelimitedTextFileWithHeaderIterator.html" title="class in picard.util">DelimitedTextFileWithHeaderIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/ByIntervalListVariantContextIterator.html#remove--">remove()</a></span> - Method in class picard.vcf.<a href="picard/vcf/ByIntervalListVariantContextIterator.html" title="class in picard.vcf">ByIntervalListVariantContextIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/filter/FilterApplyingVariantIterator.html#remove--">remove()</a></span> - Method in class picard.vcf.filter.<a href="picard/vcf/filter/FilterApplyingVariantIterator.html" title="class in picard.vcf.filter">FilterApplyingVariantIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/PairedVariantSubContextIterator.html#remove--">remove()</a></span> - Method in class picard.vcf.<a href="picard/vcf/PairedVariantSubContextIterator.html" title="class in picard.vcf">PairedVariantSubContextIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/util/PredicateFilterDecoratingClosableIterator.html#remove--">remove()</a></span> - Method in class picard.vcf.processor.util.<a href="picard/vcf/processor/util/PredicateFilterDecoratingClosableIterator.html" title="class in picard.vcf.processor.util">PredicateFilterDecoratingClosableIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertSam.html#REMOVE_ALIGNMENT_INFORMATION">REMOVE_ALIGNMENT_INFORMATION</a></span> - Variable in class picard.sam.<a href="picard/sam/RevertSam.html" title="class in picard.sam">RevertSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/PositionBasedDownsampleSam.html#REMOVE_DUPLICATE_INFORMATION">REMOVE_DUPLICATE_INFORMATION</a></span> - Variable in class picard.sam.<a href="picard/sam/PositionBasedDownsampleSam.html" title="class in picard.sam">PositionBasedDownsampleSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertSam.html#REMOVE_DUPLICATE_INFORMATION">REMOVE_DUPLICATE_INFORMATION</a></span> - Variable in class picard.sam.<a href="picard/sam/RevertSam.html" title="class in picard.sam">RevertSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html#REMOVE_DUPLICATES">REMOVE_DUPLICATES</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html" title="class in picard.sam.markduplicates.util">AbstractMarkDuplicatesCommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicates.html#REMOVE_SEQUENCING_DUPLICATES">REMOVE_SEQUENCING_DUPLICATES</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicates.html" title="class in picard.sam.markduplicates">MarkDuplicates</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/pedigree/PedFile.html#removeIncompleteTrios--">removeIncompleteTrios()</a></span> - Method in class picard.pedigree.<a href="picard/pedigree/PedFile.html" title="class in picard.pedigree">PedFile</a></dt>
<dd>
<div class="block">Scans through the pedigrees and removes all entries that do not have both paternal and maternal ids set.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMateCigar.html#removeLocationSet--">removeLocationSet()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMateCigar.html" title="class in picard.sam.markduplicates.util">ReadEndsForMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/RenameSampleInVcf.html" title="class in picard.vcf"><span class="typeNameLink">RenameSampleInVcf</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/RenameSampleInVcf.html#RenameSampleInVcf--">RenameSampleInVcf()</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/RenameSampleInVcf.html" title="class in picard.vcf">RenameSampleInVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/TileMetricsUtil.html#renderPhasingMetricsFilesFromBasecallingDirectory-java.io.File-">renderPhasingMetricsFilesFromBasecallingDirectory(File)</a></span> - Static method in class picard.illumina.parser.<a href="picard/illumina/parser/TileMetricsUtil.html" title="class in picard.illumina.parser">TileMetricsUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/TileMetricsUtil.html#renderTileMetricsFileFromBasecallingDirectory-java.io.File-boolean-">renderTileMetricsFileFromBasecallingDirectory(File, boolean)</a></span> - Static method in class picard.illumina.parser.<a href="picard/illumina/parser/TileMetricsUtil.html" title="class in picard.illumina.parser">TileMetricsUtil</a></dt>
<dd>
<div class="block">Returns the path to the TileMetrics file given the basecalling directory.</div>
</dd>
<dt><a href="picard/sam/ReorderSam.html" title="class in picard.sam"><span class="typeNameLink">ReorderSam</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block">Reorders a SAM/BAM input file according to the order of contigs in a second reference sequence</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ReorderSam.html#ReorderSam--">ReorderSam()</a></span> - Constructor for class picard.sam.<a href="picard/sam/ReorderSam.html" title="class in picard.sam">ReorderSam</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/ReorderSam.ReorderSamReferenceArgumentCollection.html" title="class in picard.sam"><span class="typeNameLink">ReorderSam.ReorderSamReferenceArgumentCollection</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ReorderSam.ReorderSamReferenceArgumentCollection.html#ReorderSamReferenceArgumentCollection--">ReorderSamReferenceArgumentCollection()</a></span> - Constructor for class picard.sam.<a href="picard/sam/ReorderSam.ReorderSamReferenceArgumentCollection.html" title="class in picard.sam">ReorderSam.ReorderSamReferenceArgumentCollection</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/BaitDesigner.html#REPEAT_TOLERANCE">REPEAT_TOLERANCE</a></span> - Variable in class picard.util.<a href="picard/util/BaitDesigner.html" title="class in picard.util">BaitDesigner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/PhysicalLocationForMateCigarSet.html#replace-picard.sam.markduplicates.util.ReadEndsForMateCigar-picard.sam.markduplicates.util.ReadEndsForMateCigar-">replace(ReadEndsForMateCigar, ReadEndsForMateCigar)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/PhysicalLocationForMateCigarSet.html" title="class in picard.sam.markduplicates.util">PhysicalLocationForMateCigarSet</a></dt>
<dd>
<div class="block">Replaces a given end with the other end.</div>
</dd>
<dt><a href="picard/sam/ReplaceSamHeader.html" title="class in picard.sam"><span class="typeNameLink">ReplaceSamHeader</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ReplaceSamHeader.html#ReplaceSamHeader--">ReplaceSamHeader()</a></span> - Constructor for class picard.sam.<a href="picard/sam/ReplaceSamHeader.html" title="class in picard.sam">ReplaceSamHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#replicationRateFromReplicateSets">replicationRateFromReplicateSets</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/util/RepresentativeReadIndexer.html" title="class in picard.sam.util"><span class="typeNameLink">RepresentativeReadIndexer</span></a> - Class in <a href="picard/sam/util/package-summary.html">picard.sam.util</a></dt>
<dd>
<div class="block">Little struct-like class to hold a record index, the index of the corresponding representative read, and duplicate set size information.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/RepresentativeReadIndexer.html#RepresentativeReadIndexer--">RepresentativeReadIndexer()</a></span> - Constructor for class picard.sam.util.<a href="picard/sam/util/RepresentativeReadIndexer.html" title="class in picard.sam.util">RepresentativeReadIndexer</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/markduplicates/util/RepresentativeReadIndexerCodec.html" title="class in picard.sam.markduplicates.util"><span class="typeNameLink">RepresentativeReadIndexerCodec</span></a> - Class in <a href="picard/sam/markduplicates/util/package-summary.html">picard.sam.markduplicates.util</a></dt>
<dd>
<div class="block">Codec for read names and integers that outputs the primitive fields and reads them back.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/RepresentativeReadIndexerCodec.html#RepresentativeReadIndexerCodec--">RepresentativeReadIndexerCodec()</a></span> - Constructor for class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/RepresentativeReadIndexerCodec.html" title="class in picard.sam.markduplicates.util">RepresentativeReadIndexerCodec</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/RepresentativeReadIndexer.html#representativeReadIndexInFile">representativeReadIndexInFile</a></span> - Variable in class picard.sam.util.<a href="picard/sam/util/RepresentativeReadIndexer.html" title="class in picard.sam.util">RepresentativeReadIndexer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/VcfFormatConverter.html#REQUIRE_INDEX">REQUIRE_INDEX</a></span> - Variable in class picard.vcf.<a href="picard/vcf/VcfFormatConverter.html" title="class in picard.vcf">VcfFormatConverter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/cmdline/argumentcollections/RequiredReferenceArgumentCollection.html" title="class in picard.cmdline.argumentcollections"><span class="typeNameLink">RequiredReferenceArgumentCollection</span></a> - Class in <a href="picard/cmdline/argumentcollections/package-summary.html">picard.cmdline.argumentcollections</a></dt>
<dd>
<div class="block">Argument collection for references that are required (and not common).</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/argumentcollections/RequiredReferenceArgumentCollection.html#RequiredReferenceArgumentCollection--">RequiredReferenceArgumentCollection()</a></span> - Constructor for class picard.cmdline.argumentcollections.<a href="picard/cmdline/argumentcollections/RequiredReferenceArgumentCollection.html" title="class in picard.cmdline.argumentcollections">RequiredReferenceArgumentCollection</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html#requiresReference--">requiresReference()</a></span> - Method in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html" title="class in picard.analysis.artifacts">CollectSequencingArtifactMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.html#requiresReference--">requiresReference()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.html" title="class in picard.analysis">CollectOxoGMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.html#requiresReference--">requiresReference()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#requiresReference--">requiresReference()</a></span> - Method in class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/reference/ExtractSequences.html#requiresReference--">requiresReference()</a></span> - Method in class picard.reference.<a href="picard/reference/ExtractSequences.html" title="class in picard.reference">ExtractSequences</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#requiresReference--">requiresReference()</a></span> - Method in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SetNmMdAndUqTags.html#requiresReference--">requiresReference()</a></span> - Method in class picard.sam.<a href="picard/sam/SetNmMdAndUqTags.html" title="class in picard.sam">SetNmMdAndUqTags</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/BaitDesigner.html#requiresReference--">requiresReference()</a></span> - Method in class picard.util.<a href="picard/util/BaitDesigner.html" title="class in picard.util">BaitDesigner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.html#resetRefSeqFileWalker--">resetRefSeqFileWalker()</a></span> - Method in class picard.sam.<a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertOriginalBaseQualitiesAndAddMateCigar.html#RESTORE_ORIGINAL_QUALITIES">RESTORE_ORIGINAL_QUALITIES</a></span> - Variable in class picard.sam.<a href="picard/sam/RevertOriginalBaseQualitiesAndAddMateCigar.html" title="class in picard.sam">RevertOriginalBaseQualitiesAndAddMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertSam.html#RESTORE_ORIGINAL_QUALITIES">RESTORE_ORIGINAL_QUALITIES</a></span> - Variable in class picard.sam.<a href="picard/sam/RevertSam.html" title="class in picard.sam">RevertSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckMetric.html#RESULT">RESULT</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckMetric.html" title="class in picard.fingerprint">CrosscheckMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CallingMetricAccumulator.html#result--">result()</a></span> - Method in class picard.vcf.<a href="picard/vcf/CallingMetricAccumulator.html" title="class in picard.vcf">CallingMetricAccumulator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VariantProcessor.Accumulator.html#result--">result()</a></span> - Method in interface picard.vcf.processor.<a href="picard/vcf/processor/VariantProcessor.Accumulator.html" title="interface in picard.vcf.processor">VariantProcessor.Accumulator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html#REV_CXT_ALT_BASES">REV_CXT_ALT_BASES</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.BaitBiasDetailMetrics</a></dt>
<dd>
<div class="block">The number of ~REF_BASE:~ALT_BASE alignments at sites complementary to the given reference context.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html#REV_CXT_REF_BASES">REV_CXT_REF_BASES</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.BaitBiasDetailMetrics</a></dt>
<dd>
<div class="block">The number of ~REF_BASE:~REF_BASE alignments at sites complementary to the given reference context.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html#REV_ERROR_RATE">REV_ERROR_RATE</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.BaitBiasDetailMetrics</a></dt>
<dd>
<div class="block">The substitution rate of ~REF_BASE:~ALT_BASE, calculated as max(1e-10, REV_CXT_ALT_BASES / (REV_CXT_ALT_BASES + REV_CXT_REF_BASES)).</div>
</dd>
<dt><a href="picard/sam/RevertOriginalBaseQualitiesAndAddMateCigar.html" title="class in picard.sam"><span class="typeNameLink">RevertOriginalBaseQualitiesAndAddMateCigar</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block">This tool reverts the original base qualities (if specified) and adds the mate cigar tag to mapped BAMs.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertOriginalBaseQualitiesAndAddMateCigar.html#RevertOriginalBaseQualitiesAndAddMateCigar--">RevertOriginalBaseQualitiesAndAddMateCigar()</a></span> - Constructor for class picard.sam.<a href="picard/sam/RevertOriginalBaseQualitiesAndAddMateCigar.html" title="class in picard.sam">RevertOriginalBaseQualitiesAndAddMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/RevertOriginalBaseQualitiesAndAddMateCigar.CanSkipSamFile.html" title="enum in picard.sam"><span class="typeNameLink">RevertOriginalBaseQualitiesAndAddMateCigar.CanSkipSamFile</span></a> - Enum in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block">Used as a return for the canSkipSAMFile function.</div>
</dd>
<dt><a href="picard/sam/RevertSam.html" title="class in picard.sam"><span class="typeNameLink">RevertSam</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block">Reverts a SAM file by optionally restoring original quality scores and by removing
 all alignment information.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertSam.html#RevertSam--">RevertSam()</a></span> - Constructor for class picard.sam.<a href="picard/sam/RevertSam.html" title="class in picard.sam">RevertSam</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/RevertSam.FileType.html" title="enum in picard.sam"><span class="typeNameLink">RevertSam.FileType</span></a> - Enum in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertSam.html#revertSamRecord-htsjdk.samtools.SAMRecord-">revertSamRecord(SAMRecord)</a></span> - Method in class picard.sam.<a href="picard/sam/RevertSam.html" title="class in picard.sam">RevertSam</a></dt>
<dd>
<div class="block">Takes an individual SAMRecord and applies the set of changes/reversions to it that
 have been requested by program level options.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/BclQualityEvaluationStrategy.html#reviseAndConditionallyLogQuality-byte-">reviseAndConditionallyLogQuality(byte)</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/BclQualityEvaluationStrategy.html" title="class in picard.illumina.parser.readers">BclQualityEvaluationStrategy</a></dt>
<dd>
<div class="block">Accepts a quality read from a BCL file and (1) returns a 1 if the value was 0 and (2) makes a note of the provided quality if it is
 low.</div>
</dd>
<dt><a href="picard/util/RExecutor.html" title="class in picard.util"><span class="typeNameLink">RExecutor</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">Util class for executing R scripts.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/RExecutor.html#RExecutor--">RExecutor()</a></span> - Constructor for class picard.util.<a href="picard/util/RExecutor.html" title="class in picard.util">RExecutor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEnds.html#RF">RF</a></span> - Static variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEnds.html" title="class in picard.sam.markduplicates.util">ReadEnds</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamToFastq.html#RG_TAG">RG_TAG</a></span> - Variable in class picard.sam.<a href="picard/sam/SamToFastq.html" title="class in picard.sam">SamToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AddOrReplaceReadGroups.html#RGCN">RGCN</a></span> - Variable in class picard.sam.<a href="picard/sam/AddOrReplaceReadGroups.html" title="class in picard.sam">AddOrReplaceReadGroups</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AddOrReplaceReadGroups.html#RGDS">RGDS</a></span> - Variable in class picard.sam.<a href="picard/sam/AddOrReplaceReadGroups.html" title="class in picard.sam">AddOrReplaceReadGroups</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AddOrReplaceReadGroups.html#RGDT">RGDT</a></span> - Variable in class picard.sam.<a href="picard/sam/AddOrReplaceReadGroups.html" title="class in picard.sam">AddOrReplaceReadGroups</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AddOrReplaceReadGroups.html#RGFO">RGFO</a></span> - Variable in class picard.sam.<a href="picard/sam/AddOrReplaceReadGroups.html" title="class in picard.sam">AddOrReplaceReadGroups</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AddOrReplaceReadGroups.html#RGID">RGID</a></span> - Variable in class picard.sam.<a href="picard/sam/AddOrReplaceReadGroups.html" title="class in picard.sam">AddOrReplaceReadGroups</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AddOrReplaceReadGroups.html#RGKS">RGKS</a></span> - Variable in class picard.sam.<a href="picard/sam/AddOrReplaceReadGroups.html" title="class in picard.sam">AddOrReplaceReadGroups</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AddOrReplaceReadGroups.html#RGLB">RGLB</a></span> - Variable in class picard.sam.<a href="picard/sam/AddOrReplaceReadGroups.html" title="class in picard.sam">AddOrReplaceReadGroups</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AddOrReplaceReadGroups.html#RGPG">RGPG</a></span> - Variable in class picard.sam.<a href="picard/sam/AddOrReplaceReadGroups.html" title="class in picard.sam">AddOrReplaceReadGroups</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AddOrReplaceReadGroups.html#RGPI">RGPI</a></span> - Variable in class picard.sam.<a href="picard/sam/AddOrReplaceReadGroups.html" title="class in picard.sam">AddOrReplaceReadGroups</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AddOrReplaceReadGroups.html#RGPL">RGPL</a></span> - Variable in class picard.sam.<a href="picard/sam/AddOrReplaceReadGroups.html" title="class in picard.sam">AddOrReplaceReadGroups</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AddOrReplaceReadGroups.html#RGPM">RGPM</a></span> - Variable in class picard.sam.<a href="picard/sam/AddOrReplaceReadGroups.html" title="class in picard.sam">AddOrReplaceReadGroups</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AddOrReplaceReadGroups.html#RGPU">RGPU</a></span> - Variable in class picard.sam.<a href="picard/sam/AddOrReplaceReadGroups.html" title="class in picard.sam">AddOrReplaceReadGroups</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AddOrReplaceReadGroups.html#RGSM">RGSM</a></span> - Variable in class picard.sam.<a href="picard/sam/AddOrReplaceReadGroups.html" title="class in picard.sam">AddOrReplaceReadGroups</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RnaSeqMetrics.html#RIBOSOMAL_BASES">RIBOSOMAL_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RnaSeqMetrics.html" title="class in picard.analysis">RnaSeqMetrics</a></dt>
<dd>
<div class="block">Number of bases in primary alignments that align to ribosomal sequence.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRnaSeqMetrics.html#RIBOSOMAL_INTERVALS">RIBOSOMAL_INTERVALS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectRnaSeqMetrics.html" title="class in picard.analysis">CollectRnaSeqMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/RnaSeqMetricsCollector.html#ribosomalInitialValue">ribosomalInitialValue</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/RnaSeqMetricsCollector.html" title="class in picard.analysis.directed">RnaSeqMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckMetric.html#RIGHT_FILE">RIGHT_FILE</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckMetric.html" title="class in picard.fingerprint">CrosscheckMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckMetric.html#RIGHT_GROUP_VALUE">RIGHT_GROUP_VALUE</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckMetric.html" title="class in picard.fingerprint">CrosscheckMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckMetric.html#RIGHT_LANE">RIGHT_LANE</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckMetric.html" title="class in picard.fingerprint">CrosscheckMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckMetric.html#RIGHT_LIBRARY">RIGHT_LIBRARY</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckMetric.html" title="class in picard.fingerprint">CrosscheckMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckMetric.html#RIGHT_MOLECULAR_BARCODE_SEQUENCE">RIGHT_MOLECULAR_BARCODE_SEQUENCE</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckMetric.html" title="class in picard.fingerprint">CrosscheckMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/BaitDesigner.html#RIGHT_PRIMER">RIGHT_PRIMER</a></span> - Variable in class picard.util.<a href="picard/util/BaitDesigner.html" title="class in picard.util">BaitDesigner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckMetric.html#RIGHT_RUN_BARCODE">RIGHT_RUN_BARCODE</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckMetric.html" title="class in picard.fingerprint">CrosscheckMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckMetric.html#RIGHT_SAMPLE">RIGHT_SAMPLE</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckMetric.html" title="class in picard.fingerprint">CrosscheckMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/PairedVariantSubContextIterator.VcfTuple.html#rightVariantContext">rightVariantContext</a></span> - Variable in class picard.vcf.<a href="picard/vcf/PairedVariantSubContextIterator.VcfTuple.html" title="class in picard.vcf">PairedVariantSubContextIterator.VcfTuple</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/RnaSeqMetrics.html" title="class in picard.analysis"><span class="typeNameLink">RnaSeqMetrics</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Metrics about the alignment of RNA-seq reads within a SAM file to genes, produced by the CollectRnaSeqMetrics
 program and usually stored in a file with the extension ".rna_metrics".</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RnaSeqMetrics.html#RnaSeqMetrics--">RnaSeqMetrics()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/RnaSeqMetrics.html" title="class in picard.analysis">RnaSeqMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/directed/RnaSeqMetricsCollector.html" title="class in picard.analysis.directed"><span class="typeNameLink">RnaSeqMetricsCollector</span></a> - Class in <a href="picard/analysis/directed/package-summary.html">picard.analysis.directed</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/RnaSeqMetricsCollector.html#RnaSeqMetricsCollector-java.util.Set-java.util.List-java.lang.Long-htsjdk.samtools.util.OverlapDetector-htsjdk.samtools.util.OverlapDetector-java.util.HashSet-int-picard.analysis.directed.RnaSeqMetricsCollector.StrandSpecificity-double-boolean-">RnaSeqMetricsCollector(Set&lt;MetricAccumulationLevel&gt;, List&lt;SAMReadGroupRecord&gt;, Long, OverlapDetector&lt;Gene&gt;, OverlapDetector&lt;Interval&gt;, HashSet&lt;Integer&gt;, int, RnaSeqMetricsCollector.StrandSpecificity, double, boolean)</a></span> - Constructor for class picard.analysis.directed.<a href="picard/analysis/directed/RnaSeqMetricsCollector.html" title="class in picard.analysis.directed">RnaSeqMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/directed/RnaSeqMetricsCollector.PerUnitRnaSeqMetricsCollector.html" title="class in picard.analysis.directed"><span class="typeNameLink">RnaSeqMetricsCollector.PerUnitRnaSeqMetricsCollector</span></a> - Class in <a href="picard/analysis/directed/package-summary.html">picard.analysis.directed</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/directed/RnaSeqMetricsCollector.StrandSpecificity.html" title="enum in picard.analysis.directed"><span class="typeNameLink">RnaSeqMetricsCollector.StrandSpecificity</span></a> - Enum in <a href="picard/analysis/directed/package-summary.html">picard.analysis.directed</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#round-double-int-">round(double, int)</a></span> - Static method in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>
<div class="block">Round off the value to the specified precision.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEnds.html#RR">RR</a></span> - Static variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEnds.html" title="class in picard.sam.markduplicates.util">ReadEnds</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/RrbsCpgDetailMetrics.html" title="class in picard.analysis"><span class="typeNameLink">RrbsCpgDetailMetrics</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Holds information about CpG sites encountered for RRBS processing QC</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RrbsCpgDetailMetrics.html#RrbsCpgDetailMetrics--">RrbsCpgDetailMetrics()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/RrbsCpgDetailMetrics.html" title="class in picard.analysis">RrbsCpgDetailMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/RrbsMetricsCollector.html" title="class in picard.analysis"><span class="typeNameLink">RrbsMetricsCollector</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RrbsMetricsCollector.html#RrbsMetricsCollector-java.util.Set-java.util.List-int-int-int-double-">RrbsMetricsCollector(Set&lt;MetricAccumulationLevel&gt;, List&lt;SAMReadGroupRecord&gt;, int, int, int, double)</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/RrbsMetricsCollector.html" title="class in picard.analysis">RrbsMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/RrbsSummaryMetrics.html" title="class in picard.analysis"><span class="typeNameLink">RrbsSummaryMetrics</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Holds summary statistics from RRBS processing QC</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RrbsSummaryMetrics.html#RrbsSummaryMetrics--">RrbsSummaryMetrics()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/RrbsSummaryMetrics.html" title="class in picard.analysis">RrbsSummaryMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRnaSeqMetrics.html#RRNA_FRAGMENT_PERCENTAGE">RRNA_FRAGMENT_PERCENTAGE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectRnaSeqMetrics.html" title="class in picard.analysis">CollectRnaSeqMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ExtractIlluminaBarcodes.PerTileBarcodeExtractor.html#run--">run()</a></span> - Method in class picard.illumina.<a href="picard/illumina/ExtractIlluminaBarcodes.PerTileBarcodeExtractor.html" title="class in picard.illumina">ExtractIlluminaBarcodes.PerTileBarcodeExtractor</a></dt>
<dd>
<div class="block">run method which extracts barcodes and accumulates metrics for an entire tile</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/Test.html#run--">run()</a></span> - Method in class picard.<a href="picard/Test.html" title="class in picard">Test</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fastq/BamToBfq.html#RUN_BARCODE">RUN_BARCODE</a></span> - Variable in class picard.fastq.<a href="picard/fastq/BamToBfq.html" title="class in picard.fastq">BamToBfq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToFastq.html#RUN_BARCODE">RUN_BARCODE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToFastq.html" title="class in picard.illumina">IlluminaBasecallsToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#RUN_BARCODE">RUN_BARCODE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FastqToSam.html#RUN_DATE">RUN_DATE</a></span> - Variable in class picard.sam.<a href="picard/sam/FastqToSam.html" title="class in picard.sam">FastqToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CollectIlluminaLaneMetrics.html#RUN_DIRECTORY">RUN_DIRECTORY</a></span> - Variable in class picard.illumina.<a href="picard/illumina/CollectIlluminaLaneMetrics.html" title="class in picard.illumina">CollectIlluminaLaneMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#RUN_START_DATE">RUN_START_DATE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="I:S">
<!--   -->
</a>
<h2 class="title">S</h2>
<dl>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.html#S_LOCS_FILE">S_LOCS_FILE</a></span> - Static variable in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.html" title="class in picard.illumina.parser.readers">AbstractIlluminaPositionFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/SamAlignmentMerger.html" title="class in picard.sam"><span class="typeNameLink">SamAlignmentMerger</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block">Class that takes in a set of alignment information in SAM format and merges it with the set
 of all reads for which alignment was attempted, stored in an unmapped SAM file.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamAlignmentMerger.html#SamAlignmentMerger-java.io.File-java.io.File-java.io.File-htsjdk.samtools.SAMProgramRecord-boolean-boolean-boolean-java.util.List-int-java.util.List-java.util.List-java.lang.Integer-java.lang.Integer-java.util.List-java.util.List-java.util.List-htsjdk.samtools.SAMFileHeader.SortOrder-picard.sam.PrimaryAlignmentSelectionStrategy-boolean-boolean-int-">SamAlignmentMerger(File, File, File, SAMProgramRecord, boolean, boolean, boolean, List&lt;File&gt;, int, List&lt;String&gt;, List&lt;String&gt;, Integer, Integer, List&lt;File&gt;, List&lt;File&gt;, List&lt;SamPairUtil.PairOrientation&gt;, SAMFileHeader.SortOrder, PrimaryAlignmentSelectionStrategy, boolean, boolean, int)</a></span> - Constructor for class picard.sam.<a href="picard/sam/SamAlignmentMerger.html" title="class in picard.sam">SamAlignmentMerger</a></dt>
<dd>
<div class="block">Constructor with a default value for unmappingReadStrategy</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamAlignmentMerger.html#SamAlignmentMerger-java.io.File-java.io.File-java.io.File-htsjdk.samtools.SAMProgramRecord-boolean-boolean-boolean-java.util.List-int-java.util.List-java.util.List-java.lang.Integer-java.lang.Integer-java.util.List-java.util.List-java.util.List-htsjdk.samtools.SAMFileHeader.SortOrder-picard.sam.PrimaryAlignmentSelectionStrategy-boolean-boolean-int-picard.sam.AbstractAlignmentMerger.UnmappingReadStrategy-java.util.List-">SamAlignmentMerger(File, File, File, SAMProgramRecord, boolean, boolean, boolean, List&lt;File&gt;, int, List&lt;String&gt;, List&lt;String&gt;, Integer, Integer, List&lt;File&gt;, List&lt;File&gt;, List&lt;SamPairUtil.PairOrientation&gt;, SAMFileHeader.SortOrder, PrimaryAlignmentSelectionStrategy, boolean, boolean, int, AbstractAlignmentMerger.UnmappingReadStrategy, List&lt;String&gt;)</a></span> - Constructor for class picard.sam.<a href="picard/sam/SamAlignmentMerger.html" title="class in picard.sam">SamAlignmentMerger</a></dt>
<dd>
<div class="block">Constructor</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CompareSAMs.html#samFiles">samFiles</a></span> - Variable in class picard.sam.<a href="picard/sam/CompareSAMs.html" title="class in picard.sam">CompareSAMs</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/SamFormatConverter.html" title="class in picard.sam"><span class="typeNameLink">SamFormatConverter</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block">Converts a BAM file to human-readable SAM output or vice versa</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamFormatConverter.html#SamFormatConverter--">SamFormatConverter()</a></span> - Constructor for class picard.sam.<a href="picard/sam/SamFormatConverter.html" title="class in picard.sam">SamFormatConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.SamHeaderAndIterator.html#SamHeaderAndIterator-htsjdk.samtools.SAMFileHeader-htsjdk.samtools.util.CloseableIterator-">SamHeaderAndIterator(SAMFileHeader, CloseableIterator&lt;SAMRecord&gt;)</a></span> - Constructor for class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.SamHeaderAndIterator.html" title="class in picard.sam.markduplicates.util">AbstractMarkDuplicatesCommandLineProgram.SamHeaderAndIterator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/cmdline/programgroups/SamOrBam.html" title="class in picard.cmdline.programgroups"><span class="typeNameLink">SamOrBam</span></a> - Class in <a href="picard/cmdline/programgroups/package-summary.html">picard.cmdline.programgroups</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/programgroups/SamOrBam.html#SamOrBam--">SamOrBam()</a></span> - Constructor for class picard.cmdline.programgroups.<a href="picard/cmdline/programgroups/SamOrBam.html" title="class in picard.cmdline.programgroups">SamOrBam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/FingerprintingDetailMetrics.html#SAMPLE">SAMPLE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/FingerprintingDetailMetrics.html" title="class in picard.analysis">FingerprintingDetailMetrics</a></dt>
<dd>
<div class="block">The name of the sample who's genotypes the sequence data was compared to.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/FingerprintingSummaryMetrics.html#SAMPLE">SAMPLE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/FingerprintingSummaryMetrics.html" title="class in picard.analysis">FingerprintingSummaryMetrics</a></dt>
<dd>
<div class="block">The sample whose known genotypes the sequence data was compared to.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/CollectIndependentReplicateMetrics.html#SAMPLE">SAMPLE</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/CollectIndependentReplicateMetrics.html" title="class in picard.analysis.replicates">CollectIndependentReplicateMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/metrics/MultilevelMetrics.html#SAMPLE">SAMPLE</a></span> - Variable in class picard.metrics.<a href="picard/metrics/MultilevelMetrics.html" title="class in picard.metrics">MultilevelMetrics</a></dt>
<dd>
<div class="block">The sample to which these metrics apply.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MakeSitesOnlyVcf.html#SAMPLE">SAMPLE</a></span> - Variable in class picard.vcf.<a href="picard/vcf/MakeSitesOnlyVcf.html" title="class in picard.vcf">MakeSitesOnlyVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html#SAMPLE_ALIAS">SAMPLE_ALIAS</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.BaitBiasDetailMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasSummaryMetrics.html#SAMPLE_ALIAS">SAMPLE_ALIAS</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasSummaryMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.BaitBiasSummaryMetrics</a></dt>
<dd>
<div class="block">The name of the sample being assayed.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterDetailMetrics.html#SAMPLE_ALIAS">SAMPLE_ALIAS</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterDetailMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.PreAdapterDetailMetrics</a></dt>
<dd>
<div class="block">The name of the sample being assayed.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterSummaryMetrics.html#SAMPLE_ALIAS">SAMPLE_ALIAS</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterSummaryMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.PreAdapterSummaryMetrics</a></dt>
<dd>
<div class="block">The name of the sample being assayed.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html#SAMPLE_ALIAS">SAMPLE_ALIAS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html" title="class in picard.analysis">CollectOxoGMetrics.CpcgMetrics</a></dt>
<dd>
<div class="block">The name of the sample being assayed.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#SAMPLE_ALIAS">SAMPLE_ALIAS</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertSam.html#SAMPLE_ALIAS">SAMPLE_ALIAS</a></span> - Variable in class picard.sam.<a href="picard/sam/RevertSam.html" title="class in picard.sam">RevertSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingDetailMetrics.html#SAMPLE_ALIAS">SAMPLE_ALIAS</a></span> - Variable in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingDetailMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingDetailMetrics</a></dt>
<dd>
<div class="block">The name of the sample being assayed</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/StandardOptionDefinitions.html#SAMPLE_ALIAS_SHORT_NAME">SAMPLE_ALIAS_SHORT_NAME</a></span> - Static variable in class picard.cmdline.<a href="picard/cmdline/StandardOptionDefinitions.html" title="class in picard.cmdline">StandardOptionDefinitions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FastqToSam.html#SAMPLE_NAME">SAMPLE_NAME</a></span> - Variable in class picard.sam.<a href="picard/sam/FastqToSam.html" title="class in picard.sam">FastqToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.html#SAMPLE_SIZE">SAMPLE_SIZE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/CollectTargetedMetrics.html#SAMPLE_SIZE">SAMPLE_SIZE</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/CollectTargetedMetrics.html" title="class in picard.analysis.directed">CollectTargetedMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadStructure.html#sampleBarcodes">sampleBarcodes</a></span> - Variable in class picard.illumina.parser.<a href="picard/illumina/parser/ReadStructure.html" title="class in picard.illumina.parser">ReadStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/TheoreticalSensitivity.RouletteWheel.html#sampleCumulativeSums-int-int-boolean-">sampleCumulativeSums(int, int, boolean)</a></span> - Method in class picard.analysis.<a href="picard/analysis/TheoreticalSensitivity.RouletteWheel.html" title="class in picard.analysis">TheoreticalSensitivity.RouletteWheel</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/metrics/SAMRecordAndReference.html" title="class in picard.metrics"><span class="typeNameLink">SAMRecordAndReference</span></a> - Class in <a href="picard/metrics/package-summary.html">picard.metrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/metrics/SAMRecordAndReference.html#SAMRecordAndReference-htsjdk.samtools.SAMRecord-htsjdk.samtools.reference.ReferenceSequence-">SAMRecordAndReference(SAMRecord, ReferenceSequence)</a></span> - Constructor for class picard.metrics.<a href="picard/metrics/SAMRecordAndReference.html" title="class in picard.metrics">SAMRecordAndReference</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/metrics/SAMRecordAndReferenceMultiLevelCollector.html" title="class in picard.metrics"><span class="typeNameLink">SAMRecordAndReferenceMultiLevelCollector</span></a>&lt;<a href="picard/metrics/SAMRecordAndReferenceMultiLevelCollector.html" title="type parameter in SAMRecordAndReferenceMultiLevelCollector">BEAN</a> extends htsjdk.samtools.metrics.MetricBase,<a href="picard/metrics/SAMRecordAndReferenceMultiLevelCollector.html" title="type parameter in SAMRecordAndReferenceMultiLevelCollector">HKEY</a> extends java.lang.Comparable&gt; - Class in <a href="picard/metrics/package-summary.html">picard.metrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/metrics/SAMRecordAndReferenceMultiLevelCollector.html#SAMRecordAndReferenceMultiLevelCollector--">SAMRecordAndReferenceMultiLevelCollector()</a></span> - Constructor for class picard.metrics.<a href="picard/metrics/SAMRecordAndReferenceMultiLevelCollector.html" title="class in picard.metrics">SAMRecordAndReferenceMultiLevelCollector</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/metrics/SAMRecordMultiLevelCollector.html" title="class in picard.metrics"><span class="typeNameLink">SAMRecordMultiLevelCollector</span></a>&lt;<a href="picard/metrics/SAMRecordMultiLevelCollector.html" title="type parameter in SAMRecordMultiLevelCollector">BEAN</a> extends htsjdk.samtools.metrics.MetricBase,<a href="picard/metrics/SAMRecordMultiLevelCollector.html" title="type parameter in SAMRecordMultiLevelCollector">HKEY</a> extends java.lang.Comparable&gt; - Class in <a href="picard/metrics/package-summary.html">picard.metrics</a></dt>
<dd>
<div class="block">Defines a MultilevelPerRecordCollector using the argument type of SAMRecord so that this doesn't have to be redefined for each subclass of MultilevelPerRecordCollector</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/metrics/SAMRecordMultiLevelCollector.html#SAMRecordMultiLevelCollector--">SAMRecordMultiLevelCollector()</a></span> - Constructor for class picard.metrics.<a href="picard/metrics/SAMRecordMultiLevelCollector.html" title="class in picard.metrics">SAMRecordMultiLevelCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMateCigar.html#samRecordWithOrdinal">samRecordWithOrdinal</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMateCigar.html" title="class in picard.sam.markduplicates.util">ReadEndsForMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/markduplicates/util/SamRecordWithOrdinalAndSetDuplicateReadFlag.html" title="class in picard.sam.markduplicates.util"><span class="typeNameLink">SamRecordWithOrdinalAndSetDuplicateReadFlag</span></a> - Class in <a href="picard/sam/markduplicates/util/package-summary.html">picard.sam.markduplicates.util</a></dt>
<dd>
<div class="block">This class sets the duplicate read flag as the result state when examining sets of records.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/SamRecordWithOrdinalAndSetDuplicateReadFlag.html#SamRecordWithOrdinalAndSetDuplicateReadFlag--">SamRecordWithOrdinalAndSetDuplicateReadFlag()</a></span> - Constructor for class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/SamRecordWithOrdinalAndSetDuplicateReadFlag.html" title="class in picard.sam.markduplicates.util">SamRecordWithOrdinalAndSetDuplicateReadFlag</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/SamRecordWithOrdinalAndSetDuplicateReadFlag.html#SamRecordWithOrdinalAndSetDuplicateReadFlag-htsjdk.samtools.SAMRecord-long-">SamRecordWithOrdinalAndSetDuplicateReadFlag(SAMRecord, long)</a></span> - Constructor for class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/SamRecordWithOrdinalAndSetDuplicateReadFlag.html" title="class in picard.sam.markduplicates.util">SamRecordWithOrdinalAndSetDuplicateReadFlag</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/SamToFastq.html" title="class in picard.sam"><span class="typeNameLink">SamToFastq</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block"><p/>
 Extracts read sequences and qualities from the input SAM/BAM file and writes them into
 the output file in Sanger fastq format.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamToFastq.html#SamToFastq--">SamToFastq()</a></span> - Constructor for class picard.sam.<a href="picard/sam/SamToFastq.html" title="class in picard.sam">SamToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertSam.html#SANITIZE">SANITIZE</a></span> - Variable in class picard.sam.<a href="picard/sam/RevertSam.html" title="class in picard.sam">RevertSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilities.html#scaledEvidenceProbabilityUsingGenotypeFrequencies-double:A-">scaledEvidenceProbabilityUsingGenotypeFrequencies(double[])</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilities.html" title="class in picard.fingerprint">HaplotypeProbabilities</a></dt>
<dd>
<div class="block">This function returns the scaled probability of the evidence collected
 given a vector of priors on the haplotype using the internal likelihood, which may be
 scaled by an unknown factor.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectGcBiasMetrics.html#SCAN_WINDOW_SIZE">SCAN_WINDOW_SIZE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectGcBiasMetrics.html" title="class in picard.analysis">CollectGcBiasMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/ClassFinder.html#scanDir-java.io.File-java.lang.String-">scanDir(File, String)</a></span> - Method in class picard.cmdline.<a href="picard/cmdline/ClassFinder.html" title="class in picard.cmdline">ClassFinder</a></dt>
<dd>
<div class="block">Scans a directory on the filesystem for classes.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/ClassFinder.html#scanJar-java.io.File-java.lang.String-">scanJar(File, String)</a></span> - Method in class picard.cmdline.<a href="picard/cmdline/ClassFinder.html" title="class in picard.cmdline">ClassFinder</a></dt>
<dd>
<div class="block">Scans the entries in a ZIP/JAR file for classes under the parent package.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/IntervalListScatterer.html#scatter-htsjdk.samtools.util.IntervalList-int-">scatter(IntervalList, int)</a></span> - Method in class picard.util.<a href="picard/util/IntervalListScatterer.html" title="class in picard.util">IntervalListScatterer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IntervalListScatterer.html#scatter-htsjdk.samtools.util.IntervalList-int-boolean-">scatter(IntervalList, int, boolean)</a></span> - Method in class picard.util.<a href="picard/util/IntervalListScatterer.html" title="class in picard.util">IntervalListScatterer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IntervalListTools.html#SCATTER_COUNT">SCATTER_COUNT</a></span> - Variable in class picard.util.<a href="picard/util/IntervalListTools.html" title="class in picard.util">IntervalListTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/ScatterIntervalsByNs.ScatterIntervalsByNReferenceArgumentCollection.html#ScatterIntervalsByNReferenceArgumentCollection--">ScatterIntervalsByNReferenceArgumentCollection()</a></span> - Constructor for class picard.util.<a href="picard/util/ScatterIntervalsByNs.ScatterIntervalsByNReferenceArgumentCollection.html" title="class in picard.util">ScatterIntervalsByNs.ScatterIntervalsByNReferenceArgumentCollection</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/ScatterIntervalsByNs.html" title="class in picard.util"><span class="typeNameLink">ScatterIntervalsByNs</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">A CLP for breaking up a reference into intervals of Ns and ACGTs bases.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/ScatterIntervalsByNs.html#ScatterIntervalsByNs--">ScatterIntervalsByNs()</a></span> - Constructor for class picard.util.<a href="picard/util/ScatterIntervalsByNs.html" title="class in picard.util">ScatterIntervalsByNs</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/ScatterIntervalsByNs.ScatterIntervalsByNReferenceArgumentCollection.html" title="class in picard.util"><span class="typeNameLink">ScatterIntervalsByNs.ScatterIntervalsByNReferenceArgumentCollection</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceScheme.html#scheme">scheme</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceScheme.html" title="class in picard.vcf">GenotypeConcordanceScheme</a></dt>
<dd>
<div class="block">The underlying scheme</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicates.html#score">score</a></span> - Variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicates.html" title="class in picard.sam.markduplicates.util">ReadEndsForMarkDuplicates</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IntervalListToBed.html#SCORE">SCORE</a></span> - Variable in class picard.util.<a href="picard/util/IntervalListToBed.html" title="class in picard.util">IntervalListToBed</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallingMetrics.html#SD_CLUSTERS_PER_TILE">SD_CLUSTERS_PER_TILE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallingMetrics.html" title="class in picard.illumina">IlluminaBasecallingMetrics</a></dt>
<dd>
<div class="block">The standard deviation of clusters per tile.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#SD_COVERAGE">SD_COVERAGE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">The standard deviation of coverage of the genome after all filters are applied.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallingMetrics.html#SD_PCT_PF_CLUSTERS_PER_TILE">SD_PCT_PF_CLUSTERS_PER_TILE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallingMetrics.html" title="class in picard.illumina">IlluminaBasecallingMetrics</a></dt>
<dd>
<div class="block">The standard deviation in percentage of pf clusters per tile.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallingMetrics.html#SD_PF_CLUSTERS_PER_TILE">SD_PF_CLUSTERS_PER_TILE</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallingMetrics.html" title="class in picard.illumina">IlluminaBasecallingMetrics</a></dt>
<dd>
<div class="block">The standard deviation in number of pf clusters per tile.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamToFastq.html#SECOND_END_FASTQ">SECOND_END_FASTQ</a></span> - Variable in class picard.sam.<a href="picard/sam/SamToFastq.html" title="class in picard.sam">SamToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckFingerprints.html#SECOND_INPUT">SECOND_INPUT</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CrosscheckFingerprints.html" title="class in picard.fingerprint">CrosscheckFingerprints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IntervalListTools.html#SECOND_INPUT">SECOND_INPUT</a></span> - Variable in class picard.util.<a href="picard/util/IntervalListTools.html" title="class in picard.util">IntervalListTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/DuplicationMetrics.html#SECONDARY_OR_SUPPLEMENTARY_RDS">SECONDARY_OR_SUPPLEMENTARY_RDS</a></span> - Variable in class picard.sam.<a href="picard/sam/DuplicationMetrics.html" title="class in picard.sam">DuplicationMetrics</a></dt>
<dd>
<div class="block">The number of reads that were either secondary or supplementary</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/BclReader.html#seek-java.util.List-picard.illumina.parser.TileIndex-int-">seek(List&lt;File&gt;, TileIndex, int)</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/BclReader.html" title="class in picard.illumina.parser.readers">BclReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaDataProvider.html#seekToTile-int-">seekToTile(int)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/IlluminaDataProvider.html" title="class in picard.illumina.parser">IlluminaDataProvider</a></dt>
<dd>
<div class="block">Jump so that the next record returned will be from the specified tile.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/MultiTileParser.html#seekToTile-int-">seekToTile(int)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/MultiTileParser.html" title="class in picard.illumina.parser">MultiTileParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/NewIlluminaDataProvider.html#seekToTile-int-">seekToTile(int)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/NewIlluminaDataProvider.html" title="class in picard.illumina.parser">NewIlluminaDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PerTileParser.html#seekToTile-int-">seekToTile(int)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/PerTileParser.html" title="class in picard.illumina.parser">PerTileParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#seq-double-double-double-">seq(double, double, double)</a></span> - Static method in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>
<div class="block">Mimic's R's seq() function to produce a sequence of equally spaced numbers.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/BedToIntervalList.html#SEQUENCE_DICTIONARY">SEQUENCE_DICTIONARY</a></span> - Variable in class picard.util.<a href="picard/util/BedToIntervalList.html" title="class in picard.util">BedToIntervalList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/LiftOverIntervalList.html#SEQUENCE_DICTIONARY">SEQUENCE_DICTIONARY</a></span> - Variable in class picard.util.<a href="picard/util/LiftOverIntervalList.html" title="class in picard.util">LiftOverIntervalList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.html#SEQUENCE_DICTIONARY">SEQUENCE_DICTIONARY</a></span> - Variable in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MergeVcfs.html#SEQUENCE_DICTIONARY">SEQUENCE_DICTIONARY</a></span> - Variable in class picard.vcf.<a href="picard/vcf/MergeVcfs.html" title="class in picard.vcf">MergeVcfs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/SortVcf.html#SEQUENCE_DICTIONARY">SEQUENCE_DICTIONARY</a></span> - Variable in class picard.vcf.<a href="picard/vcf/SortVcf.html" title="class in picard.vcf">SortVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/SplitVcfs.html#SEQUENCE_DICTIONARY">SEQUENCE_DICTIONARY</a></span> - Variable in class picard.vcf.<a href="picard/vcf/SplitVcfs.html" title="class in picard.vcf">SplitVcfs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/UpdateVcfSequenceDictionary.html#SEQUENCE_DICTIONARY">SEQUENCE_DICTIONARY</a></span> - Variable in class picard.vcf.<a href="picard/vcf/UpdateVcfSequenceDictionary.html" title="class in picard.vcf">UpdateVcfSequenceDictionary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/StandardOptionDefinitions.html#SEQUENCE_DICTIONARY_SHORT_NAME">SEQUENCE_DICTIONARY_SHORT_NAME</a></span> - Static variable in class picard.cmdline.<a href="picard/cmdline/StandardOptionDefinitions.html" title="class in picard.cmdline">StandardOptionDefinitions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RrbsCpgDetailMetrics.html#SEQUENCE_NAME">SEQUENCE_NAME</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RrbsCpgDetailMetrics.html" title="class in picard.analysis">RrbsCpgDetailMetrics</a></dt>
<dd>
<div class="block">Sequence the CpG is seen in</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRrbsMetrics.html#SEQUENCE_NAMES">SEQUENCE_NAMES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectRrbsMetrics.html" title="class in picard.analysis">CollectRrbsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#SEQUENCING_CENTER">SEQUENCING_CENTER</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FastqToSam.html#SEQUENCING_CENTER">SEQUENCING_CENTER</a></span> - Variable in class picard.sam.<a href="picard/sam/FastqToSam.html" title="class in picard.sam">FastqToSam</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/artifacts/SequencingArtifactMetrics.html" title="class in picard.analysis.artifacts"><span class="typeNameLink">SequencingArtifactMetrics</span></a> - Class in <a href="picard/analysis/artifacts/package-summary.html">picard.analysis.artifacts</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.html#SequencingArtifactMetrics--">SequencingArtifactMetrics()</a></span> - Constructor for class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasDetailMetrics.html" title="class in picard.analysis.artifacts"><span class="typeNameLink">SequencingArtifactMetrics.BaitBiasDetailMetrics</span></a> - Class in <a href="picard/analysis/artifacts/package-summary.html">picard.analysis.artifacts</a></dt>
<dd>
<div class="block">Bait bias artifacts broken down by context.</div>
</dd>
<dt><a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasSummaryMetrics.html" title="class in picard.analysis.artifacts"><span class="typeNameLink">SequencingArtifactMetrics.BaitBiasSummaryMetrics</span></a> - Class in <a href="picard/analysis/artifacts/package-summary.html">picard.analysis.artifacts</a></dt>
<dd>
<div class="block">Summary analysis of a single bait bias artifact, also known as a reference bias artifact.</div>
</dd>
<dt><a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterDetailMetrics.html" title="class in picard.analysis.artifacts"><span class="typeNameLink">SequencingArtifactMetrics.PreAdapterDetailMetrics</span></a> - Class in <a href="picard/analysis/artifacts/package-summary.html">picard.analysis.artifacts</a></dt>
<dd>
<div class="block">Pre-adapter artifacts broken down by context.</div>
</dd>
<dt><a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterSummaryMetrics.html" title="class in picard.analysis.artifacts"><span class="typeNameLink">SequencingArtifactMetrics.PreAdapterSummaryMetrics</span></a> - Class in <a href="picard/analysis/artifacts/package-summary.html">picard.analysis.artifacts</a></dt>
<dd>
<div class="block">Summary analysis of a single pre-adapter artifact.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SetNmMdAndUqTags.html#SET_ONLY_UQ">SET_ONLY_UQ</a></span> - Variable in class picard.sam.<a href="picard/sam/SetNmMdAndUqTags.html" title="class in picard.sam">SetNmMdAndUqTags</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.html#setAddPGTagToReads-boolean-">setAddPGTagToReads(boolean)</a></span> - Method in class picard.sam.<a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</a></dt>
<dd>
<div class="block">Set addPGTagToReads.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintChecker.html#setAllowDuplicateReads-boolean-">setAllowDuplicateReads(boolean)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/FingerprintChecker.html" title="class in picard.fingerprint">FingerprintChecker</a></dt>
<dd>
<div class="block">Sets whether duplicate reads should be allowed when calling genotypes from SAM files.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaDataProviderFactory.html#setApplyEamssFiltering-boolean-">setApplyEamssFiltering(boolean)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/IlluminaDataProviderFactory.html" title="class in picard.illumina.parser">IlluminaDataProviderFactory</a></dt>
<dd>
<div class="block">Sets whether or not EAMSS filtering will be applied if parsing BCL files for bases and quality scores.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.html#setAttributesToReverse-java.util.Set-">setAttributesToReverse(Set&lt;String&gt;)</a></span> - Method in class picard.sam.<a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</a></dt>
<dd>
<div class="block">Sets the set of attributes to be reversed on reads marked as negative strand.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.html#setAttributesToReverseComplement-java.util.Set-">setAttributesToReverseComplement(Set&lt;String&gt;)</a></span> - Method in class picard.sam.<a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</a></dt>
<dd>
<div class="block">Sets the set of attributes to be reverse complemented on reads marked as negative strand.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetMetricsCollector.PerUnitTargetMetricCollector.html#setBaitSetName-java.lang.String-">setBaitSetName(String)</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/TargetMetricsCollector.PerUnitTargetMetricCollector.html" title="class in picard.analysis.directed">TargetMetricsCollector.PerUnitTargetMetricCollector</a></dt>
<dd>
<div class="block">Sets the name of the bait set explicitly instead of inferring it from the bait file.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadData.html#setBases-byte:A-">setBases(byte[])</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ReadData.html" title="class in picard.illumina.parser">ReadData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/OpticalDuplicateFinder.html#setBigDuplicateSetSize-int-">setBigDuplicateSetSize(int)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/OpticalDuplicateFinder.html" title="class in picard.sam.markduplicates.util">OpticalDuplicateFinder</a></dt>
<dd>
<div class="block">Sets the size of a set that is big enough to log progress about.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.html#setClipOverlappingReads-boolean-">setClipOverlappingReads(boolean)</a></span> - Method in class picard.sam.<a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AbstractWgsMetricsCollector.html#setCounter-long-">setCounter(long)</a></span> - Method in class picard.analysis.<a href="picard/analysis/AbstractWgsMetricsCollector.html" title="class in picard.analysis">AbstractWgsMetricsCollector</a></dt>
<dd>
<div class="block">Sets the counter to the current number of processed loci.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#setDefaultHeaders-java.util.List-">setDefaultHeaders(List&lt;Header&gt;)</a></span> - Method in class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintChecker.html#setGenotypingErrorRate-double-">setGenotypingErrorRate(double)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/FingerprintChecker.html" title="class in picard.fingerprint">FingerprintChecker</a></dt>
<dd>
<div class="block">Sets the assumed genotyping error rate used when accurate error rates are not available.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.html#setIncludeSecondaryAlignments-boolean-">setIncludeSecondaryAlignments(boolean)</a></span> - Method in class picard.sam.<a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesCodec.html#setInputStream-java.io.InputStream-">setInputStream(InputStream)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesCodec.html" title="class in picard.sam.markduplicates.util">ReadEndsForMarkDuplicatesCodec</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/RepresentativeReadIndexerCodec.html#setInputStream-java.io.InputStream-">setInputStream(InputStream)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/RepresentativeReadIndexerCodec.html" title="class in picard.sam.markduplicates.util">RepresentativeReadIndexerCodec</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.html#setKeepAlignerProperPairFlags-boolean-">setKeepAlignerProperPairFlags(boolean)</a></span> - Method in class picard.sam.<a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</a></dt>
<dd>
<div class="block">If true, keep the aligner's idea of proper pairs rather than letting alignment merger decide.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ClusterData.html#setLane-int-">setLane(int)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ClusterData.html" title="class in picard.illumina.parser">ClusterData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/PhysicalLocationForMateCigar.html#setLibraryId-short-">setLibraryId(short)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/PhysicalLocationForMateCigar.html" title="class in picard.sam.markduplicates.util">PhysicalLocationForMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEnds.html#setLibraryId-short-">setLibraryId(short)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEnds.html" title="class in picard.sam.markduplicates.util">ReadEnds</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/PhysicalLocation.html#setLibraryId-short-">setLibraryId(short)</a></span> - Method in interface picard.sam.util.<a href="picard/sam/util/PhysicalLocation.html" title="interface in picard.sam.util">PhysicalLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/PhysicalLocationInt.html#setLibraryId-short-">setLibraryId(short)</a></span> - Method in class picard.sam.util.<a href="picard/sam/util/PhysicalLocationInt.html" title="class in picard.sam.util">PhysicalLocationInt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMateCigar.html#setLocationSet-picard.sam.markduplicates.util.PhysicalLocationForMateCigarSet-">setLocationSet(PhysicalLocationForMateCigarSet)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMateCigar.html" title="class in picard.sam.markduplicates.util">ReadEndsForMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ClusterData.html#setMatchedBarcode-java.lang.String-">setMatchedBarcode(String)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ClusterData.html" title="class in picard.illumina.parser">ClusterData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/OpticalDuplicateFinder.html#setMaxDuplicateSetSize-long-">setMaxDuplicateSetSize(long)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/OpticalDuplicateFinder.html" title="class in picard.sam.markduplicates.util">OpticalDuplicateFinder</a></dt>
<dd>
<div class="block">Sets the size of a set that is too big to process.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintChecker.html#setmaximalPLDifference-int-">setmaximalPLDifference(int)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/FingerprintChecker.html" title="class in picard.fingerprint">FingerprintChecker</a></dt>
<dd>
<div class="block">Sets the maximal difference in PL scores considered when reading PLs from a VCF.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/AdapterMarker.html#setMaxPairErrorRate-double-">setMaxPairErrorRate(double)</a></span> - Method in class picard.util.<a href="picard/util/AdapterMarker.html" title="class in picard.util">AdapterMarker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.html#setMaxRecordsInRam-int-">setMaxRecordsInRam(int)</a></span> - Method in class picard.sam.<a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</a></dt>
<dd>
<div class="block">Allows the caller to override the maximum records in RAM.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/AdapterMarker.html#setMaxSingleEndErrorRate-double-">setMaxSingleEndErrorRate(double)</a></span> - Method in class picard.util.<a href="picard/util/AdapterMarker.html" title="class in picard.util">AdapterMarker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintChecker.html#setMinimumBaseQuality-int-">setMinimumBaseQuality(int)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/FingerprintChecker.html" title="class in picard.fingerprint">FingerprintChecker</a></dt>
<dd>
<div class="block">Sets the minimum base quality for bases used when computing a fingerprint from sequence data.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintChecker.html#setMinimumMappingQuality-int-">setMinimumMappingQuality(int)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/FingerprintChecker.html" title="class in picard.fingerprint">FingerprintChecker</a></dt>
<dd>
<div class="block">Sets the minimum mapping quality for reads used when computing fingerprints from sequence data.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/AdapterMarker.html#setMinPairMatchBases-int-">setMinPairMatchBases(int)</a></span> - Method in class picard.util.<a href="picard/util/AdapterMarker.html" title="class in picard.util">AdapterMarker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AdapterMarker.html#setMinSingleEndMatchBases-int-">setMinSingleEndMatchBases(int)</a></span> - Method in class picard.util.<a href="picard/util/AdapterMarker.html" title="class in picard.util">AdapterMarker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetMetricsCollector.html#setNearProbeDistance-int-">setNearProbeDistance(int)</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/TargetMetricsCollector.html" title="class in picard.analysis.directed">TargetMetricsCollector</a></dt>
<dd>
<div class="block">Sets the distance that is allowed between a read and the nearest probe for it to be considered "near probe" and "selected.</div>
</dd>
<dt><a href="picard/sam/SetNmAndUqTags.html" title="class in picard.sam"><span class="typeNameLink">SetNmAndUqTags</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block"><span class="deprecatedLabel">Deprecated.</span></div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SetNmAndUqTags.html#SetNmAndUqTags--">SetNmAndUqTags()</a></span> - Constructor for class picard.sam.<a href="picard/sam/SetNmAndUqTags.html" title="class in picard.sam">SetNmAndUqTags</a></dt>
<dd>
<div class="block"><span class="deprecatedLabel">Deprecated.</span></div>
&nbsp;</dd>
<dt><a href="picard/sam/SetNmMdAndUqTags.html" title="class in picard.sam"><span class="typeNameLink">SetNmMdAndUqTags</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SetNmMdAndUqTags.html#SetNmMdAndUqTags--">SetNmMdAndUqTags()</a></span> - Constructor for class picard.sam.<a href="picard/sam/SetNmMdAndUqTags.html" title="class in picard.sam">SetNmMdAndUqTags</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadData.html#setNoise-picard.illumina.parser.FourChannelIntensityData-">setNoise(FourChannelIntensityData)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ReadData.html" title="class in picard.illumina.parser">ReadData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetMetricsCollector.html#setNoSideEffects-boolean-">setNoSideEffects(boolean)</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/TargetMetricsCollector.html" title="class in picard.analysis.directed">TargetMetricsCollector</a></dt>
<dd>
<div class="block">In the case of ignoring bases in overlapping reads from the same template,
 we choose to internally modify the SAM record's CIGAR to clip overlapping bases.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/AdapterMarker.html#setNumAdaptersToKeep-int-">setNumAdaptersToKeep(int)</a></span> - Method in class picard.util.<a href="picard/util/AdapterMarker.html" title="class in picard.util">AdapterMarker</a></dt>
<dd>
<div class="block">After seeing the thresholdForSelectingAdapters number of adapters, keep up to this many of the original adapters.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ClusterData.html#setOrCheckLane-int-">setOrCheckLane(int)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ClusterData.html" title="class in picard.illumina.parser">ClusterData</a></dt>
<dd>
<div class="block">Either set this value if not already set, or if already set, throw an exception if new value != current value.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ClusterData.html#setOrCheckPf-boolean-">setOrCheckPf(boolean)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ClusterData.html" title="class in picard.illumina.parser">ClusterData</a></dt>
<dd>
<div class="block">Either set this value if not already set, or if already set, throw an exception if new value != current value.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ClusterData.html#setOrCheckTile-int-">setOrCheckTile(int)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ClusterData.html" title="class in picard.illumina.parser">ClusterData</a></dt>
<dd>
<div class="block">Either set this value if not already set, or if already set, throw an exception if new value != current value.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ClusterData.html#setOrCheckX-int-">setOrCheckX(int)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ClusterData.html" title="class in picard.illumina.parser">ClusterData</a></dt>
<dd>
<div class="block">Either set this value if not already set, or if already set, throw an exception if new value != current value.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ClusterData.html#setOrCheckY-int-">setOrCheckY(int)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ClusterData.html" title="class in picard.illumina.parser">ClusterData</a></dt>
<dd>
<div class="block">Either set this value if not already set, or if already set, throw an exception if new value != current value.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesCodec.html#setOutputStream-java.io.OutputStream-">setOutputStream(OutputStream)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesCodec.html" title="class in picard.sam.markduplicates.util">ReadEndsForMarkDuplicatesCodec</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/RepresentativeReadIndexerCodec.html#setOutputStream-java.io.OutputStream-">setOutputStream(OutputStream)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/RepresentativeReadIndexerCodec.html" title="class in picard.sam.markduplicates.util">RepresentativeReadIndexerCodec</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetMetricsCollector.PerUnitTargetMetricCollector.html#setPerBaseOutput-java.io.File-">setPerBaseOutput(File)</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/TargetMetricsCollector.PerUnitTargetMetricCollector.html" title="class in picard.analysis.directed">TargetMetricsCollector.PerUnitTargetMetricCollector</a></dt>
<dd>
<div class="block">Sets the (optional) File to write per-base coverage information to.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetMetricsCollector.PerUnitTargetMetricCollector.html#setPerTargetOutput-java.io.File-">setPerTargetOutput(File)</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/TargetMetricsCollector.PerUnitTargetMetricCollector.html" title="class in picard.analysis.directed">TargetMetricsCollector.PerUnitTargetMetricCollector</a></dt>
<dd>
<div class="block">Sets the (optional) File to write per-target coverage information to.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ClusterData.html#setPf-boolean-">setPf(boolean)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ClusterData.html" title="class in picard.illumina.parser">ClusterData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintChecker.html#setpLossofHet-double-">setpLossofHet(double)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/FingerprintChecker.html" title="class in picard.fingerprint">FingerprintChecker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/CbclData.html#setPositionInfo-picard.illumina.parser.readers.AbstractIlluminaPositionFileReader.PositionInfo-">setPositionInfo(AbstractIlluminaPositionFileReader.PositionInfo)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/CbclData.html" title="class in picard.illumina.parser">CbclData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.html#setProgramRecord-htsjdk.samtools.SAMProgramRecord-">setProgramRecord(SAMProgramRecord)</a></span> - Method in class picard.sam.<a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectMultipleMetrics.html#setProgramsToRun-java.util.Collection-">setProgramsToRun(Collection&lt;CollectMultipleMetrics.ProgramInterface&gt;)</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectMultipleMetrics.html" title="class in picard.analysis">CollectMultipleMetrics</a></dt>
<dd>
<div class="block">Use this method when invoking CollectMultipleMetrics programmatically to run programs other than the ones
 available via enum.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadData.html#setQualities-byte:A-">setQualities(byte[])</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ReadData.html" title="class in picard.illumina.parser">ReadData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadData.html#setRawIntensities-picard.illumina.parser.FourChannelIntensityData-">setRawIntensities(FourChannelIntensityData)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ReadData.html" title="class in picard.illumina.parser">ReadData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/PhysicalLocationForMateCigar.html#setReadGroup-short-">setReadGroup(short)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/PhysicalLocationForMateCigar.html" title="class in picard.sam.markduplicates.util">PhysicalLocationForMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEnds.html#setReadGroup-short-">setReadGroup(short)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEnds.html" title="class in picard.sam.markduplicates.util">ReadEnds</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/PhysicalLocation.html#setReadGroup-short-">setReadGroup(short)</a></span> - Method in interface picard.sam.util.<a href="picard/sam/util/PhysicalLocation.html" title="interface in picard.sam.util">PhysicalLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/PhysicalLocationInt.html#setReadGroup-short-">setReadGroup(short)</a></span> - Method in class picard.sam.util.<a href="picard/sam/util/PhysicalLocationInt.html" title="class in picard.sam.util">PhysicalLocationInt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadData.html#setReadType-picard.illumina.parser.ReadType-">setReadType(ReadType)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ReadData.html" title="class in picard.illumina.parser">ReadData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/SinglePassSamProgram.html#setReferenceSequence-java.io.File-">setReferenceSequence(File)</a></span> - Method in class picard.analysis.<a href="picard/analysis/SinglePassSamProgram.html" title="class in picard.analysis">SinglePassSamProgram</a></dt>
<dd>
<div class="block">Set the reference File.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/SamRecordWithOrdinalAndSetDuplicateReadFlag.html#setResultState-boolean-">setResultState(boolean)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/SamRecordWithOrdinalAndSetDuplicateReadFlag.html" title="class in picard.sam.markduplicates.util">SamRecordWithOrdinalAndSetDuplicateReadFlag</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/RepresentativeReadIndexer.html#setSize">setSize</a></span> - Variable in class picard.sam.util.<a href="picard/sam/util/RepresentativeReadIndexer.html" title="class in picard.sam.util">RepresentativeReadIndexer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AbstractInputParser.html#setSkipBlankLines-boolean-">setSkipBlankLines(boolean)</a></span> - Method in class picard.util.<a href="picard/util/AbstractInputParser.html" title="class in picard.util">AbstractInputParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AdapterMarker.html#setThresholdForSelectingAdaptersToKeep-int-">setThresholdForSelectingAdaptersToKeep(int)</a></span> - Method in class picard.util.<a href="picard/util/AdapterMarker.html" title="class in picard.util">AdapterMarker</a></dt>
<dd>
<div class="block">When this number of adapters have been matched, discard the least-frequently matching ones.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ClusterData.html#setTile-int-">setTile(int)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ClusterData.html" title="class in picard.illumina.parser">ClusterData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/PhysicalLocation.html#setTile-short-">setTile(short)</a></span> - Method in interface picard.sam.util.<a href="picard/sam/util/PhysicalLocation.html" title="interface in picard.sam.util">PhysicalLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/PhysicalLocationInt.html#setTile-short-">setTile(short)</a></span> - Method in class picard.sam.util.<a href="picard/sam/util/PhysicalLocationInt.html" title="class in picard.sam.util">PhysicalLocationInt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/MarkQueue.html#setToMarkQueueMinimumDistance-int-">setToMarkQueueMinimumDistance(int)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/MarkQueue.html" title="class in picard.sam.markduplicates.util">MarkQueue</a></dt>
<dd>
<div class="block">Sets the minimum genomic distance such that we can be assured that all duplicates have been considered.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/AbstractInputParser.html#setTreatGroupedDelimitersAsOne-boolean-">setTreatGroupedDelimitersAsOne(boolean)</a></span> - Method in class picard.util.<a href="picard/util/AbstractInputParser.html" title="class in picard.util">AbstractInputParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html#setup-htsjdk.samtools.SAMFileHeader-java.io.File-">setup(SAMFileHeader, File)</a></span> - Method in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html" title="class in picard.analysis.artifacts">CollectSequencingArtifactMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectAlignmentSummaryMetrics.html#setup-htsjdk.samtools.SAMFileHeader-java.io.File-">setup(SAMFileHeader, File)</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectAlignmentSummaryMetrics.html" title="class in picard.analysis">CollectAlignmentSummaryMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectBaseDistributionByCycle.html#setup-htsjdk.samtools.SAMFileHeader-java.io.File-">setup(SAMFileHeader, File)</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectBaseDistributionByCycle.html" title="class in picard.analysis">CollectBaseDistributionByCycle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectGcBiasMetrics.html#setup-htsjdk.samtools.SAMFileHeader-java.io.File-">setup(SAMFileHeader, File)</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectGcBiasMetrics.html" title="class in picard.analysis">CollectGcBiasMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectInsertSizeMetrics.html#setup-htsjdk.samtools.SAMFileHeader-java.io.File-">setup(SAMFileHeader, File)</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectInsertSizeMetrics.html" title="class in picard.analysis">CollectInsertSizeMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectQualityYieldMetrics.html#setup-htsjdk.samtools.SAMFileHeader-java.io.File-">setup(SAMFileHeader, File)</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectQualityYieldMetrics.html" title="class in picard.analysis">CollectQualityYieldMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRnaSeqMetrics.html#setup-htsjdk.samtools.SAMFileHeader-java.io.File-">setup(SAMFileHeader, File)</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectRnaSeqMetrics.html" title="class in picard.analysis">CollectRnaSeqMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/MeanQualityByCycle.html#setup-htsjdk.samtools.SAMFileHeader-java.io.File-">setup(SAMFileHeader, File)</a></span> - Method in class picard.analysis.<a href="picard/analysis/MeanQualityByCycle.html" title="class in picard.analysis">MeanQualityByCycle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/QualityScoreDistribution.html#setup-htsjdk.samtools.SAMFileHeader-java.io.File-">setup(SAMFileHeader, File)</a></span> - Method in class picard.analysis.<a href="picard/analysis/QualityScoreDistribution.html" title="class in picard.analysis">QualityScoreDistribution</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/SinglePassSamProgram.html#setup-htsjdk.samtools.SAMFileHeader-java.io.File-">setup(SAMFileHeader, File)</a></span> - Method in class picard.analysis.<a href="picard/analysis/SinglePassSamProgram.html" title="class in picard.analysis">SinglePassSamProgram</a></dt>
<dd>
<div class="block">Should be implemented by subclasses to do one-time initialization work.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/metrics/MultiLevelCollector.html#setup-java.util.Set-java.util.List-">setup(Set&lt;MetricAccumulationLevel&gt;, List&lt;SAMReadGroupRecord&gt;)</a></span> - Method in class picard.metrics.<a href="picard/metrics/MultiLevelCollector.html" title="class in picard.metrics">MultiLevelCollector</a></dt>
<dd>
<div class="block">Use an init method so that overloaded methods in subclasses can pass use parameters that are initialized in their constructor</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CallingMetricAccumulator.html#setup-htsjdk.variant.vcf.VCFHeader-">setup(VCFHeader)</a></span> - Method in class picard.vcf.<a href="picard/vcf/CallingMetricAccumulator.html" title="class in picard.vcf">CallingMetricAccumulator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GvcfMetricAccumulator.html#setup-htsjdk.variant.vcf.VCFHeader-">setup(VCFHeader)</a></span> - Method in class picard.vcf.<a href="picard/vcf/GvcfMetricAccumulator.html" title="class in picard.vcf">GvcfMetricAccumulator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/AbstractOpticalDuplicateFinderCommandLineProgram.html#setupOpticalDuplicateFinder--">setupOpticalDuplicateFinder()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/AbstractOpticalDuplicateFinderCommandLineProgram.html" title="class in picard.sam.markduplicates.util">AbstractOpticalDuplicateFinderCommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/FingerprintChecker.html#setValidationStringency-htsjdk.samtools.ValidationStringency-">setValidationStringency(ValidationStringency)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/FingerprintChecker.html" title="class in picard.fingerprint">FingerprintChecker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.html#setValuesFromAlignment-htsjdk.samtools.SAMRecord-htsjdk.samtools.SAMRecord-boolean-">setValuesFromAlignment(SAMRecord, SAMRecord, boolean)</a></span> - Method in class picard.sam.<a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</a></dt>
<dd>
<div class="block">Sets the values from the alignment record on the unaligned BAM record.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/AbstractInputParser.html#setWordCount-int-">setWordCount(int)</a></span> - Method in class picard.util.<a href="picard/util/AbstractInputParser.html" title="class in picard.util">AbstractInputParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ClusterData.html#setX-int-">setX(int)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ClusterData.html" title="class in picard.illumina.parser">ClusterData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/PhysicalLocation.html#setX-int-">setX(int)</a></span> - Method in interface picard.sam.util.<a href="picard/sam/util/PhysicalLocation.html" title="interface in picard.sam.util">PhysicalLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/PhysicalLocationInt.html#setX-int-">setX(int)</a></span> - Method in class picard.sam.util.<a href="picard/sam/util/PhysicalLocationInt.html" title="class in picard.sam.util">PhysicalLocationInt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/PhysicalLocationShort.html#setX-int-">setX(int)</a></span> - Method in class picard.sam.util.<a href="picard/sam/util/PhysicalLocationShort.html" title="class in picard.sam.util">PhysicalLocationShort</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ClusterData.html#setY-int-">setY(int)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ClusterData.html" title="class in picard.illumina.parser">ClusterData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/PhysicalLocation.html#setY-int-">setY(int)</a></span> - Method in interface picard.sam.util.<a href="picard/sam/util/PhysicalLocation.html" title="interface in picard.sam.util">PhysicalLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/PhysicalLocationInt.html#setY-int-">setY(int)</a></span> - Method in class picard.sam.util.<a href="picard/sam/util/PhysicalLocationInt.html" title="class in picard.sam.util">PhysicalLocationInt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/PhysicalLocationShort.html#setY-int-">setY(int)</a></span> - Method in class picard.sam.util.<a href="picard/sam/util/PhysicalLocationShort.html" title="class in picard.sam.util">PhysicalLocationShort</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/pedigree/Sex.html" title="enum in picard.pedigree"><span class="typeNameLink">Sex</span></a> - Enum in <a href="picard/pedigree/package-summary.html">picard.pedigree</a></dt>
<dd>
<div class="block">Represents the sex of an individual.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilities.html#shiftedLogEvidenceProbability--">shiftedLogEvidenceProbability()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilities.html" title="class in picard.fingerprint">HaplotypeProbabilities</a></dt>
<dd>
<div class="block">Returns log (p(evidence)) + c assuming that the prior on haplotypes is given by
 the internal haplotypeFrequencies</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilities.html#shiftedLogEvidenceProbabilityGivenOtherEvidence-picard.fingerprint.HaplotypeProbabilities-">shiftedLogEvidenceProbabilityGivenOtherEvidence(HaplotypeProbabilities)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilities.html" title="class in picard.fingerprint">HaplotypeProbabilities</a></dt>
<dd>
<div class="block">returns the log-probability the evidence, using as priors the posteriors of another object</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilities.html#shiftedLogEvidenceProbabilityUsingGenotypeFrequencies-double:A-">shiftedLogEvidenceProbabilityUsingGenotypeFrequencies(double[])</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilities.html" title="class in picard.fingerprint">HaplotypeProbabilities</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/MarkQueue.html#shouldBeInLocations-picard.sam.markduplicates.util.ReadEndsForMateCigar-">shouldBeInLocations(ReadEndsForMateCigar)</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/MarkQueue.html" title="class in picard.sam.markduplicates.util">MarkQueue</a></dt>
<dd>
<div class="block">Returns true if we should track this for optical duplicate detection, false otherwise</div>
</dd>
<dt><a href="picard/sam/markduplicates/SimpleMarkDuplicatesWithMateCigar.html" title="class in picard.sam.markduplicates"><span class="typeNameLink">SimpleMarkDuplicatesWithMateCigar</span></a> - Class in <a href="picard/sam/markduplicates/package-summary.html">picard.sam.markduplicates</a></dt>
<dd>
<div class="block">This is a simple tool to mark duplicates using the DuplicateSetIterator, DuplicateSet, and SAMRecordDuplicateComparator.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/SimpleMarkDuplicatesWithMateCigar.html#SimpleMarkDuplicatesWithMateCigar--">SimpleMarkDuplicatesWithMateCigar()</a></span> - Constructor for class picard.sam.markduplicates.<a href="picard/sam/markduplicates/SimpleMarkDuplicatesWithMateCigar.html" title="class in picard.sam.markduplicates">SimpleMarkDuplicatesWithMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/SinglePassSamProgram.html" title="class in picard.analysis"><span class="typeNameLink">SinglePassSamProgram</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Super class that is designed to provide some consistent structure between subclasses that
 simply iterate once over a coordinate sorted BAM and collect information from the records
 as the go in order to produce some kind of output.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/SinglePassSamProgram.html#SinglePassSamProgram--">SinglePassSamProgram()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/SinglePassSamProgram.html" title="class in picard.analysis">SinglePassSamProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeBlock.html#size--">size()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeBlock.html" title="class in picard.fingerprint">HaplotypeBlock</a></dt>
<dd>
<div class="block">Returns the number of SNPs within the haplotype block.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/DiskBasedReadEndsForMarkDuplicatesMap.html#size--">size()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/DiskBasedReadEndsForMarkDuplicatesMap.html" title="class in picard.sam.markduplicates.util">DiskBasedReadEndsForMarkDuplicatesMap</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/MarkQueue.html#size--">size()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/MarkQueue.html" title="class in picard.sam.markduplicates.util">MarkQueue</a></dt>
<dd>
<div class="block">The number of records currently in this queue.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/PhysicalLocationForMateCigarSet.html#size--">size()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/PhysicalLocationForMateCigarSet.html" title="class in picard.sam.markduplicates.util">PhysicalLocationForMateCigarSet</a></dt>
<dd>
<div class="block">The number of records in this set</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesMap.html#size--">size()</a></span> - Method in interface picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesMap.html" title="interface in picard.sam.markduplicates.util">ReadEndsForMarkDuplicatesMap</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicates.html#SIZE_OF">SIZE_OF</a></span> - Static variable in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicates.html" title="class in picard.sam.markduplicates.util">ReadEndsForMarkDuplicates</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/DiskBasedReadEndsForMarkDuplicatesMap.html#sizeInRam--">sizeInRam()</a></span> - Method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/DiskBasedReadEndsForMarkDuplicatesMap.html" title="class in picard.sam.markduplicates.util">DiskBasedReadEndsForMarkDuplicatesMap</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesMap.html#sizeInRam--">sizeInRam()</a></span> - Method in interface picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/ReadEndsForMarkDuplicatesMap.html" title="interface in picard.sam.markduplicates.util">ReadEndsForMarkDuplicatesMap</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadType.html#Skip">Skip</a></span> - Static variable in enum picard.illumina.parser.<a href="picard/illumina/parser/ReadType.html" title="enum in picard.illumina.parser">ReadType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/FindMendelianViolations.html#SKIP_CHROMS">SKIP_CHROMS</a></span> - Variable in class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/FindMendelianViolations.html" title="class in picard.vcf.MendelianViolations">FindMendelianViolations</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ParameterizedFileUtil.html#skipEmptyFiles">skipEmptyFiles</a></span> - Variable in class picard.illumina.parser.<a href="picard/illumina/parser/ParameterizedFileUtil.html" title="class in picard.illumina.parser">ParameterizedFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/FilterFileReader.html#skipRecords-int-">skipRecords(int)</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/FilterFileReader.html" title="class in picard.illumina.parser.readers">FilterFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/LocsFileReader.html#skipRecords-int-">skipRecords(int)</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/LocsFileReader.html" title="class in picard.illumina.parser.readers">LocsFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadStructure.html#skips">skips</a></span> - Variable in class picard.illumina.parser.<a href="picard/illumina/parser/ReadStructure.html" title="class in picard.illumina.parser">ReadStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/FingerprintingDetailMetrics.html#SNP">SNP</a></span> - Variable in class picard.analysis.<a href="picard/analysis/FingerprintingDetailMetrics.html" title="class in picard.analysis">FingerprintingDetailMetrics</a></dt>
<dd>
<div class="block">The name of a representative SNP within the haplotype that was compared.</div>
</dd>
<dt><a href="picard/fingerprint/Snp.html" title="class in picard.fingerprint"><span class="typeNameLink">Snp</span></a> - Class in <a href="picard/fingerprint/package-summary.html">picard.fingerprint</a></dt>
<dd>
<div class="block">Class to represent a SNP in context of a haplotype block that is used in fingerprinting.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/Snp.html#Snp-java.lang.String-java.lang.String-int-byte-byte-double-java.util.List-">Snp(String, String, int, byte, byte, double, List&lt;String&gt;)</a></span> - Constructor for class picard.fingerprint.<a href="picard/fingerprint/Snp.html" title="class in picard.fingerprint">Snp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/FingerprintingDetailMetrics.html#SNP_ALLELES">SNP_ALLELES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/FingerprintingDetailMetrics.html" title="class in picard.analysis">FingerprintingDetailMetrics</a></dt>
<dd>
<div class="block">The possible alleles for the SNP.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/SplitVcfs.html#SNP_OUTPUT">SNP_OUTPUT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/SplitVcfs.html" title="class in picard.vcf">SplitVcfs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html#SNP_REFERENCE_BIAS">SNP_REFERENCE_BIAS</a></span> - Variable in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingSummaryMetrics</a></dt>
<dd>
<div class="block">The rate at which reference bases are observed at ref/alt heterozygous SNP sites.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#snpCounter">snpCounter</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/DbSnpBitSetUtil.DbSnpBitSets.html#snps">snps</a></span> - Variable in class picard.util.<a href="picard/util/DbSnpBitSetUtil.DbSnpBitSets.html" title="class in picard.util">DbSnpBitSetUtil.DbSnpBitSets</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/BedToIntervalList.html#SORT">SORT</a></span> - Variable in class picard.util.<a href="picard/util/BedToIntervalList.html" title="class in picard.util">BedToIntervalList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IntervalListToBed.html#SORT">SORT</a></span> - Variable in class picard.util.<a href="picard/util/IntervalListToBed.html" title="class in picard.util">IntervalListToBed</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IntervalListTools.html#SORT">SORT</a></span> - Variable in class picard.util.<a href="picard/util/IntervalListTools.html" title="class in picard.util">IntervalListTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AddOrReplaceReadGroups.html#SORT_ORDER">SORT_ORDER</a></span> - Variable in class picard.sam.<a href="picard/sam/AddOrReplaceReadGroups.html" title="class in picard.sam">AddOrReplaceReadGroups</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FastqToSam.html#SORT_ORDER">SORT_ORDER</a></span> - Variable in class picard.sam.<a href="picard/sam/FastqToSam.html" title="class in picard.sam">FastqToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FilterSamReads.html#SORT_ORDER">SORT_ORDER</a></span> - Variable in class picard.sam.<a href="picard/sam/FilterSamReads.html" title="class in picard.sam">FilterSamReads</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FixMateInformation.html#SORT_ORDER">SORT_ORDER</a></span> - Variable in class picard.sam.<a href="picard/sam/FixMateInformation.html" title="class in picard.sam">FixMateInformation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#SORT_ORDER">SORT_ORDER</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeSamFiles.html#SORT_ORDER">SORT_ORDER</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeSamFiles.html" title="class in picard.sam">MergeSamFiles</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertOriginalBaseQualitiesAndAddMateCigar.html#SORT_ORDER">SORT_ORDER</a></span> - Variable in class picard.sam.<a href="picard/sam/RevertOriginalBaseQualitiesAndAddMateCigar.html" title="class in picard.sam">RevertOriginalBaseQualitiesAndAddMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertSam.html#SORT_ORDER">SORT_ORDER</a></span> - Variable in class picard.sam.<a href="picard/sam/RevertSam.html" title="class in picard.sam">RevertSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SortSam.html#SORT_ORDER">SORT_ORDER</a></span> - Variable in class picard.sam.<a href="picard/sam/SortSam.html" title="class in picard.sam">SortSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/StandardOptionDefinitions.html#SORT_ORDER_SHORT_NAME">SORT_ORDER_SHORT_NAME</a></span> - Static variable in class picard.cmdline.<a href="picard/cmdline/StandardOptionDefinitions.html" title="class in picard.cmdline">StandardOptionDefinitions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicates.html#SORTING_COLLECTION_SIZE_RATIO">SORTING_COLLECTION_SIZE_RATIO</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicates.html" title="class in picard.sam.markduplicates">MarkDuplicates</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/SortSam.html" title="class in picard.sam"><span class="typeNameLink">SortSam</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SortSam.html#SortSam--">SortSam()</a></span> - Constructor for class picard.sam.<a href="picard/sam/SortSam.html" title="class in picard.sam">SortSam</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/SortVcf.html" title="class in picard.vcf"><span class="typeNameLink">SortVcf</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">Sorts one or more VCF files according to the order of the contigs in the header/sequence dictionary and then
 by coordinate.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/SortVcf.html#SortVcf--">SortVcf()</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/SortVcf.html" title="class in picard.vcf">SortVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#specialArgumentsCollection">specialArgumentsCollection</a></span> - Variable in class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CreateSequenceDictionary.html#SPECIES">SPECIES</a></span> - Variable in class picard.sam.<a href="picard/sam/CreateSequenceDictionary.html" title="class in picard.sam">CreateSequenceDictionary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/Test.html#split-java.lang.String-java.lang.String:A-java.lang.String-">split(String, String[], String)</a></span> - Method in class picard.<a href="picard/Test.html" title="class in picard">Test</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/SplitSamByLibrary.html" title="class in picard.sam"><span class="typeNameLink">SplitSamByLibrary</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block">Command-line program to split a SAM or BAM file into separate files based on
 library name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SplitSamByLibrary.html#SplitSamByLibrary--">SplitSamByLibrary()</a></span> - Constructor for class picard.sam.<a href="picard/sam/SplitSamByLibrary.html" title="class in picard.sam">SplitSamByLibrary</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/SplitVcfs.html" title="class in picard.vcf"><span class="typeNameLink">SplitVcfs</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">Splits the input VCF file into two, one for indels and one for SNPs.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/SplitVcfs.html#SplitVcfs--">SplitVcfs()</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/SplitVcfs.html" title="class in picard.vcf">SplitVcfs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/InsertSizeMetrics.html#STANDARD_DEVIATION">STANDARD_DEVIATION</a></span> - Variable in class picard.analysis.<a href="picard/analysis/InsertSizeMetrics.html" title="class in picard.analysis">InsertSizeMetrics</a></dt>
<dd>
<div class="block">Standard deviation of insert sizes over the "core" of the distribution.</div>
</dd>
<dt><a href="picard/cmdline/StandardOptionDefinitions.html" title="class in picard.cmdline"><span class="typeNameLink">StandardOptionDefinitions</span></a> - Class in <a href="picard/cmdline/package-summary.html">picard.cmdline</a></dt>
<dd>
<div class="block">A set of String constants in which the name of the constant (minus the _SHORT_NAME suffix)
 is the standard long Option name, and the value of the constant is the standard shortName.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/StandardOptionDefinitions.html#StandardOptionDefinitions--">StandardOptionDefinitions()</a></span> - Constructor for class picard.cmdline.<a href="picard/cmdline/StandardOptionDefinitions.html" title="class in picard.cmdline">StandardOptionDefinitions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/Gene.Transcript.Exon.html#start">start</a></span> - Variable in class picard.annotation.<a href="picard/annotation/Gene.Transcript.Exon.html" title="class in picard.annotation">Gene.Transcript.Exon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/Gene.Transcript.html#start--">start()</a></span> - Method in class picard.annotation.<a href="picard/annotation/Gene.Transcript.html" title="class in picard.annotation">Gene.Transcript</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/Range.html#start">start</a></span> - Variable in class picard.illumina.parser.<a href="picard/illumina/parser/Range.html" title="class in picard.illumina.parser">Range</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/help/PicardHelpDoclet.html#start-com.sun.javadoc.RootDoc-">start(RootDoc)</a></span> - Static method in class picard.util.help.<a href="picard/util/help/PicardHelpDoclet.html" title="class in picard.util.help">PicardHelpDoclet</a></dt>
<dd>
<div class="block">Create a doclet of the appropriate type and generate the FreeMarker templates properties.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VariantAccumulatorExecutor.MultiThreadedChunkBased.html#start--">start()</a></span> - Method in class picard.vcf.processor.<a href="picard/vcf/processor/VariantAccumulatorExecutor.MultiThreadedChunkBased.html" title="class in picard.vcf.processor">VariantAccumulatorExecutor.MultiThreadedChunkBased</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VariantAccumulatorExecutor.html#start--">start()</a></span> - Method in interface picard.vcf.processor.<a href="picard/vcf/processor/VariantAccumulatorExecutor.html" title="interface in picard.vcf.processor">VariantAccumulatorExecutor</a></dt>
<dd>
<div class="block">Starts the work of the executor, returning immediately.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VcfFileSegment.html#start--">start()</a></span> - Method in class picard.vcf.processor.<a href="picard/vcf/processor/VcfFileSegment.html" title="class in picard.vcf.processor">VcfFileSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#stddev-double:A-int-int-">stddev(double[], int, int)</a></span> - Static method in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>
<div class="block">Calculated the standard deviation of an array of doubles.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#stddev-double:A-int-int-double-">stddev(double[], int, int, double)</a></span> - Static method in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>
<div class="block">Calculated the standard deviation of an array of doubles.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VcfFileSegment.html#stop--">stop()</a></span> - Method in class picard.vcf.processor.<a href="picard/vcf/processor/VcfFileSegment.html" title="class in picard.vcf.processor">VcfFileSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectMultipleMetrics.html#STOP_AFTER">STOP_AFTER</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectMultipleMetrics.html" title="class in picard.analysis">CollectMultipleMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.html#STOP_AFTER">STOP_AFTER</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.html" title="class in picard.analysis">CollectOxoGMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.html#STOP_AFTER">STOP_AFTER</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/CollectIndependentReplicateMetrics.html#STOP_AFTER">STOP_AFTER</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/CollectIndependentReplicateMetrics.html" title="class in picard.analysis.replicates">CollectIndependentReplicateMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/SinglePassSamProgram.html#STOP_AFTER">STOP_AFTER</a></span> - Variable in class picard.analysis.<a href="picard/analysis/SinglePassSamProgram.html" title="class in picard.analysis">SinglePassSamProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/PositionBasedDownsampleSam.html#STOP_AFTER">STOP_AFTER</a></span> - Variable in class picard.sam.<a href="picard/sam/PositionBasedDownsampleSam.html" title="class in picard.sam">PositionBasedDownsampleSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AlignmentSummaryMetrics.html#STRAND_BALANCE">STRAND_BALANCE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/AlignmentSummaryMetrics.html" title="class in picard.analysis">AlignmentSummaryMetrics</a></dt>
<dd>
<div class="block">The number of PF reads aligned to the positive strand of the genome divided by the number of
 PF reads aligned to the genome.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRnaSeqMetrics.html#STRAND_SPECIFICITY">STRAND_SPECIFICITY</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectRnaSeqMetrics.html" title="class in picard.analysis">CollectRnaSeqMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/DownsampleSam.html#STRATEGY">STRATEGY</a></span> - Variable in class picard.sam.<a href="picard/sam/DownsampleSam.html" title="class in picard.sam">DownsampleSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/SplitVcfs.html#STRICT">STRICT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/SplitVcfs.html" title="class in picard.vcf">SplitVcfs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IlluminaUtil.html#stringSeqsToString-java.lang.String:A-java.lang.String-">stringSeqsToString(String[], String)</a></span> - Static method in class picard.util.<a href="picard/util/IlluminaUtil.html" title="class in picard.util">IlluminaUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FastqToSam.html#STRIP_UNPAIRED_MATE_NUMBER">STRIP_UNPAIRED_MATE_NUMBER</a></span> - Variable in class picard.sam.<a href="picard/sam/FastqToSam.html" title="class in picard.sam">FastqToSam</a></dt>
<dd>
<div class="block"><span class="deprecatedLabel">Deprecated.</span></div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/IntervalListTools.html#SUBDIVISION_MODE">SUBDIVISION_MODE</a></span> - Variable in class picard.util.<a href="picard/util/IntervalListTools.html" title="class in picard.util">IntervalListTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AlleleSubsettingUtils.html#subsetAlleles-htsjdk.variant.variantcontext.VariantContext-java.util.List-">subsetAlleles(VariantContext, List&lt;Allele&gt;)</a></span> - Static method in class picard.util.<a href="picard/util/AlleleSubsettingUtils.html" title="class in picard.util">AlleleSubsettingUtils</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AlleleSubsettingUtils.html#subsetAlleles-htsjdk.variant.variantcontext.GenotypesContext-java.util.List-java.util.List-">subsetAlleles(GenotypesContext, List&lt;Allele&gt;, List&lt;Allele&gt;)</a></span> - Static method in class picard.util.<a href="picard/util/AlleleSubsettingUtils.html" title="class in picard.util">AlleleSubsettingUtils</a></dt>
<dd>
<div class="block">Create the new GenotypesContext with the subsetted PLs and ADs
 Expects allelesToKeep to be in the same order
 in which they are in originalAlleles.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/AlleleSubsettingUtils.html#subsettedPLIndices-java.util.List-java.util.List-">subsettedPLIndices(List&lt;Allele&gt;, List&lt;Allele&gt;)</a></span> - Static method in class picard.util.<a href="picard/util/AlleleSubsettingUtils.html" title="class in picard.util">AlleleSubsettingUtils</a></dt>
<dd>
<div class="block">Given a list of original alleles and a subset of new alleles to retain, find the array of old PL indices that correspond
 to new PL indices i.e.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/AlleleSubsettingUtils.html#subsetVCToMatchSnp-htsjdk.variant.variantcontext.VariantContext-picard.fingerprint.Snp-">subsetVCToMatchSnp(VariantContext, Snp)</a></span> - Static method in class picard.util.<a href="picard/util/AlleleSubsettingUtils.html" title="class in picard.util">AlleleSubsettingUtils</a></dt>
<dd>
<div class="block">Method to subset the alleles in the VariantContext to those in the input snp.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/ErrorSummaryMetrics.html#SUBSTITUTION">SUBSTITUTION</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/ErrorSummaryMetrics.html" title="class in picard.analysis.artifacts">ErrorSummaryMetrics</a></dt>
<dd>
<div class="block">A single string representing the substition from REF_BASE to ALT_BASE for convenience.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/ErrorSummaryMetrics.html#SUBSTITUTION_RATE">SUBSTITUTION_RATE</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/ErrorSummaryMetrics.html" title="class in picard.analysis.artifacts">ErrorSummaryMetrics</a></dt>
<dd>
<div class="block">The rate of the substitution in question.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadStructure.Substructure.html#Substructure-java.util.List-java.util.List-">Substructure(List&lt;Integer&gt;, List&lt;Range&gt;)</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/ReadStructure.Substructure.html" title="class in picard.illumina.parser">ReadStructure.Substructure</a></dt>
<dd>
<div class="block">Indices into the ReadStructure.descriptors for this specific substructure, indices
 must be in the order they appear in the descriptors list (but the indices do NOT have to be continuous)</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.LogMath.html#sum-double...-">sum(double...)</a></span> - Method in class picard.util.<a href="picard/util/MathUtil.LogMath.html" title="class in picard.util">MathUtil.LogMath</a></dt>
<dd>
<div class="block">Computes the sum of the provided log values.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#sum-double:A-double:A-">sum(double[], double[])</a></span> - Static method in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>
<div class="block">calculates the sum of the arrays as a third array.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#sum-double:A-">sum(double[])</a></span> - Static method in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>
<div class="block">Returns the sum of the elements in the array.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/MathUtil.html#sum-long:A-int-int-">sum(long[], int, int)</a></span> - Static method in class picard.util.<a href="picard/util/MathUtil.html" title="class in picard.util">MathUtil</a></dt>
<dd>
<div class="block">Returns the sum of the elements in the array starting with start and ending before stop.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/metrics/GcBiasMetrics.html#SUMMARY">SUMMARY</a></span> - Variable in class picard.metrics.<a href="picard/metrics/GcBiasMetrics.html" title="class in picard.metrics">GcBiasMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectRrbsMetrics.html#SUMMARY_FILE_EXTENSION">SUMMARY_FILE_EXTENSION</a></span> - Static variable in class picard.analysis.<a href="picard/analysis/CollectRrbsMetrics.html" title="class in picard.analysis">CollectRrbsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#SUMMARY_METRICS_FILE_EXTENSION">SUMMARY_METRICS_FILE_EXTENSION</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectGcBiasMetrics.html#SUMMARY_OUTPUT">SUMMARY_OUTPUT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectGcBiasMetrics.html" title="class in picard.analysis">CollectGcBiasMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CheckFingerprint.html#SUMMARY_OUTPUT">SUMMARY_OUTPUT</a></span> - Variable in class picard.fingerprint.<a href="picard/fingerprint/CheckFingerprint.html" title="class in picard.fingerprint">CheckFingerprint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.html#summaryMetricsExtension">summaryMetricsExtension</a></span> - Static variable in class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/MultiTileParser.html#supportedTypes--">supportedTypes()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/MultiTileParser.html" title="class in picard.illumina.parser">MultiTileParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PosParser.html#supportedTypes--">supportedTypes()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/PosParser.html" title="class in picard.illumina.parser">PosParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectMultipleMetrics.ProgramInterface.html#supportsMetricAccumulationLevel--">supportsMetricAccumulationLevel()</a></span> - Method in interface picard.analysis.<a href="picard/analysis/CollectMultipleMetrics.ProgramInterface.html" title="interface in picard.analysis">CollectMultipleMetrics.ProgramInterface</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/AlleleSubsettingUtils.html#swapAlleles-htsjdk.variant.variantcontext.VariantContext-htsjdk.variant.variantcontext.Allele-htsjdk.variant.variantcontext.Allele-">swapAlleles(VariantContext, Allele, Allele)</a></span> - Static method in class picard.util.<a href="picard/util/AlleleSubsettingUtils.html" title="class in picard.util">AlleleSubsettingUtils</a></dt>
<dd>
<div class="block">Swaps one of the alleles in a VC (and its genotypes) with another.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeMap.html#SYNTHETIC_PHASESET_PREFIX">SYNTHETIC_PHASESET_PREFIX</a></span> - Static variable in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeMap.html" title="class in picard.fingerprint">HaplotypeMap</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="I:T">
<!--   -->
</a>
<h2 class="title">T</h2>
<dl>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/FindMendelianViolations.html#TAB_MODE">TAB_MODE</a></span> - Variable in class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/FindMendelianViolations.html" title="class in picard.vcf.MendelianViolations">FindMendelianViolations</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/TabbedInputParser.html" title="class in picard.util"><span class="typeNameLink">TabbedInputParser</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">Parser for tab-delimited files</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/TabbedInputParser.html#TabbedInputParser-boolean-java.io.InputStream...-">TabbedInputParser(boolean, InputStream...)</a></span> - Constructor for class picard.util.<a href="picard/util/TabbedInputParser.html" title="class in picard.util">TabbedInputParser</a></dt>
<dd>
<div class="block">Constructor</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/TabbedInputParser.html#TabbedInputParser-boolean-java.io.File...-">TabbedInputParser(boolean, File...)</a></span> - Constructor for class picard.util.<a href="picard/util/TabbedInputParser.html" title="class in picard.util">TabbedInputParser</a></dt>
<dd>
<div class="block">Constructor</div>
</dd>
<dt><a href="picard/util/TabbedTextFileWithHeaderParser.html" title="class in picard.util"><span class="typeNameLink">TabbedTextFileWithHeaderParser</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">Parse a tabbed text file in which columns are found by looking at a header line rather than by position.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/TabbedTextFileWithHeaderParser.html#TabbedTextFileWithHeaderParser-picard.util.TabbedInputParser-">TabbedTextFileWithHeaderParser(TabbedInputParser)</a></span> - Constructor for class picard.util.<a href="picard/util/TabbedTextFileWithHeaderParser.html" title="class in picard.util">TabbedTextFileWithHeaderParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/TabbedTextFileWithHeaderParser.html#TabbedTextFileWithHeaderParser-java.io.File-">TabbedTextFileWithHeaderParser(File)</a></span> - Constructor for class picard.util.<a href="picard/util/TabbedTextFileWithHeaderParser.html" title="class in picard.util">TabbedTextFileWithHeaderParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/TabbedTextFileWithHeaderParser.html#TabbedTextFileWithHeaderParser-java.io.File-java.lang.String:A-">TabbedTextFileWithHeaderParser(File, String[])</a></span> - Constructor for class picard.util.<a href="picard/util/TabbedTextFileWithHeaderParser.html" title="class in picard.util">TabbedTextFileWithHeaderParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/TabbedTextFileWithHeaderParser.Row.html" title="class in picard.util"><span class="typeNameLink">TabbedTextFileWithHeaderParser.Row</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicates.html#TAG_DUPLICATE_SET_MEMBERS">TAG_DUPLICATE_SET_MEMBERS</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicates.html" title="class in picard.sam.markduplicates">MarkDuplicates</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#TAG_PER_MOLECULAR_INDEX">TAG_PER_MOLECULAR_INDEX</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicates.html#TAGGING_POLICY">TAGGING_POLICY</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicates.html" title="class in picard.sam.markduplicates">MarkDuplicates</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectJumpingLibraryMetrics.html#TAIL_LIMIT">TAIL_LIMIT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectJumpingLibraryMetrics.html" title="class in picard.analysis">CollectJumpingLibraryMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html#TANDEM_READS">TANDEM_READS</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html" title="class in picard.analysis.artifacts">CollectSequencingArtifactMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/CollectTargetedMetrics.html#TARGET_INTERVALS">TARGET_INTERVALS</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/CollectTargetedMetrics.html" title="class in picard.analysis.directed">CollectTargetedMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.html#TARGET_INTERVALS">TARGET_INTERVALS</a></span> - Variable in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#TARGET_TERRITORY">TARGET_TERRITORY</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The unique number of target bases in the experiment, where the target sequence is usually exons etc.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#TARGET_TERRITORY">TARGET_TERRITORY</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The number of unique bases covered by the intervals of all targets that should be covered</div>
</dd>
<dt><a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed"><span class="typeNameLink">TargetedPcrMetrics</span></a> - Class in <a href="picard/analysis/directed/package-summary.html">picard.analysis.directed</a></dt>
<dd>
<div class="block">Metrics class for the analysis of reads obtained from targeted pcr experiments e.g.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#TargetedPcrMetrics--">TargetedPcrMetrics()</a></span> - Constructor for class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/directed/TargetedPcrMetricsCollector.html" title="class in picard.analysis.directed"><span class="typeNameLink">TargetedPcrMetricsCollector</span></a> - Class in <a href="picard/analysis/directed/package-summary.html">picard.analysis.directed</a></dt>
<dd>
<div class="block">Calculates HS metrics for a given SAM or BAM file.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetricsCollector.html#TargetedPcrMetricsCollector-java.util.Set-java.util.List-htsjdk.samtools.reference.ReferenceSequenceFile-java.io.File-java.io.File-htsjdk.samtools.util.IntervalList-htsjdk.samtools.util.IntervalList-java.lang.String-int-int-int-boolean-int-int-">TargetedPcrMetricsCollector(Set&lt;MetricAccumulationLevel&gt;, List&lt;SAMReadGroupRecord&gt;, ReferenceSequenceFile, File, File, IntervalList, IntervalList, String, int, int, int, boolean, int, int)</a></span> - Constructor for class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetricsCollector.html" title="class in picard.analysis.directed">TargetedPcrMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetricsCollector.html#TargetedPcrMetricsCollector-java.util.Set-java.util.List-htsjdk.samtools.reference.ReferenceSequenceFile-java.io.File-java.io.File-htsjdk.samtools.util.IntervalList-htsjdk.samtools.util.IntervalList-java.lang.String-int-int-int-boolean-boolean-int-int-">TargetedPcrMetricsCollector(Set&lt;MetricAccumulationLevel&gt;, List&lt;SAMReadGroupRecord&gt;, ReferenceSequenceFile, File, File, IntervalList, IntervalList, String, int, int, int, boolean, boolean, int, int)</a></span> - Constructor for class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetricsCollector.html" title="class in picard.analysis.directed">TargetedPcrMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/directed/TargetMetricsCollector.html" title="class in picard.analysis.directed"><span class="typeNameLink">TargetMetricsCollector</span></a>&lt;<a href="picard/analysis/directed/TargetMetricsCollector.html" title="type parameter in TargetMetricsCollector">METRIC_TYPE</a> extends <a href="picard/metrics/MultilevelMetrics.html" title="class in picard.metrics">MultilevelMetrics</a>&gt; - Class in <a href="picard/analysis/directed/package-summary.html">picard.analysis.directed</a></dt>
<dd>
<div class="block">TargetMetrics, are metrics to measure how well we hit specific targets (or baits) when using a targeted sequencing process like hybrid selection
 or Targeted PCR Techniques (TSCA).</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetMetricsCollector.html#TargetMetricsCollector-java.util.Set-java.util.List-htsjdk.samtools.reference.ReferenceSequenceFile-java.io.File-java.io.File-htsjdk.samtools.util.IntervalList-htsjdk.samtools.util.IntervalList-java.lang.String-int-int-int-boolean-int-int-">TargetMetricsCollector(Set&lt;MetricAccumulationLevel&gt;, List&lt;SAMReadGroupRecord&gt;, ReferenceSequenceFile, File, File, IntervalList, IntervalList, String, int, int, int, boolean, int, int)</a></span> - Constructor for class picard.analysis.directed.<a href="picard/analysis/directed/TargetMetricsCollector.html" title="class in picard.analysis.directed">TargetMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetMetricsCollector.html#TargetMetricsCollector-java.util.Set-java.util.List-htsjdk.samtools.reference.ReferenceSequenceFile-java.io.File-java.io.File-htsjdk.samtools.util.IntervalList-htsjdk.samtools.util.IntervalList-java.lang.String-int-int-int-boolean-boolean-int-int-">TargetMetricsCollector(Set&lt;MetricAccumulationLevel&gt;, List&lt;SAMReadGroupRecord&gt;, ReferenceSequenceFile, File, File, IntervalList, IntervalList, String, int, int, int, boolean, boolean, int, int)</a></span> - Constructor for class picard.analysis.directed.<a href="picard/analysis/directed/TargetMetricsCollector.html" title="class in picard.analysis.directed">TargetMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/directed/TargetMetricsCollector.Coverage.html" title="class in picard.analysis.directed"><span class="typeNameLink">TargetMetricsCollector.Coverage</span></a> - Class in <a href="picard/analysis/directed/package-summary.html">picard.analysis.directed</a></dt>
<dd>
<div class="block">A simple class that is used to store the coverage information about an interval.</div>
</dd>
<dt><a href="picard/analysis/directed/TargetMetricsCollector.PerUnitTargetMetricCollector.html" title="class in picard.analysis.directed"><span class="typeNameLink">TargetMetricsCollector.PerUnitTargetMetricCollector</span></a> - Class in <a href="picard/analysis/directed/package-summary.html">picard.analysis.directed</a></dt>
<dd>
<div class="block">Collect the Target Metrics for one unit of "accumulation" (i.e.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/BaitDesigner.html#TARGETS">TARGETS</a></span> - Variable in class picard.util.<a href="picard/util/BaitDesigner.html" title="class in picard.util">BaitDesigner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadType.html#Template">Template</a></span> - Static variable in enum picard.illumina.parser.<a href="picard/illumina/parser/ReadType.html" title="enum in picard.illumina.parser">ReadType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadStructure.html#templates">templates</a></span> - Variable in class picard.illumina.parser.<a href="picard/illumina/parser/ReadStructure.html" title="class in picard.illumina.parser">ReadStructure</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/Test.html" title="class in picard"><span class="typeNameLink">Test</span></a> - Class in <a href="picard/package-summary.html">picard</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/Test.html#Test--">Test()</a></span> - Constructor for class picard.<a href="picard/Test.html" title="class in picard">Test</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectAlignmentSummaryMetrics.html#testDoWork--">testDoWork()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectAlignmentSummaryMetrics.html" title="class in picard.analysis">CollectAlignmentSummaryMetrics</a></dt>
<dd>
<div class="block">Silly method that is necessary to give unit test access to call doWork()</div>
</dd>
<dt><a href="picard/cmdline/programgroups/Testing.html" title="class in picard.cmdline.programgroups"><span class="typeNameLink">Testing</span></a> - Class in <a href="picard/cmdline/programgroups/package-summary.html">picard.cmdline.programgroups</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/programgroups/Testing.html#Testing--">Testing()</a></span> - Constructor for class picard.cmdline.programgroups.<a href="picard/cmdline/programgroups/Testing.html" title="class in picard.cmdline.programgroups">Testing</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#theoreticalHetSensitivitySampleSize">theoreticalHetSensitivitySampleSize</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">The sample size used for theoretical het sensitivity.</div>
</dd>
<dt><a href="picard/analysis/TheoreticalSensitivity.html" title="class in picard.analysis"><span class="typeNameLink">TheoreticalSensitivity</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Created by David Benjamin on 5/13/15.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/TheoreticalSensitivity.html#TheoreticalSensitivity--">TheoreticalSensitivity()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/TheoreticalSensitivity.html" title="class in picard.analysis">TheoreticalSensitivity</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/TheoreticalSensitivity.RouletteWheel.html" title="class in picard.analysis"><span class="typeNameLink">TheoreticalSensitivity.RouletteWheel</span></a> - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.html#THREAD_COUNT">THREAD_COUNT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/FindMendelianViolations.html#THREAD_COUNT">THREAD_COUNT</a></span> - Variable in class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/FindMendelianViolations.html" title="class in picard.vcf.MendelianViolations">FindMendelianViolations</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/util/ThreadPoolExecutorWithExceptions.html" title="class in picard.util"><span class="typeNameLink">ThreadPoolExecutorWithExceptions</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">This version of the thread pool executor will throw an exception if any of the internal jobs have throw exceptions
 while executing</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/ThreadPoolExecutorWithExceptions.html#ThreadPoolExecutorWithExceptions-int-">ThreadPoolExecutorWithExceptions(int)</a></span> - Constructor for class picard.util.<a href="picard/util/ThreadPoolExecutorWithExceptions.html" title="class in picard.util">ThreadPoolExecutorWithExceptions</a></dt>
<dd>
<div class="block">Creates a fixed size thread pool executor that will rethrow exceptions from submitted jobs.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#THREE_PRIME_ADAPTER">THREE_PRIME_ADAPTER</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/MarkIlluminaAdapters.html#THREE_PRIME_ADAPTER">THREE_PRIME_ADAPTER</a></span> - Variable in class picard.illumina.<a href="picard/illumina/MarkIlluminaAdapters.html" title="class in picard.illumina">MarkIlluminaAdapters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.PositionInfo.html#tile">tile</a></span> - Variable in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.PositionInfo.html" title="class in picard.illumina.parser.readers">AbstractIlluminaPositionFileReader.PositionInfo</a></dt>
<dd>
<div class="block">The tile, which is determined from the file name</div>
</dd>
<dt><a href="picard/illumina/parser/Tile.html" title="class in picard.illumina.parser"><span class="typeNameLink">Tile</span></a> - Class in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>
<div class="block">Represents a tile from TileMetricsOut.bin.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/Tile.html#Tile-int-int-float-float-picard.illumina.parser.TilePhasingValue...-">Tile(int, int, float, float, TilePhasingValue...)</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/Tile.html" title="class in picard.illumina.parser">Tile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailDetailedMetric.html#TILE">TILE</a></span> - Variable in class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailDetailedMetric.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics.PFFailDetailedMetric</a></dt>
<dd>
<div class="block">The Tile that is described by this metric</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html#TILE">TILE</a></span> - Variable in class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailSummaryMetric.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics.PFFailSummaryMetric</a></dt>
<dd>
<div class="block">The Tile that is described by this metric.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/PhysicalLocationInt.html#tile">tile</a></span> - Variable in class picard.sam.util.<a href="picard/sam/util/PhysicalLocationInt.html" title="class in picard.sam.util">PhysicalLocationInt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToFastq.html#TILE_LIMIT">TILE_LIMIT</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToFastq.html" title="class in picard.illumina">IlluminaBasecallsToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#TILE_LIMIT">TILE_LIMIT</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/TileMetricsUtil.html#TILE_METRICS_OUT_FILE_NAME">TILE_METRICS_OUT_FILE_NAME</a></span> - Static variable in class picard.illumina.parser.<a href="picard/illumina/parser/TileMetricsUtil.html" title="class in picard.illumina.parser">TileMetricsUtil</a></dt>
<dd>
<div class="block">The expected name of the tile metrics output file.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CheckIlluminaDirectory.html#TILE_NUMBERS">TILE_NUMBERS</a></span> - Variable in class picard.illumina.<a href="picard/illumina/CheckIlluminaDirectory.html" title="class in picard.illumina">CheckIlluminaDirectory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/MultiTileFileUtil.html#tileIndex">tileIndex</a></span> - Variable in class picard.illumina.parser.<a href="picard/illumina/parser/MultiTileFileUtil.html" title="class in picard.illumina.parser">MultiTileFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/TileIndex.html" title="class in picard.illumina.parser"><span class="typeNameLink">TileIndex</span></a> - Class in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>
<div class="block">Load a file containing 8-byte records like this:
 tile number: 4-byte int
 number of clusters in tile: 4-byte int
 Number of records to read is determined by reaching EOF.</div>
</dd>
<dt><a href="picard/illumina/parser/TileIndex.TileIndexRecord.html" title="class in picard.illumina.parser"><span class="typeNameLink">TileIndex.TileIndexRecord</span></a> - Class in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ClusterData.html#tileIsSet--">tileIsSet()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ClusterData.html" title="class in picard.illumina.parser">ClusterData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaFileUtil.html#tileMetricsOut--">tileMetricsOut()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/IlluminaFileUtil.html" title="class in picard.illumina.parser">IlluminaFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/readers/TileMetricsOutReader.html" title="class in picard.illumina.parser.readers"><span class="typeNameLink">TileMetricsOutReader</span></a> - Class in <a href="picard/illumina/parser/readers/package-summary.html">picard.illumina.parser.readers</a></dt>
<dd>
<div class="block">Reads a TileMetricsOut file commonly found in the InterOp directory of an Illumina Run Folder.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/TileMetricsOutReader.html#TileMetricsOutReader-java.io.File-picard.illumina.parser.readers.TileMetricsOutReader.TileMetricsVersion-">TileMetricsOutReader(File, TileMetricsOutReader.TileMetricsVersion)</a></span> - Constructor for class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/TileMetricsOutReader.html" title="class in picard.illumina.parser.readers">TileMetricsOutReader</a></dt>
<dd>
<div class="block">Return a TileMetricsOutReader for the specified file</div>
</dd>
<dt><a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaLaneTileCode.html" title="class in picard.illumina.parser.readers"><span class="typeNameLink">TileMetricsOutReader.IlluminaLaneTileCode</span></a> - Class in <a href="picard/illumina/parser/readers/package-summary.html">picard.illumina.parser.readers</a></dt>
<dd>
<div class="block">Helper class which captures the combination of a lane, tile & metric code</div>
</dd>
<dt><a href="picard/illumina/parser/readers/TileMetricsOutReader.IlluminaTileMetrics.html" title="class in picard.illumina.parser.readers"><span class="typeNameLink">TileMetricsOutReader.IlluminaTileMetrics</span></a> - Class in <a href="picard/illumina/parser/readers/package-summary.html">picard.illumina.parser.readers</a></dt>
<dd>
<div class="block">IlluminaPhasingMetrics corresponds to a single record in a TileMetricsOut file</div>
</dd>
<dt><a href="picard/illumina/parser/readers/TileMetricsOutReader.TileMetricsVersion.html" title="enum in picard.illumina.parser.readers"><span class="typeNameLink">TileMetricsOutReader.TileMetricsVersion</span></a> - Enum in <a href="picard/illumina/parser/readers/package-summary.html">picard.illumina.parser.readers</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/TileMetricsUtil.html" title="class in picard.illumina.parser"><span class="typeNameLink">TileMetricsUtil</span></a> - Class in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>
<div class="block">Utility for reading the tile data from an Illumina run directory's TileMetricsOut.bin file</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/TileMetricsUtil.html#TileMetricsUtil--">TileMetricsUtil()</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/TileMetricsUtil.html" title="class in picard.illumina.parser">TileMetricsUtil</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/TilePhasingValue.html" title="class in picard.illumina.parser"><span class="typeNameLink">TilePhasingValue</span></a> - Class in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>
<div class="block">Captures information about a phasing value - Which read it corresponds to, which phasing type and a median value</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/TilePhasingValue.html#TilePhasingValue-picard.illumina.parser.TileTemplateRead-float-float-">TilePhasingValue(TileTemplateRead, float, float)</a></span> - Constructor for class picard.illumina.parser.<a href="picard/illumina/parser/TilePhasingValue.html" title="class in picard.illumina.parser">TilePhasingValue</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ParameterizedFileUtil.html#tiles">tiles</a></span> - Variable in class picard.illumina.parser.<a href="picard/illumina/parser/ParameterizedFileUtil.html" title="class in picard.illumina.parser">ParameterizedFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/illumina/parser/TileTemplateRead.html" title="enum in picard.illumina.parser"><span class="typeNameLink">TileTemplateRead</span></a> - Enum in <a href="picard/illumina/parser/package-summary.html">picard.illumina.parser</a></dt>
<dd>
<div class="block">Defines the first or second template read for a tile</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#TMP_DIR">TMP_DIR</a></span> - Variable in class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceContingencyMetrics.html#TN_COUNT">TN_COUNT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceContingencyMetrics.html" title="class in picard.vcf">GenotypeConcordanceContingencyMetrics</a></dt>
<dd>
<div class="block">The TN (true negative) count across all variants</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceScheme.html#TN_FN">TN_FN</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceScheme.html" title="class in picard.vcf">GenotypeConcordanceScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceScheme.html#TN_ONLY">TN_ONLY</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceScheme.html" title="class in picard.vcf">GenotypeConcordanceScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/ClassFinder.html#toClassName-java.lang.String-">toClassName(String)</a></span> - Method in class picard.cmdline.<a href="picard/cmdline/ClassFinder.html" title="class in picard.cmdline">ClassFinder</a></dt>
<dd>
<div class="block">Convert a filename to a class name by removing '.class' and converting '/'s to '.'s.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/pedigree/Sex.html#toCode--">toCode()</a></span> - Method in enum picard.pedigree.<a href="picard/pedigree/Sex.html" title="enum in picard.pedigree">Sex</a></dt>
<dd>
<div class="block">Returns the code used to encode this sex in a ped/fam file.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadStructure.Substructure.html#toReadStructure--">toReadStructure()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ReadStructure.Substructure.html" title="class in picard.illumina.parser">ReadStructure.Substructure</a></dt>
<dd>
<div class="block">Create a ReadStructure from this substructure composed of only the descriptors contained in this substructure, Any
 ReadDescriptors not in this substructure are treated as if they don't exist (e.g.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetMetricsCollector.Coverage.html#toString--">toString()</a></span> - Method in class picard.analysis.directed.<a href="picard/analysis/directed/TargetMetricsCollector.Coverage.html" title="class in picard.analysis.directed">TargetMetricsCollector.Coverage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeBlock.html#toString--">toString()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeBlock.html" title="class in picard.fingerprint">HaplotypeBlock</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/Snp.html#toString--">toString()</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/Snp.html" title="class in picard.fingerprint">Snp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallingMetrics.html#toString--">toString()</a></span> - Method in class picard.illumina.<a href="picard/illumina/IlluminaBasecallingMetrics.html" title="class in picard.illumina">IlluminaBasecallingMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ClusterData.html#toString--">toString()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ClusterData.html" title="class in picard.illumina.parser">ClusterData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/Range.html#toString--">toString()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/Range.html" title="class in picard.illumina.parser">Range</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadDescriptor.html#toString--">toString()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ReadDescriptor.html" title="class in picard.illumina.parser">ReadDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadStructure.html#toString--">toString()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ReadStructure.html" title="class in picard.illumina.parser">ReadStructure</a></dt>
<dd>
<div class="block">Converts this object into a String using rules complementary to the single string constructor above.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VcfFileSegment.html#toString--">toString()</a></span> - Method in class picard.vcf.processor.<a href="picard/vcf/processor/VcfFileSegment.html" title="class in picard.vcf.processor">VcfFileSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html#TOTAL_BASES">TOTAL_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html" title="class in picard.analysis">CollectOxoGMetrics.CpcgMetrics</a></dt>
<dd>
<div class="block">The total number of basecalls observed at all sites.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetrics.html#TOTAL_BASES">TOTAL_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetrics.html" title="class in picard.analysis">CollectQualityYieldMetrics.QualityYieldMetrics</a></dt>
<dd>
<div class="block">The total number of bases in all reads</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallingMetrics.html#TOTAL_BASES">TOTAL_BASES</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallingMetrics.html" title="class in picard.illumina">IlluminaBasecallingMetrics</a></dt>
<dd>
<div class="block">The total number of bases assigned to the index.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasSummaryMetrics.html#TOTAL_CLUSTERS">TOTAL_CLUSTERS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/GcBiasSummaryMetrics.html" title="class in picard.analysis">GcBiasSummaryMetrics</a></dt>
<dd>
<div class="block">The total number of clusters that were seen in the gc bias calculation.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallingMetrics.html#TOTAL_CLUSTERS">TOTAL_CLUSTERS</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallingMetrics.html" title="class in picard.illumina">IlluminaBasecallingMetrics</a></dt>
<dd>
<div class="block">The total number of clusters assigned to the index.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html#TOTAL_COMPLEX_INDELS">TOTAL_COMPLEX_INDELS</a></span> - Variable in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingSummaryMetrics</a></dt>
<dd>
<div class="block">The number of passing complex indel calls that were examined</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingDetailMetrics.html#TOTAL_GQ0_VARIANTS">TOTAL_GQ0_VARIANTS</a></span> - Variable in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingDetailMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingDetailMetrics</a></dt>
<dd>
<div class="block">The total number of variants in a particular sample that have a GQ score of 0.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingDetailMetrics.html#TOTAL_HET_DEPTH">TOTAL_HET_DEPTH</a></span> - Variable in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingDetailMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingDetailMetrics</a></dt>
<dd>
<div class="block">total number of reads (from AD field) for passing bi-allelic SNP hets for this sample</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html#TOTAL_INDELS">TOTAL_INDELS</a></span> - Variable in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingSummaryMetrics</a></dt>
<dd>
<div class="block">The number of passing indel calls that were examined</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html#TOTAL_MENDELIAN_VIOLATIONS">TOTAL_MENDELIAN_VIOLATIONS</a></span> - Variable in class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/MendelianViolationMetrics.html" title="class in picard.vcf.MendelianViolations">MendelianViolationMetrics</a></dt>
<dd>
<div class="block">The total of all mendelian violations observed.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html#TOTAL_MULTIALLELIC_SNPS">TOTAL_MULTIALLELIC_SNPS</a></span> - Variable in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingSummaryMetrics</a></dt>
<dd>
<div class="block">The number of passing multi-allelic SNP calls that were examined</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasSummaryMetrics.html#TOTAL_QSCORE">TOTAL_QSCORE</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasSummaryMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.BaitBiasSummaryMetrics</a></dt>
<dd>
<div class="block">The total Phred-scaled Q-score for this artifact.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterSummaryMetrics.html#TOTAL_QSCORE">TOTAL_QSCORE</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterSummaryMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.PreAdapterSummaryMetrics</a></dt>
<dd>
<div class="block">The total Phred-scaled Q-score for this artifact.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AlignmentSummaryMetrics.html#TOTAL_READS">TOTAL_READS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/AlignmentSummaryMetrics.html" title="class in picard.analysis">AlignmentSummaryMetrics</a></dt>
<dd>
<div class="block">The total number of reads including all PF and non-PF reads.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetrics.html#TOTAL_READS">TOTAL_READS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectQualityYieldMetrics.QualityYieldMetrics.html" title="class in picard.analysis">CollectQualityYieldMetrics.QualityYieldMetrics</a></dt>
<dd>
<div class="block">The total number of reads in the input file</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#TOTAL_READS">TOTAL_READS</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The total number of reads in the SAM or BAM file examined.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#TOTAL_READS">TOTAL_READS</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The total number of reads in the SAM or BAM file examined</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallingMetrics.html#TOTAL_READS">TOTAL_READS</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallingMetrics.html" title="class in picard.illumina">IlluminaBasecallingMetrics</a></dt>
<dd>
<div class="block">The total number of reads assigned to the index.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html#TOTAL_SITES">TOTAL_SITES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.CpcgMetrics.html" title="class in picard.analysis">CollectOxoGMetrics.CpcgMetrics</a></dt>
<dd>
<div class="block">The total number of sites that had at least one base covering them.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RrbsCpgDetailMetrics.html#TOTAL_SITES">TOTAL_SITES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RrbsCpgDetailMetrics.html" title="class in picard.analysis">RrbsCpgDetailMetrics</a></dt>
<dd>
<div class="block">Number of times this CpG site was encountered</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html#TOTAL_SNPS">TOTAL_SNPS</a></span> - Variable in class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingSummaryMetrics</a></dt>
<dd>
<div class="block">The number of passing bi-allelic SNPs calls (i.e.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadStructure.html#totalCycles">totalCycles</a></span> - Variable in class picard.illumina.parser.<a href="picard/illumina/parser/ReadStructure.html" title="class in picard.illumina.parser">ReadStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceContingencyMetrics.html#TP_COUNT">TP_COUNT</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceContingencyMetrics.html" title="class in picard.vcf">GenotypeConcordanceContingencyMetrics</a></dt>
<dd>
<div class="block">The TP (true positive) count across all variants</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceScheme.html#TP_FN">TP_FN</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceScheme.html" title="class in picard.vcf">GenotypeConcordanceScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceScheme.html#TP_FP">TP_FP</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceScheme.html" title="class in picard.vcf">GenotypeConcordanceScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceScheme.html#TP_FP_FN">TP_FP_FN</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceScheme.html" title="class in picard.vcf">GenotypeConcordanceScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceScheme.html#TP_ONLY">TP_ONLY</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceScheme.html" title="class in picard.vcf">GenotypeConcordanceScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceScheme.html#TP_TN">TP_TN</a></span> - Static variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceScheme.html" title="class in picard.vcf">GenotypeConcordanceScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html#trackOpticalDuplicates-java.util.List-picard.sam.markduplicates.util.ReadEnds-picard.sam.markduplicates.util.OpticalDuplicateFinder-picard.sam.markduplicates.util.LibraryIdGenerator-">trackOpticalDuplicates(List&lt;? extends ReadEnds&gt;, ReadEnds, OpticalDuplicateFinder, LibraryIdGenerator)</a></span> - Static method in class picard.sam.markduplicates.util.<a href="picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.html" title="class in picard.sam.markduplicates.util">AbstractMarkDuplicatesCommandLineProgram</a></dt>
<dd>
<div class="block">Looks through the set of reads and identifies how many of the duplicates are
 in fact optical duplicates, and stores the data in the instance level histogram.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/Gene.Transcript.html#Transcript-java.lang.String-int-int-int-int-int-">Transcript(String, int, int, int, int, int)</a></span> - Constructor for class picard.annotation.<a href="picard/annotation/Gene.Transcript.html" title="class in picard.annotation">Gene.Transcript</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/Gene.Transcript.html#transcriptionEnd">transcriptionEnd</a></span> - Variable in class picard.annotation.<a href="picard/annotation/Gene.Transcript.html" title="class in picard.annotation">Gene.Transcript</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/Gene.Transcript.html#transcriptionStart">transcriptionStart</a></span> - Variable in class picard.annotation.<a href="picard/annotation/Gene.Transcript.html" title="class in picard.annotation">Gene.Transcript</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineSyntaxTranslater.html#translatePicardStyleToPosixStyle-java.lang.String:A-">translatePicardStyleToPosixStyle(String[])</a></span> - Static method in class picard.cmdline.<a href="picard/cmdline/CommandLineSyntaxTranslater.html" title="class in picard.cmdline">CommandLineSyntaxTranslater</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/FindMendelianViolations.html#TRIOS">TRIOS</a></span> - Variable in class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/FindMendelianViolations.html" title="class in picard.vcf.MendelianViolations">FindMendelianViolations</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#triSiteHeterogeneityRate">triSiteHeterogeneityRate</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/IndependentReplicateMetric.html#triSiteHomogeneityRate">triSiteHomogeneityRate</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/IndependentReplicateMetric.html" title="class in picard.analysis.replicates">IndependentReplicateMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CreateSequenceDictionary.html#TRUNCATE_NAMES_AT_WHITESPACE">TRUNCATE_NAMES_AT_WHITESPACE</a></span> - Variable in class picard.sam.<a href="picard/sam/CreateSequenceDictionary.html" title="class in picard.sam">CreateSequenceDictionary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/reference/NormalizeFasta.html#TRUNCATE_SEQUENCE_NAMES_AT_WHITESPACE">TRUNCATE_SEQUENCE_NAMES_AT_WHITESPACE</a></span> - Variable in class picard.reference.<a href="picard/reference/NormalizeFasta.html" title="class in picard.reference">NormalizeFasta</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#TRUTH_SAMPLE">TRUTH_SAMPLE</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceContingencyMetrics.html#TRUTH_SAMPLE">TRUTH_SAMPLE</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceContingencyMetrics.html" title="class in picard.vcf">GenotypeConcordanceContingencyMetrics</a></dt>
<dd>
<div class="block">The name of the 'truth' sample</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceDetailMetrics.html#TRUTH_SAMPLE">TRUTH_SAMPLE</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceDetailMetrics.html" title="class in picard.vcf">GenotypeConcordanceDetailMetrics</a></dt>
<dd>
<div class="block">The name of the 'truth' sample</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html#TRUTH_SAMPLE">TRUTH_SAMPLE</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html" title="class in picard.vcf">GenotypeConcordanceSummaryMetrics</a></dt>
<dd>
<div class="block">The name of the 'truth' sample</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceDetailMetrics.html#TRUTH_STATE">TRUTH_STATE</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceDetailMetrics.html" title="class in picard.vcf">GenotypeConcordanceDetailMetrics</a></dt>
<dd>
<div class="block">The state of the 'truth' sample (i.e.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#TRUTH_VCF">TRUTH_VCF</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.Alleles.html#truthAllele1">truthAllele1</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.Alleles.html" title="class in picard.vcf">GenotypeConcordance.Alleles</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.Alleles.html#truthAllele2">truthAllele2</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.Alleles.html" title="class in picard.vcf">GenotypeConcordance.Alleles</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.Alleles.html#truthAlleles--">truthAlleles()</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.Alleles.html" title="class in picard.vcf">GenotypeConcordance.Alleles</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceStates.TruthAndCallStates.html#TruthAndCallStates-picard.vcf.GenotypeConcordanceStates.TruthState-picard.vcf.GenotypeConcordanceStates.CallState-">TruthAndCallStates(GenotypeConcordanceStates.TruthState, GenotypeConcordanceStates.CallState)</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/GenotypeConcordanceStates.TruthAndCallStates.html" title="class in picard.vcf">GenotypeConcordanceStates.TruthAndCallStates</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceStates.TruthAndCallStates.html#truthState">truthState</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceStates.TruthAndCallStates.html" title="class in picard.vcf">GenotypeConcordanceStates.TruthAndCallStates</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadDescriptor.html#type">type</a></span> - Variable in class picard.illumina.parser.<a href="picard/illumina/parser/ReadDescriptor.html" title="class in picard.illumina.parser">ReadDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaPhasingMetrics.html#TYPE_NAME">TYPE_NAME</a></span> - Variable in class picard.illumina.<a href="picard/illumina/IlluminaPhasingMetrics.html" title="class in picard.illumina">IlluminaPhasingMetrics</a></dt>
<dd>
<div class="block">Defines an Illumina template read number (first or second)</div>
</dd>
</dl>
<a name="I:U">
<!--   -->
</a>
<h2 class="title">U</h2>
<dl>
<dt><span class="memberNameLink"><a href="picard/util/UnsignedTypeUtil.html#uByteToInt-byte-">uByteToInt(byte)</a></span> - Static method in class picard.util.<a href="picard/util/UnsignedTypeUtil.html" title="class in picard.util">UnsignedTypeUtil</a></dt>
<dd>
<div class="block">Convert an unsigned byte to a signed int</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/UnsignedTypeUtil.html#uByteToShort-byte-">uByteToShort(byte)</a></span> - Static method in class picard.util.<a href="picard/util/UnsignedTypeUtil.html" title="class in picard.util">UnsignedTypeUtil</a></dt>
<dd>
<div class="block">Convert an unsigned byte to a signed short</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/UnsignedTypeUtil.html#uIntToFloat-int-">uIntToFloat(int)</a></span> - Static method in class picard.util.<a href="picard/util/UnsignedTypeUtil.html" title="class in picard.util">UnsignedTypeUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/UnsignedTypeUtil.html#uIntToLong-int-">uIntToLong(int)</a></span> - Static method in class picard.util.<a href="picard/util/UnsignedTypeUtil.html" title="class in picard.util">UnsignedTypeUtil</a></dt>
<dd>
<div class="block">Convert an unsigned int to a long</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/UmiMetrics.html#UMI_BASE_QUALITIES">UMI_BASE_QUALITIES</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/UmiMetrics.html" title="class in picard.sam.markduplicates">UmiMetrics</a></dt>
<dd>
<div class="block">Estimation of Phred scaled quality scores for UMIs</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/UmiAwareMarkDuplicatesWithMateCigar.html#UMI_METRICS_FILE">UMI_METRICS_FILE</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/UmiAwareMarkDuplicatesWithMateCigar.html" title="class in picard.sam.markduplicates">UmiAwareMarkDuplicatesWithMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/UmiAwareMarkDuplicatesWithMateCigar.html#UMI_TAG_NAME">UMI_TAG_NAME</a></span> - Variable in class picard.sam.markduplicates.<a href="picard/sam/markduplicates/UmiAwareMarkDuplicatesWithMateCigar.html" title="class in picard.sam.markduplicates">UmiAwareMarkDuplicatesWithMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/markduplicates/UmiAwareMarkDuplicatesWithMateCigar.html" title="class in picard.sam.markduplicates"><span class="typeNameLink">UmiAwareMarkDuplicatesWithMateCigar</span></a> - Class in <a href="picard/sam/markduplicates/package-summary.html">picard.sam.markduplicates</a></dt>
<dd>
<div class="block">This is a simple tool to mark duplicates making use of UMIs in the reads.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/UmiAwareMarkDuplicatesWithMateCigar.html#UmiAwareMarkDuplicatesWithMateCigar--">UmiAwareMarkDuplicatesWithMateCigar()</a></span> - Constructor for class picard.sam.markduplicates.<a href="picard/sam/markduplicates/UmiAwareMarkDuplicatesWithMateCigar.html" title="class in picard.sam.markduplicates">UmiAwareMarkDuplicatesWithMateCigar</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/markduplicates/UmiGraph.html" title="class in picard.sam.markduplicates"><span class="typeNameLink">UmiGraph</span></a> - Class in <a href="picard/sam/markduplicates/package-summary.html">picard.sam.markduplicates</a></dt>
<dd>
<div class="block">UmiGraph is used to identify UMIs that come from the same original source molecule.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/UmiGraph.html#UmiGraph-htsjdk.samtools.DuplicateSet-java.lang.String-java.lang.String-boolean-">UmiGraph(DuplicateSet, String, String, boolean)</a></span> - Constructor for class picard.sam.markduplicates.<a href="picard/sam/markduplicates/UmiGraph.html" title="class in picard.sam.markduplicates">UmiGraph</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/markduplicates/UmiMetrics.html" title="class in picard.sam.markduplicates"><span class="typeNameLink">UmiMetrics</span></a> - Class in <a href="picard/sam/markduplicates/package-summary.html">picard.sam.markduplicates</a></dt>
<dd>
<div class="block">Metrics that are calculated during the process of marking duplicates
 within a stream of SAMRecords using the UmiAwareDuplicateSetIterator.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/UmiMetrics.html#UmiMetrics--">UmiMetrics()</a></span> - Constructor for class picard.sam.markduplicates.<a href="picard/sam/markduplicates/UmiMetrics.html" title="class in picard.sam.markduplicates">UmiMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/UmiMetrics.html#UmiMetrics-double-int-int-int-int-int-double-double-double-">UmiMetrics(double, int, int, int, int, int, double, double, double)</a></span> - Constructor for class picard.sam.markduplicates.<a href="picard/sam/markduplicates/UmiMetrics.html" title="class in picard.sam.markduplicates">UmiMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#unfilteredBaseQHistogram">unfilteredBaseQHistogram</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">The count of bases observed with a given base quality.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AbstractWgsMetricsCollector.html#unfilteredBaseQHistogramArray">unfilteredBaseQHistogramArray</a></span> - Variable in class picard.analysis.<a href="picard/analysis/AbstractWgsMetricsCollector.html" title="class in picard.analysis">AbstractWgsMetricsCollector</a></dt>
<dd>
<div class="block">Count of bases observed with a given base quality.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#unfilteredDepthHistogram">unfilteredDepthHistogram</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">Count of sites with a given depth of coverage.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AbstractWgsMetricsCollector.html#unfilteredDepthHistogramArray">unfilteredDepthHistogramArray</a></span> - Variable in class picard.analysis.<a href="picard/analysis/AbstractWgsMetricsCollector.html" title="class in picard.analysis">AbstractWgsMetricsCollector</a></dt>
<dd>
<div class="block">Count of sites with a given depth of coverage.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/BedToIntervalList.html#UNIQUE">UNIQUE</a></span> - Variable in class picard.util.<a href="picard/util/BedToIntervalList.html" title="class in picard.util">BedToIntervalList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/util/IntervalListTools.html#UNIQUE">UNIQUE</a></span> - Variable in class picard.util.<a href="picard/util/IntervalListTools.html" title="class in picard.util">IntervalListTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/metrics/MultiLevelCollector.html#UNKNOWN">UNKNOWN</a></span> - Static variable in class picard.metrics.<a href="picard/metrics/MultiLevelCollector.html" title="class in picard.metrics">MultiLevelCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/pedigree/PedFile.html#UNKNOWN_SEX">UNKNOWN_SEX</a></span> - Static variable in class picard.pedigree.<a href="picard/pedigree/PedFile.html" title="class in picard.pedigree">PedFile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#UNMAP_CONTAMINANT_READS">UNMAP_CONTAMINANT_READS</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#UNMAPPED_BAM">UNMAPPED_BAM</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeBamAlignment.html#UNMAPPED_READ_STRATEGY">UNMAPPED_READ_STRATEGY</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeBamAlignment.html" title="class in picard.sam">MergeBamAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/DuplicationMetrics.html#UNMAPPED_READS">UNMAPPED_READS</a></span> - Variable in class picard.sam.<a href="picard/sam/DuplicationMetrics.html" title="class in picard.sam">DuplicationMetrics</a></dt>
<dd>
<div class="block">The total number of unmapped reads examined.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/SamToFastq.html#UNPAIRED_FASTQ">UNPAIRED_FASTQ</a></span> - Variable in class picard.sam.<a href="picard/sam/SamToFastq.html" title="class in picard.sam">SamToFastq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/DuplicationMetrics.html#UNPAIRED_READ_DUPLICATES">UNPAIRED_READ_DUPLICATES</a></span> - Variable in class picard.sam.<a href="picard/sam/DuplicationMetrics.html" title="class in picard.sam">DuplicationMetrics</a></dt>
<dd>
<div class="block">The number of fragments that were marked as duplicates.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/DuplicationMetrics.html#UNPAIRED_READS_EXAMINED">UNPAIRED_READS_EXAMINED</a></span> - Variable in class picard.sam.<a href="picard/sam/DuplicationMetrics.html" title="class in picard.sam">DuplicationMetrics</a></dt>
<dd>
<div class="block">The number of mapped reads examined which did not have a mapped mate pair,
 either because the read is unpaired, or the read is paired to an unmapped mate.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.html#unsafeNextInfo--">unsafeNextInfo()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.html" title="class in picard.illumina.parser.readers">AbstractIlluminaPositionFileReader</a></dt>
<dd>
<div class="block">Returns the next position info.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/ClocsFileReader.html#unsafeNextInfo--">unsafeNextInfo()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/ClocsFileReader.html" title="class in picard.illumina.parser.readers">ClocsFileReader</a></dt>
<dd>
<div class="block">Grab the next set of offset values, decompress them.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/LocsFileReader.html#unsafeNextInfo--">unsafeNextInfo()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/LocsFileReader.html" title="class in picard.illumina.parser.readers">LocsFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/PosFileReader.html#unsafeNextInfo--">unsafeNextInfo()</a></span> - Method in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/PosFileReader.html" title="class in picard.illumina.parser.readers">PosFileReader</a></dt>
<dd>
<div class="block">Read a line of text and parse it into two float values, create a PositionInfo and return it</div>
</dd>
<dt><a href="picard/util/UnsignedTypeUtil.html" title="class in picard.util"><span class="typeNameLink">UnsignedTypeUtil</span></a> - Class in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">A utility class for dealing with unsigned types.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/UnsignedTypeUtil.html#UnsignedTypeUtil--">UnsignedTypeUtil()</a></span> - Constructor for class picard.util.<a href="picard/util/UnsignedTypeUtil.html" title="class in picard.util">UnsignedTypeUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.html#updateCigarForTrimmedOrClippedBases-htsjdk.samtools.SAMRecord-htsjdk.samtools.SAMRecord-">updateCigarForTrimmedOrClippedBases(SAMRecord, SAMRecord)</a></span> - Method in class picard.sam.<a href="picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/UpdateVcfSequenceDictionary.html" title="class in picard.vcf"><span class="typeNameLink">UpdateVcfSequenceDictionary</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">Takes a VCF file and a Sequence Dictionary (from a variety of file types) and updates the Sequence Dictionary in VCF.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/UpdateVcfSequenceDictionary.html#UpdateVcfSequenceDictionary--">UpdateVcfSequenceDictionary()</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/UpdateVcfSequenceDictionary.html" title="class in picard.vcf">UpdateVcfSequenceDictionary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/CreateSequenceDictionary.html#URI">URI</a></span> - Variable in class picard.sam.<a href="picard/sam/CreateSequenceDictionary.html" title="class in picard.sam">CreateSequenceDictionary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToSam.html#USAGE">USAGE</a></span> - Static variable in class picard.illumina.<a href="picard/illumina/IlluminaBasecallsToSam.html" title="class in picard.illumina">IlluminaBasecallsToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ViewSam.html#USAGE">USAGE</a></span> - Variable in class picard.sam.<a href="picard/sam/ViewSam.html" title="class in picard.sam">ViewSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.html#USE_FAST_ALGORITHM">USE_FAST_ALGORITHM</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#USE_JDK_DEFLATER">USE_JDK_DEFLATER</a></span> - Variable in class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#USE_JDK_INFLATER">USE_JDK_INFLATER</a></span> - Variable in class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html#USE_OQ">USE_OQ</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html" title="class in picard.analysis.artifacts">CollectSequencingArtifactMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectOxoGMetrics.html#USE_OQ">USE_OQ</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectOxoGMetrics.html" title="class in picard.analysis">CollectOxoGMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectQualityYieldMetrics.html#USE_ORIGINAL_QUALITIES">USE_ORIGINAL_QUALITIES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/CollectQualityYieldMetrics.html" title="class in picard.analysis">CollectQualityYieldMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/StandardOptionDefinitions.html#USE_ORIGINAL_QUALITIES_SHORT_NAME">USE_ORIGINAL_QUALITIES_SHORT_NAME</a></span> - Static variable in class picard.cmdline.<a href="picard/cmdline/StandardOptionDefinitions.html" title="class in picard.cmdline">StandardOptionDefinitions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FastqToSam.html#USE_SEQUENTIAL_FASTQS">USE_SEQUENTIAL_FASTQS</a></span> - Variable in class picard.sam.<a href="picard/sam/FastqToSam.html" title="class in picard.sam">FastqToSam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/MergeSamFiles.html#USE_THREADING">USE_THREADING</a></span> - Variable in class picard.sam.<a href="picard/sam/MergeSamFiles.html" title="class in picard.sam">MergeSamFiles</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordance.html#USE_VCF_INDEX">USE_VCF_INDEX</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordance.html" title="class in picard.vcf">GenotypeConcordance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#useLegacyParser-java.lang.Class-">useLegacyParser(Class&lt;?&gt;)</a></span> - Static method in class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>
<div class="block">Return true if the Picard command line parser should be used in place of the Barclay command line parser,
 otherwise, false.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html#usesNoRefReads--">usesNoRefReads()</a></span> - Method in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/CollectSequencingArtifactMetrics.html" title="class in picard.analysis.artifacts">CollectSequencingArtifactMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectInsertSizeMetrics.html#usesNoRefReads--">usesNoRefReads()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectInsertSizeMetrics.html" title="class in picard.analysis">CollectInsertSizeMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectQualityYieldMetrics.html#usesNoRefReads--">usesNoRefReads()</a></span> - Method in class picard.analysis.<a href="picard/analysis/CollectQualityYieldMetrics.html" title="class in picard.analysis">CollectQualityYieldMetrics</a></dt>
<dd>
<div class="block">Ensure that we get all reads regardless of alignment status.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/SinglePassSamProgram.html#usesNoRefReads--">usesNoRefReads()</a></span> - Method in class picard.analysis.<a href="picard/analysis/SinglePassSamProgram.html" title="class in picard.analysis">SinglePassSamProgram</a></dt>
<dd>
<div class="block">Can be overridden and set to false if the section of unmapped reads at the end of the file isn't needed.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/UnsignedTypeUtil.html#uShortToInt-short-">uShortToInt(short)</a></span> - Static method in class picard.util.<a href="picard/util/UnsignedTypeUtil.html" title="class in picard.util">UnsignedTypeUtil</a></dt>
<dd>
<div class="block">Convert an unsigned short to an Int</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/RnaSeqMetrics.html#UTR_BASES">UTR_BASES</a></span> - Variable in class picard.analysis.<a href="picard/analysis/RnaSeqMetrics.html" title="class in picard.analysis">RnaSeqMetrics</a></dt>
<dd>
<div class="block">Number of bases in primary alignments that align to a UTR base for some gene, and not a coding base.</div>
</dd>
</dl>
<a name="I:V">
<!--   -->
</a>
<h2 class="title">V</h2>
<dl>
<dt><span class="memberNameLink"><a href="picard/sam/ValidateSamFile.html#VALIDATE_INDEX">VALIDATE_INDEX</a></span> - Variable in class picard.sam.<a href="picard/sam/ValidateSamFile.html" title="class in picard.sam">ValidateSamFile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceCounts.html#validateCountsAgainstScheme-picard.vcf.GenotypeConcordanceScheme-">validateCountsAgainstScheme(GenotypeConcordanceScheme)</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceCounts.html" title="class in picard.vcf">GenotypeConcordanceCounts</a></dt>
<dd>
<div class="block">Validates that there are no counts for NA states in the underlying scheme</div>
</dd>
<dt><a href="picard/sam/ValidateSamFile.html" title="class in picard.sam"><span class="typeNameLink">ValidateSamFile</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block">Command line program wrapping SamFileValidator.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ValidateSamFile.html#ValidateSamFile--">ValidateSamFile()</a></span> - Constructor for class picard.sam.<a href="picard/sam/ValidateSamFile.html" title="class in picard.sam">ValidateSamFile</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/ValidateSamFile.Mode.html" title="enum in picard.sam"><span class="typeNameLink">ValidateSamFile.Mode</span></a> - Enum in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceScheme.html#validateScheme--">validateScheme()</a></span> - Method in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceScheme.html" title="class in picard.vcf">GenotypeConcordanceScheme</a></dt>
<dd>
<div class="block">Check that all cells in the scheme exist.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#VALIDATION_STRINGENCY">VALIDATION_STRINGENCY</a></span> - Variable in class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AlignmentSummaryMetrics.Category.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.analysis.<a href="picard/analysis/AlignmentSummaryMetrics.Category.html" title="enum in picard.analysis">AlignmentSummaryMetrics.Category</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectMultipleMetrics.Program.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.analysis.<a href="picard/analysis/CollectMultipleMetrics.Program.html" title="enum in picard.analysis">CollectMultipleMetrics.Program</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverage.Category.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.analysis.<a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverage.Category.html" title="enum in picard.analysis">CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverage.Category</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/RnaSeqMetricsCollector.StrandSpecificity.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.analysis.directed.<a href="picard/analysis/directed/RnaSeqMetricsCollector.StrandSpecificity.html" title="enum in picard.analysis.directed">RnaSeqMetricsCollector.StrandSpecificity</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/MetricAccumulationLevel.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.analysis.<a href="picard/analysis/MetricAccumulationLevel.html" title="enum in picard.analysis">MetricAccumulationLevel</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/LocusFunction.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.annotation.<a href="picard/annotation/LocusFunction.html" title="enum in picard.annotation">LocusFunction</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/RefFlatReader.RefFlatColumns.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.annotation.<a href="picard/annotation/RefFlatReader.RefFlatColumns.html" title="enum in picard.annotation">RefFlatReader.RefFlatColumns</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckMetric.DataType.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.fingerprint.<a href="picard/fingerprint/CrosscheckMetric.DataType.html" title="enum in picard.fingerprint">CrosscheckMetric.DataType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckMetric.FingerprintResult.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.fingerprint.<a href="picard/fingerprint/CrosscheckMetric.FingerprintResult.html" title="enum in picard.fingerprint">CrosscheckMetric.FingerprintResult</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/DiploidGenotype.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.fingerprint.<a href="picard/fingerprint/DiploidGenotype.html" title="enum in picard.fingerprint">DiploidGenotype</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/DiploidHaplotype.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.fingerprint.<a href="picard/fingerprint/DiploidHaplotype.html" title="enum in picard.fingerprint">DiploidHaplotype</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilities.Genotype.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilities.Genotype.html" title="enum in picard.fingerprint">HaplotypeProbabilities.Genotype</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToFastq.ReadNameFormat.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.illumina.<a href="picard/illumina/IlluminaBasecallsToFastq.ReadNameFormat.html" title="enum in picard.illumina">IlluminaBasecallsToFastq.ReadNameFormat</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaDataType.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.illumina.parser.<a href="picard/illumina/parser/IlluminaDataType.html" title="enum in picard.illumina.parser">IlluminaDataType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaFileUtil.SupportedIlluminaFormat.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.illumina.parser.<a href="picard/illumina/parser/IlluminaFileUtil.SupportedIlluminaFormat.html" title="enum in picard.illumina.parser">IlluminaFileUtil.SupportedIlluminaFormat</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaMetricsCode.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.illumina.parser.<a href="picard/illumina/parser/IlluminaMetricsCode.html" title="enum in picard.illumina.parser">IlluminaMetricsCode</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IntensityChannel.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.illumina.parser.<a href="picard/illumina/parser/IntensityChannel.html" title="enum in picard.illumina.parser">IntensityChannel</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/TileMetricsOutReader.TileMetricsVersion.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/TileMetricsOutReader.TileMetricsVersion.html" title="enum in picard.illumina.parser.readers">TileMetricsOutReader.TileMetricsVersion</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadType.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.illumina.parser.<a href="picard/illumina/parser/ReadType.html" title="enum in picard.illumina.parser">ReadType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/TileTemplateRead.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.illumina.parser.<a href="picard/illumina/parser/TileTemplateRead.html" title="enum in picard.illumina.parser">TileTemplateRead</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.ReadClassifier.PfFailReason.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.ReadClassifier.PfFailReason.html" title="enum in picard.illumina.quality">CollectHiSeqXPfFailMetrics.ReadClassifier.PfFailReason</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/pedigree/Sex.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.pedigree.<a href="picard/pedigree/Sex.html" title="enum in picard.pedigree">Sex</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.UnmappingReadStrategy.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.sam.<a href="picard/sam/AbstractAlignmentMerger.UnmappingReadStrategy.html" title="enum in picard.sam">AbstractAlignmentMerger.UnmappingReadStrategy</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FilterSamReads.Filter.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.sam.<a href="picard/sam/FilterSamReads.Filter.html" title="enum in picard.sam">FilterSamReads.Filter</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicates.DuplicateTaggingPolicy.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicates.DuplicateTaggingPolicy.html" title="enum in picard.sam.markduplicates">MarkDuplicates.DuplicateTaggingPolicy</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicates.DuplicateType.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicates.DuplicateType.html" title="enum in picard.sam.markduplicates">MarkDuplicates.DuplicateType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertOriginalBaseQualitiesAndAddMateCigar.CanSkipSamFile.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.sam.<a href="picard/sam/RevertOriginalBaseQualitiesAndAddMateCigar.CanSkipSamFile.html" title="enum in picard.sam">RevertOriginalBaseQualitiesAndAddMateCigar.CanSkipSamFile</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertSam.FileType.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.sam.<a href="picard/sam/RevertSam.FileType.html" title="enum in picard.sam">RevertSam.FileType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ValidateSamFile.Mode.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.sam.<a href="picard/sam/ValidateSamFile.Mode.html" title="enum in picard.sam">ValidateSamFile.Mode</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ViewSam.AlignmentStatus.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.sam.<a href="picard/sam/ViewSam.AlignmentStatus.html" title="enum in picard.sam">ViewSam.AlignmentStatus</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ViewSam.PfStatus.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.sam.<a href="picard/sam/ViewSam.PfStatus.html" title="enum in picard.sam">ViewSam.PfStatus</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/BaitDesigner.DesignStrategy.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.util.<a href="picard/util/BaitDesigner.DesignStrategy.html" title="enum in picard.util">BaitDesigner.DesignStrategy</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/IlluminaUtil.IlluminaAdapterPair.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.util.<a href="picard/util/IlluminaUtil.IlluminaAdapterPair.html" title="enum in picard.util">IlluminaUtil.IlluminaAdapterPair</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/IntervalListScatterer.Mode.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.util.<a href="picard/util/IntervalListScatterer.Mode.html" title="enum in picard.util">IntervalListScatterer.Mode</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/IntervalListTools.Action.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.util.<a href="picard/util/IntervalListTools.Action.html" title="enum in picard.util">IntervalListTools.Action</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/VariantType.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.util.<a href="picard/util/VariantType.html" title="enum in picard.util">VariantType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceStateCodes.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.vcf.<a href="picard/vcf/GenotypeConcordanceStateCodes.html" title="enum in picard.vcf">GenotypeConcordanceStateCodes</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceStates.CallState.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.vcf.<a href="picard/vcf/GenotypeConcordanceStates.CallState.html" title="enum in picard.vcf">GenotypeConcordanceStates.CallState</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceStates.ContingencyState.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.vcf.<a href="picard/vcf/GenotypeConcordanceStates.ContingencyState.html" title="enum in picard.vcf">GenotypeConcordanceStates.ContingencyState</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceStates.TruthState.html#valueOf-java.lang.String-">valueOf(String)</a></span> - Static method in enum picard.vcf.<a href="picard/vcf/GenotypeConcordanceStates.TruthState.html" title="enum in picard.vcf">GenotypeConcordanceStates.TruthState</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/AlignmentSummaryMetrics.Category.html#values--">values()</a></span> - Static method in enum picard.analysis.<a href="picard/analysis/AlignmentSummaryMetrics.Category.html" title="enum in picard.analysis">AlignmentSummaryMetrics.Category</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectMultipleMetrics.Program.html#values--">values()</a></span> - Static method in enum picard.analysis.<a href="picard/analysis/CollectMultipleMetrics.Program.html" title="enum in picard.analysis">CollectMultipleMetrics.Program</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverage.Category.html#values--">values()</a></span> - Static method in enum picard.analysis.<a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverage.Category.html" title="enum in picard.analysis">CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverage.Category</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/RnaSeqMetricsCollector.StrandSpecificity.html#values--">values()</a></span> - Static method in enum picard.analysis.directed.<a href="picard/analysis/directed/RnaSeqMetricsCollector.StrandSpecificity.html" title="enum in picard.analysis.directed">RnaSeqMetricsCollector.StrandSpecificity</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/MetricAccumulationLevel.html#values--">values()</a></span> - Static method in enum picard.analysis.<a href="picard/analysis/MetricAccumulationLevel.html" title="enum in picard.analysis">MetricAccumulationLevel</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/LocusFunction.html#values--">values()</a></span> - Static method in enum picard.annotation.<a href="picard/annotation/LocusFunction.html" title="enum in picard.annotation">LocusFunction</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/annotation/RefFlatReader.RefFlatColumns.html#values--">values()</a></span> - Static method in enum picard.annotation.<a href="picard/annotation/RefFlatReader.RefFlatColumns.html" title="enum in picard.annotation">RefFlatReader.RefFlatColumns</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckMetric.DataType.html#values--">values()</a></span> - Static method in enum picard.fingerprint.<a href="picard/fingerprint/CrosscheckMetric.DataType.html" title="enum in picard.fingerprint">CrosscheckMetric.DataType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/CrosscheckMetric.FingerprintResult.html#values--">values()</a></span> - Static method in enum picard.fingerprint.<a href="picard/fingerprint/CrosscheckMetric.FingerprintResult.html" title="enum in picard.fingerprint">CrosscheckMetric.FingerprintResult</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/DiploidGenotype.html#values--">values()</a></span> - Static method in enum picard.fingerprint.<a href="picard/fingerprint/DiploidGenotype.html" title="enum in picard.fingerprint">DiploidGenotype</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/DiploidHaplotype.html#values--">values()</a></span> - Static method in enum picard.fingerprint.<a href="picard/fingerprint/DiploidHaplotype.html" title="enum in picard.fingerprint">DiploidHaplotype</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeProbabilities.Genotype.html#values--">values()</a></span> - Static method in enum picard.fingerprint.<a href="picard/fingerprint/HaplotypeProbabilities.Genotype.html" title="enum in picard.fingerprint">HaplotypeProbabilities.Genotype</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/IlluminaBasecallsToFastq.ReadNameFormat.html#values--">values()</a></span> - Static method in enum picard.illumina.<a href="picard/illumina/IlluminaBasecallsToFastq.ReadNameFormat.html" title="enum in picard.illumina">IlluminaBasecallsToFastq.ReadNameFormat</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaDataType.html#values--">values()</a></span> - Static method in enum picard.illumina.parser.<a href="picard/illumina/parser/IlluminaDataType.html" title="enum in picard.illumina.parser">IlluminaDataType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaFileUtil.SupportedIlluminaFormat.html#values--">values()</a></span> - Static method in enum picard.illumina.parser.<a href="picard/illumina/parser/IlluminaFileUtil.SupportedIlluminaFormat.html" title="enum in picard.illumina.parser">IlluminaFileUtil.SupportedIlluminaFormat</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IlluminaMetricsCode.html#values--">values()</a></span> - Static method in enum picard.illumina.parser.<a href="picard/illumina/parser/IlluminaMetricsCode.html" title="enum in picard.illumina.parser">IlluminaMetricsCode</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/IntensityChannel.html#values--">values()</a></span> - Static method in enum picard.illumina.parser.<a href="picard/illumina/parser/IntensityChannel.html" title="enum in picard.illumina.parser">IntensityChannel</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/TileMetricsOutReader.TileMetricsVersion.html#values--">values()</a></span> - Static method in enum picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/TileMetricsOutReader.TileMetricsVersion.html" title="enum in picard.illumina.parser.readers">TileMetricsOutReader.TileMetricsVersion</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ReadType.html#values--">values()</a></span> - Static method in enum picard.illumina.parser.<a href="picard/illumina/parser/ReadType.html" title="enum in picard.illumina.parser">ReadType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/TileTemplateRead.html#values--">values()</a></span> - Static method in enum picard.illumina.parser.<a href="picard/illumina/parser/TileTemplateRead.html" title="enum in picard.illumina.parser">TileTemplateRead</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.ReadClassifier.PfFailReason.html#values--">values()</a></span> - Static method in enum picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.ReadClassifier.PfFailReason.html" title="enum in picard.illumina.quality">CollectHiSeqXPfFailMetrics.ReadClassifier.PfFailReason</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/pedigree/Sex.html#values--">values()</a></span> - Static method in enum picard.pedigree.<a href="picard/pedigree/Sex.html" title="enum in picard.pedigree">Sex</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/AbstractAlignmentMerger.UnmappingReadStrategy.html#values--">values()</a></span> - Static method in enum picard.sam.<a href="picard/sam/AbstractAlignmentMerger.UnmappingReadStrategy.html" title="enum in picard.sam">AbstractAlignmentMerger.UnmappingReadStrategy</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FilterSamReads.Filter.html#values--">values()</a></span> - Static method in enum picard.sam.<a href="picard/sam/FilterSamReads.Filter.html" title="enum in picard.sam">FilterSamReads.Filter</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicates.DuplicateTaggingPolicy.html#values--">values()</a></span> - Static method in enum picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicates.DuplicateTaggingPolicy.html" title="enum in picard.sam.markduplicates">MarkDuplicates.DuplicateTaggingPolicy</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/markduplicates/MarkDuplicates.DuplicateType.html#values--">values()</a></span> - Static method in enum picard.sam.markduplicates.<a href="picard/sam/markduplicates/MarkDuplicates.DuplicateType.html" title="enum in picard.sam.markduplicates">MarkDuplicates.DuplicateType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertOriginalBaseQualitiesAndAddMateCigar.CanSkipSamFile.html#values--">values()</a></span> - Static method in enum picard.sam.<a href="picard/sam/RevertOriginalBaseQualitiesAndAddMateCigar.CanSkipSamFile.html" title="enum in picard.sam">RevertOriginalBaseQualitiesAndAddMateCigar.CanSkipSamFile</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/RevertSam.FileType.html#values--">values()</a></span> - Static method in enum picard.sam.<a href="picard/sam/RevertSam.FileType.html" title="enum in picard.sam">RevertSam.FileType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ValidateSamFile.Mode.html#values--">values()</a></span> - Static method in enum picard.sam.<a href="picard/sam/ValidateSamFile.Mode.html" title="enum in picard.sam">ValidateSamFile.Mode</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ViewSam.AlignmentStatus.html#values--">values()</a></span> - Static method in enum picard.sam.<a href="picard/sam/ViewSam.AlignmentStatus.html" title="enum in picard.sam">ViewSam.AlignmentStatus</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ViewSam.PfStatus.html#values--">values()</a></span> - Static method in enum picard.sam.<a href="picard/sam/ViewSam.PfStatus.html" title="enum in picard.sam">ViewSam.PfStatus</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/BaitDesigner.DesignStrategy.html#values--">values()</a></span> - Static method in enum picard.util.<a href="picard/util/BaitDesigner.DesignStrategy.html" title="enum in picard.util">BaitDesigner.DesignStrategy</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/IlluminaUtil.IlluminaAdapterPair.html#values--">values()</a></span> - Static method in enum picard.util.<a href="picard/util/IlluminaUtil.IlluminaAdapterPair.html" title="enum in picard.util">IlluminaUtil.IlluminaAdapterPair</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/IntervalListScatterer.Mode.html#values--">values()</a></span> - Static method in enum picard.util.<a href="picard/util/IntervalListScatterer.Mode.html" title="enum in picard.util">IntervalListScatterer.Mode</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/IntervalListTools.Action.html#values--">values()</a></span> - Static method in enum picard.util.<a href="picard/util/IntervalListTools.Action.html" title="enum in picard.util">IntervalListTools.Action</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/VariantType.html#values--">values()</a></span> - Static method in enum picard.util.<a href="picard/util/VariantType.html" title="enum in picard.util">VariantType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceStateCodes.html#values--">values()</a></span> - Static method in enum picard.vcf.<a href="picard/vcf/GenotypeConcordanceStateCodes.html" title="enum in picard.vcf">GenotypeConcordanceStateCodes</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceStates.CallState.html#values--">values()</a></span> - Static method in enum picard.vcf.<a href="picard/vcf/GenotypeConcordanceStates.CallState.html" title="enum in picard.vcf">GenotypeConcordanceStates.CallState</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceStates.ContingencyState.html#values--">values()</a></span> - Static method in enum picard.vcf.<a href="picard/vcf/GenotypeConcordanceStates.ContingencyState.html" title="enum in picard.vcf">GenotypeConcordanceStates.ContingencyState</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceStates.TruthState.html#values--">values()</a></span> - Static method in enum picard.vcf.<a href="picard/vcf/GenotypeConcordanceStates.TruthState.html" title="enum in picard.vcf">GenotypeConcordanceStates.TruthState</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html#VAR_PPV">VAR_PPV</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html" title="class in picard.vcf">GenotypeConcordanceSummaryMetrics</a></dt>
<dd>
<div class="block">The ppv (positive predictive value) for all (heterozygous and homozygous) variants (PPV is the TP / (TP + FP))</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html#VAR_SENSITIVITY">VAR_SENSITIVITY</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html" title="class in picard.vcf">GenotypeConcordanceSummaryMetrics</a></dt>
<dd>
<div class="block">The sensitivity for all (heterozygous and homozygous) variants (Sensitivity is TP / (TP + FN))</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html#VAR_SPECIFICITY">VAR_SPECIFICITY</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html" title="class in picard.vcf">GenotypeConcordanceSummaryMetrics</a></dt>
<dd>
<div class="block">The specificity for all (heterozygous and homozygous) variants (Specificity is TN / (FP + TN))</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceContingencyMetrics.html#VARIANT_TYPE">VARIANT_TYPE</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceContingencyMetrics.html" title="class in picard.vcf">GenotypeConcordanceContingencyMetrics</a></dt>
<dd>
<div class="block">The type of the event (i.e.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceDetailMetrics.html#VARIANT_TYPE">VARIANT_TYPE</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceDetailMetrics.html" title="class in picard.vcf">GenotypeConcordanceDetailMetrics</a></dt>
<dd>
<div class="block">The type of the event (i.e.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html#VARIANT_TYPE">VARIANT_TYPE</a></span> - Variable in class picard.vcf.<a href="picard/vcf/GenotypeConcordanceSummaryMetrics.html" title="class in picard.vcf">GenotypeConcordanceSummaryMetrics</a></dt>
<dd>
<div class="block">The type of the event (i.e.</div>
</dd>
<dt><a href="picard/vcf/processor/VariantAccumulatorExecutor.html" title="interface in picard.vcf.processor"><span class="typeNameLink">VariantAccumulatorExecutor</span></a>&lt;<a href="picard/vcf/processor/VariantAccumulatorExecutor.html" title="type parameter in VariantAccumulatorExecutor">ACCUMULATOR</a> extends <a href="picard/vcf/processor/VariantProcessor.Accumulator.html" title="interface in picard.vcf.processor">VariantProcessor.Accumulator</a>&lt;<a href="picard/vcf/processor/VariantAccumulatorExecutor.html" title="type parameter in VariantAccumulatorExecutor">RESULT</a>&gt;,<a href="picard/vcf/processor/VariantAccumulatorExecutor.html" title="type parameter in VariantAccumulatorExecutor">RESULT</a>&gt; - Interface in <a href="picard/vcf/processor/package-summary.html">picard.vcf.processor</a></dt>
<dd>
<div class="block">Describes the functionality for an executor that manages the delegation of work to <a href="picard/vcf/processor/VariantProcessor.Accumulator.html" title="interface in picard.vcf.processor"><code>VariantProcessor.Accumulator</code></a>s.</div>
</dd>
<dt><a href="picard/vcf/processor/VariantAccumulatorExecutor.MultiThreadedChunkBased.html" title="class in picard.vcf.processor"><span class="typeNameLink">VariantAccumulatorExecutor.MultiThreadedChunkBased</span></a>&lt;<a href="picard/vcf/processor/VariantAccumulatorExecutor.MultiThreadedChunkBased.html" title="type parameter in VariantAccumulatorExecutor.MultiThreadedChunkBased">A</a> extends <a href="picard/vcf/processor/VariantProcessor.Accumulator.html" title="interface in picard.vcf.processor">VariantProcessor.Accumulator</a>&lt;<a href="picard/vcf/processor/VariantAccumulatorExecutor.MultiThreadedChunkBased.html" title="type parameter in VariantAccumulatorExecutor.MultiThreadedChunkBased">R</a>&gt;,<a href="picard/vcf/processor/VariantAccumulatorExecutor.MultiThreadedChunkBased.html" title="type parameter in VariantAccumulatorExecutor.MultiThreadedChunkBased">R</a>&gt; - Class in <a href="picard/vcf/processor/package-summary.html">picard.vcf.processor</a></dt>
<dd>
<div class="block">A <a href="picard/vcf/processor/VariantAccumulatorExecutor.html" title="interface in picard.vcf.processor"><code>VariantAccumulatorExecutor</code></a> that breaks down work into chunks described by the provided <a href="picard/vcf/processor/VariantIteratorProducer.html" title="class in picard.vcf.processor"><code>VariantIteratorProducer</code></a> and
 spreads them over the indicated number of threads.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingDetailMetrics.html#VariantCallingDetailMetrics--">VariantCallingDetailMetrics()</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingDetailMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingDetailMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html#VariantCallingSummaryMetrics--">VariantCallingSummaryMetrics()</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/CollectVariantCallingMetrics.VariantCallingSummaryMetrics.html" title="class in picard.vcf">CollectVariantCallingMetrics.VariantCallingSummaryMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/filter/VariantFilter.html" title="interface in picard.vcf.filter"><span class="typeNameLink">VariantFilter</span></a> - Interface in <a href="picard/vcf/filter/package-summary.html">picard.vcf.filter</a></dt>
<dd>
<div class="block">Interface for classes that can generate filters for VariantContexts.</div>
</dd>
<dt><a href="picard/vcf/processor/VariantIteratorProducer.html" title="class in picard.vcf.processor"><span class="typeNameLink">VariantIteratorProducer</span></a> - Class in <a href="picard/vcf/processor/package-summary.html">picard.vcf.processor</a></dt>
<dd>
<div class="block">A mechanism for iterating over <code>CloseableIterator</code> of <code>VariantContext</code>s in in some fashion, given VCF files and optionally
 an interval list.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VariantIteratorProducer.html#VariantIteratorProducer--">VariantIteratorProducer()</a></span> - Constructor for class picard.vcf.processor.<a href="picard/vcf/processor/VariantIteratorProducer.html" title="class in picard.vcf.processor">VariantIteratorProducer</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/processor/VariantProcessor.html" title="class in picard.vcf.processor"><span class="typeNameLink">VariantProcessor</span></a>&lt;<a href="picard/vcf/processor/VariantProcessor.html" title="type parameter in VariantProcessor">RESULT</a>,<a href="picard/vcf/processor/VariantProcessor.html" title="type parameter in VariantProcessor">ACCUMULATOR</a> extends <a href="picard/vcf/processor/VariantProcessor.Accumulator.html" title="interface in picard.vcf.processor">VariantProcessor.Accumulator</a>&lt;<a href="picard/vcf/processor/VariantProcessor.html" title="type parameter in VariantProcessor">RESULT</a>&gt;&gt; - Class in <a href="picard/vcf/processor/package-summary.html">picard.vcf.processor</a></dt>
<dd>
<div class="block">Describes an object that processes variants and produces a result.</div>
</dd>
<dt><a href="picard/vcf/processor/VariantProcessor.Accumulator.html" title="interface in picard.vcf.processor"><span class="typeNameLink">VariantProcessor.Accumulator</span></a>&lt;<a href="picard/vcf/processor/VariantProcessor.Accumulator.html" title="type parameter in VariantProcessor.Accumulator">RESULT</a>&gt; - Interface in <a href="picard/vcf/processor/package-summary.html">picard.vcf.processor</a></dt>
<dd>
<div class="block">Handles <code>VariantContext</code>s, and accumulates their data in some fashion internally.</div>
</dd>
<dt><a href="picard/vcf/processor/VariantProcessor.AccumulatorGenerator.html" title="interface in picard.vcf.processor"><span class="typeNameLink">VariantProcessor.AccumulatorGenerator</span></a>&lt;<a href="picard/vcf/processor/VariantProcessor.AccumulatorGenerator.html" title="type parameter in VariantProcessor.AccumulatorGenerator">ACCUMULATOR</a> extends <a href="picard/vcf/processor/VariantProcessor.Accumulator.html" title="interface in picard.vcf.processor">VariantProcessor.Accumulator</a>&lt;<a href="picard/vcf/processor/VariantProcessor.AccumulatorGenerator.html" title="type parameter in VariantProcessor.AccumulatorGenerator">RESULT</a>&gt;,<a href="picard/vcf/processor/VariantProcessor.AccumulatorGenerator.html" title="type parameter in VariantProcessor.AccumulatorGenerator">RESULT</a>&gt; - Interface in <a href="picard/vcf/processor/package-summary.html">picard.vcf.processor</a></dt>
<dd>
<div class="block">Generates instances of <a href="picard/vcf/processor/VariantProcessor.Accumulator.html" title="interface in picard.vcf.processor"><code>VariantProcessor.Accumulator</code></a>s.</div>
</dd>
<dt><a href="picard/vcf/processor/VariantProcessor.Builder.html" title="class in picard.vcf.processor"><span class="typeNameLink">VariantProcessor.Builder</span></a>&lt;<a href="picard/vcf/processor/VariantProcessor.Builder.html" title="type parameter in VariantProcessor.Builder">A</a> extends <a href="picard/vcf/processor/VariantProcessor.Accumulator.html" title="interface in picard.vcf.processor">VariantProcessor.Accumulator</a>&lt;<a href="picard/vcf/processor/VariantProcessor.Builder.html" title="type parameter in VariantProcessor.Builder">R</a>&gt;,<a href="picard/vcf/processor/VariantProcessor.Builder.html" title="type parameter in VariantProcessor.Builder">R</a>&gt; - Class in <a href="picard/vcf/processor/package-summary.html">picard.vcf.processor</a></dt>
<dd>
<div class="block">Simple builder of <a href="picard/vcf/processor/VariantProcessor.html" title="class in picard.vcf.processor"><code>VariantProcessor</code></a>s.</div>
</dd>
<dt><a href="picard/vcf/processor/VariantProcessor.ResultMerger.html" title="interface in picard.vcf.processor"><span class="typeNameLink">VariantProcessor.ResultMerger</span></a>&lt;<a href="picard/vcf/processor/VariantProcessor.ResultMerger.html" title="type parameter in VariantProcessor.ResultMerger">RESULT</a>&gt; - Interface in <a href="picard/vcf/processor/package-summary.html">picard.vcf.processor</a></dt>
<dd>
<div class="block">Takes a collection of results produced by <a href="picard/vcf/processor/VariantProcessor.Accumulator.html#result--"><code>VariantProcessor.Accumulator.result()</code></a> and merges them into a single RESULT.</div>
</dd>
<dt><a href="picard/util/VariantType.html" title="enum in picard.util"><span class="typeNameLink">VariantType</span></a> - Enum in <a href="picard/util/package-summary.html">picard.util</a></dt>
<dd>
<div class="block">Enum to hold the possible types of dbSnps.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/replicates/CollectIndependentReplicateMetrics.html#VCF">VCF</a></span> - Variable in class picard.analysis.replicates.<a href="picard/analysis/replicates/CollectIndependentReplicateMetrics.html" title="class in picard.analysis.replicates">CollectIndependentReplicateMetrics</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VcfFileSegment.html#vcf--">vcf()</a></span> - Method in class picard.vcf.processor.<a href="picard/vcf/processor/VcfFileSegment.html" title="class in picard.vcf.processor">VcfFileSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/MendelianViolations/FindMendelianViolations.html#VCF_DIR">VCF_DIR</a></span> - Variable in class picard.vcf.MendelianViolations.<a href="picard/vcf/MendelianViolations/FindMendelianViolations.html" title="class in picard.vcf.MendelianViolations">FindMendelianViolations</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/processor/VcfFileSegment.html" title="class in picard.vcf.processor"><span class="typeNameLink">VcfFileSegment</span></a> - Class in <a href="picard/vcf/processor/package-summary.html">picard.vcf.processor</a></dt>
<dd>
<div class="block">Describes a segment of a particular VCF file.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VcfFileSegment.html#VcfFileSegment--">VcfFileSegment()</a></span> - Constructor for class picard.vcf.processor.<a href="picard/vcf/processor/VcfFileSegment.html" title="class in picard.vcf.processor">VcfFileSegment</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/processor/VcfFileSegmentGenerator.html" title="class in picard.vcf.processor"><span class="typeNameLink">VcfFileSegmentGenerator</span></a> - Class in <a href="picard/vcf/processor/package-summary.html">picard.vcf.processor</a></dt>
<dd>
<div class="block">Describes a mechanism for producing <a href="picard/vcf/processor/VcfFileSegment.html" title="class in picard.vcf.processor"><code>VcfFileSegment</code></a>s from a VCF file.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VcfFileSegmentGenerator.html#VcfFileSegmentGenerator--">VcfFileSegmentGenerator()</a></span> - Constructor for class picard.vcf.processor.<a href="picard/vcf/processor/VcfFileSegmentGenerator.html" title="class in picard.vcf.processor">VcfFileSegmentGenerator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/VcfFormatConverter.html" title="class in picard.vcf"><span class="typeNameLink">VcfFormatConverter</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">Converts an ASCII VCF file to a binary BCF or vice versa.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/VcfFormatConverter.html#VcfFormatConverter--">VcfFormatConverter()</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/VcfFormatConverter.html" title="class in picard.vcf">VcfFormatConverter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/cmdline/programgroups/VcfOrBcf.html" title="class in picard.cmdline.programgroups"><span class="typeNameLink">VcfOrBcf</span></a> - Class in <a href="picard/cmdline/programgroups/package-summary.html">picard.cmdline.programgroups</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/programgroups/VcfOrBcf.html#VcfOrBcf--">VcfOrBcf()</a></span> - Constructor for class picard.cmdline.programgroups.<a href="picard/cmdline/programgroups/VcfOrBcf.html" title="class in picard.cmdline.programgroups">VcfOrBcf</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/VcfToIntervalList.html" title="class in picard.vcf"><span class="typeNameLink">VcfToIntervalList</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">Creates an interval list from a VCF</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/VcfToIntervalList.html#VcfToIntervalList--">VcfToIntervalList()</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/VcfToIntervalList.html" title="class in picard.vcf">VcfToIntervalList</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/vcf/VcfUtils.html" title="class in picard.vcf"><span class="typeNameLink">VcfUtils</span></a> - Class in <a href="picard/vcf/package-summary.html">picard.vcf</a></dt>
<dd>
<div class="block">Created by farjoun on 4/1/17.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/VcfUtils.html#VcfUtils--">VcfUtils()</a></span> - Constructor for class picard.vcf.<a href="picard/vcf/VcfUtils.html" title="class in picard.vcf">VcfUtils</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/cmdline/CommandLineProgram.html#VERBOSITY">VERBOSITY</a></span> - Variable in class picard.cmdline.<a href="picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/MultiTileBclFileUtil.html#verify-java.util.List-int:A-">verify(List&lt;Integer&gt;, int[])</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/MultiTileBclFileUtil.html" title="class in picard.illumina.parser">MultiTileBclFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/MultiTileFileUtil.html#verify-java.util.List-int:A-">verify(List&lt;Integer&gt;, int[])</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/MultiTileFileUtil.html" title="class in picard.illumina.parser">MultiTileFileUtil</a></dt>
<dd>
<div class="block">expectedCycles are not checked in this implementation.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ParameterizedFileUtil.html#verify-java.util.List-int:A-">verify(List&lt;Integer&gt;, int[])</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ParameterizedFileUtil.html" title="class in picard.illumina.parser">ParameterizedFileUtil</a></dt>
<dd>
<div class="block">Given the expected tiles/expected cycles for this file type, return a list of error messages describing any
 missing/or malformed files</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PerTileFileUtil.html#verify-java.util.List-int:A-">verify(List&lt;Integer&gt;, int[])</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/PerTileFileUtil.html" title="class in picard.illumina.parser">PerTileFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PerTilePerCycleFileUtil.html#verify-java.util.List-int:A-">verify(List&lt;Integer&gt;, int[])</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/PerTilePerCycleFileUtil.html" title="class in picard.illumina.parser">PerTilePerCycleFileUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/TileIndex.html#verify-java.util.List-">verify(List&lt;Integer&gt;)</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/TileIndex.html" title="class in picard.illumina.parser">TileIndex</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/MultiTileParser.html#verifyData-java.util.List-int:A-">verifyData(List&lt;Integer&gt;, int[])</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/MultiTileParser.html" title="class in picard.illumina.parser">MultiTileParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/PerTileParser.html#verifyData-java.util.List-int:A-">verifyData(List&lt;Integer&gt;, int[])</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/PerTileParser.html" title="class in picard.illumina.parser">PerTileParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/FilterFileReader.html#version">version</a></span> - Variable in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/FilterFileReader.html" title="class in picard.illumina.parser.readers">FilterFileReader</a></dt>
<dd>
<div class="block">Version number found in the FilterFile, this should equal 3</div>
</dd>
<dt><a href="picard/sam/ViewSam.html" title="class in picard.sam"><span class="typeNameLink">ViewSam</span></a> - Class in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>
<div class="block">Very simple command that just reads a SAM or BAM file and writes out the header
 and each records to standard out.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/ViewSam.html#ViewSam--">ViewSam()</a></span> - Constructor for class picard.sam.<a href="picard/sam/ViewSam.html" title="class in picard.sam">ViewSam</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/ViewSam.AlignmentStatus.html" title="enum in picard.sam"><span class="typeNameLink">ViewSam.AlignmentStatus</span></a> - Enum in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/sam/ViewSam.PfStatus.html" title="enum in picard.sam"><span class="typeNameLink">ViewSam.PfStatus</span></a> - Enum in <a href="picard/sam/package-summary.html">picard.sam</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="I:W">
<!--   -->
</a>
<h2 class="title">W</h2>
<dl>
<dt><span class="memberNameLink"><a href="picard/vcf/LiftoverVcf.html#WARN_ON_MISSING_CONTIG">WARN_ON_MISSING_CONTIG</a></span> - Variable in class picard.vcf.<a href="picard/vcf/LiftoverVcf.html" title="class in picard.vcf">LiftoverVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#WgsMetrics--">WgsMetrics()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">Create an instance of this metric that is not mergeable.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html#WgsMetrics-htsjdk.samtools.util.IntervalList-htsjdk.samtools.util.Histogram-htsjdk.samtools.util.Histogram-double-double-double-double-double-double-double-int-htsjdk.samtools.util.Histogram-int-">WgsMetrics(IntervalList, Histogram&lt;Integer&gt;, Histogram&lt;Integer&gt;, double, double, double, double, double, double, double, int, Histogram&lt;Integer&gt;, int)</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetrics</a></dt>
<dd>
<div class="block">Create an instance of this metric that is mergeable.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetrics.WgsMetricsCollector.html#WgsMetricsCollector-picard.analysis.CollectWgsMetrics-int-htsjdk.samtools.util.IntervalList-">WgsMetricsCollector(CollectWgsMetrics, int, IntervalList)</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/CollectWgsMetrics.WgsMetricsCollector.html" title="class in picard.analysis">CollectWgsMetrics.WgsMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><a href="picard/analysis/WgsMetricsProcessor.html" title="interface in picard.analysis"><span class="typeNameLink">WgsMetricsProcessor</span></a> - Interface in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Interface for processing data and generate result for CollectWgsMetrics</div>
</dd>
<dt><a href="picard/analysis/WgsMetricsProcessorImpl.html" title="class in picard.analysis"><span class="typeNameLink">WgsMetricsProcessorImpl</span></a>&lt;<a href="picard/analysis/WgsMetricsProcessorImpl.html" title="type parameter in WgsMetricsProcessorImpl">T</a> extends htsjdk.samtools.util.AbstractRecordAndOffset&gt; - Class in <a href="picard/analysis/package-summary.html">picard.analysis</a></dt>
<dd>
<div class="block">Implementation of <a href="picard/analysis/WgsMetricsProcessor.html" title="interface in picard.analysis"><code>WgsMetricsProcessor</code></a> that gets input data from a given iterator
 and processes it with a help of collector</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/WgsMetricsProcessorImpl.html#WgsMetricsProcessorImpl-htsjdk.samtools.util.AbstractLocusIterator-htsjdk.samtools.reference.ReferenceSequenceFileWalker-picard.analysis.AbstractWgsMetricsCollector-htsjdk.samtools.util.ProgressLogger-">WgsMetricsProcessorImpl(AbstractLocusIterator&lt;T, AbstractLocusInfo&lt;T&gt;&gt;, ReferenceSequenceFileWalker, AbstractWgsMetricsCollector&lt;T&gt;, ProgressLogger)</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/WgsMetricsProcessorImpl.html" title="class in picard.analysis">WgsMetricsProcessorImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverage.html#WgsMetricsWithNonZeroCoverage--">WgsMetricsWithNonZeroCoverage()</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverage.html" title="class in picard.analysis">CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverage.html#WgsMetricsWithNonZeroCoverage-htsjdk.samtools.util.IntervalList-htsjdk.samtools.util.Histogram-htsjdk.samtools.util.Histogram-double-double-double-double-double-double-double-int-htsjdk.samtools.util.Histogram-int-">WgsMetricsWithNonZeroCoverage(IntervalList, Histogram&lt;Integer&gt;, Histogram&lt;Integer&gt;, double, double, double, double, double, double, double, int, Histogram&lt;Integer&gt;, int)</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverage.html" title="class in picard.analysis">CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverageCollector.html#WgsMetricsWithNonZeroCoverageCollector-picard.analysis.CollectWgsMetricsWithNonZeroCoverage-int-htsjdk.samtools.util.IntervalList-">WgsMetricsWithNonZeroCoverageCollector(CollectWgsMetricsWithNonZeroCoverage, int, IntervalList)</a></span> - Constructor for class picard.analysis.<a href="picard/analysis/CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverageCollector.html" title="class in picard.analysis">CollectWgsMetricsWithNonZeroCoverage.WgsMetricsWithNonZeroCoverageCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/InsertSizeMetrics.html#WIDTH_OF_10_PERCENT">WIDTH_OF_10_PERCENT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/InsertSizeMetrics.html" title="class in picard.analysis">InsertSizeMetrics</a></dt>
<dd>
<div class="block">The "width" of the bins, centered around the median, that encompass 10% of all read pairs.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/InsertSizeMetrics.html#WIDTH_OF_20_PERCENT">WIDTH_OF_20_PERCENT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/InsertSizeMetrics.html" title="class in picard.analysis">InsertSizeMetrics</a></dt>
<dd>
<div class="block">The "width" of the bins, centered around the median, that encompass 20% of all read pairs.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/InsertSizeMetrics.html#WIDTH_OF_30_PERCENT">WIDTH_OF_30_PERCENT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/InsertSizeMetrics.html" title="class in picard.analysis">InsertSizeMetrics</a></dt>
<dd>
<div class="block">The "width" of the bins, centered around the median, that encompass 30% of all read pairs.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/InsertSizeMetrics.html#WIDTH_OF_40_PERCENT">WIDTH_OF_40_PERCENT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/InsertSizeMetrics.html" title="class in picard.analysis">InsertSizeMetrics</a></dt>
<dd>
<div class="block">The "width" of the bins, centered around the median, that encompass 40% of all read pairs.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/InsertSizeMetrics.html#WIDTH_OF_50_PERCENT">WIDTH_OF_50_PERCENT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/InsertSizeMetrics.html" title="class in picard.analysis">InsertSizeMetrics</a></dt>
<dd>
<div class="block">The "width" of the bins, centered around the median, that encompass 50% of all read pairs.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/InsertSizeMetrics.html#WIDTH_OF_60_PERCENT">WIDTH_OF_60_PERCENT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/InsertSizeMetrics.html" title="class in picard.analysis">InsertSizeMetrics</a></dt>
<dd>
<div class="block">The "width" of the bins, centered around the median, that encompass 60% of all read pairs.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/InsertSizeMetrics.html#WIDTH_OF_70_PERCENT">WIDTH_OF_70_PERCENT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/InsertSizeMetrics.html" title="class in picard.analysis">InsertSizeMetrics</a></dt>
<dd>
<div class="block">The "width" of the bins, centered around the median, that encompass 70% of all read pairs.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/InsertSizeMetrics.html#WIDTH_OF_80_PERCENT">WIDTH_OF_80_PERCENT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/InsertSizeMetrics.html" title="class in picard.analysis">InsertSizeMetrics</a></dt>
<dd>
<div class="block">The "width" of the bins, centered around the median, that encompass 80% of all read pairs.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/InsertSizeMetrics.html#WIDTH_OF_90_PERCENT">WIDTH_OF_90_PERCENT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/InsertSizeMetrics.html" title="class in picard.analysis">InsertSizeMetrics</a></dt>
<dd>
<div class="block">The "width" of the bins, centered around the median, that encompass 90% of all read pairs.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/InsertSizeMetrics.html#WIDTH_OF_99_PERCENT">WIDTH_OF_99_PERCENT</a></span> - Variable in class picard.analysis.<a href="picard/analysis/InsertSizeMetrics.html" title="class in picard.analysis">InsertSizeMetrics</a></dt>
<dd>
<div class="block">The "width" of the bins, centered around the median, that encompass 100% of all read pairs.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasSummaryMetrics.html#WINDOW_SIZE">WINDOW_SIZE</a></span> - Variable in class picard.analysis.<a href="picard/analysis/GcBiasSummaryMetrics.html" title="class in picard.analysis">GcBiasSummaryMetrics</a></dt>
<dd>
<div class="block">The window size on the genome used to calculate the GC of the sequence.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/GcBiasDetailMetrics.html#WINDOWS">WINDOWS</a></span> - Variable in class picard.analysis.<a href="picard/analysis/GcBiasDetailMetrics.html" title="class in picard.analysis">GcBiasDetailMetrics</a></dt>
<dd>
<div class="block">The number of windows on the reference genome that have this G+C content.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/processor/VariantProcessor.Builder.html#withInput-java.io.File...-">withInput(File...)</a></span> - Method in class picard.vcf.processor.<a href="picard/vcf/processor/VariantProcessor.Builder.html" title="class in picard.vcf.processor">VariantProcessor.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ClusterDataToSamConverter.html#withMolecularIndexQualityTag-java.lang.String-">withMolecularIndexQualityTag(String)</a></span> - Method in class picard.illumina.<a href="picard/illumina/ClusterDataToSamConverter.html" title="class in picard.illumina">ClusterDataToSamConverter</a></dt>
<dd>
<div class="block">Sets the SAM tag to use to store the molecular index base qualities.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ClusterDataToSamConverter.html#withMolecularIndexTag-java.lang.String-">withMolecularIndexTag(String)</a></span> - Method in class picard.illumina.<a href="picard/illumina/ClusterDataToSamConverter.html" title="class in picard.illumina">ClusterDataToSamConverter</a></dt>
<dd>
<div class="block">Sets the SAM tag to use to store the molecular index bases.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeMap.html#withoutChromosomes-java.util.Set-">withoutChromosomes(Set&lt;String&gt;)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeMap.html" title="class in picard.fingerprint">HaplotypeMap</a></dt>
<dd>
<div class="block">Returns a copy of this haplotype map that excludes haplotypes on the chromosomes provided.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/ClusterDataToSamConverter.html#withTagPerMolecularIndex-java.util.List-">withTagPerMolecularIndex(List&lt;String&gt;)</a></span> - Method in class picard.illumina.<a href="picard/illumina/ClusterDataToSamConverter.html" title="class in picard.illumina">ClusterDataToSamConverter</a></dt>
<dd>
<div class="block">Sets the SAM tags to use to store the bases each molecular index.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasSummaryMetrics.html#WORST_CXT">WORST_CXT</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasSummaryMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.BaitBiasSummaryMetrics</a></dt>
<dd>
<div class="block">The sequence context (reference bases surrounding the locus of interest) having the lowest Q-score among all contexts for this artifact.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterSummaryMetrics.html#WORST_CXT">WORST_CXT</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterSummaryMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.PreAdapterSummaryMetrics</a></dt>
<dd>
<div class="block">The sequence context (reference bases surrounding the locus of interest) having the lowest Q-score among all contexts for this artifact.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasSummaryMetrics.html#WORST_CXT_QSCORE">WORST_CXT_QSCORE</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasSummaryMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.BaitBiasSummaryMetrics</a></dt>
<dd>
<div class="block">The Q-score for the worst context.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterSummaryMetrics.html#WORST_CXT_QSCORE">WORST_CXT_QSCORE</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterSummaryMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.PreAdapterSummaryMetrics</a></dt>
<dd>
<div class="block">The Q-score for the worst context.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasSummaryMetrics.html#WORST_POST_CXT">WORST_POST_CXT</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasSummaryMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.BaitBiasSummaryMetrics</a></dt>
<dd>
<div class="block">The post-context (reference bases trailing after the locus of interest) with the lowest Q-score.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterSummaryMetrics.html#WORST_POST_CXT">WORST_POST_CXT</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterSummaryMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.PreAdapterSummaryMetrics</a></dt>
<dd>
<div class="block">The post-context (reference bases trailing after the locus of interest) with the lowest Q-score.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasSummaryMetrics.html#WORST_POST_CXT_QSCORE">WORST_POST_CXT_QSCORE</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasSummaryMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.BaitBiasSummaryMetrics</a></dt>
<dd>
<div class="block">The Q-score for the worst post-context.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterSummaryMetrics.html#WORST_POST_CXT_QSCORE">WORST_POST_CXT_QSCORE</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterSummaryMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.PreAdapterSummaryMetrics</a></dt>
<dd>
<div class="block">The Q-score for the worst post-context.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasSummaryMetrics.html#WORST_PRE_CXT">WORST_PRE_CXT</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasSummaryMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.BaitBiasSummaryMetrics</a></dt>
<dd>
<div class="block">The pre-context (reference bases leading up to the locus of interest) with the lowest Q-score.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterSummaryMetrics.html#WORST_PRE_CXT">WORST_PRE_CXT</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterSummaryMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.PreAdapterSummaryMetrics</a></dt>
<dd>
<div class="block">The pre-context (reference bases leading up to the locus of interest) with the lowest Q-score.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasSummaryMetrics.html#WORST_PRE_CXT_QSCORE">WORST_PRE_CXT_QSCORE</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.BaitBiasSummaryMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.BaitBiasSummaryMetrics</a></dt>
<dd>
<div class="block">The Q-score for the worst pre-context.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterSummaryMetrics.html#WORST_PRE_CXT_QSCORE">WORST_PRE_CXT_QSCORE</a></span> - Variable in class picard.analysis.artifacts.<a href="picard/analysis/artifacts/SequencingArtifactMetrics.PreAdapterSummaryMetrics.html" title="class in picard.analysis.artifacts">SequencingArtifactMetrics.PreAdapterSummaryMetrics</a></dt>
<dd>
<div class="block">The Q-score for the worst pre-context.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/pedigree/PedFile.html#write-java.io.File-">write(File)</a></span> - Method in class picard.pedigree.<a href="picard/pedigree/PedFile.html" title="class in picard.pedigree">PedFile</a></dt>
<dd>
<div class="block">Writes a set of pedigrees out to disk.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/util/CircularByteBuffer.html#write-byte:A-int-int-">write(byte[], int, int)</a></span> - Method in class picard.util.<a href="picard/util/CircularByteBuffer.html" title="class in picard.util">CircularByteBuffer</a></dt>
<dd>
<div class="block">Write bytes into the buffer from the supplied array.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/vcf/LiftoverVcf.html#WRITE_ORIGINAL_POSITION">WRITE_ORIGINAL_POSITION</a></span> - Variable in class picard.vcf.<a href="picard/vcf/LiftoverVcf.html" title="class in picard.vcf">LiftoverVcf</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FilterSamReads.html#WRITE_READS_FILES">WRITE_READS_FILES</a></span> - Variable in class picard.sam.<a href="picard/sam/FilterSamReads.html" title="class in picard.sam">FilterSamReads</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/sam/FixMateInformation.html#writeAlignment-htsjdk.samtools.SAMRecord-">writeAlignment(SAMRecord)</a></span> - Method in class picard.sam.<a href="picard/sam/FixMateInformation.html" title="class in picard.sam">FixMateInformation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeMap.html#writeAsVcf-java.io.File-java.io.File-">writeAsVcf(File, File)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeMap.html" title="class in picard.fingerprint">HaplotypeMap</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fastq/BamToBfqWriter.html#writeBfqFiles--">writeBfqFiles()</a></span> - Method in class picard.fastq.<a href="picard/fastq/BamToBfqWriter.html" title="class in picard.fastq">BamToBfqWriter</a></dt>
<dd>
<div class="block">Writes the binary fastq file(s) to the output directory</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CollectIlluminaLaneMetrics.IlluminaLaneMetricsCollector.html#writeLaneMetrics-java.util.Map-java.io.File-java.lang.String-htsjdk.samtools.metrics.MetricsFile-java.lang.String-">writeLaneMetrics(Map&lt;Integer, ? extends Collection&lt;Tile&gt;&gt;, File, String, MetricsFile&lt;MetricBase, Comparable&lt;?&gt;&gt;, String)</a></span> - Static method in class picard.illumina.<a href="picard/illumina/CollectIlluminaLaneMetrics.IlluminaLaneMetricsCollector.html" title="class in picard.illumina">CollectIlluminaLaneMetrics.IlluminaLaneMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/CollectIlluminaLaneMetrics.IlluminaLaneMetricsCollector.html#writePhasingMetrics-java.util.Map-java.io.File-java.lang.String-htsjdk.samtools.metrics.MetricsFile-java.lang.String-boolean-">writePhasingMetrics(Map&lt;Integer, ? extends Collection&lt;Tile&gt;&gt;, File, String, MetricsFile&lt;MetricBase, Comparable&lt;?&gt;&gt;, String, boolean)</a></span> - Static method in class picard.illumina.<a href="picard/illumina/CollectIlluminaLaneMetrics.IlluminaLaneMetricsCollector.html" title="class in picard.illumina">CollectIlluminaLaneMetrics.IlluminaLaneMetricsCollector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/fingerprint/HaplotypeMap.html#writeToFile-java.io.File-">writeToFile(File)</a></span> - Method in class picard.fingerprint.<a href="picard/fingerprint/HaplotypeMap.html" title="class in picard.fingerprint">HaplotypeMap</a></dt>
<dd>
<div class="block">Writes out a HaplotypeMap file with the contents of this map.</div>
</dd>
</dl>
<a name="I:X">
<!--   -->
</a>
<h2 class="title">X</h2>
<dl>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailDetailedMetric.html#X">X</a></span> - Variable in class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailDetailedMetric.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics.PFFailDetailedMetric</a></dt>
<dd>
<div class="block">The X coordinate of the read within the tile</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/PhysicalLocationInt.html#x">x</a></span> - Variable in class picard.sam.util.<a href="picard/sam/util/PhysicalLocationInt.html" title="class in picard.sam.util">PhysicalLocationInt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ClusterData.html#xIsSet--">xIsSet()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ClusterData.html" title="class in picard.illumina.parser">ClusterData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.PositionInfo.html#xPos">xPos</a></span> - Variable in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.PositionInfo.html" title="class in picard.illumina.parser.readers">AbstractIlluminaPositionFileReader.PositionInfo</a></dt>
<dd>
<div class="block">The x-position as it occurs in the file being read</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.PositionInfo.html#xQseqCoord">xQseqCoord</a></span> - Variable in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.PositionInfo.html" title="class in picard.illumina.parser.readers">AbstractIlluminaPositionFileReader.PositionInfo</a></dt>
<dd>
<div class="block">The QSeq style x-coordinat, an integer = Math.round(xPos*10 + 1000)</div>
</dd>
</dl>
<a name="I:Y">
<!--   -->
</a>
<h2 class="title">Y</h2>
<dl>
<dt><span class="memberNameLink"><a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailDetailedMetric.html#Y">Y</a></span> - Variable in class picard.illumina.quality.<a href="picard/illumina/quality/CollectHiSeqXPfFailMetrics.PFFailDetailedMetric.html" title="class in picard.illumina.quality">CollectHiSeqXPfFailMetrics.PFFailDetailedMetric</a></dt>
<dd>
<div class="block">The Y coordinate of the read within the tile</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/sam/util/PhysicalLocationInt.html#y">y</a></span> - Variable in class picard.sam.util.<a href="picard/sam/util/PhysicalLocationInt.html" title="class in picard.sam.util">PhysicalLocationInt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/ClusterData.html#yIsSet--">yIsSet()</a></span> - Method in class picard.illumina.parser.<a href="picard/illumina/parser/ClusterData.html" title="class in picard.illumina.parser">ClusterData</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.PositionInfo.html#yPos">yPos</a></span> - Variable in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.PositionInfo.html" title="class in picard.illumina.parser.readers">AbstractIlluminaPositionFileReader.PositionInfo</a></dt>
<dd>
<div class="block">The y-position as it occurs in the file being read</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.PositionInfo.html#yQseqCoord">yQseqCoord</a></span> - Variable in class picard.illumina.parser.readers.<a href="picard/illumina/parser/readers/AbstractIlluminaPositionFileReader.PositionInfo.html" title="class in picard.illumina.parser.readers">AbstractIlluminaPositionFileReader.PositionInfo</a></dt>
<dd>
<div class="block">The QSeq style y-coordinates, an integer = Math.round(yPos*10 + 1000)</div>
</dd>
</dl>
<a name="I:Z">
<!--   -->
</a>
<h2 class="title">Z</h2>
<dl>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/HsMetrics.html#ZERO_CVG_TARGETS_PCT">ZERO_CVG_TARGETS_PCT</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/HsMetrics.html" title="class in picard.analysis.directed">HsMetrics</a></dt>
<dd>
<div class="block">The fraction of targets that did not reach coverage=1 over any base.</div>
</dd>
<dt><span class="memberNameLink"><a href="picard/analysis/directed/TargetedPcrMetrics.html#ZERO_CVG_TARGETS_PCT">ZERO_CVG_TARGETS_PCT</a></span> - Variable in class picard.analysis.directed.<a href="picard/analysis/directed/TargetedPcrMetrics.html" title="class in picard.analysis.directed">TargetedPcrMetrics</a></dt>
<dd>
<div class="block">The fraction of targets that did not reach coverage=1 over any base.</div>
</dd>
</dl>
<a href="#I:A">A</a>&nbsp;<a href="#I:B">B</a>&nbsp;<a href="#I:C">C</a>&nbsp;<a href="#I:D">D</a>&nbsp;<a href="#I:E">E</a>&nbsp;<a href="#I:F">F</a>&nbsp;<a href="#I:G">G</a>&nbsp;<a href="#I:H">H</a>&nbsp;<a href="#I:I">I</a>&nbsp;<a href="#I:J">J</a>&nbsp;<a href="#I:L">L</a>&nbsp;<a href="#I:M">M</a>&nbsp;<a href="#I:N">N</a>&nbsp;<a href="#I:O">O</a>&nbsp;<a href="#I:P">P</a>&nbsp;<a href="#I:Q">Q</a>&nbsp;<a href="#I:R">R</a>&nbsp;<a href="#I:S">S</a>&nbsp;<a href="#I:T">T</a>&nbsp;<a href="#I:U">U</a>&nbsp;<a href="#I:V">V</a>&nbsp;<a href="#I:W">W</a>&nbsp;<a href="#I:X">X</a>&nbsp;<a href="#I:Y">Y</a>&nbsp;<a href="#I:Z">Z</a>&nbsp;</div>
<!-- ======= START OF BOTTOM NAVBAR ====== -->
<div class="bottomNav"><a name="navbar.bottom">
<!--   -->
</a>
<div class="skipNav"><a href="#skip.navbar.bottom" title="Skip navigation links">Skip navigation links</a></div>
<a name="navbar.bottom.firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="overview-summary.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li><a href="overview-tree.html">Tree</a></li>
<li><a href="deprecated-list.html">Deprecated</a></li>
<li class="navBarCell1Rev">Index</li>
<li><a href="help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev</li>
<li>Next</li>
</ul>
<ul class="navList">
<li><a href="index.html?index-all.html" target="_top">Frames</a></li>
<li><a href="index-all.html" target="_top">No&nbsp;Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_bottom">
<li><a href="allclasses-noframe.html">All&nbsp;Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_bottom");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip.navbar.bottom">
<!--   -->
</a></div>
<!-- ======== END OF BOTTOM NAVBAR ======= -->
</body>
</html>
