<?php

    include '../../../common/include/common.php';
    include_once '../../config.php';
    $module = modules::GATK;
    printHeader($module, "Picard | Tool Documentation Index", "Guide");
?>

<div class='row-fluid' id="top">

	

	<?php $group = 'Fasta'; ?>

	<section class="span4">
		<aside class="well">
			<a href="index"><h4><i class='fa fa-chevron-left'></i> Back to Tool Docs Index</h4></a>
		</aside>
		<aside class="well">
			<h2>Categories</h2>
        <style>
            #sidenav .accordion-body a {
                color : gray;
            }

            .accordion-body li {
                list-style : none;
            }
        </style>
        <ul class="nav nav-pills nav-stacked" id="sidenav">
        		<hr>
        		<hr>
        		<hr>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#"></a>
							<div id=""
								<?php echo ($group == '')? 'class="accordion-body collapse in"'.chr(62) : 'class="accordion-body collapse"'.chr(62);?>
								<ul>
											<li>
												<a href="picard_sam_CreateSequenceDictionary$CreateSeqDictReferenceArgumentCollection.html">CreateSeqDictReferenceArgumentCollection</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#Fasta">Fasta</a>
							<div id="Fasta"
								<?php echo ($group == 'Fasta')? 'class="accordion-body collapse in"'.chr(62) : 'class="accordion-body collapse"'.chr(62);?>
								<ul>
											<li>
												<a href="picard_sam_CreateSequenceDictionary.html">CreateSequenceDictionary</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#Metrics">Metrics</a>
							<div id="Metrics"
								<?php echo ($group == 'Metrics')? 'class="accordion-body collapse in"'.chr(62) : 'class="accordion-body collapse"'.chr(62);?>
								<ul>
											<li>
												<a href="picard_analysis_CollectInsertSizeMetrics.html">CollectInsertSizeMetrics</a>
											</li>
								</ul>
							</div>
						</li>
        		<hr>
        </ul>
		</aside>
		<?php getForumPosts( 'CreateSequenceDictionary' ) ?>

	</section>

	<div class="span8">

		<h1>CreateSequenceDictionary</h1>

		<p class="lead">Creates a sequence dictionary for a reference sequence.  </p>

			<h3>Category
				<small> Fasta</small>
			</h3>
		<hr>
		<h2>Overview</h2>
		Create a SAM/BAM file from a fasta containing reference sequence. The output SAM file contains a header but no
 SAMRecords, and the header contains only sequence records.


				<h3>CreateSequenceDictionary specific arguments</h3>
				<p>This table summarizes the command-line arguments that are specific to this tool. For more details on each argument, see the list further down below the table or click on an argument name to jump directly to that entry in the list.</p>
				<table class="table table-striped table-bordered table-condensed">
					<thead>
					<tr>
						<th>Argument name(s)</th>
						<th>Default value</th>
						<th>Summary</th>
					</tr>
					</thead>
					<tbody>
			<tr>
				<th colspan="4" id="row-divider">Required Arguments</th>
			</tr>
				<tr>
					<td><a href="#--REFERENCE">--REFERENCE</a><br />
								&nbsp;<em>-R</em>
					</td>
					<!--<td>File</td> -->
					<td>null</td>
					<td>Input reference fasta or fasta.gz</td>
				</tr>
			<tr>
				<th colspan="4" id="row-divider">Optional Tool Arguments</th>
			</tr>
				<tr>
					<td><a href="#--GENOME_ASSEMBLY">--GENOME_ASSEMBLY</a><br />
								&nbsp;<em>-AS</em>
					</td>
					<!--<td>String</td> -->
					<td>null</td>
					<td>Put into AS field of sequence dictionary entry if supplied</td>
				</tr>
				<tr>
					<td><a href="#--NUM_SEQUENCES">--NUM_SEQUENCES</a><br />
								&nbsp;<em>-</em>
					</td>
					<!--<td>int</td> -->
					<td>2147483647</td>
					<td>Stop after writing this many sequences.  For testing.</td>
				</tr>
				<tr>
					<td><a href="#--OUTPUT">--OUTPUT</a><br />
								&nbsp;<em>-O</em>
					</td>
					<!--<td>File</td> -->
					<td>null</td>
					<td>Output SAM file containing only the sequence dictionary. By default it will use the base name of the input reference with the .dict extension</td>
				</tr>
				<tr>
					<td><a href="#--SPECIES">--SPECIES</a><br />
								&nbsp;<em>-SP</em>
					</td>
					<!--<td>String</td> -->
					<td>null</td>
					<td>Put into SP field of sequence dictionary entry</td>
				</tr>
				<tr>
					<td><a href="#--TRUNCATE_NAMES_AT_WHITESPACE">--TRUNCATE_NAMES_AT_WHITESPACE</a><br />
								&nbsp;<em>-</em>
					</td>
					<!--<td>boolean</td> -->
					<td>true</td>
					<td>Make sequence name the first word from the > line in the fasta file.  By default the entire contents of the > line is used, excluding leading and trailing whitespace.</td>
				</tr>
				<tr>
					<td><a href="#--URI">--URI</a><br />
								&nbsp;<em>-UR</em>
					</td>
					<!--<td>String</td> -->
					<td>null</td>
					<td>Put into UR field of sequence dictionary entry.  If not supplied, input reference file is used</td>
				</tr>
			<tr>
				<th colspan="4" id="row-divider">Optional Common Arguments</th>
			</tr>
				<tr>
					<td><a href="#--COMPRESSION_LEVEL">--COMPRESSION_LEVEL</a><br />
								&nbsp;<em>-</em>
					</td>
					<!--<td>int</td> -->
					<td>5</td>
					<td>Compression level for all compressed files created (e.g. BAM and GELI).</td>
				</tr>
				<tr>
					<td><a href="#--CREATE_INDEX">--CREATE_INDEX</a><br />
								&nbsp;<em>-</em>
					</td>
					<!--<td>Boolean</td> -->
					<td>false</td>
					<td>Whether to create a BAM index when writing a coordinate-sorted BAM file.</td>
				</tr>
				<tr>
					<td><a href="#--CREATE_MD5_FILE">--CREATE_MD5_FILE</a><br />
								&nbsp;<em>-</em>
					</td>
					<!--<td>boolean</td> -->
					<td>false</td>
					<td>Whether to create an MD5 digest for any BAM or FASTQ files created.  </td>
				</tr>
				<tr>
					<td><a href="#--GA4GH_CLIENT_SECRETS">--GA4GH_CLIENT_SECRETS</a><br />
								&nbsp;<em>-</em>
					</td>
					<!--<td>String</td> -->
					<td>client_secrets.json</td>
					<td>Google Genomics API client_secrets.json file path.</td>
				</tr>
				<tr>
					<td><a href="#--MAX_RECORDS_IN_RAM">--MAX_RECORDS_IN_RAM</a><br />
								&nbsp;<em>-</em>
					</td>
					<!--<td>Integer</td> -->
					<td>500000</td>
					<td>When writing SAM files that need to be sorted, this will specify the number of records stored in RAM before spilling to disk. Increasing this number reduces the number of file handles needed to sort a SAM file, and increases the amount of RAM needed.</td>
				</tr>
				<tr>
					<td><a href="#--QUIET">--QUIET</a><br />
								&nbsp;<em>-</em>
					</td>
					<!--<td>Boolean</td> -->
					<td>false</td>
					<td>Whether to suppress job-summary info on System.err.</td>
				</tr>
				<tr>
					<td><a href="#--TMP_DIR">--TMP_DIR</a><br />
								&nbsp;<em>-</em>
					</td>
					<!--<td>List[File]</td> -->
					<td>[]</td>
					<td>One or more directories with space available to be used by this program for temporary storage of working files</td>
				</tr>
				<tr>
					<td><a href="#--USE_JDK_DEFLATER">--USE_JDK_DEFLATER</a><br />
								&nbsp;<em>-use_jdk_deflater</em>
					</td>
					<!--<td>Boolean</td> -->
					<td>false</td>
					<td>Use the JDK Deflater instead of the Intel Deflater for writing compressed output</td>
				</tr>
				<tr>
					<td><a href="#--USE_JDK_INFLATER">--USE_JDK_INFLATER</a><br />
								&nbsp;<em>-use_jdk_inflater</em>
					</td>
					<!--<td>Boolean</td> -->
					<td>false</td>
					<td>Use the JDK Inflater instead of the Intel Inflater for reading compressed input</td>
				</tr>
				<tr>
					<td><a href="#--VALIDATION_STRINGENCY">--VALIDATION_STRINGENCY</a><br />
								&nbsp;<em>-</em>
					</td>
					<!--<td>ValidationStringency</td> -->
					<td>STRICT</td>
					<td>Validation stringency for all SAM files read by this program.  Setting stringency to SILENT can improve performance when processing a BAM file in which variable-length data (read, qualities, tags) do not otherwise need to be decoded.</td>
				</tr>
				<tr>
					<td><a href="#--VERBOSITY">--VERBOSITY</a><br />
								&nbsp;<em>-</em>
					</td>
					<!--<td>LogLevel</td> -->
					<td>INFO</td>
					<td>Control verbosity of logging.</td>
				</tr>
					</tbody>
				</table>

					<h3>Argument details</h3>
					<p>Arguments in this list are specific to this tool. Keep in mind that other arguments are available that are shared with other tools (e.g. command-line GATK arguments); see Inherited arguments above.</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--COMPRESSION_LEVEL">--COMPRESSION_LEVEL </a>
			 / <small>-</small>
		</h3>
		<p class="args">
			<b>Compression level for all compressed files created (e.g. BAM and GELI).</b><br />
			
		</p>
		<p>
		<span class="label label-info ">int</span>
			&nbsp;<span class="label">5</span>
			&nbsp;<span class="label label-warning">[ [ -∞</span>
			&nbsp;<span class="label label-warning">∞ ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--CREATE_INDEX">--CREATE_INDEX </a>
			 / <small>-</small>
		</h3>
		<p class="args">
			<b>Whether to create a BAM index when writing a coordinate-sorted BAM file.</b><br />
			
		</p>
		<p>
		<span class="label label-info ">Boolean</span>
			&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--CREATE_MD5_FILE">--CREATE_MD5_FILE </a>
			 / <small>-</small>
		</h3>
		<p class="args">
			<b>Whether to create an MD5 digest for any BAM or FASTQ files created.  </b><br />
			
		</p>
		<p>
		<span class="label label-info ">boolean</span>
			&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--GA4GH_CLIENT_SECRETS">--GA4GH_CLIENT_SECRETS </a>
			 / <small>-</small>
		</h3>
		<p class="args">
			<b>Google Genomics API client_secrets.json file path.</b><br />
			
		</p>
		<p>
		<span class="label label-info ">String</span>
			&nbsp;<span class="label">client_secrets.json</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--GENOME_ASSEMBLY">--GENOME_ASSEMBLY </a>
			 / <small>-AS</small>
		</h3>
		<p class="args">
			<b>Put into AS field of sequence dictionary entry if supplied</b><br />
			
		</p>
		<p>
		<span class="label label-info ">String</span>
			&nbsp;<span class="label">null</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--MAX_RECORDS_IN_RAM">--MAX_RECORDS_IN_RAM </a>
			 / <small>-</small>
		</h3>
		<p class="args">
			<b>When writing SAM files that need to be sorted, this will specify the number of records stored in RAM before spilling to disk. Increasing this number reduces the number of file handles needed to sort a SAM file, and increases the amount of RAM needed.</b><br />
			
		</p>
		<p>
		<span class="label label-info ">Integer</span>
			&nbsp;<span class="label">500000</span>
			&nbsp;<span class="label label-warning">[ [ -∞</span>
			&nbsp;<span class="label label-warning">∞ ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--NUM_SEQUENCES">--NUM_SEQUENCES </a>
			 / <small>-</small>
		</h3>
		<p class="args">
			<b>Stop after writing this many sequences.  For testing.</b><br />
			
		</p>
		<p>
		<span class="label label-info ">int</span>
			&nbsp;<span class="label">2147483647</span>
			&nbsp;<span class="label label-warning">[ [ -∞</span>
			&nbsp;<span class="label label-warning">∞ ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--OUTPUT">--OUTPUT </a>
			 / <small>-O</small>
		</h3>
		<p class="args">
			<b>Output SAM file containing only the sequence dictionary. By default it will use the base name of the input reference with the .dict extension</b><br />
			
		</p>
		<p>
		<span class="label label-info ">File</span>
			&nbsp;<span class="label">null</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--QUIET">--QUIET </a>
			 / <small>-</small>
		</h3>
		<p class="args">
			<b>Whether to suppress job-summary info on System.err.</b><br />
			
		</p>
		<p>
		<span class="label label-info ">Boolean</span>
			&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--REFERENCE">--REFERENCE </a>
			 / <small>-R</small>
		</h3>
		<p class="args">
			<b>Input reference fasta or fasta.gz</b><br />
			
		</p>
		<p>
				<span class="badge badge-important">R</span>
		<span class="label label-info ">File</span>
			&nbsp;<span class="label">null</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--SPECIES">--SPECIES </a>
			 / <small>-SP</small>
		</h3>
		<p class="args">
			<b>Put into SP field of sequence dictionary entry</b><br />
			
		</p>
		<p>
		<span class="label label-info ">String</span>
			&nbsp;<span class="label">null</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--TMP_DIR">--TMP_DIR </a>
			 / <small>-</small>
		</h3>
		<p class="args">
			<b>One or more directories with space available to be used by this program for temporary storage of working files</b><br />
			
		</p>
		<p>
		<span class="label label-info ">List[File]</span>
			&nbsp;<span class="label">[]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--TRUNCATE_NAMES_AT_WHITESPACE">--TRUNCATE_NAMES_AT_WHITESPACE </a>
			 / <small>-</small>
		</h3>
		<p class="args">
			<b>Make sequence name the first word from the > line in the fasta file.  By default the entire contents of the > line is used, excluding leading and trailing whitespace.</b><br />
			
		</p>
		<p>
		<span class="label label-info ">boolean</span>
			&nbsp;<span class="label">true</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--URI">--URI </a>
			 / <small>-UR</small>
		</h3>
		<p class="args">
			<b>Put into UR field of sequence dictionary entry.  If not supplied, input reference file is used</b><br />
			
		</p>
		<p>
		<span class="label label-info ">String</span>
			&nbsp;<span class="label">null</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--USE_JDK_DEFLATER">--USE_JDK_DEFLATER </a>
			 / <small>-use_jdk_deflater</small>
		</h3>
		<p class="args">
			<b>Use the JDK Deflater instead of the Intel Deflater for writing compressed output</b><br />
			
		</p>
		<p>
		<span class="label label-info ">Boolean</span>
			&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--USE_JDK_INFLATER">--USE_JDK_INFLATER </a>
			 / <small>-use_jdk_inflater</small>
		</h3>
		<p class="args">
			<b>Use the JDK Inflater instead of the Intel Inflater for reading compressed input</b><br />
			
		</p>
		<p>
		<span class="label label-info ">Boolean</span>
			&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--VALIDATION_STRINGENCY">--VALIDATION_STRINGENCY </a>
			 / <small>-</small>
		</h3>
		<p class="args">
			<b>Validation stringency for all SAM files read by this program.  Setting stringency to SILENT can improve performance when processing a BAM file in which variable-length data (read, qualities, tags) do not otherwise need to be decoded.</b><br />
			
		</p>
			<p>
				The --VALIDATION_STRINGENCY argument is an enumerated type (ValidationStringency), which can have one of the following values:
			<dl class="enum">
					<dt class="enum">STRICT</dt>
					<dd class="enum"></dd>
					<dt class="enum">LENIENT</dt>
					<dd class="enum"></dd>
					<dt class="enum">SILENT</dt>
					<dd class="enum"></dd>
			</dl>
			</p>
		<p>
		<span class="label label-info ">ValidationStringency</span>
			&nbsp;<span class="label">STRICT</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--VERBOSITY">--VERBOSITY </a>
			 / <small>-</small>
		</h3>
		<p class="args">
			<b>Control verbosity of logging.</b><br />
			
		</p>
			<p>
				The --VERBOSITY argument is an enumerated type (LogLevel), which can have one of the following values:
			<dl class="enum">
					<dt class="enum">ERROR</dt>
					<dd class="enum"></dd>
					<dt class="enum">WARNING</dt>
					<dd class="enum"></dd>
					<dt class="enum">INFO</dt>
					<dd class="enum"></dd>
					<dt class="enum">DEBUG</dt>
					<dd class="enum"></dd>
			</dl>
			</p>
		<p>
		<span class="label label-info ">LogLevel</span>
			&nbsp;<span class="label">INFO</span>
		</p>

        <hr>
        <p><a href='#top'><i class='fa fa-chevron-up'></i> Return to top</a></p>
        <hr>
        <p class="see-also">See also 
        	<a href="https://www.broadinstitute.org/picard/guide/">Picard Documentation Index</a> |
        	<a href="index">Tool Docs Index</a> |
        	<a href="http://gatkforums.broadinstitute.org/">Support Forum</a>
        </p>

        <p class="version">GATK version 2.10.7-2-gbfcd970-SNAPSHOT built at 05-28-2017 01:28:07.
        </p>

	</div>

	<?php printFooter($module); ?>